{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "from mmnrm.utils import set_random_seed, load_neural_model, load_model, flat_list\n",
    "from mmnrm.dataset import TestCollectionV2, sentence_splitter_builderV2, TrainSnippetsCollectionV2\n",
    "from mmnrm.evaluation import BioASQ_JavaEvaluator\n",
    "\n",
    "from collections import defaultdict\n",
    "import os\n",
    "import pickle\n",
    "import numpy as np\n",
    "import sys\n",
    "import math\n",
    "import time\n",
    "import tensorflow as tf\n",
    "from nltk.tokenize.punkt import PunktSentenceTokenizer\n",
    "from utils import *\n",
    "\n",
    "import nltk"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "# some queries to use as an example\n",
    "def build_data_generators(tokenizer, use_joint=True, queries_sw=None, docs_sw=None):\n",
    "    \n",
    "    punkt_sent_tokenizer = PunktSentenceTokenizer().span_tokenize\n",
    "\n",
    "    def maybe_tokenize(documents):\n",
    "        if \"tokens\" not in documents:\n",
    "            if use_joint:\n",
    "                split = []\n",
    "                spans = []\n",
    "                for _itter, position in enumerate(punkt_sent_tokenizer(documents[\"text\"])):\n",
    "                    start, end = position\n",
    "                    _text = documents[\"text\"][start:end]\n",
    "\n",
    "                    is_title = True\n",
    "                    if _itter>0: # fix the start and end position for the abstract\n",
    "                        \n",
    "                        if _itter == 1: # auxiliar correction to set the abstract at 0 index\n",
    "                            diff = (len(documents[\"title\"])-1)+(start-(len(documents[\"title\"])-1))\n",
    "                            \n",
    "                        start = start-diff\n",
    "                        end = end-diff\n",
    "                        is_title = False\n",
    "                    \n",
    "                    split.append(_text)\n",
    "                    spans.append({\"start\":start,\n",
    "                                  \"end\":end-1,\n",
    "                                  \"text\":_text,\n",
    "                                  \"is_title\":is_title,\n",
    "                                  \"snippet_id\":documents[\"id\"]+\"_\"+str(_itter),\n",
    "                                  \"doc_id\":documents[\"id\"]})\n",
    "                documents[\"spans\"] = spans\n",
    "            else:\n",
    "                split = nltk.sent_tokenize(documents[\"text\"])\n",
    "                \n",
    "            documents[\"tokens\"] = tokenizer.texts_to_sequences(split)\n",
    "            if docs_sw is not None:\n",
    "                for tokenized_sentence in documents[\"tokens\"]:\n",
    "                    tokenized_sentence = [token for token in tokenized_sentence if token not in docs_sw]\n",
    "\n",
    "    def test_generator(data_generator):\n",
    "        for _id, query, docs in data_generator:\n",
    "            tokenized_queries = []\n",
    "            for i in range(len(_id)):\n",
    "                # tokenization\n",
    "                tokenized_query = tokenizer.texts_to_sequences([query[i]])[0]\n",
    "\n",
    "                if queries_sw is not None:\n",
    "                    tokenized_query = [token for token in tokenized_query if token not in queries_sw] \n",
    "                \n",
    "                tokenized_queries.append(tokenized_query)\n",
    "                    \n",
    "        \n",
    "                for doc in docs[i]:\n",
    "                    maybe_tokenize(doc)\n",
    "                                                 \n",
    "            yield _id, tokenized_queries, docs\n",
    "            \n",
    "    return test_generator\n",
    "\n",
    "def get_test_generator_for_model(model, use_joint):\n",
    "\n",
    "    if \"model\" in model.savable_config:\n",
    "        cfg = model.savable_config[\"model\"]\n",
    "    \n",
    "    test_gen = build_data_generators(model.tokenizer, use_joint)\n",
    "    \n",
    "    pad_tokens = lambda x, max_len, dtype='int32': tf.keras.preprocessing.sequence.pad_sequences(x, \n",
    "                                                                                           maxlen=max_len,\n",
    "                                                                                           dtype=dtype, \n",
    "                                                                                           padding='post', \n",
    "                                                                                           truncating='post', \n",
    "                                                                                           value=0)\n",
    "\n",
    "    pad_sentences = lambda x, max_lim, dtype='int32': x[:max_lim] + [[]]*(max_lim-len(x))\n",
    "    \n",
    "    def maybe_padding(document):\n",
    "        if isinstance(document[\"tokens\"], list):\n",
    "            #overflow prevention\n",
    "            bounded_doc_passage = min(cfg[\"max_passages\"],len(document[\"tokens\"]))\n",
    "            document[\"sentences_mask\"] = [True] * bounded_doc_passage + [False] * (cfg[\"max_passages\"]-bounded_doc_passage)\n",
    "            document[\"tokens\"] = pad_tokens(pad_sentences(document[\"tokens\"], cfg[\"max_passages\"]), cfg[\"max_p_terms\"])\n",
    "            document[\"spans\"] = document[\"spans\"][:cfg[\"max_passages\"]]\n",
    "            \n",
    "    def test_generator(data_generator):\n",
    "        \n",
    "        for ids, query, docs in test_gen(data_generator):\n",
    "            \n",
    "            docs_spans = []\n",
    "            docs_ids = []\n",
    "            docs_array = []\n",
    "            docs_mask_array = []\n",
    "            query_array = []\n",
    "            query_ids = []\n",
    "            \n",
    "            for i in range(len(ids)):\n",
    "                \n",
    "                for doc in docs[i]:\n",
    "                    # pad docs, use cache here\n",
    "                    maybe_padding(doc)\n",
    "                    docs_array.append(doc[\"tokens\"])\n",
    "                    docs_mask_array.append(doc[\"sentences_mask\"])\n",
    "                    docs_ids.append(doc[\"id\"])\n",
    "                    docs_spans.append(doc[\"spans\"])\n",
    "                    \n",
    "                query_tokens = pad_tokens([query[i]], cfg[\"max_q_terms\"])[0]\n",
    "                query_tokens = [query_tokens] * len(docs[i])\n",
    "                query_array.append(query_tokens)\n",
    "                    \n",
    "                query_ids.append([ids[i]]*len(docs[i]))\n",
    "            \n",
    "            #print(np.array(docs_mask_array))\n",
    "            \n",
    "            yield flat_list(query_ids), [np.array(flat_list(query_array)), np.array(docs_array), np.array(docs_mask_array)], docs_ids, docs_spans\n",
    "            \n",
    "    return test_generator\n",
    "\n",
    "queries = load_queries(\"BioASQ-task9bPhaseA-testset3\", maps=[(\"body\",\"query\")])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "DEBUG created tokenizer bioasq_9b_RegexTokenizer\n",
      "False False\n",
      "[LOAD FROM CACHE] Load embedding matrix from /backup/BioASQ-9b/embeddings/WORD2VEC_embedding_bioasq_9b_gensim_iter_15_freq0_200_Regex_word2vec_bioasq_9b_RegexTokenizer\n",
      "Using einsum for mask bq,bps->bpqs and with embedding dim bqe,bpse->bpqs\n",
      "[EMBEDDING MATRIX SHAPE] (5322623, 200)\n"
     ]
    }
   ],
   "source": [
    "ranking_model = load_model(\"trained_models/honest-morning-60_val_collection0_doc_map@10\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [],
   "source": [
    "test_input_generator = get_test_generator_for_model(ranking_model, use_joint=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [],
   "source": [
    "def rank(model, t_collection):\n",
    "\n",
    "    generator_Y = t_collection.generator()\n",
    "                \n",
    "    q_scores = defaultdict(list)\n",
    "\n",
    "    for query_id, Y, docs_ids, docs_spans in generator_Y:\n",
    "        s_time = time.time()\n",
    "        \n",
    "        scores = model.predict(Y)\n",
    "        doc_scores = scores[0][:,0].tolist()\n",
    "        snippets_scores = scores[1].tolist()\n",
    "        \n",
    "        for i in range(len(doc_scores)):\n",
    "            \n",
    "            for j in range(len(docs_spans[i])):\n",
    "                docs_spans[i][j][\"score\"] = snippets_scores[i][j][0]\n",
    "            \n",
    "            #q_scores[query_id].extend(list(zip(docs_ids,scores)))\n",
    "            q_scores[query_id[i]].append({\"id\":docs_ids[i],\n",
    "                                          \"score\":doc_scores[i],\n",
    "                                          \"snippets\":docs_spans[i]})\n",
    "        \n",
    "        print(\"\\rEvaluation {} | time {}\".format(len(q_scores), time.time()-s_time), end=\"\\r\")\n",
    "\n",
    "    # sort the rankings\n",
    "    for query_id in q_scores.keys():\n",
    "        q_scores[query_id].sort(key=lambda x:-x[\"score\"])\n",
    "        q_scores[query_id] = q_scores[query_id]\n",
    "    \n",
    "    return q_scores\n",
    "\n",
    "def rerank_run(baseline_file, top_k):\n",
    "    run = load_document_run(baseline_file, dict_format=True)\n",
    "\n",
    "    tCollection = TestCollectionV2(queries, run)\\\n",
    "                      .batch_size(top_k)\\\n",
    "                      .set_transform_inputs_fn(test_input_generator)\n",
    "    \n",
    "    results = rank(ranking_model, tCollection)\n",
    "    \n",
    "    return results ## update the run results"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Evaluation 100 | time 0.31703734397888184\r"
     ]
    }
   ],
   "source": [
    "rerank = rerank_run(os.path.join(\"runs/rnd3\", \"bm25-baseline.run\"), 100)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [],
   "source": [
    "def snippetRank_bySort(results):\n",
    "    snippets_results = {}\n",
    "    for q in results.keys():\n",
    "        snippets_list = flat_list([x[\"snippets\"] for x in results[q]])\n",
    "        snippets_results[q] = sorted(snippets_list, key=lambda x: -x[\"score\"])\n",
    "        \n",
    "    return snippets_results\n",
    "\n",
    "def snippetRank_byThreshold(results, threshold):\n",
    "    snippets_results = {}\n",
    "    # this will follow the document order first\n",
    "    for q in results.keys():\n",
    "        snippets_results[q] = [y for y in flat_list([x[\"snippets\"] for x in results[q]]) if y[\"score\"] >= threshold]\n",
    "        \n",
    "    return snippets_results\n",
    "\n",
    "def snippetRank_byThreshold_and_TopK(results, threshold, topK):\n",
    "    snippets_results = {}\n",
    "    # this will follow the document order first\n",
    "    for q in results.keys():\n",
    "        snippets_results[q] = [y for y in flat_list([x[\"snippets\"] for x in results[q][:topK]]) if y[\"score\"] >= threshold]\n",
    "        \n",
    "    return snippets_results"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Remove /tmp/tmprt2bttm0\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "{'doc_p@10': 0.11499999999999995,\n",
       " 'doc_r@10': 0.4654141414141413,\n",
       " 'doc_f1@10': 0.16651067224828836,\n",
       " 'doc_map@10': 0.35071740362811793,\n",
       " 'doc_gmap@10': 0.0069504130851034225,\n",
       " 'snippet_p@10': 0.08505336416681805,\n",
       " 'snippet_r@10': 0.17641778927184135,\n",
       " 'snippet_f1@10': 0.10271486432212351,\n",
       " 'snippet_map@10': 0.22603522246306731,\n",
       " 'snippet_gmap@10': 0.0011442407919515175}"
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "evaluator = BioASQ_JavaEvaluator(\"BioASQ-task9bPhaseB-testset3\", \n",
    "                                 \"/home/tiagoalmeida/BioASQ-9b/BioASQEvaluator/BioASQEvaluation.jar\",\n",
    "                                 write_as_bioasq)\n",
    "\n",
    "snippets = snippetRank_bySort(rerank)\n",
    "\n",
    "run = create_document_run(queries, rerank, snippets)\n",
    "\n",
    "evaluator.evaluate(run)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 52,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'id': '6057c78994d57fd879000034',\n",
       " 'type': 'factoid',\n",
       " 'query': 'Which protein is involved in the organization and regulation of pluripotency-associated three-dimensional enhancer networks?',\n",
       " 'documents': [{'id': '31548608',\n",
       "   'score': 15.519436836242676,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 114,\n",
       "     'text': 'KLF4 is involved in the organization and regulation of pluripotency-associated three-dimensional enhancer networks.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '31548608',\n",
       "     'score': 0.7081087231636047},\n",
       "    {'start': 0,\n",
       "     'end': 259,\n",
       "     'text': 'Cell fate transitions are accompanied by global transcriptional, epigenetic and topological changes driven by transcription factors, as is exemplified by reprogramming somatic cells to pluripotent stem cells through the expression of OCT4, KLF4, SOX2 and cMYC.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31548608',\n",
       "     'score': 0.15228056907653809},\n",
       "    {'start': 261,\n",
       "     'end': 390,\n",
       "     'text': 'How transcription factors orchestrate the complex molecular changes around their target gene loci remains incompletely understood.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31548608',\n",
       "     'score': 0.05758190155029297},\n",
       "    {'start': 392,\n",
       "     'end': 669,\n",
       "     'text': 'Here, using KLF4 as a paradigm, we provide a transcription-factor-centric view of chromatin reorganization and its association with three-dimensional enhancer rewiring and transcriptional changes during the reprogramming of mouse embryonic fibroblasts to pluripotent stem cells.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31548608',\n",
       "     'score': 0.39797115325927734},\n",
       "    {'start': 671,\n",
       "     'end': 967,\n",
       "     'text': 'Inducible depletion of KLF factors in PSCs caused a genome-wide decrease in enhancer connectivity, whereas disruption of individual KLF4 binding sites within pluripotent-stem-cell-specific enhancers was sufficient to impair enhancer-promoter contacts and reduce the expression of associated genes.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31548608',\n",
       "     'score': 0.25990283489227295},\n",
       "    {'start': 969,\n",
       "     'end': 1185,\n",
       "     'text': 'Our study provides an integrative view of the complex activities of a lineage-specifying transcription factor and offers novel insights into the nature of the molecular events that follow transcription factor binding.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31548608',\n",
       "     'score': 0.05906429886817932}]},\n",
       "  {'id': '23199754',\n",
       "   'score': 11.437383651733398,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 60,\n",
       "     'text': 'Pluripotency in 3D: genome organization in pluripotent cells.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '23199754',\n",
       "     'score': 0.40288621187210083},\n",
       "    {'start': 0,\n",
       "     'end': 134,\n",
       "     'text': 'Cells face the challenge of storing two meters of DNA in the three-dimensional (3D) space of the nucleus that spans only a few microns.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '23199754',\n",
       "     'score': 0.13167187571525574},\n",
       "    {'start': 136,\n",
       "     'end': 408,\n",
       "     'text': 'The nuclear organization that is required to overcome this challenge must allow for the accessibility of the gene regulatory machinery to the DNA and, in the case of embryonic stem cells (ESCs), for the transcriptional and epigenetic changes that accompany differentiation.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '23199754',\n",
       "     'score': 0.2666993737220764},\n",
       "    {'start': 410,\n",
       "     'end': 532,\n",
       "     'text': 'Recent technological advances have allowed for the mapping of genome organization at an unprecedented resolution and scale.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '23199754',\n",
       "     'score': 0.11587250232696533},\n",
       "    {'start': 534,\n",
       "     'end': 713,\n",
       "     'text': 'These breakthroughs have led to a deluge of new data, and a sophisticated understanding of the relationship between gene regulation and 3D genome organization is beginning to form.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '23199754',\n",
       "     'score': 0.1825897991657257},\n",
       "    {'start': 715,\n",
       "     'end': 1076,\n",
       "     'text': 'In this review we summarize some of the recent findings illuminating the 3D structure of the eukaryotic genome, as well as the relationship between genome topology and function from the level of whole chromosomes to enhancer-promoter loops with a focus on features affecting genome organization in ESCs and changes in nuclear organization during differentiation.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '23199754',\n",
       "     'score': 0.49772197008132935}]},\n",
       "  {'id': '32553182',\n",
       "   'score': 11.311578750610352,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 108,\n",
       "     'text': 'Genomic Rewiring of SOX2 Chromatin Interaction Network during Differentiation of ESCs to Postmitotic Neurons.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '32553182',\n",
       "     'score': 0.29788559675216675},\n",
       "    {'start': 0,\n",
       "     'end': 128,\n",
       "     'text': 'Cellular differentiation requires dramatic changes in chromatin organization, transcriptional regulation, and protein production.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '32553182',\n",
       "     'score': 0.14769163727760315},\n",
       "    {'start': 130,\n",
       "     'end': 467,\n",
       "     'text': 'To understand the regulatory connections between these processes, we generated proteomic, transcriptomic, and chromatin accessibility data during differentiation of mouse embryonic stem cells (ESCs) into postmitotic neurons and found extensive associations between different molecular layers within and across differentiation time points.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '32553182',\n",
       "     'score': 0.2127506136894226},\n",
       "    {'start': 469,\n",
       "     'end': 690,\n",
       "     'text': 'We observed that SOX2, as a regulator of pluripotency and neuronal genes, redistributes from pluripotency enhancers to neuronal promoters during differentiation, likely driven by changes in its protein interaction network.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '32553182',\n",
       "     'score': 0.5031372904777527},\n",
       "    {'start': 692,\n",
       "     'end': 912,\n",
       "     'text': 'We identified ATRX as a major SOX2 partner in neurons, whose co-localization correlated with an increase in active enhancer marks and increased expression of nearby genes, which we experimentally confirmed for three loci.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '32553182',\n",
       "     'score': 0.15763896703720093},\n",
       "    {'start': 914,\n",
       "     'end': 1144,\n",
       "     'text': 'Collectively, our data provide key insights into the regulatory transformation of SOX2 during neuronal differentiation, and we highlight the significance of multi-omic approaches in understanding gene regulation in complex systems.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '32553182',\n",
       "     'score': 0.10985422134399414}]},\n",
       "  {'id': '27471257',\n",
       "   'score': 11.235023498535156,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 128,\n",
       "     'text': 'Sall4 controls differentiation of pluripotent cells independently of the Nucleosome Remodelling and Deacetylation (NuRD) complex.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '27471257',\n",
       "     'score': 0.24639463424682617},\n",
       "    {'start': 0,\n",
       "     'end': 116,\n",
       "     'text': 'Sall4 is an essential transcription factor for early mammalian development and is frequently overexpressed in cancer.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '27471257',\n",
       "     'score': 0.0972265899181366},\n",
       "    {'start': 118,\n",
       "     'end': 275,\n",
       "     'text': 'Although it is reported to play an important role in embryonic stem cell (ESC) self-renewal, whether it is an essential pluripotency factor has been disputed.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '27471257',\n",
       "     'score': 0.21590715646743774},\n",
       "    {'start': 277,\n",
       "     'end': 343,\n",
       "     'text': 'Here, we show that Sall4 is dispensable for mouse ESC pluripotency.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '27471257',\n",
       "     'score': 0.23391544818878174},\n",
       "    {'start': 345,\n",
       "     'end': 545,\n",
       "     'text': 'Sall4 is an enhancer-binding protein that prevents precocious activation of the neural gene expression programme in ESCs but is not required for maintenance of the pluripotency gene regulatory network.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '27471257',\n",
       "     'score': 0.4841834604740143},\n",
       "    {'start': 547,\n",
       "     'end': 824,\n",
       "     'text': 'Although a proportion of Sall4 protein physically associates with the Nucleosome Remodelling and Deacetylase (NuRD) complex, Sall4 neither recruits NuRD to chromatin nor influences transcription via NuRD; rather, free Sall4 protein regulates transcription independently of NuRD.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '27471257',\n",
       "     'score': 0.10221952199935913},\n",
       "    {'start': 826,\n",
       "     'end': 1033,\n",
       "     'text': 'We propose a model whereby enhancer binding by Sall4 and other pluripotency-associated transcription factors is responsible for maintaining the balance between transcriptional programmes in pluripotent cells.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '27471257',\n",
       "     'score': 0.44995638728141785}]},\n",
       "  {'id': '29372665',\n",
       "   'score': 10.940218925476074,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 188,\n",
       "     'text': 'DNA methylation of the Oct4A enhancers in embryonal carcinoma cells after etoposide treatment is associated with alternative splicing and altered pluripotency in reversibly senescent cells.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '29372665',\n",
       "     'score': 0.36239680647850037},\n",
       "    {'start': 0,\n",
       "     'end': 140,\n",
       "     'text': 'The epigenetic mechanisms underlying chemoresistance in cancer cells resulting from drug-induced reversible senescence are poorly understood.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '29372665',\n",
       "     'score': 0.09827521443367004},\n",
       "    {'start': 142,\n",
       "     'end': 394,\n",
       "     'text': 'Chemoresistant ESC-like embryonal carcinoma PA1 cells treated with etoposide (ETO) were previously found to undergo prolonged G2 arrest with transient p53-dependent upregulation of opposing fate regulators, p21CIP1 (senescence) and OCT4A (self-renewal).',\n",
       "     'is_title': False,\n",
       "     'doc_id': '29372665',\n",
       "     'score': 0.09764865040779114},\n",
       "    {'start': 396,\n",
       "     'end': 554,\n",
       "     'text': 'Here we report on the analysis of the DNA methylation state of the distal enhancer (DE) and proximal enhancer (PE) of the Oct4A gene during this dual response.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '29372665',\n",
       "     'score': 0.15672793984413147},\n",
       "    {'start': 556,\n",
       "     'end': 696,\n",
       "     'text': 'When compared to non-treated controls the methylation level increased from 1.3% to 12.5% and from 3% to 19.4%, in the DE and PE respectively.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '29372665',\n",
       "     'score': 0.09951764345169067},\n",
       "    {'start': 698,\n",
       "     'end': 804,\n",
       "     'text': 'It included CpG and non-CpG methylation, which was not chaotic but presented two patterns in each enhancer.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '29372665',\n",
       "     'score': 0.16123738884925842},\n",
       "    {'start': 806,\n",
       "     'end': 1071,\n",
       "     'text': 'Discorrelating with methylation of enhancers, the transcription of Oct4A increased, however, a strong expression of the splicing form Oct4B was also induced, along with down-regulation of the Oct4A partners of in the pluripotency/self-renewal network Sox2 and Lin28.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '29372665',\n",
       "     'score': 0.4819476008415222},\n",
       "    {'start': 1073,\n",
       "     'end': 1154,\n",
       "     'text': 'WB demonstrated disjoining of the OCT4A protein from the chromatin-bound fraction.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '29372665',\n",
       "     'score': 0.10324835777282715},\n",
       "    {'start': 1156,\n",
       "     'end': 1285,\n",
       "     'text': 'In survival clones, methylation of the DE was considerably erased, while some remnant of methylation of the PE was still observed.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '29372665',\n",
       "     'score': 0.09949445724487305},\n",
       "    {'start': 1287,\n",
       "     'end': 1473,\n",
       "     'text': 'The alternative splicing for Oct4B was reduced, Oct4A level insignificantly decreased, while the expression of Sox2 and Lin28 recovered, all three became proportionally above the control.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '29372665',\n",
       "     'score': 0.10344123840332031},\n",
       "    {'start': 1475,\n",
       "     'end': 1739,\n",
       "     'text': 'These findings indicate the involvement of the transient patterned methylation of the Oct4A enhancers and alternative splicing in the adaptive regulation of cell fate choice during the p53-dependant dual state of reversible senescence in ESC-like cancer stem cells.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '29372665',\n",
       "     'score': 0.18598923087120056}]},\n",
       "  {'id': '26765775',\n",
       "   'score': 10.674539566040039,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 74,\n",
       "     'text': 'Role of ALKBH1 in the Core Transcriptional Network of Embryonic Stem Cells.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '26765775',\n",
       "     'score': 0.1245814859867096},\n",
       "    {'start': 0,\n",
       "     'end': 171,\n",
       "     'text': 'BACKGROUND/AIMS\\nALKBH1, an AlkB homologue in the 2-oxoglutarate and Fe2+ dependent hydroxylase family, is a histone dioxygenase that removes methyl groups from histone H2A.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '26765775',\n",
       "     'score': 0.10476219654083252},\n",
       "    {'start': 173,\n",
       "     'end': 282,\n",
       "     'text': 'Studies of transgenic mice lacking Alkbh1 reveal that most Alkbh1-/- embryos die during embryonic development.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '26765775',\n",
       "     'score': 0.09883207082748413},\n",
       "    {'start': 284,\n",
       "     'end': 537,\n",
       "     'text': 'Embryonic stem cells (ESCs) derived from these mice have prolonged expression of pluripotency markers and delayed induction of genes involved in neural differentiation, indicating that ALKBH1 is involved in regulation of pluripotency and differentiation.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '26765775',\n",
       "     'score': 0.3430096507072449},\n",
       "    {'start': 539,\n",
       "     'end': 624,\n",
       "     'text': 'The aim of this study was to further investigate the role ALKBH1 in early development.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '26765775',\n",
       "     'score': 0.10196158289909363},\n",
       "    {'start': 628,\n",
       "     'end': 839,\n",
       "     'text': 'METHODS\\nDouble-filter methods for nitrocellulose-filter binding, dot blot, enzyme-linked immunosorbent assay (ELISA), immonocytochemistry, cell culture and differentiation of mouse ESCs, Co-IP and miRNA analysis.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '26765775',\n",
       "     'score': 0.2339635193347931},\n",
       "    {'start': 843,\n",
       "     'end': 1032,\n",
       "     'text': 'RESULTS\\nWe found that SOX2 and NANOG bind the ALKBH1 promoter, and we identified protein-protein interactions between ALKBH1 and these core transcription factors of the pluripotency network.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '26765775',\n",
       "     'score': 0.4504778981208801},\n",
       "    {'start': 1034,\n",
       "     'end': 1201,\n",
       "     'text': 'Furthermore, lack of ALKBH1 affected the expression of developmentally important miRNAs, which are involved in the regulation of NANOG, SOX2 and neural differentiation.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '26765775',\n",
       "     'score': 0.31529873609542847},\n",
       "    {'start': 1205,\n",
       "     'end': 1382,\n",
       "     'text': 'CONCLUSION\\nOur results suggest that ALKBH1 interacts with the core transcriptional pluripotency network of ESCs and is involved in regulation of pluripotency and differentiation.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '26765775',\n",
       "     'score': 0.4017235338687897}]},\n",
       "  {'id': '24777452',\n",
       "   'score': 10.655994415283203,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 128,\n",
       "     'text': 'Deep transcriptome profiling of mammalian stem cells supports a regulatory role for retrotransposons in pluripotency maintenance.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '24777452',\n",
       "     'score': 0.24576252698898315},\n",
       "    {'start': 0,\n",
       "     'end': 190,\n",
       "     'text': 'The importance of microRNAs and long noncoding RNAs in the regulation of pluripotency has been documented; however, the noncoding components of stem cell gene networks remain largely unknown.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24777452',\n",
       "     'score': 0.38223984837532043},\n",
       "    {'start': 192,\n",
       "     'end': 340,\n",
       "     'text': 'Here we investigate the role of noncoding RNAs in the pluripotent state, with particular emphasis on nuclear and retrotransposon-derived transcripts.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24777452',\n",
       "     'score': 0.25615397095680237},\n",
       "    {'start': 342,\n",
       "     'end': 527,\n",
       "     'text': 'We have performed deep profiling of the nuclear and cytoplasmic transcriptomes of human and mouse stem cells, identifying a class of previously undetected stem cell-specific transcripts.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24777452',\n",
       "     'score': 0.10201424360275269},\n",
       "    {'start': 529,\n",
       "     'end': 668,\n",
       "     'text': 'We show that long terminal repeat (LTR)-derived transcripts contribute extensively to the complexity of the stem cell nuclear transcriptome.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24777452',\n",
       "     'score': 0.10034841299057007},\n",
       "    {'start': 670,\n",
       "     'end': 800,\n",
       "     'text': 'Some LTR-derived transcripts are associated with enhancer regions and are likely to be involved in the maintenance of pluripotency.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24777452',\n",
       "     'score': 0.45888859033584595}]},\n",
       "  {'id': '31221664',\n",
       "   'score': 10.504301071166992,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 121,\n",
       "     'text': 'KLF4 protein stability regulated by interaction with pluripotency transcription factors overrides transcriptional control.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '31221664',\n",
       "     'score': 0.2726683020591736},\n",
       "    {'start': 0,\n",
       "     'end': 113,\n",
       "     'text': 'Embryonic stem (ES) cells are regulated by a network of transcription factors that maintain the pluripotent state.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31221664',\n",
       "     'score': 0.2761201858520508},\n",
       "    {'start': 115,\n",
       "     'end': 218,\n",
       "     'text': 'Differentiation relies on down-regulation of pluripotency transcription factors disrupting this network.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31221664',\n",
       "     'score': 0.3756549060344696},\n",
       "    {'start': 220,\n",
       "     'end': 597,\n",
       "     'text': 'While investigating transcriptional regulation of the pluripotency transcription factor Kruppel-like factor 4 (Klf4), we observed that homozygous deletion of distal enhancers caused a 17-fold decrease in Klf4 transcript but surprisingly decreased protein levels by less than twofold, indicating that posttranscriptional control of KLF4 protein overrides transcriptional control.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31221664',\n",
       "     'score': 0.4509628713130951},\n",
       "    {'start': 599,\n",
       "     'end': 698,\n",
       "     'text': 'The lack of sensitivity of KLF4 to transcription is due to high protein stability (half-life >24 h).',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31221664',\n",
       "     'score': 0.11053293943405151},\n",
       "    {'start': 700,\n",
       "     'end': 1020,\n",
       "     'text': 'This stability is context-dependent and is disrupted during differentiation, as evidenced by a shift to a half-life of <2 h. KLF4 protein stability is maintained through interaction with other pluripotency transcription factors (NANOG, SOX2, and STAT3) that together facilitate association of KLF4 with RNA polymerase II.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31221664',\n",
       "     'score': 0.268865168094635},\n",
       "    {'start': 1022,\n",
       "     'end': 1131,\n",
       "     'text': 'In addition, the KLF4 DNA-binding and transactivation domains are required for optimal KLF4 protein stability.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31221664',\n",
       "     'score': 0.11121487617492676},\n",
       "    {'start': 1133,\n",
       "     'end': 1310,\n",
       "     'text': 'Posttranslational modification of KLF4 destabilizes the protein as cells exit the pluripotent state, and mutations that prevent this destabilization also prevent differentiation.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31221664',\n",
       "     'score': 0.2682322859764099},\n",
       "    {'start': 1312,\n",
       "     'end': 1569,\n",
       "     'text': 'These data indicate that the core pluripotency transcription factors are integrated by posttranslational mechanisms to maintain the pluripotent state and identify mutations that increase KLF4 protein stability while maintaining transcription factor function.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31221664',\n",
       "     'score': 0.23399245738983154}]},\n",
       "  {'id': '30266927',\n",
       "   'score': 10.469011306762695,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 171,\n",
       "     'text': 'Three-Dimensional Retinal Organoids Facilitate the Investigation of Retinal Ganglion Cell Development, Organization and Neurite Outgrowth from Human Pluripotent Stem Cells.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '30266927',\n",
       "     'score': 0.46932312846183777},\n",
       "    {'start': 0,\n",
       "     'end': 235,\n",
       "     'text': 'Retinal organoids are three-dimensional structures derived from human pluripotent stem cells (hPSCs) which recapitulate the spatial and temporal differentiation of the retina, serving as effective in vitro models of retinal development.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '30266927',\n",
       "     'score': 0.31366026401519775},\n",
       "    {'start': 237,\n",
       "     'end': 376,\n",
       "     'text': 'However, a lack of emphasis has been placed upon the development and organization of retinal ganglion cells (RGCs) within retinal organoids.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '30266927',\n",
       "     'score': 0.13068628311157227},\n",
       "    {'start': 378,\n",
       "     'end': 623,\n",
       "     'text': 'Thus, initial efforts were made to characterize RGC differentiation throughout early stages of organoid development, with a clearly defined RGC layer developing in a temporally-appropriate manner expressing a complement of RGC-associated markers.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '30266927',\n",
       "     'score': 0.10587576031684875},\n",
       "    {'start': 625,\n",
       "     'end': 804,\n",
       "     'text': 'Beyond studies of RGC development, retinal organoids may also prove useful for cellular replacement in which extensive axonal outgrowth is necessary to reach post-synaptic targets.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '30266927',\n",
       "     'score': 0.10580062866210938},\n",
       "    {'start': 806,\n",
       "     'end': 973,\n",
       "     'text': 'Organoid-derived RGCs could help to elucidate factors promoting axonal outgrowth, thereby identifying approaches to circumvent a formidable obstacle to RGC replacement.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '30266927',\n",
       "     'score': 0.10016009211540222},\n",
       "    {'start': 975,\n",
       "     'end': 1137,\n",
       "     'text': 'As such, additional efforts demonstrated significant enhancement of neurite outgrowth through modulation of both substrate composition and growth factor signaling.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '30266927',\n",
       "     'score': 0.10293316841125488},\n",
       "    {'start': 1139,\n",
       "     'end': 1280,\n",
       "     'text': 'Additionally, organoid-derived RGCs exhibited diverse phenotypes, extending elaborate growth cones and expressing numerous guidance receptors.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '30266927',\n",
       "     'score': 0.10172152519226074},\n",
       "    {'start': 1282,\n",
       "     'end': 1540,\n",
       "     'text': 'Collectively, these results establish retinal organoids as a valuable tool for studies of RGC development, and demonstrate the utility of organoid-derived RGCs as an effective platform to study factors influencing neurite outgrowth from organoid-derived RGCs.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '30266927',\n",
       "     'score': 0.1039593517780304}]},\n",
       "  {'id': '30863449',\n",
       "   'score': 10.369046211242676,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 137,\n",
       "     'text': 'The Role of RNA Polymerase II Contiguity and Long-Range Interactions in the Regulation of Gene Expression in Human Pluripotent Stem Cells.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '30863449',\n",
       "     'score': 0.30869418382644653},\n",
       "    {'start': 0,\n",
       "     'end': 189,\n",
       "     'text': 'The eukaryotic nucleus is a highly complex structure that carries out multiple functions primarily needed for gene expression, and among them, transcription seems to be the most fundamental.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '30863449',\n",
       "     'score': 0.10597193241119385},\n",
       "    {'start': 191,\n",
       "     'end': 412,\n",
       "     'text': 'Diverse approaches have demonstrated that transcription takes place at discrete sites known as transcription factories, wherein RNA polymerase II (RNAP II) is attached to the factory and immobilized while transcribing DNA.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '30863449',\n",
       "     'score': 0.10560482740402222},\n",
       "    {'start': 414,\n",
       "     'end': 598,\n",
       "     'text': 'It has been proposed that transcription factories promote chromatin loop formation, creating long-range interactions in which relatively distant genes can be transcribed simultaneously.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '30863449',\n",
       "     'score': 0.10410195589065552},\n",
       "    {'start': 600,\n",
       "     'end': 909,\n",
       "     'text': 'In this study, we examined long-range interactions between the POU5F1 gene and genes previously identified as being POU5F1 enhancer-interacting, namely, CDYL, TLE2, RARG, and MSX1 (all involved in transcriptional regulation), in human pluripotent stem cells (hPSCs) and their early differentiated counterparts.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '30863449',\n",
       "     'score': 0.4484449028968811},\n",
       "    {'start': 911,\n",
       "     'end': 1038,\n",
       "     'text': 'As a control gene, RUNX1 was used, which is expressed during hematopoietic differentiation and not associated with pluripotency.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '30863449',\n",
       "     'score': 0.26265716552734375},\n",
       "    {'start': 1040,\n",
       "     'end': 1313,\n",
       "     'text': 'To reveal how these long-range interactions between POU5F1 and the selected genes change with the onset of differentiation and upon RNAP II inhibition, we performed three-dimensional fluorescence in situ hybridization (3D-FISH) followed by computational simulation analysis.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '30863449',\n",
       "     'score': 0.14999845623970032},\n",
       "    {'start': 1315,\n",
       "     'end': 1519,\n",
       "     'text': 'Our analysis showed that the numbers of long-range interactions between specific genes decrease during differentiation, suggesting that the transcription of monitored genes is associated with pluripotency.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '30863449',\n",
       "     'score': 0.25966376066207886},\n",
       "    {'start': 1521,\n",
       "     'end': 1640,\n",
       "     'text': 'In addition, we showed that upon inhibition of RNAP II, long-range associations do not disintegrate and remain constant.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '30863449',\n",
       "     'score': 0.10640057921409607},\n",
       "    {'start': 1642,\n",
       "     'end': 1833,\n",
       "     'text': 'We also analyzed the distance distributions of these genes in the context of their positions in the nucleus and revealed that they tend to have similar patterns resembling normal distribution.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '30863449',\n",
       "     'score': 0.10745862126350403},\n",
       "    {'start': 1835,\n",
       "     'end': 1945,\n",
       "     'text': 'Furthermore, we compared data created in vitro and in silico to assess the biological relevance of our results.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '30863449',\n",
       "     'score': 0.10668230056762695}]},\n",
       "  {'id': '32562758',\n",
       "   'score': 10.34384536743164,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 49,\n",
       "     'text': 'DNA methylation and the core pluripotency network.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '32562758',\n",
       "     'score': 0.28680557012557983},\n",
       "    {'start': 0,\n",
       "     'end': 87,\n",
       "     'text': 'From the onset of fertilization, the genome undergoes cell division and differentiation.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '32562758',\n",
       "     'score': 0.10561397671699524},\n",
       "    {'start': 89,\n",
       "     'end': 227,\n",
       "     'text': 'All of these developmental transitions and differentiation processes include cell-specific signatures and gradual changes of the epigenome.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '32562758',\n",
       "     'score': 0.10598498582839966},\n",
       "    {'start': 229,\n",
       "     'end': 380,\n",
       "     'text': 'Understanding what keeps stem cells in the pluripotent state and what leads to differentiation are fascinating and biomedically highly important issues.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '32562758',\n",
       "     'score': 0.25422459840774536},\n",
       "    {'start': 382,\n",
       "     'end': 490,\n",
       "     'text': 'Numerous studies have identified genes, proteins, microRNAs and small molecules that exert essential effects.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '32562758',\n",
       "     'score': 0.10321137309074402},\n",
       "    {'start': 492,\n",
       "     'end': 611,\n",
       "     'text': 'Notably, there exists a core pluripotency network that consists of several transcription factors and accessory proteins.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '32562758',\n",
       "     'score': 0.2775859832763672},\n",
       "    {'start': 613,\n",
       "     'end': 719,\n",
       "     'text': 'Three eminent transcription factors, OCT4, SOX2 and NANOG, serve as hubs in this core pluripotency network.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '32562758',\n",
       "     'score': 0.32389211654663086},\n",
       "    {'start': 721,\n",
       "     'end': 924,\n",
       "     'text': 'They bind to the enhancer regions of their target genes and modulate, among others, the expression levels of genes that are associated with Gene Ontology terms related to differentiation and self-renewal.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '32562758',\n",
       "     'score': 0.17643365263938904},\n",
       "    {'start': 926,\n",
       "     'end': 1027,\n",
       "     'text': 'Also, much has been learned about the epigenetic rewiring processes during these changes of cell fate.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '32562758',\n",
       "     'score': 0.10387203097343445},\n",
       "    {'start': 1029,\n",
       "     'end': 1106,\n",
       "     'text': 'For example, DNA methylation dynamics is pivotal during embryonic development.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '32562758',\n",
       "     'score': 0.10323336720466614},\n",
       "    {'start': 1108,\n",
       "     'end': 1449,\n",
       "     'text': 'The main goal of this review is to highlight an intricate interplay of (a) DNA methyltransferases controlling the expression levels of core pluripotency factors by modulation of the DNA methylation levels in their enhancer regions, and of (b) the core pluripotency factors controlling the transcriptional regulation of DNA methyltransferases.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '32562758',\n",
       "     'score': 0.4437042474746704},\n",
       "    {'start': 1451,\n",
       "     'end': 1601,\n",
       "     'text': 'We discuss these processes both at the global level and in atomistic detail based on information from structural studies and from computer simulations.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '32562758',\n",
       "     'score': 0.10622531175613403}]},\n",
       "  {'id': '29514091',\n",
       "   'score': 10.248522758483887,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 130,\n",
       "     'text': 'Long-Range Enhancer Interactions Are Prevalent in Mouse Embryonic Stem Cells and Are Reorganized upon Pluripotent State Transition.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '29514091',\n",
       "     'score': 0.3677775263786316},\n",
       "    {'start': 0,\n",
       "     'end': 148,\n",
       "     'text': 'Transcriptional enhancers, including super-enhancers (SEs), form physical interactions with promoters to regulate cell-type-specific gene expression.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '29514091',\n",
       "     'score': 0.16429558396339417},\n",
       "    {'start': 150,\n",
       "     'end': 332,\n",
       "     'text': 'SEs are characterized by high transcription factor occupancy and large domains of active chromatin, and they are commonly assigned to target promoters using computational predictions.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '29514091',\n",
       "     'score': 0.10517621040344238},\n",
       "    {'start': 334,\n",
       "     'end': 481,\n",
       "     'text': 'How promoter-SE interactions change upon cell state transitions, and whether transcription factors maintain SE interactions, have not been reported.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '29514091',\n",
       "     'score': 0.16661447286605835},\n",
       "    {'start': 483,\n",
       "     'end': 600,\n",
       "     'text': 'Here, we used promoter-capture Hi-C to identify promoters that interact with SEs in mouse embryonic stem cells (ESCs).',\n",
       "     'is_title': False,\n",
       "     'doc_id': '29514091',\n",
       "     'score': 0.32012271881103516},\n",
       "    {'start': 602,\n",
       "     'end': 779,\n",
       "     'text': 'We found that SEs form complex, spatial networks in which individual SEs contact multiple promoters, and a rewiring of promoter-SE interactions occurs between pluripotent states.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '29514091',\n",
       "     'score': 0.4290946424007416},\n",
       "    {'start': 781,\n",
       "     'end': 922,\n",
       "     'text': 'We also show that long-range promoter-SE interactions are more prevalent in ESCs than in epiblast stem cells (EpiSCs) or Nanog-deficient ESCs.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '29514091',\n",
       "     'score': 0.3685828745365143},\n",
       "    {'start': 924,\n",
       "     'end': 1132,\n",
       "     'text': 'We conclude that SEs form cell-type-specific interaction networks that are partly dependent on core transcription factors, thereby providing insights into the gene regulatory organization of pluripotent cells.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '29514091',\n",
       "     'score': 0.3570590019226074}]},\n",
       "  {'id': '28147265',\n",
       "   'score': 10.137882232666016,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 81,\n",
       "     'text': 'Gender Differences in Global but Not Targeted Demethylation in iPSC Reprogramming.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '28147265',\n",
       "     'score': 0.10552334785461426},\n",
       "    {'start': 0,\n",
       "     'end': 188,\n",
       "     'text': 'Global DNA demethylation is an integral part of reprogramming processes in\\xa0vivo and in\\xa0vitro, but whether it occurs in the derivation of induced pluripotent stem cells (iPSCs) is not known.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28147265',\n",
       "     'score': 0.2805059552192688},\n",
       "    {'start': 190,\n",
       "     'end': 345,\n",
       "     'text': 'Here, we show that iPSC reprogramming involves both global and targeted demethylation, which are separable mechanistically and by their biological outcomes.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28147265',\n",
       "     'score': 0.10664835572242737},\n",
       "    {'start': 347,\n",
       "     'end': 483,\n",
       "     'text': 'Cells at intermediate-late stages of reprogramming undergo transient genome-wide demethylation, which is more pronounced in female cells.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28147265',\n",
       "     'score': 0.11006316542625427},\n",
       "    {'start': 485,\n",
       "     'end': 692,\n",
       "     'text': 'Global demethylation requires activation-induced cytidine deaminase (AID)-mediated downregulation of UHRF1 protein, and abolishing demethylation leaves thousands of hypermethylated regions in the iPSC genome.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28147265',\n",
       "     'score': 0.1180504858493805},\n",
       "    {'start': 694,\n",
       "     'end': 916,\n",
       "     'text': 'Independently of AID and global demethylation, regulatory regions, particularly ESC enhancers and super-enhancers, are specifically targeted for hypomethylation in association with transcription of the pluripotency network.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28147265',\n",
       "     'score': 0.4567837417125702},\n",
       "    {'start': 918,\n",
       "     'end': 1145,\n",
       "     'text': 'Our results show that global and targeted DNA demethylation are conserved and\\xa0distinct reprogramming processes, presumably because of their respective roles in epigenetic memory erasure and in the establishment of cell identity.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28147265',\n",
       "     'score': 0.11022317409515381}]},\n",
       "  {'id': '29180618',\n",
       "   'score': 10.04347038269043,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 73,\n",
       "     'text': 'A transcribed enhancer dictates mesendoderm specification in pluripotency.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '29180618',\n",
       "     'score': 0.40541893243789673},\n",
       "    {'start': 0,\n",
       "     'end': 108,\n",
       "     'text': 'Enhancers and long noncoding RNAs (lncRNAs) are key determinants of lineage specification during development.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '29180618',\n",
       "     'score': 0.1812184453010559},\n",
       "    {'start': 110,\n",
       "     'end': 240,\n",
       "     'text': 'Here, we evaluate remodeling of the enhancer landscape and modulation of the lncRNA transcriptome during mesendoderm specification.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '29180618',\n",
       "     'score': 0.18145865201950073},\n",
       "    {'start': 242,\n",
       "     'end': 485,\n",
       "     'text': 'We sort mesendodermal progenitors from differentiating embryonic stem cells (ESCs) according to Eomes expression, and find that enhancer usage is coordinated with mesendoderm-specific expression of key lineage-determining transcription factors.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '29180618',\n",
       "     'score': 0.3563392162322998},\n",
       "    {'start': 487,\n",
       "     'end': 556,\n",
       "     'text': 'Many of these enhancers are associated with the expression of lncRNAs.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '29180618',\n",
       "     'score': 0.18528583645820618},\n",
       "    {'start': 558,\n",
       "     'end': 763,\n",
       "     'text': 'Examination of ESC-specific enhancers interacting in three-dimensional space with mesendoderm-specifying transcription factor loci identifies MesEndoderm Transcriptional Enhancer Organizing Region (Meteor).',\n",
       "     'is_title': False,\n",
       "     'doc_id': '29180618',\n",
       "     'score': 0.26574966311454773},\n",
       "    {'start': 765,\n",
       "     'end': 985,\n",
       "     'text': 'Genetic and epigenetic manipulation of the Meteor enhancer reveal its indispensable role during mesendoderm specification and subsequent cardiogenic differentiation via transcription-independent and -dependent mechanisms.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '29180618',\n",
       "     'score': 0.1795782446861267},\n",
       "    {'start': 987,\n",
       "     'end': 1084,\n",
       "     'text': 'Interestingly, Meteor-deleted ESCs are epigenetically redirected towards neuroectodermal lineages.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '29180618',\n",
       "     'score': 0.2370816469192505},\n",
       "    {'start': 1086,\n",
       "     'end': 1288,\n",
       "     'text': 'Loci, topologically associating a transcribed enhancer and its cognate protein coding gene, appear to represent therefore a class of genomic elements controlling developmental competence in pluripotency.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '29180618',\n",
       "     'score': 0.4206128716468811}]},\n",
       "  {'id': '23549118',\n",
       "   'score': 9.938804626464844,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 75,\n",
       "     'text': 'The interactomes of POU5F1 and SOX2 enhancers in human embryonic stem cells.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '23549118',\n",
       "     'score': 0.18976709246635437},\n",
       "    {'start': 0,\n",
       "     'end': 152,\n",
       "     'text': 'The genes POU5F1 and SOX2 are critical for pluripotency and reprogramming, yet the chromosomal organization around these genes remains poorly understood.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '23549118',\n",
       "     'score': 0.31996476650238037},\n",
       "    {'start': 154,\n",
       "     'end': 308,\n",
       "     'text': 'We assayed long-range chromosomal interactions on putative enhancers of POU5F1 and SOX2 genes in human embryonic stem cells (hESCs) using 4C-Seq technique.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '23549118',\n",
       "     'score': 0.39866986870765686},\n",
       "    {'start': 310,\n",
       "     'end': 413,\n",
       "     'text': 'We discovered that their frequent interacting regions mainly overlap with early DNA replication domains.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '23549118',\n",
       "     'score': 0.10612431168556213},\n",
       "    {'start': 415,\n",
       "     'end': 520,\n",
       "     'text': 'The interactomes are associated with active histone marks and enriched with 5-hydroxymethylcytosine sites.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '23549118',\n",
       "     'score': 0.11248305439949036},\n",
       "    {'start': 522,\n",
       "     'end': 586,\n",
       "     'text': 'In hESCs, genes within the interactomes have elevated expression.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '23549118',\n",
       "     'score': 0.25428152084350586},\n",
       "    {'start': 588,\n",
       "     'end': 675,\n",
       "     'text': 'Additionally, some genes associated with the POU5F1 enhancer contribute to pluripotency.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '23549118',\n",
       "     'score': 0.41570454835891724},\n",
       "    {'start': 677,\n",
       "     'end': 816,\n",
       "     'text': 'Binding sites for multiple DNA binding proteins, including ATF3, CTCF, GABPA, JUND, NANOG, RAD21 and YY1, are enriched in both interactomes.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '23549118',\n",
       "     'score': 0.25257909297943115},\n",
       "    {'start': 818,\n",
       "     'end': 958,\n",
       "     'text': 'The RARG locus, frequently interacting with the POU5F1 locus, has abundant RAD21 binding sites co-localized with other protein binding sites.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '23549118',\n",
       "     'score': 0.11369895935058594},\n",
       "    {'start': 960,\n",
       "     'end': 1075,\n",
       "     'text': 'Thus the interactomes of these two pluripotency genes could be an important part of the regulatory network in hESCs.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '23549118',\n",
       "     'score': 0.3358808755874634}]},\n",
       "  {'id': '24648406',\n",
       "   'score': 9.893735885620117,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 121,\n",
       "     'text': 'Coactivators p300 and CBP maintain the identity of mouse embryonic stem cells by mediating long-range chromatin structure.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '24648406',\n",
       "     'score': 0.11095663905143738},\n",
       "    {'start': 0,\n",
       "     'end': 189,\n",
       "     'text': 'Master transcription factors Oct4, Sox2, and Nanog are required to maintain the pluripotency and self-renewal of embryonic stem cells (ESCs) by regulating a specific transcriptional network.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24648406',\n",
       "     'score': 0.3523593842983246},\n",
       "    {'start': 191,\n",
       "     'end': 440,\n",
       "     'text': 'A few other transcription factors have been shown to be important in ESCs by interacting with these master transcription factors; however, little is known about the transcriptional mechanisms regulated by coregulators (coactivators and corepressors).',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24648406',\n",
       "     'score': 0.25966501235961914},\n",
       "    {'start': 442,\n",
       "     'end': 565,\n",
       "     'text': 'In this study, we examined the function of two highly homologous coactivators, p300 and CREB-binding protein (CBP), in ESCs.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24648406',\n",
       "     'score': 0.3012901842594147},\n",
       "    {'start': 567,\n",
       "     'end': 673,\n",
       "     'text': 'We find that these two coactivators play redundant roles in maintaining the undifferentiated state of ESCs.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24648406',\n",
       "     'score': 0.2795216143131256},\n",
       "    {'start': 675,\n",
       "     'end': 882,\n",
       "     'text': 'They are recruited by Nanog through physical interaction to Nanog binding loci, mediating the formation of long-range chromatin looping structures, which is essential to maintain ESC-specific gene expression.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24648406',\n",
       "     'score': 0.24124464392662048},\n",
       "    {'start': 884,\n",
       "     'end': 1219,\n",
       "     'text': 'Further functional studies reveal that the p300/CBP binding looping fragments contain enhancer activities, suggesting that the formation of p300/CBP-mediated looping structures may recruit distal enhancers to create a concentration of factors for the transcription activation of genes that are involved in self-renewal and pluripotency.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24648406',\n",
       "     'score': 0.4191829264163971},\n",
       "    {'start': 1221,\n",
       "     'end': 1485,\n",
       "     'text': 'Overall, these results provide a total new insight into the transcriptional regulation mechanism of coactivators p300 and CBP in ESCs, which is important in maintaining self-renewal and pluripotency, by mediating the formation of higher order chromosome structures.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24648406',\n",
       "     'score': 0.2935965359210968}]},\n",
       "  {'id': '29042476',\n",
       "   'score': 9.866009712219238,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 92,\n",
       "     'text': 'The three-dimensional genome: regulating gene expression during pluripotency and development.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '29042476',\n",
       "     'score': 0.4218951165676117},\n",
       "    {'start': 0,\n",
       "     'end': 234,\n",
       "     'text': 'The precise expression of genes in time and space during embryogenesis is largely influenced by communication between enhancers and promoters, which is propagated and governed by the physical proximity of these elements in the nucleus.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '29042476',\n",
       "     'score': 0.18712130188941956},\n",
       "    {'start': 236,\n",
       "     'end': 412,\n",
       "     'text': 'Here, we review how chromatin domains organize the genome by guiding enhancers to their target genes thereby preventing non-specific interactions with other neighboring regions.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '29042476',\n",
       "     'score': 0.16716381907463074},\n",
       "    {'start': 414,\n",
       "     'end': 610,\n",
       "     'text': 'We also discuss the dynamics of chromatin interactions between enhancers and promoters, as well as the consequent changes in gene expression, that occur in pluripotent cells and during development.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '29042476',\n",
       "     'score': 0.3728809356689453},\n",
       "    {'start': 612,\n",
       "     'end': 875,\n",
       "     'text': 'Finally, we evaluate how genomic changes such as deletions, inversions and duplications affect 3D chromatin configuration overall and lead to ectopic enhancer-promoter contacts, and thus gene misexpression, which can contribute to abnormal development and disease.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '29042476',\n",
       "     'score': 0.22391435503959656}]},\n",
       "  {'id': '28622076',\n",
       "   'score': 9.783202171325684,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 110,\n",
       "     'text': 'Human Induced Pluripotent Stem Cell-Derived Endothelial Cells for Three-Dimensional Microphysiological Systems.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '28622076',\n",
       "     'score': 0.36644959449768066},\n",
       "    {'start': 0,\n",
       "     'end': 165,\n",
       "     'text': 'Microphysiological systems (MPS), or \"organ-on-a-chip\" platforms, aim to recapitulate in vivo physiology using small-scale in vitro tissue models of human physiology.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28622076',\n",
       "     'score': 0.11068680882453918},\n",
       "    {'start': 167,\n",
       "     'end': 314,\n",
       "     'text': 'While significant efforts have been made to create vascularized tissues, most reports utilize primary endothelial cells that hinder reproducibility.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28622076',\n",
       "     'score': 0.10793313384056091},\n",
       "    {'start': 316,\n",
       "     'end': 483,\n",
       "     'text': 'In this study, we report the use of human induced pluripotent stem cell-derived endothelial cells (iPS-ECs) in developing three-dimensional (3D) microvascular networks.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28622076',\n",
       "     'score': 0.4245969355106354},\n",
       "    {'start': 485,\n",
       "     'end': 610,\n",
       "     'text': 'We established a CDH5-mCherry reporter iPS cell line, which expresses the vascular endothelial (VE)-cadherin fused to mCherry.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28622076',\n",
       "     'score': 0.1105991005897522},\n",
       "    {'start': 612,\n",
       "     'end': 896,\n",
       "     'text': 'The iPS-ECs demonstrate physiological functions characteristic of primary endothelial cells in a series of in vitro assays, including permeability, response to shear stress, and the expression of endothelial markers (CD31, von Willibrand factor, and endothelial nitric oxide synthase).',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28622076',\n",
       "     'score': 0.1143372654914856},\n",
       "    {'start': 898,\n",
       "     'end': 1018,\n",
       "     'text': 'The iPS-ECs form stable, perfusable microvessels over the course of 14 days when cultured within 3D microfluidic devices.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28622076',\n",
       "     'score': 0.14495491981506348},\n",
       "    {'start': 1020,\n",
       "     'end': 1125,\n",
       "     'text': 'We also demonstrate that inhibition of TGF-β signaling improves vascular network formation by the iPS-ECs.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28622076',\n",
       "     'score': 0.12957125902175903},\n",
       "    {'start': 1127,\n",
       "     'end': 1304,\n",
       "     'text': 'We conclude that iPS-ECs can be a source of endothelial cells in MPS providing opportunities for human disease modeling and improving the reproducibility of 3D vascular networks.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28622076',\n",
       "     'score': 0.17117729783058167}]},\n",
       "  {'id': '31315906',\n",
       "   'score': 9.766156196594238,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 149,\n",
       "     'text': 'Profiling the long noncoding RNA interaction network in the regulatory elements of target genes by chromatin in situ reverse transcription sequencing.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '31315906',\n",
       "     'score': 0.13434267044067383},\n",
       "    {'start': 0,\n",
       "     'end': 134,\n",
       "     'text': 'Long noncoding RNAs (lncRNAs) can regulate the activity of target genes by participating in the organization of chromatin architecture.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31315906',\n",
       "     'score': 0.1428370177745819},\n",
       "    {'start': 136,\n",
       "     'end': 399,\n",
       "     'text': 'We have devised a \"chromatin-RNA in situ reverse transcription sequencing\" (CRIST-seq) approach to profile the lncRNA interaction network in gene regulatory elements by combining the simplicity of RNA biotin labeling with the specificity of the CRISPR/Cas9 system.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31315906',\n",
       "     'score': 0.13011813163757324},\n",
       "    {'start': 401,\n",
       "     'end': 612,\n",
       "     'text': 'Using gene-specific gRNAs, we describe a pluripotency-specific lncRNA interacting network in the promoters of Sox2 and Pou5f1, two critical stem cell factors that are required for the maintenance of pluripotency.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31315906',\n",
       "     'score': 0.3435637354850769},\n",
       "    {'start': 614,\n",
       "     'end': 713,\n",
       "     'text': 'The promoter-interacting lncRNAs were specifically activated during reprogramming into pluripotency.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31315906',\n",
       "     'score': 0.4100116491317749},\n",
       "    {'start': 715,\n",
       "     'end': 789,\n",
       "     'text': 'Knockdown of these lncRNAs caused the stem cells to exit from pluripotency.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31315906',\n",
       "     'score': 0.27840617299079895},\n",
       "    {'start': 791,\n",
       "     'end': 967,\n",
       "     'text': 'In contrast, overexpression of the pluripotency-associated lncRNA activated the promoters of core stem cell factor genes and enhanced fibroblast reprogramming into pluripotency.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31315906',\n",
       "     'score': 0.32683682441711426},\n",
       "    {'start': 969,\n",
       "     'end': 1151,\n",
       "     'text': 'These CRIST-seq data suggest that the Sox2 and Pou5f1 promoters are organized within a unique lncRNA interaction network that determines the fate of pluripotency during reprogramming.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31315906',\n",
       "     'score': 0.3261426091194153},\n",
       "    {'start': 1153,\n",
       "     'end': 1260,\n",
       "     'text': 'This CRIST approach may be broadly used to map lncRNA interaction networks at target loci across the genome.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31315906',\n",
       "     'score': 0.12960880994796753}]},\n",
       "  {'id': '24170349',\n",
       "   'score': 9.674530982971191,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 76,\n",
       "     'text': 'Autophagy regulates homeostasis of pluripotency-associated proteins in hESCs.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '24170349',\n",
       "     'score': 0.3461417555809021},\n",
       "    {'start': 0,\n",
       "     'end': 163,\n",
       "     'text': 'The pluripotency of embryonic stem cells (ESCs) is maintained by intracellular networks of many pluripotency-associated (PA) proteins such as OCT4, SOX2, and NANOG.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24170349',\n",
       "     'score': 0.4057402014732361},\n",
       "    {'start': 165,\n",
       "     'end': 269,\n",
       "     'text': 'However, the mechanisms underlying the regulation of protein homeostasis for pluripotency remain elusive.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24170349',\n",
       "     'score': 0.3350452780723572},\n",
       "    {'start': 271,\n",
       "     'end': 429,\n",
       "     'text': 'Here, we first demonstrate that autophagy acts together with the ubiquitin-proteasome system (UPS) to modulate the levels of PA proteins in human ESCs (hESCs).',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24170349',\n",
       "     'score': 0.247096985578537},\n",
       "    {'start': 431,\n",
       "     'end': 519,\n",
       "     'text': 'Autophagy inhibition impaired the pluripotency despite increment of PA proteins in hESCs.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24170349',\n",
       "     'score': 0.27625173330307007},\n",
       "    {'start': 521,\n",
       "     'end': 614,\n",
       "     'text': 'Immunogold-electron microscopy confirmed localization of OCT4 molecules within autophagosomes.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24170349',\n",
       "     'score': 0.29148149490356445},\n",
       "    {'start': 616,\n",
       "     'end': 723,\n",
       "     'text': 'Also, knockdown of LC3 expression led to accumulation of PA proteins and reduction of pluripotency in hESCs.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24170349',\n",
       "     'score': 0.3075977861881256},\n",
       "    {'start': 725,\n",
       "     'end': 824,\n",
       "     'text': 'Interestingly, autophagy and the UPS showed differential kinetics in the degradation of PA proteins.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24170349',\n",
       "     'score': 0.11589106917381287},\n",
       "    {'start': 826,\n",
       "     'end': 997,\n",
       "     'text': 'Autophagy inhibition caused enhanced accumulation of both cytoplasmic and nuclear PA proteins, whereas the UPS inhibition led to preferentially degrade nuclear PA proteins.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24170349',\n",
       "     'score': 0.11371856927871704},\n",
       "    {'start': 999,\n",
       "     'end': 1139,\n",
       "     'text': 'Our findings suggest that autophagy modulates homeostasis of PA proteins, providing a new insight in the regulation of pluripotency in hESCs.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24170349',\n",
       "     'score': 0.35039037466049194}]},\n",
       "  {'id': '27036552',\n",
       "   'score': 9.63797664642334,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 100,\n",
       "     'text': 'Hmga2 is necessary for Otx2-dependent exit of embryonic stem cells from the pluripotent ground state.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '27036552',\n",
       "     'score': 0.2798989415168762},\n",
       "    {'start': 0,\n",
       "     'end': 257,\n",
       "     'text': \"BACKGROUND\\nA crucial event in the differentiation of mouse embryonic stem cells (ESCs) is the exit from the pluripotent ground state that leads to the acquisition of the 'primed' pluripotent phenotype, characteristic of the epiblast-like stem cells (EpiLCs).\",\n",
       "     'is_title': False,\n",
       "     'doc_id': '27036552',\n",
       "     'score': 0.2700392007827759},\n",
       "    {'start': 259,\n",
       "     'end': 333,\n",
       "     'text': 'The transcription factors Oct4 and Otx2 play a key role in this phenomenon.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '27036552',\n",
       "     'score': 0.27299344539642334},\n",
       "    {'start': 335,\n",
       "     'end': 517,\n",
       "     'text': 'In particular, Otx2 pioneers and activates new enhancers, which are silent in ESCs and which control the transcription of genes responsible for the acquisition of the EpiLC phenotype.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '27036552',\n",
       "     'score': 0.4107334613800049},\n",
       "    {'start': 519,\n",
       "     'end': 660,\n",
       "     'text': 'An important point that remains to be addressed is the mechanism through which Otx2 engages the new enhancers and stably associates with them.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '27036552',\n",
       "     'score': 0.17143890261650085},\n",
       "    {'start': 662,\n",
       "     'end': 850,\n",
       "     'text': 'Hmga2 is a member of the high-mobility group family of proteins, non-histone components of chromatin whose expression is high during embryogenesis and becomes low or undetectable in adults.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '27036552',\n",
       "     'score': 0.11868694424629211},\n",
       "    {'start': 852,\n",
       "     'end': 952,\n",
       "     'text': 'Its high expression during embryogenesis suggests that Hmga2 fulfills important roles in development.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '27036552',\n",
       "     'score': 0.1116006076335907},\n",
       "    {'start': 956,\n",
       "     'end': 1055,\n",
       "     'text': 'RESULTS\\nHere, we demonstrate that Hmga2 accumulates soon after the induction of ESC differentiation.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '27036552',\n",
       "     'score': 0.11226573586463928},\n",
       "    {'start': 1057,\n",
       "     'end': 1169,\n",
       "     'text': 'Its suppression hampers the exit of ESCs from the pluripotent ground state and their differentiation into EpiLCs.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '27036552',\n",
       "     'score': 0.2754358649253845},\n",
       "    {'start': 1171,\n",
       "     'end': 1290,\n",
       "     'text': 'Mechanistically, Hmga2 controls the differentiation process by cooperating with Otx2 in the pioneering of new enhancers.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '27036552',\n",
       "     'score': 0.18212589621543884},\n",
       "    {'start': 1292,\n",
       "     'end': 1430,\n",
       "     'text': 'In Hmga2 null induced pluripotent stem cells we observe that Otx2 fails to regulate its target genes upon the induction of differentiation.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '27036552',\n",
       "     'score': 0.27476048469543457},\n",
       "    {'start': 1432,\n",
       "     'end': 1640,\n",
       "     'text': 'Hmga2 associates to Otx2-bound loci in EpiLCs, and in Hmga2 KO cells Otx2 is unable to engage and activate the new enhancers, thus indicating that Hmga2 is required for the binding of Otx2 to its cis-elements.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '27036552',\n",
       "     'score': 0.1892279088497162},\n",
       "    {'start': 1642,\n",
       "     'end': 1797,\n",
       "     'text': 'We find that this mechanism also operates on the Hmga2 gene, which is one of the targets of Otx2, thus indicating the existence of a positive feedback loop.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '27036552',\n",
       "     'score': 0.11528739333152771},\n",
       "    {'start': 1801,\n",
       "     'end': 1915,\n",
       "     'text': 'CONCLUSIONS\\nOur findings reveal a novel mechanism necessary for the exit of ESCs from the pluripotent ground state.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '27036552',\n",
       "     'score': 0.2726737856864929},\n",
       "    {'start': 1917,\n",
       "     'end': 2063,\n",
       "     'text': 'Upon the induction of ESC differentiation, Otx2 alone or in combination with Oct4 engages new enhancers, which are silent in undifferentiated ESCs.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '27036552',\n",
       "     'score': 0.36384570598602295},\n",
       "    {'start': 2065,\n",
       "     'end': 2230,\n",
       "     'text': 'The Hmga2 gene is activated by Otx2 and Hmga2 protein binds to the enhancers targeted by Otx2, thus facilitating the engagement and/or the stable association of Otx2.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '27036552',\n",
       "     'score': 0.20535767078399658},\n",
       "    {'start': 2232,\n",
       "     'end': 2384,\n",
       "     'text': 'Therefore, our results demonstrate that Hmga2 is a key element of the regulatory network that governs the exit of ESCs from the pluripotent ground state.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '27036552',\n",
       "     'score': 0.3137584924697876}]},\n",
       "  {'id': '31304567',\n",
       "   'score': 9.54745101928711,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 118,\n",
       "     'text': 'Three-dimensional cultured liver-on-a-Chip with mature hepatocyte-like cells derived from human pluripotent stem cells.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '31304567',\n",
       "     'score': 0.42351478338241577},\n",
       "    {'start': 0,\n",
       "     'end': 118,\n",
       "     'text': 'Liver-on-a-Chip technology holds considerable potential for applications in drug screening and chemical-safety testing.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31304567',\n",
       "     'score': 0.11414846777915955},\n",
       "    {'start': 120,\n",
       "     'end': 352,\n",
       "     'text': 'To establish such platforms, functional hepatocytes are required; however, primary hepatocytes are commonly used, despite problems involving donor limitations, lot-to-lot variation, and unsatisfactory two-dimensional culture methods.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31304567',\n",
       "     'score': 0.15832966566085815},\n",
       "    {'start': 354,\n",
       "     'end': 597,\n",
       "     'text': 'Although human pluripotent stem cells (hPSCs) may represent a strong alternative contender to address the aforementioned issues, remaining technological challenges include the robust, highly efficient production of high-purity hepatic clusters.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31304567',\n",
       "     'score': 0.23991736769676208},\n",
       "    {'start': 599,\n",
       "     'end': 723,\n",
       "     'text': 'In addition, current Liver-on-a-Chip platforms are relatively complicated and not applicable for high-throughput experiments.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31304567',\n",
       "     'score': 0.11474660038948059},\n",
       "    {'start': 725,\n",
       "     'end': 848,\n",
       "     'text': 'Here, we develop a very simple Liver-on-a-Chip platform with mature and functional hepatocyte-like cells derived from hPSCs.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31304567',\n",
       "     'score': 0.11265712976455688},\n",
       "    {'start': 850,\n",
       "     'end': 1182,\n",
       "     'text': 'To establish a method for hepatic differentiation of hPSCs, cells were first treated by inhibiting the phosphoinositide 3-kinase- and Rho-associated protein kinase-signaling pathways to stop self-renewal and improve survival, respectively, which enabled the formation of a well-defined endoderm and facilitated hepatocyte commitment.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31304567',\n",
       "     'score': 0.1260453462600708},\n",
       "    {'start': 1184,\n",
       "     'end': 1433,\n",
       "     'text': 'Next, a simple microfluidic device was used to create a three-dimensional (3D) culture environment that enhanced the maturation and function of hepatocyte-like cells by increasing the expression of both hepatic maturation markers and cytochrome P450.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31304567',\n",
       "     'score': 0.16047409176826477},\n",
       "    {'start': 1435,\n",
       "     'end': 1605,\n",
       "     'text': 'Finally, we confirmed improvements in hepatic functions, such as drug uptake/excretion capabilities, in >90% of 3D-matured hepatocyte-like cells by indocyanin green assay.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31304567',\n",
       "     'score': 0.1466076672077179},\n",
       "    {'start': 1607,\n",
       "     'end': 1803,\n",
       "     'text': 'These results indicated that the incorporation of hPSC-derived hepatocytes on our Liver-on-a-Chip platform may serve to enhance the processes involved in drug screening and chemical-safety testing.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31304567',\n",
       "     'score': 0.12223529815673828}]},\n",
       "  {'id': '27125671',\n",
       "   'score': 9.520529747009277,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 114,\n",
       "     'text': 'The pluripotency factor Nanog regulates pericentromeric heterochromatin organization in mouse embryonic stem cells.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '27125671',\n",
       "     'score': 0.36134791374206543},\n",
       "    {'start': 0,\n",
       "     'end': 85,\n",
       "     'text': 'An open and decondensed chromatin organization is a defining property of pluripotency.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '27125671',\n",
       "     'score': 0.3744581341743469},\n",
       "    {'start': 87,\n",
       "     'end': 261,\n",
       "     'text': 'Several epigenetic regulators have been implicated in maintaining an open chromatin organization, but how these processes are connected to the pluripotency network is unknown.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '27125671',\n",
       "     'score': 0.37988194823265076},\n",
       "    {'start': 263,\n",
       "     'end': 460,\n",
       "     'text': 'Here, we identified a new role for the transcription factor NANOG as a key regulator connecting the pluripotency network with constitutive heterochromatin organization in mouse embryonic stem cells.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '27125671',\n",
       "     'score': 0.35795527696609497},\n",
       "    {'start': 462,\n",
       "     'end': 555,\n",
       "     'text': 'Deletion of Nanog leads to chromatin compaction and the remodeling of heterochromatin domains.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '27125671',\n",
       "     'score': 0.3050897717475891},\n",
       "    {'start': 557,\n",
       "     'end': 643,\n",
       "     'text': 'Forced expression of NANOG in epiblast stem cells is sufficient to decompact chromatin.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '27125671',\n",
       "     'score': 0.2971142828464508},\n",
       "    {'start': 645,\n",
       "     'end': 944,\n",
       "     'text': 'NANOG associates with satellite repeats within heterochromatin domains, contributing to an architecture characterized by highly dispersed chromatin fibers, low levels of H3K9me3, and high major satellite transcription, and the strong transactivation domain of NANOG is required for this organization.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '27125671',\n",
       "     'score': 0.3552335798740387},\n",
       "    {'start': 946,\n",
       "     'end': 1205,\n",
       "     'text': 'The heterochromatin-associated protein SALL1 is a direct cofactor for NANOG, and loss of Sall1 recapitulates the Nanog-null phenotype, but the loss of Sall1 can be circumvented through direct recruitment of the NANOG transactivation domain to major satellites.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '27125671',\n",
       "     'score': 0.2978903651237488},\n",
       "    {'start': 1207,\n",
       "     'end': 1432,\n",
       "     'text': 'These results establish a direct connection between the pluripotency network and chromatin organization and emphasize that maintaining an open heterochromatin architecture is a highly regulated process in embryonic stem cells.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '27125671',\n",
       "     'score': 0.39368849992752075}]},\n",
       "  {'id': '30350469',\n",
       "   'score': 9.502934455871582,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 180,\n",
       "     'text': 'Tuning Forkhead Box D3 overexpression to promote specific osteogenic differentiation of human embryonic stem cells while reducing pluripotency in a three-dimensional culture system.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '30350469',\n",
       "     'score': 0.39350152015686035},\n",
       "    {'start': 0,\n",
       "     'end': 190,\n",
       "     'text': 'Clinical use of human embryonic stem cells (hESCs) in bone regeneration applications requires that their osteogenic differentiation be highly controllable as well as time- and cost-effective.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '30350469',\n",
       "     'score': 0.2443694770336151},\n",
       "    {'start': 192,\n",
       "     'end': 540,\n",
       "     'text': 'The main goals of the current work were thus (a) to assess whether overexpression of pluripotency regulator Forkhead Box D3 (FOXD3) can enhance the osteogenic commitment of hESCs seeded in three-dimensional (3D) scaffolds and (b) to evaluate if the degree of FOXD3 overexpression regulates the strength and specificity of hESC osteogenic commitment.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '30350469',\n",
       "     'score': 0.3956056535243988},\n",
       "    {'start': 542,\n",
       "     'end': 707,\n",
       "     'text': 'In conducting these studies, an interpenetrating hydrogel network consisting of poly(ethylene glycol) diacrylate and collagen I was utilized as a 3D culture platform.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '30350469',\n",
       "     'score': 0.18305140733718872},\n",
       "    {'start': 709,\n",
       "     'end': 940,\n",
       "     'text': 'Expression of osteogenic, chondrogenic, pluripotency, and germ layer markers by encapsulated hESCs was measured after 2\\xa0weeks of culture in osteogenic medium in the presence or absence doxycycline-induced FOXD3 transgene expression.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '30350469',\n",
       "     'score': 0.3054622411727905},\n",
       "    {'start': 942,\n",
       "     'end': 1183,\n",
       "     'text': 'Towards the first goal, FOXD3 overexpression initiated 24\\xa0hr prior to hESC encapsulation, relative to unstimulated controls, resulted in upregulation of osteogenic markers and enhanced calcium deposition, without promoting off-target effects.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '30350469',\n",
       "     'score': 0.11817711591720581},\n",
       "    {'start': 1185,\n",
       "     'end': 1374,\n",
       "     'text': 'However, when initiation of FOXD3 overexpression was increased from 24 to 48\\xa0hr prior to encapsulation, hESC osteogenic commitment was not further enhanced and off-target effects were noted.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '30350469',\n",
       "     'score': 0.1143130362033844},\n",
       "    {'start': 1376,\n",
       "     'end': 1621,\n",
       "     'text': 'Specifically, relative to 24-hr prestimulation, initiation of FOXD3 overexpression 48\\xa0hr prior to encapsulation yielded increased expression of pluripotency markers while reducing mesodermal but increasing endodermal germ layer marker expression.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '30350469',\n",
       "     'score': 0.29410868883132935},\n",
       "    {'start': 1623,\n",
       "     'end': 1796,\n",
       "     'text': 'Combined, the current results indicate that the controlled overexpression of FOXD3 warrants further investigation as a mechanism to guide enhanced hESC osteogenic commitment.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '30350469',\n",
       "     'score': 0.11418530344963074}]},\n",
       "  {'id': '28614297',\n",
       "   'score': 9.433808326721191,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 82,\n",
       "     'text': 'Multilineage communication regulates human liver bud development from pluripotency.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '28614297',\n",
       "     'score': 0.2864711880683899},\n",
       "    {'start': 0,\n",
       "     'end': 133,\n",
       "     'text': 'Conventional two-dimensional differentiation from pluripotency fails to recapitulate cell interactions occurring during organogenesis.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28614297',\n",
       "     'score': 0.39616042375564575},\n",
       "    {'start': 135,\n",
       "     'end': 275,\n",
       "     'text': 'Three-dimensional organoids generate complex organ-like tissues; however, it is unclear how heterotypic interactions affect lineage identity.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28614297',\n",
       "     'score': 0.16951856017112732},\n",
       "    {'start': 277,\n",
       "     'end': 411,\n",
       "     'text': 'Here we use single-cell RNA sequencing to reconstruct hepatocyte-like lineage progression from pluripotency in two-dimensional culture.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28614297',\n",
       "     'score': 0.3961751461029053},\n",
       "    {'start': 413,\n",
       "     'end': 590,\n",
       "     'text': 'We then derive three-dimensional liver bud organoids by reconstituting hepatic, stromal, and endothelial interactions, and deconstruct heterogeneity during liver bud development.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28614297',\n",
       "     'score': 0.1645735800266266},\n",
       "    {'start': 592,\n",
       "     'end': 745,\n",
       "     'text': 'We find that liver bud hepatoblasts diverge from the two-dimensional lineage, and express epithelial migration signatures characteristic of organ budding.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28614297',\n",
       "     'score': 0.1639573574066162},\n",
       "    {'start': 747,\n",
       "     'end': 954,\n",
       "     'text': 'We benchmark three-dimensional liver buds against fetal and adult human liver single-cell RNA sequencing data, and find a striking correspondence between the three-dimensional liver bud and fetal liver cells.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28614297',\n",
       "     'score': 0.16483372449874878},\n",
       "    {'start': 956,\n",
       "     'end': 1216,\n",
       "     'text': 'We use a receptor-ligand pairing analysis and a high-throughput inhibitor assay to interrogate signalling in liver buds, and show that vascular endothelial growth factor (VEGF) crosstalk potentiates endothelial network formation and hepatoblast differentiation.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28614297',\n",
       "     'score': 0.13404500484466553},\n",
       "    {'start': 1218,\n",
       "     'end': 1385,\n",
       "     'text': 'Our molecular dissection reveals interlineage communication regulating organoid development, and illuminates previously inaccessible aspects of human liver development.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28614297',\n",
       "     'score': 0.1296645998954773}]},\n",
       "  {'id': '26399336',\n",
       "   'score': 9.394570350646973,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 118,\n",
       "     'text': 'Identification of novel proteins differentially expressed in pluripotent embryonic stem cells and differentiated cells.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '26399336',\n",
       "     'score': 0.271136999130249},\n",
       "    {'start': 0,\n",
       "     'end': 84,\n",
       "     'text': 'Mammalian pluripotent stem cells possess properties of self-renewal and pluripotency.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '26399336',\n",
       "     'score': 0.2874419093132019},\n",
       "    {'start': 86,\n",
       "     'end': 257,\n",
       "     'text': 'These abilities are maintained by the strict regulation of pluripotent stem cell-specific transcription factor network and unique properties of chromatin in the stem cells.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '26399336',\n",
       "     'score': 0.3912312984466553},\n",
       "    {'start': 259,\n",
       "     'end': 475,\n",
       "     'text': 'Although these major signaling pathways robustly control the characteristics of stem cells, other regulatory factors, such as metabolic pathways, are also known to modulate stem cell proliferation and differentiation.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '26399336',\n",
       "     'score': 0.11688193678855896},\n",
       "    {'start': 477,\n",
       "     'end': 622,\n",
       "     'text': 'In this study, we fractionated protein samples from mouse embryonic stem (ES) cells cultured with or without the leukemia inhibitory factor (LIF).',\n",
       "     'is_title': False,\n",
       "     'doc_id': '26399336',\n",
       "     'score': 0.12306603789329529},\n",
       "    {'start': 624,\n",
       "     'end': 717,\n",
       "     'text': 'Protein expression was quantified by 2-dimensional differential gel electrophoresis (2D-DIGE).',\n",
       "     'is_title': False,\n",
       "     'doc_id': '26399336',\n",
       "     'score': 0.16299530863761902},\n",
       "    {'start': 719,\n",
       "     'end': 849,\n",
       "     'text': 'In total, 44 proteins were identified as being differentially expressed in the pluripotent stem cells and the differentiated cells.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '26399336',\n",
       "     'score': 0.29089367389678955},\n",
       "    {'start': 851,\n",
       "     'end': 1023,\n",
       "     'text': 'Surprisingly, half of the identified proteins were the proteins localized in mitochondria, which supply cellular energy and regulate cell cycle, development, and cell death.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '26399336',\n",
       "     'score': 0.12032938003540039},\n",
       "    {'start': 1025,\n",
       "     'end': 1132,\n",
       "     'text': 'Some of these identified proteins are involved in the metabolic function and the regulation of pluripotency.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '26399336',\n",
       "     'score': 0.38898223638534546},\n",
       "    {'start': 1134,\n",
       "     'end': 1256,\n",
       "     'text': 'Further analysis of the identified proteins could provide new information for the manipulation of pluripotency in ES cells.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '26399336',\n",
       "     'score': 0.31028246879577637}]},\n",
       "  {'id': '28361318',\n",
       "   'score': 9.375205039978027,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 78,\n",
       "     'text': 'Generation of a Three-Dimensional Kidney Structure from Pluripotent Stem Cells.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '28361318',\n",
       "     'score': 0.39970484375953674},\n",
       "    {'start': 0,\n",
       "     'end': 143,\n",
       "     'text': 'The kidney is a vital organ that has an important role in the maintenance of homeostasis by fluid volume regulation and waste product excretion.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28361318',\n",
       "     'score': 0.13902965188026428},\n",
       "    {'start': 145,\n",
       "     'end': 233,\n",
       "     'text': 'This role cannot be performed without the three-dimensional (3D) structure of the kidney.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28361318',\n",
       "     'score': 0.1641119420528412},\n",
       "    {'start': 235,\n",
       "     'end': 390,\n",
       "     'text': 'Therefore, it is important to generate the 3D structure of the kidney when inducing functional kidney tissue or the whole organ from pluripotent stem cells.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28361318',\n",
       "     'score': 0.3040994703769684},\n",
       "    {'start': 392,\n",
       "     'end': 549,\n",
       "     'text': 'In this chapter, we describe the detailed methods to induce kidney progenitor cells from pluripotent stem cells, which are based on embryological development.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28361318',\n",
       "     'score': 0.26723235845565796},\n",
       "    {'start': 551,\n",
       "     'end': 653,\n",
       "     'text': 'We also provide a method to generate 3D kidney tissue with vascularized glomeruli upon transplantation.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28361318',\n",
       "     'score': 0.1442633867263794}]},\n",
       "  {'id': '16150921',\n",
       "   'score': 9.327903747558594,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 91,\n",
       "     'text': 'Three-dimensional nanofibrillar surfaces promote self-renewal in mouse embryonic stem cells.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '16150921',\n",
       "     'score': 0.1721205711364746},\n",
       "    {'start': 0,\n",
       "     'end': 167,\n",
       "     'text': 'The regulation of mouse embryonic stem cell (mESC) fate is controlled by the interplay of signaling networks that either promote self-renewal or induce differentiation.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '16150921',\n",
       "     'score': 0.16373324394226074},\n",
       "    {'start': 169,\n",
       "     'end': 301,\n",
       "     'text': 'Leukemia inhibitory factor (LIF) is a cytokine that is required for stem cell renewal in mouse but not in human embryonic stem cells.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '16150921',\n",
       "     'score': 0.11700060963630676},\n",
       "    {'start': 303,\n",
       "     'end': 638,\n",
       "     'text': 'However, feeder layers of embryonic fibroblasts are capable of inducing stem cell renewal in both cell types, suggesting that the self-renewal signaling pathways may also be promoted by other triggers, such as alternative cytokines and/or chemical or physical properties of the extracellular matrix (ECM) secreted by feeder fibroblasts.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '16150921',\n",
       "     'score': 0.11970141530036926},\n",
       "    {'start': 640,\n",
       "     'end': 794,\n",
       "     'text': 'We have recently used a synthetic polyamide matrix (Ultra-Web) whose three-dimensional (3D) nanofibrillar organization resembles the ECM/basement membrane.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '16150921',\n",
       "     'score': 0.19708123803138733},\n",
       "    {'start': 796,\n",
       "     'end': 1006,\n",
       "     'text': 'Growth of mESCs on this nanofibrillar surface greatly enhanced proliferation and self-renewal in comparison with growth on tissue culture surfaces without nanofibers, despite the presence of LIF in both systems.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '16150921',\n",
       "     'score': 0.29000887274742126},\n",
       "    {'start': 1008,\n",
       "     'end': 1306,\n",
       "     'text': 'Enhanced proliferation and self-renewal of the stem cells on nanofibrillar surfaces were correlated with the activation of the small GTPase Rac, the activation of phosphoinositide 3-kinase (PI3K) pathway, and the enhanced expression of Nanog, a homeoprotein required for maintenance of pluripotency.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '16150921',\n",
       "     'score': 0.2947448790073395},\n",
       "    {'start': 1308,\n",
       "     'end': 1411,\n",
       "     'text': 'Inhibitors of PI3K reduced the expression level of Nanog in mESCs cultured on 3D nanofibrillar surfaces.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '16150921',\n",
       "     'score': 0.3962307870388031},\n",
       "    {'start': 1413,\n",
       "     'end': 1639,\n",
       "     'text': 'These results provide support for the view that the three-dimensionality of the culture surface may function as a cue for the activation of Rac and PI3K signaling pathways, resulting in stem cell proliferation and self-renewal.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '16150921',\n",
       "     'score': 0.12439730763435364}]},\n",
       "  {'id': '30414923',\n",
       "   'score': 9.274703979492188,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 76,\n",
       "     'text': 'Gain of CTCF-Anchored Chromatin Loops Marks the Exit from Naive Pluripotency.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '30414923',\n",
       "     'score': 0.2822280824184418},\n",
       "    {'start': 0,\n",
       "     'end': 160,\n",
       "     'text': 'The genome of pluripotent stem cells adopts a unique three-dimensional architecture featuring weakly condensed heterochromatin and large nucleosome-free regions.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '30414923',\n",
       "     'score': 0.3918892443180084},\n",
       "    {'start': 162,\n",
       "     'end': 325,\n",
       "     'text': 'Yet, it is unknown whether structural loops and contact domains display characteristics that distinguish embryonic stem cells (ESCs) from differentiated cell types.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '30414923',\n",
       "     'score': 0.2605474591255188},\n",
       "    {'start': 327,\n",
       "     'end': 493,\n",
       "     'text': 'We used genome-wide chromosome conformation capture and super-resolution imaging to determine nuclear organization in mouse ESC and neural stem cell (NSC) derivatives.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '30414923',\n",
       "     'score': 0.1448041796684265},\n",
       "    {'start': 495,\n",
       "     'end': 583,\n",
       "     'text': 'We found that loss of pluripotency is accompanied by widespread gain of structural loops.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '30414923',\n",
       "     'score': 0.31085795164108276},\n",
       "    {'start': 585,\n",
       "     'end': 770,\n",
       "     'text': 'This general architectural change correlates with enhanced binding of CTCF and cohesins and more pronounced insulation of contacts across chromatin boundaries in lineage-committed cells.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '30414923',\n",
       "     'score': 0.1192331314086914},\n",
       "    {'start': 772,\n",
       "     'end': 858,\n",
       "     'text': 'Reprogramming NSCs to pluripotency restores the unique features of ESC domain topology.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '30414923',\n",
       "     'score': 0.28120577335357666},\n",
       "    {'start': 860,\n",
       "     'end': 969,\n",
       "     'text': 'Domains defined by the anchors of loops established upon differentiation are enriched for developmental genes.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '30414923',\n",
       "     'score': 0.11662551760673523},\n",
       "    {'start': 971,\n",
       "     'end': 1160,\n",
       "     'text': 'Chromatin loop formation is a pervasive structural alteration to the genome that accompanies exit from pluripotency and delineates the spatial segregation of developmentally regulated genes.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '30414923',\n",
       "     'score': 0.28757819533348083}]},\n",
       "  {'id': '33304995',\n",
       "   'score': 9.233806610107422,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 85,\n",
       "     'text': 'Functional and Sustainable 3D Human Neural Network Models from Pluripotent Stem Cells.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '33304995',\n",
       "     'score': 0.3980183005332947},\n",
       "    {'start': 0,\n",
       "     'end': 229,\n",
       "     'text': 'Three-dimensional in vitro cell culture models, particularly for the central nervous system, allow for the exploration of mechanisms of organ development, cellular interactions, and disease progression within defined environments.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '33304995',\n",
       "     'score': 0.1784084439277649},\n",
       "    {'start': 231,\n",
       "     'end': 405,\n",
       "     'text': 'Here we describe the development and characterization of three-dimensional tissue models that promote the differentiation and long-term survival of functional neural networks.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '33304995',\n",
       "     'score': 0.19315552711486816},\n",
       "    {'start': 407,\n",
       "     'end': 649,\n",
       "     'text': 'These tissue cultures show diverse cell populations including neurons and glial cells (astrocytes) interacting in 3D with spontaneous neural activity confirmed through electrophysiological recordings and calcium imaging over at least 8 months.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '33304995',\n",
       "     'score': 0.15080520510673523},\n",
       "    {'start': 651,\n",
       "     'end': 964,\n",
       "     'text': 'This approach allows for the direct integration of pluripotent stem cells into the 3D construct bypassing early neural differentiation steps (embryoid bodies and neural rosettes), which streamlines the process while also providing a system that can be manipulated to support a variety of experimental applications.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '33304995',\n",
       "     'score': 0.3149849772453308},\n",
       "    {'start': 966,\n",
       "     'end': 1250,\n",
       "     'text': \"This tissue model has been tested in stem cells derived from healthy individuals as well as Alzheimer's and Parkinson's disease patients, with similar growth and gene expression responses indicating potential use in the modeling of disease states related to neurodegenerative diseases.\",\n",
       "     'is_title': False,\n",
       "     'doc_id': '33304995',\n",
       "     'score': 0.1207895576953888}]},\n",
       "  {'id': '20953172',\n",
       "   'score': 9.224075317382812,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 84,\n",
       "     'text': 'A genome-wide RNAi screen reveals determinants of human embryonic stem cell identity.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '20953172',\n",
       "     'score': 0.11572203040122986},\n",
       "    {'start': 0,\n",
       "     'end': 117,\n",
       "     'text': 'The derivation of human ES cells (hESCs) from human blastocysts represents one of the milestones in stem cell biology.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '20953172',\n",
       "     'score': 0.26213860511779785},\n",
       "    {'start': 119,\n",
       "     'end': 285,\n",
       "     'text': 'The full potential of hESCs in research and clinical applications requires a detailed understanding of the genetic network that governs the unique properties of hESCs.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '20953172',\n",
       "     'score': 0.34298673272132874},\n",
       "    {'start': 287,\n",
       "     'end': 423,\n",
       "     'text': 'Here, we report a genome-wide RNA interference screen to identify genes which regulate self-renewal and pluripotency properties in hESCs.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '20953172',\n",
       "     'score': 0.27338868379592896},\n",
       "    {'start': 425,\n",
       "     'end': 583,\n",
       "     'text': 'Interestingly, functionally distinct complexes involved in transcriptional regulation and chromatin remodelling are among the factors identified in the screen.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '20953172',\n",
       "     'score': 0.14615768194198608},\n",
       "    {'start': 585,\n",
       "     'end': 768,\n",
       "     'text': 'To understand the roles of these potential regulators of hESCs, we studied transcription factor PRDM14 to gain new insights into its functional roles in the regulation of pluripotency.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '20953172',\n",
       "     'score': 0.3653643727302551},\n",
       "    {'start': 770,\n",
       "     'end': 887,\n",
       "     'text': 'We showed that PRDM14 regulates directly the expression of key pluripotency gene POU5F1 through its proximal enhancer.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '20953172',\n",
       "     'score': 0.3836219310760498},\n",
       "    {'start': 889,\n",
       "     'end': 1129,\n",
       "     'text': 'Genome-wide location profiling experiments revealed that PRDM14 colocalized extensively with other key transcription factors such as OCT4, NANOG and SOX2, indicating that PRDM14 is integrated into the core transcriptional regulatory network.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '20953172',\n",
       "     'score': 0.31281960010528564},\n",
       "    {'start': 1131,\n",
       "     'end': 1311,\n",
       "     'text': 'More importantly, in a gain-of-function assay, we showed that PRDM14 is able to enhance the efficiency of reprogramming of human fibroblasts in conjunction with OCT4, SOX2 and KLF4.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '20953172',\n",
       "     'score': 0.2948090732097626},\n",
       "    {'start': 1313,\n",
       "     'end': 1538,\n",
       "     'text': 'Altogether, our study uncovers a wealth of novel hESC regulators wherein PRDM14 exemplifies a key transcription factor required for the maintenance of hESC identity and the reacquisition of pluripotency in human somatic cells.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '20953172',\n",
       "     'score': 0.3012164533138275}]},\n",
       "  {'id': '22548825',\n",
       "   'score': 9.151025772094727,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 80,\n",
       "     'text': 'Derivation of an interaction/regulation network describing pluripotency in human.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '22548825',\n",
       "     'score': 0.38108497858047485},\n",
       "    {'start': 0,\n",
       "     'end': 169,\n",
       "     'text': 'Identification of the key genes/proteins of pluripotency and their interrelationships is an important step in understanding the induction and maintenance of pluripotency.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '22548825',\n",
       "     'score': 0.3298477530479431},\n",
       "    {'start': 171,\n",
       "     'end': 265,\n",
       "     'text': 'Experimental approaches have accumulated large amounts of interaction/regulation data in mouse.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '22548825',\n",
       "     'score': 0.13394737243652344},\n",
       "    {'start': 267,\n",
       "     'end': 418,\n",
       "     'text': 'We investigate how far such information can be transferred to human, the species of maximum interest, for which experimental data are much more limited.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '22548825',\n",
       "     'score': 0.11906912922859192},\n",
       "    {'start': 420,\n",
       "     'end': 519,\n",
       "     'text': 'To address this issue, we mapped an existing mouse pluripotency network (the PluriNetWork) to human.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '22548825',\n",
       "     'score': 0.30386391282081604},\n",
       "    {'start': 521,\n",
       "     'end': 701,\n",
       "     'text': 'We transferred interaction and regulation links between genes/proteins from mouse to human on the basis of orthologous relationship of the genes/proteins (called interolog mapping).',\n",
       "     'is_title': False,\n",
       "     'doc_id': '22548825',\n",
       "     'score': 0.13766825199127197},\n",
       "    {'start': 703,\n",
       "     'end': 887,\n",
       "     'text': 'To reduce the number of false positives, we used four different methods: phylogenetic profiling, Gene Ontology semantic similarity, gene co-expression, and RNA interference (RNAi) data.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '22548825',\n",
       "     'score': 0.12338033318519592},\n",
       "    {'start': 889,\n",
       "     'end': 1057,\n",
       "     'text': 'The methods and the resulting networks were evaluated by a novel approach using the information about the genes known to be involved in pluripotency from the literature.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '22548825',\n",
       "     'score': 0.3402690291404724},\n",
       "    {'start': 1059,\n",
       "     'end': 1192,\n",
       "     'text': 'The RNAi method proved best for filtering out unlikely interactions, so it was used to construct the final human pluripotency network.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '22548825',\n",
       "     'score': 0.3018547296524048},\n",
       "    {'start': 1194,\n",
       "     'end': 1330,\n",
       "     'text': 'The RNAi data are based on human embryonic stem cells (hESCs) that are generally considered to be in a (primed) epiblast stem cell state.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '22548825',\n",
       "     'score': 0.26566484570503235},\n",
       "    {'start': 1332,\n",
       "     'end': 1538,\n",
       "     'text': 'Therefore, we assume that the final human network may reflect the (primed) epiblast stem cell state more closely, while the mouse network reflects the (unprimed/naïve) embryonic stem cell state more closely.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '22548825',\n",
       "     'score': 0.13441291451454163}]},\n",
       "  {'id': '26206085',\n",
       "   'score': 9.135479927062988,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 94,\n",
       "     'text': 'Chromatin organization in pluripotent cells: emerging approaches to study and disrupt function.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '26206085',\n",
       "     'score': 0.3437819480895996},\n",
       "    {'start': 0,\n",
       "     'end': 182,\n",
       "     'text': 'Translating the vast amounts of genomic and epigenomic information accumulated on the linear genome into three-dimensional models of nuclear organization is a current major challenge.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '26206085',\n",
       "     'score': 0.21401411294937134},\n",
       "    {'start': 184,\n",
       "     'end': 430,\n",
       "     'text': 'In response to this challenge, recent technological innovations based on chromosome conformation capture methods in combination with increasingly powerful functional approaches have revealed exciting insights into key aspects of genome regulation.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '26206085',\n",
       "     'score': 0.13456183671951294},\n",
       "    {'start': 432,\n",
       "     'end': 699,\n",
       "     'text': 'These findings have led to an emerging model where the genome is folded and compartmentalized into highly conserved topological domains that are further divided into functional subdomains containing physical loops that bring cis-regulatory elements to close proximity.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '26206085',\n",
       "     'score': 0.12079134583473206},\n",
       "    {'start': 701,\n",
       "     'end': 902,\n",
       "     'text': 'Targeted functional experiments, largely based on designable DNA-binding proteins, have begun to define the major architectural proteins required to establish and maintain appropriate genome regulation.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '26206085',\n",
       "     'score': 0.1325986683368683},\n",
       "    {'start': 904,\n",
       "     'end': 1105,\n",
       "     'text': 'Here, we focus on the accessible and well-characterized system of pluripotent cells to review the functional role of chromatin organization in regulating pluripotency, differentiation and reprogramming.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '26206085',\n",
       "     'score': 0.39008259773254395}]},\n",
       "  {'id': '24035354',\n",
       "   'score': 9.134991645812988,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 137,\n",
       "     'text': 'Long-range chromatin contacts in embryonic stem cells reveal a role for pluripotency factors and polycomb proteins in genome organization.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '24035354',\n",
       "     'score': 0.3310115337371826},\n",
       "    {'start': 0,\n",
       "     'end': 104,\n",
       "     'text': 'The relationship between 3D organization of the genome and gene-regulatory networks is poorly understood.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24035354',\n",
       "     'score': 0.2418501079082489},\n",
       "    {'start': 106,\n",
       "     'end': 341,\n",
       "     'text': 'Here, we examined long-range chromatin interactions genome-wide in mouse embryonic stem cells (ESCs), iPSCs, and fibroblasts and uncovered a pluripotency-specific genome organization that is gradually reestablished during reprogramming.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24035354',\n",
       "     'score': 0.3077548146247864},\n",
       "    {'start': 343,\n",
       "     'end': 523,\n",
       "     'text': 'Our data confirm that long-range chromatin interactions are primarily associated with the spatial segregation of open and closed chromatin, defining overall chromosome conformation.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24035354',\n",
       "     'score': 0.12268230319023132},\n",
       "    {'start': 525,\n",
       "     'end': 752,\n",
       "     'text': 'Additionally, we identified two further levels of genome organization in ESCs characterized by colocalization of regions with high pluripotency factor occupancy and strong enrichment for Polycomb proteins/H3K27me3, respectively.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24035354',\n",
       "     'score': 0.35384947061538696},\n",
       "    {'start': 754,\n",
       "     'end': 1001,\n",
       "     'text': 'Underlining the independence of these networks and their functional relevance for genome organization, loss of the Polycomb protein Eed diminishes interactions between Polycomb-regulated regions without altering overarching chromosome conformation.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24035354',\n",
       "     'score': 0.18397995829582214},\n",
       "    {'start': 1003,\n",
       "     'end': 1252,\n",
       "     'text': 'Together, our data highlight a pluripotency-specific genome organization in which pluripotency factors such as Nanog and H3K27me3 occupy distinct nuclear spaces and reveal a role for cell-type-specific gene-regulatory networks in genome organization.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24035354',\n",
       "     'score': 0.3784448206424713}]},\n",
       "  {'id': '26526723',\n",
       "   'score': 9.108065605163574,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 112,\n",
       "     'text': 'Coordination of m(6)A mRNA Methylation and Gene Transcription by ZFP217 Regulates Pluripotency and Reprogramming.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '26526723',\n",
       "     'score': 0.29806822538375854},\n",
       "    {'start': 0,\n",
       "     'end': 160,\n",
       "     'text': 'Epigenetic and epitranscriptomic networks have important functions in maintaining the pluripotency of embryonic stem cells (ESCs) and somatic cell reprogramming.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '26526723',\n",
       "     'score': 0.38562652468681335},\n",
       "    {'start': 162,\n",
       "     'end': 266,\n",
       "     'text': 'However, the mechanisms integrating the actions of these distinct networks are only partially understood.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '26526723',\n",
       "     'score': 0.14018943905830383},\n",
       "    {'start': 268,\n",
       "     'end': 399,\n",
       "     'text': 'Here we show that the chromatin-associated zinc finger protein 217 (ZFP217) coordinates epigenetic and epitranscriptomic regulation.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '26526723',\n",
       "     'score': 0.14924296736717224},\n",
       "    {'start': 401,\n",
       "     'end': 648,\n",
       "     'text': 'ZFP217 interacts with several epigenetic regulators, activates the transcription of key pluripotency genes, and modulates N6-methyladenosine (m(6)A) deposition on their transcripts by sequestering the enzyme m(6)A methyltransferase-like 3 (METTL3).',\n",
       "     'is_title': False,\n",
       "     'doc_id': '26526723',\n",
       "     'score': 0.268758624792099},\n",
       "    {'start': 650,\n",
       "     'end': 881,\n",
       "     'text': 'Consistently, Zfp217 depletion compromises ESC self-renewal and somatic cell reprogramming, globally increases m(6)A RNA levels, and enhances m(6)A modification of the Nanog, Sox2, Klf4, and c-Myc mRNAs, promoting their degradation.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '26526723',\n",
       "     'score': 0.2757059633731842},\n",
       "    {'start': 883,\n",
       "     'end': 1015,\n",
       "     'text': 'ZFP217 binds its own target gene mRNAs, which are also METTL3 associated, and is enriched at promoters of m(6)A-modified transcripts.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '26526723',\n",
       "     'score': 0.1268824338912964},\n",
       "    {'start': 1017,\n",
       "     'end': 1173,\n",
       "     'text': 'Collectively, these findings shed light on how a transcription factor can tightly couple gene transcription to m(6)A RNA modification to ensure ESC identity.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '26526723',\n",
       "     'score': 0.11609819531440735}]},\n",
       "  {'id': '8707862',\n",
       "   'score': 9.094643592834473,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 96,\n",
       "     'text': 'Regulation of protein abundance in pluripotent cells undergoing commitment to the neural lineage.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '8707862',\n",
       "     'score': 0.38138315081596375},\n",
       "    {'start': 0,\n",
       "     'end': 254,\n",
       "     'text': 'The P19 cell line is a widely studied model of neural differentiation When pluripotent P19 cells are cultured as aggregates in the presence of retinoic acid for 4 days, the cells commit to the neural fate, but have not yet undergone overt differentiation.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '8707862',\n",
       "     'score': 0.27461665868759155},\n",
       "    {'start': 256,\n",
       "     'end': 376,\n",
       "     'text': 'Two-dimensional polyacrylamide gel electrophoresis was used to analyze cellular protein expression during this induction.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '8707862',\n",
       "     'score': 0.1835336685180664},\n",
       "    {'start': 378,\n",
       "     'end': 431,\n",
       "     'text': 'Approximately 500 abundant polypeptides were analyzed.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '8707862',\n",
       "     'score': 0.11626586318016052},\n",
       "    {'start': 433,\n",
       "     'end': 577,\n",
       "     'text': 'Seventeen polypeptides were upregulated during induction; several of these were significantly regulated 48 h after the addition of retinoic acid.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '8707862',\n",
       "     'score': 0.11912712454795837},\n",
       "    {'start': 579,\n",
       "     'end': 611,\n",
       "     'text': 'No downregulations were observed.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '8707862',\n",
       "     'score': 0.11626586318016052},\n",
       "    {'start': 613,\n",
       "     'end': 705,\n",
       "     'text': 'Fifteen of the 17 polypeptides continued to be expressed throughout terminal differentiation.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '8707862',\n",
       "     'score': 0.11900350451469421},\n",
       "    {'start': 707,\n",
       "     'end': 846,\n",
       "     'text': 'The upregulation of 14 of the 17 polypeptides requires both retinoic acid and aggregation, which alone do not induce neural differentiation.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '8707862',\n",
       "     'score': 0.12266531586647034},\n",
       "    {'start': 848,\n",
       "     'end': 981,\n",
       "     'text': 'Furthermore, these regulated polypeptides are expressed in neural tissue, suggesting they are associated with neural function in vivo.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '8707862',\n",
       "     'score': 0.12146833539009094},\n",
       "    {'start': 983,\n",
       "     'end': 1096,\n",
       "     'text': 'Embryonic stem cells, a totipotent line, also neurally differentiate in response to retinoic acid and aggregation.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '8707862',\n",
       "     'score': 0.12005940079689026},\n",
       "    {'start': 1098,\n",
       "     'end': 1260,\n",
       "     'text': 'Comparison of embryonic stem cells to P19 cells shows that the two systems regulate a similar set of polypeptides and are thus likely to utilize a similar pathway.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '8707862',\n",
       "     'score': 0.12565675377845764},\n",
       "    {'start': 1262,\n",
       "     'end': 1402,\n",
       "     'text': 'These studies are a step toward determining the full extent of regulation involved in the commitment of pluripotent cells to the neural fate.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '8707862',\n",
       "     'score': 0.3657374382019043}]},\n",
       "  {'id': '24104477',\n",
       "   'score': 9.066771507263184,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 127,\n",
       "     'text': 'A competitive protein interaction network buffers Oct4-mediated differentiation to promote pluripotency in embryonic stem cells.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '24104477',\n",
       "     'score': 0.346407413482666},\n",
       "    {'start': 0,\n",
       "     'end': 217,\n",
       "     'text': \"Pluripotency in embryonic stem cells is maintained through the activity of a small set of transcription factors centred around Oct4 and Nanog, which control the expression of 'self-renewal' and 'differentiation' genes.\",\n",
       "     'is_title': False,\n",
       "     'doc_id': '24104477',\n",
       "     'score': 0.3075852394104004},\n",
       "    {'start': 219,\n",
       "     'end': 419,\n",
       "     'text': 'Here, we combine single-cell quantitative immunofluorescence microscopy and gene expression analysis, together with theoretical modelling, to investigate how the activity of those factors is regulated.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24104477',\n",
       "     'score': 0.12300476431846619},\n",
       "    {'start': 421,\n",
       "     'end': 584,\n",
       "     'text': 'We uncover a key role for post-translational regulation in the maintenance of pluripotency, which complements the well-established transcriptional regulatory layer.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24104477',\n",
       "     'score': 0.3389921188354492},\n",
       "    {'start': 586,\n",
       "     'end': 778,\n",
       "     'text': 'Specifically, we find that the activity of a network of protein complexes involving Nanog, Oct4, Tcf3, and β-catenin suffices to account for the behavior of ES cells under different conditions.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24104477',\n",
       "     'score': 0.3235238790512085},\n",
       "    {'start': 780,\n",
       "     'end': 945,\n",
       "     'text': 'Our results suggest that the function of the network is to buffer the transcriptional activity of Oct4, which appears to be the main determinant to exit pluripotency.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24104477',\n",
       "     'score': 0.31913793087005615},\n",
       "    {'start': 947,\n",
       "     'end': 1137,\n",
       "     'text': 'The protein network explains the mechanisms underlying the gain and loss of function in different mutants, and brings us closer to a full understanding of the molecular basis of pluripotency.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24104477',\n",
       "     'score': 0.3752123713493347}]},\n",
       "  {'id': '30157624',\n",
       "   'score': 9.063106536865234,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 138,\n",
       "     'text': 'Embryonic Stem Cell Engineering with a Glycomimetic FGF2/BMP4 Co-Receptor Drives Mesodermal Differentiation in a Three-Dimensional Culture.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '30157624',\n",
       "     'score': 0.17118901014328003},\n",
       "    {'start': 0,\n",
       "     'end': 300,\n",
       "     'text': 'Cell surface glycans, such as heparan sulfate (HS), are increasingly identified as co-regulators of growth factor signaling in early embryonic development; therefore, chemical tailoring of HS activity within the cellular glycocalyx of stem cells offers an opportunity to control their differentiation.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '30157624',\n",
       "     'score': 0.12159267067909241},\n",
       "    {'start': 302,\n",
       "     'end': 506,\n",
       "     'text': 'The growth factors FGF2 and BMP4 are involved in mediating the exit of murine embryonic stem cells (mESCs) from their pluripotent state and their differentiation toward mesodermal cell types, respectively.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '30157624',\n",
       "     'score': 0.311210036277771},\n",
       "    {'start': 508,\n",
       "     'end': 690,\n",
       "     'text': 'Here, we report a method for remodeling the glycocalyx of mutant Ext1-/- mESCs with defective biosynthesis of HS to drive their mesodermal differentiation in an embryoid body culture.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '30157624',\n",
       "     'score': 0.27280616760253906},\n",
       "    {'start': 692,\n",
       "     'end': 999,\n",
       "     'text': 'Lipid-functionalized synthetic HS-mimetic glycopolymers with affinity for both FGF2 and BMP4 were introduced into the plasma membrane of Ext1-/- mESCs, where they acted as functional co-receptors of these growth factors and facilitated signal transduction through associated MAPK and Smad signaling pathways.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '30157624',\n",
       "     'score': 0.2697141170501709},\n",
       "    {'start': 1001,\n",
       "     'end': 1226,\n",
       "     'text': 'We demonstrate that these materials can be employed to remodel Ext1-/- mESCs within three-dimensional embryoid body structures, providing enhanced association of BMP4 at the cell surface and driving mesodermal differentiation.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '30157624',\n",
       "     'score': 0.38241466879844666},\n",
       "    {'start': 1228,\n",
       "     'end': 1491,\n",
       "     'text': 'As a more complete understanding of the function of HS in regulating development continues to emerge, this simple glycocalyx engineering method is poised to enable precise control over growth factor signaling activity and outcomes of differentiation in stem cells.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '30157624',\n",
       "     'score': 0.1398715078830719}]},\n",
       "  {'id': '23444350',\n",
       "   'score': 9.045125007629395,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 116,\n",
       "     'text': 'A membrane-associated β-catenin/Oct4 complex correlates with ground-state pluripotency in mouse embryonic stem cells.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '23444350',\n",
       "     'score': 0.2991563379764557},\n",
       "    {'start': 0,\n",
       "     'end': 255,\n",
       "     'text': 'The maintenance of pluripotency in mouse embryonic stem cells (mESCs) relies on the activity of a transcriptional network that is fuelled by the activity of three transcription factors (Nanog, Oct4 and Sox2) and balanced by the repressive activity of Tcf3.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '23444350',\n",
       "     'score': 0.37772685289382935},\n",
       "    {'start': 257,\n",
       "     'end': 370,\n",
       "     'text': 'Extracellular signals modulate the activity of the network and regulate the differentiation capacity of the cells.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '23444350',\n",
       "     'score': 0.141049325466156},\n",
       "    {'start': 372,\n",
       "     'end': 556,\n",
       "     'text': 'Wnt/β-catenin signaling has emerged as a significant potentiator of pluripotency: increases in the levels of β-catenin regulate the activity of Oct4 and Nanog, and enhance pluripotency.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '23444350',\n",
       "     'score': 0.3263707458972931},\n",
       "    {'start': 558,\n",
       "     'end': 737,\n",
       "     'text': 'A recent report shows that β-catenin achieves some of these effects by modulating the activity of Tcf3, and that this effect does not require its transcriptional activation domain.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '23444350',\n",
       "     'score': 0.12156513333320618},\n",
       "    {'start': 739,\n",
       "     'end': 952,\n",
       "     'text': 'Here, we show that during self-renewal there is negligible transcriptional activity of β-catenin and that this is due to its tight association with membranes, where we find it in a complex with Oct4 and E-cadherin.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '23444350',\n",
       "     'score': 0.2583959698677063},\n",
       "    {'start': 954,\n",
       "     'end': 1079,\n",
       "     'text': 'Differentiation triggers a burst of Wnt/β-catenin transcriptional activity that coincides with the disassembly of the complex.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '23444350',\n",
       "     'score': 0.11976107954978943},\n",
       "    {'start': 1081,\n",
       "     'end': 1223,\n",
       "     'text': 'Our results establish that β-catenin, but not its transcriptional activity, is central to pluripotency acting through a β-catenin/Oct4 complex.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '23444350',\n",
       "     'score': 0.28451353311538696}]},\n",
       "  {'id': '22046437',\n",
       "   'score': 9.023152351379395,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 33,\n",
       "     'text': 'CARM1 mediates modulation of Sox2.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '22046437',\n",
       "     'score': 0.11844712495803833},\n",
       "    {'start': 0,\n",
       "     'end': 127,\n",
       "     'text': 'Sox2 is a key component of the transcription factor network that maintains the pluripotent state of embryonic stem cells (ESCs).',\n",
       "     'is_title': False,\n",
       "     'doc_id': '22046437',\n",
       "     'score': 0.34139561653137207},\n",
       "    {'start': 129,\n",
       "     'end': 263,\n",
       "     'text': 'Sox2 is regulated by multiple post-translational modifications, including ubiquitination, sumoylation, acetylation and phosphorylation.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '22046437',\n",
       "     'score': 0.12168490886688232},\n",
       "    {'start': 265,\n",
       "     'end': 462,\n",
       "     'text': 'Here we report that Sox2 is in association with and methylated by coactivator-associated arginine methyltransferase 1 (CARM1), a protein arginine methyltransferase that plays a pivotal role in ESCs.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '22046437',\n",
       "     'score': 0.30046582221984863},\n",
       "    {'start': 464,\n",
       "     'end': 570,\n",
       "     'text': 'We found that CARM1 facilitates Sox2-mediated transactivation and directly methylates Sox2 at arginine 113.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '22046437',\n",
       "     'score': 0.11919710040092468},\n",
       "    {'start': 572,\n",
       "     'end': 625,\n",
       "     'text': 'This methylation event enhances Sox2 self-association.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '22046437',\n",
       "     'score': 0.11716225743293762},\n",
       "    {'start': 627,\n",
       "     'end': 725,\n",
       "     'text': 'Furthermore, the physiological retention of Sox2 on chromatin restricts the Sox2 methylation level.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '22046437',\n",
       "     'score': 0.12018075585365295},\n",
       "    {'start': 727,\n",
       "     'end': 902,\n",
       "     'text': 'Our study reveals the direct regulation of Sox2 by CARM1 that sheds lights on how arginine methylation signals are integrated into the pluripotent transcription factor network.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '22046437',\n",
       "     'score': 0.3782678246498108}]},\n",
       "  {'id': '31428613',\n",
       "   'score': 9.018759727478027,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 64,\n",
       "     'text': 'Chaperones and Beyond as Key Players in Pluripotency Maintenance.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '31428613',\n",
       "     'score': 0.2854463458061218},\n",
       "    {'start': 0,\n",
       "     'end': 161,\n",
       "     'text': 'Pluripotency is orchestrated by distinct players and chaperones and their partners have emerged as pivotal molecules in proteostasis control to maintain stemness.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31428613',\n",
       "     'score': 0.30126267671585083},\n",
       "    {'start': 163,\n",
       "     'end': 344,\n",
       "     'text': 'The proteostasis network consists of diverse interconnected pathways that function dynamically according to the needs of the cell to quality control and maintain protein homeostasis.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31428613',\n",
       "     'score': 0.15134593844413757},\n",
       "    {'start': 346,\n",
       "     'end': 529,\n",
       "     'text': 'The proteostasis machinery of pluripotent stem cells (PSCs) is finely adjusted in response to distinct stimuli during cell fate commitment to determine successful organism development.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31428613',\n",
       "     'score': 0.3076562285423279},\n",
       "    {'start': 531,\n",
       "     'end': 635,\n",
       "     'text': 'Growing evidence has shown different classes of chaperones regulating crucial cellular processes in PSCs.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31428613',\n",
       "     'score': 0.13689666986465454},\n",
       "    {'start': 637,\n",
       "     'end': 811,\n",
       "     'text': \"Histones chaperones promote proper nucleosome assembly and modulate the epigenetic regulation of factors involved in PSCs' rapid turnover from pluripotency to differentiation.\",\n",
       "     'is_title': False,\n",
       "     'doc_id': '31428613',\n",
       "     'score': 0.3611997961997986},\n",
       "    {'start': 813,\n",
       "     'end': 1021,\n",
       "     'text': 'The life cycle of pluripotency proteins from synthesis and folding, transport and degradation is finely regulated by chaperones and co-factors either to maintain the stemness status or to cell fate commitment.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31428613',\n",
       "     'score': 0.3082446753978729},\n",
       "    {'start': 1023,\n",
       "     'end': 1175,\n",
       "     'text': 'Here, we summarize current knowledge of the chaperone network that govern stemness and present the versatile role of chaperones in stem cells resilience.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31428613',\n",
       "     'score': 0.14538276195526123},\n",
       "    {'start': 1177,\n",
       "     'end': 1443,\n",
       "     'text': 'Elucidation of the intricate regulation of pluripotency, dissecting in detail molecular determinants and drivers, is fundamental to understanding the properties of stem cells in order to provide a reliable foundation for biomedical research and regenerative medicine.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31428613',\n",
       "     'score': 0.37499818205833435}]},\n",
       "  {'id': '24194720',\n",
       "   'score': 9.001581192016602,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 118,\n",
       "     'text': 'Qualitative modeling identifies IL-11 as a novel regulator in maintaining self-renewal in human pluripotent stem cells.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '24194720',\n",
       "     'score': 0.2609670162200928},\n",
       "    {'start': 0,\n",
       "     'end': 161,\n",
       "     'text': 'Pluripotency in human embryonic stem cells (hESCs) and induced pluripotent stem cells (iPSCs) is regulated by three transcription factors-OCT3/4, SOX2, and NANOG.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24194720',\n",
       "     'score': 0.3290077745914459},\n",
       "    {'start': 163,\n",
       "     'end': 329,\n",
       "     'text': 'To fully exploit the therapeutic potential of these cells it is essential to have a good mechanistic understanding of the maintenance of self-renewal and pluripotency.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24194720',\n",
       "     'score': 0.293235182762146},\n",
       "    {'start': 331,\n",
       "     'end': 567,\n",
       "     'text': 'In this study, we demonstrate a powerful systems biology approach in which we first expand literature-based network encompassing the core regulators of pluripotency by assessing the behavior of genes targeted by perturbation experiments.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24194720',\n",
       "     'score': 0.3735601305961609},\n",
       "    {'start': 569,\n",
       "     'end': 752,\n",
       "     'text': 'We focused our attention on highly regulated genes encoding cell surface and secreted proteins as these can be more easily manipulated by the use of inhibitors or recombinant proteins.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24194720',\n",
       "     'score': 0.1216198205947876},\n",
       "    {'start': 754,\n",
       "     'end': 911,\n",
       "     'text': 'Qualitative modeling based on combining boolean networks and in silico perturbation experiments were employed to identify novel pluripotency-regulating genes.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24194720',\n",
       "     'score': 0.372661292552948},\n",
       "    {'start': 913,\n",
       "     'end': 1112,\n",
       "     'text': 'We validated Interleukin-11 (IL-11) and demonstrate that this cytokine is a novel pluripotency-associated factor capable of supporting self-renewal in the absence of exogenously added bFGF in culture.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24194720',\n",
       "     'score': 0.33625221252441406},\n",
       "    {'start': 1114,\n",
       "     'end': 1267,\n",
       "     'text': 'To date, the various protocols for hESCs maintenance require supplementation with bFGF to activate the Activin/Nodal branch of the TGFβ signaling pathway.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24194720',\n",
       "     'score': 0.2501968741416931},\n",
       "    {'start': 1269,\n",
       "     'end': 1460,\n",
       "     'text': 'Additional evidence supporting our findings is that IL-11 belongs to the same protein family as LIF, which is known to be necessary for maintaining pluripotency in mouse but not in human ESCs.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24194720',\n",
       "     'score': 0.2922254800796509},\n",
       "    {'start': 1462,\n",
       "     'end': 1552,\n",
       "     'text': 'These cytokines operate through the same gp130 receptor which interacts with Janus kinases.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24194720',\n",
       "     'score': 0.12059038877487183},\n",
       "    {'start': 1554,\n",
       "     'end': 1694,\n",
       "     'text': 'Our finding might explain why mESCs are in a more naïve cell state compared to hESCs and how to convert primed hESCs back to the naïve state.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24194720',\n",
       "     'score': 0.2690315842628479},\n",
       "    {'start': 1696,\n",
       "     'end': 1926,\n",
       "     'text': 'Taken together, our integrative modeling approach has identified novel genes as putative candidates to be incorporated into the expansion of the current gene regulatory network responsible for inducing and maintaining pluripotency.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24194720',\n",
       "     'score': 0.3067598342895508}]},\n",
       "  {'id': '19416884',\n",
       "   'score': 8.993880271911621,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 114,\n",
       "     'text': 'mTOR supports long-term self-renewal and suppresses mesoderm and endoderm activities of human embryonic stem cells.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '19416884',\n",
       "     'score': 0.12078273296356201},\n",
       "    {'start': 0,\n",
       "     'end': 237,\n",
       "     'text': 'Despite the recent identification of the transcriptional regulatory circuitry involving SOX2, NANOG, and OCT-4, the intracellular signaling networks that control pluripotency of human embryonic stem cells (hESCs) remain largely undefined.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '19416884',\n",
       "     'score': 0.37591099739074707},\n",
       "    {'start': 239,\n",
       "     'end': 407,\n",
       "     'text': 'Here, we demonstrate an essential role for the serine/threonine protein kinase mammalian target of rapamycin (mTOR) in regulating hESC long-term undifferentiated growth.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '19416884',\n",
       "     'score': 0.14699479937553406},\n",
       "    {'start': 409,\n",
       "     'end': 533,\n",
       "     'text': 'Inhibition of mTOR impairs pluripotency, prevents cell proliferation, and enhances mesoderm and endoderm activities in hESCs.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '19416884',\n",
       "     'score': 0.30433404445648193},\n",
       "    {'start': 535,\n",
       "     'end': 779,\n",
       "     'text': 'At the molecular level, mTOR integrates signals from extrinsic pluripotency-supporting factors and represses the transcriptional activities of a subset of developmental and growth-inhibitory genes, as revealed by genome-wide microarray analyses.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '19416884',\n",
       "     'score': 0.2596684694290161},\n",
       "    {'start': 781,\n",
       "     'end': 880,\n",
       "     'text': 'Repression of the developmental genes by mTOR is necessary for the maintenance of hESC pluripotency.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '19416884',\n",
       "     'score': 0.3138730823993683},\n",
       "    {'start': 882,\n",
       "     'end': 978,\n",
       "     'text': 'These results uncover a novel signaling mechanism by which mTOR controls fate decisions in hESCs.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '19416884',\n",
       "     'score': 0.2693483829498291},\n",
       "    {'start': 980,\n",
       "     'end': 1066,\n",
       "     'text': 'Our findings may contribute to effective strategies for tissue repair and regeneration.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '19416884',\n",
       "     'score': 0.11936423182487488}]},\n",
       "  {'id': '31724494',\n",
       "   'score': 8.987876892089844,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 104,\n",
       "     'text': 'Microvessel Network Formation and Interactions with Pancreatic Islets in Three-Dimensional Chip Cultures.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '31724494',\n",
       "     'score': 0.20889729261398315},\n",
       "    {'start': 0,\n",
       "     'end': 157,\n",
       "     'text': 'The pancreatic islet is a highly vascularized micro-organ, and rapid revascularization postislet transplantation is important for islet survival and function.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31724494',\n",
       "     'score': 0.12302398681640625},\n",
       "    {'start': 159,\n",
       "     'end': 314,\n",
       "     'text': 'However, the various mechanisms involved in islet revascularization are not fully understood, and we currently lack good in vitro platforms to explore this.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31724494',\n",
       "     'score': 0.128957599401474},\n",
       "    {'start': 316,\n",
       "     'end': 555,\n",
       "     'text': 'Our aim for this study was to generate perfusable microvascular networks in a microfluidic chip device, in which islets could be easily integrated, to establish an in vitro platform for investigations on islet-microvasculature interactions.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31724494',\n",
       "     'score': 0.14003503322601318},\n",
       "    {'start': 557,\n",
       "     'end': 942,\n",
       "     'text': 'We compared the ability of mesenchymal stem cells (MSCs) and fibroblasts to support microvascular network formation by human umbilical vein endothelial cells (HUVECs) and human induced pluripotent stem cell-derived endothelial colony-forming cell in two-dimensional and three-dimensional models of angiogenesis, and tested the effect of different culture media on microvessel formation.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31724494',\n",
       "     'score': 0.4021335542201996},\n",
       "    {'start': 944,\n",
       "     'end': 1093,\n",
       "     'text': 'HUVECs that were supported by MSCs formed patent and perfusable networks in a fibrin gel, whereas networks supported by fibroblasts rapidly regressed.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31724494',\n",
       "     'score': 0.1437842845916748},\n",
       "    {'start': 1095,\n",
       "     'end': 1182,\n",
       "     'text': 'Network morphology could be controlled by adjusting relative cell numbers and densities.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31724494',\n",
       "     'score': 0.14021751284599304},\n",
       "    {'start': 1184,\n",
       "     'end': 1379,\n",
       "     'text': 'Incorporation of isolated rat islets demonstrated that islets recruit local microvasculature in vitro, but that the microvessels did not invade islets, at least during the course of these studies.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31724494',\n",
       "     'score': 0.12237435579299927},\n",
       "    {'start': 1381,\n",
       "     'end': 1616,\n",
       "     'text': 'This in vitro microvascularization platform can provide a useful tool to study how various parameters affect islet integration with microvascular networks and could also be utilized for studies of vascularization of other organ systems.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31724494',\n",
       "     'score': 0.14367058873176575},\n",
       "    {'start': 1618,\n",
       "     'end': 1760,\n",
       "     'text': 'Impact statement To improve pancreatic islet graft survival and function posttransplantation, rapid and adequate revascularization is critical.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31724494',\n",
       "     'score': 0.12122046947479248},\n",
       "    {'start': 1762,\n",
       "     'end': 1897,\n",
       "     'text': 'Efforts to improve islet revascularization are demanding due to an insufficient understanding of the mechanisms involved in the process.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31724494',\n",
       "     'score': 0.12830641865730286},\n",
       "    {'start': 1899,\n",
       "     'end': 2086,\n",
       "     'text': 'We have applied a microfluidics platform to generate microvascular networks, and by incorporating pancreatic islets, we were able to study microvasculature-islet interactions in real time.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31724494',\n",
       "     'score': 0.14255136251449585},\n",
       "    {'start': 2088,\n",
       "     'end': 2259,\n",
       "     'text': 'This platform can provide a useful tool to study islet integration with microvascular networks, and could be utilized for studies of vascularization of other organ systems.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31724494',\n",
       "     'score': 0.14122667908668518},\n",
       "    {'start': 2261,\n",
       "     'end': 2367,\n",
       "     'text': 'Moreover, this work may be adapted toward developing a prevascularized islet construct for transplantation.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31724494',\n",
       "     'score': 0.11760681867599487}]},\n",
       "  {'id': '28936269',\n",
       "   'score': 8.968226432800293,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 109,\n",
       "     'text': 'Microfibrous Scaffolds Enhance Endothelial Differentiation and Organization of Induced Pluripotent Stem Cells.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '28936269',\n",
       "     'score': 0.35005834698677063},\n",
       "    {'start': 0,\n",
       "     'end': 180,\n",
       "     'text': 'INTRODUCTION\\nHuman induced pluripotent stem cells (iPSCs) are a promising source of endothelial cells (iPSC-ECs) for engineering three-dimensional (3D) vascularized cardiac tissues.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28936269',\n",
       "     'score': 0.3848375380039215},\n",
       "    {'start': 182,\n",
       "     'end': 470,\n",
       "     'text': 'To mimic cardiac microvasculature, in which capillaries are oriented in parallel, we hypothesized that endothelial differentiation of iPSCs within topographically aligned 3D scaffolds would be a facile one-step approach to generate iPSC-ECs as well as induce aligned vascular organization.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28936269',\n",
       "     'score': 0.19390136003494263},\n",
       "    {'start': 474,\n",
       "     'end': 650,\n",
       "     'text': 'METHODS\\nHuman iPSCs underwent endothelial differentiation within electrospun 3D polycaprolactone (PCL) scaffolds having either randomly oriented or parallel-aligned microfibers.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28936269',\n",
       "     'score': 0.15493014454841614},\n",
       "    {'start': 652,\n",
       "     'end': 879,\n",
       "     'text': 'Using transcriptional, protein, and endothelial functional assays, endothelial differentiation was compared between conventional two-dimensional (2D) films and 3D scaffolds having either randomly oriented or aligned microfibers.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28936269',\n",
       "     'score': 0.17965486645698547},\n",
       "    {'start': 881,\n",
       "     'end': 1001,\n",
       "     'text': 'Furthermore, the role of parallel-aligned microfiber patterning on the organization of vessel-like networks was assessed.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28936269',\n",
       "     'score': 0.17952904105186462},\n",
       "    {'start': 1005,\n",
       "     'end': 1207,\n",
       "     'text': 'RESULTS\\nThe cells in both the randomly oriented and aligned 3D scaffolds demonstrated an 11-fold upregulation in gene expression of the endothelial phenotypic marker, CD31, compared to cells on 2D films.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28936269',\n",
       "     'score': 0.16266173124313354},\n",
       "    {'start': 1209,\n",
       "     'end': 1324,\n",
       "     'text': 'This upregulation corresponded to >3-fold increase in CD31 protein expression in 3D scaffolds, compared to 2D films.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28936269',\n",
       "     'score': 0.16749972105026245},\n",
       "    {'start': 1326,\n",
       "     'end': 1533,\n",
       "     'text': 'Concomitantly, other endothelial phenotypic markers including CD144 and endothelial nitric oxide synthase also showed significant transcriptional upregulation in 3D scaffolds by >7-fold, compared to 2D films.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28936269',\n",
       "     'score': 0.1590895652770996},\n",
       "    {'start': 1535,\n",
       "     'end': 1689,\n",
       "     'text': 'Nitric oxide production, which is characteristic of endothelial function, was produced 4-fold more abundantly in 3D scaffolds, compared to on 2D PCL films.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28936269',\n",
       "     'score': 0.16083160042762756},\n",
       "    {'start': 1691,\n",
       "     'end': 1950,\n",
       "     'text': 'Within aligned scaffolds, the iPSC-ECs displayed parallel-aligned vascular-like networks with 70% longer branch length, compared to cells in randomly oriented scaffolds, suggesting that fiber topography modulates vascular network-like formation and patterning.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28936269',\n",
       "     'score': 0.14416229724884033},\n",
       "    {'start': 1954,\n",
       "     'end': 2184,\n",
       "     'text': 'CONCLUSION\\nTogether, these results demonstrate that 3D scaffold structure promotes endothelial differentiation, compared to 2D substrates, and that aligned topographical patterning induces anisotropic vascular network organization.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28936269',\n",
       "     'score': 0.2151528298854828}]},\n",
       "  {'id': '23768116',\n",
       "   'score': 8.95314884185791,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 186,\n",
       "     'text': 'Proteomic analysis of early reprogramming events in murine somatic cells incubated with Xenopus laevis oocyte extracts demonstrates network associations with induced pluripotency markers.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '23768116',\n",
       "     'score': 0.3259560763835907},\n",
       "    {'start': 0,\n",
       "     'end': 192,\n",
       "     'text': 'The reprogramming of somatic cells into a pluripotent/embryonic-like state holds great potential for regenerative medicine, bypassing ethical issues associated with embryonic stem cells (ESCs).',\n",
       "     'is_title': False,\n",
       "     'doc_id': '23768116',\n",
       "     'score': 0.3115348219871521},\n",
       "    {'start': 194,\n",
       "     'end': 456,\n",
       "     'text': 'Numerous methods, including somatic cell nuclear transfer (SCNT), fusion to pluripotent cells, the use of cell extracts, and expression of transcription factors, have been used to reprogram cells into ES-like cells [termed induced pluripotent stem cells (iPSCs)].',\n",
       "     'is_title': False,\n",
       "     'doc_id': '23768116',\n",
       "     'score': 0.2667217254638672},\n",
       "    {'start': 458,\n",
       "     'end': 735,\n",
       "     'text': 'This study investigated early events in the nuclei of permeabilized murine somatic cells incubated in cytoplasmic extract prepared from Xenopus laevis germinal vesicle-stage oocytes by identifying proteins that showed significant quantitative changes using proteomic techniques.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '23768116',\n",
       "     'score': 0.12321022152900696},\n",
       "    {'start': 737,\n",
       "     'end': 939,\n",
       "     'text': 'A total of 69 protein spots from two-dimensional electrophoresis were identified as being significantly altered in expression after treatment, and 38 proteins were identified by tandem mass spectrometry.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '23768116',\n",
       "     'score': 0.17819857597351074},\n",
       "    {'start': 941,\n",
       "     'end': 1175,\n",
       "     'text': 'Network analysis was used to highlight pathway connections and interactions between these identified proteins, which were found to be involved in many functions--primarily nuclear structure and dynamics, transcription, and translation.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '23768116',\n",
       "     'score': 0.15581241250038147},\n",
       "    {'start': 1177,\n",
       "     'end': 1462,\n",
       "     'text': 'The pluripotency markers Klf4, c-Myc, Nanog, and POU5F1 were highlighted by the interaction network analysis, as well as other compounds/proteins known to be repressed in pluripotent cells [e.g., protein kinase C (PRKC)] or enhanced during differentiation of ESCs (e.g., retinoic acid).',\n",
       "     'is_title': False,\n",
       "     'doc_id': '23768116',\n",
       "     'score': 0.37398213148117065},\n",
       "    {'start': 1464,\n",
       "     'end': 1583,\n",
       "     'text': 'The network analysis also indicated additional proteins and pathways potentially involved in early reprogramming events.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '23768116',\n",
       "     'score': 0.15523281693458557}]},\n",
       "  {'id': '25857441',\n",
       "   'score': 8.932638168334961,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 46,\n",
       "     'text': 'Establishing pluripotency in early development.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '25857441',\n",
       "     'score': 0.3000926971435547},\n",
       "    {'start': 0,\n",
       "     'end': 197,\n",
       "     'text': 'The earliest steps of embryonic development involve important changes in chromatin and transcription factor networks, which are orchestrated to establish pluripotent cells that will form the embryo.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '25857441',\n",
       "     'score': 0.32913291454315186},\n",
       "    {'start': 199,\n",
       "     'end': 399,\n",
       "     'text': 'DNA methylation, histone modifications, the pluripotency regulatory network of transcription factors, maternal factors and newly translated proteins all contribute to these transitions in dynamic ways.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '25857441',\n",
       "     'score': 0.3381897211074829},\n",
       "    {'start': 401,\n",
       "     'end': 474,\n",
       "     'text': 'Moreover, these dynamics are linked to the onset of zygotic transcription.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '25857441',\n",
       "     'score': 0.1210588812828064},\n",
       "    {'start': 476,\n",
       "     'end': 679,\n",
       "     'text': 'We will review recent progress in our understanding of chromatin state and regulation of gene expression in the context of embryonic development in vertebrates, in particular mouse, Xenopus and zebrafish.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '25857441',\n",
       "     'score': 0.1436864137649536},\n",
       "    {'start': 681,\n",
       "     'end': 856,\n",
       "     'text': 'We include work on mouse embryonic stem cells and highlight work that illustrates how early embryonic dynamics establish gene regulatory networks and the state of pluripotency.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '25857441',\n",
       "     'score': 0.3701986074447632}]},\n",
       "  {'id': '23056006',\n",
       "   'score': 8.93102741241455,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 53,\n",
       "     'text': 'Role of CTCF in the regulation of microRNA expression.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '23056006',\n",
       "     'score': 0.14439508318901062},\n",
       "    {'start': 0,\n",
       "     'end': 93,\n",
       "     'text': 'MicroRNAs (miRNAs) are small non-coding RNAs that regulate expression of various target genes.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '23056006',\n",
       "     'score': 0.11944600939750671},\n",
       "    {'start': 95,\n",
       "     'end': 222,\n",
       "     'text': 'miRNAs are expressed in a tissue-specific manner and play important roles in cell proliferation, apoptosis, and differentiation.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '23056006',\n",
       "     'score': 0.12247636914253235},\n",
       "    {'start': 224,\n",
       "     'end': 385,\n",
       "     'text': 'Epigenetic alterations such as DNA methylation and histone modification are essential for chromatin remodeling and regulation of gene expression including miRNAs.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '23056006',\n",
       "     'score': 0.13989046216011047},\n",
       "    {'start': 387,\n",
       "     'end': 593,\n",
       "     'text': 'The CCCTC-binding factor, CTCF, is known to bind insulators and exhibits an enhancer-blocking and barrier function, and more recently, it also contributes to the three-dimensional organization of the genome.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '23056006',\n",
       "     'score': 0.34126001596450806},\n",
       "    {'start': 595,\n",
       "     'end': 742,\n",
       "     'text': 'CTCF can also serve as a barrier against the spread of DNA methylation and histone repressive marks over promoter regions of tumor suppressor genes.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '23056006',\n",
       "     'score': 0.1955692172050476},\n",
       "    {'start': 744,\n",
       "     'end': 907,\n",
       "     'text': 'Recent studies have shown that CTCF is also involved in the regulation of miRNAs such as miR-125b1, miR-375, and the miR-290 cluster in cancer cells and stem cells.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '23056006',\n",
       "     'score': 0.15107238292694092},\n",
       "    {'start': 909,\n",
       "     'end': 992,\n",
       "     'text': 'miR-125b1 is a candidate of tumor suppressor and is silenced in breast cancer cells.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '23056006',\n",
       "     'score': 0.12520340085029602},\n",
       "    {'start': 994,\n",
       "     'end': 1092,\n",
       "     'text': 'On the other hand, miR-375 may have oncogenic function and is overexpressed in breast cancer cells.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '23056006',\n",
       "     'score': 0.12647035717964172},\n",
       "    {'start': 1094,\n",
       "     'end': 1253,\n",
       "     'text': 'CTCF is involved in the regulation of both miR-125b1 and miR-375, indicating that there are various patterns of CTCF-associated epigenetic regulation of miRNAs.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '23056006',\n",
       "     'score': 0.16101661324501038},\n",
       "    {'start': 1255,\n",
       "     'end': 1355,\n",
       "     'text': 'CTCF may also play a key role in the pluripotency of cells through the regulation of miR-290 cluster.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '23056006',\n",
       "     'score': 0.3826563060283661},\n",
       "    {'start': 1357,\n",
       "     'end': 1494,\n",
       "     'text': 'These observations suggest that CTCF-mediated regulation of miRNAs could be a novel approach for cancer therapy and regenerative medicine.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '23056006',\n",
       "     'score': 0.13874012231826782}]},\n",
       "  {'id': '27773674',\n",
       "   'score': 8.929633140563965,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 95,\n",
       "     'text': 'Expanding the Circuitry of Pluripotency by Selective Isolation of Chromatin-Associated Proteins.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '27773674',\n",
       "     'score': 0.32780998945236206},\n",
       "    {'start': 0,\n",
       "     'end': 255,\n",
       "     'text': 'Maintenance of pluripotency is regulated by a network of transcription factors coordinated by Oct4, Sox2, and Nanog (OSN), yet a systematic investigation of the composition and dynamics of the OSN protein network specifically on chromatin is still missing.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '27773674',\n",
       "     'score': 0.37387555837631226},\n",
       "    {'start': 257,\n",
       "     'end': 461,\n",
       "     'text': 'Here we have developed a method combining ChIP with selective isolation of chromatin-associated proteins (SICAP) followed by mass spectrometry to identify chromatin-bound partners of a protein of interest.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '27773674',\n",
       "     'score': 0.1308966875076294},\n",
       "    {'start': 463,\n",
       "     'end': 630,\n",
       "     'text': 'ChIP-SICAP in mouse embryonic stem cells (ESCs) identified over 400 proteins associating with OSN, including several whose interaction depends on the pluripotent state.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '27773674',\n",
       "     'score': 0.26358672976493835},\n",
       "    {'start': 632,\n",
       "     'end': 817,\n",
       "     'text': 'Trim24, a previously unrecognized protein in the network, converges with OSN on multiple enhancers and suppresses the expression of developmental genes while activating cell cycle genes.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '27773674',\n",
       "     'score': 0.24970591068267822},\n",
       "    {'start': 819,\n",
       "     'end': 968,\n",
       "     'text': 'Consistently, Trim24 significantly improved efficiency of cellular reprogramming, demonstrating its direct functionality in establishing pluripotency.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '27773674',\n",
       "     'score': 0.2762302756309509},\n",
       "    {'start': 970,\n",
       "     'end': 1125,\n",
       "     'text': 'Collectively, ChIP-SICAP provides a powerful tool to decode chromatin protein composition, further enhanced by its integrative capacity to perform ChIP-seq.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '27773674',\n",
       "     'score': 0.12471142411231995}]},\n",
       "  {'id': '31951024',\n",
       "   'score': 8.92789077758789,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 113,\n",
       "     'text': 'The role of Hippo signaling pathway and mechanotransduction in tuning embryoid body formation and differentiation.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '31951024',\n",
       "     'score': 0.12618011236190796},\n",
       "    {'start': 0,\n",
       "     'end': 149,\n",
       "     'text': 'Embryoid bodies (EBs) are the three-dimensional aggregates of pluripotent stem cells that are used as a model system for the in vitro differentiation.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31951024',\n",
       "     'score': 0.40712907910346985},\n",
       "    {'start': 151,\n",
       "     'end': 273,\n",
       "     'text': 'EBs mimic the early stages of embryogenesis and are considered as a potential biomimetic body in tuning the stem cell fate.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31951024',\n",
       "     'score': 0.1248508095741272},\n",
       "    {'start': 275,\n",
       "     'end': 513,\n",
       "     'text': 'Although EBs have a spheroid shape, they are not formed accidentally by the agglomeration of cells; they are formed by the deliberate and programmed aggregation of stem cells in a complex topological and biophysical microstructure instead.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31951024',\n",
       "     'score': 0.12491637468338013},\n",
       "    {'start': 515,\n",
       "     'end': 695,\n",
       "     'text': 'EBs could be programmed to promisingly differentiate into the desired germ layers with specific cell lineages, in response to intra- and extra-biochemical and biomechanical signals.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31951024',\n",
       "     'score': 0.12371566891670227},\n",
       "    {'start': 697,\n",
       "     'end': 813,\n",
       "     'text': 'Hippo signaling and mechanotransduction are the key pathways in controlling the formation and differentiation of EBs.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31951024',\n",
       "     'score': 0.12538403272628784},\n",
       "    {'start': 815,\n",
       "     'end': 1003,\n",
       "     'text': 'The activity of the Hippo pathway strongly relies on cell-cell junctions, cell polarity, cellular architecture, cellular metabolism, and mechanical cues in the surrounding microenvironment.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31951024',\n",
       "     'score': 0.12559983134269714},\n",
       "    {'start': 1005,\n",
       "     'end': 1246,\n",
       "     'text': 'Although the Hippo pathway was initially thought to limit the size of the organ by inhibiting the proliferation and the promotion of apoptosis, the evidence suggests that this pathway even regulates stem cell self-renewal and differentiation.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31951024',\n",
       "     'score': 0.12291550636291504},\n",
       "    {'start': 1248,\n",
       "     'end': 1500,\n",
       "     'text': 'Considering the abovementioned explanations, the present study investigated the interplay of the Hippo signaling pathway, mechanotransduction, differentiation, and proliferation pathways to draw the molecular network involved in the control of EBs fate.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31951024',\n",
       "     'score': 0.15443626046180725},\n",
       "    {'start': 1502,\n",
       "     'end': 1688,\n",
       "     'text': 'In addition, this study highlighted several neglected critical parameters regarding EB formation, in the interplay with the Hippo core component involved in the promising differentiation.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31951024',\n",
       "     'score': 0.12807956337928772}]},\n",
       "  {'id': '28052084',\n",
       "   'score': 8.91479778289795,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 160,\n",
       "     'text': 'Linking Hematopoietic Differentiation to Co-Expressed Sets of Pluripotency-Associated and Imprinted Genes and to Regulatory microRNA-Transcription Factor Motifs.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '28052084',\n",
       "     'score': 0.3573504388332367},\n",
       "    {'start': 0,\n",
       "     'end': 177,\n",
       "     'text': 'Maintenance of cell pluripotency, differentiation, and reprogramming are regulated by complex gene regulatory networks (GRNs) including monoallelically-expressed imprinted genes.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28052084',\n",
       "     'score': 0.37089645862579346},\n",
       "    {'start': 179,\n",
       "     'end': 289,\n",
       "     'text': 'Besides transcriptional control, epigenetic modifications and microRNAs contribute to cellular differentiation.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28052084',\n",
       "     'score': 0.12057551741600037},\n",
       "    {'start': 291,\n",
       "     'end': 531,\n",
       "     'text': 'As a model system for studying the capacity of cells to preserve their pluripotency state and the onset of differentiation and subsequent specialization, murine hematopoiesis was used and compared to embryonic stem cells (ESCs) as a control.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28052084',\n",
       "     'score': 0.2716447710990906},\n",
       "    {'start': 533,\n",
       "     'end': 696,\n",
       "     'text': 'Using published microarray data, the expression profiles of two sets of genes, pluripotent and imprinted, were compared to a third set of known hematopoietic genes.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28052084',\n",
       "     'score': 0.27532678842544556},\n",
       "    {'start': 698,\n",
       "     'end': 845,\n",
       "     'text': 'We found that more than half of the pluripotent and imprinted genes are clearly upregulated in ESCs but subsequently repressed during hematopoiesis.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28052084',\n",
       "     'score': 0.2780001163482666},\n",
       "    {'start': 847,\n",
       "     'end': 952,\n",
       "     'text': 'The remaining genes were either upregulated in hematopoietic progenitors or in differentiated blood cells.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28052084',\n",
       "     'score': 0.12251365184783936},\n",
       "    {'start': 954,\n",
       "     'end': 1126,\n",
       "     'text': 'The three gene sets each consist of three similarly behaving gene groups with similar expression profiles in various lineages of the hematopoietic system as well as in ESCs.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28052084',\n",
       "     'score': 0.28763651847839355},\n",
       "    {'start': 1128,\n",
       "     'end': 1344,\n",
       "     'text': 'To explain this co-regulation behavior, we explored the transcriptional and post-transcriptional mechanisms of pluripotent and imprinted genes and their regulator/target miRNAs in six different hematopoietic lineages.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28052084',\n",
       "     'score': 0.33364272117614746},\n",
       "    {'start': 1346,\n",
       "     'end': 1519,\n",
       "     'text': 'Therewith, lineage-specific transcription factor (TF)-miRNA regulatory networks were generated and their topologies and functional impacts during hematopoiesis were analyzed.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28052084',\n",
       "     'score': 0.14304688572883606},\n",
       "    {'start': 1521,\n",
       "     'end': 1756,\n",
       "     'text': 'This led to the identification of TF-miRNA co-regulatory motifs, for which we validated the contribution to the cellular development of the corresponding lineage in terms of statistical significance and relevance to biological evidence.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28052084',\n",
       "     'score': 0.12596312165260315},\n",
       "    {'start': 1758,\n",
       "     'end': 1876,\n",
       "     'text': 'This analysis also identified key miRNAs and TFs/genes that might play important roles in the derived lineage networks.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28052084',\n",
       "     'score': 0.14833849668502808},\n",
       "    {'start': 1878,\n",
       "     'end': 2205,\n",
       "     'text': 'These molecular associations suggest new aspects of the cellular regulation of the onset of cellular differentiation and during hematopoiesis involving, on one hand, pluripotent genes that were previously not discussed in the context of hematopoiesis and, on the other hand, involve genes that are related to genomic imprinting.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28052084',\n",
       "     'score': 0.3046821355819702},\n",
       "    {'start': 2207,\n",
       "     'end': 2329,\n",
       "     'text': 'These are new links between hematopoiesis and cellular differentiation and the important field of epigenetic modifications.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28052084',\n",
       "     'score': 0.1235848069190979}]},\n",
       "  {'id': '21589869',\n",
       "   'score': 8.911394119262695,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 107,\n",
       "     'text': 'RAD21 cooperates with pluripotency transcription factors in the maintenance of embryonic stem cell identity.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '21589869',\n",
       "     'score': 0.30433428287506104},\n",
       "    {'start': 0,\n",
       "     'end': 240,\n",
       "     'text': 'For self-renewal, embryonic stem cells (ESCs) require the expression of specific transcription factors accompanied by a particular chromosome organization to maintain a balance between pluripotency and the capacity for rapid differentiation.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '21589869',\n",
       "     'score': 0.3559303879737854},\n",
       "    {'start': 242,\n",
       "     'end': 349,\n",
       "     'text': 'However, how transcriptional regulation is linked to chromosome organization in ESCs is not well understood.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '21589869',\n",
       "     'score': 0.36969661712646484},\n",
       "    {'start': 351,\n",
       "     'end': 517,\n",
       "     'text': 'Here we show that the cohesin component RAD21 exhibits a functional role in maintaining ESC identity through association with the pluripotency transcriptional network.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '21589869',\n",
       "     'score': 0.31612884998321533},\n",
       "    {'start': 519,\n",
       "     'end': 743,\n",
       "     'text': 'ChIP-seq analyses of RAD21 reveal an ESC specific cohesin binding pattern that is characterized by CTCF independent co-localization of cohesin with pluripotency related transcription factors Oct4, Nanog, Sox2, Esrrb and Klf4.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '21589869',\n",
       "     'score': 0.3019070029258728},\n",
       "    {'start': 745,\n",
       "     'end': 967,\n",
       "     'text': 'Upon ESC differentiation, most of these binding sites disappear and instead new CTCF independent RAD21 binding sites emerge, which are enriched for binding sites of transcription factors implicated in early differentiation.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '21589869',\n",
       "     'score': 0.1257777214050293},\n",
       "    {'start': 969,\n",
       "     'end': 1191,\n",
       "     'text': 'Furthermore, knock-down of RAD21 causes expression changes that are similar to expression changes after Nanog depletion, demonstrating the functional relevance of the RAD21--pluripotency transcriptional network association.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '21589869',\n",
       "     'score': 0.3287307918071747},\n",
       "    {'start': 1193,\n",
       "     'end': 1345,\n",
       "     'text': 'Finally, we show that Nanog physically interacts with the cohesin or cohesin interacting proteins STAG1 and WAPL further substantiating this association.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '21589869',\n",
       "     'score': 0.2604931890964508},\n",
       "    {'start': 1347,\n",
       "     'end': 1534,\n",
       "     'text': 'Based on these findings we propose that a dynamic placement of cohesin by pluripotency transcription factors contributes to a chromosome organization supporting the ESC expression program.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '21589869',\n",
       "     'score': 0.332438588142395}]},\n",
       "  {'id': '22641845',\n",
       "   'score': 8.90600872039795,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 91,\n",
       "     'text': 'Roles of Dppa2 in the regulation of the present status and future of pluripotent stem cells.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '22641845',\n",
       "     'score': 0.3610232174396515},\n",
       "    {'start': 0,\n",
       "     'end': 156,\n",
       "     'text': 'Maintenance of undifferentiated states of pluripotent embryonic stem cells is regulated by a complex network of transcription factors and signaling pathways.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '22641845',\n",
       "     'score': 0.36264264583587646},\n",
       "    {'start': 158,\n",
       "     'end': 372,\n",
       "     'text': 'Recent reports revealed that developmental pluripotency-associated 2 (Dppa2), which regulates chromatin structures, plays important roles in the maintenance of pluripotency and proliferation of embryonic stem cells.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '22641845',\n",
       "     'score': 0.3484847843647003},\n",
       "    {'start': 374,\n",
       "     'end': 543,\n",
       "     'text': 'Interestingly, developmental pluripotency-associated 2 is involved not only in the normal development of lung but also in the pathogenesis of non-small cell lung cancers.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '22641845',\n",
       "     'score': 0.36342453956604004},\n",
       "    {'start': 545,\n",
       "     'end': 683,\n",
       "     'text': 'These results suggest that an epigenetic regulator of pluripotent stem cells plays important roles in normal development and tumorigenesis.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '22641845',\n",
       "     'score': 0.31424176692962646}]},\n",
       "  {'id': '20202218',\n",
       "   'score': 8.801796913146973,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 111,\n",
       "     'text': 'AutoSOME: a clustering method for identifying gene expression modules without prior knowledge of cluster number.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '20202218',\n",
       "     'score': 0.1210375726222992},\n",
       "    {'start': 0,\n",
       "     'end': 142,\n",
       "     'text': 'BACKGROUND\\nClustering the information content of large high-dimensional gene expression datasets has widespread application in \"omics\" biology.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '20202218',\n",
       "     'score': 0.16202124953269958},\n",
       "    {'start': 144,\n",
       "     'end': 342,\n",
       "     'text': 'Unfortunately, the underlying structure of these natural datasets is often fuzzy, and the computational identification of data clusters generally requires knowledge about cluster number and geometry.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '20202218',\n",
       "     'score': 0.1266900599002838},\n",
       "    {'start': 346,\n",
       "     'end': 545,\n",
       "     'text': 'RESULTS\\nWe integrated strategies from machine learning, cartography, and graph theory into a new informatics method for automatically clustering self-organizing map ensembles of high-dimensional data.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '20202218',\n",
       "     'score': 0.1627674102783203},\n",
       "    {'start': 547,\n",
       "     'end': 745,\n",
       "     'text': 'Our new method, called AutoSOME, readily identifies discrete and fuzzy data clusters without prior knowledge of cluster number or structure in diverse datasets including whole genome microarray data.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '20202218',\n",
       "     'score': 0.12472221255302429},\n",
       "    {'start': 747,\n",
       "     'end': 901,\n",
       "     'text': 'Visualization of AutoSOME output using network diagrams and differential heat maps reveals unexpected variation among well-characterized cancer cell lines.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '20202218',\n",
       "     'score': 0.14467346668243408},\n",
       "    {'start': 903,\n",
       "     'end': 1214,\n",
       "     'text': 'Co-expression analysis of data from human embryonic and induced pluripotent stem cells using AutoSOME identifies >3400 up-regulated genes associated with pluripotency, and indicates that a recently identified protein-protein interaction network characterizing pluripotency was underestimated by a factor of four.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '20202218',\n",
       "     'score': 0.39451032876968384},\n",
       "    {'start': 1218,\n",
       "     'end': 1462,\n",
       "     'text': 'CONCLUSIONS\\nBy effectively extracting important information from high-dimensional microarray data without prior knowledge or the need for data filtration, AutoSOME can yield systems-level insights from whole genome microarray expression studies.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '20202218',\n",
       "     'score': 0.15597054362297058},\n",
       "    {'start': 1464,\n",
       "     'end': 1636,\n",
       "     'text': 'Due to its generality, this new method should also have practical utility for a variety of data-intensive applications, including the results of deep sequencing experiments.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '20202218',\n",
       "     'score': 0.12257242202758789},\n",
       "    {'start': 1638,\n",
       "     'end': 1726,\n",
       "     'text': 'AutoSOME is available for download at http://jimcooperlab.mcdb.ucsb.edu/autosome webcite.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '20202218',\n",
       "     'score': 0.12189999222755432}]},\n",
       "  {'id': '23577087',\n",
       "   'score': 8.787529945373535,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 178,\n",
       "     'text': 'Glycogen synthase kinase-3 inhibition enhances translation of pluripotency-associated transcription factors to contribute to maintenance of mouse embryonic stem cell self-renewal.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '23577087',\n",
       "     'score': 0.3627505302429199},\n",
       "    {'start': 0,\n",
       "     'end': 167,\n",
       "     'text': 'Maintenance of embryonic stem cell (ESC) self-renewal and pluripotency are controlled by extrinsic factors, molecular signaling pathways and transcriptional regulators.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '23577087',\n",
       "     'score': 0.30400508642196655},\n",
       "    {'start': 169,\n",
       "     'end': 370,\n",
       "     'text': 'While many of the key players have been studied in depth, how the molecular signals interact with transcription factors of the pluripotency network to regulate their action remains less well understood.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '23577087',\n",
       "     'score': 0.3423035740852356},\n",
       "    {'start': 372,\n",
       "     'end': 675,\n",
       "     'text': 'Inhibition of glycogen synthase kinase 3 (Gsk-3) has been implicated in the maintenance of mouse ESC pluripotency, although there is contradictory data on its role, with enhancement of cell survival and metabolism, stabilisation of c-Myc and activation of Wnt signalling proposed as potential mechanisms.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '23577087',\n",
       "     'score': 0.3175240755081177},\n",
       "    {'start': 677,\n",
       "     'end': 856,\n",
       "     'text': 'We have discovered that suppression of Gsk-3 activity leads to enhanced protein levels of key transcriptional regulators of the pluripotency network, notably Nanog, Tbx3 and c-Myc.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '23577087',\n",
       "     'score': 0.36189576983451843},\n",
       "    {'start': 858,\n",
       "     'end': 1343,\n",
       "     'text': 'Protein stability was unchanged following Gsk-3 inhibition, although interestingly, Nanog and Tbx3 proteins were found to have half-lives of 1-3 h, while that of Oct4 protein was longer, at 6 h. We demonstrate that the effects on protein levels seen following inhibition of Gsk-3 are due to both enhanced de novo synthesis of Nanog protein and increases in the proportion of Nanog and Tbx3 RNAs bound to polysomes, findings consistent with Gsk-3 regulating translation of these factors.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '23577087',\n",
       "     'score': 0.3224031627178192},\n",
       "    {'start': 1345,\n",
       "     'end': 1499,\n",
       "     'text': \"These effects were not due to changes in regulators of general translation initiation machinery nor mediated via the 5' or 3' UTR sequences of Nanog alone.\",\n",
       "     'is_title': False,\n",
       "     'doc_id': '23577087',\n",
       "     'score': 0.3050016760826111},\n",
       "    {'start': 1501,\n",
       "     'end': 1764,\n",
       "     'text': 'The data we present provide both new conceptual insight into the mechanisms regulated by Gsk-3 that may contribute to ESC self-renewal and, importantly, establish control of protein translation as an additional mechanism involved in modulation of ESC pluripotency.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '23577087',\n",
       "     'score': 0.342555433511734}]},\n",
       "  {'id': '29999566',\n",
       "   'score': 8.785465240478516,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 109,\n",
       "     'text': 'An Optimized System for Effective Derivation of Three-Dimensional Retinal Tissue via Wnt Signaling Regulation.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '29999566',\n",
       "     'score': 0.1967775523662567},\n",
       "    {'start': 0,\n",
       "     'end': 217,\n",
       "     'text': 'Effective derivation of three-dimensional (3D) retinal tissue from human-induced pluripotent stem cells (hiPSCs) could provide models for drug screening and facilitate patient-specific retinal cell replacement therapy.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '29999566',\n",
       "     'score': 0.3850555419921875},\n",
       "    {'start': 219,\n",
       "     'end': 351,\n",
       "     'text': 'However, some hiPSC lines cannot undergo 3D self-organization and show inadequate differentiation efficiency to meet clinical demand.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '29999566',\n",
       "     'score': 0.21032124757766724},\n",
       "    {'start': 353,\n",
       "     'end': 436,\n",
       "     'text': 'In this study, we developed an optimized system for derivation of 3D retinal tissue.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '29999566',\n",
       "     'score': 0.1707819700241089},\n",
       "    {'start': 438,\n",
       "     'end': 600,\n",
       "     'text': 'We found that the Wnt signaling pathway antagonist Dickkopf-related protein 1 (DKK-1) rescued the inability of differentiated retinal progenitors to self-organize.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '29999566',\n",
       "     'score': 0.1292334496974945},\n",
       "    {'start': 602,\n",
       "     'end': 789,\n",
       "     'text': 'By evaluating DKK-1 expression and supplying DKK-1 if necessary, retinal organoids were differentiated from six hiPSC lines, which were reprogramed from three common initiating cell types.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '29999566',\n",
       "     'score': 0.1281784474849701},\n",
       "    {'start': 791,\n",
       "     'end': 908,\n",
       "     'text': 'Retinal tissues derived from the optimized system were well organized and capable of surviving for further maturation.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '29999566',\n",
       "     'score': 0.1246325671672821},\n",
       "    {'start': 910,\n",
       "     'end': 1009,\n",
       "     'text': 'Thus, using this system, we generated retinal tissues from various hiPSC lines with high efficiency.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '29999566',\n",
       "     'score': 0.1201859712600708},\n",
       "    {'start': 1011,\n",
       "     'end': 1107,\n",
       "     'text': 'This novel system has many potential applications in regenerative therapy and precision medicine.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '29999566',\n",
       "     'score': 0.12381336092948914},\n",
       "    {'start': 1109,\n",
       "     'end': 1137,\n",
       "     'text': 'Stem Cells 2018;36:1709-1722.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '29999566',\n",
       "     'score': 0.1201859712600708}]},\n",
       "  {'id': '21222196',\n",
       "   'score': 8.779594421386719,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 34,\n",
       "     'text': 'Nuclear architecture in stem cells.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '21222196',\n",
       "     'score': 0.12231820821762085},\n",
       "    {'start': 0,\n",
       "     'end': 203,\n",
       "     'text': 'Fundamental features of genome regulation depend on the linear DNA sequence, cell type specific modification of DNA and chromatin-associated proteins, which locally control the expression of single genes.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '21222196',\n",
       "     'score': 0.14852508902549744},\n",
       "    {'start': 205,\n",
       "     'end': 468,\n",
       "     'text': 'Architectural features of genome organization within the three-dimensional (3D) nuclear space establish preferential positioning of genes relative to nuclear subcompartments associated with specific biochemical activities, thereby influencing states of expression.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '21222196',\n",
       "     'score': 0.22650891542434692},\n",
       "    {'start': 470,\n",
       "     'end': 715,\n",
       "     'text': 'The structural and temporal organization of the genome within the nucleus of stem cells, together with specific features of epigenetic and transcriptional regulation are emerging as key players that influence pluripotency and differentiation.1,2.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '21222196',\n",
       "     'score': 0.38951271772384644}]},\n",
       "  {'id': '24380431',\n",
       "   'score': 8.774561882019043,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 44,\n",
       "     'text': 'Patz1 regulates embryonic stem cell identity.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '24380431',\n",
       "     'score': 0.1203262209892273},\n",
       "    {'start': 0,\n",
       "     'end': 97,\n",
       "     'text': 'Embryonic stem cells (ESCs) derived from the inner cell mass (ICM) of blastocysts are pluripotent.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24380431',\n",
       "     'score': 0.2773019075393677},\n",
       "    {'start': 99,\n",
       "     'end': 200,\n",
       "     'text': 'Pluripotency is maintained by a transcriptional network in which Oct4 and Nanog are master regulators.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24380431',\n",
       "     'score': 0.3662261962890625},\n",
       "    {'start': 202,\n",
       "     'end': 289,\n",
       "     'text': 'Notably, several zinc finger transcription factors have important roles in this network.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24380431',\n",
       "     'score': 0.14457696676254272},\n",
       "    {'start': 291,\n",
       "     'end': 417,\n",
       "     'text': 'Patz1, a BTB/POZ-domain-containing zinc finger protein, is expressed at higher levels in the ICM relative to the trophectoderm.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24380431',\n",
       "     'score': 0.13567835092544556},\n",
       "    {'start': 419,\n",
       "     'end': 480,\n",
       "     'text': 'However, its function in pluripotency has been poorly studied.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24380431',\n",
       "     'score': 0.2901040315628052},\n",
       "    {'start': 482,\n",
       "     'end': 556,\n",
       "     'text': 'Here, we show that Patz1 is an important regulator of pluripotency in ESCs.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24380431',\n",
       "     'score': 0.3411720395088196},\n",
       "    {'start': 558,\n",
       "     'end': 683,\n",
       "     'text': 'Patz1 RNAi, chromatin immunoprecipitation (ChIP), and reporter assays indicate that Patz1 directly regulates Pou5f1 and Nanog.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24380431',\n",
       "     'score': 0.2827426791191101},\n",
       "    {'start': 685,\n",
       "     'end': 845,\n",
       "     'text': 'Global transcriptome changes upon Patz1 knockdown largely involve upregulation of apoptotic genes and downregulation of cell cycle and cellular metabolism genes.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24380431',\n",
       "     'score': 0.12364223599433899},\n",
       "    {'start': 847,\n",
       "     'end': 988,\n",
       "     'text': 'Patz1 ChIP sequencing further identified more than 5,000 binding sites of Patz1 in mouse genome, from which two binding motifs were extracted.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24380431',\n",
       "     'score': 0.13018068671226501},\n",
       "    {'start': 990,\n",
       "     'end': 1121,\n",
       "     'text': 'Further, gene ontology analysis of genes associated with the binding sites displays enrichment for proximity to developmental genes.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24380431',\n",
       "     'score': 0.1267530918121338},\n",
       "    {'start': 1123,\n",
       "     'end': 1205,\n",
       "     'text': 'In addition, embryoid body assays suggest that Patz1 represses developmental genes.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24380431',\n",
       "     'score': 0.12269043922424316},\n",
       "    {'start': 1207,\n",
       "     'end': 1283,\n",
       "     'text': 'Together, these results propose that Patz1 is important for ESC pluripotency.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24380431',\n",
       "     'score': 0.2886764407157898}]},\n",
       "  {'id': '31428157',\n",
       "   'score': 8.760375022888184,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 77,\n",
       "     'text': 'Protein Kinases and Their Inhibitors in Pluripotent Stem Cell Fate Regulation.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '31428157',\n",
       "     'score': 0.32658523321151733},\n",
       "    {'start': 0,\n",
       "     'end': 193,\n",
       "     'text': 'Protein kinases modulate the reversible postmodifications of substrate proteins to their phosphorylated forms as an essential process in regulating intracellular signaling transduction cascades.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31428157',\n",
       "     'score': 0.1494547426700592},\n",
       "    {'start': 195,\n",
       "     'end': 480,\n",
       "     'text': 'Moreover, phosphorylation has recently been shown to tightly control the regulatory network of kinases responsible for the induction and maintenance of pluripotency, defined as the particular ability to differentiate pluripotent stem cells (PSCs) into every cell type in the adult body.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31428157',\n",
       "     'score': 0.3523107171058655},\n",
       "    {'start': 482,\n",
       "     'end': 670,\n",
       "     'text': 'In particular, emerging evidence indicates that the balance between the self-renewal and differentiation of PSCs is regulated by the small molecules that modulate kinase signaling pathways.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31428157',\n",
       "     'score': 0.13033080101013184},\n",
       "    {'start': 672,\n",
       "     'end': 923,\n",
       "     'text': 'Furthermore, new reprogramming technologies have been developed using kinase modulators, which have provided novel insight of the mechanisms underlying the kinase regulatory networks involved in the generation of induced pluripotent stem cells (iPSCs).',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31428157',\n",
       "     'score': 0.3503033518791199},\n",
       "    {'start': 925,\n",
       "     'end': 1184,\n",
       "     'text': 'In this review, we highlight the recent progress made in defining the roles of protein kinase signaling pathways and their small molecule modulators in regulating the pluripotent states, self-renewal, reprogramming process, and lineage differentiation of PSCs.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31428157',\n",
       "     'score': 0.3596958816051483}]},\n",
       "  {'id': '32179818',\n",
       "   'score': 8.710554122924805,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 79,\n",
       "     'text': 'The Transcription Factor-microRNA Regulatory Network during hESC-chondrogenesis.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '32179818',\n",
       "     'score': 0.14655354619026184},\n",
       "    {'start': 0,\n",
       "     'end': 96,\n",
       "     'text': 'Human embryonic stem cells (ESCs) offer a promising therapeutic approach for osteoarthritis (OA).',\n",
       "     'is_title': False,\n",
       "     'doc_id': '32179818',\n",
       "     'score': 0.27788612246513367},\n",
       "    {'start': 98,\n",
       "     'end': 263,\n",
       "     'text': 'The unlimited source of cells capable of differentiating to chondrocytes has potential for repairing damaged cartilage or to generate disease models via gene editing.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '32179818',\n",
       "     'score': 0.12423697113990784},\n",
       "    {'start': 265,\n",
       "     'end': 343,\n",
       "     'text': 'However their use is limited by the efficiency of chondrogenic differentiation.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '32179818',\n",
       "     'score': 0.12568259239196777},\n",
       "    {'start': 345,\n",
       "     'end': 517,\n",
       "     'text': 'An improved understanding of the transcriptional and post-transcriptional regulation of chondrogenesis will enable us to improve hESC chondrogenic differentiation protocols.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '32179818',\n",
       "     'score': 0.1457291543483734},\n",
       "    {'start': 519,\n",
       "     'end': 632,\n",
       "     'text': 'Small RNA-seq and whole transcriptome sequencing was performed on distinct stages of hESC-directed chondrogenesis.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '32179818',\n",
       "     'score': 0.12464895844459534},\n",
       "    {'start': 634,\n",
       "     'end': 875,\n",
       "     'text': 'This revealed significant changes in the expression of several microRNAs including upregulation of known cartilage associated microRNAs and those transcribed from the Hox complexes, and the downregulation of pluripotency associated microRNAs.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '32179818',\n",
       "     'score': 0.3660372197628021},\n",
       "    {'start': 877,\n",
       "     'end': 1151,\n",
       "     'text': 'Integration of miRomes and transcriptomes generated during hESC-directed chondrogenesis identified key functionally related clusters of co-expressed microRNAs and protein coding genes, associated with pluripotency, primitive streak, limb development and extracellular matrix.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '32179818',\n",
       "     'score': 0.32369381189346313},\n",
       "    {'start': 1153,\n",
       "     'end': 1474,\n",
       "     'text': \"Analysis identified regulators of hESC-directed chondrogenesis such as miR-29c-3p with 10 of its established targets identified as co-regulated 'ECM organisation' genes and miR-22-3p which is highly co-expressed with ECM genes and may regulate these genes indirectly by targeting the chondrogenic regulators SP1 and HDAC4.\",\n",
       "     'is_title': False,\n",
       "     'doc_id': '32179818',\n",
       "     'score': 0.16089516878128052},\n",
       "    {'start': 1476,\n",
       "     'end': 1662,\n",
       "     'text': 'We identified several upregulated transcription factors including HOXA9/A10/D13 involved in limb patterning and RELA, JUN and NFAT5, which have targets enriched with ECM associated genes.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '32179818',\n",
       "     'score': 0.1360066831111908},\n",
       "    {'start': 1664,\n",
       "     'end': 1913,\n",
       "     'text': 'We have developed an unbiased approach for integrating transcriptome and miRome using protein-protein interactions, transcription factor regulation and miRNA target interactions and identified key regulatory networks prominent in hESC chondrogenesis.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '32179818',\n",
       "     'score': 0.18140000104904175}]},\n",
       "  {'id': '28740264',\n",
       "   'score': 8.70011043548584,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 95,\n",
       "     'text': 'PRDM15 safeguards naive pluripotency by transcriptionally regulating WNT and MAPK-ERK signaling.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '28740264',\n",
       "     'score': 0.32720839977264404},\n",
       "    {'start': 0,\n",
       "     'end': 173,\n",
       "     'text': 'The transcriptional network acting downstream of LIF, WNT and MAPK-ERK to stabilize mouse embryonic stem cells (ESCs) in their naive state has been extensively characterized.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28740264',\n",
       "     'score': 0.3634990453720093},\n",
       "    {'start': 175,\n",
       "     'end': 271,\n",
       "     'text': 'However, the upstream factors regulating these three signaling pathways remain largely uncharted.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28740264',\n",
       "     'score': 0.14423120021820068},\n",
       "    {'start': 273,\n",
       "     'end': 437,\n",
       "     'text': 'PR-domain-containing proteins (PRDMs) are zinc-finger sequence-specific chromatin factors that have essential roles in embryonic development and cell fate decisions.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28740264',\n",
       "     'score': 0.12530022859573364},\n",
       "    {'start': 439,\n",
       "     'end': 574,\n",
       "     'text': 'Here we characterize the transcriptional regulator PRDM15, which acts independently of PRDM14 to regulate the naive state of mouse ESCs.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28740264',\n",
       "     'score': 0.2784096300601959},\n",
       "    {'start': 576,\n",
       "     'end': 716,\n",
       "     'text': 'Mechanistically, PRDM15 modulates WNT and MAPK-ERK signaling by directly promoting the expression of Rspo1 (R-spondin1) and Spry1 (Sprouty1).',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28740264',\n",
       "     'score': 0.1260775327682495},\n",
       "    {'start': 718,\n",
       "     'end': 966,\n",
       "     'text': 'Consistent with these findings, CRISPR-Cas9-mediated disruption of PRDM15-binding sites in the Rspo1 and Spry1 promoters recapitulates PRDM15 depletion, both in terms of local chromatin organization and the transcriptional modulation of these genes.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28740264',\n",
       "     'score': 0.16184517741203308},\n",
       "    {'start': 968,\n",
       "     'end': 1171,\n",
       "     'text': 'Collectively, our findings uncover an essential role for PRDM15 as a chromatin factor that modulates the transcription of upstream regulators of WNT and MAPK-ERK signaling to safeguard naive pluripotency.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28740264',\n",
       "     'score': 0.2829882502555847}]},\n",
       "  {'id': '31659113',\n",
       "   'score': 8.640697479248047,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 126,\n",
       "     'text': 'DNA Replication Inhibitor Geminin and Retinoic Acid Signaling Participate in Complex Interactions Associated With Pluripotency.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '31659113',\n",
       "     'score': 0.3197113871574402},\n",
       "    {'start': 0,\n",
       "     'end': 112,\n",
       "     'text': 'BACKGROUND/AIM\\nSeveral links between DNA replication, pluripotency and development have been recently identified.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31659113',\n",
       "     'score': 0.2944391965866089},\n",
       "    {'start': 114,\n",
       "     'end': 228,\n",
       "     'text': 'The involvement of miRNA in the regulation of cell cycle events and pluripotency factors has also gained attention.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31659113',\n",
       "     'score': 0.36008602380752563},\n",
       "    {'start': 232,\n",
       "     'end': 461,\n",
       "     'text': 'MATERIALS AND METHODS\\nIn the present study, we used the g:Profiler platform to analyze transcription factor binding sites, miRNA networks and protein-protein interactions to identify novel links among the aforementioned processes.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31659113',\n",
       "     'score': 0.16147297620773315},\n",
       "    {'start': 465,\n",
       "     'end': 653,\n",
       "     'text': 'RESULTS AND CONCLUSION\\nA complex circuitry between retinoic acid signaling, SWI/SNF components, pluripotency factors including Oct4, Sox2 and Nanog and cell cycle regulators was identified.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31659113',\n",
       "     'score': 0.2799634337425232},\n",
       "    {'start': 655,\n",
       "     'end': 752,\n",
       "     'text': 'It is suggested that the DNA replication inhibitor geminin plays a central role in this circuitry.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31659113',\n",
       "     'score': 0.12780997157096863}]},\n",
       "  {'id': '27146218',\n",
       "   'score': 8.638813018798828,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 59,\n",
       "     'text': 'Nanog, Oct4 and Tet1 interplay in establishing pluripotency.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '27146218',\n",
       "     'score': 0.3121066093444824},\n",
       "    {'start': 0,\n",
       "     'end': 163,\n",
       "     'text': \"A few central transcription factors inside mouse embryonic stem (ES) cells and induced pluripotent stem (iPS) cells are believed to control the cells' pluripotency.\",\n",
       "     'is_title': False,\n",
       "     'doc_id': '27146218',\n",
       "     'score': 0.2721973657608032},\n",
       "    {'start': 165,\n",
       "     'end': 271,\n",
       "     'text': 'Characterizations of pluripotent state were put forward on both transcription factor and epigenetic levels.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '27146218',\n",
       "     'score': 0.3134813606739044},\n",
       "    {'start': 273,\n",
       "     'end': 460,\n",
       "     'text': 'Whereas core players have been identified, it is desirable to map out gene regulatory networks which govern the reprogramming of somatic cells as well as the early developmental decisions.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '27146218',\n",
       "     'score': 0.15231949090957642},\n",
       "    {'start': 462,\n",
       "     'end': 652,\n",
       "     'text': 'Here we propose a multiple level model where the regulatory network of Oct4, Nanog and Tet1 includes positive feedback loops involving DNA-demethylation around the promoters of Oct4 and Tet1.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '27146218',\n",
       "     'score': 0.35640037059783936},\n",
       "    {'start': 654,\n",
       "     'end': 1168,\n",
       "     'text': 'We put forward a mechanistic understanding of the regulatory dynamics which account for i) Oct4 overexpression is sufficient to induce pluripotency in somatic cell types expressing the other Yamanaka reprogramming factors endogenously; ii) Tet1 can replace Oct4 in reprogramming cocktail; iii) Nanog is not necessary for reprogramming however its over-expression leads to enhanced self-renewal; iv) DNA methylation is the key to the regulation of pluripotency genes; v) Lif withdrawal leads to loss of pluripotency.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '27146218',\n",
       "     'score': 0.35113525390625},\n",
       "    {'start': 1170,\n",
       "     'end': 1416,\n",
       "     'text': 'Overall, our paper proposes a novel framework combining transcription regulation with DNA methylation modifications which, takes into account the multi-layer nature of regulatory mechanisms governing pluripotency acquisition through reprogramming.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '27146218',\n",
       "     'score': 0.31794777512550354}]},\n",
       "  {'id': '24998240',\n",
       "   'score': 8.566682815551758,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 88,\n",
       "     'text': 'Protein kinases and associated pathways in pluripotent state and lineage differentiation.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '24998240',\n",
       "     'score': 0.31817471981048584},\n",
       "    {'start': 0,\n",
       "     'end': 177,\n",
       "     'text': 'Protein kinases (PKs) mediate the reversible conversion of substrate proteins to phosphorylated forms, a key process in controlling intracellular signaling transduction cascades.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24998240',\n",
       "     'score': 0.13507691025733948},\n",
       "    {'start': 179,\n",
       "     'end': 358,\n",
       "     'text': 'Pluripotency is, among others, characterized by specifically expressed PKs forming a highly interconnected regulatory network that culminates in a finely-balanced molecular switch.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24998240',\n",
       "     'score': 0.35417917370796204},\n",
       "    {'start': 360,\n",
       "     'end': 511,\n",
       "     'text': 'Current high-throughput phosphoproteomic approaches have shed light on the specific regulatory PKs and their function in controlling pluripotent states.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24998240',\n",
       "     'score': 0.2677290439605713},\n",
       "    {'start': 513,\n",
       "     'end': 681,\n",
       "     'text': 'Pluripotent cell-derived endothelial and hematopoietic developments represent an example of the importance of pluripotency in cancer therapeutics and organ regeneration.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24998240',\n",
       "     'score': 0.33453667163848877},\n",
       "    {'start': 683,\n",
       "     'end': 834,\n",
       "     'text': 'This review attempts to provide the hitherto known kinome profile and the individual characterization of PK-related pathways that regulate pluripotency.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24998240',\n",
       "     'score': 0.2740197777748108},\n",
       "    {'start': 836,\n",
       "     'end': 1213,\n",
       "     'text': 'Elucidating the underlying intrinsic and extrinsic signals may improve our understanding of the different pluripotent states, the maintenance or induction of pluripotency, and the ability to tailor lineage differentiation, with a particular focus on endothelial cell differentiation for anti-cancer treatment, cell-based tissue engineering, and regenerative medicine strategies.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24998240',\n",
       "     'score': 0.31702467799186707}]},\n",
       "  {'id': '22797927',\n",
       "   'score': 8.52734375,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 57,\n",
       "     'text': 'Mechanosensitive mechanisms in transcriptional regulation.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '22797927',\n",
       "     'score': 0.14120548963546753},\n",
       "    {'start': 0,\n",
       "     'end': 144,\n",
       "     'text': 'Transcriptional regulation contributes to the maintenance of pluripotency, self-renewal and differentiation in embryonic cells and in stem cells.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '22797927',\n",
       "     'score': 0.3630203902721405},\n",
       "    {'start': 146,\n",
       "     'end': 349,\n",
       "     'text': 'Therefore, control of gene expression at the level of transcription is crucial for embryonic development, as well as for organogenesis, functional adaptation, and regeneration in adult tissues and organs.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '22797927',\n",
       "     'score': 0.13322484493255615},\n",
       "    {'start': 351,\n",
       "     'end': 560,\n",
       "     'text': 'In the past, most work has focused on how transcriptional regulation results from the complex interplay between chemical cues, adhesion signals, transcription factors and their co-regulators during development.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '22797927',\n",
       "     'score': 0.14933377504348755},\n",
       "    {'start': 562,\n",
       "     'end': 766,\n",
       "     'text': 'However, chemical signaling alone is not sufficient to explain how three-dimensional (3D) tissues and organs are constructed and maintained through the spatiotemporal control of transcriptional activities.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '22797927',\n",
       "     'score': 0.1849413812160492},\n",
       "    {'start': 768,\n",
       "     'end': 855,\n",
       "     'text': 'Accumulated evidence indicates that mechanical cues, which include physical forces (e.g.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '22797927',\n",
       "     'score': 0.12499180436134338},\n",
       "    {'start': 857,\n",
       "     'end': 1123,\n",
       "     'text': 'tension, compression or shear stress), alterations in extracellular matrix (ECM) mechanics and changes in cell shape, are transmitted to the nucleus directly or indirectly to orchestrate transcriptional activities that are crucial for embryogenesis and organogenesis.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '22797927',\n",
       "     'score': 0.12888354063034058},\n",
       "    {'start': 1125,\n",
       "     'end': 1429,\n",
       "     'text': 'In this Commentary, we review how the mechanical control of gene transcription contributes to the maintenance of pluripotency, determination of cell fate, pattern formation and organogenesis, as well as how it is involved in the control of cell and tissue function throughout embryogenesis and adult life.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '22797927',\n",
       "     'score': 0.35613518953323364},\n",
       "    {'start': 1431,\n",
       "     'end': 1592,\n",
       "     'text': 'A deeper understanding of these mechanosensitive transcriptional control mechanisms should lead to new approaches to tissue engineering and regenerative medicine.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '22797927',\n",
       "     'score': 0.1266523003578186}]},\n",
       "  {'id': '21141733',\n",
       "   'score': 8.480884552001953,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 59,\n",
       "     'text': 'Transcriptional regulatory networks in embryonic stem cells.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '21141733',\n",
       "     'score': 0.1519191861152649},\n",
       "    {'start': 0,\n",
       "     'end': 172,\n",
       "     'text': 'Transcriptional regulation is one of the most fundamental processes in biology, governing the morphology, function, and behavior of cells and thus the survival of organisms.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '21141733',\n",
       "     'score': 0.14974936842918396},\n",
       "    {'start': 174,\n",
       "     'end': 299,\n",
       "     'text': 'The embryonic stem cell (ESC) provides a good model for the understanding of transcriptional regulation in vertebrate systems.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '21141733',\n",
       "     'score': 0.14885759353637695},\n",
       "    {'start': 301,\n",
       "     'end': 452,\n",
       "     'text': 'Recent efforts have led to the identification of molecular events, which confer upon these cells the unique properties of pluripotency and self renewal.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '21141733',\n",
       "     'score': 0.32400214672088623},\n",
       "    {'start': 454,\n",
       "     'end': 566,\n",
       "     'text': 'The core regulatory network maintaining the ESC identity involves three master regulators: Oct4, Sox2, and Nanog.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '21141733',\n",
       "     'score': 0.3651459217071533},\n",
       "    {'start': 568,\n",
       "     'end': 730,\n",
       "     'text': 'Large-scale mapping studies interrogating the binding sites of these and other transcription factors showed co-occupancy of distinct sets of transcription factors.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '21141733',\n",
       "     'score': 0.1291850209236145},\n",
       "    {'start': 732,\n",
       "     'end': 879,\n",
       "     'text': 'The assembly of multitranscription factor complexes could serve as a mechanism for providing specificity in regulating ESC-specific gene expression.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '21141733',\n",
       "     'score': 0.14708253741264343},\n",
       "    {'start': 881,\n",
       "     'end': 993,\n",
       "     'text': 'These studies are also beginning to unravel the transcriptional regulatory networks that govern the ESC identity.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '21141733',\n",
       "     'score': 0.14661765098571777},\n",
       "    {'start': 995,\n",
       "     'end': 1119,\n",
       "     'text': 'Loss-of-function RNAi screens also identified novel regulatory molecules involved in the stable propagation of the ESC state.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '21141733',\n",
       "     'score': 0.1355399191379547},\n",
       "    {'start': 1121,\n",
       "     'end': 1403,\n",
       "     'text': 'This argues for an ESC transcriptional regulation program in which interconnected transcriptional regulatory networks involving large numbers of transcription factors and epigenetic modifiers work in concert on ESC- and differentiation-specific genes to achieve cell state stability.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '21141733',\n",
       "     'score': 0.17287743091583252},\n",
       "    {'start': 1405,\n",
       "     'end': 1606,\n",
       "     'text': 'This chapter traces the major efforts made over the past decade in dissecting the transcriptional regulatory network governing ESC identity and offers perspectives on the future directions of the field.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '21141733',\n",
       "     'score': 0.1503174901008606}]},\n",
       "  {'id': '31695196',\n",
       "   'score': 8.473475456237793,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 71,\n",
       "     'text': 'Dynamic lineage priming is driven via direct enhancer regulation by ERK.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '31695196',\n",
       "     'score': 0.22688543796539307},\n",
       "    {'start': 0,\n",
       "     'end': 185,\n",
       "     'text': 'Central to understanding cellular behaviour in multi-cellular organisms is the question of how a cell exits one transcriptional state to adopt and eventually become committed to another.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31695196',\n",
       "     'score': 0.13288241624832153},\n",
       "    {'start': 187,\n",
       "     'end': 442,\n",
       "     'text': 'Fibroblast growth factor-extracellular signal-regulated kinase (FGF -ERK) signalling drives differentiation of mouse embryonic stem cells (ES cells) and pre-implantation embryos towards primitive endoderm, and inhibiting ERK supports ES cell self-renewal1.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31695196',\n",
       "     'score': 0.12736180424690247},\n",
       "    {'start': 444,\n",
       "     'end': 631,\n",
       "     'text': 'Paracrine FGF-ERK signalling induces heterogeneity, whereby cells reversibly progress from pluripotency towards primitive endoderm while retaining their capacity to re-enter self-renewal2.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31695196',\n",
       "     'score': 0.308447927236557},\n",
       "    {'start': 633,\n",
       "     'end': 800,\n",
       "     'text': 'Here we find that ERK reversibly regulates transcription in ES cells by directly affecting enhancer activity without requiring a change in transcription factor binding.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31695196',\n",
       "     'score': 0.19502729177474976},\n",
       "    {'start': 802,\n",
       "     'end': 1035,\n",
       "     'text': 'ERK triggers the reversible association and disassociation of RNA polymerase II and associated co-factors from genes and enhancers with the mediator component MED24 having an essential role in ERK-dependent transcriptional regulation.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31695196',\n",
       "     'score': 0.22788551449775696},\n",
       "    {'start': 1037,\n",
       "     'end': 1284,\n",
       "     'text': 'Though the binding of mediator components responds directly to signalling, the persistent binding of pluripotency factors to both induced and repressed genes marks them for activation and/or reactivation in response to fluctuations in ERK activity.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31695196',\n",
       "     'score': 0.3259464502334595},\n",
       "    {'start': 1286,\n",
       "     'end': 1519,\n",
       "     'text': 'Among the repressed genes are several core components of the pluripotency network that act to drive their own expression and maintain the ES cell state; if their binding is lost, the ability to reactivate transcription is compromised.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31695196',\n",
       "     'score': 0.3520236015319824},\n",
       "    {'start': 1521,\n",
       "     'end': 1671,\n",
       "     'text': 'Thus, as long as transcription factor occupancy is maintained, so is plasticity, enabling cells to distinguish between transient and sustained signals.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31695196',\n",
       "     'score': 0.1280554234981537},\n",
       "    {'start': 1673,\n",
       "     'end': 1830,\n",
       "     'text': 'If ERK signalling persists, pluripotency transcription factor levels are reduced by protein turnover and irreversible gene silencing and commitment can occur.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31695196',\n",
       "     'score': 0.3071673512458801}]},\n",
       "  {'id': '32718852',\n",
       "   'score': 8.386555671691895,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 47,\n",
       "     'text': 'Taming human brain organoids one cell at a time.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '32718852',\n",
       "     'score': 0.12541189789772034},\n",
       "    {'start': 0,\n",
       "     'end': 210,\n",
       "     'text': 'Human brain organoids are self-organizing three-dimensional structures that emerge from human pluripotent stem cells and mimic aspects of the cellular composition and functionality of the developing human brain.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '32718852',\n",
       "     'score': 0.37641382217407227},\n",
       "    {'start': 212,\n",
       "     'end': 467,\n",
       "     'text': 'Despite their impressive self-organizing capacity, organoids lack the stereotypic structural anatomy of their in vivo counterpart, making conventional analysis techniques underpowered to assess cellular composition and gene network regulation in organoids.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '32718852',\n",
       "     'score': 0.17145931720733643},\n",
       "    {'start': 469,\n",
       "     'end': 705,\n",
       "     'text': 'Advances in single cell transcriptomics have recently allowed characterization and improvement of organoid protocols, as they continue to evolve, by enabling identification of cell types and states along with their developmental origins.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '32718852',\n",
       "     'score': 0.1307586431503296},\n",
       "    {'start': 707,\n",
       "     'end': 857,\n",
       "     'text': \"In this review, we summarize recent approaches, progresses and challenges in resolving brain organoid's complexity through single-cell transcriptomics.\",\n",
       "     'is_title': False,\n",
       "     'doc_id': '32718852',\n",
       "     'score': 0.12988168001174927},\n",
       "    {'start': 859,\n",
       "     'end': 1054,\n",
       "     'text': 'We then discuss emerging technologies that may complement single-cell RNA sequencing by providing additional readouts of cellular states to generate an organ-level view of developmental processes.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '32718852',\n",
       "     'score': 0.1264784336090088},\n",
       "    {'start': 1056,\n",
       "     'end': 1352,\n",
       "     'text': 'Altogether, these integrative technologies will allow monitoring of global gene regulation in thousands of individual cells and will offer an unprecedented opportunity to investigate features of human brain development and disease across multiple cellular modalities and with cell-type resolution.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '32718852',\n",
       "     'score': 0.14569297432899475}]},\n",
       "  {'id': '32215392',\n",
       "   'score': 8.381769180297852,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 115,\n",
       "     'text': 'Assembly of FN-silk with laminin-521 to integrate hPSCs into a three-dimensional culture for neural differentiation.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '32215392',\n",
       "     'score': 0.18806323409080505},\n",
       "    {'start': 0,\n",
       "     'end': 185,\n",
       "     'text': 'Three-dimensional (3D) neural tissue cultures recapitulate the basic concepts during development and disease better than what can be obtained using conventional two-dimensional cultures.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '32215392',\n",
       "     'score': 0.1900118589401245},\n",
       "    {'start': 187,\n",
       "     'end': 410,\n",
       "     'text': 'Here, we use a recombinant spider silk protein functionalized with a cell binding motif from fibronectin (FN-silk) in combination with a human recombinant laminin 521 (LN-521) to create a fully defined stem cell niche in 3D.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '32215392',\n",
       "     'score': 0.17017602920532227},\n",
       "    {'start': 412,\n",
       "     'end': 648,\n",
       "     'text': 'A novel method to assemble silk blended with LN-521 together with human pluripotent stem cells (hPSC) is used to create centimeter-sized foams, which upon cultivation develop into 3D cell constructs supported by a microfibrillar network.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '32215392',\n",
       "     'score': 0.36491045355796814},\n",
       "    {'start': 650,\n",
       "     'end': 837,\n",
       "     'text': 'After initial cell expansion, neural differentiation was induced to form a homogenous layer of continuous neuroectodermal tissue that allows further differentiation into neuronal subtypes.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '32215392',\n",
       "     'score': 0.12646859884262085},\n",
       "    {'start': 839,\n",
       "     'end': 1047,\n",
       "     'text': 'The silk-supported 3D cell constructs could then be detached from the bottom of the well and cultured as floating entities, where cells appeared in distinctive radial organization resembling early neural tube.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '32215392',\n",
       "     'score': 0.2044319212436676},\n",
       "    {'start': 1049,\n",
       "     'end': 1178,\n",
       "     'text': 'This shows that the neural progenitors retain their cellular self-organization ability in the FN-silk/LN-521-supported 3D culture.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '32215392',\n",
       "     'score': 0.20968371629714966},\n",
       "    {'start': 1180,\n",
       "     'end': 1288,\n",
       "     'text': 'Calcium imaging demonstrated spontaneous activity, which is important for the formation of neuronal networks.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '32215392',\n",
       "     'score': 0.1544681191444397},\n",
       "    {'start': 1290,\n",
       "     'end': 1456,\n",
       "     'text': 'Together, the results show that hPSCs integrated into FN-silk/LN-521 foam develop into neural progenitors and that these stay viable during long-term differentiations.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '32215392',\n",
       "     'score': 0.1290324330329895},\n",
       "    {'start': 1458,\n",
       "     'end': 1606,\n",
       "     'text': 'FN-silk/LN-521 also supports morphogenesis mimicking the human brain development and can serve as base for engineering of hPSC-derived neural tissue.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '32215392',\n",
       "     'score': 0.13067004084587097}]},\n",
       "  {'id': '25736782',\n",
       "   'score': 8.36679744720459,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 90,\n",
       "     'text': 'Proteomic and network analysis of proteins regulated by REX1 in human embryonic stem cells.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '25736782',\n",
       "     'score': 0.15493479371070862},\n",
       "    {'start': 0,\n",
       "     'end': 138,\n",
       "     'text': 'Recent studies have suggested that REX1 (reduced expression 1) plays an important role in pluripotency, proliferation, and differentiation.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '25736782',\n",
       "     'score': 0.29200220108032227},\n",
       "    {'start': 140,\n",
       "     'end': 256,\n",
       "     'text': 'However, the molecular mechanisms involved in REX1-dependent regulation of diverse cellular processes remain unclear.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '25736782',\n",
       "     'score': 0.15819910168647766},\n",
       "    {'start': 258,\n",
       "     'end': 428,\n",
       "     'text': 'To elucidate the regulatory functions of REX1 in human embryonic stem cells (hESCs), comparative proteomic analysis was performed on REX1 RNAi specifically silenced hESCs.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '25736782',\n",
       "     'score': 0.3034142851829529},\n",
       "    {'start': 430,\n",
       "     'end': 744,\n",
       "     'text': 'Analysis of the proteome via nano-LC-MS/MS identified 140 differentially expressed proteins (DEPs) displaying a >2-fold difference in expression level between control and REX1 knockdown (KD) hESCs, which were then compared with transcriptome data and validated by quantitative real-time RT-PCR and Western blotting.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '25736782',\n",
       "     'score': 0.27733102440834045},\n",
       "    {'start': 746,\n",
       "     'end': 905,\n",
       "     'text': 'These DEPs were analyzed by GO, pathway, and functional clustering analyses to determine the molecular functions of the proteins and pathways regulated by REX1.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '25736782',\n",
       "     'score': 0.13070592284202576},\n",
       "    {'start': 907,\n",
       "     'end': 1056,\n",
       "     'text': 'The REX1 KD-mediated DEPs mapped to major biological processes involved in the regulation of ribosome-mediated translation and mitochondrial function.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '25736782',\n",
       "     'score': 0.16216900944709778},\n",
       "    {'start': 1058,\n",
       "     'end': 1283,\n",
       "     'text': 'Functional network analysis revealed a highly interconnected network among these DEPs and indicated that these interconnected proteins are predominantly involved in translation and the regulation of mitochondrial organization.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '25736782',\n",
       "     'score': 0.2326642870903015},\n",
       "    {'start': 1285,\n",
       "     'end': 1527,\n",
       "     'text': 'These findings regarding REX1-mediated regulatory network have revealed the contributions of REX1 to maintaining the status of hESCs and have improved our understanding of the molecular events that underlie the fundamental properties of hESCs.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '25736782',\n",
       "     'score': 0.34918534755706787}]},\n",
       "  {'id': '27889317',\n",
       "   'score': 8.35023307800293,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 104,\n",
       "     'text': 'The p53 Family Coordinates Wnt and Nodal Inputs in Mesendodermal Differentiation of Embryonic Stem Cells.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '27889317',\n",
       "     'score': 0.13433071970939636},\n",
       "    {'start': 0,\n",
       "     'end': 224,\n",
       "     'text': 'In this study, we outline a regulatory network that involves the p53 tumor suppressor family and the Wnt pathway acting together with the TGF-β pathway in mesendodermal differentiation of mouse and human embryonic stem cells.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '27889317',\n",
       "     'score': 0.15927597880363464},\n",
       "    {'start': 226,\n",
       "     'end': 400,\n",
       "     'text': 'Knockout of all three members, p53, p63, and p73, shows that the p53 family is essential for mesendoderm specification during exit from pluripotency in embryos and in culture.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '27889317',\n",
       "     'score': 0.3394626975059509},\n",
       "    {'start': 402,\n",
       "     'end': 584,\n",
       "     'text': 'Wnt3 and its receptor Fzd1 are direct p53 family target genes in this context, and induction of Wnt\\xa0signaling by p53 is critical for activation of\\xa0mesendodermal differentiation genes.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '27889317',\n",
       "     'score': 0.1339360773563385},\n",
       "    {'start': 586,\n",
       "     'end': 764,\n",
       "     'text': 'Globally, Wnt3-activated Tcf3 and nodal-activated Smad2/3 transcription factors depend on each other for co-occupancy of target enhancers associated with key differentiation loci.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '27889317',\n",
       "     'score': 0.21422529220581055},\n",
       "    {'start': 766,\n",
       "     'end': 1021,\n",
       "     'text': 'Our results therefore highlight an unanticipated role for p53 family proteins in a regulatory network that integrates essential Wnt-Tcf and nodal-Smad inputs in a selective and interdependent way to drive mesendodermal differentiation of pluripotent cells.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '27889317',\n",
       "     'score': 0.3530239760875702}]},\n",
       "  {'id': '25818411',\n",
       "   'score': 8.315384864807129,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 88,\n",
       "     'text': 'On human pluripotent stem cell control: The rise of 3D bioengineering and mechanobiology.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '25818411',\n",
       "     'score': 0.35448741912841797},\n",
       "    {'start': 0,\n",
       "     'end': 147,\n",
       "     'text': 'Human pluripotent stem cells (hPSCs) provide promising resources for regenerating tissues and organs and modeling development and diseases in vitro.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '25818411',\n",
       "     'score': 0.27423030138015747},\n",
       "    {'start': 149,\n",
       "     'end': 363,\n",
       "     'text': 'To fulfill their promise, the fate, function, and organization of hPSCs need to be precisely regulated in a three-dimensional (3D) environment to mimic cellular structures and functions of native tissues and organs.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '25818411',\n",
       "     'score': 0.23433026671409607},\n",
       "    {'start': 365,\n",
       "     'end': 527,\n",
       "     'text': 'In the past decade, innovations in 3D culture systems with functional biomaterials have enabled efficient and versatile control of hPSC fate at the cellular level.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '25818411',\n",
       "     'score': 0.18543899059295654},\n",
       "    {'start': 529,\n",
       "     'end': 670,\n",
       "     'text': 'However, we are just at the beginning of bringing hPSC-based regeneration and development and disease modeling to the tissue and organ levels.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '25818411',\n",
       "     'score': 0.13100382685661316},\n",
       "    {'start': 672,\n",
       "     'end': 886,\n",
       "     'text': 'In this review, we summarize existing bioengineered culture platforms for controlling hPSC fate and function by regulating inductive mechanical and biochemical cues coexisting in the synthetic cell microenvironment.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '25818411',\n",
       "     'score': 0.151963472366333},\n",
       "    {'start': 888,\n",
       "     'end': 1047,\n",
       "     'text': 'We highlight recent excitements in developing 3D hPSC-based in vitro tissue and organ models with in vivo-like cellular structures, interactions, and functions.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '25818411',\n",
       "     'score': 0.17055419087409973},\n",
       "    {'start': 1049,\n",
       "     'end': 1268,\n",
       "     'text': 'We further discuss an emerging multifaceted mechanotransductive signaling network--with transcriptional coactivators YAP and TAZ at the center stage--that regulate fates and behaviors of mammalian cells, including hPSCs.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '25818411',\n",
       "     'score': 0.15300211310386658},\n",
       "    {'start': 1270,\n",
       "     'end': 1520,\n",
       "     'text': 'Future development of 3D biomaterial systems should incorporate dynamically modulated mechanical and chemical properties targeting specific intracellular signaling events leading to desirable hPSC fate patterning and functional tissue formation in 3D.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '25818411',\n",
       "     'score': 0.17026853561401367}]},\n",
       "  {'id': '22820889',\n",
       "   'score': 8.289949417114258,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 61,\n",
       "     'text': 'Navigating the epigenetic landscape of pluripotent stem cells.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '22820889',\n",
       "     'score': 0.3059931993484497},\n",
       "    {'start': 0,\n",
       "     'end': 236,\n",
       "     'text': 'Pluripotent stem cells, which include embryonic stem cells and induced pluripotent stem cells, use a complex network of genetic and epigenetic pathways to maintain a delicate balance between self-renewal and multilineage differentiation.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '22820889',\n",
       "     'score': 0.3193248212337494},\n",
       "    {'start': 238,\n",
       "     'end': 366,\n",
       "     'text': 'Recently developed high-throughput genomic tools greatly facilitate the study of epigenetic regulation in pluripotent stem cells.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '22820889',\n",
       "     'score': 0.34369325637817383},\n",
       "    {'start': 368,\n",
       "     'end': 501,\n",
       "     'text': 'Increasing evidence suggests the existence of extensive crosstalk among epigenetic pathways that modify DNA, histones and nucleosomes.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '22820889',\n",
       "     'score': 0.13160884380340576},\n",
       "    {'start': 503,\n",
       "     'end': 818,\n",
       "     'text': 'Novel methods of mapping higher-order chromatin structure and chromatin-nuclear matrix interactions also provide the first insight into the three-dimensional organization of the genome and a framework in which existing genomic data of epigenetic regulation can be integrated to discover new rules of gene regulation.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '22820889',\n",
       "     'score': 0.2500002384185791}]},\n",
       "  {'id': '27664238',\n",
       "   'score': 8.231114387512207,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 94,\n",
       "     'text': 'Cap-independent translation by DAP5 controls cell fate decisions in human embryonic stem cells.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '27664238',\n",
       "     'score': 0.12957125902175903},\n",
       "    {'start': 0,\n",
       "     'end': 100,\n",
       "     'text': 'Multiple transcriptional and epigenetic changes drive differentiation of embryonic stem cells (ESCs).',\n",
       "     'is_title': False,\n",
       "     'doc_id': '27664238',\n",
       "     'score': 0.2989666163921356},\n",
       "    {'start': 102,\n",
       "     'end': 249,\n",
       "     'text': 'This study unveils an additional level of gene expression regulation involving noncanonical, cap-independent translation of a select group of mRNAs.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '27664238',\n",
       "     'score': 0.14663276076316833},\n",
       "    {'start': 251,\n",
       "     'end': 384,\n",
       "     'text': 'This is driven by death-associated protein 5 (DAP5/eIF4G2/NAT1), a translation initiation factor mediating IRES-dependent translation.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '27664238',\n",
       "     'score': 0.14073753356933594},\n",
       "    {'start': 386,\n",
       "     'end': 618,\n",
       "     'text': 'We found that the DAP5 knockdown from human ESCs (hESCs) resulted in persistence of pluripotent gene expression, delayed induction of differentiation-associated genes in different cell lineages, and defective embryoid body formation.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '27664238',\n",
       "     'score': 0.34000664949417114},\n",
       "    {'start': 620,\n",
       "     'end': 727,\n",
       "     'text': 'The latter involved improper cellular organization, lack of cavitation, and enhanced mislocalized apoptosis.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '27664238',\n",
       "     'score': 0.16661763191223145},\n",
       "    {'start': 729,\n",
       "     'end': 853,\n",
       "     'text': 'RNA sequencing of polysome-associated mRNAs identified candidates with reduced translation efficiency in DAP5-depleted hESCs.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '27664238',\n",
       "     'score': 0.3179702162742615},\n",
       "    {'start': 855,\n",
       "     'end': 1126,\n",
       "     'text': 'These were enriched in mitochondrial proteins involved in oxidative respiration, a pathway essential for differentiation, the significance of which was confirmed by the aberrant mitochondrial morphology and decreased oxidative respiratory activity in DAP5 knockdown cells.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '27664238',\n",
       "     'score': 0.14124780893325806},\n",
       "    {'start': 1128,\n",
       "     'end': 1283,\n",
       "     'text': 'Further analysis identified the chromatin modifier HMGN3 as a cap-independent DAP5 translation target whose knockdown resulted in defective differentiation.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '27664238',\n",
       "     'score': 0.13116124272346497},\n",
       "    {'start': 1285,\n",
       "     'end': 1502,\n",
       "     'text': 'Thus, DAP5-mediated translation of a specific set of proteins is critical for the transition from pluripotency to differentiation, highlighting the importance of cap-independent translation in stem cell fate decisions.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '27664238',\n",
       "     'score': 0.3211459517478943}]},\n",
       "  {'id': '19551524',\n",
       "   'score': 8.122164726257324,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 107,\n",
       "     'text': 'The high mobility group protein HMGA2: a co-regulator of chromatin structure and pluripotency in stem cells?',\n",
       "     'is_title': True,\n",
       "     'doc_id': '19551524',\n",
       "     'score': 0.34748154878616333},\n",
       "    {'start': 0,\n",
       "     'end': 162,\n",
       "     'text': 'The small, chromatin-associated HMGA proteins contain three separate DNA binding domains, so-called AT hooks, which bind preferentially to short AT-rich sequences.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '19551524',\n",
       "     'score': 0.14240971207618713},\n",
       "    {'start': 164,\n",
       "     'end': 312,\n",
       "     'text': 'These proteins are abundant in pluripotent embryonic stem (ES) cells and most malignant human tumors, but are not detectable in normal somatic cells.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '19551524',\n",
       "     'score': 0.28186798095703125},\n",
       "    {'start': 314,\n",
       "     'end': 513,\n",
       "     'text': 'They act both as activator and repressor of gene expression, and most likely facilitate DNA architectural changes during formation of specialized nucleoprotein structures at selected promoter regions.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '19551524',\n",
       "     'score': 0.20867693424224854},\n",
       "    {'start': 515,\n",
       "     'end': 704,\n",
       "     'text': 'For example, HMGA2 is involved in transcriptional activation of certain cell proliferation genes, which likely contributes to its well-established oncogenic potential during tumor formation.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '19551524',\n",
       "     'score': 0.14384377002716064},\n",
       "    {'start': 706,\n",
       "     'end': 861,\n",
       "     'text': 'However, surprisingly little is known about how HMGA proteins bind DNA packaged in chromatin and how this affects the chromatin structure at a larger scale.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '19551524',\n",
       "     'score': 0.13657796382904053},\n",
       "    {'start': 863,\n",
       "     'end': 963,\n",
       "     'text': 'Experimental evidence suggests that HMGA2 competes with binding of histone H1 in the chromatin fiber.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '19551524',\n",
       "     'score': 0.1348886489868164},\n",
       "    {'start': 965,\n",
       "     'end': 1102,\n",
       "     'text': 'This could substantially alter chromatin domain structures in ES cells and contribute to the activation of certain transcription networks.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '19551524',\n",
       "     'score': 0.16274115443229675},\n",
       "    {'start': 1104,\n",
       "     'end': 1220,\n",
       "     'text': 'HMGA2 also seems capable of recruiting enzymes directly involved in histone modifications to trigger gene expression.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '19551524',\n",
       "     'score': 0.1381511688232422},\n",
       "    {'start': 1222,\n",
       "     'end': 1351,\n",
       "     'text': 'Furthermore, it was shown that multiple HMGA2 molecules bind stably to a single nucleosome core particle whose structure is known.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '19551524',\n",
       "     'score': 0.13096222281455994},\n",
       "    {'start': 1353,\n",
       "     'end': 1446,\n",
       "     'text': 'How these features of HMGA2 impinge on chromatin organization inside a living cell is unknown.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '19551524',\n",
       "     'score': 0.16340762376785278},\n",
       "    {'start': 1448,\n",
       "     'end': 1685,\n",
       "     'text': 'In this commentary, we propose that HMGA2, through the action of three independent DNA binding domains, substantially contributes to the plasticity of ES cell chromatin and is involved in the maintenance of a un-differentiated cell state.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '19551524',\n",
       "     'score': 0.1536206305027008}]},\n",
       "  {'id': '28196802',\n",
       "   'score': 8.021608352661133,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 68,\n",
       "     'text': 'Metabolic remodeling during the loss and acquisition of pluripotency.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '28196802',\n",
       "     'score': 0.3320280909538269},\n",
       "    {'start': 0,\n",
       "     'end': 183,\n",
       "     'text': 'Pluripotent cells from the early stages of embryonic development have the unlimited capacity to self-renew and undergo differentiation into all of the cell types of the adult organism.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28196802',\n",
       "     'score': 0.30469852685928345},\n",
       "    {'start': 185,\n",
       "     'end': 401,\n",
       "     'text': 'These properties are regulated by tightly controlled networks of gene expression, which in turn are governed by the availability of transcription factors and their interaction with the underlying epigenetic landscape.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28196802',\n",
       "     'score': 0.16373902559280396},\n",
       "    {'start': 403,\n",
       "     'end': 558,\n",
       "     'text': 'Recent data suggest that, perhaps unexpectedly, some key epigenetic marks, and thereby gene expression, are regulated by the levels of specific metabolites.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28196802',\n",
       "     'score': 0.13528311252593994},\n",
       "    {'start': 560,\n",
       "     'end': 696,\n",
       "     'text': 'Hence, cellular metabolism plays a vital role beyond simply the production of energy, and may be involved in the regulation of cell fate.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28196802',\n",
       "     'score': 0.1640891432762146},\n",
       "    {'start': 698,\n",
       "     'end': 1014,\n",
       "     'text': 'In this Review, we discuss the metabolic changes that occur during the transitions between different pluripotent states both in vitro and in vivo, including during reprogramming to pluripotency and the onset of differentiation, and we discuss the extent to which distinct metabolites might regulate these transitions.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28196802',\n",
       "     'score': 0.32108139991760254}]},\n",
       "  {'id': '26417860',\n",
       "   'score': 8.011797904968262,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 63,\n",
       "     'text': 'PRC1 proteins orchestrate three-dimensional genome architecture.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '26417860',\n",
       "     'score': 0.1906348466873169},\n",
       "    {'start': 0,\n",
       "     'end': 144,\n",
       "     'text': 'The three-dimensional organization of the genome has an important role in orchestrating gene expression, but its regulation is poorly understood.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '26417860',\n",
       "     'score': 0.27219128608703613},\n",
       "    {'start': 146,\n",
       "     'end': 316,\n",
       "     'text': 'Now, a new study uncovers a major role for Polycomb components of the PRC1 complex in organizing physical networks of genes that are co-repressed to maintain pluripotency.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '26417860',\n",
       "     'score': 0.3448425233364105}]},\n",
       "  {'id': '28726548',\n",
       "   'score': 7.960502624511719,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 111,\n",
       "     'text': 'Wnt/Yes-Associated Protein Interactions During Neural Tissue Patterning of Human Induced Pluripotent Stem Cells.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '28726548',\n",
       "     'score': 0.33925867080688477},\n",
       "    {'start': 0,\n",
       "     'end': 135,\n",
       "     'text': 'Human induced pluripotent stem cells (hiPSCs) have special ability to self-assemble into neural spheroids or mini-brain-like structures.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28726548',\n",
       "     'score': 0.2946568727493286},\n",
       "    {'start': 137,\n",
       "     'end': 304,\n",
       "     'text': 'During the self-assembly process, Wnt signaling plays an important role in regional patterning and establishing positional identity of hiPSC-derived neural progenitors.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28726548',\n",
       "     'score': 0.1388946771621704},\n",
       "    {'start': 306,\n",
       "     'end': 521,\n",
       "     'text': 'Recently, the role of Wnt signaling in regulating Yes-associated protein (YAP) expression (nuclear or cytoplasmic), the pivotal regulator during organ growth and tissue generation, has attracted increasing interests.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28726548',\n",
       "     'score': 0.1723305881023407},\n",
       "    {'start': 523,\n",
       "     'end': 642,\n",
       "     'text': 'However, the interactions between Wnt and YAP expression for neural lineage commitment of hiPSCs remain poorly explored.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28726548',\n",
       "     'score': 0.13696685433387756},\n",
       "    {'start': 644,\n",
       "     'end': 827,\n",
       "     'text': 'The objective of this study is to investigate the effects of Wnt signaling and YAP expression on the cellular population in three-dimensional (3D) neural spheroids derived from hiPSCs.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28726548',\n",
       "     'score': 0.20046281814575195},\n",
       "    {'start': 829,\n",
       "     'end': 974,\n",
       "     'text': 'In this study, Wnt signaling was activated using CHIR99021 for 3D neural spheroids derived from human iPSK3 cells through embryoid body formation.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28726548',\n",
       "     'score': 0.18246540427207947},\n",
       "    {'start': 976,\n",
       "     'end': 1130,\n",
       "     'text': 'Our results indicate that Wnt activation induces nuclear localization of YAP and upregulates the expression of HOXB4, the marker for hindbrain/spinal cord.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28726548',\n",
       "     'score': 0.13647699356079102},\n",
       "    {'start': 1132,\n",
       "     'end': 1245,\n",
       "     'text': 'By contrast, the cells exhibit more rostral forebrain neural identity (expression of TBR1) without Wnt activation.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28726548',\n",
       "     'score': 0.13439282774925232},\n",
       "    {'start': 1247,\n",
       "     'end': 1355,\n",
       "     'text': 'Cytochalasin D was then used to induce cytoplasmic YAP and the results showed the decreased HOXB4 expression.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28726548',\n",
       "     'score': 0.13506647944450378},\n",
       "    {'start': 1357,\n",
       "     'end': 1488,\n",
       "     'text': 'In addition, the incorporation of microparticles in the neural spheroids was investigated for the perturbation of neural patterning.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28726548',\n",
       "     'score': 0.1369967758655548},\n",
       "    {'start': 1490,\n",
       "     'end': 1722,\n",
       "     'text': 'This study may indicate the bidirectional interactions of Wnt signaling and YAP expression during neural tissue patterning, which have the significance in neurological disease modeling, drug screening, and neural tissue regeneration.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28726548',\n",
       "     'score': 0.14020735025405884}]},\n",
       "  {'id': '12789278',\n",
       "   'score': 7.949369430541992,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 110,\n",
       "     'text': 'Microphthalamia-associated transcription factor: a critical regulator of pigment cell development and survival.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '12789278',\n",
       "     'score': 0.13920381665229797},\n",
       "    {'start': 0,\n",
       "     'end': 162,\n",
       "     'text': 'The microphthalamia-associated transcription factor (MITF) is an integral transcriptional regulator in melanocyte, the lineage from which melanoma cells originate.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '12789278',\n",
       "     'score': 0.143486887216568},\n",
       "    {'start': 164,\n",
       "     'end': 339,\n",
       "     'text': 'This basic-helix-loop-helix-leucine-zipper (bHLHzip) protein is critical for melanocyte cell-fate choice during commitment from pluripotent precursor cells in the neural crest.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '12789278',\n",
       "     'score': 0.35395193099975586},\n",
       "    {'start': 341,\n",
       "     'end': 561,\n",
       "     'text': 'Its role in differentiation pathways has been highlighted by its potent transcriptional and lineage-specific regulation of the three major pigment enzymes: tyrosinase, Tyrp1, and Dct as well as other pigmentation factors.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '12789278',\n",
       "     'score': 0.1626070737838745},\n",
       "    {'start': 563,\n",
       "     'end': 754,\n",
       "     'text': 'However, the cellular functions of MITF seem to be wider than differentiation and cell-fate pathways alone, since melanocytes and melanoma cells appear to require an expression of this factor.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '12789278',\n",
       "     'score': 0.13690334558486938},\n",
       "    {'start': 756,\n",
       "     'end': 902,\n",
       "     'text': 'Here, we discuss the transcriptional networks in which MITF is thought to reside and describe signaling pathways in the cell which impinge on MITF.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '12789278',\n",
       "     'score': 0.16772770881652832},\n",
       "    {'start': 904,\n",
       "     'end': 1062,\n",
       "     'text': 'Accumulating evidence supports the notion that MITF is involved in survival pathways during normal development as well as during neoplastic growth of melanoma.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '12789278',\n",
       "     'score': 0.14641043543815613}]},\n",
       "  {'id': '26323060',\n",
       "   'score': 7.937893867492676,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 90,\n",
       "     'text': 'Polycomb repressive complex PRC1 spatially constrains the mouse embryonic stem cell genome.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '26323060',\n",
       "     'score': 0.13326194882392883},\n",
       "    {'start': 0,\n",
       "     'end': 157,\n",
       "     'text': 'The Polycomb repressive complexes PRC1 and PRC2 maintain embryonic stem cell (ESC) pluripotency by silencing lineage-specifying developmental regulator genes.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '26323060',\n",
       "     'score': 0.33374887704849243},\n",
       "    {'start': 159,\n",
       "     'end': 261,\n",
       "     'text': 'Emerging evidence suggests that Polycomb complexes act through controlling spatial genome organization.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '26323060',\n",
       "     'score': 0.15924254059791565},\n",
       "    {'start': 263,\n",
       "     'end': 407,\n",
       "     'text': 'We show that PRC1 functions as a master regulator of mouse ESC genome architecture by organizing genes in three-dimensional interaction networks.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '26323060',\n",
       "     'score': 0.23159152269363403},\n",
       "    {'start': 409,\n",
       "     'end': 579,\n",
       "     'text': 'The strongest spatial network is composed of the four Hox gene clusters and early developmental transcription factor genes, the majority of which contact poised enhancers.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '26323060',\n",
       "     'score': 0.24867889285087585},\n",
       "    {'start': 581,\n",
       "     'end': 685,\n",
       "     'text': 'Removal of Polycomb repression leads to disruption of promoter-promoter contacts in the Hox gene network.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '26323060',\n",
       "     'score': 0.2731490135192871},\n",
       "    {'start': 687,\n",
       "     'end': 934,\n",
       "     'text': 'In contrast, promoter-enhancer contacts are maintained in the absence of Polycomb repression, with accompanying widespread acquisition of active chromatin signatures at network enhancers and pronounced transcriptional upregulation of network genes.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '26323060',\n",
       "     'score': 0.2736440896987915},\n",
       "    {'start': 936,\n",
       "     'end': 1070,\n",
       "     'text': 'Thus, PRC1 physically constrains developmental transcription factor genes and their enhancers in a silenced but poised spatial network.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '26323060',\n",
       "     'score': 0.23642989993095398},\n",
       "    {'start': 1072,\n",
       "     'end': 1213,\n",
       "     'text': 'We propose that the selective release of genes from this spatial network underlies cell fate specification during early embryonic development.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '26323060',\n",
       "     'score': 0.15765175223350525}]},\n",
       "  {'id': '23277454',\n",
       "   'score': 7.814926624298096,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 104,\n",
       "     'text': 'Poly(ADP-ribose) polymerase 1 regulates nuclear reprogramming and promotes iPSC generation without c-Myc.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '23277454',\n",
       "     'score': 0.13552239537239075},\n",
       "    {'start': 0,\n",
       "     'end': 152,\n",
       "     'text': 'Poly(ADP-ribose) polymerase 1 (Parp1) catalyzes poly(ADP-ribosylation) (PARylation) and induces replication networks involved in multiple nuclear events.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '23277454',\n",
       "     'score': 0.17222949862480164},\n",
       "    {'start': 154,\n",
       "     'end': 406,\n",
       "     'text': 'Using mass spectrometry and Western blotting, Parp1 and PARylation activity were intensively detected in induced pluripotent stem cells (iPSCs) and embryonic stem cells, but they were lower in mouse embryonic fibroblasts (MEFs) and differentiated cells.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '23277454',\n",
       "     'score': 0.27230048179626465},\n",
       "    {'start': 408,\n",
       "     'end': 563,\n",
       "     'text': 'We show that knockdown of Parp1 and pharmacological inhibition of PARylation both reduced the efficiency of iPSC generation induced by Oct4/Sox2/Klf4/c-Myc.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '23277454',\n",
       "     'score': 0.3131505250930786},\n",
       "    {'start': 565,\n",
       "     'end': 661,\n",
       "     'text': 'Furthermore, Parp1 is able to replace Klf4 or c-Myc to enhance the efficiency of iPSC generation.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '23277454',\n",
       "     'score': 0.13825595378875732},\n",
       "    {'start': 663,\n",
       "     'end': 764,\n",
       "     'text': 'In addition, mouse iPSCs generated from Oct4/Sox2/Parp1-overexpressing MEFs formed chimeric offspring.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '23277454',\n",
       "     'score': 0.32201915979385376},\n",
       "    {'start': 766,\n",
       "     'end': 892,\n",
       "     'text': 'Notably, the endogenous Parp1 and PARylation activity was enhanced by overexpression of c-Myc and repressed by c-Myc knockdown.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '23277454',\n",
       "     'score': 0.13864994049072266},\n",
       "    {'start': 894,\n",
       "     'end': 994,\n",
       "     'text': 'A chromatin immunoprecipitation assay revealed a direct interaction of c-Myc with the Parp1 promoter.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '23277454',\n",
       "     'score': 0.21714571118354797},\n",
       "    {'start': 996,\n",
       "     'end': 1211,\n",
       "     'text': 'PAR-resin pulldown, followed by proteomic analysis, demonstrated high levels of PARylated Chd1L, DNA ligase III, SSrp1, Xrcc-6/Ku70, and Parp2 in pluripotent cells, which decreased during the differentiation process.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '23277454',\n",
       "     'score': 0.2946053743362427},\n",
       "    {'start': 1213,\n",
       "     'end': 1428,\n",
       "     'text': 'These data show that the activation of Parp1, partly regulated by endogenous c-Myc, effectively promotes iPSC production and helps to maintain a pluripotent state by posttranslationally modulating protein PARylation.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '23277454',\n",
       "     'score': 0.31614190340042114}]},\n",
       "  {'id': '31724731',\n",
       "   'score': 7.616585731506348,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 69,\n",
       "     'text': 'Super-enhancers in transcriptional regulation and genome organization.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '31724731',\n",
       "     'score': 0.3204605281352997},\n",
       "    {'start': 0,\n",
       "     'end': 187,\n",
       "     'text': 'Gene expression is precisely controlled in a stage and cell-type-specific manner, largely through the interaction between cis-regulatory elements and their associated trans-acting factors.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31724731',\n",
       "     'score': 0.14798542857170105},\n",
       "    {'start': 189,\n",
       "     'end': 425,\n",
       "     'text': \"Where these components aggregate in promoters and enhancers, they are able to cooperate to modulate chromatin structure and support the engagement in long-range 3D superstructures that shape the dynamics of a cell's genomic architecture.\",\n",
       "     'is_title': False,\n",
       "     'doc_id': '31724731',\n",
       "     'score': 0.260100781917572},\n",
       "    {'start': 427,\n",
       "     'end': 653,\n",
       "     'text': \"Recently, the term 'super-enhancer' has been introduced to describe a hyper-active regulatory domain comprising a complex array of sequence elements that work together to control the key gene networks involved in cell identity.\",\n",
       "     'is_title': False,\n",
       "     'doc_id': '31724731',\n",
       "     'score': 0.27670320868492126},\n",
       "    {'start': 655,\n",
       "     'end': 848,\n",
       "     'text': 'Here, we survey the unique characteristics of super-enhancers compared to other enhancer types and summarize the recent advances in our understanding of their biological role in gene regulation.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31724731',\n",
       "     'score': 0.23841539025306702},\n",
       "    {'start': 850,\n",
       "     'end': 1051,\n",
       "     'text': 'In particular, we discuss their capacity to attract the formation of phase-separated condensates, and capacity to generate three-dimensional genome structures that precisely activate their target genes.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31724731',\n",
       "     'score': 0.20552003383636475},\n",
       "    {'start': 1053,\n",
       "     'end': 1262,\n",
       "     'text': 'We also propose a multi-stage transition model to explain the evolutionary pressure driving the development of super-enhancers in complex organisms, and highlight the potential for involvement in tumorigenesis.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31724731',\n",
       "     'score': 0.21455782651901245},\n",
       "    {'start': 1264,\n",
       "     'end': 1401,\n",
       "     'text': 'Finally, we discuss more broadly the role of super-enhancers in human health disorders and related potential in therapeutic interventions.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31724731',\n",
       "     'score': 0.22466906905174255}]},\n",
       "  {'id': '28361319',\n",
       "   'score': 7.486038684844971,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 91,\n",
       "     'text': 'Making a Kidney Organoid Using the Directed Differentiation of Human Pluripotent Stem Cells.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '28361319',\n",
       "     'score': 0.30767688155174255},\n",
       "    {'start': 0,\n",
       "     'end': 114,\n",
       "     'text': 'An organoid can be defined as a three-dimensional organ-like structure formed from organ-specific progenitor cells.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28361319',\n",
       "     'score': 0.19676324725151062},\n",
       "    {'start': 116,\n",
       "     'end': 254,\n",
       "     'text': 'Organ progenitor cells were empirically found to self-organize three-dimensional tissues when they were aggregated and cultivated in vitro.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28361319',\n",
       "     'score': 0.20012500882148743},\n",
       "    {'start': 256,\n",
       "     'end': 581,\n",
       "     'text': 'While this nature power of progenitor cells has an amazing potential to recreate artificial organs in vitro, there had been difficulty to apply this technology to human organs due to the inaccessibility to human progenitor cells until human-induced pluripotent stem cell (hiPSC) was invented by Takahashi and Yamanaka in 2007.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28361319',\n",
       "     'score': 0.28519439697265625},\n",
       "    {'start': 583,\n",
       "     'end': 738,\n",
       "     'text': 'As embryonic stem cells do, hiPSCs also have pluripotency to give rise to any organs/tissues cell types, including the kidney, via directed differentiation.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28361319',\n",
       "     'score': 0.30977320671081543},\n",
       "    {'start': 740,\n",
       "     'end': 843,\n",
       "     'text': 'Here, we provide a detailed protocol for generating kidney organoids using human pluripotent stem cells.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28361319',\n",
       "     'score': 0.2971879839897156},\n",
       "    {'start': 845,\n",
       "     'end': 937,\n",
       "     'text': 'The protocol differentiates human pluripotent stem cells into the posterior primitive streak.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28361319',\n",
       "     'score': 0.286166787147522},\n",
       "    {'start': 939,\n",
       "     'end': 1121,\n",
       "     'text': 'This is followed by the simultaneous induction of posterior and anterior intermediate mesoderm that are subsequently aggregated and undergo self-organization into the kidney organoid.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28361319',\n",
       "     'score': 0.17479455471038818},\n",
       "    {'start': 1123,\n",
       "     'end': 1371,\n",
       "     'text': 'Such kidney organoids are comprised of all anticipated kidney cell types including nephrons segmented into the glomerulus, proximal tubule, loop of Henle, and distal tubule as well as the collecting duct, endothelial network, and renal interstitium.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28361319',\n",
       "     'score': 0.16491156816482544}]},\n",
       "  {'id': '26918198',\n",
       "   'score': 7.448629379272461,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 60,\n",
       "     'text': 'Nanoengineered Platforms to Guide Pluripotent Stem Cell Fate.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '26918198',\n",
       "     'score': 0.3092994689941406},\n",
       "    {'start': 0,\n",
       "     'end': 129,\n",
       "     'text': 'Tissue engineering utilizes cells, signaling molecules, and scaffolds towards creating functional tissue to repair damaged organs.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '26918198',\n",
       "     'score': 0.1405961513519287},\n",
       "    {'start': 131,\n",
       "     'end': 301,\n",
       "     'text': 'Pluripotent stem cells (PSCs) are a promising cell source due to their ability to self-renewal indefinitely and their potential to differentiate into almost any cell type.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '26918198',\n",
       "     'score': 0.30063775181770325},\n",
       "    {'start': 303,\n",
       "     'end': 451,\n",
       "     'text': 'Great strides have been taken to parse the physiological mechanisms by which PSCs respond to their microenvironment and commit to a specific lineage.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '26918198',\n",
       "     'score': 0.1437987983226776},\n",
       "    {'start': 453,\n",
       "     'end': 681,\n",
       "     'text': 'The combination of physical cues and chemical factors is thought to have the most profound influence on stem cell behavior, therefore a major focus of tissue engineering strategies is scaffold design to incorporate these signals.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '26918198',\n",
       "     'score': 0.1452803611755371},\n",
       "    {'start': 683,\n",
       "     'end': 909,\n",
       "     'text': 'One overlooked component of the in vivo microenvironment researchers attempt to recapitulate with three dimensional (3D) substrates is the nanoarchitecture formed by the fibrillar network of extracellular matrix (ECM) proteins.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '26918198',\n",
       "     'score': 0.2397085428237915},\n",
       "    {'start': 911,\n",
       "     'end': 1026,\n",
       "     'text': 'These nanoscale features have the ability to impact cell adhesion, migration, proliferation, and lineage commitment.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '26918198',\n",
       "     'score': 0.1422543227672577},\n",
       "    {'start': 1028,\n",
       "     'end': 1284,\n",
       "     'text': 'Significant advances have been made in deciphering how these nanoscale cues interact with stem cells to determine phenotype, but much is still unknown as to how the interplay between physical and chemical signals regulate in vitro and in vivo cellular fate.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '26918198',\n",
       "     'score': 0.14635738730430603},\n",
       "    {'start': 1286,\n",
       "     'end': 1468,\n",
       "     'text': 'This review dives deeper to investigate nanoscale platforms for engineering tissue, as well use the use of these nanotechnologies to drive pluripotent stem cell lineage determination.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '26918198',\n",
       "     'score': 0.2841383218765259}]},\n",
       "  {'id': '25165871',\n",
       "   'score': 7.442572116851807,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 122,\n",
       "     'text': 'Chromatin insulator factors involved in long-range DNA interactions and their role in the folding of the Drosophila genome.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '25165871',\n",
       "     'score': 0.15337121486663818},\n",
       "    {'start': 0,\n",
       "     'end': 121,\n",
       "     'text': 'Chromatin insulators are genetic elements implicated in the organization of chromatin and the regulation of transcription.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '25165871',\n",
       "     'score': 0.2160864770412445},\n",
       "    {'start': 123,\n",
       "     'end': 255,\n",
       "     'text': 'In Drosophila, different insulator types were characterized by their locus-specific composition of insulator proteins and co-factors.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '25165871',\n",
       "     'score': 0.14510449767112732},\n",
       "    {'start': 257,\n",
       "     'end': 499,\n",
       "     'text': 'Insulators mediate specific long-range DNA contacts required for the three dimensional organization of the interphase nucleus and for transcription regulation, but the mechanisms underlying the formation of these contacts is currently unknown.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '25165871',\n",
       "     'score': 0.2787514626979828},\n",
       "    {'start': 501,\n",
       "     'end': 813,\n",
       "     'text': 'Here, we investigate the molecular associations between different components of insulator complexes (BEAF32, CP190 and Chromator) by biochemical and biophysical means, and develop a novel single-molecule assay to determine what factors are necessary and essential for the formation of long-range DNA interactions.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '25165871',\n",
       "     'score': 0.1438683569431305},\n",
       "    {'start': 815,\n",
       "     'end': 939,\n",
       "     'text': 'We show that BEAF32 is able to bind DNA specifically and with high affinity, but not to bridge long-range interactions (LRI).',\n",
       "     'is_title': False,\n",
       "     'doc_id': '25165871',\n",
       "     'score': 0.14279693365097046},\n",
       "    {'start': 941,\n",
       "     'end': 1077,\n",
       "     'text': 'In contrast, we show that CP190 and Chromator are able to mediate LRI between specifically-bound BEAF32 nucleoprotein complexes in vitro.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '25165871',\n",
       "     'score': 0.14328351616859436},\n",
       "    {'start': 1079,\n",
       "     'end': 1253,\n",
       "     'text': 'This ability of CP190 and Chromator to establish LRI requires specific contacts between BEAF32 and their C-terminal domains, and dimerization through their N-terminal domains.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '25165871',\n",
       "     'score': 0.14200228452682495},\n",
       "    {'start': 1255,\n",
       "     'end': 1399,\n",
       "     'text': 'In particular, the BTB/POZ domains of CP190 form a strict homodimer, and its C-terminal domain interacts with several insulator binding proteins.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '25165871',\n",
       "     'score': 0.14701470732688904},\n",
       "    {'start': 1401,\n",
       "     'end': 1644,\n",
       "     'text': 'We propose a general model for insulator function in which BEAF32/dCTCF/Su(HW) provide DNA specificity (first layer proteins) whereas CP190/Chromator are responsible for the physical interactions required for long-range contacts (second layer).',\n",
       "     'is_title': False,\n",
       "     'doc_id': '25165871',\n",
       "     'score': 0.1443822681903839},\n",
       "    {'start': 1646,\n",
       "     'end': 1949,\n",
       "     'text': 'This network of organized, multi-layer interactions could explain the different activities of insulators as chromatin barriers, enhancer blockers, and transcriptional regulators, and suggest a general mechanism for how insulators may shape the organization of higher-order chromatin during cell division.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '25165871',\n",
       "     'score': 0.3165259063243866}]},\n",
       "  {'id': '27760116',\n",
       "   'score': 7.407301425933838,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 85,\n",
       "     'text': 'Chromosome conformation elucidates regulatory relationships in developing human brain.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '27760116',\n",
       "     'score': 0.14131638407707214},\n",
       "    {'start': 0,\n",
       "     'end': 123,\n",
       "     'text': 'Three-dimensional physical interactions within chromosomes dynamically regulate gene expression in a tissue-specific manner.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '27760116',\n",
       "     'score': 0.20864352583885193},\n",
       "    {'start': 125,\n",
       "     'end': 331,\n",
       "     'text': 'However, the 3D organization of chromosomes during human brain development and its role in regulating gene networks dysregulated in neurodevelopmental disorders, such as autism or schizophrenia, are unknown.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '27760116',\n",
       "     'score': 0.31868794560432434},\n",
       "    {'start': 333,\n",
       "     'end': 570,\n",
       "     'text': 'Here we generate high-resolution 3D maps of chromatin contacts during human corticogenesis, permitting large-scale annotation of previously uncharacterized regulatory relationships relevant to the evolution of human cognition and disease.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '27760116',\n",
       "     'score': 0.19164606928825378},\n",
       "    {'start': 572,\n",
       "     'end': 769,\n",
       "     'text': 'Our analyses identify hundreds of genes that physically interact with enhancers gained on the human lineage, many of which are under purifying selection and associated with human cognitive function.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '27760116',\n",
       "     'score': 0.22818276286125183},\n",
       "    {'start': 771,\n",
       "     'end': 1172,\n",
       "     'text': 'We integrate chromatin contacts with non-coding variants identified in schizophrenia genome-wide association studies (GWAS), highlighting multiple candidate schizophrenia risk genes and pathways, including transcription factors involved in neurogenesis, and cholinergic signalling molecules, several of which are supported by independent expression quantitative trait loci and gene expression analyses.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '27760116',\n",
       "     'score': 0.1516258716583252},\n",
       "    {'start': 1174,\n",
       "     'end': 1361,\n",
       "     'text': 'Genome editing in human neural progenitors suggests that one of these distal schizophrenia GWAS loci regulates FOXG1 expression, supporting its potential role as a schizophrenia risk gene.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '27760116',\n",
       "     'score': 0.14247703552246094},\n",
       "    {'start': 1363,\n",
       "     'end': 1585,\n",
       "     'text': 'This work provides a framework for understanding the effect of non-coding regulatory elements on human brain development and the evolution of cognition, and highlights novel mechanisms underlying neuropsychiatric disorders.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '27760116',\n",
       "     'score': 0.14432692527770996}]},\n",
       "  {'id': '28275009',\n",
       "   'score': 7.384666919708252,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 94,\n",
       "     'text': 'Paracrine signals regulate human liver organoid maturation from induced pluripotent stem cells.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '28275009',\n",
       "     'score': 0.3126864433288574},\n",
       "    {'start': 0,\n",
       "     'end': 108,\n",
       "     'text': 'A self-organizing organoid model provides a new approach to study the mechanism of human liver organogenesis.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28275009',\n",
       "     'score': 0.14264750480651855},\n",
       "    {'start': 110,\n",
       "     'end': 250,\n",
       "     'text': 'Previous animal models documented that simultaneous paracrine signaling and cell-to-cell surface contact regulate hepatocyte differentiation.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28275009',\n",
       "     'score': 0.14155784249305725},\n",
       "    {'start': 252,\n",
       "     'end': 510,\n",
       "     'text': 'To dissect the relative contributions of the paracrine effects, we first established a liver organoid using human induced pluripotent stem cells (iPSCs), mesenchymal stem cells (MSCs) and human umbilical vein endothelial cells (HUVECs) as previously reported.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28275009',\n",
       "     'score': 0.3001575767993927},\n",
       "    {'start': 512,\n",
       "     'end': 671,\n",
       "     'text': 'Time-lapse imaging showed that hepatic-specified endoderm iPSCs (HE-iPSCs) self-assembled into three-dimensional organoids, resulting in hepatic gene induction.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28275009',\n",
       "     'score': 0.19780898094177246},\n",
       "    {'start': 673,\n",
       "     'end': 786,\n",
       "     'text': 'Progressive differentiation was demonstrated by hepatic protein production after in vivo organoid transplantation.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28275009',\n",
       "     'score': 0.14900216460227966},\n",
       "    {'start': 788,\n",
       "     'end': 927,\n",
       "     'text': 'To assess the paracrine contributions, we employed a Transwell system in which HE-iPSCs were separately co-cultured with MSCs and/or HUVECs.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28275009',\n",
       "     'score': 0.1466226875782013},\n",
       "    {'start': 929,\n",
       "     'end': 1107,\n",
       "     'text': 'Although the three-dimensional structure did not form, their soluble factors induced a hepatocyte-like phenotype in HE-iPSCs, resulting in the expression of bile salt export pump.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28275009',\n",
       "     'score': 0.20642510056495667},\n",
       "    {'start': 1109,\n",
       "     'end': 1281,\n",
       "     'text': 'In conclusion, the mesoderm-derived paracrine signals promote hepatocyte maturation in liver organoids, but organoid self-organization requires cell-to-cell surface contact.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28275009',\n",
       "     'score': 0.17506101727485657},\n",
       "    {'start': 1283,\n",
       "     'end': 1427,\n",
       "     'text': 'Our in vitro model demonstrates a novel approach to identify developmental paracrine signals regulating the differentiation of human hepatocytes.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '28275009',\n",
       "     'score': 0.1632860004901886}]},\n",
       "  {'id': '24213634',\n",
       "   'score': 7.333868980407715,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 84,\n",
       "     'text': 'Chromatin connectivity maps reveal dynamic promoter-enhancer long-range associations.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '24213634',\n",
       "     'score': 0.23710381984710693},\n",
       "    {'start': 0,\n",
       "     'end': 147,\n",
       "     'text': 'In multicellular organisms, transcription regulation is one of the central mechanisms modelling lineage differentiation and cell-fate determination.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24213634',\n",
       "     'score': 0.17123103141784668},\n",
       "    {'start': 149,\n",
       "     'end': 273,\n",
       "     'text': 'Transcription requires dynamic chromatin configurations between promoters and their corresponding distal regulatory elements.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24213634',\n",
       "     'score': 0.14240393042564392},\n",
       "    {'start': 275,\n",
       "     'end': 444,\n",
       "     'text': 'It is believed that their communication occurs within large discrete foci of aggregated RNA polymerases termed transcription factories in three-dimensional nuclear space.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24213634',\n",
       "     'score': 0.208286315202713},\n",
       "    {'start': 446,\n",
       "     'end': 551,\n",
       "     'text': 'However, the dynamic nature of chromatin connectivity has not been characterized at the genome-wide level.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24213634',\n",
       "     'score': 0.14365434646606445},\n",
       "    {'start': 553,\n",
       "     'end': 927,\n",
       "     'text': 'Here, through a chromatin interaction analysis with paired-end tagging approach using an antibody that primarily recognizes the pre-initiation complexes of RNA polymerase II, we explore the transcriptional interactomes of three mouse cells of progressive lineage commitment, including pluripotent embryonic stem cells, neural stem cells and neurosphere stem/progenitor cells.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24213634',\n",
       "     'score': 0.3128579556941986},\n",
       "    {'start': 929,\n",
       "     'end': 1121,\n",
       "     'text': 'Our global chromatin connectivity maps reveal approximately 40,000 long-range interactions, suggest precise enhancer-promoter associations and delineate cell-type-specific chromatin structures.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24213634',\n",
       "     'score': 0.22451671957969666},\n",
       "    {'start': 1123,\n",
       "     'end': 1259,\n",
       "     'text': 'Analysis of the complex regulatory repertoire shows that there are extensive colocalizations among promoters and distal-acting enhancers.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24213634',\n",
       "     'score': 0.22484946250915527},\n",
       "    {'start': 1261,\n",
       "     'end': 1459,\n",
       "     'text': 'Most of the enhancers associate with promoters located beyond their nearest active genes, indicating that the linear juxtaposition is not the only guiding principle driving enhancer target selection.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24213634',\n",
       "     'score': 0.22459429502487183},\n",
       "    {'start': 1461,\n",
       "     'end': 1644,\n",
       "     'text': 'Although promoter-enhancer interactions exhibit high cell-type specificity, promoters involved in interactions are found to be generally common and mostly active among different cells.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24213634',\n",
       "     'score': 0.2469051480293274},\n",
       "    {'start': 1646,\n",
       "     'end': 1877,\n",
       "     'text': 'Chromatin connectivity networks reveal that the pivotal genes of reprogramming functions are transcribed within physical proximity to each other in embryonic stem cells, linking chromatin architecture to coordinated gene expression.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24213634',\n",
       "     'score': 0.174902081489563},\n",
       "    {'start': 1879,\n",
       "     'end': 2020,\n",
       "     'text': 'Our study sets the stage for the full-scale dissection of spatial and temporal genome structures and their roles in orchestrating development.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24213634',\n",
       "     'score': 0.14775970578193665}]},\n",
       "  {'id': '15652148',\n",
       "   'score': 7.320226192474365,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 71,\n",
       "     'text': 'High-dimensional switches and the modelling of cellular differentiation.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '15652148',\n",
       "     'score': 0.19714757800102234},\n",
       "    {'start': 0,\n",
       "     'end': 191,\n",
       "     'text': 'Many genes have been identified as driving cellular differentiation, but because of their complex interactions, the understanding of their collective behaviour requires mathematical modelling.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '15652148',\n",
       "     'score': 0.14331617951393127},\n",
       "    {'start': 193,\n",
       "     'end': 453,\n",
       "     'text': 'Intriguingly, it has been observed in numerous developmental contexts, and particularly haematopoiesis, that genes regulating differentiation are initially co-expressed in progenitors despite their antagonism, before one is upregulated and others downregulated.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '15652148',\n",
       "     'score': 0.1641356647014618},\n",
       "    {'start': 455,\n",
       "     'end': 877,\n",
       "     'text': 'We characterise conditions under which three classes of generic \"master regulatory networks\", modelled at the molecular level after experimentally observed interactions (including bHLH protein dimerisation), and including an arbitrary number of antagonistic components, can behave as a \"multi-switch\", directing differentiation in an all-or-none fashion to a specific cell-type chosen among more than two possible outcomes.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '15652148',\n",
       "     'score': 0.18904632329940796},\n",
       "    {'start': 879,\n",
       "     'end': 1029,\n",
       "     'text': 'bHLH dimerisation networks can readily display coexistence of many antagonistic factors when competition is low (a simple characterisation is derived).',\n",
       "     'is_title': False,\n",
       "     'doc_id': '15652148',\n",
       "     'score': 0.1690405309200287},\n",
       "    {'start': 1031,\n",
       "     'end': 1349,\n",
       "     'text': 'Decision-making can be forced by a transient increase in competition, which could correspond to some unexplained experimental observations related to Id proteins; the speed of response varies with the initial conditions the network is subjected to, which could explain some aspects of cell behaviour upon reprogramming.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '15652148',\n",
       "     'score': 0.16980090737342834},\n",
       "    {'start': 1351,\n",
       "     'end': 1588,\n",
       "     'text': 'The coexistence of antagonistic factors at low levels, early in the differentiation process or in pluripotent stem cells, could be an intrinsic property of the interaction between those factors, not requiring a specific regulatory system.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '15652148',\n",
       "     'score': 0.3194504678249359}]},\n",
       "  {'id': '32611787',\n",
       "   'score': 7.291994094848633,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 63,\n",
       "     'text': 'The SUMO E3 Ligase SIZ1 Negatively Regulates Shoot Regeneration.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '32611787',\n",
       "     'score': 0.14275389909744263},\n",
       "    {'start': 0,\n",
       "     'end': 94,\n",
       "     'text': 'Plants form calluses and regenerate new organs when incubated on phytohormone-containing media.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '32611787',\n",
       "     'score': 0.14283525943756104},\n",
       "    {'start': 96,\n",
       "     'end': 358,\n",
       "     'text': 'While accumulating evidence suggests that these regenerative processes are governed by transcriptional networks orchestrating wound response and developmental transitions, it remains unknown if posttranslational regulatory mechanisms are involved in this process.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '32611787',\n",
       "     'score': 0.18269506096839905},\n",
       "    {'start': 360,\n",
       "     'end': 639,\n",
       "     'text': 'In this study, we demonstrate that SAP AND MIZ1 DOMAIN- CONTAINING LIGASE1 (SIZ1), an E3 ligase-catalyzing attachment of the SMALL UBIQUITIN-LIKE MODIFIER (SUMO) to proteins, regulates wound-induced signal transduction and organ regeneration in Arabidopsis (Arabidopsis thaliana).',\n",
       "     'is_title': False,\n",
       "     'doc_id': '32611787',\n",
       "     'score': 0.1490306258201599},\n",
       "    {'start': 641,\n",
       "     'end': 1040,\n",
       "     'text': 'We show that loss-of-function mutants for SIZ1 exhibit overproduction of shoot meristems under in vitro tissue culture conditions, while this defect is rescued in a complementation line expressing pSIZ1::SIZ1 RNA sequencing analysis revealed that siz1-2 mutants exhibit enhanced transcriptional responses to wound stress, resulting in the hyper-induction of over 400 genes immediately after wounding.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '32611787',\n",
       "     'score': 0.14854979515075684},\n",
       "    {'start': 1042,\n",
       "     'end': 1352,\n",
       "     'text': 'Among them, we show that elevated levels of WOUND INDUCED DEDIFFERENTIATION1 (WIND1) and WIND2 contribute to the enhanced shoot regeneration observed in siz1 mutants, as expression of the dominant-negative chimeric protein WIND1-SRDX (SUPERMAN repression domain) in siz1-3 mutants partly rescues this phenotype.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '32611787',\n",
       "     'score': 0.15515625476837158},\n",
       "    {'start': 1354,\n",
       "     'end': 1671,\n",
       "     'text': 'Although compromised SIZ1 function does not modify the transcription of genes implicated in auxin-induced callus formation and/or pluripotency acquisition, it does lead to enhanced induction of cytokinin-induced shoot meristem regulators such as WUSCHEL, promoting the formation of WUSCHEL-expressing foci in explants.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '32611787',\n",
       "     'score': 0.3235282301902771},\n",
       "    {'start': 1673,\n",
       "     'end': 1811,\n",
       "     'text': 'This study thus suggests that SIZ1 negatively regulates shoot regeneration in part by repressing wound-induced developmental reprogramming.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '32611787',\n",
       "     'score': 0.14303883910179138}]},\n",
       "  {'id': '18369139',\n",
       "   'score': 6.995852947235107,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 38,\n",
       "     'text': 'Gene regulation in the third dimension.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '18369139',\n",
       "     'score': 0.17092499136924744},\n",
       "    {'start': 0,\n",
       "     'end': 122,\n",
       "     'text': 'Analysis of the spatial organization of chromosomes reveals complex three-dimensional networks of chromosomal interactions.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '18369139',\n",
       "     'score': 0.30421969294548035},\n",
       "    {'start': 124,\n",
       "     'end': 322,\n",
       "     'text': 'These interactions affect gene expression at multiple levels, including long-range control by distant enhancers and repressors, coordinated expression of genes, and modification of epigenetic states.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '18369139',\n",
       "     'score': 0.23709824681282043},\n",
       "    {'start': 324,\n",
       "     'end': 491,\n",
       "     'text': 'Major challenges now include deciphering the mechanisms by which loci come together and understanding the functional consequences of these often transient associations.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '18369139',\n",
       "     'score': 0.1520516574382782}]},\n",
       "  {'id': '31216471',\n",
       "   'score': 6.87188720703125,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 106,\n",
       "     'text': 'Specific Contributions of Cohesin-SA1 and Cohesin-SA2 to TADs and Polycomb Domains in Embryonic Stem Cells.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '31216471',\n",
       "     'score': 0.15367034077644348},\n",
       "    {'start': 0,\n",
       "     'end': 62,\n",
       "     'text': 'Cohesin exists in two variants carrying either STAG/SA1 or SA2.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31216471',\n",
       "     'score': 0.1548101305961609},\n",
       "    {'start': 64,\n",
       "     'end': 338,\n",
       "     'text': 'Here we have addressed their specific contributions to the unique spatial organization of the mouse embryonic stem cell genome, which ensures super-enhancer-dependent transcription of\\xa0pluripotency factors and repression of lineage-specification genes within Polycomb domains.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31216471',\n",
       "     'score': 0.2934204339981079},\n",
       "    {'start': 340,\n",
       "     'end': 606,\n",
       "     'text': 'We find that cohesin-SA2 facilitates Polycomb domain compaction through Polycomb repressing complex 1 (PRC1) recruitment and promotes the establishment of long-range interaction networks between distant Polycomb-bound promoters that are important for gene repression.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31216471',\n",
       "     'score': 0.17637240886688232},\n",
       "    {'start': 608,\n",
       "     'end': 722,\n",
       "     'text': 'Cohesin-SA1, in contrast, disrupts these networks, while preserving topologically associating domain (TAD) borders.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31216471',\n",
       "     'score': 0.1741761863231659},\n",
       "    {'start': 724,\n",
       "     'end': 823,\n",
       "     'text': 'The diverse effects of both complexes on genome topology may reflect two modes of action of cohesin.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31216471',\n",
       "     'score': 0.15721797943115234},\n",
       "    {'start': 825,\n",
       "     'end': 997,\n",
       "     'text': 'One, likely involving loop extrusion, establishes overall genome arrangement in TADs together with CTCF and prevents excessive segregation of same-class compartment regions.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31216471',\n",
       "     'score': 0.1531020700931549},\n",
       "    {'start': 999,\n",
       "     'end': 1140,\n",
       "     'text': 'The other is required for organization of local transcriptional hubs such as Polycomb domains and super-enhancers, which define cell identity.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31216471',\n",
       "     'score': 0.2941708564758301}]},\n",
       "  {'id': '31035373',\n",
       "   'score': 6.704160213470459,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 35,\n",
       "     'text': 'A Cleared View on Retinal Organoids.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '31035373',\n",
       "     'score': 0.1496291160583496},\n",
       "    {'start': 0,\n",
       "     'end': 306,\n",
       "     'text': 'Human induced pluripotent stem cell (hiPSC)-derived organoids mimicking tissues and organs in vitro have advanced medical research, as they opened up new possibilities for in-depth basic research on human organ development as well as providing a human in vitro model for personalized therapeutic approaches.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31035373',\n",
       "     'score': 0.2895902991294861},\n",
       "    {'start': 308,\n",
       "     'end': 477,\n",
       "     'text': 'hiPSC-derived retinal organoids have proven to be of great value for modeling the human retina featuring a very similar cellular composition, layering, and functionality.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31035373',\n",
       "     'score': 0.155246764421463},\n",
       "    {'start': 479,\n",
       "     'end': 647,\n",
       "     'text': 'The technically challenging imaging of three-dimensional structures such as retinal organoids has, however, raised the need for robust whole-organoid imaging techniques.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31035373',\n",
       "     'score': 0.21311861276626587},\n",
       "    {'start': 649,\n",
       "     'end': 817,\n",
       "     'text': 'To improve imaging of retinal organoids we optimized a passive clearing technique (PACT), which enables high-resolution visualization of fragile intra-tissue structures.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31035373',\n",
       "     'score': 0.15271759033203125},\n",
       "    {'start': 819,\n",
       "     'end': 1008,\n",
       "     'text': 'Using cleared retinal organoids, we could greatly enhance the antibody labeling efficiency and depth of imaging at high resolution, thereby improving the three-dimensional microscopy output.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31035373',\n",
       "     'score': 0.21031630039215088},\n",
       "    {'start': 1010,\n",
       "     'end': 1153,\n",
       "     'text': 'In that course, we were able to identify the spatial morphological shape and organization of, e.g., photoreceptor cells and bipolar cell layers.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31035373',\n",
       "     'score': 0.20043876767158508},\n",
       "    {'start': 1155,\n",
       "     'end': 1326,\n",
       "     'text': 'Moreover, we used the synaptic protein CtBP2/Ribeye to visualize the interconnection points of photoreceptor and bipolar cells forming the retinal-specific ribbon synapses.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '31035373',\n",
       "     'score': 0.16256222128868103}]},\n",
       "  {'id': '20180650',\n",
       "   'score': 6.360445022583008,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 87,\n",
       "     'text': 'A structural network associated with the kallikrein-kinin and renin-angiotensin systems.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '20180650',\n",
       "     'score': 0.19083353877067566},\n",
       "    {'start': 0,\n",
       "     'end': 181,\n",
       "     'text': 'The kallikrein-kinin and renin-angiotensin (KKS-RAS) systems represent two highly regulated proteolytic systems that are involved in several physiological and pathological processes.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '20180650',\n",
       "     'score': 0.1764903962612152},\n",
       "    {'start': 183,\n",
       "     'end': 438,\n",
       "     'text': 'Although their protein-protein interactions can be studied using experimental approaches, it is difficult to differentiate between direct physical interactions and functional associations, which do not involve direct atomic contacts between macromolecules.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '20180650',\n",
       "     'score': 0.17073673009872437},\n",
       "    {'start': 440,\n",
       "     'end': 541,\n",
       "     'text': 'This information can be obtained from an atomic-resolution characterization of the protein interfaces.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '20180650',\n",
       "     'score': 0.1675952672958374},\n",
       "    {'start': 543,\n",
       "     'end': 655,\n",
       "     'text': 'As a result of this, various three-dimensional-based protein-protein interaction databases have become available.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '20180650',\n",
       "     'score': 0.2343921959400177},\n",
       "    {'start': 657,\n",
       "     'end': 824,\n",
       "     'text': 'To gain insight into the multilayered interaction of the KKS-RAS systems, we present a protein network that is built up on three-dimensional domain-domain interactions.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '20180650',\n",
       "     'score': 0.27577322721481323},\n",
       "    {'start': 826,\n",
       "     'end': 1092,\n",
       "     'text': 'The essential domains that link these systems are as follows: Cystatin, Peptidase_C1, Thyroglobulin_1, Insulin, CIMR (Cation-independent mannose-6-phosphate receptor repeat), fn2 (Fibronectin type II domain), fn1 (Fibronectin type I domain), EGF, Trypsin, and Serpin.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '20180650',\n",
       "     'score': 0.15941724181175232},\n",
       "    {'start': 1094,\n",
       "     'end': 1191,\n",
       "     'text': 'We found that the CIMR domain is located at the core of the network, thus connecting both systems.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '20180650',\n",
       "     'score': 0.1862839162349701},\n",
       "    {'start': 1193,\n",
       "     'end': 1348,\n",
       "     'text': 'From the latter, all domain interactors up to level 4 were retrieved, thus displaying a more comprehensive representation of the KKS-RAS structural network.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '20180650',\n",
       "     'score': 0.1840573251247406}]},\n",
       "  {'id': '11702195',\n",
       "   'score': 6.13373327255249,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 163,\n",
       "     'text': 'Ep-CAM transfection in thymic epithelial cell lines triggers the formation of dynamic actin-rich protrusions involved in the organization of epithelial cell layers.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '11702195',\n",
       "     'score': 0.21225932240486145},\n",
       "    {'start': 0,\n",
       "     'end': 117,\n",
       "     'text': 'Thymic epithelium is organized in a highly connected three-dimensional network through which thymocytes differentiate.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '11702195',\n",
       "     'score': 0.2675233483314514},\n",
       "    {'start': 119,\n",
       "     'end': 190,\n",
       "     'text': 'The molecular mechanisms underlying this organization are still unknown.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '11702195',\n",
       "     'score': 0.19785204529762268},\n",
       "    {'start': 192,\n",
       "     'end': 347,\n",
       "     'text': 'In thymic medulla, a major site of tolerance induction, the development of the epithelial cell net is tightly regulated by the needs of thymocyte selection.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '11702195',\n",
       "     'score': 0.16782167553901672},\n",
       "    {'start': 349,\n",
       "     'end': 426,\n",
       "     'text': 'These reticulated epithelial cells express high levels of the Ep-CAM molecule.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '11702195',\n",
       "     'score': 0.16200625896453857},\n",
       "    {'start': 428,\n",
       "     'end': 689,\n",
       "     'text': 'Using different thymic epithelial cell lines as a model system, we found that transfection of Ep-CAM enhances cell growth and leads to a rapid reorganization of the actin cytoskeleton by inducing the formation of numerous stress fibers and long cell protrusions.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '11702195',\n",
       "     'score': 0.1645474135875702},\n",
       "    {'start': 691,\n",
       "     'end': 837,\n",
       "     'text': 'Finally, the crosslinking of the extracellular domain of a chimeric CD25ec/Ep-CAMic molecule is sufficient to trigger the formation of protrusions.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '11702195',\n",
       "     'score': 0.16455599665641785},\n",
       "    {'start': 839,\n",
       "     'end': 1036,\n",
       "     'text': 'These results suggest that expression of Ep-CAM might balance the organizing capacity of cadherin molecules and may be participating in the formation of a dynamic stromal cell network in the thymus.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '11702195',\n",
       "     'score': 0.19614019989967346}]},\n",
       "  {'id': '12850452',\n",
       "   'score': 5.854151725769043,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 61,\n",
       "     'text': 'Organization of the cytokeratin network in an epithelial cell.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '12850452',\n",
       "     'score': 0.2511327862739563},\n",
       "    {'start': 0,\n",
       "     'end': 89,\n",
       "     'text': 'The cytoskeleton is a dynamic three-dimensional structure mainly located in the cytoplasm.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '12850452',\n",
       "     'score': 0.24609526991844177},\n",
       "    {'start': 91,\n",
       "     'end': 201,\n",
       "     'text': 'It is involved in many cell functions such as mechanical signal transduction and maintenance of cell integrity.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '12850452',\n",
       "     'score': 0.18173441290855408},\n",
       "    {'start': 203,\n",
       "     'end': 353,\n",
       "     'text': 'Among the three cytoskeletal components, intermediate filaments (the cytokeratin in epithelial cells) are the best candidates for this mechanical role.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '12850452',\n",
       "     'score': 0.1731824278831482},\n",
       "    {'start': 355,\n",
       "     'end': 539,\n",
       "     'text': 'A model of the establishment of the cytokeratin network of an epithelial cell is proposed to study the dependence of its structural organization on extracellular mechanical environment.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '12850452',\n",
       "     'score': 0.2371448278427124},\n",
       "    {'start': 541,\n",
       "     'end': 667,\n",
       "     'text': 'To implicitly describe the latter and its effects on the intracellular domain, we use mechanically regulated protein synthesis.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '12850452',\n",
       "     'score': 0.17581528425216675},\n",
       "    {'start': 669,\n",
       "     'end': 889,\n",
       "     'text': 'Our model is a hybrid of a partial differential equation of parabolic type, governing the evolution of the concentration of cytokeratin, and a set of stochastic differential equations describing the dynamics of filaments.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '12850452',\n",
       "     'score': 0.17147886753082275},\n",
       "    {'start': 891,\n",
       "     'end': 1086,\n",
       "     'text': 'Each filament is described by a stochastic differential equation that reflects both the local interactions with the environment and the non-local interactions via the past history of the filament.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '12850452',\n",
       "     'score': 0.17153769731521606},\n",
       "    {'start': 1088,\n",
       "     'end': 1164,\n",
       "     'text': 'A three-dimensional simulation model is derived from this mathematical model.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '12850452',\n",
       "     'score': 0.23270553350448608},\n",
       "    {'start': 1166,\n",
       "     'end': 1340,\n",
       "     'text': 'This simulation model is then used to obtain examples of cytokeratin network architectures under given mechanical conditions, and to study the influence of several parameters.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '12850452',\n",
       "     'score': 0.20032727718353271}]},\n",
       "  {'id': '24277632',\n",
       "   'score': 5.824346542358398,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 103,\n",
       "     'text': 'Making connections: insulators organize eukaryotic chromosomes into independent cis-regulatory networks.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '24277632',\n",
       "     'score': 0.1947794258594513},\n",
       "    {'start': 0,\n",
       "     'end': 204,\n",
       "     'text': 'Insulators play a central role in subdividing the chromosome into a series of discrete topologically independent domains and in ensuring that enhancers and silencers contact their appropriate target genes.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24277632',\n",
       "     'score': 0.2537003755569458},\n",
       "    {'start': 206,\n",
       "     'end': 324,\n",
       "     'text': 'In this review we first discuss the general characteristics of insulator elements and their associated protein factors.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24277632',\n",
       "     'score': 0.18847844004631042},\n",
       "    {'start': 326,\n",
       "     'end': 466,\n",
       "     'text': 'A growing collection of insulator proteins have been identified including a family of proteins whose expression is developmentally regulated.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24277632',\n",
       "     'score': 0.16808632016181946},\n",
       "    {'start': 468,\n",
       "     'end': 612,\n",
       "     'text': 'We next consider several unexpected discoveries that require us to completely rethink how insulators function (and how they can best be assayed).',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24277632',\n",
       "     'score': 0.1661384105682373},\n",
       "    {'start': 614,\n",
       "     'end': 800,\n",
       "     'text': 'These discoveries also require a reevaluation of how insulators might restrict or orchestrate (by preventing or promoting) interactions between regulatory elements and their target genes.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24277632',\n",
       "     'score': 0.16746357083320618},\n",
       "    {'start': 802,\n",
       "     'end': 1058,\n",
       "     'text': 'We conclude by connecting these new insights into the mechanisms of insulator action to dynamic changes in the three-dimensional topology of the chromatin fiber and the generation of specific patterns of gene activity during development and differentiation.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '24277632',\n",
       "     'score': 0.23895177245140076}]},\n",
       "  {'id': '22696678',\n",
       "   'score': 5.657094478607178,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 179,\n",
       "     'text': 'Grainyhead-like 2 regulates epithelial morphogenesis by establishing functional tight junctions through the organization of a molecular network among claudin3, claudin4, and Rab25.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '22696678',\n",
       "     'score': 0.2452041208744049},\n",
       "    {'start': 0,\n",
       "     'end': 212,\n",
       "     'text': 'During development, epithelial progenitors establish intercellular junctions, including tight junctions (TJs), and form three-dimensional (3D) tissue structures, which are often associated with luminal structures.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '22696678',\n",
       "     'score': 0.23981311917304993},\n",
       "    {'start': 214,\n",
       "     'end': 364,\n",
       "     'text': 'Here we identify grainyhead-like 2 (Grhl2) as a transcription factor that regulates the size of luminal space surrounded by polarized epithelial cells.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '22696678',\n",
       "     'score': 0.16978731751441956},\n",
       "    {'start': 366,\n",
       "     'end': 510,\n",
       "     'text': 'We show that HPPL, a liver progenitor cell line, transfected with Grhl2 cDNA forms remarkably larger cysts than the control cells in 3D cultures.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '22696678',\n",
       "     'score': 0.21562442183494568},\n",
       "    {'start': 512,\n",
       "     'end': 638,\n",
       "     'text': 'We find that Grhl2 up-regulates claudin (Cldn) 3 and Cldn4, and their functions are necessary for the formation of large cysts.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '22696678',\n",
       "     'score': 0.17221930623054504},\n",
       "    {'start': 640,\n",
       "     'end': 696,\n",
       "     'text': 'Overexpression of Cldn3 alone induces the cyst expansion.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '22696678',\n",
       "     'score': 0.1708110272884369},\n",
       "    {'start': 698,\n",
       "     'end': 793,\n",
       "     'text': 'In contrast, expression of Cldn4 alone does not induce expansion, as it is not localized at TJs.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '22696678',\n",
       "     'score': 0.1740792691707611},\n",
       "    {'start': 795,\n",
       "     'end': 916,\n",
       "     'text': 'Of interest, Rab25, another Grhl2 target, not only increases the Cldn4 protein, but also enhances its localization at TJs.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '22696678',\n",
       "     'score': 0.1804715096950531},\n",
       "    {'start': 918,\n",
       "     'end': 1141,\n",
       "     'text': 'Taken together, the results indicate that Grhl2 regulates epithelial morphogenesis through transcriptional up-regulation of Cldn3 and Cldn4, as well as of Rab25, which increases the Cldn4 protein and its localization at TJs.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '22696678',\n",
       "     'score': 0.20861735939979553},\n",
       "    {'start': 1143,\n",
       "     'end': 1238,\n",
       "     'text': 'The results reveal a molecular network regulating epithelial lumen formation organized by Grhl2.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '22696678',\n",
       "     'score': 0.217901349067688}]},\n",
       "  {'id': '29485124',\n",
       "   'score': 5.512683868408203,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 120,\n",
       "     'text': 'Structure function relations in PDZ-domain-containing proteins: Implications for protein networks in cellular signalling.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '29485124',\n",
       "     'score': 0.21302258968353271},\n",
       "    {'start': 0,\n",
       "     'end': 143,\n",
       "     'text': 'Protein scaffolds as essential backbones for organization of supramolecular signalling complexes are a recurrent theme in several model systems.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '29485124',\n",
       "     'score': 0.22705060243606567},\n",
       "    {'start': 145,\n",
       "     'end': 256,\n",
       "     'text': 'Scaffold proteins preferentially employ linear peptide binding motifs for recruiting their interaction partners.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '29485124',\n",
       "     'score': 0.16901996731758118},\n",
       "    {'start': 258,\n",
       "     'end': 404,\n",
       "     'text': 'PDZ domains are one of the more commonly encountered peptide binding domains in several proteins including those involved in scaffolding functions.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '29485124',\n",
       "     'score': 0.18271741271018982},\n",
       "    {'start': 406,\n",
       "     'end': 584,\n",
       "     'text': 'This domain is known for its promiscuity both in terms of ligand selection, mode of interaction with its ligands as well as its association with other protein interaction domains.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '29485124',\n",
       "     'score': 0.18577778339385986},\n",
       "    {'start': 586,\n",
       "     'end': 681,\n",
       "     'text': 'PDZ domains are subject to several means of regulations by virtue of their functional diversity.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '29485124',\n",
       "     'score': 0.17047181725502014},\n",
       "    {'start': 683,\n",
       "     'end': 834,\n",
       "     'text': 'Additionally, the PDZ domains are refractive to the effect of mutations and maintain their three-dimensional architecture under extreme mutational load.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '29485124',\n",
       "     'score': 0.24068427085876465},\n",
       "    {'start': 836,\n",
       "     'end': 964,\n",
       "     'text': 'The biochemical and biophysical basis for this selectivity as well as promiscuity has been investigated and reviewed extensively.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '29485124',\n",
       "     'score': 0.17380714416503906},\n",
       "    {'start': 966,\n",
       "     'end': 1154,\n",
       "     'text': 'The present review focuses on the plasticity inherent in PDZ domains and its implications for modular organization as well as evolution of cellular signalling pathways in higher eukaryotes.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '29485124',\n",
       "     'score': 0.21782386302947998}]},\n",
       "  {'id': '32988592',\n",
       "   'score': 5.505514621734619,\n",
       "   'snippets': [{'start': 0,\n",
       "     'end': 89,\n",
       "     'text': 'Interstitial flow regulates in\\xa0vitro three-dimensional self-organized brain micro-vessels.',\n",
       "     'is_title': True,\n",
       "     'doc_id': '32988592',\n",
       "     'score': 0.23765534162521362},\n",
       "    {'start': 0,\n",
       "     'end': 122,\n",
       "     'text': 'Cell culture under medium flow has been shown to favor human brain microvascular endothelial cells function and maturation.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '32988592',\n",
       "     'score': 0.17177963256835938},\n",
       "    {'start': 124,\n",
       "     'end': 453,\n",
       "     'text': 'Here a three-dimensional in\\xa0vitro model of the human brain microvasculature, comprising brain microvascular endothelial cells but also astrocytes, pericytes and a collagen type I microfiber - fibrin based matrix, was cultured under continuous medium flow in a pressure driven microphysiological system (10\\xa0kPa, in 60-30\\xa0s cycles).',\n",
       "     'is_title': False,\n",
       "     'doc_id': '32988592',\n",
       "     'score': 0.24118348956108093},\n",
       "    {'start': 455,\n",
       "     'end': 528,\n",
       "     'text': 'The cells self-organized in micro-vessels perpendicular to the shear flow.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '32988592',\n",
       "     'score': 0.17417865991592407},\n",
       "    {'start': 530,\n",
       "     'end': 827,\n",
       "     'text': 'Comparison with static culture showed that the resulting interstitial flow enhanced a more defined micro-vasculature network, with slightly more numerous lumens, and a higher expression of transporters, carriers and tight junction genes and proteins, essential to the blood-brain barrier functions.',\n",
       "     'is_title': False,\n",
       "     'doc_id': '32988592',\n",
       "     'score': 0.20121333003044128}]}],\n",
       " 'snippets': [{'start': 0,\n",
       "   'end': 114,\n",
       "   'text': 'KLF4 is involved in the organization and regulation of pluripotency-associated three-dimensional enhancer networks.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '31548608',\n",
       "   'score': 0.7081087231636047},\n",
       "  {'start': 0,\n",
       "   'end': 259,\n",
       "   'text': 'Cell fate transitions are accompanied by global transcriptional, epigenetic and topological changes driven by transcription factors, as is exemplified by reprogramming somatic cells to pluripotent stem cells through the expression of OCT4, KLF4, SOX2 and cMYC.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31548608',\n",
       "   'score': 0.15228056907653809},\n",
       "  {'start': 392,\n",
       "   'end': 669,\n",
       "   'text': 'Here, using KLF4 as a paradigm, we provide a transcription-factor-centric view of chromatin reorganization and its association with three-dimensional enhancer rewiring and transcriptional changes during the reprogramming of mouse embryonic fibroblasts to pluripotent stem cells.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31548608',\n",
       "   'score': 0.39797115325927734},\n",
       "  {'start': 671,\n",
       "   'end': 967,\n",
       "   'text': 'Inducible depletion of KLF factors in PSCs caused a genome-wide decrease in enhancer connectivity, whereas disruption of individual KLF4 binding sites within pluripotent-stem-cell-specific enhancers was sufficient to impair enhancer-promoter contacts and reduce the expression of associated genes.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31548608',\n",
       "   'score': 0.25990283489227295},\n",
       "  {'start': 0,\n",
       "   'end': 60,\n",
       "   'text': 'Pluripotency in 3D: genome organization in pluripotent cells.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '23199754',\n",
       "   'score': 0.40288621187210083},\n",
       "  {'start': 0,\n",
       "   'end': 134,\n",
       "   'text': 'Cells face the challenge of storing two meters of DNA in the three-dimensional (3D) space of the nucleus that spans only a few microns.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '23199754',\n",
       "   'score': 0.13167187571525574},\n",
       "  {'start': 136,\n",
       "   'end': 408,\n",
       "   'text': 'The nuclear organization that is required to overcome this challenge must allow for the accessibility of the gene regulatory machinery to the DNA and, in the case of embryonic stem cells (ESCs), for the transcriptional and epigenetic changes that accompany differentiation.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '23199754',\n",
       "   'score': 0.2666993737220764},\n",
       "  {'start': 410,\n",
       "   'end': 532,\n",
       "   'text': 'Recent technological advances have allowed for the mapping of genome organization at an unprecedented resolution and scale.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '23199754',\n",
       "   'score': 0.11587250232696533},\n",
       "  {'start': 534,\n",
       "   'end': 713,\n",
       "   'text': 'These breakthroughs have led to a deluge of new data, and a sophisticated understanding of the relationship between gene regulation and 3D genome organization is beginning to form.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '23199754',\n",
       "   'score': 0.1825897991657257},\n",
       "  {'start': 715,\n",
       "   'end': 1076,\n",
       "   'text': 'In this review we summarize some of the recent findings illuminating the 3D structure of the eukaryotic genome, as well as the relationship between genome topology and function from the level of whole chromosomes to enhancer-promoter loops with a focus on features affecting genome organization in ESCs and changes in nuclear organization during differentiation.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '23199754',\n",
       "   'score': 0.49772197008132935},\n",
       "  {'start': 0,\n",
       "   'end': 108,\n",
       "   'text': 'Genomic Rewiring of SOX2 Chromatin Interaction Network during Differentiation of ESCs to Postmitotic Neurons.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '32553182',\n",
       "   'score': 0.29788559675216675},\n",
       "  {'start': 0,\n",
       "   'end': 128,\n",
       "   'text': 'Cellular differentiation requires dramatic changes in chromatin organization, transcriptional regulation, and protein production.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '32553182',\n",
       "   'score': 0.14769163727760315},\n",
       "  {'start': 130,\n",
       "   'end': 467,\n",
       "   'text': 'To understand the regulatory connections between these processes, we generated proteomic, transcriptomic, and chromatin accessibility data during differentiation of mouse embryonic stem cells (ESCs) into postmitotic neurons and found extensive associations between different molecular layers within and across differentiation time points.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '32553182',\n",
       "   'score': 0.2127506136894226},\n",
       "  {'start': 469,\n",
       "   'end': 690,\n",
       "   'text': 'We observed that SOX2, as a regulator of pluripotency and neuronal genes, redistributes from pluripotency enhancers to neuronal promoters during differentiation, likely driven by changes in its protein interaction network.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '32553182',\n",
       "   'score': 0.5031372904777527},\n",
       "  {'start': 692,\n",
       "   'end': 912,\n",
       "   'text': 'We identified ATRX as a major SOX2 partner in neurons, whose co-localization correlated with an increase in active enhancer marks and increased expression of nearby genes, which we experimentally confirmed for three loci.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '32553182',\n",
       "   'score': 0.15763896703720093},\n",
       "  {'start': 914,\n",
       "   'end': 1144,\n",
       "   'text': 'Collectively, our data provide key insights into the regulatory transformation of SOX2 during neuronal differentiation, and we highlight the significance of multi-omic approaches in understanding gene regulation in complex systems.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '32553182',\n",
       "   'score': 0.10985422134399414},\n",
       "  {'start': 0,\n",
       "   'end': 128,\n",
       "   'text': 'Sall4 controls differentiation of pluripotent cells independently of the Nucleosome Remodelling and Deacetylation (NuRD) complex.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '27471257',\n",
       "   'score': 0.24639463424682617},\n",
       "  {'start': 0,\n",
       "   'end': 116,\n",
       "   'text': 'Sall4 is an essential transcription factor for early mammalian development and is frequently overexpressed in cancer.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '27471257',\n",
       "   'score': 0.0972265899181366},\n",
       "  {'start': 118,\n",
       "   'end': 275,\n",
       "   'text': 'Although it is reported to play an important role in embryonic stem cell (ESC) self-renewal, whether it is an essential pluripotency factor has been disputed.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '27471257',\n",
       "   'score': 0.21590715646743774},\n",
       "  {'start': 277,\n",
       "   'end': 343,\n",
       "   'text': 'Here, we show that Sall4 is dispensable for mouse ESC pluripotency.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '27471257',\n",
       "   'score': 0.23391544818878174},\n",
       "  {'start': 345,\n",
       "   'end': 545,\n",
       "   'text': 'Sall4 is an enhancer-binding protein that prevents precocious activation of the neural gene expression programme in ESCs but is not required for maintenance of the pluripotency gene regulatory network.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '27471257',\n",
       "   'score': 0.4841834604740143},\n",
       "  {'start': 547,\n",
       "   'end': 824,\n",
       "   'text': 'Although a proportion of Sall4 protein physically associates with the Nucleosome Remodelling and Deacetylase (NuRD) complex, Sall4 neither recruits NuRD to chromatin nor influences transcription via NuRD; rather, free Sall4 protein regulates transcription independently of NuRD.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '27471257',\n",
       "   'score': 0.10221952199935913},\n",
       "  {'start': 826,\n",
       "   'end': 1033,\n",
       "   'text': 'We propose a model whereby enhancer binding by Sall4 and other pluripotency-associated transcription factors is responsible for maintaining the balance between transcriptional programmes in pluripotent cells.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '27471257',\n",
       "   'score': 0.44995638728141785},\n",
       "  {'start': 0,\n",
       "   'end': 188,\n",
       "   'text': 'DNA methylation of the Oct4A enhancers in embryonal carcinoma cells after etoposide treatment is associated with alternative splicing and altered pluripotency in reversibly senescent cells.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '29372665',\n",
       "   'score': 0.36239680647850037},\n",
       "  {'start': 0,\n",
       "   'end': 140,\n",
       "   'text': 'The epigenetic mechanisms underlying chemoresistance in cancer cells resulting from drug-induced reversible senescence are poorly understood.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '29372665',\n",
       "   'score': 0.09827521443367004},\n",
       "  {'start': 142,\n",
       "   'end': 394,\n",
       "   'text': 'Chemoresistant ESC-like embryonal carcinoma PA1 cells treated with etoposide (ETO) were previously found to undergo prolonged G2 arrest with transient p53-dependent upregulation of opposing fate regulators, p21CIP1 (senescence) and OCT4A (self-renewal).',\n",
       "   'is_title': False,\n",
       "   'doc_id': '29372665',\n",
       "   'score': 0.09764865040779114},\n",
       "  {'start': 396,\n",
       "   'end': 554,\n",
       "   'text': 'Here we report on the analysis of the DNA methylation state of the distal enhancer (DE) and proximal enhancer (PE) of the Oct4A gene during this dual response.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '29372665',\n",
       "   'score': 0.15672793984413147},\n",
       "  {'start': 556,\n",
       "   'end': 696,\n",
       "   'text': 'When compared to non-treated controls the methylation level increased from 1.3% to 12.5% and from 3% to 19.4%, in the DE and PE respectively.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '29372665',\n",
       "   'score': 0.09951764345169067},\n",
       "  {'start': 698,\n",
       "   'end': 804,\n",
       "   'text': 'It included CpG and non-CpG methylation, which was not chaotic but presented two patterns in each enhancer.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '29372665',\n",
       "   'score': 0.16123738884925842},\n",
       "  {'start': 806,\n",
       "   'end': 1071,\n",
       "   'text': 'Discorrelating with methylation of enhancers, the transcription of Oct4A increased, however, a strong expression of the splicing form Oct4B was also induced, along with down-regulation of the Oct4A partners of in the pluripotency/self-renewal network Sox2 and Lin28.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '29372665',\n",
       "   'score': 0.4819476008415222},\n",
       "  {'start': 1073,\n",
       "   'end': 1154,\n",
       "   'text': 'WB demonstrated disjoining of the OCT4A protein from the chromatin-bound fraction.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '29372665',\n",
       "   'score': 0.10324835777282715},\n",
       "  {'start': 1156,\n",
       "   'end': 1285,\n",
       "   'text': 'In survival clones, methylation of the DE was considerably erased, while some remnant of methylation of the PE was still observed.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '29372665',\n",
       "   'score': 0.09949445724487305},\n",
       "  {'start': 1287,\n",
       "   'end': 1473,\n",
       "   'text': 'The alternative splicing for Oct4B was reduced, Oct4A level insignificantly decreased, while the expression of Sox2 and Lin28 recovered, all three became proportionally above the control.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '29372665',\n",
       "   'score': 0.10344123840332031},\n",
       "  {'start': 1475,\n",
       "   'end': 1739,\n",
       "   'text': 'These findings indicate the involvement of the transient patterned methylation of the Oct4A enhancers and alternative splicing in the adaptive regulation of cell fate choice during the p53-dependant dual state of reversible senescence in ESC-like cancer stem cells.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '29372665',\n",
       "   'score': 0.18598923087120056},\n",
       "  {'start': 0,\n",
       "   'end': 74,\n",
       "   'text': 'Role of ALKBH1 in the Core Transcriptional Network of Embryonic Stem Cells.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '26765775',\n",
       "   'score': 0.1245814859867096},\n",
       "  {'start': 0,\n",
       "   'end': 171,\n",
       "   'text': 'BACKGROUND/AIMS\\nALKBH1, an AlkB homologue in the 2-oxoglutarate and Fe2+ dependent hydroxylase family, is a histone dioxygenase that removes methyl groups from histone H2A.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '26765775',\n",
       "   'score': 0.10476219654083252},\n",
       "  {'start': 173,\n",
       "   'end': 282,\n",
       "   'text': 'Studies of transgenic mice lacking Alkbh1 reveal that most Alkbh1-/- embryos die during embryonic development.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '26765775',\n",
       "   'score': 0.09883207082748413},\n",
       "  {'start': 284,\n",
       "   'end': 537,\n",
       "   'text': 'Embryonic stem cells (ESCs) derived from these mice have prolonged expression of pluripotency markers and delayed induction of genes involved in neural differentiation, indicating that ALKBH1 is involved in regulation of pluripotency and differentiation.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '26765775',\n",
       "   'score': 0.3430096507072449},\n",
       "  {'start': 539,\n",
       "   'end': 624,\n",
       "   'text': 'The aim of this study was to further investigate the role ALKBH1 in early development.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '26765775',\n",
       "   'score': 0.10196158289909363},\n",
       "  {'start': 628,\n",
       "   'end': 839,\n",
       "   'text': 'METHODS\\nDouble-filter methods for nitrocellulose-filter binding, dot blot, enzyme-linked immunosorbent assay (ELISA), immonocytochemistry, cell culture and differentiation of mouse ESCs, Co-IP and miRNA analysis.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '26765775',\n",
       "   'score': 0.2339635193347931},\n",
       "  {'start': 843,\n",
       "   'end': 1032,\n",
       "   'text': 'RESULTS\\nWe found that SOX2 and NANOG bind the ALKBH1 promoter, and we identified protein-protein interactions between ALKBH1 and these core transcription factors of the pluripotency network.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '26765775',\n",
       "   'score': 0.4504778981208801},\n",
       "  {'start': 1034,\n",
       "   'end': 1201,\n",
       "   'text': 'Furthermore, lack of ALKBH1 affected the expression of developmentally important miRNAs, which are involved in the regulation of NANOG, SOX2 and neural differentiation.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '26765775',\n",
       "   'score': 0.31529873609542847},\n",
       "  {'start': 1205,\n",
       "   'end': 1382,\n",
       "   'text': 'CONCLUSION\\nOur results suggest that ALKBH1 interacts with the core transcriptional pluripotency network of ESCs and is involved in regulation of pluripotency and differentiation.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '26765775',\n",
       "   'score': 0.4017235338687897},\n",
       "  {'start': 0,\n",
       "   'end': 128,\n",
       "   'text': 'Deep transcriptome profiling of mammalian stem cells supports a regulatory role for retrotransposons in pluripotency maintenance.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '24777452',\n",
       "   'score': 0.24576252698898315},\n",
       "  {'start': 0,\n",
       "   'end': 190,\n",
       "   'text': 'The importance of microRNAs and long noncoding RNAs in the regulation of pluripotency has been documented; however, the noncoding components of stem cell gene networks remain largely unknown.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24777452',\n",
       "   'score': 0.38223984837532043},\n",
       "  {'start': 192,\n",
       "   'end': 340,\n",
       "   'text': 'Here we investigate the role of noncoding RNAs in the pluripotent state, with particular emphasis on nuclear and retrotransposon-derived transcripts.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24777452',\n",
       "   'score': 0.25615397095680237},\n",
       "  {'start': 342,\n",
       "   'end': 527,\n",
       "   'text': 'We have performed deep profiling of the nuclear and cytoplasmic transcriptomes of human and mouse stem cells, identifying a class of previously undetected stem cell-specific transcripts.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24777452',\n",
       "   'score': 0.10201424360275269},\n",
       "  {'start': 529,\n",
       "   'end': 668,\n",
       "   'text': 'We show that long terminal repeat (LTR)-derived transcripts contribute extensively to the complexity of the stem cell nuclear transcriptome.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24777452',\n",
       "   'score': 0.10034841299057007},\n",
       "  {'start': 670,\n",
       "   'end': 800,\n",
       "   'text': 'Some LTR-derived transcripts are associated with enhancer regions and are likely to be involved in the maintenance of pluripotency.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24777452',\n",
       "   'score': 0.45888859033584595},\n",
       "  {'start': 0,\n",
       "   'end': 121,\n",
       "   'text': 'KLF4 protein stability regulated by interaction with pluripotency transcription factors overrides transcriptional control.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '31221664',\n",
       "   'score': 0.2726683020591736},\n",
       "  {'start': 0,\n",
       "   'end': 113,\n",
       "   'text': 'Embryonic stem (ES) cells are regulated by a network of transcription factors that maintain the pluripotent state.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31221664',\n",
       "   'score': 0.2761201858520508},\n",
       "  {'start': 115,\n",
       "   'end': 218,\n",
       "   'text': 'Differentiation relies on down-regulation of pluripotency transcription factors disrupting this network.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31221664',\n",
       "   'score': 0.3756549060344696},\n",
       "  {'start': 220,\n",
       "   'end': 597,\n",
       "   'text': 'While investigating transcriptional regulation of the pluripotency transcription factor Kruppel-like factor 4 (Klf4), we observed that homozygous deletion of distal enhancers caused a 17-fold decrease in Klf4 transcript but surprisingly decreased protein levels by less than twofold, indicating that posttranscriptional control of KLF4 protein overrides transcriptional control.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31221664',\n",
       "   'score': 0.4509628713130951},\n",
       "  {'start': 599,\n",
       "   'end': 698,\n",
       "   'text': 'The lack of sensitivity of KLF4 to transcription is due to high protein stability (half-life >24 h).',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31221664',\n",
       "   'score': 0.11053293943405151},\n",
       "  {'start': 700,\n",
       "   'end': 1020,\n",
       "   'text': 'This stability is context-dependent and is disrupted during differentiation, as evidenced by a shift to a half-life of <2 h. KLF4 protein stability is maintained through interaction with other pluripotency transcription factors (NANOG, SOX2, and STAT3) that together facilitate association of KLF4 with RNA polymerase II.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31221664',\n",
       "   'score': 0.268865168094635},\n",
       "  {'start': 1022,\n",
       "   'end': 1131,\n",
       "   'text': 'In addition, the KLF4 DNA-binding and transactivation domains are required for optimal KLF4 protein stability.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31221664',\n",
       "   'score': 0.11121487617492676},\n",
       "  {'start': 1133,\n",
       "   'end': 1310,\n",
       "   'text': 'Posttranslational modification of KLF4 destabilizes the protein as cells exit the pluripotent state, and mutations that prevent this destabilization also prevent differentiation.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31221664',\n",
       "   'score': 0.2682322859764099},\n",
       "  {'start': 1312,\n",
       "   'end': 1569,\n",
       "   'text': 'These data indicate that the core pluripotency transcription factors are integrated by posttranslational mechanisms to maintain the pluripotent state and identify mutations that increase KLF4 protein stability while maintaining transcription factor function.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31221664',\n",
       "   'score': 0.23399245738983154},\n",
       "  {'start': 0,\n",
       "   'end': 171,\n",
       "   'text': 'Three-Dimensional Retinal Organoids Facilitate the Investigation of Retinal Ganglion Cell Development, Organization and Neurite Outgrowth from Human Pluripotent Stem Cells.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '30266927',\n",
       "   'score': 0.46932312846183777},\n",
       "  {'start': 0,\n",
       "   'end': 235,\n",
       "   'text': 'Retinal organoids are three-dimensional structures derived from human pluripotent stem cells (hPSCs) which recapitulate the spatial and temporal differentiation of the retina, serving as effective in vitro models of retinal development.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '30266927',\n",
       "   'score': 0.31366026401519775},\n",
       "  {'start': 237,\n",
       "   'end': 376,\n",
       "   'text': 'However, a lack of emphasis has been placed upon the development and organization of retinal ganglion cells (RGCs) within retinal organoids.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '30266927',\n",
       "   'score': 0.13068628311157227},\n",
       "  {'start': 378,\n",
       "   'end': 623,\n",
       "   'text': 'Thus, initial efforts were made to characterize RGC differentiation throughout early stages of organoid development, with a clearly defined RGC layer developing in a temporally-appropriate manner expressing a complement of RGC-associated markers.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '30266927',\n",
       "   'score': 0.10587576031684875},\n",
       "  {'start': 625,\n",
       "   'end': 804,\n",
       "   'text': 'Beyond studies of RGC development, retinal organoids may also prove useful for cellular replacement in which extensive axonal outgrowth is necessary to reach post-synaptic targets.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '30266927',\n",
       "   'score': 0.10580062866210938},\n",
       "  {'start': 806,\n",
       "   'end': 973,\n",
       "   'text': 'Organoid-derived RGCs could help to elucidate factors promoting axonal outgrowth, thereby identifying approaches to circumvent a formidable obstacle to RGC replacement.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '30266927',\n",
       "   'score': 0.10016009211540222},\n",
       "  {'start': 975,\n",
       "   'end': 1137,\n",
       "   'text': 'As such, additional efforts demonstrated significant enhancement of neurite outgrowth through modulation of both substrate composition and growth factor signaling.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '30266927',\n",
       "   'score': 0.10293316841125488},\n",
       "  {'start': 1139,\n",
       "   'end': 1280,\n",
       "   'text': 'Additionally, organoid-derived RGCs exhibited diverse phenotypes, extending elaborate growth cones and expressing numerous guidance receptors.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '30266927',\n",
       "   'score': 0.10172152519226074},\n",
       "  {'start': 1282,\n",
       "   'end': 1540,\n",
       "   'text': 'Collectively, these results establish retinal organoids as a valuable tool for studies of RGC development, and demonstrate the utility of organoid-derived RGCs as an effective platform to study factors influencing neurite outgrowth from organoid-derived RGCs.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '30266927',\n",
       "   'score': 0.1039593517780304},\n",
       "  {'start': 0,\n",
       "   'end': 137,\n",
       "   'text': 'The Role of RNA Polymerase II Contiguity and Long-Range Interactions in the Regulation of Gene Expression in Human Pluripotent Stem Cells.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '30863449',\n",
       "   'score': 0.30869418382644653},\n",
       "  {'start': 0,\n",
       "   'end': 189,\n",
       "   'text': 'The eukaryotic nucleus is a highly complex structure that carries out multiple functions primarily needed for gene expression, and among them, transcription seems to be the most fundamental.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '30863449',\n",
       "   'score': 0.10597193241119385},\n",
       "  {'start': 191,\n",
       "   'end': 412,\n",
       "   'text': 'Diverse approaches have demonstrated that transcription takes place at discrete sites known as transcription factories, wherein RNA polymerase II (RNAP II) is attached to the factory and immobilized while transcribing DNA.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '30863449',\n",
       "   'score': 0.10560482740402222},\n",
       "  {'start': 414,\n",
       "   'end': 598,\n",
       "   'text': 'It has been proposed that transcription factories promote chromatin loop formation, creating long-range interactions in which relatively distant genes can be transcribed simultaneously.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '30863449',\n",
       "   'score': 0.10410195589065552},\n",
       "  {'start': 600,\n",
       "   'end': 909,\n",
       "   'text': 'In this study, we examined long-range interactions between the POU5F1 gene and genes previously identified as being POU5F1 enhancer-interacting, namely, CDYL, TLE2, RARG, and MSX1 (all involved in transcriptional regulation), in human pluripotent stem cells (hPSCs) and their early differentiated counterparts.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '30863449',\n",
       "   'score': 0.4484449028968811},\n",
       "  {'start': 911,\n",
       "   'end': 1038,\n",
       "   'text': 'As a control gene, RUNX1 was used, which is expressed during hematopoietic differentiation and not associated with pluripotency.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '30863449',\n",
       "   'score': 0.26265716552734375},\n",
       "  {'start': 1040,\n",
       "   'end': 1313,\n",
       "   'text': 'To reveal how these long-range interactions between POU5F1 and the selected genes change with the onset of differentiation and upon RNAP II inhibition, we performed three-dimensional fluorescence in situ hybridization (3D-FISH) followed by computational simulation analysis.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '30863449',\n",
       "   'score': 0.14999845623970032},\n",
       "  {'start': 1315,\n",
       "   'end': 1519,\n",
       "   'text': 'Our analysis showed that the numbers of long-range interactions between specific genes decrease during differentiation, suggesting that the transcription of monitored genes is associated with pluripotency.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '30863449',\n",
       "   'score': 0.25966376066207886},\n",
       "  {'start': 1521,\n",
       "   'end': 1640,\n",
       "   'text': 'In addition, we showed that upon inhibition of RNAP II, long-range associations do not disintegrate and remain constant.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '30863449',\n",
       "   'score': 0.10640057921409607},\n",
       "  {'start': 1642,\n",
       "   'end': 1833,\n",
       "   'text': 'We also analyzed the distance distributions of these genes in the context of their positions in the nucleus and revealed that they tend to have similar patterns resembling normal distribution.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '30863449',\n",
       "   'score': 0.10745862126350403},\n",
       "  {'start': 1835,\n",
       "   'end': 1945,\n",
       "   'text': 'Furthermore, we compared data created in vitro and in silico to assess the biological relevance of our results.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '30863449',\n",
       "   'score': 0.10668230056762695},\n",
       "  {'start': 0,\n",
       "   'end': 49,\n",
       "   'text': 'DNA methylation and the core pluripotency network.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '32562758',\n",
       "   'score': 0.28680557012557983},\n",
       "  {'start': 0,\n",
       "   'end': 87,\n",
       "   'text': 'From the onset of fertilization, the genome undergoes cell division and differentiation.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '32562758',\n",
       "   'score': 0.10561397671699524},\n",
       "  {'start': 89,\n",
       "   'end': 227,\n",
       "   'text': 'All of these developmental transitions and differentiation processes include cell-specific signatures and gradual changes of the epigenome.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '32562758',\n",
       "   'score': 0.10598498582839966},\n",
       "  {'start': 229,\n",
       "   'end': 380,\n",
       "   'text': 'Understanding what keeps stem cells in the pluripotent state and what leads to differentiation are fascinating and biomedically highly important issues.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '32562758',\n",
       "   'score': 0.25422459840774536},\n",
       "  {'start': 382,\n",
       "   'end': 490,\n",
       "   'text': 'Numerous studies have identified genes, proteins, microRNAs and small molecules that exert essential effects.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '32562758',\n",
       "   'score': 0.10321137309074402},\n",
       "  {'start': 492,\n",
       "   'end': 611,\n",
       "   'text': 'Notably, there exists a core pluripotency network that consists of several transcription factors and accessory proteins.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '32562758',\n",
       "   'score': 0.2775859832763672},\n",
       "  {'start': 613,\n",
       "   'end': 719,\n",
       "   'text': 'Three eminent transcription factors, OCT4, SOX2 and NANOG, serve as hubs in this core pluripotency network.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '32562758',\n",
       "   'score': 0.32389211654663086},\n",
       "  {'start': 721,\n",
       "   'end': 924,\n",
       "   'text': 'They bind to the enhancer regions of their target genes and modulate, among others, the expression levels of genes that are associated with Gene Ontology terms related to differentiation and self-renewal.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '32562758',\n",
       "   'score': 0.17643365263938904},\n",
       "  {'start': 926,\n",
       "   'end': 1027,\n",
       "   'text': 'Also, much has been learned about the epigenetic rewiring processes during these changes of cell fate.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '32562758',\n",
       "   'score': 0.10387203097343445},\n",
       "  {'start': 1029,\n",
       "   'end': 1106,\n",
       "   'text': 'For example, DNA methylation dynamics is pivotal during embryonic development.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '32562758',\n",
       "   'score': 0.10323336720466614},\n",
       "  {'start': 1108,\n",
       "   'end': 1449,\n",
       "   'text': 'The main goal of this review is to highlight an intricate interplay of (a) DNA methyltransferases controlling the expression levels of core pluripotency factors by modulation of the DNA methylation levels in their enhancer regions, and of (b) the core pluripotency factors controlling the transcriptional regulation of DNA methyltransferases.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '32562758',\n",
       "   'score': 0.4437042474746704},\n",
       "  {'start': 1451,\n",
       "   'end': 1601,\n",
       "   'text': 'We discuss these processes both at the global level and in atomistic detail based on information from structural studies and from computer simulations.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '32562758',\n",
       "   'score': 0.10622531175613403},\n",
       "  {'start': 0,\n",
       "   'end': 130,\n",
       "   'text': 'Long-Range Enhancer Interactions Are Prevalent in Mouse Embryonic Stem Cells and Are Reorganized upon Pluripotent State Transition.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '29514091',\n",
       "   'score': 0.3677775263786316},\n",
       "  {'start': 0,\n",
       "   'end': 148,\n",
       "   'text': 'Transcriptional enhancers, including super-enhancers (SEs), form physical interactions with promoters to regulate cell-type-specific gene expression.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '29514091',\n",
       "   'score': 0.16429558396339417},\n",
       "  {'start': 150,\n",
       "   'end': 332,\n",
       "   'text': 'SEs are characterized by high transcription factor occupancy and large domains of active chromatin, and they are commonly assigned to target promoters using computational predictions.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '29514091',\n",
       "   'score': 0.10517621040344238},\n",
       "  {'start': 334,\n",
       "   'end': 481,\n",
       "   'text': 'How promoter-SE interactions change upon cell state transitions, and whether transcription factors maintain SE interactions, have not been reported.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '29514091',\n",
       "   'score': 0.16661447286605835},\n",
       "  {'start': 483,\n",
       "   'end': 600,\n",
       "   'text': 'Here, we used promoter-capture Hi-C to identify promoters that interact with SEs in mouse embryonic stem cells (ESCs).',\n",
       "   'is_title': False,\n",
       "   'doc_id': '29514091',\n",
       "   'score': 0.32012271881103516},\n",
       "  {'start': 602,\n",
       "   'end': 779,\n",
       "   'text': 'We found that SEs form complex, spatial networks in which individual SEs contact multiple promoters, and a rewiring of promoter-SE interactions occurs between pluripotent states.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '29514091',\n",
       "   'score': 0.4290946424007416},\n",
       "  {'start': 781,\n",
       "   'end': 922,\n",
       "   'text': 'We also show that long-range promoter-SE interactions are more prevalent in ESCs than in epiblast stem cells (EpiSCs) or Nanog-deficient ESCs.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '29514091',\n",
       "   'score': 0.3685828745365143},\n",
       "  {'start': 924,\n",
       "   'end': 1132,\n",
       "   'text': 'We conclude that SEs form cell-type-specific interaction networks that are partly dependent on core transcription factors, thereby providing insights into the gene regulatory organization of pluripotent cells.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '29514091',\n",
       "   'score': 0.3570590019226074},\n",
       "  {'start': 0,\n",
       "   'end': 81,\n",
       "   'text': 'Gender Differences in Global but Not Targeted Demethylation in iPSC Reprogramming.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '28147265',\n",
       "   'score': 0.10552334785461426},\n",
       "  {'start': 0,\n",
       "   'end': 188,\n",
       "   'text': 'Global DNA demethylation is an integral part of reprogramming processes in\\xa0vivo and in\\xa0vitro, but whether it occurs in the derivation of induced pluripotent stem cells (iPSCs) is not known.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28147265',\n",
       "   'score': 0.2805059552192688},\n",
       "  {'start': 190,\n",
       "   'end': 345,\n",
       "   'text': 'Here, we show that iPSC reprogramming involves both global and targeted demethylation, which are separable mechanistically and by their biological outcomes.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28147265',\n",
       "   'score': 0.10664835572242737},\n",
       "  {'start': 347,\n",
       "   'end': 483,\n",
       "   'text': 'Cells at intermediate-late stages of reprogramming undergo transient genome-wide demethylation, which is more pronounced in female cells.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28147265',\n",
       "   'score': 0.11006316542625427},\n",
       "  {'start': 485,\n",
       "   'end': 692,\n",
       "   'text': 'Global demethylation requires activation-induced cytidine deaminase (AID)-mediated downregulation of UHRF1 protein, and abolishing demethylation leaves thousands of hypermethylated regions in the iPSC genome.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28147265',\n",
       "   'score': 0.1180504858493805},\n",
       "  {'start': 694,\n",
       "   'end': 916,\n",
       "   'text': 'Independently of AID and global demethylation, regulatory regions, particularly ESC enhancers and super-enhancers, are specifically targeted for hypomethylation in association with transcription of the pluripotency network.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28147265',\n",
       "   'score': 0.4567837417125702},\n",
       "  {'start': 918,\n",
       "   'end': 1145,\n",
       "   'text': 'Our results show that global and targeted DNA demethylation are conserved and\\xa0distinct reprogramming processes, presumably because of their respective roles in epigenetic memory erasure and in the establishment of cell identity.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28147265',\n",
       "   'score': 0.11022317409515381},\n",
       "  {'start': 0,\n",
       "   'end': 73,\n",
       "   'text': 'A transcribed enhancer dictates mesendoderm specification in pluripotency.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '29180618',\n",
       "   'score': 0.40541893243789673},\n",
       "  {'start': 0,\n",
       "   'end': 108,\n",
       "   'text': 'Enhancers and long noncoding RNAs (lncRNAs) are key determinants of lineage specification during development.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '29180618',\n",
       "   'score': 0.1812184453010559},\n",
       "  {'start': 110,\n",
       "   'end': 240,\n",
       "   'text': 'Here, we evaluate remodeling of the enhancer landscape and modulation of the lncRNA transcriptome during mesendoderm specification.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '29180618',\n",
       "   'score': 0.18145865201950073},\n",
       "  {'start': 242,\n",
       "   'end': 485,\n",
       "   'text': 'We sort mesendodermal progenitors from differentiating embryonic stem cells (ESCs) according to Eomes expression, and find that enhancer usage is coordinated with mesendoderm-specific expression of key lineage-determining transcription factors.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '29180618',\n",
       "   'score': 0.3563392162322998},\n",
       "  {'start': 487,\n",
       "   'end': 556,\n",
       "   'text': 'Many of these enhancers are associated with the expression of lncRNAs.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '29180618',\n",
       "   'score': 0.18528583645820618},\n",
       "  {'start': 558,\n",
       "   'end': 763,\n",
       "   'text': 'Examination of ESC-specific enhancers interacting in three-dimensional space with mesendoderm-specifying transcription factor loci identifies MesEndoderm Transcriptional Enhancer Organizing Region (Meteor).',\n",
       "   'is_title': False,\n",
       "   'doc_id': '29180618',\n",
       "   'score': 0.26574966311454773},\n",
       "  {'start': 765,\n",
       "   'end': 985,\n",
       "   'text': 'Genetic and epigenetic manipulation of the Meteor enhancer reveal its indispensable role during mesendoderm specification and subsequent cardiogenic differentiation via transcription-independent and -dependent mechanisms.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '29180618',\n",
       "   'score': 0.1795782446861267},\n",
       "  {'start': 987,\n",
       "   'end': 1084,\n",
       "   'text': 'Interestingly, Meteor-deleted ESCs are epigenetically redirected towards neuroectodermal lineages.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '29180618',\n",
       "   'score': 0.2370816469192505},\n",
       "  {'start': 1086,\n",
       "   'end': 1288,\n",
       "   'text': 'Loci, topologically associating a transcribed enhancer and its cognate protein coding gene, appear to represent therefore a class of genomic elements controlling developmental competence in pluripotency.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '29180618',\n",
       "   'score': 0.4206128716468811},\n",
       "  {'start': 0,\n",
       "   'end': 75,\n",
       "   'text': 'The interactomes of POU5F1 and SOX2 enhancers in human embryonic stem cells.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '23549118',\n",
       "   'score': 0.18976709246635437},\n",
       "  {'start': 0,\n",
       "   'end': 152,\n",
       "   'text': 'The genes POU5F1 and SOX2 are critical for pluripotency and reprogramming, yet the chromosomal organization around these genes remains poorly understood.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '23549118',\n",
       "   'score': 0.31996476650238037},\n",
       "  {'start': 154,\n",
       "   'end': 308,\n",
       "   'text': 'We assayed long-range chromosomal interactions on putative enhancers of POU5F1 and SOX2 genes in human embryonic stem cells (hESCs) using 4C-Seq technique.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '23549118',\n",
       "   'score': 0.39866986870765686},\n",
       "  {'start': 310,\n",
       "   'end': 413,\n",
       "   'text': 'We discovered that their frequent interacting regions mainly overlap with early DNA replication domains.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '23549118',\n",
       "   'score': 0.10612431168556213},\n",
       "  {'start': 415,\n",
       "   'end': 520,\n",
       "   'text': 'The interactomes are associated with active histone marks and enriched with 5-hydroxymethylcytosine sites.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '23549118',\n",
       "   'score': 0.11248305439949036},\n",
       "  {'start': 522,\n",
       "   'end': 586,\n",
       "   'text': 'In hESCs, genes within the interactomes have elevated expression.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '23549118',\n",
       "   'score': 0.25428152084350586},\n",
       "  {'start': 588,\n",
       "   'end': 675,\n",
       "   'text': 'Additionally, some genes associated with the POU5F1 enhancer contribute to pluripotency.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '23549118',\n",
       "   'score': 0.41570454835891724},\n",
       "  {'start': 677,\n",
       "   'end': 816,\n",
       "   'text': 'Binding sites for multiple DNA binding proteins, including ATF3, CTCF, GABPA, JUND, NANOG, RAD21 and YY1, are enriched in both interactomes.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '23549118',\n",
       "   'score': 0.25257909297943115},\n",
       "  {'start': 818,\n",
       "   'end': 958,\n",
       "   'text': 'The RARG locus, frequently interacting with the POU5F1 locus, has abundant RAD21 binding sites co-localized with other protein binding sites.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '23549118',\n",
       "   'score': 0.11369895935058594},\n",
       "  {'start': 960,\n",
       "   'end': 1075,\n",
       "   'text': 'Thus the interactomes of these two pluripotency genes could be an important part of the regulatory network in hESCs.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '23549118',\n",
       "   'score': 0.3358808755874634},\n",
       "  {'start': 0,\n",
       "   'end': 121,\n",
       "   'text': 'Coactivators p300 and CBP maintain the identity of mouse embryonic stem cells by mediating long-range chromatin structure.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '24648406',\n",
       "   'score': 0.11095663905143738},\n",
       "  {'start': 0,\n",
       "   'end': 189,\n",
       "   'text': 'Master transcription factors Oct4, Sox2, and Nanog are required to maintain the pluripotency and self-renewal of embryonic stem cells (ESCs) by regulating a specific transcriptional network.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24648406',\n",
       "   'score': 0.3523593842983246},\n",
       "  {'start': 191,\n",
       "   'end': 440,\n",
       "   'text': 'A few other transcription factors have been shown to be important in ESCs by interacting with these master transcription factors; however, little is known about the transcriptional mechanisms regulated by coregulators (coactivators and corepressors).',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24648406',\n",
       "   'score': 0.25966501235961914},\n",
       "  {'start': 442,\n",
       "   'end': 565,\n",
       "   'text': 'In this study, we examined the function of two highly homologous coactivators, p300 and CREB-binding protein (CBP), in ESCs.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24648406',\n",
       "   'score': 0.3012901842594147},\n",
       "  {'start': 567,\n",
       "   'end': 673,\n",
       "   'text': 'We find that these two coactivators play redundant roles in maintaining the undifferentiated state of ESCs.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24648406',\n",
       "   'score': 0.2795216143131256},\n",
       "  {'start': 675,\n",
       "   'end': 882,\n",
       "   'text': 'They are recruited by Nanog through physical interaction to Nanog binding loci, mediating the formation of long-range chromatin looping structures, which is essential to maintain ESC-specific gene expression.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24648406',\n",
       "   'score': 0.24124464392662048},\n",
       "  {'start': 884,\n",
       "   'end': 1219,\n",
       "   'text': 'Further functional studies reveal that the p300/CBP binding looping fragments contain enhancer activities, suggesting that the formation of p300/CBP-mediated looping structures may recruit distal enhancers to create a concentration of factors for the transcription activation of genes that are involved in self-renewal and pluripotency.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24648406',\n",
       "   'score': 0.4191829264163971},\n",
       "  {'start': 1221,\n",
       "   'end': 1485,\n",
       "   'text': 'Overall, these results provide a total new insight into the transcriptional regulation mechanism of coactivators p300 and CBP in ESCs, which is important in maintaining self-renewal and pluripotency, by mediating the formation of higher order chromosome structures.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24648406',\n",
       "   'score': 0.2935965359210968},\n",
       "  {'start': 0,\n",
       "   'end': 92,\n",
       "   'text': 'The three-dimensional genome: regulating gene expression during pluripotency and development.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '29042476',\n",
       "   'score': 0.4218951165676117},\n",
       "  {'start': 0,\n",
       "   'end': 234,\n",
       "   'text': 'The precise expression of genes in time and space during embryogenesis is largely influenced by communication between enhancers and promoters, which is propagated and governed by the physical proximity of these elements in the nucleus.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '29042476',\n",
       "   'score': 0.18712130188941956},\n",
       "  {'start': 236,\n",
       "   'end': 412,\n",
       "   'text': 'Here, we review how chromatin domains organize the genome by guiding enhancers to their target genes thereby preventing non-specific interactions with other neighboring regions.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '29042476',\n",
       "   'score': 0.16716381907463074},\n",
       "  {'start': 414,\n",
       "   'end': 610,\n",
       "   'text': 'We also discuss the dynamics of chromatin interactions between enhancers and promoters, as well as the consequent changes in gene expression, that occur in pluripotent cells and during development.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '29042476',\n",
       "   'score': 0.3728809356689453},\n",
       "  {'start': 612,\n",
       "   'end': 875,\n",
       "   'text': 'Finally, we evaluate how genomic changes such as deletions, inversions and duplications affect 3D chromatin configuration overall and lead to ectopic enhancer-promoter contacts, and thus gene misexpression, which can contribute to abnormal development and disease.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '29042476',\n",
       "   'score': 0.22391435503959656},\n",
       "  {'start': 0,\n",
       "   'end': 110,\n",
       "   'text': 'Human Induced Pluripotent Stem Cell-Derived Endothelial Cells for Three-Dimensional Microphysiological Systems.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '28622076',\n",
       "   'score': 0.36644959449768066},\n",
       "  {'start': 0,\n",
       "   'end': 165,\n",
       "   'text': 'Microphysiological systems (MPS), or \"organ-on-a-chip\" platforms, aim to recapitulate in vivo physiology using small-scale in vitro tissue models of human physiology.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28622076',\n",
       "   'score': 0.11068680882453918},\n",
       "  {'start': 167,\n",
       "   'end': 314,\n",
       "   'text': 'While significant efforts have been made to create vascularized tissues, most reports utilize primary endothelial cells that hinder reproducibility.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28622076',\n",
       "   'score': 0.10793313384056091},\n",
       "  {'start': 316,\n",
       "   'end': 483,\n",
       "   'text': 'In this study, we report the use of human induced pluripotent stem cell-derived endothelial cells (iPS-ECs) in developing three-dimensional (3D) microvascular networks.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28622076',\n",
       "   'score': 0.4245969355106354},\n",
       "  {'start': 485,\n",
       "   'end': 610,\n",
       "   'text': 'We established a CDH5-mCherry reporter iPS cell line, which expresses the vascular endothelial (VE)-cadherin fused to mCherry.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28622076',\n",
       "   'score': 0.1105991005897522},\n",
       "  {'start': 612,\n",
       "   'end': 896,\n",
       "   'text': 'The iPS-ECs demonstrate physiological functions characteristic of primary endothelial cells in a series of in vitro assays, including permeability, response to shear stress, and the expression of endothelial markers (CD31, von Willibrand factor, and endothelial nitric oxide synthase).',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28622076',\n",
       "   'score': 0.1143372654914856},\n",
       "  {'start': 898,\n",
       "   'end': 1018,\n",
       "   'text': 'The iPS-ECs form stable, perfusable microvessels over the course of 14 days when cultured within 3D microfluidic devices.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28622076',\n",
       "   'score': 0.14495491981506348},\n",
       "  {'start': 1020,\n",
       "   'end': 1125,\n",
       "   'text': 'We also demonstrate that inhibition of TGF-β signaling improves vascular network formation by the iPS-ECs.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28622076',\n",
       "   'score': 0.12957125902175903},\n",
       "  {'start': 1127,\n",
       "   'end': 1304,\n",
       "   'text': 'We conclude that iPS-ECs can be a source of endothelial cells in MPS providing opportunities for human disease modeling and improving the reproducibility of 3D vascular networks.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28622076',\n",
       "   'score': 0.17117729783058167},\n",
       "  {'start': 0,\n",
       "   'end': 149,\n",
       "   'text': 'Profiling the long noncoding RNA interaction network in the regulatory elements of target genes by chromatin in situ reverse transcription sequencing.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '31315906',\n",
       "   'score': 0.13434267044067383},\n",
       "  {'start': 0,\n",
       "   'end': 134,\n",
       "   'text': 'Long noncoding RNAs (lncRNAs) can regulate the activity of target genes by participating in the organization of chromatin architecture.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31315906',\n",
       "   'score': 0.1428370177745819},\n",
       "  {'start': 136,\n",
       "   'end': 399,\n",
       "   'text': 'We have devised a \"chromatin-RNA in situ reverse transcription sequencing\" (CRIST-seq) approach to profile the lncRNA interaction network in gene regulatory elements by combining the simplicity of RNA biotin labeling with the specificity of the CRISPR/Cas9 system.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31315906',\n",
       "   'score': 0.13011813163757324},\n",
       "  {'start': 401,\n",
       "   'end': 612,\n",
       "   'text': 'Using gene-specific gRNAs, we describe a pluripotency-specific lncRNA interacting network in the promoters of Sox2 and Pou5f1, two critical stem cell factors that are required for the maintenance of pluripotency.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31315906',\n",
       "   'score': 0.3435637354850769},\n",
       "  {'start': 614,\n",
       "   'end': 713,\n",
       "   'text': 'The promoter-interacting lncRNAs were specifically activated during reprogramming into pluripotency.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31315906',\n",
       "   'score': 0.4100116491317749},\n",
       "  {'start': 715,\n",
       "   'end': 789,\n",
       "   'text': 'Knockdown of these lncRNAs caused the stem cells to exit from pluripotency.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31315906',\n",
       "   'score': 0.27840617299079895},\n",
       "  {'start': 791,\n",
       "   'end': 967,\n",
       "   'text': 'In contrast, overexpression of the pluripotency-associated lncRNA activated the promoters of core stem cell factor genes and enhanced fibroblast reprogramming into pluripotency.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31315906',\n",
       "   'score': 0.32683682441711426},\n",
       "  {'start': 969,\n",
       "   'end': 1151,\n",
       "   'text': 'These CRIST-seq data suggest that the Sox2 and Pou5f1 promoters are organized within a unique lncRNA interaction network that determines the fate of pluripotency during reprogramming.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31315906',\n",
       "   'score': 0.3261426091194153},\n",
       "  {'start': 1153,\n",
       "   'end': 1260,\n",
       "   'text': 'This CRIST approach may be broadly used to map lncRNA interaction networks at target loci across the genome.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31315906',\n",
       "   'score': 0.12960880994796753},\n",
       "  {'start': 0,\n",
       "   'end': 76,\n",
       "   'text': 'Autophagy regulates homeostasis of pluripotency-associated proteins in hESCs.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '24170349',\n",
       "   'score': 0.3461417555809021},\n",
       "  {'start': 0,\n",
       "   'end': 163,\n",
       "   'text': 'The pluripotency of embryonic stem cells (ESCs) is maintained by intracellular networks of many pluripotency-associated (PA) proteins such as OCT4, SOX2, and NANOG.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24170349',\n",
       "   'score': 0.4057402014732361},\n",
       "  {'start': 165,\n",
       "   'end': 269,\n",
       "   'text': 'However, the mechanisms underlying the regulation of protein homeostasis for pluripotency remain elusive.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24170349',\n",
       "   'score': 0.3350452780723572},\n",
       "  {'start': 271,\n",
       "   'end': 429,\n",
       "   'text': 'Here, we first demonstrate that autophagy acts together with the ubiquitin-proteasome system (UPS) to modulate the levels of PA proteins in human ESCs (hESCs).',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24170349',\n",
       "   'score': 0.247096985578537},\n",
       "  {'start': 431,\n",
       "   'end': 519,\n",
       "   'text': 'Autophagy inhibition impaired the pluripotency despite increment of PA proteins in hESCs.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24170349',\n",
       "   'score': 0.27625173330307007},\n",
       "  {'start': 521,\n",
       "   'end': 614,\n",
       "   'text': 'Immunogold-electron microscopy confirmed localization of OCT4 molecules within autophagosomes.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24170349',\n",
       "   'score': 0.29148149490356445},\n",
       "  {'start': 616,\n",
       "   'end': 723,\n",
       "   'text': 'Also, knockdown of LC3 expression led to accumulation of PA proteins and reduction of pluripotency in hESCs.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24170349',\n",
       "   'score': 0.3075977861881256},\n",
       "  {'start': 725,\n",
       "   'end': 824,\n",
       "   'text': 'Interestingly, autophagy and the UPS showed differential kinetics in the degradation of PA proteins.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24170349',\n",
       "   'score': 0.11589106917381287},\n",
       "  {'start': 826,\n",
       "   'end': 997,\n",
       "   'text': 'Autophagy inhibition caused enhanced accumulation of both cytoplasmic and nuclear PA proteins, whereas the UPS inhibition led to preferentially degrade nuclear PA proteins.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24170349',\n",
       "   'score': 0.11371856927871704},\n",
       "  {'start': 999,\n",
       "   'end': 1139,\n",
       "   'text': 'Our findings suggest that autophagy modulates homeostasis of PA proteins, providing a new insight in the regulation of pluripotency in hESCs.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24170349',\n",
       "   'score': 0.35039037466049194},\n",
       "  {'start': 0,\n",
       "   'end': 100,\n",
       "   'text': 'Hmga2 is necessary for Otx2-dependent exit of embryonic stem cells from the pluripotent ground state.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '27036552',\n",
       "   'score': 0.2798989415168762},\n",
       "  {'start': 0,\n",
       "   'end': 257,\n",
       "   'text': \"BACKGROUND\\nA crucial event in the differentiation of mouse embryonic stem cells (ESCs) is the exit from the pluripotent ground state that leads to the acquisition of the 'primed' pluripotent phenotype, characteristic of the epiblast-like stem cells (EpiLCs).\",\n",
       "   'is_title': False,\n",
       "   'doc_id': '27036552',\n",
       "   'score': 0.2700392007827759},\n",
       "  {'start': 259,\n",
       "   'end': 333,\n",
       "   'text': 'The transcription factors Oct4 and Otx2 play a key role in this phenomenon.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '27036552',\n",
       "   'score': 0.27299344539642334},\n",
       "  {'start': 335,\n",
       "   'end': 517,\n",
       "   'text': 'In particular, Otx2 pioneers and activates new enhancers, which are silent in ESCs and which control the transcription of genes responsible for the acquisition of the EpiLC phenotype.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '27036552',\n",
       "   'score': 0.4107334613800049},\n",
       "  {'start': 519,\n",
       "   'end': 660,\n",
       "   'text': 'An important point that remains to be addressed is the mechanism through which Otx2 engages the new enhancers and stably associates with them.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '27036552',\n",
       "   'score': 0.17143890261650085},\n",
       "  {'start': 662,\n",
       "   'end': 850,\n",
       "   'text': 'Hmga2 is a member of the high-mobility group family of proteins, non-histone components of chromatin whose expression is high during embryogenesis and becomes low or undetectable in adults.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '27036552',\n",
       "   'score': 0.11868694424629211},\n",
       "  {'start': 852,\n",
       "   'end': 952,\n",
       "   'text': 'Its high expression during embryogenesis suggests that Hmga2 fulfills important roles in development.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '27036552',\n",
       "   'score': 0.1116006076335907},\n",
       "  {'start': 956,\n",
       "   'end': 1055,\n",
       "   'text': 'RESULTS\\nHere, we demonstrate that Hmga2 accumulates soon after the induction of ESC differentiation.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '27036552',\n",
       "   'score': 0.11226573586463928},\n",
       "  {'start': 1057,\n",
       "   'end': 1169,\n",
       "   'text': 'Its suppression hampers the exit of ESCs from the pluripotent ground state and their differentiation into EpiLCs.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '27036552',\n",
       "   'score': 0.2754358649253845},\n",
       "  {'start': 1171,\n",
       "   'end': 1290,\n",
       "   'text': 'Mechanistically, Hmga2 controls the differentiation process by cooperating with Otx2 in the pioneering of new enhancers.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '27036552',\n",
       "   'score': 0.18212589621543884},\n",
       "  {'start': 1292,\n",
       "   'end': 1430,\n",
       "   'text': 'In Hmga2 null induced pluripotent stem cells we observe that Otx2 fails to regulate its target genes upon the induction of differentiation.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '27036552',\n",
       "   'score': 0.27476048469543457},\n",
       "  {'start': 1432,\n",
       "   'end': 1640,\n",
       "   'text': 'Hmga2 associates to Otx2-bound loci in EpiLCs, and in Hmga2 KO cells Otx2 is unable to engage and activate the new enhancers, thus indicating that Hmga2 is required for the binding of Otx2 to its cis-elements.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '27036552',\n",
       "   'score': 0.1892279088497162},\n",
       "  {'start': 1642,\n",
       "   'end': 1797,\n",
       "   'text': 'We find that this mechanism also operates on the Hmga2 gene, which is one of the targets of Otx2, thus indicating the existence of a positive feedback loop.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '27036552',\n",
       "   'score': 0.11528739333152771},\n",
       "  {'start': 1801,\n",
       "   'end': 1915,\n",
       "   'text': 'CONCLUSIONS\\nOur findings reveal a novel mechanism necessary for the exit of ESCs from the pluripotent ground state.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '27036552',\n",
       "   'score': 0.2726737856864929},\n",
       "  {'start': 1917,\n",
       "   'end': 2063,\n",
       "   'text': 'Upon the induction of ESC differentiation, Otx2 alone or in combination with Oct4 engages new enhancers, which are silent in undifferentiated ESCs.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '27036552',\n",
       "   'score': 0.36384570598602295},\n",
       "  {'start': 2065,\n",
       "   'end': 2230,\n",
       "   'text': 'The Hmga2 gene is activated by Otx2 and Hmga2 protein binds to the enhancers targeted by Otx2, thus facilitating the engagement and/or the stable association of Otx2.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '27036552',\n",
       "   'score': 0.20535767078399658},\n",
       "  {'start': 2232,\n",
       "   'end': 2384,\n",
       "   'text': 'Therefore, our results demonstrate that Hmga2 is a key element of the regulatory network that governs the exit of ESCs from the pluripotent ground state.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '27036552',\n",
       "   'score': 0.3137584924697876},\n",
       "  {'start': 0,\n",
       "   'end': 118,\n",
       "   'text': 'Three-dimensional cultured liver-on-a-Chip with mature hepatocyte-like cells derived from human pluripotent stem cells.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '31304567',\n",
       "   'score': 0.42351478338241577},\n",
       "  {'start': 0,\n",
       "   'end': 118,\n",
       "   'text': 'Liver-on-a-Chip technology holds considerable potential for applications in drug screening and chemical-safety testing.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31304567',\n",
       "   'score': 0.11414846777915955},\n",
       "  {'start': 120,\n",
       "   'end': 352,\n",
       "   'text': 'To establish such platforms, functional hepatocytes are required; however, primary hepatocytes are commonly used, despite problems involving donor limitations, lot-to-lot variation, and unsatisfactory two-dimensional culture methods.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31304567',\n",
       "   'score': 0.15832966566085815},\n",
       "  {'start': 354,\n",
       "   'end': 597,\n",
       "   'text': 'Although human pluripotent stem cells (hPSCs) may represent a strong alternative contender to address the aforementioned issues, remaining technological challenges include the robust, highly efficient production of high-purity hepatic clusters.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31304567',\n",
       "   'score': 0.23991736769676208},\n",
       "  {'start': 599,\n",
       "   'end': 723,\n",
       "   'text': 'In addition, current Liver-on-a-Chip platforms are relatively complicated and not applicable for high-throughput experiments.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31304567',\n",
       "   'score': 0.11474660038948059},\n",
       "  {'start': 725,\n",
       "   'end': 848,\n",
       "   'text': 'Here, we develop a very simple Liver-on-a-Chip platform with mature and functional hepatocyte-like cells derived from hPSCs.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31304567',\n",
       "   'score': 0.11265712976455688},\n",
       "  {'start': 850,\n",
       "   'end': 1182,\n",
       "   'text': 'To establish a method for hepatic differentiation of hPSCs, cells were first treated by inhibiting the phosphoinositide 3-kinase- and Rho-associated protein kinase-signaling pathways to stop self-renewal and improve survival, respectively, which enabled the formation of a well-defined endoderm and facilitated hepatocyte commitment.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31304567',\n",
       "   'score': 0.1260453462600708},\n",
       "  {'start': 1184,\n",
       "   'end': 1433,\n",
       "   'text': 'Next, a simple microfluidic device was used to create a three-dimensional (3D) culture environment that enhanced the maturation and function of hepatocyte-like cells by increasing the expression of both hepatic maturation markers and cytochrome P450.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31304567',\n",
       "   'score': 0.16047409176826477},\n",
       "  {'start': 1435,\n",
       "   'end': 1605,\n",
       "   'text': 'Finally, we confirmed improvements in hepatic functions, such as drug uptake/excretion capabilities, in >90% of 3D-matured hepatocyte-like cells by indocyanin green assay.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31304567',\n",
       "   'score': 0.1466076672077179},\n",
       "  {'start': 1607,\n",
       "   'end': 1803,\n",
       "   'text': 'These results indicated that the incorporation of hPSC-derived hepatocytes on our Liver-on-a-Chip platform may serve to enhance the processes involved in drug screening and chemical-safety testing.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31304567',\n",
       "   'score': 0.12223529815673828},\n",
       "  {'start': 0,\n",
       "   'end': 114,\n",
       "   'text': 'The pluripotency factor Nanog regulates pericentromeric heterochromatin organization in mouse embryonic stem cells.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '27125671',\n",
       "   'score': 0.36134791374206543},\n",
       "  {'start': 0,\n",
       "   'end': 85,\n",
       "   'text': 'An open and decondensed chromatin organization is a defining property of pluripotency.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '27125671',\n",
       "   'score': 0.3744581341743469},\n",
       "  {'start': 87,\n",
       "   'end': 261,\n",
       "   'text': 'Several epigenetic regulators have been implicated in maintaining an open chromatin organization, but how these processes are connected to the pluripotency network is unknown.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '27125671',\n",
       "   'score': 0.37988194823265076},\n",
       "  {'start': 263,\n",
       "   'end': 460,\n",
       "   'text': 'Here, we identified a new role for the transcription factor NANOG as a key regulator connecting the pluripotency network with constitutive heterochromatin organization in mouse embryonic stem cells.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '27125671',\n",
       "   'score': 0.35795527696609497},\n",
       "  {'start': 462,\n",
       "   'end': 555,\n",
       "   'text': 'Deletion of Nanog leads to chromatin compaction and the remodeling of heterochromatin domains.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '27125671',\n",
       "   'score': 0.3050897717475891},\n",
       "  {'start': 557,\n",
       "   'end': 643,\n",
       "   'text': 'Forced expression of NANOG in epiblast stem cells is sufficient to decompact chromatin.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '27125671',\n",
       "   'score': 0.2971142828464508},\n",
       "  {'start': 645,\n",
       "   'end': 944,\n",
       "   'text': 'NANOG associates with satellite repeats within heterochromatin domains, contributing to an architecture characterized by highly dispersed chromatin fibers, low levels of H3K9me3, and high major satellite transcription, and the strong transactivation domain of NANOG is required for this organization.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '27125671',\n",
       "   'score': 0.3552335798740387},\n",
       "  {'start': 946,\n",
       "   'end': 1205,\n",
       "   'text': 'The heterochromatin-associated protein SALL1 is a direct cofactor for NANOG, and loss of Sall1 recapitulates the Nanog-null phenotype, but the loss of Sall1 can be circumvented through direct recruitment of the NANOG transactivation domain to major satellites.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '27125671',\n",
       "   'score': 0.2978903651237488},\n",
       "  {'start': 1207,\n",
       "   'end': 1432,\n",
       "   'text': 'These results establish a direct connection between the pluripotency network and chromatin organization and emphasize that maintaining an open heterochromatin architecture is a highly regulated process in embryonic stem cells.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '27125671',\n",
       "   'score': 0.39368849992752075},\n",
       "  {'start': 0,\n",
       "   'end': 180,\n",
       "   'text': 'Tuning Forkhead Box D3 overexpression to promote specific osteogenic differentiation of human embryonic stem cells while reducing pluripotency in a three-dimensional culture system.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '30350469',\n",
       "   'score': 0.39350152015686035},\n",
       "  {'start': 0,\n",
       "   'end': 190,\n",
       "   'text': 'Clinical use of human embryonic stem cells (hESCs) in bone regeneration applications requires that their osteogenic differentiation be highly controllable as well as time- and cost-effective.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '30350469',\n",
       "   'score': 0.2443694770336151},\n",
       "  {'start': 192,\n",
       "   'end': 540,\n",
       "   'text': 'The main goals of the current work were thus (a) to assess whether overexpression of pluripotency regulator Forkhead Box D3 (FOXD3) can enhance the osteogenic commitment of hESCs seeded in three-dimensional (3D) scaffolds and (b) to evaluate if the degree of FOXD3 overexpression regulates the strength and specificity of hESC osteogenic commitment.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '30350469',\n",
       "   'score': 0.3956056535243988},\n",
       "  {'start': 542,\n",
       "   'end': 707,\n",
       "   'text': 'In conducting these studies, an interpenetrating hydrogel network consisting of poly(ethylene glycol) diacrylate and collagen I was utilized as a 3D culture platform.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '30350469',\n",
       "   'score': 0.18305140733718872},\n",
       "  {'start': 709,\n",
       "   'end': 940,\n",
       "   'text': 'Expression of osteogenic, chondrogenic, pluripotency, and germ layer markers by encapsulated hESCs was measured after 2\\xa0weeks of culture in osteogenic medium in the presence or absence doxycycline-induced FOXD3 transgene expression.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '30350469',\n",
       "   'score': 0.3054622411727905},\n",
       "  {'start': 942,\n",
       "   'end': 1183,\n",
       "   'text': 'Towards the first goal, FOXD3 overexpression initiated 24\\xa0hr prior to hESC encapsulation, relative to unstimulated controls, resulted in upregulation of osteogenic markers and enhanced calcium deposition, without promoting off-target effects.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '30350469',\n",
       "   'score': 0.11817711591720581},\n",
       "  {'start': 1185,\n",
       "   'end': 1374,\n",
       "   'text': 'However, when initiation of FOXD3 overexpression was increased from 24 to 48\\xa0hr prior to encapsulation, hESC osteogenic commitment was not further enhanced and off-target effects were noted.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '30350469',\n",
       "   'score': 0.1143130362033844},\n",
       "  {'start': 1376,\n",
       "   'end': 1621,\n",
       "   'text': 'Specifically, relative to 24-hr prestimulation, initiation of FOXD3 overexpression 48\\xa0hr prior to encapsulation yielded increased expression of pluripotency markers while reducing mesodermal but increasing endodermal germ layer marker expression.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '30350469',\n",
       "   'score': 0.29410868883132935},\n",
       "  {'start': 1623,\n",
       "   'end': 1796,\n",
       "   'text': 'Combined, the current results indicate that the controlled overexpression of FOXD3 warrants further investigation as a mechanism to guide enhanced hESC osteogenic commitment.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '30350469',\n",
       "   'score': 0.11418530344963074},\n",
       "  {'start': 0,\n",
       "   'end': 82,\n",
       "   'text': 'Multilineage communication regulates human liver bud development from pluripotency.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '28614297',\n",
       "   'score': 0.2864711880683899},\n",
       "  {'start': 0,\n",
       "   'end': 133,\n",
       "   'text': 'Conventional two-dimensional differentiation from pluripotency fails to recapitulate cell interactions occurring during organogenesis.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28614297',\n",
       "   'score': 0.39616042375564575},\n",
       "  {'start': 135,\n",
       "   'end': 275,\n",
       "   'text': 'Three-dimensional organoids generate complex organ-like tissues; however, it is unclear how heterotypic interactions affect lineage identity.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28614297',\n",
       "   'score': 0.16951856017112732},\n",
       "  {'start': 277,\n",
       "   'end': 411,\n",
       "   'text': 'Here we use single-cell RNA sequencing to reconstruct hepatocyte-like lineage progression from pluripotency in two-dimensional culture.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28614297',\n",
       "   'score': 0.3961751461029053},\n",
       "  {'start': 413,\n",
       "   'end': 590,\n",
       "   'text': 'We then derive three-dimensional liver bud organoids by reconstituting hepatic, stromal, and endothelial interactions, and deconstruct heterogeneity during liver bud development.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28614297',\n",
       "   'score': 0.1645735800266266},\n",
       "  {'start': 592,\n",
       "   'end': 745,\n",
       "   'text': 'We find that liver bud hepatoblasts diverge from the two-dimensional lineage, and express epithelial migration signatures characteristic of organ budding.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28614297',\n",
       "   'score': 0.1639573574066162},\n",
       "  {'start': 747,\n",
       "   'end': 954,\n",
       "   'text': 'We benchmark three-dimensional liver buds against fetal and adult human liver single-cell RNA sequencing data, and find a striking correspondence between the three-dimensional liver bud and fetal liver cells.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28614297',\n",
       "   'score': 0.16483372449874878},\n",
       "  {'start': 956,\n",
       "   'end': 1216,\n",
       "   'text': 'We use a receptor-ligand pairing analysis and a high-throughput inhibitor assay to interrogate signalling in liver buds, and show that vascular endothelial growth factor (VEGF) crosstalk potentiates endothelial network formation and hepatoblast differentiation.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28614297',\n",
       "   'score': 0.13404500484466553},\n",
       "  {'start': 1218,\n",
       "   'end': 1385,\n",
       "   'text': 'Our molecular dissection reveals interlineage communication regulating organoid development, and illuminates previously inaccessible aspects of human liver development.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28614297',\n",
       "   'score': 0.1296645998954773},\n",
       "  {'start': 0,\n",
       "   'end': 118,\n",
       "   'text': 'Identification of novel proteins differentially expressed in pluripotent embryonic stem cells and differentiated cells.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '26399336',\n",
       "   'score': 0.271136999130249},\n",
       "  {'start': 0,\n",
       "   'end': 84,\n",
       "   'text': 'Mammalian pluripotent stem cells possess properties of self-renewal and pluripotency.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '26399336',\n",
       "   'score': 0.2874419093132019},\n",
       "  {'start': 86,\n",
       "   'end': 257,\n",
       "   'text': 'These abilities are maintained by the strict regulation of pluripotent stem cell-specific transcription factor network and unique properties of chromatin in the stem cells.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '26399336',\n",
       "   'score': 0.3912312984466553},\n",
       "  {'start': 259,\n",
       "   'end': 475,\n",
       "   'text': 'Although these major signaling pathways robustly control the characteristics of stem cells, other regulatory factors, such as metabolic pathways, are also known to modulate stem cell proliferation and differentiation.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '26399336',\n",
       "   'score': 0.11688193678855896},\n",
       "  {'start': 477,\n",
       "   'end': 622,\n",
       "   'text': 'In this study, we fractionated protein samples from mouse embryonic stem (ES) cells cultured with or without the leukemia inhibitory factor (LIF).',\n",
       "   'is_title': False,\n",
       "   'doc_id': '26399336',\n",
       "   'score': 0.12306603789329529},\n",
       "  {'start': 624,\n",
       "   'end': 717,\n",
       "   'text': 'Protein expression was quantified by 2-dimensional differential gel electrophoresis (2D-DIGE).',\n",
       "   'is_title': False,\n",
       "   'doc_id': '26399336',\n",
       "   'score': 0.16299530863761902},\n",
       "  {'start': 719,\n",
       "   'end': 849,\n",
       "   'text': 'In total, 44 proteins were identified as being differentially expressed in the pluripotent stem cells and the differentiated cells.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '26399336',\n",
       "   'score': 0.29089367389678955},\n",
       "  {'start': 851,\n",
       "   'end': 1023,\n",
       "   'text': 'Surprisingly, half of the identified proteins were the proteins localized in mitochondria, which supply cellular energy and regulate cell cycle, development, and cell death.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '26399336',\n",
       "   'score': 0.12032938003540039},\n",
       "  {'start': 1025,\n",
       "   'end': 1132,\n",
       "   'text': 'Some of these identified proteins are involved in the metabolic function and the regulation of pluripotency.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '26399336',\n",
       "   'score': 0.38898223638534546},\n",
       "  {'start': 1134,\n",
       "   'end': 1256,\n",
       "   'text': 'Further analysis of the identified proteins could provide new information for the manipulation of pluripotency in ES cells.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '26399336',\n",
       "   'score': 0.31028246879577637},\n",
       "  {'start': 0,\n",
       "   'end': 78,\n",
       "   'text': 'Generation of a Three-Dimensional Kidney Structure from Pluripotent Stem Cells.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '28361318',\n",
       "   'score': 0.39970484375953674},\n",
       "  {'start': 0,\n",
       "   'end': 143,\n",
       "   'text': 'The kidney is a vital organ that has an important role in the maintenance of homeostasis by fluid volume regulation and waste product excretion.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28361318',\n",
       "   'score': 0.13902965188026428},\n",
       "  {'start': 145,\n",
       "   'end': 233,\n",
       "   'text': 'This role cannot be performed without the three-dimensional (3D) structure of the kidney.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28361318',\n",
       "   'score': 0.1641119420528412},\n",
       "  {'start': 235,\n",
       "   'end': 390,\n",
       "   'text': 'Therefore, it is important to generate the 3D structure of the kidney when inducing functional kidney tissue or the whole organ from pluripotent stem cells.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28361318',\n",
       "   'score': 0.3040994703769684},\n",
       "  {'start': 392,\n",
       "   'end': 549,\n",
       "   'text': 'In this chapter, we describe the detailed methods to induce kidney progenitor cells from pluripotent stem cells, which are based on embryological development.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28361318',\n",
       "   'score': 0.26723235845565796},\n",
       "  {'start': 551,\n",
       "   'end': 653,\n",
       "   'text': 'We also provide a method to generate 3D kidney tissue with vascularized glomeruli upon transplantation.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28361318',\n",
       "   'score': 0.1442633867263794},\n",
       "  {'start': 0,\n",
       "   'end': 91,\n",
       "   'text': 'Three-dimensional nanofibrillar surfaces promote self-renewal in mouse embryonic stem cells.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '16150921',\n",
       "   'score': 0.1721205711364746},\n",
       "  {'start': 0,\n",
       "   'end': 167,\n",
       "   'text': 'The regulation of mouse embryonic stem cell (mESC) fate is controlled by the interplay of signaling networks that either promote self-renewal or induce differentiation.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '16150921',\n",
       "   'score': 0.16373324394226074},\n",
       "  {'start': 169,\n",
       "   'end': 301,\n",
       "   'text': 'Leukemia inhibitory factor (LIF) is a cytokine that is required for stem cell renewal in mouse but not in human embryonic stem cells.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '16150921',\n",
       "   'score': 0.11700060963630676},\n",
       "  {'start': 303,\n",
       "   'end': 638,\n",
       "   'text': 'However, feeder layers of embryonic fibroblasts are capable of inducing stem cell renewal in both cell types, suggesting that the self-renewal signaling pathways may also be promoted by other triggers, such as alternative cytokines and/or chemical or physical properties of the extracellular matrix (ECM) secreted by feeder fibroblasts.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '16150921',\n",
       "   'score': 0.11970141530036926},\n",
       "  {'start': 640,\n",
       "   'end': 794,\n",
       "   'text': 'We have recently used a synthetic polyamide matrix (Ultra-Web) whose three-dimensional (3D) nanofibrillar organization resembles the ECM/basement membrane.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '16150921',\n",
       "   'score': 0.19708123803138733},\n",
       "  {'start': 796,\n",
       "   'end': 1006,\n",
       "   'text': 'Growth of mESCs on this nanofibrillar surface greatly enhanced proliferation and self-renewal in comparison with growth on tissue culture surfaces without nanofibers, despite the presence of LIF in both systems.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '16150921',\n",
       "   'score': 0.29000887274742126},\n",
       "  {'start': 1008,\n",
       "   'end': 1306,\n",
       "   'text': 'Enhanced proliferation and self-renewal of the stem cells on nanofibrillar surfaces were correlated with the activation of the small GTPase Rac, the activation of phosphoinositide 3-kinase (PI3K) pathway, and the enhanced expression of Nanog, a homeoprotein required for maintenance of pluripotency.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '16150921',\n",
       "   'score': 0.2947448790073395},\n",
       "  {'start': 1308,\n",
       "   'end': 1411,\n",
       "   'text': 'Inhibitors of PI3K reduced the expression level of Nanog in mESCs cultured on 3D nanofibrillar surfaces.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '16150921',\n",
       "   'score': 0.3962307870388031},\n",
       "  {'start': 1413,\n",
       "   'end': 1639,\n",
       "   'text': 'These results provide support for the view that the three-dimensionality of the culture surface may function as a cue for the activation of Rac and PI3K signaling pathways, resulting in stem cell proliferation and self-renewal.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '16150921',\n",
       "   'score': 0.12439730763435364},\n",
       "  {'start': 0,\n",
       "   'end': 76,\n",
       "   'text': 'Gain of CTCF-Anchored Chromatin Loops Marks the Exit from Naive Pluripotency.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '30414923',\n",
       "   'score': 0.2822280824184418},\n",
       "  {'start': 0,\n",
       "   'end': 160,\n",
       "   'text': 'The genome of pluripotent stem cells adopts a unique three-dimensional architecture featuring weakly condensed heterochromatin and large nucleosome-free regions.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '30414923',\n",
       "   'score': 0.3918892443180084},\n",
       "  {'start': 162,\n",
       "   'end': 325,\n",
       "   'text': 'Yet, it is unknown whether structural loops and contact domains display characteristics that distinguish embryonic stem cells (ESCs) from differentiated cell types.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '30414923',\n",
       "   'score': 0.2605474591255188},\n",
       "  {'start': 327,\n",
       "   'end': 493,\n",
       "   'text': 'We used genome-wide chromosome conformation capture and super-resolution imaging to determine nuclear organization in mouse ESC and neural stem cell (NSC) derivatives.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '30414923',\n",
       "   'score': 0.1448041796684265},\n",
       "  {'start': 495,\n",
       "   'end': 583,\n",
       "   'text': 'We found that loss of pluripotency is accompanied by widespread gain of structural loops.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '30414923',\n",
       "   'score': 0.31085795164108276},\n",
       "  {'start': 585,\n",
       "   'end': 770,\n",
       "   'text': 'This general architectural change correlates with enhanced binding of CTCF and cohesins and more pronounced insulation of contacts across chromatin boundaries in lineage-committed cells.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '30414923',\n",
       "   'score': 0.1192331314086914},\n",
       "  {'start': 772,\n",
       "   'end': 858,\n",
       "   'text': 'Reprogramming NSCs to pluripotency restores the unique features of ESC domain topology.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '30414923',\n",
       "   'score': 0.28120577335357666},\n",
       "  {'start': 860,\n",
       "   'end': 969,\n",
       "   'text': 'Domains defined by the anchors of loops established upon differentiation are enriched for developmental genes.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '30414923',\n",
       "   'score': 0.11662551760673523},\n",
       "  {'start': 971,\n",
       "   'end': 1160,\n",
       "   'text': 'Chromatin loop formation is a pervasive structural alteration to the genome that accompanies exit from pluripotency and delineates the spatial segregation of developmentally regulated genes.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '30414923',\n",
       "   'score': 0.28757819533348083},\n",
       "  {'start': 0,\n",
       "   'end': 85,\n",
       "   'text': 'Functional and Sustainable 3D Human Neural Network Models from Pluripotent Stem Cells.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '33304995',\n",
       "   'score': 0.3980183005332947},\n",
       "  {'start': 0,\n",
       "   'end': 229,\n",
       "   'text': 'Three-dimensional in vitro cell culture models, particularly for the central nervous system, allow for the exploration of mechanisms of organ development, cellular interactions, and disease progression within defined environments.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '33304995',\n",
       "   'score': 0.1784084439277649},\n",
       "  {'start': 231,\n",
       "   'end': 405,\n",
       "   'text': 'Here we describe the development and characterization of three-dimensional tissue models that promote the differentiation and long-term survival of functional neural networks.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '33304995',\n",
       "   'score': 0.19315552711486816},\n",
       "  {'start': 407,\n",
       "   'end': 649,\n",
       "   'text': 'These tissue cultures show diverse cell populations including neurons and glial cells (astrocytes) interacting in 3D with spontaneous neural activity confirmed through electrophysiological recordings and calcium imaging over at least 8 months.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '33304995',\n",
       "   'score': 0.15080520510673523},\n",
       "  {'start': 651,\n",
       "   'end': 964,\n",
       "   'text': 'This approach allows for the direct integration of pluripotent stem cells into the 3D construct bypassing early neural differentiation steps (embryoid bodies and neural rosettes), which streamlines the process while also providing a system that can be manipulated to support a variety of experimental applications.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '33304995',\n",
       "   'score': 0.3149849772453308},\n",
       "  {'start': 966,\n",
       "   'end': 1250,\n",
       "   'text': \"This tissue model has been tested in stem cells derived from healthy individuals as well as Alzheimer's and Parkinson's disease patients, with similar growth and gene expression responses indicating potential use in the modeling of disease states related to neurodegenerative diseases.\",\n",
       "   'is_title': False,\n",
       "   'doc_id': '33304995',\n",
       "   'score': 0.1207895576953888},\n",
       "  {'start': 0,\n",
       "   'end': 84,\n",
       "   'text': 'A genome-wide RNAi screen reveals determinants of human embryonic stem cell identity.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '20953172',\n",
       "   'score': 0.11572203040122986},\n",
       "  {'start': 0,\n",
       "   'end': 117,\n",
       "   'text': 'The derivation of human ES cells (hESCs) from human blastocysts represents one of the milestones in stem cell biology.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '20953172',\n",
       "   'score': 0.26213860511779785},\n",
       "  {'start': 119,\n",
       "   'end': 285,\n",
       "   'text': 'The full potential of hESCs in research and clinical applications requires a detailed understanding of the genetic network that governs the unique properties of hESCs.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '20953172',\n",
       "   'score': 0.34298673272132874},\n",
       "  {'start': 287,\n",
       "   'end': 423,\n",
       "   'text': 'Here, we report a genome-wide RNA interference screen to identify genes which regulate self-renewal and pluripotency properties in hESCs.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '20953172',\n",
       "   'score': 0.27338868379592896},\n",
       "  {'start': 425,\n",
       "   'end': 583,\n",
       "   'text': 'Interestingly, functionally distinct complexes involved in transcriptional regulation and chromatin remodelling are among the factors identified in the screen.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '20953172',\n",
       "   'score': 0.14615768194198608},\n",
       "  {'start': 585,\n",
       "   'end': 768,\n",
       "   'text': 'To understand the roles of these potential regulators of hESCs, we studied transcription factor PRDM14 to gain new insights into its functional roles in the regulation of pluripotency.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '20953172',\n",
       "   'score': 0.3653643727302551},\n",
       "  {'start': 770,\n",
       "   'end': 887,\n",
       "   'text': 'We showed that PRDM14 regulates directly the expression of key pluripotency gene POU5F1 through its proximal enhancer.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '20953172',\n",
       "   'score': 0.3836219310760498},\n",
       "  {'start': 889,\n",
       "   'end': 1129,\n",
       "   'text': 'Genome-wide location profiling experiments revealed that PRDM14 colocalized extensively with other key transcription factors such as OCT4, NANOG and SOX2, indicating that PRDM14 is integrated into the core transcriptional regulatory network.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '20953172',\n",
       "   'score': 0.31281960010528564},\n",
       "  {'start': 1131,\n",
       "   'end': 1311,\n",
       "   'text': 'More importantly, in a gain-of-function assay, we showed that PRDM14 is able to enhance the efficiency of reprogramming of human fibroblasts in conjunction with OCT4, SOX2 and KLF4.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '20953172',\n",
       "   'score': 0.2948090732097626},\n",
       "  {'start': 1313,\n",
       "   'end': 1538,\n",
       "   'text': 'Altogether, our study uncovers a wealth of novel hESC regulators wherein PRDM14 exemplifies a key transcription factor required for the maintenance of hESC identity and the reacquisition of pluripotency in human somatic cells.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '20953172',\n",
       "   'score': 0.3012164533138275},\n",
       "  {'start': 0,\n",
       "   'end': 80,\n",
       "   'text': 'Derivation of an interaction/regulation network describing pluripotency in human.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '22548825',\n",
       "   'score': 0.38108497858047485},\n",
       "  {'start': 0,\n",
       "   'end': 169,\n",
       "   'text': 'Identification of the key genes/proteins of pluripotency and their interrelationships is an important step in understanding the induction and maintenance of pluripotency.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '22548825',\n",
       "   'score': 0.3298477530479431},\n",
       "  {'start': 171,\n",
       "   'end': 265,\n",
       "   'text': 'Experimental approaches have accumulated large amounts of interaction/regulation data in mouse.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '22548825',\n",
       "   'score': 0.13394737243652344},\n",
       "  {'start': 267,\n",
       "   'end': 418,\n",
       "   'text': 'We investigate how far such information can be transferred to human, the species of maximum interest, for which experimental data are much more limited.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '22548825',\n",
       "   'score': 0.11906912922859192},\n",
       "  {'start': 420,\n",
       "   'end': 519,\n",
       "   'text': 'To address this issue, we mapped an existing mouse pluripotency network (the PluriNetWork) to human.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '22548825',\n",
       "   'score': 0.30386391282081604},\n",
       "  {'start': 521,\n",
       "   'end': 701,\n",
       "   'text': 'We transferred interaction and regulation links between genes/proteins from mouse to human on the basis of orthologous relationship of the genes/proteins (called interolog mapping).',\n",
       "   'is_title': False,\n",
       "   'doc_id': '22548825',\n",
       "   'score': 0.13766825199127197},\n",
       "  {'start': 703,\n",
       "   'end': 887,\n",
       "   'text': 'To reduce the number of false positives, we used four different methods: phylogenetic profiling, Gene Ontology semantic similarity, gene co-expression, and RNA interference (RNAi) data.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '22548825',\n",
       "   'score': 0.12338033318519592},\n",
       "  {'start': 889,\n",
       "   'end': 1057,\n",
       "   'text': 'The methods and the resulting networks were evaluated by a novel approach using the information about the genes known to be involved in pluripotency from the literature.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '22548825',\n",
       "   'score': 0.3402690291404724},\n",
       "  {'start': 1059,\n",
       "   'end': 1192,\n",
       "   'text': 'The RNAi method proved best for filtering out unlikely interactions, so it was used to construct the final human pluripotency network.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '22548825',\n",
       "   'score': 0.3018547296524048},\n",
       "  {'start': 1194,\n",
       "   'end': 1330,\n",
       "   'text': 'The RNAi data are based on human embryonic stem cells (hESCs) that are generally considered to be in a (primed) epiblast stem cell state.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '22548825',\n",
       "   'score': 0.26566484570503235},\n",
       "  {'start': 1332,\n",
       "   'end': 1538,\n",
       "   'text': 'Therefore, we assume that the final human network may reflect the (primed) epiblast stem cell state more closely, while the mouse network reflects the (unprimed/naïve) embryonic stem cell state more closely.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '22548825',\n",
       "   'score': 0.13441291451454163},\n",
       "  {'start': 0,\n",
       "   'end': 94,\n",
       "   'text': 'Chromatin organization in pluripotent cells: emerging approaches to study and disrupt function.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '26206085',\n",
       "   'score': 0.3437819480895996},\n",
       "  {'start': 0,\n",
       "   'end': 182,\n",
       "   'text': 'Translating the vast amounts of genomic and epigenomic information accumulated on the linear genome into three-dimensional models of nuclear organization is a current major challenge.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '26206085',\n",
       "   'score': 0.21401411294937134},\n",
       "  {'start': 184,\n",
       "   'end': 430,\n",
       "   'text': 'In response to this challenge, recent technological innovations based on chromosome conformation capture methods in combination with increasingly powerful functional approaches have revealed exciting insights into key aspects of genome regulation.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '26206085',\n",
       "   'score': 0.13456183671951294},\n",
       "  {'start': 432,\n",
       "   'end': 699,\n",
       "   'text': 'These findings have led to an emerging model where the genome is folded and compartmentalized into highly conserved topological domains that are further divided into functional subdomains containing physical loops that bring cis-regulatory elements to close proximity.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '26206085',\n",
       "   'score': 0.12079134583473206},\n",
       "  {'start': 701,\n",
       "   'end': 902,\n",
       "   'text': 'Targeted functional experiments, largely based on designable DNA-binding proteins, have begun to define the major architectural proteins required to establish and maintain appropriate genome regulation.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '26206085',\n",
       "   'score': 0.1325986683368683},\n",
       "  {'start': 904,\n",
       "   'end': 1105,\n",
       "   'text': 'Here, we focus on the accessible and well-characterized system of pluripotent cells to review the functional role of chromatin organization in regulating pluripotency, differentiation and reprogramming.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '26206085',\n",
       "   'score': 0.39008259773254395},\n",
       "  {'start': 0,\n",
       "   'end': 137,\n",
       "   'text': 'Long-range chromatin contacts in embryonic stem cells reveal a role for pluripotency factors and polycomb proteins in genome organization.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '24035354',\n",
       "   'score': 0.3310115337371826},\n",
       "  {'start': 0,\n",
       "   'end': 104,\n",
       "   'text': 'The relationship between 3D organization of the genome and gene-regulatory networks is poorly understood.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24035354',\n",
       "   'score': 0.2418501079082489},\n",
       "  {'start': 106,\n",
       "   'end': 341,\n",
       "   'text': 'Here, we examined long-range chromatin interactions genome-wide in mouse embryonic stem cells (ESCs), iPSCs, and fibroblasts and uncovered a pluripotency-specific genome organization that is gradually reestablished during reprogramming.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24035354',\n",
       "   'score': 0.3077548146247864},\n",
       "  {'start': 343,\n",
       "   'end': 523,\n",
       "   'text': 'Our data confirm that long-range chromatin interactions are primarily associated with the spatial segregation of open and closed chromatin, defining overall chromosome conformation.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24035354',\n",
       "   'score': 0.12268230319023132},\n",
       "  {'start': 525,\n",
       "   'end': 752,\n",
       "   'text': 'Additionally, we identified two further levels of genome organization in ESCs characterized by colocalization of regions with high pluripotency factor occupancy and strong enrichment for Polycomb proteins/H3K27me3, respectively.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24035354',\n",
       "   'score': 0.35384947061538696},\n",
       "  {'start': 754,\n",
       "   'end': 1001,\n",
       "   'text': 'Underlining the independence of these networks and their functional relevance for genome organization, loss of the Polycomb protein Eed diminishes interactions between Polycomb-regulated regions without altering overarching chromosome conformation.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24035354',\n",
       "   'score': 0.18397995829582214},\n",
       "  {'start': 1003,\n",
       "   'end': 1252,\n",
       "   'text': 'Together, our data highlight a pluripotency-specific genome organization in which pluripotency factors such as Nanog and H3K27me3 occupy distinct nuclear spaces and reveal a role for cell-type-specific gene-regulatory networks in genome organization.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24035354',\n",
       "   'score': 0.3784448206424713},\n",
       "  {'start': 0,\n",
       "   'end': 112,\n",
       "   'text': 'Coordination of m(6)A mRNA Methylation and Gene Transcription by ZFP217 Regulates Pluripotency and Reprogramming.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '26526723',\n",
       "   'score': 0.29806822538375854},\n",
       "  {'start': 0,\n",
       "   'end': 160,\n",
       "   'text': 'Epigenetic and epitranscriptomic networks have important functions in maintaining the pluripotency of embryonic stem cells (ESCs) and somatic cell reprogramming.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '26526723',\n",
       "   'score': 0.38562652468681335},\n",
       "  {'start': 162,\n",
       "   'end': 266,\n",
       "   'text': 'However, the mechanisms integrating the actions of these distinct networks are only partially understood.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '26526723',\n",
       "   'score': 0.14018943905830383},\n",
       "  {'start': 268,\n",
       "   'end': 399,\n",
       "   'text': 'Here we show that the chromatin-associated zinc finger protein 217 (ZFP217) coordinates epigenetic and epitranscriptomic regulation.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '26526723',\n",
       "   'score': 0.14924296736717224},\n",
       "  {'start': 401,\n",
       "   'end': 648,\n",
       "   'text': 'ZFP217 interacts with several epigenetic regulators, activates the transcription of key pluripotency genes, and modulates N6-methyladenosine (m(6)A) deposition on their transcripts by sequestering the enzyme m(6)A methyltransferase-like 3 (METTL3).',\n",
       "   'is_title': False,\n",
       "   'doc_id': '26526723',\n",
       "   'score': 0.268758624792099},\n",
       "  {'start': 650,\n",
       "   'end': 881,\n",
       "   'text': 'Consistently, Zfp217 depletion compromises ESC self-renewal and somatic cell reprogramming, globally increases m(6)A RNA levels, and enhances m(6)A modification of the Nanog, Sox2, Klf4, and c-Myc mRNAs, promoting their degradation.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '26526723',\n",
       "   'score': 0.2757059633731842},\n",
       "  {'start': 883,\n",
       "   'end': 1015,\n",
       "   'text': 'ZFP217 binds its own target gene mRNAs, which are also METTL3 associated, and is enriched at promoters of m(6)A-modified transcripts.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '26526723',\n",
       "   'score': 0.1268824338912964},\n",
       "  {'start': 1017,\n",
       "   'end': 1173,\n",
       "   'text': 'Collectively, these findings shed light on how a transcription factor can tightly couple gene transcription to m(6)A RNA modification to ensure ESC identity.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '26526723',\n",
       "   'score': 0.11609819531440735},\n",
       "  {'start': 0,\n",
       "   'end': 96,\n",
       "   'text': 'Regulation of protein abundance in pluripotent cells undergoing commitment to the neural lineage.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '8707862',\n",
       "   'score': 0.38138315081596375},\n",
       "  {'start': 0,\n",
       "   'end': 254,\n",
       "   'text': 'The P19 cell line is a widely studied model of neural differentiation When pluripotent P19 cells are cultured as aggregates in the presence of retinoic acid for 4 days, the cells commit to the neural fate, but have not yet undergone overt differentiation.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '8707862',\n",
       "   'score': 0.27461665868759155},\n",
       "  {'start': 256,\n",
       "   'end': 376,\n",
       "   'text': 'Two-dimensional polyacrylamide gel electrophoresis was used to analyze cellular protein expression during this induction.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '8707862',\n",
       "   'score': 0.1835336685180664},\n",
       "  {'start': 378,\n",
       "   'end': 431,\n",
       "   'text': 'Approximately 500 abundant polypeptides were analyzed.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '8707862',\n",
       "   'score': 0.11626586318016052},\n",
       "  {'start': 433,\n",
       "   'end': 577,\n",
       "   'text': 'Seventeen polypeptides were upregulated during induction; several of these were significantly regulated 48 h after the addition of retinoic acid.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '8707862',\n",
       "   'score': 0.11912712454795837},\n",
       "  {'start': 579,\n",
       "   'end': 611,\n",
       "   'text': 'No downregulations were observed.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '8707862',\n",
       "   'score': 0.11626586318016052},\n",
       "  {'start': 613,\n",
       "   'end': 705,\n",
       "   'text': 'Fifteen of the 17 polypeptides continued to be expressed throughout terminal differentiation.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '8707862',\n",
       "   'score': 0.11900350451469421},\n",
       "  {'start': 707,\n",
       "   'end': 846,\n",
       "   'text': 'The upregulation of 14 of the 17 polypeptides requires both retinoic acid and aggregation, which alone do not induce neural differentiation.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '8707862',\n",
       "   'score': 0.12266531586647034},\n",
       "  {'start': 848,\n",
       "   'end': 981,\n",
       "   'text': 'Furthermore, these regulated polypeptides are expressed in neural tissue, suggesting they are associated with neural function in vivo.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '8707862',\n",
       "   'score': 0.12146833539009094},\n",
       "  {'start': 983,\n",
       "   'end': 1096,\n",
       "   'text': 'Embryonic stem cells, a totipotent line, also neurally differentiate in response to retinoic acid and aggregation.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '8707862',\n",
       "   'score': 0.12005940079689026},\n",
       "  {'start': 1098,\n",
       "   'end': 1260,\n",
       "   'text': 'Comparison of embryonic stem cells to P19 cells shows that the two systems regulate a similar set of polypeptides and are thus likely to utilize a similar pathway.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '8707862',\n",
       "   'score': 0.12565675377845764},\n",
       "  {'start': 1262,\n",
       "   'end': 1402,\n",
       "   'text': 'These studies are a step toward determining the full extent of regulation involved in the commitment of pluripotent cells to the neural fate.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '8707862',\n",
       "   'score': 0.3657374382019043},\n",
       "  {'start': 0,\n",
       "   'end': 127,\n",
       "   'text': 'A competitive protein interaction network buffers Oct4-mediated differentiation to promote pluripotency in embryonic stem cells.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '24104477',\n",
       "   'score': 0.346407413482666},\n",
       "  {'start': 0,\n",
       "   'end': 217,\n",
       "   'text': \"Pluripotency in embryonic stem cells is maintained through the activity of a small set of transcription factors centred around Oct4 and Nanog, which control the expression of 'self-renewal' and 'differentiation' genes.\",\n",
       "   'is_title': False,\n",
       "   'doc_id': '24104477',\n",
       "   'score': 0.3075852394104004},\n",
       "  {'start': 219,\n",
       "   'end': 419,\n",
       "   'text': 'Here, we combine single-cell quantitative immunofluorescence microscopy and gene expression analysis, together with theoretical modelling, to investigate how the activity of those factors is regulated.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24104477',\n",
       "   'score': 0.12300476431846619},\n",
       "  {'start': 421,\n",
       "   'end': 584,\n",
       "   'text': 'We uncover a key role for post-translational regulation in the maintenance of pluripotency, which complements the well-established transcriptional regulatory layer.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24104477',\n",
       "   'score': 0.3389921188354492},\n",
       "  {'start': 586,\n",
       "   'end': 778,\n",
       "   'text': 'Specifically, we find that the activity of a network of protein complexes involving Nanog, Oct4, Tcf3, and β-catenin suffices to account for the behavior of ES cells under different conditions.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24104477',\n",
       "   'score': 0.3235238790512085},\n",
       "  {'start': 780,\n",
       "   'end': 945,\n",
       "   'text': 'Our results suggest that the function of the network is to buffer the transcriptional activity of Oct4, which appears to be the main determinant to exit pluripotency.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24104477',\n",
       "   'score': 0.31913793087005615},\n",
       "  {'start': 947,\n",
       "   'end': 1137,\n",
       "   'text': 'The protein network explains the mechanisms underlying the gain and loss of function in different mutants, and brings us closer to a full understanding of the molecular basis of pluripotency.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24104477',\n",
       "   'score': 0.3752123713493347},\n",
       "  {'start': 0,\n",
       "   'end': 138,\n",
       "   'text': 'Embryonic Stem Cell Engineering with a Glycomimetic FGF2/BMP4 Co-Receptor Drives Mesodermal Differentiation in a Three-Dimensional Culture.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '30157624',\n",
       "   'score': 0.17118901014328003},\n",
       "  {'start': 0,\n",
       "   'end': 300,\n",
       "   'text': 'Cell surface glycans, such as heparan sulfate (HS), are increasingly identified as co-regulators of growth factor signaling in early embryonic development; therefore, chemical tailoring of HS activity within the cellular glycocalyx of stem cells offers an opportunity to control their differentiation.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '30157624',\n",
       "   'score': 0.12159267067909241},\n",
       "  {'start': 302,\n",
       "   'end': 506,\n",
       "   'text': 'The growth factors FGF2 and BMP4 are involved in mediating the exit of murine embryonic stem cells (mESCs) from their pluripotent state and their differentiation toward mesodermal cell types, respectively.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '30157624',\n",
       "   'score': 0.311210036277771},\n",
       "  {'start': 508,\n",
       "   'end': 690,\n",
       "   'text': 'Here, we report a method for remodeling the glycocalyx of mutant Ext1-/- mESCs with defective biosynthesis of HS to drive their mesodermal differentiation in an embryoid body culture.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '30157624',\n",
       "   'score': 0.27280616760253906},\n",
       "  {'start': 692,\n",
       "   'end': 999,\n",
       "   'text': 'Lipid-functionalized synthetic HS-mimetic glycopolymers with affinity for both FGF2 and BMP4 were introduced into the plasma membrane of Ext1-/- mESCs, where they acted as functional co-receptors of these growth factors and facilitated signal transduction through associated MAPK and Smad signaling pathways.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '30157624',\n",
       "   'score': 0.2697141170501709},\n",
       "  {'start': 1001,\n",
       "   'end': 1226,\n",
       "   'text': 'We demonstrate that these materials can be employed to remodel Ext1-/- mESCs within three-dimensional embryoid body structures, providing enhanced association of BMP4 at the cell surface and driving mesodermal differentiation.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '30157624',\n",
       "   'score': 0.38241466879844666},\n",
       "  {'start': 1228,\n",
       "   'end': 1491,\n",
       "   'text': 'As a more complete understanding of the function of HS in regulating development continues to emerge, this simple glycocalyx engineering method is poised to enable precise control over growth factor signaling activity and outcomes of differentiation in stem cells.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '30157624',\n",
       "   'score': 0.1398715078830719},\n",
       "  {'start': 0,\n",
       "   'end': 116,\n",
       "   'text': 'A membrane-associated β-catenin/Oct4 complex correlates with ground-state pluripotency in mouse embryonic stem cells.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '23444350',\n",
       "   'score': 0.2991563379764557},\n",
       "  {'start': 0,\n",
       "   'end': 255,\n",
       "   'text': 'The maintenance of pluripotency in mouse embryonic stem cells (mESCs) relies on the activity of a transcriptional network that is fuelled by the activity of three transcription factors (Nanog, Oct4 and Sox2) and balanced by the repressive activity of Tcf3.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '23444350',\n",
       "   'score': 0.37772685289382935},\n",
       "  {'start': 257,\n",
       "   'end': 370,\n",
       "   'text': 'Extracellular signals modulate the activity of the network and regulate the differentiation capacity of the cells.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '23444350',\n",
       "   'score': 0.141049325466156},\n",
       "  {'start': 372,\n",
       "   'end': 556,\n",
       "   'text': 'Wnt/β-catenin signaling has emerged as a significant potentiator of pluripotency: increases in the levels of β-catenin regulate the activity of Oct4 and Nanog, and enhance pluripotency.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '23444350',\n",
       "   'score': 0.3263707458972931},\n",
       "  {'start': 558,\n",
       "   'end': 737,\n",
       "   'text': 'A recent report shows that β-catenin achieves some of these effects by modulating the activity of Tcf3, and that this effect does not require its transcriptional activation domain.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '23444350',\n",
       "   'score': 0.12156513333320618},\n",
       "  {'start': 739,\n",
       "   'end': 952,\n",
       "   'text': 'Here, we show that during self-renewal there is negligible transcriptional activity of β-catenin and that this is due to its tight association with membranes, where we find it in a complex with Oct4 and E-cadherin.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '23444350',\n",
       "   'score': 0.2583959698677063},\n",
       "  {'start': 954,\n",
       "   'end': 1079,\n",
       "   'text': 'Differentiation triggers a burst of Wnt/β-catenin transcriptional activity that coincides with the disassembly of the complex.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '23444350',\n",
       "   'score': 0.11976107954978943},\n",
       "  {'start': 1081,\n",
       "   'end': 1223,\n",
       "   'text': 'Our results establish that β-catenin, but not its transcriptional activity, is central to pluripotency acting through a β-catenin/Oct4 complex.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '23444350',\n",
       "   'score': 0.28451353311538696},\n",
       "  {'start': 0,\n",
       "   'end': 33,\n",
       "   'text': 'CARM1 mediates modulation of Sox2.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '22046437',\n",
       "   'score': 0.11844712495803833},\n",
       "  {'start': 0,\n",
       "   'end': 127,\n",
       "   'text': 'Sox2 is a key component of the transcription factor network that maintains the pluripotent state of embryonic stem cells (ESCs).',\n",
       "   'is_title': False,\n",
       "   'doc_id': '22046437',\n",
       "   'score': 0.34139561653137207},\n",
       "  {'start': 129,\n",
       "   'end': 263,\n",
       "   'text': 'Sox2 is regulated by multiple post-translational modifications, including ubiquitination, sumoylation, acetylation and phosphorylation.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '22046437',\n",
       "   'score': 0.12168490886688232},\n",
       "  {'start': 265,\n",
       "   'end': 462,\n",
       "   'text': 'Here we report that Sox2 is in association with and methylated by coactivator-associated arginine methyltransferase 1 (CARM1), a protein arginine methyltransferase that plays a pivotal role in ESCs.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '22046437',\n",
       "   'score': 0.30046582221984863},\n",
       "  {'start': 464,\n",
       "   'end': 570,\n",
       "   'text': 'We found that CARM1 facilitates Sox2-mediated transactivation and directly methylates Sox2 at arginine 113.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '22046437',\n",
       "   'score': 0.11919710040092468},\n",
       "  {'start': 572,\n",
       "   'end': 625,\n",
       "   'text': 'This methylation event enhances Sox2 self-association.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '22046437',\n",
       "   'score': 0.11716225743293762},\n",
       "  {'start': 627,\n",
       "   'end': 725,\n",
       "   'text': 'Furthermore, the physiological retention of Sox2 on chromatin restricts the Sox2 methylation level.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '22046437',\n",
       "   'score': 0.12018075585365295},\n",
       "  {'start': 727,\n",
       "   'end': 902,\n",
       "   'text': 'Our study reveals the direct regulation of Sox2 by CARM1 that sheds lights on how arginine methylation signals are integrated into the pluripotent transcription factor network.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '22046437',\n",
       "   'score': 0.3782678246498108},\n",
       "  {'start': 0,\n",
       "   'end': 64,\n",
       "   'text': 'Chaperones and Beyond as Key Players in Pluripotency Maintenance.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '31428613',\n",
       "   'score': 0.2854463458061218},\n",
       "  {'start': 0,\n",
       "   'end': 161,\n",
       "   'text': 'Pluripotency is orchestrated by distinct players and chaperones and their partners have emerged as pivotal molecules in proteostasis control to maintain stemness.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31428613',\n",
       "   'score': 0.30126267671585083},\n",
       "  {'start': 163,\n",
       "   'end': 344,\n",
       "   'text': 'The proteostasis network consists of diverse interconnected pathways that function dynamically according to the needs of the cell to quality control and maintain protein homeostasis.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31428613',\n",
       "   'score': 0.15134593844413757},\n",
       "  {'start': 346,\n",
       "   'end': 529,\n",
       "   'text': 'The proteostasis machinery of pluripotent stem cells (PSCs) is finely adjusted in response to distinct stimuli during cell fate commitment to determine successful organism development.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31428613',\n",
       "   'score': 0.3076562285423279},\n",
       "  {'start': 531,\n",
       "   'end': 635,\n",
       "   'text': 'Growing evidence has shown different classes of chaperones regulating crucial cellular processes in PSCs.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31428613',\n",
       "   'score': 0.13689666986465454},\n",
       "  {'start': 637,\n",
       "   'end': 811,\n",
       "   'text': \"Histones chaperones promote proper nucleosome assembly and modulate the epigenetic regulation of factors involved in PSCs' rapid turnover from pluripotency to differentiation.\",\n",
       "   'is_title': False,\n",
       "   'doc_id': '31428613',\n",
       "   'score': 0.3611997961997986},\n",
       "  {'start': 813,\n",
       "   'end': 1021,\n",
       "   'text': 'The life cycle of pluripotency proteins from synthesis and folding, transport and degradation is finely regulated by chaperones and co-factors either to maintain the stemness status or to cell fate commitment.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31428613',\n",
       "   'score': 0.3082446753978729},\n",
       "  {'start': 1023,\n",
       "   'end': 1175,\n",
       "   'text': 'Here, we summarize current knowledge of the chaperone network that govern stemness and present the versatile role of chaperones in stem cells resilience.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31428613',\n",
       "   'score': 0.14538276195526123},\n",
       "  {'start': 1177,\n",
       "   'end': 1443,\n",
       "   'text': 'Elucidation of the intricate regulation of pluripotency, dissecting in detail molecular determinants and drivers, is fundamental to understanding the properties of stem cells in order to provide a reliable foundation for biomedical research and regenerative medicine.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31428613',\n",
       "   'score': 0.37499818205833435},\n",
       "  {'start': 0,\n",
       "   'end': 118,\n",
       "   'text': 'Qualitative modeling identifies IL-11 as a novel regulator in maintaining self-renewal in human pluripotent stem cells.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '24194720',\n",
       "   'score': 0.2609670162200928},\n",
       "  {'start': 0,\n",
       "   'end': 161,\n",
       "   'text': 'Pluripotency in human embryonic stem cells (hESCs) and induced pluripotent stem cells (iPSCs) is regulated by three transcription factors-OCT3/4, SOX2, and NANOG.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24194720',\n",
       "   'score': 0.3290077745914459},\n",
       "  {'start': 163,\n",
       "   'end': 329,\n",
       "   'text': 'To fully exploit the therapeutic potential of these cells it is essential to have a good mechanistic understanding of the maintenance of self-renewal and pluripotency.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24194720',\n",
       "   'score': 0.293235182762146},\n",
       "  {'start': 331,\n",
       "   'end': 567,\n",
       "   'text': 'In this study, we demonstrate a powerful systems biology approach in which we first expand literature-based network encompassing the core regulators of pluripotency by assessing the behavior of genes targeted by perturbation experiments.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24194720',\n",
       "   'score': 0.3735601305961609},\n",
       "  {'start': 569,\n",
       "   'end': 752,\n",
       "   'text': 'We focused our attention on highly regulated genes encoding cell surface and secreted proteins as these can be more easily manipulated by the use of inhibitors or recombinant proteins.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24194720',\n",
       "   'score': 0.1216198205947876},\n",
       "  {'start': 754,\n",
       "   'end': 911,\n",
       "   'text': 'Qualitative modeling based on combining boolean networks and in silico perturbation experiments were employed to identify novel pluripotency-regulating genes.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24194720',\n",
       "   'score': 0.372661292552948},\n",
       "  {'start': 913,\n",
       "   'end': 1112,\n",
       "   'text': 'We validated Interleukin-11 (IL-11) and demonstrate that this cytokine is a novel pluripotency-associated factor capable of supporting self-renewal in the absence of exogenously added bFGF in culture.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24194720',\n",
       "   'score': 0.33625221252441406},\n",
       "  {'start': 1114,\n",
       "   'end': 1267,\n",
       "   'text': 'To date, the various protocols for hESCs maintenance require supplementation with bFGF to activate the Activin/Nodal branch of the TGFβ signaling pathway.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24194720',\n",
       "   'score': 0.2501968741416931},\n",
       "  {'start': 1269,\n",
       "   'end': 1460,\n",
       "   'text': 'Additional evidence supporting our findings is that IL-11 belongs to the same protein family as LIF, which is known to be necessary for maintaining pluripotency in mouse but not in human ESCs.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24194720',\n",
       "   'score': 0.2922254800796509},\n",
       "  {'start': 1462,\n",
       "   'end': 1552,\n",
       "   'text': 'These cytokines operate through the same gp130 receptor which interacts with Janus kinases.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24194720',\n",
       "   'score': 0.12059038877487183},\n",
       "  {'start': 1554,\n",
       "   'end': 1694,\n",
       "   'text': 'Our finding might explain why mESCs are in a more naïve cell state compared to hESCs and how to convert primed hESCs back to the naïve state.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24194720',\n",
       "   'score': 0.2690315842628479},\n",
       "  {'start': 1696,\n",
       "   'end': 1926,\n",
       "   'text': 'Taken together, our integrative modeling approach has identified novel genes as putative candidates to be incorporated into the expansion of the current gene regulatory network responsible for inducing and maintaining pluripotency.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24194720',\n",
       "   'score': 0.3067598342895508},\n",
       "  {'start': 0,\n",
       "   'end': 114,\n",
       "   'text': 'mTOR supports long-term self-renewal and suppresses mesoderm and endoderm activities of human embryonic stem cells.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '19416884',\n",
       "   'score': 0.12078273296356201},\n",
       "  {'start': 0,\n",
       "   'end': 237,\n",
       "   'text': 'Despite the recent identification of the transcriptional regulatory circuitry involving SOX2, NANOG, and OCT-4, the intracellular signaling networks that control pluripotency of human embryonic stem cells (hESCs) remain largely undefined.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '19416884',\n",
       "   'score': 0.37591099739074707},\n",
       "  {'start': 239,\n",
       "   'end': 407,\n",
       "   'text': 'Here, we demonstrate an essential role for the serine/threonine protein kinase mammalian target of rapamycin (mTOR) in regulating hESC long-term undifferentiated growth.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '19416884',\n",
       "   'score': 0.14699479937553406},\n",
       "  {'start': 409,\n",
       "   'end': 533,\n",
       "   'text': 'Inhibition of mTOR impairs pluripotency, prevents cell proliferation, and enhances mesoderm and endoderm activities in hESCs.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '19416884',\n",
       "   'score': 0.30433404445648193},\n",
       "  {'start': 535,\n",
       "   'end': 779,\n",
       "   'text': 'At the molecular level, mTOR integrates signals from extrinsic pluripotency-supporting factors and represses the transcriptional activities of a subset of developmental and growth-inhibitory genes, as revealed by genome-wide microarray analyses.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '19416884',\n",
       "   'score': 0.2596684694290161},\n",
       "  {'start': 781,\n",
       "   'end': 880,\n",
       "   'text': 'Repression of the developmental genes by mTOR is necessary for the maintenance of hESC pluripotency.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '19416884',\n",
       "   'score': 0.3138730823993683},\n",
       "  {'start': 882,\n",
       "   'end': 978,\n",
       "   'text': 'These results uncover a novel signaling mechanism by which mTOR controls fate decisions in hESCs.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '19416884',\n",
       "   'score': 0.2693483829498291},\n",
       "  {'start': 980,\n",
       "   'end': 1066,\n",
       "   'text': 'Our findings may contribute to effective strategies for tissue repair and regeneration.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '19416884',\n",
       "   'score': 0.11936423182487488},\n",
       "  {'start': 0,\n",
       "   'end': 104,\n",
       "   'text': 'Microvessel Network Formation and Interactions with Pancreatic Islets in Three-Dimensional Chip Cultures.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '31724494',\n",
       "   'score': 0.20889729261398315},\n",
       "  {'start': 0,\n",
       "   'end': 157,\n",
       "   'text': 'The pancreatic islet is a highly vascularized micro-organ, and rapid revascularization postislet transplantation is important for islet survival and function.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31724494',\n",
       "   'score': 0.12302398681640625},\n",
       "  {'start': 159,\n",
       "   'end': 314,\n",
       "   'text': 'However, the various mechanisms involved in islet revascularization are not fully understood, and we currently lack good in vitro platforms to explore this.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31724494',\n",
       "   'score': 0.128957599401474},\n",
       "  {'start': 316,\n",
       "   'end': 555,\n",
       "   'text': 'Our aim for this study was to generate perfusable microvascular networks in a microfluidic chip device, in which islets could be easily integrated, to establish an in vitro platform for investigations on islet-microvasculature interactions.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31724494',\n",
       "   'score': 0.14003503322601318},\n",
       "  {'start': 557,\n",
       "   'end': 942,\n",
       "   'text': 'We compared the ability of mesenchymal stem cells (MSCs) and fibroblasts to support microvascular network formation by human umbilical vein endothelial cells (HUVECs) and human induced pluripotent stem cell-derived endothelial colony-forming cell in two-dimensional and three-dimensional models of angiogenesis, and tested the effect of different culture media on microvessel formation.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31724494',\n",
       "   'score': 0.4021335542201996},\n",
       "  {'start': 944,\n",
       "   'end': 1093,\n",
       "   'text': 'HUVECs that were supported by MSCs formed patent and perfusable networks in a fibrin gel, whereas networks supported by fibroblasts rapidly regressed.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31724494',\n",
       "   'score': 0.1437842845916748},\n",
       "  {'start': 1095,\n",
       "   'end': 1182,\n",
       "   'text': 'Network morphology could be controlled by adjusting relative cell numbers and densities.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31724494',\n",
       "   'score': 0.14021751284599304},\n",
       "  {'start': 1184,\n",
       "   'end': 1379,\n",
       "   'text': 'Incorporation of isolated rat islets demonstrated that islets recruit local microvasculature in vitro, but that the microvessels did not invade islets, at least during the course of these studies.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31724494',\n",
       "   'score': 0.12237435579299927},\n",
       "  {'start': 1381,\n",
       "   'end': 1616,\n",
       "   'text': 'This in vitro microvascularization platform can provide a useful tool to study how various parameters affect islet integration with microvascular networks and could also be utilized for studies of vascularization of other organ systems.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31724494',\n",
       "   'score': 0.14367058873176575},\n",
       "  {'start': 1618,\n",
       "   'end': 1760,\n",
       "   'text': 'Impact statement To improve pancreatic islet graft survival and function posttransplantation, rapid and adequate revascularization is critical.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31724494',\n",
       "   'score': 0.12122046947479248},\n",
       "  {'start': 1762,\n",
       "   'end': 1897,\n",
       "   'text': 'Efforts to improve islet revascularization are demanding due to an insufficient understanding of the mechanisms involved in the process.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31724494',\n",
       "   'score': 0.12830641865730286},\n",
       "  {'start': 1899,\n",
       "   'end': 2086,\n",
       "   'text': 'We have applied a microfluidics platform to generate microvascular networks, and by incorporating pancreatic islets, we were able to study microvasculature-islet interactions in real time.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31724494',\n",
       "   'score': 0.14255136251449585},\n",
       "  {'start': 2088,\n",
       "   'end': 2259,\n",
       "   'text': 'This platform can provide a useful tool to study islet integration with microvascular networks, and could be utilized for studies of vascularization of other organ systems.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31724494',\n",
       "   'score': 0.14122667908668518},\n",
       "  {'start': 2261,\n",
       "   'end': 2367,\n",
       "   'text': 'Moreover, this work may be adapted toward developing a prevascularized islet construct for transplantation.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31724494',\n",
       "   'score': 0.11760681867599487},\n",
       "  {'start': 0,\n",
       "   'end': 109,\n",
       "   'text': 'Microfibrous Scaffolds Enhance Endothelial Differentiation and Organization of Induced Pluripotent Stem Cells.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '28936269',\n",
       "   'score': 0.35005834698677063},\n",
       "  {'start': 0,\n",
       "   'end': 180,\n",
       "   'text': 'INTRODUCTION\\nHuman induced pluripotent stem cells (iPSCs) are a promising source of endothelial cells (iPSC-ECs) for engineering three-dimensional (3D) vascularized cardiac tissues.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28936269',\n",
       "   'score': 0.3848375380039215},\n",
       "  {'start': 182,\n",
       "   'end': 470,\n",
       "   'text': 'To mimic cardiac microvasculature, in which capillaries are oriented in parallel, we hypothesized that endothelial differentiation of iPSCs within topographically aligned 3D scaffolds would be a facile one-step approach to generate iPSC-ECs as well as induce aligned vascular organization.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28936269',\n",
       "   'score': 0.19390136003494263},\n",
       "  {'start': 474,\n",
       "   'end': 650,\n",
       "   'text': 'METHODS\\nHuman iPSCs underwent endothelial differentiation within electrospun 3D polycaprolactone (PCL) scaffolds having either randomly oriented or parallel-aligned microfibers.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28936269',\n",
       "   'score': 0.15493014454841614},\n",
       "  {'start': 652,\n",
       "   'end': 879,\n",
       "   'text': 'Using transcriptional, protein, and endothelial functional assays, endothelial differentiation was compared between conventional two-dimensional (2D) films and 3D scaffolds having either randomly oriented or aligned microfibers.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28936269',\n",
       "   'score': 0.17965486645698547},\n",
       "  {'start': 881,\n",
       "   'end': 1001,\n",
       "   'text': 'Furthermore, the role of parallel-aligned microfiber patterning on the organization of vessel-like networks was assessed.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28936269',\n",
       "   'score': 0.17952904105186462},\n",
       "  {'start': 1005,\n",
       "   'end': 1207,\n",
       "   'text': 'RESULTS\\nThe cells in both the randomly oriented and aligned 3D scaffolds demonstrated an 11-fold upregulation in gene expression of the endothelial phenotypic marker, CD31, compared to cells on 2D films.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28936269',\n",
       "   'score': 0.16266173124313354},\n",
       "  {'start': 1209,\n",
       "   'end': 1324,\n",
       "   'text': 'This upregulation corresponded to >3-fold increase in CD31 protein expression in 3D scaffolds, compared to 2D films.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28936269',\n",
       "   'score': 0.16749972105026245},\n",
       "  {'start': 1326,\n",
       "   'end': 1533,\n",
       "   'text': 'Concomitantly, other endothelial phenotypic markers including CD144 and endothelial nitric oxide synthase also showed significant transcriptional upregulation in 3D scaffolds by >7-fold, compared to 2D films.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28936269',\n",
       "   'score': 0.1590895652770996},\n",
       "  {'start': 1535,\n",
       "   'end': 1689,\n",
       "   'text': 'Nitric oxide production, which is characteristic of endothelial function, was produced 4-fold more abundantly in 3D scaffolds, compared to on 2D PCL films.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28936269',\n",
       "   'score': 0.16083160042762756},\n",
       "  {'start': 1691,\n",
       "   'end': 1950,\n",
       "   'text': 'Within aligned scaffolds, the iPSC-ECs displayed parallel-aligned vascular-like networks with 70% longer branch length, compared to cells in randomly oriented scaffolds, suggesting that fiber topography modulates vascular network-like formation and patterning.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28936269',\n",
       "   'score': 0.14416229724884033},\n",
       "  {'start': 1954,\n",
       "   'end': 2184,\n",
       "   'text': 'CONCLUSION\\nTogether, these results demonstrate that 3D scaffold structure promotes endothelial differentiation, compared to 2D substrates, and that aligned topographical patterning induces anisotropic vascular network organization.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28936269',\n",
       "   'score': 0.2151528298854828},\n",
       "  {'start': 0,\n",
       "   'end': 186,\n",
       "   'text': 'Proteomic analysis of early reprogramming events in murine somatic cells incubated with Xenopus laevis oocyte extracts demonstrates network associations with induced pluripotency markers.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '23768116',\n",
       "   'score': 0.3259560763835907},\n",
       "  {'start': 0,\n",
       "   'end': 192,\n",
       "   'text': 'The reprogramming of somatic cells into a pluripotent/embryonic-like state holds great potential for regenerative medicine, bypassing ethical issues associated with embryonic stem cells (ESCs).',\n",
       "   'is_title': False,\n",
       "   'doc_id': '23768116',\n",
       "   'score': 0.3115348219871521},\n",
       "  {'start': 194,\n",
       "   'end': 456,\n",
       "   'text': 'Numerous methods, including somatic cell nuclear transfer (SCNT), fusion to pluripotent cells, the use of cell extracts, and expression of transcription factors, have been used to reprogram cells into ES-like cells [termed induced pluripotent stem cells (iPSCs)].',\n",
       "   'is_title': False,\n",
       "   'doc_id': '23768116',\n",
       "   'score': 0.2667217254638672},\n",
       "  {'start': 458,\n",
       "   'end': 735,\n",
       "   'text': 'This study investigated early events in the nuclei of permeabilized murine somatic cells incubated in cytoplasmic extract prepared from Xenopus laevis germinal vesicle-stage oocytes by identifying proteins that showed significant quantitative changes using proteomic techniques.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '23768116',\n",
       "   'score': 0.12321022152900696},\n",
       "  {'start': 737,\n",
       "   'end': 939,\n",
       "   'text': 'A total of 69 protein spots from two-dimensional electrophoresis were identified as being significantly altered in expression after treatment, and 38 proteins were identified by tandem mass spectrometry.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '23768116',\n",
       "   'score': 0.17819857597351074},\n",
       "  {'start': 941,\n",
       "   'end': 1175,\n",
       "   'text': 'Network analysis was used to highlight pathway connections and interactions between these identified proteins, which were found to be involved in many functions--primarily nuclear structure and dynamics, transcription, and translation.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '23768116',\n",
       "   'score': 0.15581241250038147},\n",
       "  {'start': 1177,\n",
       "   'end': 1462,\n",
       "   'text': 'The pluripotency markers Klf4, c-Myc, Nanog, and POU5F1 were highlighted by the interaction network analysis, as well as other compounds/proteins known to be repressed in pluripotent cells [e.g., protein kinase C (PRKC)] or enhanced during differentiation of ESCs (e.g., retinoic acid).',\n",
       "   'is_title': False,\n",
       "   'doc_id': '23768116',\n",
       "   'score': 0.37398213148117065},\n",
       "  {'start': 1464,\n",
       "   'end': 1583,\n",
       "   'text': 'The network analysis also indicated additional proteins and pathways potentially involved in early reprogramming events.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '23768116',\n",
       "   'score': 0.15523281693458557},\n",
       "  {'start': 0,\n",
       "   'end': 46,\n",
       "   'text': 'Establishing pluripotency in early development.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '25857441',\n",
       "   'score': 0.3000926971435547},\n",
       "  {'start': 0,\n",
       "   'end': 197,\n",
       "   'text': 'The earliest steps of embryonic development involve important changes in chromatin and transcription factor networks, which are orchestrated to establish pluripotent cells that will form the embryo.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '25857441',\n",
       "   'score': 0.32913291454315186},\n",
       "  {'start': 199,\n",
       "   'end': 399,\n",
       "   'text': 'DNA methylation, histone modifications, the pluripotency regulatory network of transcription factors, maternal factors and newly translated proteins all contribute to these transitions in dynamic ways.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '25857441',\n",
       "   'score': 0.3381897211074829},\n",
       "  {'start': 401,\n",
       "   'end': 474,\n",
       "   'text': 'Moreover, these dynamics are linked to the onset of zygotic transcription.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '25857441',\n",
       "   'score': 0.1210588812828064},\n",
       "  {'start': 476,\n",
       "   'end': 679,\n",
       "   'text': 'We will review recent progress in our understanding of chromatin state and regulation of gene expression in the context of embryonic development in vertebrates, in particular mouse, Xenopus and zebrafish.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '25857441',\n",
       "   'score': 0.1436864137649536},\n",
       "  {'start': 681,\n",
       "   'end': 856,\n",
       "   'text': 'We include work on mouse embryonic stem cells and highlight work that illustrates how early embryonic dynamics establish gene regulatory networks and the state of pluripotency.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '25857441',\n",
       "   'score': 0.3701986074447632},\n",
       "  {'start': 0,\n",
       "   'end': 53,\n",
       "   'text': 'Role of CTCF in the regulation of microRNA expression.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '23056006',\n",
       "   'score': 0.14439508318901062},\n",
       "  {'start': 0,\n",
       "   'end': 93,\n",
       "   'text': 'MicroRNAs (miRNAs) are small non-coding RNAs that regulate expression of various target genes.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '23056006',\n",
       "   'score': 0.11944600939750671},\n",
       "  {'start': 95,\n",
       "   'end': 222,\n",
       "   'text': 'miRNAs are expressed in a tissue-specific manner and play important roles in cell proliferation, apoptosis, and differentiation.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '23056006',\n",
       "   'score': 0.12247636914253235},\n",
       "  {'start': 224,\n",
       "   'end': 385,\n",
       "   'text': 'Epigenetic alterations such as DNA methylation and histone modification are essential for chromatin remodeling and regulation of gene expression including miRNAs.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '23056006',\n",
       "   'score': 0.13989046216011047},\n",
       "  {'start': 387,\n",
       "   'end': 593,\n",
       "   'text': 'The CCCTC-binding factor, CTCF, is known to bind insulators and exhibits an enhancer-blocking and barrier function, and more recently, it also contributes to the three-dimensional organization of the genome.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '23056006',\n",
       "   'score': 0.34126001596450806},\n",
       "  {'start': 595,\n",
       "   'end': 742,\n",
       "   'text': 'CTCF can also serve as a barrier against the spread of DNA methylation and histone repressive marks over promoter regions of tumor suppressor genes.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '23056006',\n",
       "   'score': 0.1955692172050476},\n",
       "  {'start': 744,\n",
       "   'end': 907,\n",
       "   'text': 'Recent studies have shown that CTCF is also involved in the regulation of miRNAs such as miR-125b1, miR-375, and the miR-290 cluster in cancer cells and stem cells.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '23056006',\n",
       "   'score': 0.15107238292694092},\n",
       "  {'start': 909,\n",
       "   'end': 992,\n",
       "   'text': 'miR-125b1 is a candidate of tumor suppressor and is silenced in breast cancer cells.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '23056006',\n",
       "   'score': 0.12520340085029602},\n",
       "  {'start': 994,\n",
       "   'end': 1092,\n",
       "   'text': 'On the other hand, miR-375 may have oncogenic function and is overexpressed in breast cancer cells.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '23056006',\n",
       "   'score': 0.12647035717964172},\n",
       "  {'start': 1094,\n",
       "   'end': 1253,\n",
       "   'text': 'CTCF is involved in the regulation of both miR-125b1 and miR-375, indicating that there are various patterns of CTCF-associated epigenetic regulation of miRNAs.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '23056006',\n",
       "   'score': 0.16101661324501038},\n",
       "  {'start': 1255,\n",
       "   'end': 1355,\n",
       "   'text': 'CTCF may also play a key role in the pluripotency of cells through the regulation of miR-290 cluster.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '23056006',\n",
       "   'score': 0.3826563060283661},\n",
       "  {'start': 1357,\n",
       "   'end': 1494,\n",
       "   'text': 'These observations suggest that CTCF-mediated regulation of miRNAs could be a novel approach for cancer therapy and regenerative medicine.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '23056006',\n",
       "   'score': 0.13874012231826782},\n",
       "  {'start': 0,\n",
       "   'end': 95,\n",
       "   'text': 'Expanding the Circuitry of Pluripotency by Selective Isolation of Chromatin-Associated Proteins.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '27773674',\n",
       "   'score': 0.32780998945236206},\n",
       "  {'start': 0,\n",
       "   'end': 255,\n",
       "   'text': 'Maintenance of pluripotency is regulated by a network of transcription factors coordinated by Oct4, Sox2, and Nanog (OSN), yet a systematic investigation of the composition and dynamics of the OSN protein network specifically on chromatin is still missing.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '27773674',\n",
       "   'score': 0.37387555837631226},\n",
       "  {'start': 257,\n",
       "   'end': 461,\n",
       "   'text': 'Here we have developed a method combining ChIP with selective isolation of chromatin-associated proteins (SICAP) followed by mass spectrometry to identify chromatin-bound partners of a protein of interest.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '27773674',\n",
       "   'score': 0.1308966875076294},\n",
       "  {'start': 463,\n",
       "   'end': 630,\n",
       "   'text': 'ChIP-SICAP in mouse embryonic stem cells (ESCs) identified over 400 proteins associating with OSN, including several whose interaction depends on the pluripotent state.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '27773674',\n",
       "   'score': 0.26358672976493835},\n",
       "  {'start': 632,\n",
       "   'end': 817,\n",
       "   'text': 'Trim24, a previously unrecognized protein in the network, converges with OSN on multiple enhancers and suppresses the expression of developmental genes while activating cell cycle genes.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '27773674',\n",
       "   'score': 0.24970591068267822},\n",
       "  {'start': 819,\n",
       "   'end': 968,\n",
       "   'text': 'Consistently, Trim24 significantly improved efficiency of cellular reprogramming, demonstrating its direct functionality in establishing pluripotency.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '27773674',\n",
       "   'score': 0.2762302756309509},\n",
       "  {'start': 970,\n",
       "   'end': 1125,\n",
       "   'text': 'Collectively, ChIP-SICAP provides a powerful tool to decode chromatin protein composition, further enhanced by its integrative capacity to perform ChIP-seq.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '27773674',\n",
       "   'score': 0.12471142411231995},\n",
       "  {'start': 0,\n",
       "   'end': 113,\n",
       "   'text': 'The role of Hippo signaling pathway and mechanotransduction in tuning embryoid body formation and differentiation.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '31951024',\n",
       "   'score': 0.12618011236190796},\n",
       "  {'start': 0,\n",
       "   'end': 149,\n",
       "   'text': 'Embryoid bodies (EBs) are the three-dimensional aggregates of pluripotent stem cells that are used as a model system for the in vitro differentiation.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31951024',\n",
       "   'score': 0.40712907910346985},\n",
       "  {'start': 151,\n",
       "   'end': 273,\n",
       "   'text': 'EBs mimic the early stages of embryogenesis and are considered as a potential biomimetic body in tuning the stem cell fate.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31951024',\n",
       "   'score': 0.1248508095741272},\n",
       "  {'start': 275,\n",
       "   'end': 513,\n",
       "   'text': 'Although EBs have a spheroid shape, they are not formed accidentally by the agglomeration of cells; they are formed by the deliberate and programmed aggregation of stem cells in a complex topological and biophysical microstructure instead.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31951024',\n",
       "   'score': 0.12491637468338013},\n",
       "  {'start': 515,\n",
       "   'end': 695,\n",
       "   'text': 'EBs could be programmed to promisingly differentiate into the desired germ layers with specific cell lineages, in response to intra- and extra-biochemical and biomechanical signals.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31951024',\n",
       "   'score': 0.12371566891670227},\n",
       "  {'start': 697,\n",
       "   'end': 813,\n",
       "   'text': 'Hippo signaling and mechanotransduction are the key pathways in controlling the formation and differentiation of EBs.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31951024',\n",
       "   'score': 0.12538403272628784},\n",
       "  {'start': 815,\n",
       "   'end': 1003,\n",
       "   'text': 'The activity of the Hippo pathway strongly relies on cell-cell junctions, cell polarity, cellular architecture, cellular metabolism, and mechanical cues in the surrounding microenvironment.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31951024',\n",
       "   'score': 0.12559983134269714},\n",
       "  {'start': 1005,\n",
       "   'end': 1246,\n",
       "   'text': 'Although the Hippo pathway was initially thought to limit the size of the organ by inhibiting the proliferation and the promotion of apoptosis, the evidence suggests that this pathway even regulates stem cell self-renewal and differentiation.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31951024',\n",
       "   'score': 0.12291550636291504},\n",
       "  {'start': 1248,\n",
       "   'end': 1500,\n",
       "   'text': 'Considering the abovementioned explanations, the present study investigated the interplay of the Hippo signaling pathway, mechanotransduction, differentiation, and proliferation pathways to draw the molecular network involved in the control of EBs fate.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31951024',\n",
       "   'score': 0.15443626046180725},\n",
       "  {'start': 1502,\n",
       "   'end': 1688,\n",
       "   'text': 'In addition, this study highlighted several neglected critical parameters regarding EB formation, in the interplay with the Hippo core component involved in the promising differentiation.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31951024',\n",
       "   'score': 0.12807956337928772},\n",
       "  {'start': 0,\n",
       "   'end': 160,\n",
       "   'text': 'Linking Hematopoietic Differentiation to Co-Expressed Sets of Pluripotency-Associated and Imprinted Genes and to Regulatory microRNA-Transcription Factor Motifs.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '28052084',\n",
       "   'score': 0.3573504388332367},\n",
       "  {'start': 0,\n",
       "   'end': 177,\n",
       "   'text': 'Maintenance of cell pluripotency, differentiation, and reprogramming are regulated by complex gene regulatory networks (GRNs) including monoallelically-expressed imprinted genes.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28052084',\n",
       "   'score': 0.37089645862579346},\n",
       "  {'start': 179,\n",
       "   'end': 289,\n",
       "   'text': 'Besides transcriptional control, epigenetic modifications and microRNAs contribute to cellular differentiation.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28052084',\n",
       "   'score': 0.12057551741600037},\n",
       "  {'start': 291,\n",
       "   'end': 531,\n",
       "   'text': 'As a model system for studying the capacity of cells to preserve their pluripotency state and the onset of differentiation and subsequent specialization, murine hematopoiesis was used and compared to embryonic stem cells (ESCs) as a control.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28052084',\n",
       "   'score': 0.2716447710990906},\n",
       "  {'start': 533,\n",
       "   'end': 696,\n",
       "   'text': 'Using published microarray data, the expression profiles of two sets of genes, pluripotent and imprinted, were compared to a third set of known hematopoietic genes.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28052084',\n",
       "   'score': 0.27532678842544556},\n",
       "  {'start': 698,\n",
       "   'end': 845,\n",
       "   'text': 'We found that more than half of the pluripotent and imprinted genes are clearly upregulated in ESCs but subsequently repressed during hematopoiesis.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28052084',\n",
       "   'score': 0.2780001163482666},\n",
       "  {'start': 847,\n",
       "   'end': 952,\n",
       "   'text': 'The remaining genes were either upregulated in hematopoietic progenitors or in differentiated blood cells.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28052084',\n",
       "   'score': 0.12251365184783936},\n",
       "  {'start': 954,\n",
       "   'end': 1126,\n",
       "   'text': 'The three gene sets each consist of three similarly behaving gene groups with similar expression profiles in various lineages of the hematopoietic system as well as in ESCs.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28052084',\n",
       "   'score': 0.28763651847839355},\n",
       "  {'start': 1128,\n",
       "   'end': 1344,\n",
       "   'text': 'To explain this co-regulation behavior, we explored the transcriptional and post-transcriptional mechanisms of pluripotent and imprinted genes and their regulator/target miRNAs in six different hematopoietic lineages.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28052084',\n",
       "   'score': 0.33364272117614746},\n",
       "  {'start': 1346,\n",
       "   'end': 1519,\n",
       "   'text': 'Therewith, lineage-specific transcription factor (TF)-miRNA regulatory networks were generated and their topologies and functional impacts during hematopoiesis were analyzed.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28052084',\n",
       "   'score': 0.14304688572883606},\n",
       "  {'start': 1521,\n",
       "   'end': 1756,\n",
       "   'text': 'This led to the identification of TF-miRNA co-regulatory motifs, for which we validated the contribution to the cellular development of the corresponding lineage in terms of statistical significance and relevance to biological evidence.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28052084',\n",
       "   'score': 0.12596312165260315},\n",
       "  {'start': 1758,\n",
       "   'end': 1876,\n",
       "   'text': 'This analysis also identified key miRNAs and TFs/genes that might play important roles in the derived lineage networks.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28052084',\n",
       "   'score': 0.14833849668502808},\n",
       "  {'start': 1878,\n",
       "   'end': 2205,\n",
       "   'text': 'These molecular associations suggest new aspects of the cellular regulation of the onset of cellular differentiation and during hematopoiesis involving, on one hand, pluripotent genes that were previously not discussed in the context of hematopoiesis and, on the other hand, involve genes that are related to genomic imprinting.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28052084',\n",
       "   'score': 0.3046821355819702},\n",
       "  {'start': 2207,\n",
       "   'end': 2329,\n",
       "   'text': 'These are new links between hematopoiesis and cellular differentiation and the important field of epigenetic modifications.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28052084',\n",
       "   'score': 0.1235848069190979},\n",
       "  {'start': 0,\n",
       "   'end': 107,\n",
       "   'text': 'RAD21 cooperates with pluripotency transcription factors in the maintenance of embryonic stem cell identity.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '21589869',\n",
       "   'score': 0.30433428287506104},\n",
       "  {'start': 0,\n",
       "   'end': 240,\n",
       "   'text': 'For self-renewal, embryonic stem cells (ESCs) require the expression of specific transcription factors accompanied by a particular chromosome organization to maintain a balance between pluripotency and the capacity for rapid differentiation.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '21589869',\n",
       "   'score': 0.3559303879737854},\n",
       "  {'start': 242,\n",
       "   'end': 349,\n",
       "   'text': 'However, how transcriptional regulation is linked to chromosome organization in ESCs is not well understood.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '21589869',\n",
       "   'score': 0.36969661712646484},\n",
       "  {'start': 351,\n",
       "   'end': 517,\n",
       "   'text': 'Here we show that the cohesin component RAD21 exhibits a functional role in maintaining ESC identity through association with the pluripotency transcriptional network.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '21589869',\n",
       "   'score': 0.31612884998321533},\n",
       "  {'start': 519,\n",
       "   'end': 743,\n",
       "   'text': 'ChIP-seq analyses of RAD21 reveal an ESC specific cohesin binding pattern that is characterized by CTCF independent co-localization of cohesin with pluripotency related transcription factors Oct4, Nanog, Sox2, Esrrb and Klf4.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '21589869',\n",
       "   'score': 0.3019070029258728},\n",
       "  {'start': 745,\n",
       "   'end': 967,\n",
       "   'text': 'Upon ESC differentiation, most of these binding sites disappear and instead new CTCF independent RAD21 binding sites emerge, which are enriched for binding sites of transcription factors implicated in early differentiation.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '21589869',\n",
       "   'score': 0.1257777214050293},\n",
       "  {'start': 969,\n",
       "   'end': 1191,\n",
       "   'text': 'Furthermore, knock-down of RAD21 causes expression changes that are similar to expression changes after Nanog depletion, demonstrating the functional relevance of the RAD21--pluripotency transcriptional network association.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '21589869',\n",
       "   'score': 0.3287307918071747},\n",
       "  {'start': 1193,\n",
       "   'end': 1345,\n",
       "   'text': 'Finally, we show that Nanog physically interacts with the cohesin or cohesin interacting proteins STAG1 and WAPL further substantiating this association.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '21589869',\n",
       "   'score': 0.2604931890964508},\n",
       "  {'start': 1347,\n",
       "   'end': 1534,\n",
       "   'text': 'Based on these findings we propose that a dynamic placement of cohesin by pluripotency transcription factors contributes to a chromosome organization supporting the ESC expression program.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '21589869',\n",
       "   'score': 0.332438588142395},\n",
       "  {'start': 0,\n",
       "   'end': 91,\n",
       "   'text': 'Roles of Dppa2 in the regulation of the present status and future of pluripotent stem cells.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '22641845',\n",
       "   'score': 0.3610232174396515},\n",
       "  {'start': 0,\n",
       "   'end': 156,\n",
       "   'text': 'Maintenance of undifferentiated states of pluripotent embryonic stem cells is regulated by a complex network of transcription factors and signaling pathways.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '22641845',\n",
       "   'score': 0.36264264583587646},\n",
       "  {'start': 158,\n",
       "   'end': 372,\n",
       "   'text': 'Recent reports revealed that developmental pluripotency-associated 2 (Dppa2), which regulates chromatin structures, plays important roles in the maintenance of pluripotency and proliferation of embryonic stem cells.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '22641845',\n",
       "   'score': 0.3484847843647003},\n",
       "  {'start': 374,\n",
       "   'end': 543,\n",
       "   'text': 'Interestingly, developmental pluripotency-associated 2 is involved not only in the normal development of lung but also in the pathogenesis of non-small cell lung cancers.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '22641845',\n",
       "   'score': 0.36342453956604004},\n",
       "  {'start': 545,\n",
       "   'end': 683,\n",
       "   'text': 'These results suggest that an epigenetic regulator of pluripotent stem cells plays important roles in normal development and tumorigenesis.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '22641845',\n",
       "   'score': 0.31424176692962646},\n",
       "  {'start': 0,\n",
       "   'end': 111,\n",
       "   'text': 'AutoSOME: a clustering method for identifying gene expression modules without prior knowledge of cluster number.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '20202218',\n",
       "   'score': 0.1210375726222992},\n",
       "  {'start': 0,\n",
       "   'end': 142,\n",
       "   'text': 'BACKGROUND\\nClustering the information content of large high-dimensional gene expression datasets has widespread application in \"omics\" biology.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '20202218',\n",
       "   'score': 0.16202124953269958},\n",
       "  {'start': 144,\n",
       "   'end': 342,\n",
       "   'text': 'Unfortunately, the underlying structure of these natural datasets is often fuzzy, and the computational identification of data clusters generally requires knowledge about cluster number and geometry.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '20202218',\n",
       "   'score': 0.1266900599002838},\n",
       "  {'start': 346,\n",
       "   'end': 545,\n",
       "   'text': 'RESULTS\\nWe integrated strategies from machine learning, cartography, and graph theory into a new informatics method for automatically clustering self-organizing map ensembles of high-dimensional data.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '20202218',\n",
       "   'score': 0.1627674102783203},\n",
       "  {'start': 547,\n",
       "   'end': 745,\n",
       "   'text': 'Our new method, called AutoSOME, readily identifies discrete and fuzzy data clusters without prior knowledge of cluster number or structure in diverse datasets including whole genome microarray data.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '20202218',\n",
       "   'score': 0.12472221255302429},\n",
       "  {'start': 747,\n",
       "   'end': 901,\n",
       "   'text': 'Visualization of AutoSOME output using network diagrams and differential heat maps reveals unexpected variation among well-characterized cancer cell lines.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '20202218',\n",
       "   'score': 0.14467346668243408},\n",
       "  {'start': 903,\n",
       "   'end': 1214,\n",
       "   'text': 'Co-expression analysis of data from human embryonic and induced pluripotent stem cells using AutoSOME identifies >3400 up-regulated genes associated with pluripotency, and indicates that a recently identified protein-protein interaction network characterizing pluripotency was underestimated by a factor of four.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '20202218',\n",
       "   'score': 0.39451032876968384},\n",
       "  {'start': 1218,\n",
       "   'end': 1462,\n",
       "   'text': 'CONCLUSIONS\\nBy effectively extracting important information from high-dimensional microarray data without prior knowledge or the need for data filtration, AutoSOME can yield systems-level insights from whole genome microarray expression studies.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '20202218',\n",
       "   'score': 0.15597054362297058},\n",
       "  {'start': 1464,\n",
       "   'end': 1636,\n",
       "   'text': 'Due to its generality, this new method should also have practical utility for a variety of data-intensive applications, including the results of deep sequencing experiments.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '20202218',\n",
       "   'score': 0.12257242202758789},\n",
       "  {'start': 1638,\n",
       "   'end': 1726,\n",
       "   'text': 'AutoSOME is available for download at http://jimcooperlab.mcdb.ucsb.edu/autosome webcite.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '20202218',\n",
       "   'score': 0.12189999222755432},\n",
       "  {'start': 0,\n",
       "   'end': 178,\n",
       "   'text': 'Glycogen synthase kinase-3 inhibition enhances translation of pluripotency-associated transcription factors to contribute to maintenance of mouse embryonic stem cell self-renewal.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '23577087',\n",
       "   'score': 0.3627505302429199},\n",
       "  {'start': 0,\n",
       "   'end': 167,\n",
       "   'text': 'Maintenance of embryonic stem cell (ESC) self-renewal and pluripotency are controlled by extrinsic factors, molecular signaling pathways and transcriptional regulators.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '23577087',\n",
       "   'score': 0.30400508642196655},\n",
       "  {'start': 169,\n",
       "   'end': 370,\n",
       "   'text': 'While many of the key players have been studied in depth, how the molecular signals interact with transcription factors of the pluripotency network to regulate their action remains less well understood.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '23577087',\n",
       "   'score': 0.3423035740852356},\n",
       "  {'start': 372,\n",
       "   'end': 675,\n",
       "   'text': 'Inhibition of glycogen synthase kinase 3 (Gsk-3) has been implicated in the maintenance of mouse ESC pluripotency, although there is contradictory data on its role, with enhancement of cell survival and metabolism, stabilisation of c-Myc and activation of Wnt signalling proposed as potential mechanisms.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '23577087',\n",
       "   'score': 0.3175240755081177},\n",
       "  {'start': 677,\n",
       "   'end': 856,\n",
       "   'text': 'We have discovered that suppression of Gsk-3 activity leads to enhanced protein levels of key transcriptional regulators of the pluripotency network, notably Nanog, Tbx3 and c-Myc.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '23577087',\n",
       "   'score': 0.36189576983451843},\n",
       "  {'start': 858,\n",
       "   'end': 1343,\n",
       "   'text': 'Protein stability was unchanged following Gsk-3 inhibition, although interestingly, Nanog and Tbx3 proteins were found to have half-lives of 1-3 h, while that of Oct4 protein was longer, at 6 h. We demonstrate that the effects on protein levels seen following inhibition of Gsk-3 are due to both enhanced de novo synthesis of Nanog protein and increases in the proportion of Nanog and Tbx3 RNAs bound to polysomes, findings consistent with Gsk-3 regulating translation of these factors.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '23577087',\n",
       "   'score': 0.3224031627178192},\n",
       "  {'start': 1345,\n",
       "   'end': 1499,\n",
       "   'text': \"These effects were not due to changes in regulators of general translation initiation machinery nor mediated via the 5' or 3' UTR sequences of Nanog alone.\",\n",
       "   'is_title': False,\n",
       "   'doc_id': '23577087',\n",
       "   'score': 0.3050016760826111},\n",
       "  {'start': 1501,\n",
       "   'end': 1764,\n",
       "   'text': 'The data we present provide both new conceptual insight into the mechanisms regulated by Gsk-3 that may contribute to ESC self-renewal and, importantly, establish control of protein translation as an additional mechanism involved in modulation of ESC pluripotency.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '23577087',\n",
       "   'score': 0.342555433511734},\n",
       "  {'start': 0,\n",
       "   'end': 109,\n",
       "   'text': 'An Optimized System for Effective Derivation of Three-Dimensional Retinal Tissue via Wnt Signaling Regulation.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '29999566',\n",
       "   'score': 0.1967775523662567},\n",
       "  {'start': 0,\n",
       "   'end': 217,\n",
       "   'text': 'Effective derivation of three-dimensional (3D) retinal tissue from human-induced pluripotent stem cells (hiPSCs) could provide models for drug screening and facilitate patient-specific retinal cell replacement therapy.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '29999566',\n",
       "   'score': 0.3850555419921875},\n",
       "  {'start': 219,\n",
       "   'end': 351,\n",
       "   'text': 'However, some hiPSC lines cannot undergo 3D self-organization and show inadequate differentiation efficiency to meet clinical demand.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '29999566',\n",
       "   'score': 0.21032124757766724},\n",
       "  {'start': 353,\n",
       "   'end': 436,\n",
       "   'text': 'In this study, we developed an optimized system for derivation of 3D retinal tissue.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '29999566',\n",
       "   'score': 0.1707819700241089},\n",
       "  {'start': 438,\n",
       "   'end': 600,\n",
       "   'text': 'We found that the Wnt signaling pathway antagonist Dickkopf-related protein 1 (DKK-1) rescued the inability of differentiated retinal progenitors to self-organize.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '29999566',\n",
       "   'score': 0.1292334496974945},\n",
       "  {'start': 602,\n",
       "   'end': 789,\n",
       "   'text': 'By evaluating DKK-1 expression and supplying DKK-1 if necessary, retinal organoids were differentiated from six hiPSC lines, which were reprogramed from three common initiating cell types.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '29999566',\n",
       "   'score': 0.1281784474849701},\n",
       "  {'start': 791,\n",
       "   'end': 908,\n",
       "   'text': 'Retinal tissues derived from the optimized system were well organized and capable of surviving for further maturation.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '29999566',\n",
       "   'score': 0.1246325671672821},\n",
       "  {'start': 910,\n",
       "   'end': 1009,\n",
       "   'text': 'Thus, using this system, we generated retinal tissues from various hiPSC lines with high efficiency.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '29999566',\n",
       "   'score': 0.1201859712600708},\n",
       "  {'start': 1011,\n",
       "   'end': 1107,\n",
       "   'text': 'This novel system has many potential applications in regenerative therapy and precision medicine.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '29999566',\n",
       "   'score': 0.12381336092948914},\n",
       "  {'start': 1109,\n",
       "   'end': 1137,\n",
       "   'text': 'Stem Cells 2018;36:1709-1722.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '29999566',\n",
       "   'score': 0.1201859712600708},\n",
       "  {'start': 0,\n",
       "   'end': 34,\n",
       "   'text': 'Nuclear architecture in stem cells.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '21222196',\n",
       "   'score': 0.12231820821762085},\n",
       "  {'start': 0,\n",
       "   'end': 203,\n",
       "   'text': 'Fundamental features of genome regulation depend on the linear DNA sequence, cell type specific modification of DNA and chromatin-associated proteins, which locally control the expression of single genes.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '21222196',\n",
       "   'score': 0.14852508902549744},\n",
       "  {'start': 205,\n",
       "   'end': 468,\n",
       "   'text': 'Architectural features of genome organization within the three-dimensional (3D) nuclear space establish preferential positioning of genes relative to nuclear subcompartments associated with specific biochemical activities, thereby influencing states of expression.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '21222196',\n",
       "   'score': 0.22650891542434692},\n",
       "  {'start': 470,\n",
       "   'end': 715,\n",
       "   'text': 'The structural and temporal organization of the genome within the nucleus of stem cells, together with specific features of epigenetic and transcriptional regulation are emerging as key players that influence pluripotency and differentiation.1,2.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '21222196',\n",
       "   'score': 0.38951271772384644},\n",
       "  {'start': 0,\n",
       "   'end': 44,\n",
       "   'text': 'Patz1 regulates embryonic stem cell identity.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '24380431',\n",
       "   'score': 0.1203262209892273},\n",
       "  {'start': 0,\n",
       "   'end': 97,\n",
       "   'text': 'Embryonic stem cells (ESCs) derived from the inner cell mass (ICM) of blastocysts are pluripotent.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24380431',\n",
       "   'score': 0.2773019075393677},\n",
       "  {'start': 99,\n",
       "   'end': 200,\n",
       "   'text': 'Pluripotency is maintained by a transcriptional network in which Oct4 and Nanog are master regulators.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24380431',\n",
       "   'score': 0.3662261962890625},\n",
       "  {'start': 202,\n",
       "   'end': 289,\n",
       "   'text': 'Notably, several zinc finger transcription factors have important roles in this network.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24380431',\n",
       "   'score': 0.14457696676254272},\n",
       "  {'start': 291,\n",
       "   'end': 417,\n",
       "   'text': 'Patz1, a BTB/POZ-domain-containing zinc finger protein, is expressed at higher levels in the ICM relative to the trophectoderm.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24380431',\n",
       "   'score': 0.13567835092544556},\n",
       "  {'start': 419,\n",
       "   'end': 480,\n",
       "   'text': 'However, its function in pluripotency has been poorly studied.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24380431',\n",
       "   'score': 0.2901040315628052},\n",
       "  {'start': 482,\n",
       "   'end': 556,\n",
       "   'text': 'Here, we show that Patz1 is an important regulator of pluripotency in ESCs.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24380431',\n",
       "   'score': 0.3411720395088196},\n",
       "  {'start': 558,\n",
       "   'end': 683,\n",
       "   'text': 'Patz1 RNAi, chromatin immunoprecipitation (ChIP), and reporter assays indicate that Patz1 directly regulates Pou5f1 and Nanog.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24380431',\n",
       "   'score': 0.2827426791191101},\n",
       "  {'start': 685,\n",
       "   'end': 845,\n",
       "   'text': 'Global transcriptome changes upon Patz1 knockdown largely involve upregulation of apoptotic genes and downregulation of cell cycle and cellular metabolism genes.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24380431',\n",
       "   'score': 0.12364223599433899},\n",
       "  {'start': 847,\n",
       "   'end': 988,\n",
       "   'text': 'Patz1 ChIP sequencing further identified more than 5,000 binding sites of Patz1 in mouse genome, from which two binding motifs were extracted.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24380431',\n",
       "   'score': 0.13018068671226501},\n",
       "  {'start': 990,\n",
       "   'end': 1121,\n",
       "   'text': 'Further, gene ontology analysis of genes associated with the binding sites displays enrichment for proximity to developmental genes.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24380431',\n",
       "   'score': 0.1267530918121338},\n",
       "  {'start': 1123,\n",
       "   'end': 1205,\n",
       "   'text': 'In addition, embryoid body assays suggest that Patz1 represses developmental genes.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24380431',\n",
       "   'score': 0.12269043922424316},\n",
       "  {'start': 1207,\n",
       "   'end': 1283,\n",
       "   'text': 'Together, these results propose that Patz1 is important for ESC pluripotency.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24380431',\n",
       "   'score': 0.2886764407157898},\n",
       "  {'start': 0,\n",
       "   'end': 77,\n",
       "   'text': 'Protein Kinases and Their Inhibitors in Pluripotent Stem Cell Fate Regulation.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '31428157',\n",
       "   'score': 0.32658523321151733},\n",
       "  {'start': 0,\n",
       "   'end': 193,\n",
       "   'text': 'Protein kinases modulate the reversible postmodifications of substrate proteins to their phosphorylated forms as an essential process in regulating intracellular signaling transduction cascades.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31428157',\n",
       "   'score': 0.1494547426700592},\n",
       "  {'start': 195,\n",
       "   'end': 480,\n",
       "   'text': 'Moreover, phosphorylation has recently been shown to tightly control the regulatory network of kinases responsible for the induction and maintenance of pluripotency, defined as the particular ability to differentiate pluripotent stem cells (PSCs) into every cell type in the adult body.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31428157',\n",
       "   'score': 0.3523107171058655},\n",
       "  {'start': 482,\n",
       "   'end': 670,\n",
       "   'text': 'In particular, emerging evidence indicates that the balance between the self-renewal and differentiation of PSCs is regulated by the small molecules that modulate kinase signaling pathways.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31428157',\n",
       "   'score': 0.13033080101013184},\n",
       "  {'start': 672,\n",
       "   'end': 923,\n",
       "   'text': 'Furthermore, new reprogramming technologies have been developed using kinase modulators, which have provided novel insight of the mechanisms underlying the kinase regulatory networks involved in the generation of induced pluripotent stem cells (iPSCs).',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31428157',\n",
       "   'score': 0.3503033518791199},\n",
       "  {'start': 925,\n",
       "   'end': 1184,\n",
       "   'text': 'In this review, we highlight the recent progress made in defining the roles of protein kinase signaling pathways and their small molecule modulators in regulating the pluripotent states, self-renewal, reprogramming process, and lineage differentiation of PSCs.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31428157',\n",
       "   'score': 0.3596958816051483},\n",
       "  {'start': 0,\n",
       "   'end': 79,\n",
       "   'text': 'The Transcription Factor-microRNA Regulatory Network during hESC-chondrogenesis.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '32179818',\n",
       "   'score': 0.14655354619026184},\n",
       "  {'start': 0,\n",
       "   'end': 96,\n",
       "   'text': 'Human embryonic stem cells (ESCs) offer a promising therapeutic approach for osteoarthritis (OA).',\n",
       "   'is_title': False,\n",
       "   'doc_id': '32179818',\n",
       "   'score': 0.27788612246513367},\n",
       "  {'start': 98,\n",
       "   'end': 263,\n",
       "   'text': 'The unlimited source of cells capable of differentiating to chondrocytes has potential for repairing damaged cartilage or to generate disease models via gene editing.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '32179818',\n",
       "   'score': 0.12423697113990784},\n",
       "  {'start': 265,\n",
       "   'end': 343,\n",
       "   'text': 'However their use is limited by the efficiency of chondrogenic differentiation.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '32179818',\n",
       "   'score': 0.12568259239196777},\n",
       "  {'start': 345,\n",
       "   'end': 517,\n",
       "   'text': 'An improved understanding of the transcriptional and post-transcriptional regulation of chondrogenesis will enable us to improve hESC chondrogenic differentiation protocols.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '32179818',\n",
       "   'score': 0.1457291543483734},\n",
       "  {'start': 519,\n",
       "   'end': 632,\n",
       "   'text': 'Small RNA-seq and whole transcriptome sequencing was performed on distinct stages of hESC-directed chondrogenesis.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '32179818',\n",
       "   'score': 0.12464895844459534},\n",
       "  {'start': 634,\n",
       "   'end': 875,\n",
       "   'text': 'This revealed significant changes in the expression of several microRNAs including upregulation of known cartilage associated microRNAs and those transcribed from the Hox complexes, and the downregulation of pluripotency associated microRNAs.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '32179818',\n",
       "   'score': 0.3660372197628021},\n",
       "  {'start': 877,\n",
       "   'end': 1151,\n",
       "   'text': 'Integration of miRomes and transcriptomes generated during hESC-directed chondrogenesis identified key functionally related clusters of co-expressed microRNAs and protein coding genes, associated with pluripotency, primitive streak, limb development and extracellular matrix.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '32179818',\n",
       "   'score': 0.32369381189346313},\n",
       "  {'start': 1153,\n",
       "   'end': 1474,\n",
       "   'text': \"Analysis identified regulators of hESC-directed chondrogenesis such as miR-29c-3p with 10 of its established targets identified as co-regulated 'ECM organisation' genes and miR-22-3p which is highly co-expressed with ECM genes and may regulate these genes indirectly by targeting the chondrogenic regulators SP1 and HDAC4.\",\n",
       "   'is_title': False,\n",
       "   'doc_id': '32179818',\n",
       "   'score': 0.16089516878128052},\n",
       "  {'start': 1476,\n",
       "   'end': 1662,\n",
       "   'text': 'We identified several upregulated transcription factors including HOXA9/A10/D13 involved in limb patterning and RELA, JUN and NFAT5, which have targets enriched with ECM associated genes.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '32179818',\n",
       "   'score': 0.1360066831111908},\n",
       "  {'start': 1664,\n",
       "   'end': 1913,\n",
       "   'text': 'We have developed an unbiased approach for integrating transcriptome and miRome using protein-protein interactions, transcription factor regulation and miRNA target interactions and identified key regulatory networks prominent in hESC chondrogenesis.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '32179818',\n",
       "   'score': 0.18140000104904175},\n",
       "  {'start': 0,\n",
       "   'end': 95,\n",
       "   'text': 'PRDM15 safeguards naive pluripotency by transcriptionally regulating WNT and MAPK-ERK signaling.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '28740264',\n",
       "   'score': 0.32720839977264404},\n",
       "  {'start': 0,\n",
       "   'end': 173,\n",
       "   'text': 'The transcriptional network acting downstream of LIF, WNT and MAPK-ERK to stabilize mouse embryonic stem cells (ESCs) in their naive state has been extensively characterized.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28740264',\n",
       "   'score': 0.3634990453720093},\n",
       "  {'start': 175,\n",
       "   'end': 271,\n",
       "   'text': 'However, the upstream factors regulating these three signaling pathways remain largely uncharted.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28740264',\n",
       "   'score': 0.14423120021820068},\n",
       "  {'start': 273,\n",
       "   'end': 437,\n",
       "   'text': 'PR-domain-containing proteins (PRDMs) are zinc-finger sequence-specific chromatin factors that have essential roles in embryonic development and cell fate decisions.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28740264',\n",
       "   'score': 0.12530022859573364},\n",
       "  {'start': 439,\n",
       "   'end': 574,\n",
       "   'text': 'Here we characterize the transcriptional regulator PRDM15, which acts independently of PRDM14 to regulate the naive state of mouse ESCs.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28740264',\n",
       "   'score': 0.2784096300601959},\n",
       "  {'start': 576,\n",
       "   'end': 716,\n",
       "   'text': 'Mechanistically, PRDM15 modulates WNT and MAPK-ERK signaling by directly promoting the expression of Rspo1 (R-spondin1) and Spry1 (Sprouty1).',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28740264',\n",
       "   'score': 0.1260775327682495},\n",
       "  {'start': 718,\n",
       "   'end': 966,\n",
       "   'text': 'Consistent with these findings, CRISPR-Cas9-mediated disruption of PRDM15-binding sites in the Rspo1 and Spry1 promoters recapitulates PRDM15 depletion, both in terms of local chromatin organization and the transcriptional modulation of these genes.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28740264',\n",
       "   'score': 0.16184517741203308},\n",
       "  {'start': 968,\n",
       "   'end': 1171,\n",
       "   'text': 'Collectively, our findings uncover an essential role for PRDM15 as a chromatin factor that modulates the transcription of upstream regulators of WNT and MAPK-ERK signaling to safeguard naive pluripotency.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28740264',\n",
       "   'score': 0.2829882502555847},\n",
       "  {'start': 0,\n",
       "   'end': 126,\n",
       "   'text': 'DNA Replication Inhibitor Geminin and Retinoic Acid Signaling Participate in Complex Interactions Associated With Pluripotency.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '31659113',\n",
       "   'score': 0.3197113871574402},\n",
       "  {'start': 0,\n",
       "   'end': 112,\n",
       "   'text': 'BACKGROUND/AIM\\nSeveral links between DNA replication, pluripotency and development have been recently identified.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31659113',\n",
       "   'score': 0.2944391965866089},\n",
       "  {'start': 114,\n",
       "   'end': 228,\n",
       "   'text': 'The involvement of miRNA in the regulation of cell cycle events and pluripotency factors has also gained attention.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31659113',\n",
       "   'score': 0.36008602380752563},\n",
       "  {'start': 232,\n",
       "   'end': 461,\n",
       "   'text': 'MATERIALS AND METHODS\\nIn the present study, we used the g:Profiler platform to analyze transcription factor binding sites, miRNA networks and protein-protein interactions to identify novel links among the aforementioned processes.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31659113',\n",
       "   'score': 0.16147297620773315},\n",
       "  {'start': 465,\n",
       "   'end': 653,\n",
       "   'text': 'RESULTS AND CONCLUSION\\nA complex circuitry between retinoic acid signaling, SWI/SNF components, pluripotency factors including Oct4, Sox2 and Nanog and cell cycle regulators was identified.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31659113',\n",
       "   'score': 0.2799634337425232},\n",
       "  {'start': 655,\n",
       "   'end': 752,\n",
       "   'text': 'It is suggested that the DNA replication inhibitor geminin plays a central role in this circuitry.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31659113',\n",
       "   'score': 0.12780997157096863},\n",
       "  {'start': 0,\n",
       "   'end': 59,\n",
       "   'text': 'Nanog, Oct4 and Tet1 interplay in establishing pluripotency.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '27146218',\n",
       "   'score': 0.3121066093444824},\n",
       "  {'start': 0,\n",
       "   'end': 163,\n",
       "   'text': \"A few central transcription factors inside mouse embryonic stem (ES) cells and induced pluripotent stem (iPS) cells are believed to control the cells' pluripotency.\",\n",
       "   'is_title': False,\n",
       "   'doc_id': '27146218',\n",
       "   'score': 0.2721973657608032},\n",
       "  {'start': 165,\n",
       "   'end': 271,\n",
       "   'text': 'Characterizations of pluripotent state were put forward on both transcription factor and epigenetic levels.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '27146218',\n",
       "   'score': 0.3134813606739044},\n",
       "  {'start': 273,\n",
       "   'end': 460,\n",
       "   'text': 'Whereas core players have been identified, it is desirable to map out gene regulatory networks which govern the reprogramming of somatic cells as well as the early developmental decisions.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '27146218',\n",
       "   'score': 0.15231949090957642},\n",
       "  {'start': 462,\n",
       "   'end': 652,\n",
       "   'text': 'Here we propose a multiple level model where the regulatory network of Oct4, Nanog and Tet1 includes positive feedback loops involving DNA-demethylation around the promoters of Oct4 and Tet1.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '27146218',\n",
       "   'score': 0.35640037059783936},\n",
       "  {'start': 654,\n",
       "   'end': 1168,\n",
       "   'text': 'We put forward a mechanistic understanding of the regulatory dynamics which account for i) Oct4 overexpression is sufficient to induce pluripotency in somatic cell types expressing the other Yamanaka reprogramming factors endogenously; ii) Tet1 can replace Oct4 in reprogramming cocktail; iii) Nanog is not necessary for reprogramming however its over-expression leads to enhanced self-renewal; iv) DNA methylation is the key to the regulation of pluripotency genes; v) Lif withdrawal leads to loss of pluripotency.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '27146218',\n",
       "   'score': 0.35113525390625},\n",
       "  {'start': 1170,\n",
       "   'end': 1416,\n",
       "   'text': 'Overall, our paper proposes a novel framework combining transcription regulation with DNA methylation modifications which, takes into account the multi-layer nature of regulatory mechanisms governing pluripotency acquisition through reprogramming.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '27146218',\n",
       "   'score': 0.31794777512550354},\n",
       "  {'start': 0,\n",
       "   'end': 88,\n",
       "   'text': 'Protein kinases and associated pathways in pluripotent state and lineage differentiation.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '24998240',\n",
       "   'score': 0.31817471981048584},\n",
       "  {'start': 0,\n",
       "   'end': 177,\n",
       "   'text': 'Protein kinases (PKs) mediate the reversible conversion of substrate proteins to phosphorylated forms, a key process in controlling intracellular signaling transduction cascades.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24998240',\n",
       "   'score': 0.13507691025733948},\n",
       "  {'start': 179,\n",
       "   'end': 358,\n",
       "   'text': 'Pluripotency is, among others, characterized by specifically expressed PKs forming a highly interconnected regulatory network that culminates in a finely-balanced molecular switch.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24998240',\n",
       "   'score': 0.35417917370796204},\n",
       "  {'start': 360,\n",
       "   'end': 511,\n",
       "   'text': 'Current high-throughput phosphoproteomic approaches have shed light on the specific regulatory PKs and their function in controlling pluripotent states.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24998240',\n",
       "   'score': 0.2677290439605713},\n",
       "  {'start': 513,\n",
       "   'end': 681,\n",
       "   'text': 'Pluripotent cell-derived endothelial and hematopoietic developments represent an example of the importance of pluripotency in cancer therapeutics and organ regeneration.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24998240',\n",
       "   'score': 0.33453667163848877},\n",
       "  {'start': 683,\n",
       "   'end': 834,\n",
       "   'text': 'This review attempts to provide the hitherto known kinome profile and the individual characterization of PK-related pathways that regulate pluripotency.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24998240',\n",
       "   'score': 0.2740197777748108},\n",
       "  {'start': 836,\n",
       "   'end': 1213,\n",
       "   'text': 'Elucidating the underlying intrinsic and extrinsic signals may improve our understanding of the different pluripotent states, the maintenance or induction of pluripotency, and the ability to tailor lineage differentiation, with a particular focus on endothelial cell differentiation for anti-cancer treatment, cell-based tissue engineering, and regenerative medicine strategies.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24998240',\n",
       "   'score': 0.31702467799186707},\n",
       "  {'start': 0,\n",
       "   'end': 57,\n",
       "   'text': 'Mechanosensitive mechanisms in transcriptional regulation.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '22797927',\n",
       "   'score': 0.14120548963546753},\n",
       "  {'start': 0,\n",
       "   'end': 144,\n",
       "   'text': 'Transcriptional regulation contributes to the maintenance of pluripotency, self-renewal and differentiation in embryonic cells and in stem cells.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '22797927',\n",
       "   'score': 0.3630203902721405},\n",
       "  {'start': 146,\n",
       "   'end': 349,\n",
       "   'text': 'Therefore, control of gene expression at the level of transcription is crucial for embryonic development, as well as for organogenesis, functional adaptation, and regeneration in adult tissues and organs.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '22797927',\n",
       "   'score': 0.13322484493255615},\n",
       "  {'start': 351,\n",
       "   'end': 560,\n",
       "   'text': 'In the past, most work has focused on how transcriptional regulation results from the complex interplay between chemical cues, adhesion signals, transcription factors and their co-regulators during development.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '22797927',\n",
       "   'score': 0.14933377504348755},\n",
       "  {'start': 562,\n",
       "   'end': 766,\n",
       "   'text': 'However, chemical signaling alone is not sufficient to explain how three-dimensional (3D) tissues and organs are constructed and maintained through the spatiotemporal control of transcriptional activities.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '22797927',\n",
       "   'score': 0.1849413812160492},\n",
       "  {'start': 768,\n",
       "   'end': 855,\n",
       "   'text': 'Accumulated evidence indicates that mechanical cues, which include physical forces (e.g.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '22797927',\n",
       "   'score': 0.12499180436134338},\n",
       "  {'start': 857,\n",
       "   'end': 1123,\n",
       "   'text': 'tension, compression or shear stress), alterations in extracellular matrix (ECM) mechanics and changes in cell shape, are transmitted to the nucleus directly or indirectly to orchestrate transcriptional activities that are crucial for embryogenesis and organogenesis.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '22797927',\n",
       "   'score': 0.12888354063034058},\n",
       "  {'start': 1125,\n",
       "   'end': 1429,\n",
       "   'text': 'In this Commentary, we review how the mechanical control of gene transcription contributes to the maintenance of pluripotency, determination of cell fate, pattern formation and organogenesis, as well as how it is involved in the control of cell and tissue function throughout embryogenesis and adult life.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '22797927',\n",
       "   'score': 0.35613518953323364},\n",
       "  {'start': 1431,\n",
       "   'end': 1592,\n",
       "   'text': 'A deeper understanding of these mechanosensitive transcriptional control mechanisms should lead to new approaches to tissue engineering and regenerative medicine.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '22797927',\n",
       "   'score': 0.1266523003578186},\n",
       "  {'start': 0,\n",
       "   'end': 59,\n",
       "   'text': 'Transcriptional regulatory networks in embryonic stem cells.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '21141733',\n",
       "   'score': 0.1519191861152649},\n",
       "  {'start': 0,\n",
       "   'end': 172,\n",
       "   'text': 'Transcriptional regulation is one of the most fundamental processes in biology, governing the morphology, function, and behavior of cells and thus the survival of organisms.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '21141733',\n",
       "   'score': 0.14974936842918396},\n",
       "  {'start': 174,\n",
       "   'end': 299,\n",
       "   'text': 'The embryonic stem cell (ESC) provides a good model for the understanding of transcriptional regulation in vertebrate systems.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '21141733',\n",
       "   'score': 0.14885759353637695},\n",
       "  {'start': 301,\n",
       "   'end': 452,\n",
       "   'text': 'Recent efforts have led to the identification of molecular events, which confer upon these cells the unique properties of pluripotency and self renewal.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '21141733',\n",
       "   'score': 0.32400214672088623},\n",
       "  {'start': 454,\n",
       "   'end': 566,\n",
       "   'text': 'The core regulatory network maintaining the ESC identity involves three master regulators: Oct4, Sox2, and Nanog.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '21141733',\n",
       "   'score': 0.3651459217071533},\n",
       "  {'start': 568,\n",
       "   'end': 730,\n",
       "   'text': 'Large-scale mapping studies interrogating the binding sites of these and other transcription factors showed co-occupancy of distinct sets of transcription factors.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '21141733',\n",
       "   'score': 0.1291850209236145},\n",
       "  {'start': 732,\n",
       "   'end': 879,\n",
       "   'text': 'The assembly of multitranscription factor complexes could serve as a mechanism for providing specificity in regulating ESC-specific gene expression.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '21141733',\n",
       "   'score': 0.14708253741264343},\n",
       "  {'start': 881,\n",
       "   'end': 993,\n",
       "   'text': 'These studies are also beginning to unravel the transcriptional regulatory networks that govern the ESC identity.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '21141733',\n",
       "   'score': 0.14661765098571777},\n",
       "  {'start': 995,\n",
       "   'end': 1119,\n",
       "   'text': 'Loss-of-function RNAi screens also identified novel regulatory molecules involved in the stable propagation of the ESC state.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '21141733',\n",
       "   'score': 0.1355399191379547},\n",
       "  {'start': 1121,\n",
       "   'end': 1403,\n",
       "   'text': 'This argues for an ESC transcriptional regulation program in which interconnected transcriptional regulatory networks involving large numbers of transcription factors and epigenetic modifiers work in concert on ESC- and differentiation-specific genes to achieve cell state stability.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '21141733',\n",
       "   'score': 0.17287743091583252},\n",
       "  {'start': 1405,\n",
       "   'end': 1606,\n",
       "   'text': 'This chapter traces the major efforts made over the past decade in dissecting the transcriptional regulatory network governing ESC identity and offers perspectives on the future directions of the field.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '21141733',\n",
       "   'score': 0.1503174901008606},\n",
       "  {'start': 0,\n",
       "   'end': 71,\n",
       "   'text': 'Dynamic lineage priming is driven via direct enhancer regulation by ERK.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '31695196',\n",
       "   'score': 0.22688543796539307},\n",
       "  {'start': 0,\n",
       "   'end': 185,\n",
       "   'text': 'Central to understanding cellular behaviour in multi-cellular organisms is the question of how a cell exits one transcriptional state to adopt and eventually become committed to another.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31695196',\n",
       "   'score': 0.13288241624832153},\n",
       "  {'start': 187,\n",
       "   'end': 442,\n",
       "   'text': 'Fibroblast growth factor-extracellular signal-regulated kinase (FGF -ERK) signalling drives differentiation of mouse embryonic stem cells (ES cells) and pre-implantation embryos towards primitive endoderm, and inhibiting ERK supports ES cell self-renewal1.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31695196',\n",
       "   'score': 0.12736180424690247},\n",
       "  {'start': 444,\n",
       "   'end': 631,\n",
       "   'text': 'Paracrine FGF-ERK signalling induces heterogeneity, whereby cells reversibly progress from pluripotency towards primitive endoderm while retaining their capacity to re-enter self-renewal2.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31695196',\n",
       "   'score': 0.308447927236557},\n",
       "  {'start': 633,\n",
       "   'end': 800,\n",
       "   'text': 'Here we find that ERK reversibly regulates transcription in ES cells by directly affecting enhancer activity without requiring a change in transcription factor binding.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31695196',\n",
       "   'score': 0.19502729177474976},\n",
       "  {'start': 802,\n",
       "   'end': 1035,\n",
       "   'text': 'ERK triggers the reversible association and disassociation of RNA polymerase II and associated co-factors from genes and enhancers with the mediator component MED24 having an essential role in ERK-dependent transcriptional regulation.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31695196',\n",
       "   'score': 0.22788551449775696},\n",
       "  {'start': 1037,\n",
       "   'end': 1284,\n",
       "   'text': 'Though the binding of mediator components responds directly to signalling, the persistent binding of pluripotency factors to both induced and repressed genes marks them for activation and/or reactivation in response to fluctuations in ERK activity.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31695196',\n",
       "   'score': 0.3259464502334595},\n",
       "  {'start': 1286,\n",
       "   'end': 1519,\n",
       "   'text': 'Among the repressed genes are several core components of the pluripotency network that act to drive their own expression and maintain the ES cell state; if their binding is lost, the ability to reactivate transcription is compromised.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31695196',\n",
       "   'score': 0.3520236015319824},\n",
       "  {'start': 1521,\n",
       "   'end': 1671,\n",
       "   'text': 'Thus, as long as transcription factor occupancy is maintained, so is plasticity, enabling cells to distinguish between transient and sustained signals.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31695196',\n",
       "   'score': 0.1280554234981537},\n",
       "  {'start': 1673,\n",
       "   'end': 1830,\n",
       "   'text': 'If ERK signalling persists, pluripotency transcription factor levels are reduced by protein turnover and irreversible gene silencing and commitment can occur.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31695196',\n",
       "   'score': 0.3071673512458801},\n",
       "  {'start': 0,\n",
       "   'end': 47,\n",
       "   'text': 'Taming human brain organoids one cell at a time.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '32718852',\n",
       "   'score': 0.12541189789772034},\n",
       "  {'start': 0,\n",
       "   'end': 210,\n",
       "   'text': 'Human brain organoids are self-organizing three-dimensional structures that emerge from human pluripotent stem cells and mimic aspects of the cellular composition and functionality of the developing human brain.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '32718852',\n",
       "   'score': 0.37641382217407227},\n",
       "  {'start': 212,\n",
       "   'end': 467,\n",
       "   'text': 'Despite their impressive self-organizing capacity, organoids lack the stereotypic structural anatomy of their in vivo counterpart, making conventional analysis techniques underpowered to assess cellular composition and gene network regulation in organoids.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '32718852',\n",
       "   'score': 0.17145931720733643},\n",
       "  {'start': 469,\n",
       "   'end': 705,\n",
       "   'text': 'Advances in single cell transcriptomics have recently allowed characterization and improvement of organoid protocols, as they continue to evolve, by enabling identification of cell types and states along with their developmental origins.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '32718852',\n",
       "   'score': 0.1307586431503296},\n",
       "  {'start': 707,\n",
       "   'end': 857,\n",
       "   'text': \"In this review, we summarize recent approaches, progresses and challenges in resolving brain organoid's complexity through single-cell transcriptomics.\",\n",
       "   'is_title': False,\n",
       "   'doc_id': '32718852',\n",
       "   'score': 0.12988168001174927},\n",
       "  {'start': 859,\n",
       "   'end': 1054,\n",
       "   'text': 'We then discuss emerging technologies that may complement single-cell RNA sequencing by providing additional readouts of cellular states to generate an organ-level view of developmental processes.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '32718852',\n",
       "   'score': 0.1264784336090088},\n",
       "  {'start': 1056,\n",
       "   'end': 1352,\n",
       "   'text': 'Altogether, these integrative technologies will allow monitoring of global gene regulation in thousands of individual cells and will offer an unprecedented opportunity to investigate features of human brain development and disease across multiple cellular modalities and with cell-type resolution.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '32718852',\n",
       "   'score': 0.14569297432899475},\n",
       "  {'start': 0,\n",
       "   'end': 115,\n",
       "   'text': 'Assembly of FN-silk with laminin-521 to integrate hPSCs into a three-dimensional culture for neural differentiation.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '32215392',\n",
       "   'score': 0.18806323409080505},\n",
       "  {'start': 0,\n",
       "   'end': 185,\n",
       "   'text': 'Three-dimensional (3D) neural tissue cultures recapitulate the basic concepts during development and disease better than what can be obtained using conventional two-dimensional cultures.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '32215392',\n",
       "   'score': 0.1900118589401245},\n",
       "  {'start': 187,\n",
       "   'end': 410,\n",
       "   'text': 'Here, we use a recombinant spider silk protein functionalized with a cell binding motif from fibronectin (FN-silk) in combination with a human recombinant laminin 521 (LN-521) to create a fully defined stem cell niche in 3D.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '32215392',\n",
       "   'score': 0.17017602920532227},\n",
       "  {'start': 412,\n",
       "   'end': 648,\n",
       "   'text': 'A novel method to assemble silk blended with LN-521 together with human pluripotent stem cells (hPSC) is used to create centimeter-sized foams, which upon cultivation develop into 3D cell constructs supported by a microfibrillar network.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '32215392',\n",
       "   'score': 0.36491045355796814},\n",
       "  {'start': 650,\n",
       "   'end': 837,\n",
       "   'text': 'After initial cell expansion, neural differentiation was induced to form a homogenous layer of continuous neuroectodermal tissue that allows further differentiation into neuronal subtypes.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '32215392',\n",
       "   'score': 0.12646859884262085},\n",
       "  {'start': 839,\n",
       "   'end': 1047,\n",
       "   'text': 'The silk-supported 3D cell constructs could then be detached from the bottom of the well and cultured as floating entities, where cells appeared in distinctive radial organization resembling early neural tube.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '32215392',\n",
       "   'score': 0.2044319212436676},\n",
       "  {'start': 1049,\n",
       "   'end': 1178,\n",
       "   'text': 'This shows that the neural progenitors retain their cellular self-organization ability in the FN-silk/LN-521-supported 3D culture.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '32215392',\n",
       "   'score': 0.20968371629714966},\n",
       "  {'start': 1180,\n",
       "   'end': 1288,\n",
       "   'text': 'Calcium imaging demonstrated spontaneous activity, which is important for the formation of neuronal networks.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '32215392',\n",
       "   'score': 0.1544681191444397},\n",
       "  {'start': 1290,\n",
       "   'end': 1456,\n",
       "   'text': 'Together, the results show that hPSCs integrated into FN-silk/LN-521 foam develop into neural progenitors and that these stay viable during long-term differentiations.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '32215392',\n",
       "   'score': 0.1290324330329895},\n",
       "  {'start': 1458,\n",
       "   'end': 1606,\n",
       "   'text': 'FN-silk/LN-521 also supports morphogenesis mimicking the human brain development and can serve as base for engineering of hPSC-derived neural tissue.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '32215392',\n",
       "   'score': 0.13067004084587097},\n",
       "  {'start': 0,\n",
       "   'end': 90,\n",
       "   'text': 'Proteomic and network analysis of proteins regulated by REX1 in human embryonic stem cells.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '25736782',\n",
       "   'score': 0.15493479371070862},\n",
       "  {'start': 0,\n",
       "   'end': 138,\n",
       "   'text': 'Recent studies have suggested that REX1 (reduced expression 1) plays an important role in pluripotency, proliferation, and differentiation.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '25736782',\n",
       "   'score': 0.29200220108032227},\n",
       "  {'start': 140,\n",
       "   'end': 256,\n",
       "   'text': 'However, the molecular mechanisms involved in REX1-dependent regulation of diverse cellular processes remain unclear.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '25736782',\n",
       "   'score': 0.15819910168647766},\n",
       "  {'start': 258,\n",
       "   'end': 428,\n",
       "   'text': 'To elucidate the regulatory functions of REX1 in human embryonic stem cells (hESCs), comparative proteomic analysis was performed on REX1 RNAi specifically silenced hESCs.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '25736782',\n",
       "   'score': 0.3034142851829529},\n",
       "  {'start': 430,\n",
       "   'end': 744,\n",
       "   'text': 'Analysis of the proteome via nano-LC-MS/MS identified 140 differentially expressed proteins (DEPs) displaying a >2-fold difference in expression level between control and REX1 knockdown (KD) hESCs, which were then compared with transcriptome data and validated by quantitative real-time RT-PCR and Western blotting.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '25736782',\n",
       "   'score': 0.27733102440834045},\n",
       "  {'start': 746,\n",
       "   'end': 905,\n",
       "   'text': 'These DEPs were analyzed by GO, pathway, and functional clustering analyses to determine the molecular functions of the proteins and pathways regulated by REX1.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '25736782',\n",
       "   'score': 0.13070592284202576},\n",
       "  {'start': 907,\n",
       "   'end': 1056,\n",
       "   'text': 'The REX1 KD-mediated DEPs mapped to major biological processes involved in the regulation of ribosome-mediated translation and mitochondrial function.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '25736782',\n",
       "   'score': 0.16216900944709778},\n",
       "  {'start': 1058,\n",
       "   'end': 1283,\n",
       "   'text': 'Functional network analysis revealed a highly interconnected network among these DEPs and indicated that these interconnected proteins are predominantly involved in translation and the regulation of mitochondrial organization.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '25736782',\n",
       "   'score': 0.2326642870903015},\n",
       "  {'start': 1285,\n",
       "   'end': 1527,\n",
       "   'text': 'These findings regarding REX1-mediated regulatory network have revealed the contributions of REX1 to maintaining the status of hESCs and have improved our understanding of the molecular events that underlie the fundamental properties of hESCs.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '25736782',\n",
       "   'score': 0.34918534755706787},\n",
       "  {'start': 0,\n",
       "   'end': 104,\n",
       "   'text': 'The p53 Family Coordinates Wnt and Nodal Inputs in Mesendodermal Differentiation of Embryonic Stem Cells.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '27889317',\n",
       "   'score': 0.13433071970939636},\n",
       "  {'start': 0,\n",
       "   'end': 224,\n",
       "   'text': 'In this study, we outline a regulatory network that involves the p53 tumor suppressor family and the Wnt pathway acting together with the TGF-β pathway in mesendodermal differentiation of mouse and human embryonic stem cells.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '27889317',\n",
       "   'score': 0.15927597880363464},\n",
       "  {'start': 226,\n",
       "   'end': 400,\n",
       "   'text': 'Knockout of all three members, p53, p63, and p73, shows that the p53 family is essential for mesendoderm specification during exit from pluripotency in embryos and in culture.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '27889317',\n",
       "   'score': 0.3394626975059509},\n",
       "  {'start': 402,\n",
       "   'end': 584,\n",
       "   'text': 'Wnt3 and its receptor Fzd1 are direct p53 family target genes in this context, and induction of Wnt\\xa0signaling by p53 is critical for activation of\\xa0mesendodermal differentiation genes.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '27889317',\n",
       "   'score': 0.1339360773563385},\n",
       "  {'start': 586,\n",
       "   'end': 764,\n",
       "   'text': 'Globally, Wnt3-activated Tcf3 and nodal-activated Smad2/3 transcription factors depend on each other for co-occupancy of target enhancers associated with key differentiation loci.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '27889317',\n",
       "   'score': 0.21422529220581055},\n",
       "  {'start': 766,\n",
       "   'end': 1021,\n",
       "   'text': 'Our results therefore highlight an unanticipated role for p53 family proteins in a regulatory network that integrates essential Wnt-Tcf and nodal-Smad inputs in a selective and interdependent way to drive mesendodermal differentiation of pluripotent cells.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '27889317',\n",
       "   'score': 0.3530239760875702},\n",
       "  {'start': 0,\n",
       "   'end': 88,\n",
       "   'text': 'On human pluripotent stem cell control: The rise of 3D bioengineering and mechanobiology.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '25818411',\n",
       "   'score': 0.35448741912841797},\n",
       "  {'start': 0,\n",
       "   'end': 147,\n",
       "   'text': 'Human pluripotent stem cells (hPSCs) provide promising resources for regenerating tissues and organs and modeling development and diseases in vitro.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '25818411',\n",
       "   'score': 0.27423030138015747},\n",
       "  {'start': 149,\n",
       "   'end': 363,\n",
       "   'text': 'To fulfill their promise, the fate, function, and organization of hPSCs need to be precisely regulated in a three-dimensional (3D) environment to mimic cellular structures and functions of native tissues and organs.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '25818411',\n",
       "   'score': 0.23433026671409607},\n",
       "  {'start': 365,\n",
       "   'end': 527,\n",
       "   'text': 'In the past decade, innovations in 3D culture systems with functional biomaterials have enabled efficient and versatile control of hPSC fate at the cellular level.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '25818411',\n",
       "   'score': 0.18543899059295654},\n",
       "  {'start': 529,\n",
       "   'end': 670,\n",
       "   'text': 'However, we are just at the beginning of bringing hPSC-based regeneration and development and disease modeling to the tissue and organ levels.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '25818411',\n",
       "   'score': 0.13100382685661316},\n",
       "  {'start': 672,\n",
       "   'end': 886,\n",
       "   'text': 'In this review, we summarize existing bioengineered culture platforms for controlling hPSC fate and function by regulating inductive mechanical and biochemical cues coexisting in the synthetic cell microenvironment.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '25818411',\n",
       "   'score': 0.151963472366333},\n",
       "  {'start': 888,\n",
       "   'end': 1047,\n",
       "   'text': 'We highlight recent excitements in developing 3D hPSC-based in vitro tissue and organ models with in vivo-like cellular structures, interactions, and functions.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '25818411',\n",
       "   'score': 0.17055419087409973},\n",
       "  {'start': 1049,\n",
       "   'end': 1268,\n",
       "   'text': 'We further discuss an emerging multifaceted mechanotransductive signaling network--with transcriptional coactivators YAP and TAZ at the center stage--that regulate fates and behaviors of mammalian cells, including hPSCs.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '25818411',\n",
       "   'score': 0.15300211310386658},\n",
       "  {'start': 1270,\n",
       "   'end': 1520,\n",
       "   'text': 'Future development of 3D biomaterial systems should incorporate dynamically modulated mechanical and chemical properties targeting specific intracellular signaling events leading to desirable hPSC fate patterning and functional tissue formation in 3D.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '25818411',\n",
       "   'score': 0.17026853561401367},\n",
       "  {'start': 0,\n",
       "   'end': 61,\n",
       "   'text': 'Navigating the epigenetic landscape of pluripotent stem cells.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '22820889',\n",
       "   'score': 0.3059931993484497},\n",
       "  {'start': 0,\n",
       "   'end': 236,\n",
       "   'text': 'Pluripotent stem cells, which include embryonic stem cells and induced pluripotent stem cells, use a complex network of genetic and epigenetic pathways to maintain a delicate balance between self-renewal and multilineage differentiation.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '22820889',\n",
       "   'score': 0.3193248212337494},\n",
       "  {'start': 238,\n",
       "   'end': 366,\n",
       "   'text': 'Recently developed high-throughput genomic tools greatly facilitate the study of epigenetic regulation in pluripotent stem cells.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '22820889',\n",
       "   'score': 0.34369325637817383},\n",
       "  {'start': 368,\n",
       "   'end': 501,\n",
       "   'text': 'Increasing evidence suggests the existence of extensive crosstalk among epigenetic pathways that modify DNA, histones and nucleosomes.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '22820889',\n",
       "   'score': 0.13160884380340576},\n",
       "  {'start': 503,\n",
       "   'end': 818,\n",
       "   'text': 'Novel methods of mapping higher-order chromatin structure and chromatin-nuclear matrix interactions also provide the first insight into the three-dimensional organization of the genome and a framework in which existing genomic data of epigenetic regulation can be integrated to discover new rules of gene regulation.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '22820889',\n",
       "   'score': 0.2500002384185791},\n",
       "  {'start': 0,\n",
       "   'end': 94,\n",
       "   'text': 'Cap-independent translation by DAP5 controls cell fate decisions in human embryonic stem cells.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '27664238',\n",
       "   'score': 0.12957125902175903},\n",
       "  {'start': 0,\n",
       "   'end': 100,\n",
       "   'text': 'Multiple transcriptional and epigenetic changes drive differentiation of embryonic stem cells (ESCs).',\n",
       "   'is_title': False,\n",
       "   'doc_id': '27664238',\n",
       "   'score': 0.2989666163921356},\n",
       "  {'start': 102,\n",
       "   'end': 249,\n",
       "   'text': 'This study unveils an additional level of gene expression regulation involving noncanonical, cap-independent translation of a select group of mRNAs.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '27664238',\n",
       "   'score': 0.14663276076316833},\n",
       "  {'start': 251,\n",
       "   'end': 384,\n",
       "   'text': 'This is driven by death-associated protein 5 (DAP5/eIF4G2/NAT1), a translation initiation factor mediating IRES-dependent translation.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '27664238',\n",
       "   'score': 0.14073753356933594},\n",
       "  {'start': 386,\n",
       "   'end': 618,\n",
       "   'text': 'We found that the DAP5 knockdown from human ESCs (hESCs) resulted in persistence of pluripotent gene expression, delayed induction of differentiation-associated genes in different cell lineages, and defective embryoid body formation.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '27664238',\n",
       "   'score': 0.34000664949417114},\n",
       "  {'start': 620,\n",
       "   'end': 727,\n",
       "   'text': 'The latter involved improper cellular organization, lack of cavitation, and enhanced mislocalized apoptosis.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '27664238',\n",
       "   'score': 0.16661763191223145},\n",
       "  {'start': 729,\n",
       "   'end': 853,\n",
       "   'text': 'RNA sequencing of polysome-associated mRNAs identified candidates with reduced translation efficiency in DAP5-depleted hESCs.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '27664238',\n",
       "   'score': 0.3179702162742615},\n",
       "  {'start': 855,\n",
       "   'end': 1126,\n",
       "   'text': 'These were enriched in mitochondrial proteins involved in oxidative respiration, a pathway essential for differentiation, the significance of which was confirmed by the aberrant mitochondrial morphology and decreased oxidative respiratory activity in DAP5 knockdown cells.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '27664238',\n",
       "   'score': 0.14124780893325806},\n",
       "  {'start': 1128,\n",
       "   'end': 1283,\n",
       "   'text': 'Further analysis identified the chromatin modifier HMGN3 as a cap-independent DAP5 translation target whose knockdown resulted in defective differentiation.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '27664238',\n",
       "   'score': 0.13116124272346497},\n",
       "  {'start': 1285,\n",
       "   'end': 1502,\n",
       "   'text': 'Thus, DAP5-mediated translation of a specific set of proteins is critical for the transition from pluripotency to differentiation, highlighting the importance of cap-independent translation in stem cell fate decisions.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '27664238',\n",
       "   'score': 0.3211459517478943},\n",
       "  {'start': 0,\n",
       "   'end': 107,\n",
       "   'text': 'The high mobility group protein HMGA2: a co-regulator of chromatin structure and pluripotency in stem cells?',\n",
       "   'is_title': True,\n",
       "   'doc_id': '19551524',\n",
       "   'score': 0.34748154878616333},\n",
       "  {'start': 0,\n",
       "   'end': 162,\n",
       "   'text': 'The small, chromatin-associated HMGA proteins contain three separate DNA binding domains, so-called AT hooks, which bind preferentially to short AT-rich sequences.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '19551524',\n",
       "   'score': 0.14240971207618713},\n",
       "  {'start': 164,\n",
       "   'end': 312,\n",
       "   'text': 'These proteins are abundant in pluripotent embryonic stem (ES) cells and most malignant human tumors, but are not detectable in normal somatic cells.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '19551524',\n",
       "   'score': 0.28186798095703125},\n",
       "  {'start': 314,\n",
       "   'end': 513,\n",
       "   'text': 'They act both as activator and repressor of gene expression, and most likely facilitate DNA architectural changes during formation of specialized nucleoprotein structures at selected promoter regions.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '19551524',\n",
       "   'score': 0.20867693424224854},\n",
       "  {'start': 515,\n",
       "   'end': 704,\n",
       "   'text': 'For example, HMGA2 is involved in transcriptional activation of certain cell proliferation genes, which likely contributes to its well-established oncogenic potential during tumor formation.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '19551524',\n",
       "   'score': 0.14384377002716064},\n",
       "  {'start': 706,\n",
       "   'end': 861,\n",
       "   'text': 'However, surprisingly little is known about how HMGA proteins bind DNA packaged in chromatin and how this affects the chromatin structure at a larger scale.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '19551524',\n",
       "   'score': 0.13657796382904053},\n",
       "  {'start': 863,\n",
       "   'end': 963,\n",
       "   'text': 'Experimental evidence suggests that HMGA2 competes with binding of histone H1 in the chromatin fiber.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '19551524',\n",
       "   'score': 0.1348886489868164},\n",
       "  {'start': 965,\n",
       "   'end': 1102,\n",
       "   'text': 'This could substantially alter chromatin domain structures in ES cells and contribute to the activation of certain transcription networks.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '19551524',\n",
       "   'score': 0.16274115443229675},\n",
       "  {'start': 1104,\n",
       "   'end': 1220,\n",
       "   'text': 'HMGA2 also seems capable of recruiting enzymes directly involved in histone modifications to trigger gene expression.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '19551524',\n",
       "   'score': 0.1381511688232422},\n",
       "  {'start': 1222,\n",
       "   'end': 1351,\n",
       "   'text': 'Furthermore, it was shown that multiple HMGA2 molecules bind stably to a single nucleosome core particle whose structure is known.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '19551524',\n",
       "   'score': 0.13096222281455994},\n",
       "  {'start': 1353,\n",
       "   'end': 1446,\n",
       "   'text': 'How these features of HMGA2 impinge on chromatin organization inside a living cell is unknown.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '19551524',\n",
       "   'score': 0.16340762376785278},\n",
       "  {'start': 1448,\n",
       "   'end': 1685,\n",
       "   'text': 'In this commentary, we propose that HMGA2, through the action of three independent DNA binding domains, substantially contributes to the plasticity of ES cell chromatin and is involved in the maintenance of a un-differentiated cell state.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '19551524',\n",
       "   'score': 0.1536206305027008},\n",
       "  {'start': 0,\n",
       "   'end': 68,\n",
       "   'text': 'Metabolic remodeling during the loss and acquisition of pluripotency.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '28196802',\n",
       "   'score': 0.3320280909538269},\n",
       "  {'start': 0,\n",
       "   'end': 183,\n",
       "   'text': 'Pluripotent cells from the early stages of embryonic development have the unlimited capacity to self-renew and undergo differentiation into all of the cell types of the adult organism.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28196802',\n",
       "   'score': 0.30469852685928345},\n",
       "  {'start': 185,\n",
       "   'end': 401,\n",
       "   'text': 'These properties are regulated by tightly controlled networks of gene expression, which in turn are governed by the availability of transcription factors and their interaction with the underlying epigenetic landscape.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28196802',\n",
       "   'score': 0.16373902559280396},\n",
       "  {'start': 403,\n",
       "   'end': 558,\n",
       "   'text': 'Recent data suggest that, perhaps unexpectedly, some key epigenetic marks, and thereby gene expression, are regulated by the levels of specific metabolites.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28196802',\n",
       "   'score': 0.13528311252593994},\n",
       "  {'start': 560,\n",
       "   'end': 696,\n",
       "   'text': 'Hence, cellular metabolism plays a vital role beyond simply the production of energy, and may be involved in the regulation of cell fate.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28196802',\n",
       "   'score': 0.1640891432762146},\n",
       "  {'start': 698,\n",
       "   'end': 1014,\n",
       "   'text': 'In this Review, we discuss the metabolic changes that occur during the transitions between different pluripotent states both in vitro and in vivo, including during reprogramming to pluripotency and the onset of differentiation, and we discuss the extent to which distinct metabolites might regulate these transitions.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28196802',\n",
       "   'score': 0.32108139991760254},\n",
       "  {'start': 0,\n",
       "   'end': 63,\n",
       "   'text': 'PRC1 proteins orchestrate three-dimensional genome architecture.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '26417860',\n",
       "   'score': 0.1906348466873169},\n",
       "  {'start': 0,\n",
       "   'end': 144,\n",
       "   'text': 'The three-dimensional organization of the genome has an important role in orchestrating gene expression, but its regulation is poorly understood.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '26417860',\n",
       "   'score': 0.27219128608703613},\n",
       "  {'start': 146,\n",
       "   'end': 316,\n",
       "   'text': 'Now, a new study uncovers a major role for Polycomb components of the PRC1 complex in organizing physical networks of genes that are co-repressed to maintain pluripotency.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '26417860',\n",
       "   'score': 0.3448425233364105},\n",
       "  {'start': 0,\n",
       "   'end': 111,\n",
       "   'text': 'Wnt/Yes-Associated Protein Interactions During Neural Tissue Patterning of Human Induced Pluripotent Stem Cells.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '28726548',\n",
       "   'score': 0.33925867080688477},\n",
       "  {'start': 0,\n",
       "   'end': 135,\n",
       "   'text': 'Human induced pluripotent stem cells (hiPSCs) have special ability to self-assemble into neural spheroids or mini-brain-like structures.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28726548',\n",
       "   'score': 0.2946568727493286},\n",
       "  {'start': 137,\n",
       "   'end': 304,\n",
       "   'text': 'During the self-assembly process, Wnt signaling plays an important role in regional patterning and establishing positional identity of hiPSC-derived neural progenitors.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28726548',\n",
       "   'score': 0.1388946771621704},\n",
       "  {'start': 306,\n",
       "   'end': 521,\n",
       "   'text': 'Recently, the role of Wnt signaling in regulating Yes-associated protein (YAP) expression (nuclear or cytoplasmic), the pivotal regulator during organ growth and tissue generation, has attracted increasing interests.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28726548',\n",
       "   'score': 0.1723305881023407},\n",
       "  {'start': 523,\n",
       "   'end': 642,\n",
       "   'text': 'However, the interactions between Wnt and YAP expression for neural lineage commitment of hiPSCs remain poorly explored.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28726548',\n",
       "   'score': 0.13696685433387756},\n",
       "  {'start': 644,\n",
       "   'end': 827,\n",
       "   'text': 'The objective of this study is to investigate the effects of Wnt signaling and YAP expression on the cellular population in three-dimensional (3D) neural spheroids derived from hiPSCs.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28726548',\n",
       "   'score': 0.20046281814575195},\n",
       "  {'start': 829,\n",
       "   'end': 974,\n",
       "   'text': 'In this study, Wnt signaling was activated using CHIR99021 for 3D neural spheroids derived from human iPSK3 cells through embryoid body formation.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28726548',\n",
       "   'score': 0.18246540427207947},\n",
       "  {'start': 976,\n",
       "   'end': 1130,\n",
       "   'text': 'Our results indicate that Wnt activation induces nuclear localization of YAP and upregulates the expression of HOXB4, the marker for hindbrain/spinal cord.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28726548',\n",
       "   'score': 0.13647699356079102},\n",
       "  {'start': 1132,\n",
       "   'end': 1245,\n",
       "   'text': 'By contrast, the cells exhibit more rostral forebrain neural identity (expression of TBR1) without Wnt activation.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28726548',\n",
       "   'score': 0.13439282774925232},\n",
       "  {'start': 1247,\n",
       "   'end': 1355,\n",
       "   'text': 'Cytochalasin D was then used to induce cytoplasmic YAP and the results showed the decreased HOXB4 expression.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28726548',\n",
       "   'score': 0.13506647944450378},\n",
       "  {'start': 1357,\n",
       "   'end': 1488,\n",
       "   'text': 'In addition, the incorporation of microparticles in the neural spheroids was investigated for the perturbation of neural patterning.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28726548',\n",
       "   'score': 0.1369967758655548},\n",
       "  {'start': 1490,\n",
       "   'end': 1722,\n",
       "   'text': 'This study may indicate the bidirectional interactions of Wnt signaling and YAP expression during neural tissue patterning, which have the significance in neurological disease modeling, drug screening, and neural tissue regeneration.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28726548',\n",
       "   'score': 0.14020735025405884},\n",
       "  {'start': 0,\n",
       "   'end': 110,\n",
       "   'text': 'Microphthalamia-associated transcription factor: a critical regulator of pigment cell development and survival.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '12789278',\n",
       "   'score': 0.13920381665229797},\n",
       "  {'start': 0,\n",
       "   'end': 162,\n",
       "   'text': 'The microphthalamia-associated transcription factor (MITF) is an integral transcriptional regulator in melanocyte, the lineage from which melanoma cells originate.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '12789278',\n",
       "   'score': 0.143486887216568},\n",
       "  {'start': 164,\n",
       "   'end': 339,\n",
       "   'text': 'This basic-helix-loop-helix-leucine-zipper (bHLHzip) protein is critical for melanocyte cell-fate choice during commitment from pluripotent precursor cells in the neural crest.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '12789278',\n",
       "   'score': 0.35395193099975586},\n",
       "  {'start': 341,\n",
       "   'end': 561,\n",
       "   'text': 'Its role in differentiation pathways has been highlighted by its potent transcriptional and lineage-specific regulation of the three major pigment enzymes: tyrosinase, Tyrp1, and Dct as well as other pigmentation factors.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '12789278',\n",
       "   'score': 0.1626070737838745},\n",
       "  {'start': 563,\n",
       "   'end': 754,\n",
       "   'text': 'However, the cellular functions of MITF seem to be wider than differentiation and cell-fate pathways alone, since melanocytes and melanoma cells appear to require an expression of this factor.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '12789278',\n",
       "   'score': 0.13690334558486938},\n",
       "  {'start': 756,\n",
       "   'end': 902,\n",
       "   'text': 'Here, we discuss the transcriptional networks in which MITF is thought to reside and describe signaling pathways in the cell which impinge on MITF.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '12789278',\n",
       "   'score': 0.16772770881652832},\n",
       "  {'start': 904,\n",
       "   'end': 1062,\n",
       "   'text': 'Accumulating evidence supports the notion that MITF is involved in survival pathways during normal development as well as during neoplastic growth of melanoma.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '12789278',\n",
       "   'score': 0.14641043543815613},\n",
       "  {'start': 0,\n",
       "   'end': 90,\n",
       "   'text': 'Polycomb repressive complex PRC1 spatially constrains the mouse embryonic stem cell genome.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '26323060',\n",
       "   'score': 0.13326194882392883},\n",
       "  {'start': 0,\n",
       "   'end': 157,\n",
       "   'text': 'The Polycomb repressive complexes PRC1 and PRC2 maintain embryonic stem cell (ESC) pluripotency by silencing lineage-specifying developmental regulator genes.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '26323060',\n",
       "   'score': 0.33374887704849243},\n",
       "  {'start': 159,\n",
       "   'end': 261,\n",
       "   'text': 'Emerging evidence suggests that Polycomb complexes act through controlling spatial genome organization.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '26323060',\n",
       "   'score': 0.15924254059791565},\n",
       "  {'start': 263,\n",
       "   'end': 407,\n",
       "   'text': 'We show that PRC1 functions as a master regulator of mouse ESC genome architecture by organizing genes in three-dimensional interaction networks.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '26323060',\n",
       "   'score': 0.23159152269363403},\n",
       "  {'start': 409,\n",
       "   'end': 579,\n",
       "   'text': 'The strongest spatial network is composed of the four Hox gene clusters and early developmental transcription factor genes, the majority of which contact poised enhancers.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '26323060',\n",
       "   'score': 0.24867889285087585},\n",
       "  {'start': 581,\n",
       "   'end': 685,\n",
       "   'text': 'Removal of Polycomb repression leads to disruption of promoter-promoter contacts in the Hox gene network.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '26323060',\n",
       "   'score': 0.2731490135192871},\n",
       "  {'start': 687,\n",
       "   'end': 934,\n",
       "   'text': 'In contrast, promoter-enhancer contacts are maintained in the absence of Polycomb repression, with accompanying widespread acquisition of active chromatin signatures at network enhancers and pronounced transcriptional upregulation of network genes.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '26323060',\n",
       "   'score': 0.2736440896987915},\n",
       "  {'start': 936,\n",
       "   'end': 1070,\n",
       "   'text': 'Thus, PRC1 physically constrains developmental transcription factor genes and their enhancers in a silenced but poised spatial network.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '26323060',\n",
       "   'score': 0.23642989993095398},\n",
       "  {'start': 1072,\n",
       "   'end': 1213,\n",
       "   'text': 'We propose that the selective release of genes from this spatial network underlies cell fate specification during early embryonic development.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '26323060',\n",
       "   'score': 0.15765175223350525},\n",
       "  {'start': 0,\n",
       "   'end': 104,\n",
       "   'text': 'Poly(ADP-ribose) polymerase 1 regulates nuclear reprogramming and promotes iPSC generation without c-Myc.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '23277454',\n",
       "   'score': 0.13552239537239075},\n",
       "  {'start': 0,\n",
       "   'end': 152,\n",
       "   'text': 'Poly(ADP-ribose) polymerase 1 (Parp1) catalyzes poly(ADP-ribosylation) (PARylation) and induces replication networks involved in multiple nuclear events.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '23277454',\n",
       "   'score': 0.17222949862480164},\n",
       "  {'start': 154,\n",
       "   'end': 406,\n",
       "   'text': 'Using mass spectrometry and Western blotting, Parp1 and PARylation activity were intensively detected in induced pluripotent stem cells (iPSCs) and embryonic stem cells, but they were lower in mouse embryonic fibroblasts (MEFs) and differentiated cells.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '23277454',\n",
       "   'score': 0.27230048179626465},\n",
       "  {'start': 408,\n",
       "   'end': 563,\n",
       "   'text': 'We show that knockdown of Parp1 and pharmacological inhibition of PARylation both reduced the efficiency of iPSC generation induced by Oct4/Sox2/Klf4/c-Myc.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '23277454',\n",
       "   'score': 0.3131505250930786},\n",
       "  {'start': 565,\n",
       "   'end': 661,\n",
       "   'text': 'Furthermore, Parp1 is able to replace Klf4 or c-Myc to enhance the efficiency of iPSC generation.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '23277454',\n",
       "   'score': 0.13825595378875732},\n",
       "  {'start': 663,\n",
       "   'end': 764,\n",
       "   'text': 'In addition, mouse iPSCs generated from Oct4/Sox2/Parp1-overexpressing MEFs formed chimeric offspring.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '23277454',\n",
       "   'score': 0.32201915979385376},\n",
       "  {'start': 766,\n",
       "   'end': 892,\n",
       "   'text': 'Notably, the endogenous Parp1 and PARylation activity was enhanced by overexpression of c-Myc and repressed by c-Myc knockdown.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '23277454',\n",
       "   'score': 0.13864994049072266},\n",
       "  {'start': 894,\n",
       "   'end': 994,\n",
       "   'text': 'A chromatin immunoprecipitation assay revealed a direct interaction of c-Myc with the Parp1 promoter.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '23277454',\n",
       "   'score': 0.21714571118354797},\n",
       "  {'start': 996,\n",
       "   'end': 1211,\n",
       "   'text': 'PAR-resin pulldown, followed by proteomic analysis, demonstrated high levels of PARylated Chd1L, DNA ligase III, SSrp1, Xrcc-6/Ku70, and Parp2 in pluripotent cells, which decreased during the differentiation process.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '23277454',\n",
       "   'score': 0.2946053743362427},\n",
       "  {'start': 1213,\n",
       "   'end': 1428,\n",
       "   'text': 'These data show that the activation of Parp1, partly regulated by endogenous c-Myc, effectively promotes iPSC production and helps to maintain a pluripotent state by posttranslationally modulating protein PARylation.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '23277454',\n",
       "   'score': 0.31614190340042114},\n",
       "  {'start': 0,\n",
       "   'end': 69,\n",
       "   'text': 'Super-enhancers in transcriptional regulation and genome organization.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '31724731',\n",
       "   'score': 0.3204605281352997},\n",
       "  {'start': 0,\n",
       "   'end': 187,\n",
       "   'text': 'Gene expression is precisely controlled in a stage and cell-type-specific manner, largely through the interaction between cis-regulatory elements and their associated trans-acting factors.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31724731',\n",
       "   'score': 0.14798542857170105},\n",
       "  {'start': 189,\n",
       "   'end': 425,\n",
       "   'text': \"Where these components aggregate in promoters and enhancers, they are able to cooperate to modulate chromatin structure and support the engagement in long-range 3D superstructures that shape the dynamics of a cell's genomic architecture.\",\n",
       "   'is_title': False,\n",
       "   'doc_id': '31724731',\n",
       "   'score': 0.260100781917572},\n",
       "  {'start': 427,\n",
       "   'end': 653,\n",
       "   'text': \"Recently, the term 'super-enhancer' has been introduced to describe a hyper-active regulatory domain comprising a complex array of sequence elements that work together to control the key gene networks involved in cell identity.\",\n",
       "   'is_title': False,\n",
       "   'doc_id': '31724731',\n",
       "   'score': 0.27670320868492126},\n",
       "  {'start': 655,\n",
       "   'end': 848,\n",
       "   'text': 'Here, we survey the unique characteristics of super-enhancers compared to other enhancer types and summarize the recent advances in our understanding of their biological role in gene regulation.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31724731',\n",
       "   'score': 0.23841539025306702},\n",
       "  {'start': 850,\n",
       "   'end': 1051,\n",
       "   'text': 'In particular, we discuss their capacity to attract the formation of phase-separated condensates, and capacity to generate three-dimensional genome structures that precisely activate their target genes.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31724731',\n",
       "   'score': 0.20552003383636475},\n",
       "  {'start': 1053,\n",
       "   'end': 1262,\n",
       "   'text': 'We also propose a multi-stage transition model to explain the evolutionary pressure driving the development of super-enhancers in complex organisms, and highlight the potential for involvement in tumorigenesis.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31724731',\n",
       "   'score': 0.21455782651901245},\n",
       "  {'start': 1264,\n",
       "   'end': 1401,\n",
       "   'text': 'Finally, we discuss more broadly the role of super-enhancers in human health disorders and related potential in therapeutic interventions.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31724731',\n",
       "   'score': 0.22466906905174255},\n",
       "  {'start': 0,\n",
       "   'end': 91,\n",
       "   'text': 'Making a Kidney Organoid Using the Directed Differentiation of Human Pluripotent Stem Cells.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '28361319',\n",
       "   'score': 0.30767688155174255},\n",
       "  {'start': 0,\n",
       "   'end': 114,\n",
       "   'text': 'An organoid can be defined as a three-dimensional organ-like structure formed from organ-specific progenitor cells.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28361319',\n",
       "   'score': 0.19676324725151062},\n",
       "  {'start': 116,\n",
       "   'end': 254,\n",
       "   'text': 'Organ progenitor cells were empirically found to self-organize three-dimensional tissues when they were aggregated and cultivated in vitro.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28361319',\n",
       "   'score': 0.20012500882148743},\n",
       "  {'start': 256,\n",
       "   'end': 581,\n",
       "   'text': 'While this nature power of progenitor cells has an amazing potential to recreate artificial organs in vitro, there had been difficulty to apply this technology to human organs due to the inaccessibility to human progenitor cells until human-induced pluripotent stem cell (hiPSC) was invented by Takahashi and Yamanaka in 2007.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28361319',\n",
       "   'score': 0.28519439697265625},\n",
       "  {'start': 583,\n",
       "   'end': 738,\n",
       "   'text': 'As embryonic stem cells do, hiPSCs also have pluripotency to give rise to any organs/tissues cell types, including the kidney, via directed differentiation.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28361319',\n",
       "   'score': 0.30977320671081543},\n",
       "  {'start': 740,\n",
       "   'end': 843,\n",
       "   'text': 'Here, we provide a detailed protocol for generating kidney organoids using human pluripotent stem cells.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28361319',\n",
       "   'score': 0.2971879839897156},\n",
       "  {'start': 845,\n",
       "   'end': 937,\n",
       "   'text': 'The protocol differentiates human pluripotent stem cells into the posterior primitive streak.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28361319',\n",
       "   'score': 0.286166787147522},\n",
       "  {'start': 939,\n",
       "   'end': 1121,\n",
       "   'text': 'This is followed by the simultaneous induction of posterior and anterior intermediate mesoderm that are subsequently aggregated and undergo self-organization into the kidney organoid.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28361319',\n",
       "   'score': 0.17479455471038818},\n",
       "  {'start': 1123,\n",
       "   'end': 1371,\n",
       "   'text': 'Such kidney organoids are comprised of all anticipated kidney cell types including nephrons segmented into the glomerulus, proximal tubule, loop of Henle, and distal tubule as well as the collecting duct, endothelial network, and renal interstitium.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28361319',\n",
       "   'score': 0.16491156816482544},\n",
       "  {'start': 0,\n",
       "   'end': 60,\n",
       "   'text': 'Nanoengineered Platforms to Guide Pluripotent Stem Cell Fate.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '26918198',\n",
       "   'score': 0.3092994689941406},\n",
       "  {'start': 0,\n",
       "   'end': 129,\n",
       "   'text': 'Tissue engineering utilizes cells, signaling molecules, and scaffolds towards creating functional tissue to repair damaged organs.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '26918198',\n",
       "   'score': 0.1405961513519287},\n",
       "  {'start': 131,\n",
       "   'end': 301,\n",
       "   'text': 'Pluripotent stem cells (PSCs) are a promising cell source due to their ability to self-renewal indefinitely and their potential to differentiate into almost any cell type.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '26918198',\n",
       "   'score': 0.30063775181770325},\n",
       "  {'start': 303,\n",
       "   'end': 451,\n",
       "   'text': 'Great strides have been taken to parse the physiological mechanisms by which PSCs respond to their microenvironment and commit to a specific lineage.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '26918198',\n",
       "   'score': 0.1437987983226776},\n",
       "  {'start': 453,\n",
       "   'end': 681,\n",
       "   'text': 'The combination of physical cues and chemical factors is thought to have the most profound influence on stem cell behavior, therefore a major focus of tissue engineering strategies is scaffold design to incorporate these signals.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '26918198',\n",
       "   'score': 0.1452803611755371},\n",
       "  {'start': 683,\n",
       "   'end': 909,\n",
       "   'text': 'One overlooked component of the in vivo microenvironment researchers attempt to recapitulate with three dimensional (3D) substrates is the nanoarchitecture formed by the fibrillar network of extracellular matrix (ECM) proteins.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '26918198',\n",
       "   'score': 0.2397085428237915},\n",
       "  {'start': 911,\n",
       "   'end': 1026,\n",
       "   'text': 'These nanoscale features have the ability to impact cell adhesion, migration, proliferation, and lineage commitment.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '26918198',\n",
       "   'score': 0.1422543227672577},\n",
       "  {'start': 1028,\n",
       "   'end': 1284,\n",
       "   'text': 'Significant advances have been made in deciphering how these nanoscale cues interact with stem cells to determine phenotype, but much is still unknown as to how the interplay between physical and chemical signals regulate in vitro and in vivo cellular fate.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '26918198',\n",
       "   'score': 0.14635738730430603},\n",
       "  {'start': 1286,\n",
       "   'end': 1468,\n",
       "   'text': 'This review dives deeper to investigate nanoscale platforms for engineering tissue, as well use the use of these nanotechnologies to drive pluripotent stem cell lineage determination.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '26918198',\n",
       "   'score': 0.2841383218765259},\n",
       "  {'start': 0,\n",
       "   'end': 122,\n",
       "   'text': 'Chromatin insulator factors involved in long-range DNA interactions and their role in the folding of the Drosophila genome.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '25165871',\n",
       "   'score': 0.15337121486663818},\n",
       "  {'start': 0,\n",
       "   'end': 121,\n",
       "   'text': 'Chromatin insulators are genetic elements implicated in the organization of chromatin and the regulation of transcription.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '25165871',\n",
       "   'score': 0.2160864770412445},\n",
       "  {'start': 123,\n",
       "   'end': 255,\n",
       "   'text': 'In Drosophila, different insulator types were characterized by their locus-specific composition of insulator proteins and co-factors.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '25165871',\n",
       "   'score': 0.14510449767112732},\n",
       "  {'start': 257,\n",
       "   'end': 499,\n",
       "   'text': 'Insulators mediate specific long-range DNA contacts required for the three dimensional organization of the interphase nucleus and for transcription regulation, but the mechanisms underlying the formation of these contacts is currently unknown.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '25165871',\n",
       "   'score': 0.2787514626979828},\n",
       "  {'start': 501,\n",
       "   'end': 813,\n",
       "   'text': 'Here, we investigate the molecular associations between different components of insulator complexes (BEAF32, CP190 and Chromator) by biochemical and biophysical means, and develop a novel single-molecule assay to determine what factors are necessary and essential for the formation of long-range DNA interactions.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '25165871',\n",
       "   'score': 0.1438683569431305},\n",
       "  {'start': 815,\n",
       "   'end': 939,\n",
       "   'text': 'We show that BEAF32 is able to bind DNA specifically and with high affinity, but not to bridge long-range interactions (LRI).',\n",
       "   'is_title': False,\n",
       "   'doc_id': '25165871',\n",
       "   'score': 0.14279693365097046},\n",
       "  {'start': 941,\n",
       "   'end': 1077,\n",
       "   'text': 'In contrast, we show that CP190 and Chromator are able to mediate LRI between specifically-bound BEAF32 nucleoprotein complexes in vitro.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '25165871',\n",
       "   'score': 0.14328351616859436},\n",
       "  {'start': 1079,\n",
       "   'end': 1253,\n",
       "   'text': 'This ability of CP190 and Chromator to establish LRI requires specific contacts between BEAF32 and their C-terminal domains, and dimerization through their N-terminal domains.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '25165871',\n",
       "   'score': 0.14200228452682495},\n",
       "  {'start': 1255,\n",
       "   'end': 1399,\n",
       "   'text': 'In particular, the BTB/POZ domains of CP190 form a strict homodimer, and its C-terminal domain interacts with several insulator binding proteins.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '25165871',\n",
       "   'score': 0.14701470732688904},\n",
       "  {'start': 1401,\n",
       "   'end': 1644,\n",
       "   'text': 'We propose a general model for insulator function in which BEAF32/dCTCF/Su(HW) provide DNA specificity (first layer proteins) whereas CP190/Chromator are responsible for the physical interactions required for long-range contacts (second layer).',\n",
       "   'is_title': False,\n",
       "   'doc_id': '25165871',\n",
       "   'score': 0.1443822681903839},\n",
       "  {'start': 1646,\n",
       "   'end': 1949,\n",
       "   'text': 'This network of organized, multi-layer interactions could explain the different activities of insulators as chromatin barriers, enhancer blockers, and transcriptional regulators, and suggest a general mechanism for how insulators may shape the organization of higher-order chromatin during cell division.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '25165871',\n",
       "   'score': 0.3165259063243866},\n",
       "  {'start': 0,\n",
       "   'end': 85,\n",
       "   'text': 'Chromosome conformation elucidates regulatory relationships in developing human brain.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '27760116',\n",
       "   'score': 0.14131638407707214},\n",
       "  {'start': 0,\n",
       "   'end': 123,\n",
       "   'text': 'Three-dimensional physical interactions within chromosomes dynamically regulate gene expression in a tissue-specific manner.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '27760116',\n",
       "   'score': 0.20864352583885193},\n",
       "  {'start': 125,\n",
       "   'end': 331,\n",
       "   'text': 'However, the 3D organization of chromosomes during human brain development and its role in regulating gene networks dysregulated in neurodevelopmental disorders, such as autism or schizophrenia, are unknown.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '27760116',\n",
       "   'score': 0.31868794560432434},\n",
       "  {'start': 333,\n",
       "   'end': 570,\n",
       "   'text': 'Here we generate high-resolution 3D maps of chromatin contacts during human corticogenesis, permitting large-scale annotation of previously uncharacterized regulatory relationships relevant to the evolution of human cognition and disease.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '27760116',\n",
       "   'score': 0.19164606928825378},\n",
       "  {'start': 572,\n",
       "   'end': 769,\n",
       "   'text': 'Our analyses identify hundreds of genes that physically interact with enhancers gained on the human lineage, many of which are under purifying selection and associated with human cognitive function.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '27760116',\n",
       "   'score': 0.22818276286125183},\n",
       "  {'start': 771,\n",
       "   'end': 1172,\n",
       "   'text': 'We integrate chromatin contacts with non-coding variants identified in schizophrenia genome-wide association studies (GWAS), highlighting multiple candidate schizophrenia risk genes and pathways, including transcription factors involved in neurogenesis, and cholinergic signalling molecules, several of which are supported by independent expression quantitative trait loci and gene expression analyses.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '27760116',\n",
       "   'score': 0.1516258716583252},\n",
       "  {'start': 1174,\n",
       "   'end': 1361,\n",
       "   'text': 'Genome editing in human neural progenitors suggests that one of these distal schizophrenia GWAS loci regulates FOXG1 expression, supporting its potential role as a schizophrenia risk gene.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '27760116',\n",
       "   'score': 0.14247703552246094},\n",
       "  {'start': 1363,\n",
       "   'end': 1585,\n",
       "   'text': 'This work provides a framework for understanding the effect of non-coding regulatory elements on human brain development and the evolution of cognition, and highlights novel mechanisms underlying neuropsychiatric disorders.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '27760116',\n",
       "   'score': 0.14432692527770996},\n",
       "  {'start': 0,\n",
       "   'end': 94,\n",
       "   'text': 'Paracrine signals regulate human liver organoid maturation from induced pluripotent stem cells.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '28275009',\n",
       "   'score': 0.3126864433288574},\n",
       "  {'start': 0,\n",
       "   'end': 108,\n",
       "   'text': 'A self-organizing organoid model provides a new approach to study the mechanism of human liver organogenesis.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28275009',\n",
       "   'score': 0.14264750480651855},\n",
       "  {'start': 110,\n",
       "   'end': 250,\n",
       "   'text': 'Previous animal models documented that simultaneous paracrine signaling and cell-to-cell surface contact regulate hepatocyte differentiation.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28275009',\n",
       "   'score': 0.14155784249305725},\n",
       "  {'start': 252,\n",
       "   'end': 510,\n",
       "   'text': 'To dissect the relative contributions of the paracrine effects, we first established a liver organoid using human induced pluripotent stem cells (iPSCs), mesenchymal stem cells (MSCs) and human umbilical vein endothelial cells (HUVECs) as previously reported.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28275009',\n",
       "   'score': 0.3001575767993927},\n",
       "  {'start': 512,\n",
       "   'end': 671,\n",
       "   'text': 'Time-lapse imaging showed that hepatic-specified endoderm iPSCs (HE-iPSCs) self-assembled into three-dimensional organoids, resulting in hepatic gene induction.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28275009',\n",
       "   'score': 0.19780898094177246},\n",
       "  {'start': 673,\n",
       "   'end': 786,\n",
       "   'text': 'Progressive differentiation was demonstrated by hepatic protein production after in vivo organoid transplantation.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28275009',\n",
       "   'score': 0.14900216460227966},\n",
       "  {'start': 788,\n",
       "   'end': 927,\n",
       "   'text': 'To assess the paracrine contributions, we employed a Transwell system in which HE-iPSCs were separately co-cultured with MSCs and/or HUVECs.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28275009',\n",
       "   'score': 0.1466226875782013},\n",
       "  {'start': 929,\n",
       "   'end': 1107,\n",
       "   'text': 'Although the three-dimensional structure did not form, their soluble factors induced a hepatocyte-like phenotype in HE-iPSCs, resulting in the expression of bile salt export pump.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28275009',\n",
       "   'score': 0.20642510056495667},\n",
       "  {'start': 1109,\n",
       "   'end': 1281,\n",
       "   'text': 'In conclusion, the mesoderm-derived paracrine signals promote hepatocyte maturation in liver organoids, but organoid self-organization requires cell-to-cell surface contact.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28275009',\n",
       "   'score': 0.17506101727485657},\n",
       "  {'start': 1283,\n",
       "   'end': 1427,\n",
       "   'text': 'Our in vitro model demonstrates a novel approach to identify developmental paracrine signals regulating the differentiation of human hepatocytes.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '28275009',\n",
       "   'score': 0.1632860004901886},\n",
       "  {'start': 0,\n",
       "   'end': 84,\n",
       "   'text': 'Chromatin connectivity maps reveal dynamic promoter-enhancer long-range associations.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '24213634',\n",
       "   'score': 0.23710381984710693},\n",
       "  {'start': 0,\n",
       "   'end': 147,\n",
       "   'text': 'In multicellular organisms, transcription regulation is one of the central mechanisms modelling lineage differentiation and cell-fate determination.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24213634',\n",
       "   'score': 0.17123103141784668},\n",
       "  {'start': 149,\n",
       "   'end': 273,\n",
       "   'text': 'Transcription requires dynamic chromatin configurations between promoters and their corresponding distal regulatory elements.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24213634',\n",
       "   'score': 0.14240393042564392},\n",
       "  {'start': 275,\n",
       "   'end': 444,\n",
       "   'text': 'It is believed that their communication occurs within large discrete foci of aggregated RNA polymerases termed transcription factories in three-dimensional nuclear space.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24213634',\n",
       "   'score': 0.208286315202713},\n",
       "  {'start': 446,\n",
       "   'end': 551,\n",
       "   'text': 'However, the dynamic nature of chromatin connectivity has not been characterized at the genome-wide level.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24213634',\n",
       "   'score': 0.14365434646606445},\n",
       "  {'start': 553,\n",
       "   'end': 927,\n",
       "   'text': 'Here, through a chromatin interaction analysis with paired-end tagging approach using an antibody that primarily recognizes the pre-initiation complexes of RNA polymerase II, we explore the transcriptional interactomes of three mouse cells of progressive lineage commitment, including pluripotent embryonic stem cells, neural stem cells and neurosphere stem/progenitor cells.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24213634',\n",
       "   'score': 0.3128579556941986},\n",
       "  {'start': 929,\n",
       "   'end': 1121,\n",
       "   'text': 'Our global chromatin connectivity maps reveal approximately 40,000 long-range interactions, suggest precise enhancer-promoter associations and delineate cell-type-specific chromatin structures.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24213634',\n",
       "   'score': 0.22451671957969666},\n",
       "  {'start': 1123,\n",
       "   'end': 1259,\n",
       "   'text': 'Analysis of the complex regulatory repertoire shows that there are extensive colocalizations among promoters and distal-acting enhancers.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24213634',\n",
       "   'score': 0.22484946250915527},\n",
       "  {'start': 1261,\n",
       "   'end': 1459,\n",
       "   'text': 'Most of the enhancers associate with promoters located beyond their nearest active genes, indicating that the linear juxtaposition is not the only guiding principle driving enhancer target selection.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24213634',\n",
       "   'score': 0.22459429502487183},\n",
       "  {'start': 1461,\n",
       "   'end': 1644,\n",
       "   'text': 'Although promoter-enhancer interactions exhibit high cell-type specificity, promoters involved in interactions are found to be generally common and mostly active among different cells.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24213634',\n",
       "   'score': 0.2469051480293274},\n",
       "  {'start': 1646,\n",
       "   'end': 1877,\n",
       "   'text': 'Chromatin connectivity networks reveal that the pivotal genes of reprogramming functions are transcribed within physical proximity to each other in embryonic stem cells, linking chromatin architecture to coordinated gene expression.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24213634',\n",
       "   'score': 0.174902081489563},\n",
       "  {'start': 1879,\n",
       "   'end': 2020,\n",
       "   'text': 'Our study sets the stage for the full-scale dissection of spatial and temporal genome structures and their roles in orchestrating development.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24213634',\n",
       "   'score': 0.14775970578193665},\n",
       "  {'start': 0,\n",
       "   'end': 71,\n",
       "   'text': 'High-dimensional switches and the modelling of cellular differentiation.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '15652148',\n",
       "   'score': 0.19714757800102234},\n",
       "  {'start': 0,\n",
       "   'end': 191,\n",
       "   'text': 'Many genes have been identified as driving cellular differentiation, but because of their complex interactions, the understanding of their collective behaviour requires mathematical modelling.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '15652148',\n",
       "   'score': 0.14331617951393127},\n",
       "  {'start': 193,\n",
       "   'end': 453,\n",
       "   'text': 'Intriguingly, it has been observed in numerous developmental contexts, and particularly haematopoiesis, that genes regulating differentiation are initially co-expressed in progenitors despite their antagonism, before one is upregulated and others downregulated.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '15652148',\n",
       "   'score': 0.1641356647014618},\n",
       "  {'start': 455,\n",
       "   'end': 877,\n",
       "   'text': 'We characterise conditions under which three classes of generic \"master regulatory networks\", modelled at the molecular level after experimentally observed interactions (including bHLH protein dimerisation), and including an arbitrary number of antagonistic components, can behave as a \"multi-switch\", directing differentiation in an all-or-none fashion to a specific cell-type chosen among more than two possible outcomes.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '15652148',\n",
       "   'score': 0.18904632329940796},\n",
       "  {'start': 879,\n",
       "   'end': 1029,\n",
       "   'text': 'bHLH dimerisation networks can readily display coexistence of many antagonistic factors when competition is low (a simple characterisation is derived).',\n",
       "   'is_title': False,\n",
       "   'doc_id': '15652148',\n",
       "   'score': 0.1690405309200287},\n",
       "  {'start': 1031,\n",
       "   'end': 1349,\n",
       "   'text': 'Decision-making can be forced by a transient increase in competition, which could correspond to some unexplained experimental observations related to Id proteins; the speed of response varies with the initial conditions the network is subjected to, which could explain some aspects of cell behaviour upon reprogramming.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '15652148',\n",
       "   'score': 0.16980090737342834},\n",
       "  {'start': 1351,\n",
       "   'end': 1588,\n",
       "   'text': 'The coexistence of antagonistic factors at low levels, early in the differentiation process or in pluripotent stem cells, could be an intrinsic property of the interaction between those factors, not requiring a specific regulatory system.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '15652148',\n",
       "   'score': 0.3194504678249359},\n",
       "  {'start': 0,\n",
       "   'end': 63,\n",
       "   'text': 'The SUMO E3 Ligase SIZ1 Negatively Regulates Shoot Regeneration.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '32611787',\n",
       "   'score': 0.14275389909744263},\n",
       "  {'start': 0,\n",
       "   'end': 94,\n",
       "   'text': 'Plants form calluses and regenerate new organs when incubated on phytohormone-containing media.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '32611787',\n",
       "   'score': 0.14283525943756104},\n",
       "  {'start': 96,\n",
       "   'end': 358,\n",
       "   'text': 'While accumulating evidence suggests that these regenerative processes are governed by transcriptional networks orchestrating wound response and developmental transitions, it remains unknown if posttranslational regulatory mechanisms are involved in this process.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '32611787',\n",
       "   'score': 0.18269506096839905},\n",
       "  {'start': 360,\n",
       "   'end': 639,\n",
       "   'text': 'In this study, we demonstrate that SAP AND MIZ1 DOMAIN- CONTAINING LIGASE1 (SIZ1), an E3 ligase-catalyzing attachment of the SMALL UBIQUITIN-LIKE MODIFIER (SUMO) to proteins, regulates wound-induced signal transduction and organ regeneration in Arabidopsis (Arabidopsis thaliana).',\n",
       "   'is_title': False,\n",
       "   'doc_id': '32611787',\n",
       "   'score': 0.1490306258201599},\n",
       "  {'start': 641,\n",
       "   'end': 1040,\n",
       "   'text': 'We show that loss-of-function mutants for SIZ1 exhibit overproduction of shoot meristems under in vitro tissue culture conditions, while this defect is rescued in a complementation line expressing pSIZ1::SIZ1 RNA sequencing analysis revealed that siz1-2 mutants exhibit enhanced transcriptional responses to wound stress, resulting in the hyper-induction of over 400 genes immediately after wounding.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '32611787',\n",
       "   'score': 0.14854979515075684},\n",
       "  {'start': 1042,\n",
       "   'end': 1352,\n",
       "   'text': 'Among them, we show that elevated levels of WOUND INDUCED DEDIFFERENTIATION1 (WIND1) and WIND2 contribute to the enhanced shoot regeneration observed in siz1 mutants, as expression of the dominant-negative chimeric protein WIND1-SRDX (SUPERMAN repression domain) in siz1-3 mutants partly rescues this phenotype.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '32611787',\n",
       "   'score': 0.15515625476837158},\n",
       "  {'start': 1354,\n",
       "   'end': 1671,\n",
       "   'text': 'Although compromised SIZ1 function does not modify the transcription of genes implicated in auxin-induced callus formation and/or pluripotency acquisition, it does lead to enhanced induction of cytokinin-induced shoot meristem regulators such as WUSCHEL, promoting the formation of WUSCHEL-expressing foci in explants.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '32611787',\n",
       "   'score': 0.3235282301902771},\n",
       "  {'start': 1673,\n",
       "   'end': 1811,\n",
       "   'text': 'This study thus suggests that SIZ1 negatively regulates shoot regeneration in part by repressing wound-induced developmental reprogramming.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '32611787',\n",
       "   'score': 0.14303883910179138},\n",
       "  {'start': 0,\n",
       "   'end': 38,\n",
       "   'text': 'Gene regulation in the third dimension.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '18369139',\n",
       "   'score': 0.17092499136924744},\n",
       "  {'start': 0,\n",
       "   'end': 122,\n",
       "   'text': 'Analysis of the spatial organization of chromosomes reveals complex three-dimensional networks of chromosomal interactions.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '18369139',\n",
       "   'score': 0.30421969294548035},\n",
       "  {'start': 124,\n",
       "   'end': 322,\n",
       "   'text': 'These interactions affect gene expression at multiple levels, including long-range control by distant enhancers and repressors, coordinated expression of genes, and modification of epigenetic states.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '18369139',\n",
       "   'score': 0.23709824681282043},\n",
       "  {'start': 324,\n",
       "   'end': 491,\n",
       "   'text': 'Major challenges now include deciphering the mechanisms by which loci come together and understanding the functional consequences of these often transient associations.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '18369139',\n",
       "   'score': 0.1520516574382782},\n",
       "  {'start': 0,\n",
       "   'end': 106,\n",
       "   'text': 'Specific Contributions of Cohesin-SA1 and Cohesin-SA2 to TADs and Polycomb Domains in Embryonic Stem Cells.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '31216471',\n",
       "   'score': 0.15367034077644348},\n",
       "  {'start': 0,\n",
       "   'end': 62,\n",
       "   'text': 'Cohesin exists in two variants carrying either STAG/SA1 or SA2.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31216471',\n",
       "   'score': 0.1548101305961609},\n",
       "  {'start': 64,\n",
       "   'end': 338,\n",
       "   'text': 'Here we have addressed their specific contributions to the unique spatial organization of the mouse embryonic stem cell genome, which ensures super-enhancer-dependent transcription of\\xa0pluripotency factors and repression of lineage-specification genes within Polycomb domains.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31216471',\n",
       "   'score': 0.2934204339981079},\n",
       "  {'start': 340,\n",
       "   'end': 606,\n",
       "   'text': 'We find that cohesin-SA2 facilitates Polycomb domain compaction through Polycomb repressing complex 1 (PRC1) recruitment and promotes the establishment of long-range interaction networks between distant Polycomb-bound promoters that are important for gene repression.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31216471',\n",
       "   'score': 0.17637240886688232},\n",
       "  {'start': 608,\n",
       "   'end': 722,\n",
       "   'text': 'Cohesin-SA1, in contrast, disrupts these networks, while preserving topologically associating domain (TAD) borders.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31216471',\n",
       "   'score': 0.1741761863231659},\n",
       "  {'start': 724,\n",
       "   'end': 823,\n",
       "   'text': 'The diverse effects of both complexes on genome topology may reflect two modes of action of cohesin.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31216471',\n",
       "   'score': 0.15721797943115234},\n",
       "  {'start': 825,\n",
       "   'end': 997,\n",
       "   'text': 'One, likely involving loop extrusion, establishes overall genome arrangement in TADs together with CTCF and prevents excessive segregation of same-class compartment regions.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31216471',\n",
       "   'score': 0.1531020700931549},\n",
       "  {'start': 999,\n",
       "   'end': 1140,\n",
       "   'text': 'The other is required for organization of local transcriptional hubs such as Polycomb domains and super-enhancers, which define cell identity.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31216471',\n",
       "   'score': 0.2941708564758301},\n",
       "  {'start': 0,\n",
       "   'end': 35,\n",
       "   'text': 'A Cleared View on Retinal Organoids.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '31035373',\n",
       "   'score': 0.1496291160583496},\n",
       "  {'start': 0,\n",
       "   'end': 306,\n",
       "   'text': 'Human induced pluripotent stem cell (hiPSC)-derived organoids mimicking tissues and organs in vitro have advanced medical research, as they opened up new possibilities for in-depth basic research on human organ development as well as providing a human in vitro model for personalized therapeutic approaches.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31035373',\n",
       "   'score': 0.2895902991294861},\n",
       "  {'start': 308,\n",
       "   'end': 477,\n",
       "   'text': 'hiPSC-derived retinal organoids have proven to be of great value for modeling the human retina featuring a very similar cellular composition, layering, and functionality.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31035373',\n",
       "   'score': 0.155246764421463},\n",
       "  {'start': 479,\n",
       "   'end': 647,\n",
       "   'text': 'The technically challenging imaging of three-dimensional structures such as retinal organoids has, however, raised the need for robust whole-organoid imaging techniques.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31035373',\n",
       "   'score': 0.21311861276626587},\n",
       "  {'start': 649,\n",
       "   'end': 817,\n",
       "   'text': 'To improve imaging of retinal organoids we optimized a passive clearing technique (PACT), which enables high-resolution visualization of fragile intra-tissue structures.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31035373',\n",
       "   'score': 0.15271759033203125},\n",
       "  {'start': 819,\n",
       "   'end': 1008,\n",
       "   'text': 'Using cleared retinal organoids, we could greatly enhance the antibody labeling efficiency and depth of imaging at high resolution, thereby improving the three-dimensional microscopy output.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31035373',\n",
       "   'score': 0.21031630039215088},\n",
       "  {'start': 1010,\n",
       "   'end': 1153,\n",
       "   'text': 'In that course, we were able to identify the spatial morphological shape and organization of, e.g., photoreceptor cells and bipolar cell layers.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31035373',\n",
       "   'score': 0.20043876767158508},\n",
       "  {'start': 1155,\n",
       "   'end': 1326,\n",
       "   'text': 'Moreover, we used the synaptic protein CtBP2/Ribeye to visualize the interconnection points of photoreceptor and bipolar cells forming the retinal-specific ribbon synapses.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '31035373',\n",
       "   'score': 0.16256222128868103},\n",
       "  {'start': 0,\n",
       "   'end': 87,\n",
       "   'text': 'A structural network associated with the kallikrein-kinin and renin-angiotensin systems.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '20180650',\n",
       "   'score': 0.19083353877067566},\n",
       "  {'start': 0,\n",
       "   'end': 181,\n",
       "   'text': 'The kallikrein-kinin and renin-angiotensin (KKS-RAS) systems represent two highly regulated proteolytic systems that are involved in several physiological and pathological processes.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '20180650',\n",
       "   'score': 0.1764903962612152},\n",
       "  {'start': 183,\n",
       "   'end': 438,\n",
       "   'text': 'Although their protein-protein interactions can be studied using experimental approaches, it is difficult to differentiate between direct physical interactions and functional associations, which do not involve direct atomic contacts between macromolecules.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '20180650',\n",
       "   'score': 0.17073673009872437},\n",
       "  {'start': 440,\n",
       "   'end': 541,\n",
       "   'text': 'This information can be obtained from an atomic-resolution characterization of the protein interfaces.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '20180650',\n",
       "   'score': 0.1675952672958374},\n",
       "  {'start': 543,\n",
       "   'end': 655,\n",
       "   'text': 'As a result of this, various three-dimensional-based protein-protein interaction databases have become available.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '20180650',\n",
       "   'score': 0.2343921959400177},\n",
       "  {'start': 657,\n",
       "   'end': 824,\n",
       "   'text': 'To gain insight into the multilayered interaction of the KKS-RAS systems, we present a protein network that is built up on three-dimensional domain-domain interactions.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '20180650',\n",
       "   'score': 0.27577322721481323},\n",
       "  {'start': 826,\n",
       "   'end': 1092,\n",
       "   'text': 'The essential domains that link these systems are as follows: Cystatin, Peptidase_C1, Thyroglobulin_1, Insulin, CIMR (Cation-independent mannose-6-phosphate receptor repeat), fn2 (Fibronectin type II domain), fn1 (Fibronectin type I domain), EGF, Trypsin, and Serpin.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '20180650',\n",
       "   'score': 0.15941724181175232},\n",
       "  {'start': 1094,\n",
       "   'end': 1191,\n",
       "   'text': 'We found that the CIMR domain is located at the core of the network, thus connecting both systems.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '20180650',\n",
       "   'score': 0.1862839162349701},\n",
       "  {'start': 1193,\n",
       "   'end': 1348,\n",
       "   'text': 'From the latter, all domain interactors up to level 4 were retrieved, thus displaying a more comprehensive representation of the KKS-RAS structural network.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '20180650',\n",
       "   'score': 0.1840573251247406},\n",
       "  {'start': 0,\n",
       "   'end': 163,\n",
       "   'text': 'Ep-CAM transfection in thymic epithelial cell lines triggers the formation of dynamic actin-rich protrusions involved in the organization of epithelial cell layers.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '11702195',\n",
       "   'score': 0.21225932240486145},\n",
       "  {'start': 0,\n",
       "   'end': 117,\n",
       "   'text': 'Thymic epithelium is organized in a highly connected three-dimensional network through which thymocytes differentiate.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '11702195',\n",
       "   'score': 0.2675233483314514},\n",
       "  {'start': 119,\n",
       "   'end': 190,\n",
       "   'text': 'The molecular mechanisms underlying this organization are still unknown.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '11702195',\n",
       "   'score': 0.19785204529762268},\n",
       "  {'start': 192,\n",
       "   'end': 347,\n",
       "   'text': 'In thymic medulla, a major site of tolerance induction, the development of the epithelial cell net is tightly regulated by the needs of thymocyte selection.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '11702195',\n",
       "   'score': 0.16782167553901672},\n",
       "  {'start': 349,\n",
       "   'end': 426,\n",
       "   'text': 'These reticulated epithelial cells express high levels of the Ep-CAM molecule.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '11702195',\n",
       "   'score': 0.16200625896453857},\n",
       "  {'start': 428,\n",
       "   'end': 689,\n",
       "   'text': 'Using different thymic epithelial cell lines as a model system, we found that transfection of Ep-CAM enhances cell growth and leads to a rapid reorganization of the actin cytoskeleton by inducing the formation of numerous stress fibers and long cell protrusions.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '11702195',\n",
       "   'score': 0.1645474135875702},\n",
       "  {'start': 691,\n",
       "   'end': 837,\n",
       "   'text': 'Finally, the crosslinking of the extracellular domain of a chimeric CD25ec/Ep-CAMic molecule is sufficient to trigger the formation of protrusions.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '11702195',\n",
       "   'score': 0.16455599665641785},\n",
       "  {'start': 839,\n",
       "   'end': 1036,\n",
       "   'text': 'These results suggest that expression of Ep-CAM might balance the organizing capacity of cadherin molecules and may be participating in the formation of a dynamic stromal cell network in the thymus.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '11702195',\n",
       "   'score': 0.19614019989967346},\n",
       "  {'start': 0,\n",
       "   'end': 61,\n",
       "   'text': 'Organization of the cytokeratin network in an epithelial cell.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '12850452',\n",
       "   'score': 0.2511327862739563},\n",
       "  {'start': 0,\n",
       "   'end': 89,\n",
       "   'text': 'The cytoskeleton is a dynamic three-dimensional structure mainly located in the cytoplasm.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '12850452',\n",
       "   'score': 0.24609526991844177},\n",
       "  {'start': 91,\n",
       "   'end': 201,\n",
       "   'text': 'It is involved in many cell functions such as mechanical signal transduction and maintenance of cell integrity.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '12850452',\n",
       "   'score': 0.18173441290855408},\n",
       "  {'start': 203,\n",
       "   'end': 353,\n",
       "   'text': 'Among the three cytoskeletal components, intermediate filaments (the cytokeratin in epithelial cells) are the best candidates for this mechanical role.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '12850452',\n",
       "   'score': 0.1731824278831482},\n",
       "  {'start': 355,\n",
       "   'end': 539,\n",
       "   'text': 'A model of the establishment of the cytokeratin network of an epithelial cell is proposed to study the dependence of its structural organization on extracellular mechanical environment.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '12850452',\n",
       "   'score': 0.2371448278427124},\n",
       "  {'start': 541,\n",
       "   'end': 667,\n",
       "   'text': 'To implicitly describe the latter and its effects on the intracellular domain, we use mechanically regulated protein synthesis.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '12850452',\n",
       "   'score': 0.17581528425216675},\n",
       "  {'start': 669,\n",
       "   'end': 889,\n",
       "   'text': 'Our model is a hybrid of a partial differential equation of parabolic type, governing the evolution of the concentration of cytokeratin, and a set of stochastic differential equations describing the dynamics of filaments.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '12850452',\n",
       "   'score': 0.17147886753082275},\n",
       "  {'start': 891,\n",
       "   'end': 1086,\n",
       "   'text': 'Each filament is described by a stochastic differential equation that reflects both the local interactions with the environment and the non-local interactions via the past history of the filament.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '12850452',\n",
       "   'score': 0.17153769731521606},\n",
       "  {'start': 1088,\n",
       "   'end': 1164,\n",
       "   'text': 'A three-dimensional simulation model is derived from this mathematical model.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '12850452',\n",
       "   'score': 0.23270553350448608},\n",
       "  {'start': 1166,\n",
       "   'end': 1340,\n",
       "   'text': 'This simulation model is then used to obtain examples of cytokeratin network architectures under given mechanical conditions, and to study the influence of several parameters.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '12850452',\n",
       "   'score': 0.20032727718353271},\n",
       "  {'start': 0,\n",
       "   'end': 103,\n",
       "   'text': 'Making connections: insulators organize eukaryotic chromosomes into independent cis-regulatory networks.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '24277632',\n",
       "   'score': 0.1947794258594513},\n",
       "  {'start': 0,\n",
       "   'end': 204,\n",
       "   'text': 'Insulators play a central role in subdividing the chromosome into a series of discrete topologically independent domains and in ensuring that enhancers and silencers contact their appropriate target genes.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24277632',\n",
       "   'score': 0.2537003755569458},\n",
       "  {'start': 206,\n",
       "   'end': 324,\n",
       "   'text': 'In this review we first discuss the general characteristics of insulator elements and their associated protein factors.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24277632',\n",
       "   'score': 0.18847844004631042},\n",
       "  {'start': 326,\n",
       "   'end': 466,\n",
       "   'text': 'A growing collection of insulator proteins have been identified including a family of proteins whose expression is developmentally regulated.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24277632',\n",
       "   'score': 0.16808632016181946},\n",
       "  {'start': 468,\n",
       "   'end': 612,\n",
       "   'text': 'We next consider several unexpected discoveries that require us to completely rethink how insulators function (and how they can best be assayed).',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24277632',\n",
       "   'score': 0.1661384105682373},\n",
       "  {'start': 614,\n",
       "   'end': 800,\n",
       "   'text': 'These discoveries also require a reevaluation of how insulators might restrict or orchestrate (by preventing or promoting) interactions between regulatory elements and their target genes.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24277632',\n",
       "   'score': 0.16746357083320618},\n",
       "  {'start': 802,\n",
       "   'end': 1058,\n",
       "   'text': 'We conclude by connecting these new insights into the mechanisms of insulator action to dynamic changes in the three-dimensional topology of the chromatin fiber and the generation of specific patterns of gene activity during development and differentiation.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '24277632',\n",
       "   'score': 0.23895177245140076},\n",
       "  {'start': 0,\n",
       "   'end': 179,\n",
       "   'text': 'Grainyhead-like 2 regulates epithelial morphogenesis by establishing functional tight junctions through the organization of a molecular network among claudin3, claudin4, and Rab25.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '22696678',\n",
       "   'score': 0.2452041208744049},\n",
       "  {'start': 0,\n",
       "   'end': 212,\n",
       "   'text': 'During development, epithelial progenitors establish intercellular junctions, including tight junctions (TJs), and form three-dimensional (3D) tissue structures, which are often associated with luminal structures.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '22696678',\n",
       "   'score': 0.23981311917304993},\n",
       "  {'start': 214,\n",
       "   'end': 364,\n",
       "   'text': 'Here we identify grainyhead-like 2 (Grhl2) as a transcription factor that regulates the size of luminal space surrounded by polarized epithelial cells.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '22696678',\n",
       "   'score': 0.16978731751441956},\n",
       "  {'start': 366,\n",
       "   'end': 510,\n",
       "   'text': 'We show that HPPL, a liver progenitor cell line, transfected with Grhl2 cDNA forms remarkably larger cysts than the control cells in 3D cultures.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '22696678',\n",
       "   'score': 0.21562442183494568},\n",
       "  {'start': 512,\n",
       "   'end': 638,\n",
       "   'text': 'We find that Grhl2 up-regulates claudin (Cldn) 3 and Cldn4, and their functions are necessary for the formation of large cysts.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '22696678',\n",
       "   'score': 0.17221930623054504},\n",
       "  {'start': 640,\n",
       "   'end': 696,\n",
       "   'text': 'Overexpression of Cldn3 alone induces the cyst expansion.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '22696678',\n",
       "   'score': 0.1708110272884369},\n",
       "  {'start': 698,\n",
       "   'end': 793,\n",
       "   'text': 'In contrast, expression of Cldn4 alone does not induce expansion, as it is not localized at TJs.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '22696678',\n",
       "   'score': 0.1740792691707611},\n",
       "  {'start': 795,\n",
       "   'end': 916,\n",
       "   'text': 'Of interest, Rab25, another Grhl2 target, not only increases the Cldn4 protein, but also enhances its localization at TJs.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '22696678',\n",
       "   'score': 0.1804715096950531},\n",
       "  {'start': 918,\n",
       "   'end': 1141,\n",
       "   'text': 'Taken together, the results indicate that Grhl2 regulates epithelial morphogenesis through transcriptional up-regulation of Cldn3 and Cldn4, as well as of Rab25, which increases the Cldn4 protein and its localization at TJs.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '22696678',\n",
       "   'score': 0.20861735939979553},\n",
       "  {'start': 1143,\n",
       "   'end': 1238,\n",
       "   'text': 'The results reveal a molecular network regulating epithelial lumen formation organized by Grhl2.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '22696678',\n",
       "   'score': 0.217901349067688},\n",
       "  {'start': 0,\n",
       "   'end': 120,\n",
       "   'text': 'Structure function relations in PDZ-domain-containing proteins: Implications for protein networks in cellular signalling.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '29485124',\n",
       "   'score': 0.21302258968353271},\n",
       "  {'start': 0,\n",
       "   'end': 143,\n",
       "   'text': 'Protein scaffolds as essential backbones for organization of supramolecular signalling complexes are a recurrent theme in several model systems.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '29485124',\n",
       "   'score': 0.22705060243606567},\n",
       "  {'start': 145,\n",
       "   'end': 256,\n",
       "   'text': 'Scaffold proteins preferentially employ linear peptide binding motifs for recruiting their interaction partners.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '29485124',\n",
       "   'score': 0.16901996731758118},\n",
       "  {'start': 258,\n",
       "   'end': 404,\n",
       "   'text': 'PDZ domains are one of the more commonly encountered peptide binding domains in several proteins including those involved in scaffolding functions.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '29485124',\n",
       "   'score': 0.18271741271018982},\n",
       "  {'start': 406,\n",
       "   'end': 584,\n",
       "   'text': 'This domain is known for its promiscuity both in terms of ligand selection, mode of interaction with its ligands as well as its association with other protein interaction domains.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '29485124',\n",
       "   'score': 0.18577778339385986},\n",
       "  {'start': 586,\n",
       "   'end': 681,\n",
       "   'text': 'PDZ domains are subject to several means of regulations by virtue of their functional diversity.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '29485124',\n",
       "   'score': 0.17047181725502014},\n",
       "  {'start': 683,\n",
       "   'end': 834,\n",
       "   'text': 'Additionally, the PDZ domains are refractive to the effect of mutations and maintain their three-dimensional architecture under extreme mutational load.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '29485124',\n",
       "   'score': 0.24068427085876465},\n",
       "  {'start': 836,\n",
       "   'end': 964,\n",
       "   'text': 'The biochemical and biophysical basis for this selectivity as well as promiscuity has been investigated and reviewed extensively.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '29485124',\n",
       "   'score': 0.17380714416503906},\n",
       "  {'start': 966,\n",
       "   'end': 1154,\n",
       "   'text': 'The present review focuses on the plasticity inherent in PDZ domains and its implications for modular organization as well as evolution of cellular signalling pathways in higher eukaryotes.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '29485124',\n",
       "   'score': 0.21782386302947998},\n",
       "  {'start': 0,\n",
       "   'end': 89,\n",
       "   'text': 'Interstitial flow regulates in\\xa0vitro three-dimensional self-organized brain micro-vessels.',\n",
       "   'is_title': True,\n",
       "   'doc_id': '32988592',\n",
       "   'score': 0.23765534162521362},\n",
       "  {'start': 0,\n",
       "   'end': 122,\n",
       "   'text': 'Cell culture under medium flow has been shown to favor human brain microvascular endothelial cells function and maturation.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '32988592',\n",
       "   'score': 0.17177963256835938},\n",
       "  {'start': 124,\n",
       "   'end': 453,\n",
       "   'text': 'Here a three-dimensional in\\xa0vitro model of the human brain microvasculature, comprising brain microvascular endothelial cells but also astrocytes, pericytes and a collagen type I microfiber - fibrin based matrix, was cultured under continuous medium flow in a pressure driven microphysiological system (10\\xa0kPa, in 60-30\\xa0s cycles).',\n",
       "   'is_title': False,\n",
       "   'doc_id': '32988592',\n",
       "   'score': 0.24118348956108093},\n",
       "  {'start': 455,\n",
       "   'end': 528,\n",
       "   'text': 'The cells self-organized in micro-vessels perpendicular to the shear flow.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '32988592',\n",
       "   'score': 0.17417865991592407},\n",
       "  {'start': 530,\n",
       "   'end': 827,\n",
       "   'text': 'Comparison with static culture showed that the resulting interstitial flow enhanced a more defined micro-vasculature network, with slightly more numerous lumens, and a higher expression of transporters, carriers and tight junction genes and proteins, essential to the blood-brain barrier functions.',\n",
       "   'is_title': False,\n",
       "   'doc_id': '32988592',\n",
       "   'score': 0.20121333003044128}]}"
      ]
     },
     "execution_count": 52,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "run[0]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 54,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[{'start': 0,\n",
       "  'end': 114,\n",
       "  'text': 'KLF4 is involved in the organization and regulation of pluripotency-associated three-dimensional enhancer networks.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '31548608',\n",
       "  'score': 0.7081087231636047},\n",
       " {'start': 0,\n",
       "  'end': 259,\n",
       "  'text': 'Cell fate transitions are accompanied by global transcriptional, epigenetic and topological changes driven by transcription factors, as is exemplified by reprogramming somatic cells to pluripotent stem cells through the expression of OCT4, KLF4, SOX2 and cMYC.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31548608',\n",
       "  'score': 0.15228056907653809},\n",
       " {'start': 392,\n",
       "  'end': 669,\n",
       "  'text': 'Here, using KLF4 as a paradigm, we provide a transcription-factor-centric view of chromatin reorganization and its association with three-dimensional enhancer rewiring and transcriptional changes during the reprogramming of mouse embryonic fibroblasts to pluripotent stem cells.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31548608',\n",
       "  'score': 0.39797115325927734},\n",
       " {'start': 671,\n",
       "  'end': 967,\n",
       "  'text': 'Inducible depletion of KLF factors in PSCs caused a genome-wide decrease in enhancer connectivity, whereas disruption of individual KLF4 binding sites within pluripotent-stem-cell-specific enhancers was sufficient to impair enhancer-promoter contacts and reduce the expression of associated genes.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31548608',\n",
       "  'score': 0.25990283489227295},\n",
       " {'start': 0,\n",
       "  'end': 60,\n",
       "  'text': 'Pluripotency in 3D: genome organization in pluripotent cells.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '23199754',\n",
       "  'score': 0.40288621187210083},\n",
       " {'start': 0,\n",
       "  'end': 134,\n",
       "  'text': 'Cells face the challenge of storing two meters of DNA in the three-dimensional (3D) space of the nucleus that spans only a few microns.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '23199754',\n",
       "  'score': 0.13167187571525574},\n",
       " {'start': 136,\n",
       "  'end': 408,\n",
       "  'text': 'The nuclear organization that is required to overcome this challenge must allow for the accessibility of the gene regulatory machinery to the DNA and, in the case of embryonic stem cells (ESCs), for the transcriptional and epigenetic changes that accompany differentiation.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '23199754',\n",
       "  'score': 0.2666993737220764},\n",
       " {'start': 410,\n",
       "  'end': 532,\n",
       "  'text': 'Recent technological advances have allowed for the mapping of genome organization at an unprecedented resolution and scale.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '23199754',\n",
       "  'score': 0.11587250232696533},\n",
       " {'start': 534,\n",
       "  'end': 713,\n",
       "  'text': 'These breakthroughs have led to a deluge of new data, and a sophisticated understanding of the relationship between gene regulation and 3D genome organization is beginning to form.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '23199754',\n",
       "  'score': 0.1825897991657257},\n",
       " {'start': 715,\n",
       "  'end': 1076,\n",
       "  'text': 'In this review we summarize some of the recent findings illuminating the 3D structure of the eukaryotic genome, as well as the relationship between genome topology and function from the level of whole chromosomes to enhancer-promoter loops with a focus on features affecting genome organization in ESCs and changes in nuclear organization during differentiation.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '23199754',\n",
       "  'score': 0.49772197008132935},\n",
       " {'start': 0,\n",
       "  'end': 108,\n",
       "  'text': 'Genomic Rewiring of SOX2 Chromatin Interaction Network during Differentiation of ESCs to Postmitotic Neurons.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '32553182',\n",
       "  'score': 0.29788559675216675},\n",
       " {'start': 0,\n",
       "  'end': 128,\n",
       "  'text': 'Cellular differentiation requires dramatic changes in chromatin organization, transcriptional regulation, and protein production.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '32553182',\n",
       "  'score': 0.14769163727760315},\n",
       " {'start': 130,\n",
       "  'end': 467,\n",
       "  'text': 'To understand the regulatory connections between these processes, we generated proteomic, transcriptomic, and chromatin accessibility data during differentiation of mouse embryonic stem cells (ESCs) into postmitotic neurons and found extensive associations between different molecular layers within and across differentiation time points.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '32553182',\n",
       "  'score': 0.2127506136894226},\n",
       " {'start': 469,\n",
       "  'end': 690,\n",
       "  'text': 'We observed that SOX2, as a regulator of pluripotency and neuronal genes, redistributes from pluripotency enhancers to neuronal promoters during differentiation, likely driven by changes in its protein interaction network.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '32553182',\n",
       "  'score': 0.5031372904777527},\n",
       " {'start': 692,\n",
       "  'end': 912,\n",
       "  'text': 'We identified ATRX as a major SOX2 partner in neurons, whose co-localization correlated with an increase in active enhancer marks and increased expression of nearby genes, which we experimentally confirmed for three loci.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '32553182',\n",
       "  'score': 0.15763896703720093},\n",
       " {'start': 914,\n",
       "  'end': 1144,\n",
       "  'text': 'Collectively, our data provide key insights into the regulatory transformation of SOX2 during neuronal differentiation, and we highlight the significance of multi-omic approaches in understanding gene regulation in complex systems.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '32553182',\n",
       "  'score': 0.10985422134399414},\n",
       " {'start': 0,\n",
       "  'end': 128,\n",
       "  'text': 'Sall4 controls differentiation of pluripotent cells independently of the Nucleosome Remodelling and Deacetylation (NuRD) complex.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '27471257',\n",
       "  'score': 0.24639463424682617},\n",
       " {'start': 0,\n",
       "  'end': 116,\n",
       "  'text': 'Sall4 is an essential transcription factor for early mammalian development and is frequently overexpressed in cancer.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '27471257',\n",
       "  'score': 0.0972265899181366},\n",
       " {'start': 118,\n",
       "  'end': 275,\n",
       "  'text': 'Although it is reported to play an important role in embryonic stem cell (ESC) self-renewal, whether it is an essential pluripotency factor has been disputed.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '27471257',\n",
       "  'score': 0.21590715646743774},\n",
       " {'start': 277,\n",
       "  'end': 343,\n",
       "  'text': 'Here, we show that Sall4 is dispensable for mouse ESC pluripotency.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '27471257',\n",
       "  'score': 0.23391544818878174},\n",
       " {'start': 345,\n",
       "  'end': 545,\n",
       "  'text': 'Sall4 is an enhancer-binding protein that prevents precocious activation of the neural gene expression programme in ESCs but is not required for maintenance of the pluripotency gene regulatory network.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '27471257',\n",
       "  'score': 0.4841834604740143},\n",
       " {'start': 547,\n",
       "  'end': 824,\n",
       "  'text': 'Although a proportion of Sall4 protein physically associates with the Nucleosome Remodelling and Deacetylase (NuRD) complex, Sall4 neither recruits NuRD to chromatin nor influences transcription via NuRD; rather, free Sall4 protein regulates transcription independently of NuRD.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '27471257',\n",
       "  'score': 0.10221952199935913},\n",
       " {'start': 826,\n",
       "  'end': 1033,\n",
       "  'text': 'We propose a model whereby enhancer binding by Sall4 and other pluripotency-associated transcription factors is responsible for maintaining the balance between transcriptional programmes in pluripotent cells.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '27471257',\n",
       "  'score': 0.44995638728141785},\n",
       " {'start': 0,\n",
       "  'end': 188,\n",
       "  'text': 'DNA methylation of the Oct4A enhancers in embryonal carcinoma cells after etoposide treatment is associated with alternative splicing and altered pluripotency in reversibly senescent cells.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '29372665',\n",
       "  'score': 0.36239680647850037},\n",
       " {'start': 0,\n",
       "  'end': 140,\n",
       "  'text': 'The epigenetic mechanisms underlying chemoresistance in cancer cells resulting from drug-induced reversible senescence are poorly understood.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '29372665',\n",
       "  'score': 0.09827521443367004},\n",
       " {'start': 142,\n",
       "  'end': 394,\n",
       "  'text': 'Chemoresistant ESC-like embryonal carcinoma PA1 cells treated with etoposide (ETO) were previously found to undergo prolonged G2 arrest with transient p53-dependent upregulation of opposing fate regulators, p21CIP1 (senescence) and OCT4A (self-renewal).',\n",
       "  'is_title': False,\n",
       "  'doc_id': '29372665',\n",
       "  'score': 0.09764865040779114},\n",
       " {'start': 396,\n",
       "  'end': 554,\n",
       "  'text': 'Here we report on the analysis of the DNA methylation state of the distal enhancer (DE) and proximal enhancer (PE) of the Oct4A gene during this dual response.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '29372665',\n",
       "  'score': 0.15672793984413147},\n",
       " {'start': 556,\n",
       "  'end': 696,\n",
       "  'text': 'When compared to non-treated controls the methylation level increased from 1.3% to 12.5% and from 3% to 19.4%, in the DE and PE respectively.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '29372665',\n",
       "  'score': 0.09951764345169067},\n",
       " {'start': 698,\n",
       "  'end': 804,\n",
       "  'text': 'It included CpG and non-CpG methylation, which was not chaotic but presented two patterns in each enhancer.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '29372665',\n",
       "  'score': 0.16123738884925842},\n",
       " {'start': 806,\n",
       "  'end': 1071,\n",
       "  'text': 'Discorrelating with methylation of enhancers, the transcription of Oct4A increased, however, a strong expression of the splicing form Oct4B was also induced, along with down-regulation of the Oct4A partners of in the pluripotency/self-renewal network Sox2 and Lin28.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '29372665',\n",
       "  'score': 0.4819476008415222},\n",
       " {'start': 1073,\n",
       "  'end': 1154,\n",
       "  'text': 'WB demonstrated disjoining of the OCT4A protein from the chromatin-bound fraction.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '29372665',\n",
       "  'score': 0.10324835777282715},\n",
       " {'start': 1156,\n",
       "  'end': 1285,\n",
       "  'text': 'In survival clones, methylation of the DE was considerably erased, while some remnant of methylation of the PE was still observed.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '29372665',\n",
       "  'score': 0.09949445724487305},\n",
       " {'start': 1287,\n",
       "  'end': 1473,\n",
       "  'text': 'The alternative splicing for Oct4B was reduced, Oct4A level insignificantly decreased, while the expression of Sox2 and Lin28 recovered, all three became proportionally above the control.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '29372665',\n",
       "  'score': 0.10344123840332031},\n",
       " {'start': 1475,\n",
       "  'end': 1739,\n",
       "  'text': 'These findings indicate the involvement of the transient patterned methylation of the Oct4A enhancers and alternative splicing in the adaptive regulation of cell fate choice during the p53-dependant dual state of reversible senescence in ESC-like cancer stem cells.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '29372665',\n",
       "  'score': 0.18598923087120056},\n",
       " {'start': 0,\n",
       "  'end': 74,\n",
       "  'text': 'Role of ALKBH1 in the Core Transcriptional Network of Embryonic Stem Cells.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '26765775',\n",
       "  'score': 0.1245814859867096},\n",
       " {'start': 0,\n",
       "  'end': 171,\n",
       "  'text': 'BACKGROUND/AIMS\\nALKBH1, an AlkB homologue in the 2-oxoglutarate and Fe2+ dependent hydroxylase family, is a histone dioxygenase that removes methyl groups from histone H2A.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '26765775',\n",
       "  'score': 0.10476219654083252},\n",
       " {'start': 173,\n",
       "  'end': 282,\n",
       "  'text': 'Studies of transgenic mice lacking Alkbh1 reveal that most Alkbh1-/- embryos die during embryonic development.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '26765775',\n",
       "  'score': 0.09883207082748413},\n",
       " {'start': 284,\n",
       "  'end': 537,\n",
       "  'text': 'Embryonic stem cells (ESCs) derived from these mice have prolonged expression of pluripotency markers and delayed induction of genes involved in neural differentiation, indicating that ALKBH1 is involved in regulation of pluripotency and differentiation.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '26765775',\n",
       "  'score': 0.3430096507072449},\n",
       " {'start': 539,\n",
       "  'end': 624,\n",
       "  'text': 'The aim of this study was to further investigate the role ALKBH1 in early development.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '26765775',\n",
       "  'score': 0.10196158289909363},\n",
       " {'start': 628,\n",
       "  'end': 839,\n",
       "  'text': 'METHODS\\nDouble-filter methods for nitrocellulose-filter binding, dot blot, enzyme-linked immunosorbent assay (ELISA), immonocytochemistry, cell culture and differentiation of mouse ESCs, Co-IP and miRNA analysis.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '26765775',\n",
       "  'score': 0.2339635193347931},\n",
       " {'start': 843,\n",
       "  'end': 1032,\n",
       "  'text': 'RESULTS\\nWe found that SOX2 and NANOG bind the ALKBH1 promoter, and we identified protein-protein interactions between ALKBH1 and these core transcription factors of the pluripotency network.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '26765775',\n",
       "  'score': 0.4504778981208801},\n",
       " {'start': 1034,\n",
       "  'end': 1201,\n",
       "  'text': 'Furthermore, lack of ALKBH1 affected the expression of developmentally important miRNAs, which are involved in the regulation of NANOG, SOX2 and neural differentiation.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '26765775',\n",
       "  'score': 0.31529873609542847},\n",
       " {'start': 1205,\n",
       "  'end': 1382,\n",
       "  'text': 'CONCLUSION\\nOur results suggest that ALKBH1 interacts with the core transcriptional pluripotency network of ESCs and is involved in regulation of pluripotency and differentiation.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '26765775',\n",
       "  'score': 0.4017235338687897},\n",
       " {'start': 0,\n",
       "  'end': 128,\n",
       "  'text': 'Deep transcriptome profiling of mammalian stem cells supports a regulatory role for retrotransposons in pluripotency maintenance.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '24777452',\n",
       "  'score': 0.24576252698898315},\n",
       " {'start': 0,\n",
       "  'end': 190,\n",
       "  'text': 'The importance of microRNAs and long noncoding RNAs in the regulation of pluripotency has been documented; however, the noncoding components of stem cell gene networks remain largely unknown.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24777452',\n",
       "  'score': 0.38223984837532043},\n",
       " {'start': 192,\n",
       "  'end': 340,\n",
       "  'text': 'Here we investigate the role of noncoding RNAs in the pluripotent state, with particular emphasis on nuclear and retrotransposon-derived transcripts.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24777452',\n",
       "  'score': 0.25615397095680237},\n",
       " {'start': 342,\n",
       "  'end': 527,\n",
       "  'text': 'We have performed deep profiling of the nuclear and cytoplasmic transcriptomes of human and mouse stem cells, identifying a class of previously undetected stem cell-specific transcripts.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24777452',\n",
       "  'score': 0.10201424360275269},\n",
       " {'start': 529,\n",
       "  'end': 668,\n",
       "  'text': 'We show that long terminal repeat (LTR)-derived transcripts contribute extensively to the complexity of the stem cell nuclear transcriptome.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24777452',\n",
       "  'score': 0.10034841299057007},\n",
       " {'start': 670,\n",
       "  'end': 800,\n",
       "  'text': 'Some LTR-derived transcripts are associated with enhancer regions and are likely to be involved in the maintenance of pluripotency.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24777452',\n",
       "  'score': 0.45888859033584595},\n",
       " {'start': 0,\n",
       "  'end': 121,\n",
       "  'text': 'KLF4 protein stability regulated by interaction with pluripotency transcription factors overrides transcriptional control.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '31221664',\n",
       "  'score': 0.2726683020591736},\n",
       " {'start': 0,\n",
       "  'end': 113,\n",
       "  'text': 'Embryonic stem (ES) cells are regulated by a network of transcription factors that maintain the pluripotent state.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31221664',\n",
       "  'score': 0.2761201858520508},\n",
       " {'start': 115,\n",
       "  'end': 218,\n",
       "  'text': 'Differentiation relies on down-regulation of pluripotency transcription factors disrupting this network.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31221664',\n",
       "  'score': 0.3756549060344696},\n",
       " {'start': 220,\n",
       "  'end': 597,\n",
       "  'text': 'While investigating transcriptional regulation of the pluripotency transcription factor Kruppel-like factor 4 (Klf4), we observed that homozygous deletion of distal enhancers caused a 17-fold decrease in Klf4 transcript but surprisingly decreased protein levels by less than twofold, indicating that posttranscriptional control of KLF4 protein overrides transcriptional control.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31221664',\n",
       "  'score': 0.4509628713130951},\n",
       " {'start': 599,\n",
       "  'end': 698,\n",
       "  'text': 'The lack of sensitivity of KLF4 to transcription is due to high protein stability (half-life >24 h).',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31221664',\n",
       "  'score': 0.11053293943405151},\n",
       " {'start': 700,\n",
       "  'end': 1020,\n",
       "  'text': 'This stability is context-dependent and is disrupted during differentiation, as evidenced by a shift to a half-life of <2 h. KLF4 protein stability is maintained through interaction with other pluripotency transcription factors (NANOG, SOX2, and STAT3) that together facilitate association of KLF4 with RNA polymerase II.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31221664',\n",
       "  'score': 0.268865168094635},\n",
       " {'start': 1022,\n",
       "  'end': 1131,\n",
       "  'text': 'In addition, the KLF4 DNA-binding and transactivation domains are required for optimal KLF4 protein stability.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31221664',\n",
       "  'score': 0.11121487617492676},\n",
       " {'start': 1133,\n",
       "  'end': 1310,\n",
       "  'text': 'Posttranslational modification of KLF4 destabilizes the protein as cells exit the pluripotent state, and mutations that prevent this destabilization also prevent differentiation.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31221664',\n",
       "  'score': 0.2682322859764099},\n",
       " {'start': 1312,\n",
       "  'end': 1569,\n",
       "  'text': 'These data indicate that the core pluripotency transcription factors are integrated by posttranslational mechanisms to maintain the pluripotent state and identify mutations that increase KLF4 protein stability while maintaining transcription factor function.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31221664',\n",
       "  'score': 0.23399245738983154},\n",
       " {'start': 0,\n",
       "  'end': 171,\n",
       "  'text': 'Three-Dimensional Retinal Organoids Facilitate the Investigation of Retinal Ganglion Cell Development, Organization and Neurite Outgrowth from Human Pluripotent Stem Cells.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '30266927',\n",
       "  'score': 0.46932312846183777},\n",
       " {'start': 0,\n",
       "  'end': 235,\n",
       "  'text': 'Retinal organoids are three-dimensional structures derived from human pluripotent stem cells (hPSCs) which recapitulate the spatial and temporal differentiation of the retina, serving as effective in vitro models of retinal development.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '30266927',\n",
       "  'score': 0.31366026401519775},\n",
       " {'start': 237,\n",
       "  'end': 376,\n",
       "  'text': 'However, a lack of emphasis has been placed upon the development and organization of retinal ganglion cells (RGCs) within retinal organoids.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '30266927',\n",
       "  'score': 0.13068628311157227},\n",
       " {'start': 378,\n",
       "  'end': 623,\n",
       "  'text': 'Thus, initial efforts were made to characterize RGC differentiation throughout early stages of organoid development, with a clearly defined RGC layer developing in a temporally-appropriate manner expressing a complement of RGC-associated markers.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '30266927',\n",
       "  'score': 0.10587576031684875},\n",
       " {'start': 625,\n",
       "  'end': 804,\n",
       "  'text': 'Beyond studies of RGC development, retinal organoids may also prove useful for cellular replacement in which extensive axonal outgrowth is necessary to reach post-synaptic targets.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '30266927',\n",
       "  'score': 0.10580062866210938},\n",
       " {'start': 806,\n",
       "  'end': 973,\n",
       "  'text': 'Organoid-derived RGCs could help to elucidate factors promoting axonal outgrowth, thereby identifying approaches to circumvent a formidable obstacle to RGC replacement.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '30266927',\n",
       "  'score': 0.10016009211540222},\n",
       " {'start': 975,\n",
       "  'end': 1137,\n",
       "  'text': 'As such, additional efforts demonstrated significant enhancement of neurite outgrowth through modulation of both substrate composition and growth factor signaling.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '30266927',\n",
       "  'score': 0.10293316841125488},\n",
       " {'start': 1139,\n",
       "  'end': 1280,\n",
       "  'text': 'Additionally, organoid-derived RGCs exhibited diverse phenotypes, extending elaborate growth cones and expressing numerous guidance receptors.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '30266927',\n",
       "  'score': 0.10172152519226074},\n",
       " {'start': 1282,\n",
       "  'end': 1540,\n",
       "  'text': 'Collectively, these results establish retinal organoids as a valuable tool for studies of RGC development, and demonstrate the utility of organoid-derived RGCs as an effective platform to study factors influencing neurite outgrowth from organoid-derived RGCs.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '30266927',\n",
       "  'score': 0.1039593517780304},\n",
       " {'start': 0,\n",
       "  'end': 137,\n",
       "  'text': 'The Role of RNA Polymerase II Contiguity and Long-Range Interactions in the Regulation of Gene Expression in Human Pluripotent Stem Cells.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '30863449',\n",
       "  'score': 0.30869418382644653},\n",
       " {'start': 0,\n",
       "  'end': 189,\n",
       "  'text': 'The eukaryotic nucleus is a highly complex structure that carries out multiple functions primarily needed for gene expression, and among them, transcription seems to be the most fundamental.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '30863449',\n",
       "  'score': 0.10597193241119385},\n",
       " {'start': 191,\n",
       "  'end': 412,\n",
       "  'text': 'Diverse approaches have demonstrated that transcription takes place at discrete sites known as transcription factories, wherein RNA polymerase II (RNAP II) is attached to the factory and immobilized while transcribing DNA.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '30863449',\n",
       "  'score': 0.10560482740402222},\n",
       " {'start': 414,\n",
       "  'end': 598,\n",
       "  'text': 'It has been proposed that transcription factories promote chromatin loop formation, creating long-range interactions in which relatively distant genes can be transcribed simultaneously.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '30863449',\n",
       "  'score': 0.10410195589065552},\n",
       " {'start': 600,\n",
       "  'end': 909,\n",
       "  'text': 'In this study, we examined long-range interactions between the POU5F1 gene and genes previously identified as being POU5F1 enhancer-interacting, namely, CDYL, TLE2, RARG, and MSX1 (all involved in transcriptional regulation), in human pluripotent stem cells (hPSCs) and their early differentiated counterparts.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '30863449',\n",
       "  'score': 0.4484449028968811},\n",
       " {'start': 911,\n",
       "  'end': 1038,\n",
       "  'text': 'As a control gene, RUNX1 was used, which is expressed during hematopoietic differentiation and not associated with pluripotency.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '30863449',\n",
       "  'score': 0.26265716552734375},\n",
       " {'start': 1040,\n",
       "  'end': 1313,\n",
       "  'text': 'To reveal how these long-range interactions between POU5F1 and the selected genes change with the onset of differentiation and upon RNAP II inhibition, we performed three-dimensional fluorescence in situ hybridization (3D-FISH) followed by computational simulation analysis.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '30863449',\n",
       "  'score': 0.14999845623970032},\n",
       " {'start': 1315,\n",
       "  'end': 1519,\n",
       "  'text': 'Our analysis showed that the numbers of long-range interactions between specific genes decrease during differentiation, suggesting that the transcription of monitored genes is associated with pluripotency.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '30863449',\n",
       "  'score': 0.25966376066207886},\n",
       " {'start': 1521,\n",
       "  'end': 1640,\n",
       "  'text': 'In addition, we showed that upon inhibition of RNAP II, long-range associations do not disintegrate and remain constant.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '30863449',\n",
       "  'score': 0.10640057921409607},\n",
       " {'start': 1642,\n",
       "  'end': 1833,\n",
       "  'text': 'We also analyzed the distance distributions of these genes in the context of their positions in the nucleus and revealed that they tend to have similar patterns resembling normal distribution.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '30863449',\n",
       "  'score': 0.10745862126350403},\n",
       " {'start': 1835,\n",
       "  'end': 1945,\n",
       "  'text': 'Furthermore, we compared data created in vitro and in silico to assess the biological relevance of our results.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '30863449',\n",
       "  'score': 0.10668230056762695},\n",
       " {'start': 0,\n",
       "  'end': 49,\n",
       "  'text': 'DNA methylation and the core pluripotency network.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '32562758',\n",
       "  'score': 0.28680557012557983},\n",
       " {'start': 0,\n",
       "  'end': 87,\n",
       "  'text': 'From the onset of fertilization, the genome undergoes cell division and differentiation.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '32562758',\n",
       "  'score': 0.10561397671699524},\n",
       " {'start': 89,\n",
       "  'end': 227,\n",
       "  'text': 'All of these developmental transitions and differentiation processes include cell-specific signatures and gradual changes of the epigenome.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '32562758',\n",
       "  'score': 0.10598498582839966},\n",
       " {'start': 229,\n",
       "  'end': 380,\n",
       "  'text': 'Understanding what keeps stem cells in the pluripotent state and what leads to differentiation are fascinating and biomedically highly important issues.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '32562758',\n",
       "  'score': 0.25422459840774536},\n",
       " {'start': 382,\n",
       "  'end': 490,\n",
       "  'text': 'Numerous studies have identified genes, proteins, microRNAs and small molecules that exert essential effects.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '32562758',\n",
       "  'score': 0.10321137309074402},\n",
       " {'start': 492,\n",
       "  'end': 611,\n",
       "  'text': 'Notably, there exists a core pluripotency network that consists of several transcription factors and accessory proteins.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '32562758',\n",
       "  'score': 0.2775859832763672},\n",
       " {'start': 613,\n",
       "  'end': 719,\n",
       "  'text': 'Three eminent transcription factors, OCT4, SOX2 and NANOG, serve as hubs in this core pluripotency network.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '32562758',\n",
       "  'score': 0.32389211654663086},\n",
       " {'start': 721,\n",
       "  'end': 924,\n",
       "  'text': 'They bind to the enhancer regions of their target genes and modulate, among others, the expression levels of genes that are associated with Gene Ontology terms related to differentiation and self-renewal.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '32562758',\n",
       "  'score': 0.17643365263938904},\n",
       " {'start': 926,\n",
       "  'end': 1027,\n",
       "  'text': 'Also, much has been learned about the epigenetic rewiring processes during these changes of cell fate.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '32562758',\n",
       "  'score': 0.10387203097343445},\n",
       " {'start': 1029,\n",
       "  'end': 1106,\n",
       "  'text': 'For example, DNA methylation dynamics is pivotal during embryonic development.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '32562758',\n",
       "  'score': 0.10323336720466614},\n",
       " {'start': 1108,\n",
       "  'end': 1449,\n",
       "  'text': 'The main goal of this review is to highlight an intricate interplay of (a) DNA methyltransferases controlling the expression levels of core pluripotency factors by modulation of the DNA methylation levels in their enhancer regions, and of (b) the core pluripotency factors controlling the transcriptional regulation of DNA methyltransferases.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '32562758',\n",
       "  'score': 0.4437042474746704},\n",
       " {'start': 1451,\n",
       "  'end': 1601,\n",
       "  'text': 'We discuss these processes both at the global level and in atomistic detail based on information from structural studies and from computer simulations.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '32562758',\n",
       "  'score': 0.10622531175613403},\n",
       " {'start': 0,\n",
       "  'end': 130,\n",
       "  'text': 'Long-Range Enhancer Interactions Are Prevalent in Mouse Embryonic Stem Cells and Are Reorganized upon Pluripotent State Transition.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '29514091',\n",
       "  'score': 0.3677775263786316},\n",
       " {'start': 0,\n",
       "  'end': 148,\n",
       "  'text': 'Transcriptional enhancers, including super-enhancers (SEs), form physical interactions with promoters to regulate cell-type-specific gene expression.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '29514091',\n",
       "  'score': 0.16429558396339417},\n",
       " {'start': 150,\n",
       "  'end': 332,\n",
       "  'text': 'SEs are characterized by high transcription factor occupancy and large domains of active chromatin, and they are commonly assigned to target promoters using computational predictions.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '29514091',\n",
       "  'score': 0.10517621040344238},\n",
       " {'start': 334,\n",
       "  'end': 481,\n",
       "  'text': 'How promoter-SE interactions change upon cell state transitions, and whether transcription factors maintain SE interactions, have not been reported.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '29514091',\n",
       "  'score': 0.16661447286605835},\n",
       " {'start': 483,\n",
       "  'end': 600,\n",
       "  'text': 'Here, we used promoter-capture Hi-C to identify promoters that interact with SEs in mouse embryonic stem cells (ESCs).',\n",
       "  'is_title': False,\n",
       "  'doc_id': '29514091',\n",
       "  'score': 0.32012271881103516},\n",
       " {'start': 602,\n",
       "  'end': 779,\n",
       "  'text': 'We found that SEs form complex, spatial networks in which individual SEs contact multiple promoters, and a rewiring of promoter-SE interactions occurs between pluripotent states.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '29514091',\n",
       "  'score': 0.4290946424007416},\n",
       " {'start': 781,\n",
       "  'end': 922,\n",
       "  'text': 'We also show that long-range promoter-SE interactions are more prevalent in ESCs than in epiblast stem cells (EpiSCs) or Nanog-deficient ESCs.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '29514091',\n",
       "  'score': 0.3685828745365143},\n",
       " {'start': 924,\n",
       "  'end': 1132,\n",
       "  'text': 'We conclude that SEs form cell-type-specific interaction networks that are partly dependent on core transcription factors, thereby providing insights into the gene regulatory organization of pluripotent cells.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '29514091',\n",
       "  'score': 0.3570590019226074},\n",
       " {'start': 0,\n",
       "  'end': 81,\n",
       "  'text': 'Gender Differences in Global but Not Targeted Demethylation in iPSC Reprogramming.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '28147265',\n",
       "  'score': 0.10552334785461426},\n",
       " {'start': 0,\n",
       "  'end': 188,\n",
       "  'text': 'Global DNA demethylation is an integral part of reprogramming processes in\\xa0vivo and in\\xa0vitro, but whether it occurs in the derivation of induced pluripotent stem cells (iPSCs) is not known.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28147265',\n",
       "  'score': 0.2805059552192688},\n",
       " {'start': 190,\n",
       "  'end': 345,\n",
       "  'text': 'Here, we show that iPSC reprogramming involves both global and targeted demethylation, which are separable mechanistically and by their biological outcomes.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28147265',\n",
       "  'score': 0.10664835572242737},\n",
       " {'start': 347,\n",
       "  'end': 483,\n",
       "  'text': 'Cells at intermediate-late stages of reprogramming undergo transient genome-wide demethylation, which is more pronounced in female cells.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28147265',\n",
       "  'score': 0.11006316542625427},\n",
       " {'start': 485,\n",
       "  'end': 692,\n",
       "  'text': 'Global demethylation requires activation-induced cytidine deaminase (AID)-mediated downregulation of UHRF1 protein, and abolishing demethylation leaves thousands of hypermethylated regions in the iPSC genome.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28147265',\n",
       "  'score': 0.1180504858493805},\n",
       " {'start': 694,\n",
       "  'end': 916,\n",
       "  'text': 'Independently of AID and global demethylation, regulatory regions, particularly ESC enhancers and super-enhancers, are specifically targeted for hypomethylation in association with transcription of the pluripotency network.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28147265',\n",
       "  'score': 0.4567837417125702},\n",
       " {'start': 918,\n",
       "  'end': 1145,\n",
       "  'text': 'Our results show that global and targeted DNA demethylation are conserved and\\xa0distinct reprogramming processes, presumably because of their respective roles in epigenetic memory erasure and in the establishment of cell identity.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28147265',\n",
       "  'score': 0.11022317409515381},\n",
       " {'start': 0,\n",
       "  'end': 73,\n",
       "  'text': 'A transcribed enhancer dictates mesendoderm specification in pluripotency.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '29180618',\n",
       "  'score': 0.40541893243789673},\n",
       " {'start': 0,\n",
       "  'end': 108,\n",
       "  'text': 'Enhancers and long noncoding RNAs (lncRNAs) are key determinants of lineage specification during development.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '29180618',\n",
       "  'score': 0.1812184453010559},\n",
       " {'start': 110,\n",
       "  'end': 240,\n",
       "  'text': 'Here, we evaluate remodeling of the enhancer landscape and modulation of the lncRNA transcriptome during mesendoderm specification.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '29180618',\n",
       "  'score': 0.18145865201950073},\n",
       " {'start': 242,\n",
       "  'end': 485,\n",
       "  'text': 'We sort mesendodermal progenitors from differentiating embryonic stem cells (ESCs) according to Eomes expression, and find that enhancer usage is coordinated with mesendoderm-specific expression of key lineage-determining transcription factors.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '29180618',\n",
       "  'score': 0.3563392162322998},\n",
       " {'start': 487,\n",
       "  'end': 556,\n",
       "  'text': 'Many of these enhancers are associated with the expression of lncRNAs.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '29180618',\n",
       "  'score': 0.18528583645820618},\n",
       " {'start': 558,\n",
       "  'end': 763,\n",
       "  'text': 'Examination of ESC-specific enhancers interacting in three-dimensional space with mesendoderm-specifying transcription factor loci identifies MesEndoderm Transcriptional Enhancer Organizing Region (Meteor).',\n",
       "  'is_title': False,\n",
       "  'doc_id': '29180618',\n",
       "  'score': 0.26574966311454773},\n",
       " {'start': 765,\n",
       "  'end': 985,\n",
       "  'text': 'Genetic and epigenetic manipulation of the Meteor enhancer reveal its indispensable role during mesendoderm specification and subsequent cardiogenic differentiation via transcription-independent and -dependent mechanisms.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '29180618',\n",
       "  'score': 0.1795782446861267},\n",
       " {'start': 987,\n",
       "  'end': 1084,\n",
       "  'text': 'Interestingly, Meteor-deleted ESCs are epigenetically redirected towards neuroectodermal lineages.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '29180618',\n",
       "  'score': 0.2370816469192505},\n",
       " {'start': 1086,\n",
       "  'end': 1288,\n",
       "  'text': 'Loci, topologically associating a transcribed enhancer and its cognate protein coding gene, appear to represent therefore a class of genomic elements controlling developmental competence in pluripotency.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '29180618',\n",
       "  'score': 0.4206128716468811},\n",
       " {'start': 0,\n",
       "  'end': 75,\n",
       "  'text': 'The interactomes of POU5F1 and SOX2 enhancers in human embryonic stem cells.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '23549118',\n",
       "  'score': 0.18976709246635437},\n",
       " {'start': 0,\n",
       "  'end': 152,\n",
       "  'text': 'The genes POU5F1 and SOX2 are critical for pluripotency and reprogramming, yet the chromosomal organization around these genes remains poorly understood.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '23549118',\n",
       "  'score': 0.31996476650238037},\n",
       " {'start': 154,\n",
       "  'end': 308,\n",
       "  'text': 'We assayed long-range chromosomal interactions on putative enhancers of POU5F1 and SOX2 genes in human embryonic stem cells (hESCs) using 4C-Seq technique.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '23549118',\n",
       "  'score': 0.39866986870765686},\n",
       " {'start': 310,\n",
       "  'end': 413,\n",
       "  'text': 'We discovered that their frequent interacting regions mainly overlap with early DNA replication domains.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '23549118',\n",
       "  'score': 0.10612431168556213},\n",
       " {'start': 415,\n",
       "  'end': 520,\n",
       "  'text': 'The interactomes are associated with active histone marks and enriched with 5-hydroxymethylcytosine sites.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '23549118',\n",
       "  'score': 0.11248305439949036},\n",
       " {'start': 522,\n",
       "  'end': 586,\n",
       "  'text': 'In hESCs, genes within the interactomes have elevated expression.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '23549118',\n",
       "  'score': 0.25428152084350586},\n",
       " {'start': 588,\n",
       "  'end': 675,\n",
       "  'text': 'Additionally, some genes associated with the POU5F1 enhancer contribute to pluripotency.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '23549118',\n",
       "  'score': 0.41570454835891724},\n",
       " {'start': 677,\n",
       "  'end': 816,\n",
       "  'text': 'Binding sites for multiple DNA binding proteins, including ATF3, CTCF, GABPA, JUND, NANOG, RAD21 and YY1, are enriched in both interactomes.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '23549118',\n",
       "  'score': 0.25257909297943115},\n",
       " {'start': 818,\n",
       "  'end': 958,\n",
       "  'text': 'The RARG locus, frequently interacting with the POU5F1 locus, has abundant RAD21 binding sites co-localized with other protein binding sites.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '23549118',\n",
       "  'score': 0.11369895935058594},\n",
       " {'start': 960,\n",
       "  'end': 1075,\n",
       "  'text': 'Thus the interactomes of these two pluripotency genes could be an important part of the regulatory network in hESCs.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '23549118',\n",
       "  'score': 0.3358808755874634},\n",
       " {'start': 0,\n",
       "  'end': 121,\n",
       "  'text': 'Coactivators p300 and CBP maintain the identity of mouse embryonic stem cells by mediating long-range chromatin structure.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '24648406',\n",
       "  'score': 0.11095663905143738},\n",
       " {'start': 0,\n",
       "  'end': 189,\n",
       "  'text': 'Master transcription factors Oct4, Sox2, and Nanog are required to maintain the pluripotency and self-renewal of embryonic stem cells (ESCs) by regulating a specific transcriptional network.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24648406',\n",
       "  'score': 0.3523593842983246},\n",
       " {'start': 191,\n",
       "  'end': 440,\n",
       "  'text': 'A few other transcription factors have been shown to be important in ESCs by interacting with these master transcription factors; however, little is known about the transcriptional mechanisms regulated by coregulators (coactivators and corepressors).',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24648406',\n",
       "  'score': 0.25966501235961914},\n",
       " {'start': 442,\n",
       "  'end': 565,\n",
       "  'text': 'In this study, we examined the function of two highly homologous coactivators, p300 and CREB-binding protein (CBP), in ESCs.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24648406',\n",
       "  'score': 0.3012901842594147},\n",
       " {'start': 567,\n",
       "  'end': 673,\n",
       "  'text': 'We find that these two coactivators play redundant roles in maintaining the undifferentiated state of ESCs.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24648406',\n",
       "  'score': 0.2795216143131256},\n",
       " {'start': 675,\n",
       "  'end': 882,\n",
       "  'text': 'They are recruited by Nanog through physical interaction to Nanog binding loci, mediating the formation of long-range chromatin looping structures, which is essential to maintain ESC-specific gene expression.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24648406',\n",
       "  'score': 0.24124464392662048},\n",
       " {'start': 884,\n",
       "  'end': 1219,\n",
       "  'text': 'Further functional studies reveal that the p300/CBP binding looping fragments contain enhancer activities, suggesting that the formation of p300/CBP-mediated looping structures may recruit distal enhancers to create a concentration of factors for the transcription activation of genes that are involved in self-renewal and pluripotency.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24648406',\n",
       "  'score': 0.4191829264163971},\n",
       " {'start': 1221,\n",
       "  'end': 1485,\n",
       "  'text': 'Overall, these results provide a total new insight into the transcriptional regulation mechanism of coactivators p300 and CBP in ESCs, which is important in maintaining self-renewal and pluripotency, by mediating the formation of higher order chromosome structures.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24648406',\n",
       "  'score': 0.2935965359210968},\n",
       " {'start': 0,\n",
       "  'end': 92,\n",
       "  'text': 'The three-dimensional genome: regulating gene expression during pluripotency and development.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '29042476',\n",
       "  'score': 0.4218951165676117},\n",
       " {'start': 0,\n",
       "  'end': 234,\n",
       "  'text': 'The precise expression of genes in time and space during embryogenesis is largely influenced by communication between enhancers and promoters, which is propagated and governed by the physical proximity of these elements in the nucleus.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '29042476',\n",
       "  'score': 0.18712130188941956},\n",
       " {'start': 236,\n",
       "  'end': 412,\n",
       "  'text': 'Here, we review how chromatin domains organize the genome by guiding enhancers to their target genes thereby preventing non-specific interactions with other neighboring regions.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '29042476',\n",
       "  'score': 0.16716381907463074},\n",
       " {'start': 414,\n",
       "  'end': 610,\n",
       "  'text': 'We also discuss the dynamics of chromatin interactions between enhancers and promoters, as well as the consequent changes in gene expression, that occur in pluripotent cells and during development.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '29042476',\n",
       "  'score': 0.3728809356689453},\n",
       " {'start': 612,\n",
       "  'end': 875,\n",
       "  'text': 'Finally, we evaluate how genomic changes such as deletions, inversions and duplications affect 3D chromatin configuration overall and lead to ectopic enhancer-promoter contacts, and thus gene misexpression, which can contribute to abnormal development and disease.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '29042476',\n",
       "  'score': 0.22391435503959656},\n",
       " {'start': 0,\n",
       "  'end': 110,\n",
       "  'text': 'Human Induced Pluripotent Stem Cell-Derived Endothelial Cells for Three-Dimensional Microphysiological Systems.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '28622076',\n",
       "  'score': 0.36644959449768066},\n",
       " {'start': 0,\n",
       "  'end': 165,\n",
       "  'text': 'Microphysiological systems (MPS), or \"organ-on-a-chip\" platforms, aim to recapitulate in vivo physiology using small-scale in vitro tissue models of human physiology.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28622076',\n",
       "  'score': 0.11068680882453918},\n",
       " {'start': 167,\n",
       "  'end': 314,\n",
       "  'text': 'While significant efforts have been made to create vascularized tissues, most reports utilize primary endothelial cells that hinder reproducibility.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28622076',\n",
       "  'score': 0.10793313384056091},\n",
       " {'start': 316,\n",
       "  'end': 483,\n",
       "  'text': 'In this study, we report the use of human induced pluripotent stem cell-derived endothelial cells (iPS-ECs) in developing three-dimensional (3D) microvascular networks.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28622076',\n",
       "  'score': 0.4245969355106354},\n",
       " {'start': 485,\n",
       "  'end': 610,\n",
       "  'text': 'We established a CDH5-mCherry reporter iPS cell line, which expresses the vascular endothelial (VE)-cadherin fused to mCherry.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28622076',\n",
       "  'score': 0.1105991005897522},\n",
       " {'start': 612,\n",
       "  'end': 896,\n",
       "  'text': 'The iPS-ECs demonstrate physiological functions characteristic of primary endothelial cells in a series of in vitro assays, including permeability, response to shear stress, and the expression of endothelial markers (CD31, von Willibrand factor, and endothelial nitric oxide synthase).',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28622076',\n",
       "  'score': 0.1143372654914856},\n",
       " {'start': 898,\n",
       "  'end': 1018,\n",
       "  'text': 'The iPS-ECs form stable, perfusable microvessels over the course of 14 days when cultured within 3D microfluidic devices.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28622076',\n",
       "  'score': 0.14495491981506348},\n",
       " {'start': 1020,\n",
       "  'end': 1125,\n",
       "  'text': 'We also demonstrate that inhibition of TGF-β signaling improves vascular network formation by the iPS-ECs.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28622076',\n",
       "  'score': 0.12957125902175903},\n",
       " {'start': 1127,\n",
       "  'end': 1304,\n",
       "  'text': 'We conclude that iPS-ECs can be a source of endothelial cells in MPS providing opportunities for human disease modeling and improving the reproducibility of 3D vascular networks.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28622076',\n",
       "  'score': 0.17117729783058167},\n",
       " {'start': 0,\n",
       "  'end': 149,\n",
       "  'text': 'Profiling the long noncoding RNA interaction network in the regulatory elements of target genes by chromatin in situ reverse transcription sequencing.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '31315906',\n",
       "  'score': 0.13434267044067383},\n",
       " {'start': 0,\n",
       "  'end': 134,\n",
       "  'text': 'Long noncoding RNAs (lncRNAs) can regulate the activity of target genes by participating in the organization of chromatin architecture.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31315906',\n",
       "  'score': 0.1428370177745819},\n",
       " {'start': 136,\n",
       "  'end': 399,\n",
       "  'text': 'We have devised a \"chromatin-RNA in situ reverse transcription sequencing\" (CRIST-seq) approach to profile the lncRNA interaction network in gene regulatory elements by combining the simplicity of RNA biotin labeling with the specificity of the CRISPR/Cas9 system.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31315906',\n",
       "  'score': 0.13011813163757324},\n",
       " {'start': 401,\n",
       "  'end': 612,\n",
       "  'text': 'Using gene-specific gRNAs, we describe a pluripotency-specific lncRNA interacting network in the promoters of Sox2 and Pou5f1, two critical stem cell factors that are required for the maintenance of pluripotency.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31315906',\n",
       "  'score': 0.3435637354850769},\n",
       " {'start': 614,\n",
       "  'end': 713,\n",
       "  'text': 'The promoter-interacting lncRNAs were specifically activated during reprogramming into pluripotency.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31315906',\n",
       "  'score': 0.4100116491317749},\n",
       " {'start': 715,\n",
       "  'end': 789,\n",
       "  'text': 'Knockdown of these lncRNAs caused the stem cells to exit from pluripotency.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31315906',\n",
       "  'score': 0.27840617299079895},\n",
       " {'start': 791,\n",
       "  'end': 967,\n",
       "  'text': 'In contrast, overexpression of the pluripotency-associated lncRNA activated the promoters of core stem cell factor genes and enhanced fibroblast reprogramming into pluripotency.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31315906',\n",
       "  'score': 0.32683682441711426},\n",
       " {'start': 969,\n",
       "  'end': 1151,\n",
       "  'text': 'These CRIST-seq data suggest that the Sox2 and Pou5f1 promoters are organized within a unique lncRNA interaction network that determines the fate of pluripotency during reprogramming.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31315906',\n",
       "  'score': 0.3261426091194153},\n",
       " {'start': 1153,\n",
       "  'end': 1260,\n",
       "  'text': 'This CRIST approach may be broadly used to map lncRNA interaction networks at target loci across the genome.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31315906',\n",
       "  'score': 0.12960880994796753},\n",
       " {'start': 0,\n",
       "  'end': 76,\n",
       "  'text': 'Autophagy regulates homeostasis of pluripotency-associated proteins in hESCs.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '24170349',\n",
       "  'score': 0.3461417555809021},\n",
       " {'start': 0,\n",
       "  'end': 163,\n",
       "  'text': 'The pluripotency of embryonic stem cells (ESCs) is maintained by intracellular networks of many pluripotency-associated (PA) proteins such as OCT4, SOX2, and NANOG.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24170349',\n",
       "  'score': 0.4057402014732361},\n",
       " {'start': 165,\n",
       "  'end': 269,\n",
       "  'text': 'However, the mechanisms underlying the regulation of protein homeostasis for pluripotency remain elusive.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24170349',\n",
       "  'score': 0.3350452780723572},\n",
       " {'start': 271,\n",
       "  'end': 429,\n",
       "  'text': 'Here, we first demonstrate that autophagy acts together with the ubiquitin-proteasome system (UPS) to modulate the levels of PA proteins in human ESCs (hESCs).',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24170349',\n",
       "  'score': 0.247096985578537},\n",
       " {'start': 431,\n",
       "  'end': 519,\n",
       "  'text': 'Autophagy inhibition impaired the pluripotency despite increment of PA proteins in hESCs.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24170349',\n",
       "  'score': 0.27625173330307007},\n",
       " {'start': 521,\n",
       "  'end': 614,\n",
       "  'text': 'Immunogold-electron microscopy confirmed localization of OCT4 molecules within autophagosomes.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24170349',\n",
       "  'score': 0.29148149490356445},\n",
       " {'start': 616,\n",
       "  'end': 723,\n",
       "  'text': 'Also, knockdown of LC3 expression led to accumulation of PA proteins and reduction of pluripotency in hESCs.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24170349',\n",
       "  'score': 0.3075977861881256},\n",
       " {'start': 725,\n",
       "  'end': 824,\n",
       "  'text': 'Interestingly, autophagy and the UPS showed differential kinetics in the degradation of PA proteins.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24170349',\n",
       "  'score': 0.11589106917381287},\n",
       " {'start': 826,\n",
       "  'end': 997,\n",
       "  'text': 'Autophagy inhibition caused enhanced accumulation of both cytoplasmic and nuclear PA proteins, whereas the UPS inhibition led to preferentially degrade nuclear PA proteins.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24170349',\n",
       "  'score': 0.11371856927871704},\n",
       " {'start': 999,\n",
       "  'end': 1139,\n",
       "  'text': 'Our findings suggest that autophagy modulates homeostasis of PA proteins, providing a new insight in the regulation of pluripotency in hESCs.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24170349',\n",
       "  'score': 0.35039037466049194},\n",
       " {'start': 0,\n",
       "  'end': 100,\n",
       "  'text': 'Hmga2 is necessary for Otx2-dependent exit of embryonic stem cells from the pluripotent ground state.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '27036552',\n",
       "  'score': 0.2798989415168762},\n",
       " {'start': 0,\n",
       "  'end': 257,\n",
       "  'text': \"BACKGROUND\\nA crucial event in the differentiation of mouse embryonic stem cells (ESCs) is the exit from the pluripotent ground state that leads to the acquisition of the 'primed' pluripotent phenotype, characteristic of the epiblast-like stem cells (EpiLCs).\",\n",
       "  'is_title': False,\n",
       "  'doc_id': '27036552',\n",
       "  'score': 0.2700392007827759},\n",
       " {'start': 259,\n",
       "  'end': 333,\n",
       "  'text': 'The transcription factors Oct4 and Otx2 play a key role in this phenomenon.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '27036552',\n",
       "  'score': 0.27299344539642334},\n",
       " {'start': 335,\n",
       "  'end': 517,\n",
       "  'text': 'In particular, Otx2 pioneers and activates new enhancers, which are silent in ESCs and which control the transcription of genes responsible for the acquisition of the EpiLC phenotype.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '27036552',\n",
       "  'score': 0.4107334613800049},\n",
       " {'start': 519,\n",
       "  'end': 660,\n",
       "  'text': 'An important point that remains to be addressed is the mechanism through which Otx2 engages the new enhancers and stably associates with them.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '27036552',\n",
       "  'score': 0.17143890261650085},\n",
       " {'start': 662,\n",
       "  'end': 850,\n",
       "  'text': 'Hmga2 is a member of the high-mobility group family of proteins, non-histone components of chromatin whose expression is high during embryogenesis and becomes low or undetectable in adults.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '27036552',\n",
       "  'score': 0.11868694424629211},\n",
       " {'start': 852,\n",
       "  'end': 952,\n",
       "  'text': 'Its high expression during embryogenesis suggests that Hmga2 fulfills important roles in development.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '27036552',\n",
       "  'score': 0.1116006076335907},\n",
       " {'start': 956,\n",
       "  'end': 1055,\n",
       "  'text': 'RESULTS\\nHere, we demonstrate that Hmga2 accumulates soon after the induction of ESC differentiation.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '27036552',\n",
       "  'score': 0.11226573586463928},\n",
       " {'start': 1057,\n",
       "  'end': 1169,\n",
       "  'text': 'Its suppression hampers the exit of ESCs from the pluripotent ground state and their differentiation into EpiLCs.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '27036552',\n",
       "  'score': 0.2754358649253845},\n",
       " {'start': 1171,\n",
       "  'end': 1290,\n",
       "  'text': 'Mechanistically, Hmga2 controls the differentiation process by cooperating with Otx2 in the pioneering of new enhancers.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '27036552',\n",
       "  'score': 0.18212589621543884},\n",
       " {'start': 1292,\n",
       "  'end': 1430,\n",
       "  'text': 'In Hmga2 null induced pluripotent stem cells we observe that Otx2 fails to regulate its target genes upon the induction of differentiation.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '27036552',\n",
       "  'score': 0.27476048469543457},\n",
       " {'start': 1432,\n",
       "  'end': 1640,\n",
       "  'text': 'Hmga2 associates to Otx2-bound loci in EpiLCs, and in Hmga2 KO cells Otx2 is unable to engage and activate the new enhancers, thus indicating that Hmga2 is required for the binding of Otx2 to its cis-elements.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '27036552',\n",
       "  'score': 0.1892279088497162},\n",
       " {'start': 1642,\n",
       "  'end': 1797,\n",
       "  'text': 'We find that this mechanism also operates on the Hmga2 gene, which is one of the targets of Otx2, thus indicating the existence of a positive feedback loop.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '27036552',\n",
       "  'score': 0.11528739333152771},\n",
       " {'start': 1801,\n",
       "  'end': 1915,\n",
       "  'text': 'CONCLUSIONS\\nOur findings reveal a novel mechanism necessary for the exit of ESCs from the pluripotent ground state.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '27036552',\n",
       "  'score': 0.2726737856864929},\n",
       " {'start': 1917,\n",
       "  'end': 2063,\n",
       "  'text': 'Upon the induction of ESC differentiation, Otx2 alone or in combination with Oct4 engages new enhancers, which are silent in undifferentiated ESCs.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '27036552',\n",
       "  'score': 0.36384570598602295},\n",
       " {'start': 2065,\n",
       "  'end': 2230,\n",
       "  'text': 'The Hmga2 gene is activated by Otx2 and Hmga2 protein binds to the enhancers targeted by Otx2, thus facilitating the engagement and/or the stable association of Otx2.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '27036552',\n",
       "  'score': 0.20535767078399658},\n",
       " {'start': 2232,\n",
       "  'end': 2384,\n",
       "  'text': 'Therefore, our results demonstrate that Hmga2 is a key element of the regulatory network that governs the exit of ESCs from the pluripotent ground state.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '27036552',\n",
       "  'score': 0.3137584924697876},\n",
       " {'start': 0,\n",
       "  'end': 118,\n",
       "  'text': 'Three-dimensional cultured liver-on-a-Chip with mature hepatocyte-like cells derived from human pluripotent stem cells.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '31304567',\n",
       "  'score': 0.42351478338241577},\n",
       " {'start': 0,\n",
       "  'end': 118,\n",
       "  'text': 'Liver-on-a-Chip technology holds considerable potential for applications in drug screening and chemical-safety testing.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31304567',\n",
       "  'score': 0.11414846777915955},\n",
       " {'start': 120,\n",
       "  'end': 352,\n",
       "  'text': 'To establish such platforms, functional hepatocytes are required; however, primary hepatocytes are commonly used, despite problems involving donor limitations, lot-to-lot variation, and unsatisfactory two-dimensional culture methods.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31304567',\n",
       "  'score': 0.15832966566085815},\n",
       " {'start': 354,\n",
       "  'end': 597,\n",
       "  'text': 'Although human pluripotent stem cells (hPSCs) may represent a strong alternative contender to address the aforementioned issues, remaining technological challenges include the robust, highly efficient production of high-purity hepatic clusters.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31304567',\n",
       "  'score': 0.23991736769676208},\n",
       " {'start': 599,\n",
       "  'end': 723,\n",
       "  'text': 'In addition, current Liver-on-a-Chip platforms are relatively complicated and not applicable for high-throughput experiments.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31304567',\n",
       "  'score': 0.11474660038948059},\n",
       " {'start': 725,\n",
       "  'end': 848,\n",
       "  'text': 'Here, we develop a very simple Liver-on-a-Chip platform with mature and functional hepatocyte-like cells derived from hPSCs.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31304567',\n",
       "  'score': 0.11265712976455688},\n",
       " {'start': 850,\n",
       "  'end': 1182,\n",
       "  'text': 'To establish a method for hepatic differentiation of hPSCs, cells were first treated by inhibiting the phosphoinositide 3-kinase- and Rho-associated protein kinase-signaling pathways to stop self-renewal and improve survival, respectively, which enabled the formation of a well-defined endoderm and facilitated hepatocyte commitment.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31304567',\n",
       "  'score': 0.1260453462600708},\n",
       " {'start': 1184,\n",
       "  'end': 1433,\n",
       "  'text': 'Next, a simple microfluidic device was used to create a three-dimensional (3D) culture environment that enhanced the maturation and function of hepatocyte-like cells by increasing the expression of both hepatic maturation markers and cytochrome P450.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31304567',\n",
       "  'score': 0.16047409176826477},\n",
       " {'start': 1435,\n",
       "  'end': 1605,\n",
       "  'text': 'Finally, we confirmed improvements in hepatic functions, such as drug uptake/excretion capabilities, in >90% of 3D-matured hepatocyte-like cells by indocyanin green assay.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31304567',\n",
       "  'score': 0.1466076672077179},\n",
       " {'start': 1607,\n",
       "  'end': 1803,\n",
       "  'text': 'These results indicated that the incorporation of hPSC-derived hepatocytes on our Liver-on-a-Chip platform may serve to enhance the processes involved in drug screening and chemical-safety testing.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31304567',\n",
       "  'score': 0.12223529815673828},\n",
       " {'start': 0,\n",
       "  'end': 114,\n",
       "  'text': 'The pluripotency factor Nanog regulates pericentromeric heterochromatin organization in mouse embryonic stem cells.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '27125671',\n",
       "  'score': 0.36134791374206543},\n",
       " {'start': 0,\n",
       "  'end': 85,\n",
       "  'text': 'An open and decondensed chromatin organization is a defining property of pluripotency.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '27125671',\n",
       "  'score': 0.3744581341743469},\n",
       " {'start': 87,\n",
       "  'end': 261,\n",
       "  'text': 'Several epigenetic regulators have been implicated in maintaining an open chromatin organization, but how these processes are connected to the pluripotency network is unknown.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '27125671',\n",
       "  'score': 0.37988194823265076},\n",
       " {'start': 263,\n",
       "  'end': 460,\n",
       "  'text': 'Here, we identified a new role for the transcription factor NANOG as a key regulator connecting the pluripotency network with constitutive heterochromatin organization in mouse embryonic stem cells.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '27125671',\n",
       "  'score': 0.35795527696609497},\n",
       " {'start': 462,\n",
       "  'end': 555,\n",
       "  'text': 'Deletion of Nanog leads to chromatin compaction and the remodeling of heterochromatin domains.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '27125671',\n",
       "  'score': 0.3050897717475891},\n",
       " {'start': 557,\n",
       "  'end': 643,\n",
       "  'text': 'Forced expression of NANOG in epiblast stem cells is sufficient to decompact chromatin.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '27125671',\n",
       "  'score': 0.2971142828464508},\n",
       " {'start': 645,\n",
       "  'end': 944,\n",
       "  'text': 'NANOG associates with satellite repeats within heterochromatin domains, contributing to an architecture characterized by highly dispersed chromatin fibers, low levels of H3K9me3, and high major satellite transcription, and the strong transactivation domain of NANOG is required for this organization.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '27125671',\n",
       "  'score': 0.3552335798740387},\n",
       " {'start': 946,\n",
       "  'end': 1205,\n",
       "  'text': 'The heterochromatin-associated protein SALL1 is a direct cofactor for NANOG, and loss of Sall1 recapitulates the Nanog-null phenotype, but the loss of Sall1 can be circumvented through direct recruitment of the NANOG transactivation domain to major satellites.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '27125671',\n",
       "  'score': 0.2978903651237488},\n",
       " {'start': 1207,\n",
       "  'end': 1432,\n",
       "  'text': 'These results establish a direct connection between the pluripotency network and chromatin organization and emphasize that maintaining an open heterochromatin architecture is a highly regulated process in embryonic stem cells.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '27125671',\n",
       "  'score': 0.39368849992752075},\n",
       " {'start': 0,\n",
       "  'end': 180,\n",
       "  'text': 'Tuning Forkhead Box D3 overexpression to promote specific osteogenic differentiation of human embryonic stem cells while reducing pluripotency in a three-dimensional culture system.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '30350469',\n",
       "  'score': 0.39350152015686035},\n",
       " {'start': 0,\n",
       "  'end': 190,\n",
       "  'text': 'Clinical use of human embryonic stem cells (hESCs) in bone regeneration applications requires that their osteogenic differentiation be highly controllable as well as time- and cost-effective.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '30350469',\n",
       "  'score': 0.2443694770336151},\n",
       " {'start': 192,\n",
       "  'end': 540,\n",
       "  'text': 'The main goals of the current work were thus (a) to assess whether overexpression of pluripotency regulator Forkhead Box D3 (FOXD3) can enhance the osteogenic commitment of hESCs seeded in three-dimensional (3D) scaffolds and (b) to evaluate if the degree of FOXD3 overexpression regulates the strength and specificity of hESC osteogenic commitment.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '30350469',\n",
       "  'score': 0.3956056535243988},\n",
       " {'start': 542,\n",
       "  'end': 707,\n",
       "  'text': 'In conducting these studies, an interpenetrating hydrogel network consisting of poly(ethylene glycol) diacrylate and collagen I was utilized as a 3D culture platform.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '30350469',\n",
       "  'score': 0.18305140733718872},\n",
       " {'start': 709,\n",
       "  'end': 940,\n",
       "  'text': 'Expression of osteogenic, chondrogenic, pluripotency, and germ layer markers by encapsulated hESCs was measured after 2\\xa0weeks of culture in osteogenic medium in the presence or absence doxycycline-induced FOXD3 transgene expression.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '30350469',\n",
       "  'score': 0.3054622411727905},\n",
       " {'start': 942,\n",
       "  'end': 1183,\n",
       "  'text': 'Towards the first goal, FOXD3 overexpression initiated 24\\xa0hr prior to hESC encapsulation, relative to unstimulated controls, resulted in upregulation of osteogenic markers and enhanced calcium deposition, without promoting off-target effects.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '30350469',\n",
       "  'score': 0.11817711591720581},\n",
       " {'start': 1185,\n",
       "  'end': 1374,\n",
       "  'text': 'However, when initiation of FOXD3 overexpression was increased from 24 to 48\\xa0hr prior to encapsulation, hESC osteogenic commitment was not further enhanced and off-target effects were noted.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '30350469',\n",
       "  'score': 0.1143130362033844},\n",
       " {'start': 1376,\n",
       "  'end': 1621,\n",
       "  'text': 'Specifically, relative to 24-hr prestimulation, initiation of FOXD3 overexpression 48\\xa0hr prior to encapsulation yielded increased expression of pluripotency markers while reducing mesodermal but increasing endodermal germ layer marker expression.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '30350469',\n",
       "  'score': 0.29410868883132935},\n",
       " {'start': 1623,\n",
       "  'end': 1796,\n",
       "  'text': 'Combined, the current results indicate that the controlled overexpression of FOXD3 warrants further investigation as a mechanism to guide enhanced hESC osteogenic commitment.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '30350469',\n",
       "  'score': 0.11418530344963074},\n",
       " {'start': 0,\n",
       "  'end': 82,\n",
       "  'text': 'Multilineage communication regulates human liver bud development from pluripotency.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '28614297',\n",
       "  'score': 0.2864711880683899},\n",
       " {'start': 0,\n",
       "  'end': 133,\n",
       "  'text': 'Conventional two-dimensional differentiation from pluripotency fails to recapitulate cell interactions occurring during organogenesis.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28614297',\n",
       "  'score': 0.39616042375564575},\n",
       " {'start': 135,\n",
       "  'end': 275,\n",
       "  'text': 'Three-dimensional organoids generate complex organ-like tissues; however, it is unclear how heterotypic interactions affect lineage identity.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28614297',\n",
       "  'score': 0.16951856017112732},\n",
       " {'start': 277,\n",
       "  'end': 411,\n",
       "  'text': 'Here we use single-cell RNA sequencing to reconstruct hepatocyte-like lineage progression from pluripotency in two-dimensional culture.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28614297',\n",
       "  'score': 0.3961751461029053},\n",
       " {'start': 413,\n",
       "  'end': 590,\n",
       "  'text': 'We then derive three-dimensional liver bud organoids by reconstituting hepatic, stromal, and endothelial interactions, and deconstruct heterogeneity during liver bud development.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28614297',\n",
       "  'score': 0.1645735800266266},\n",
       " {'start': 592,\n",
       "  'end': 745,\n",
       "  'text': 'We find that liver bud hepatoblasts diverge from the two-dimensional lineage, and express epithelial migration signatures characteristic of organ budding.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28614297',\n",
       "  'score': 0.1639573574066162},\n",
       " {'start': 747,\n",
       "  'end': 954,\n",
       "  'text': 'We benchmark three-dimensional liver buds against fetal and adult human liver single-cell RNA sequencing data, and find a striking correspondence between the three-dimensional liver bud and fetal liver cells.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28614297',\n",
       "  'score': 0.16483372449874878},\n",
       " {'start': 956,\n",
       "  'end': 1216,\n",
       "  'text': 'We use a receptor-ligand pairing analysis and a high-throughput inhibitor assay to interrogate signalling in liver buds, and show that vascular endothelial growth factor (VEGF) crosstalk potentiates endothelial network formation and hepatoblast differentiation.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28614297',\n",
       "  'score': 0.13404500484466553},\n",
       " {'start': 1218,\n",
       "  'end': 1385,\n",
       "  'text': 'Our molecular dissection reveals interlineage communication regulating organoid development, and illuminates previously inaccessible aspects of human liver development.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28614297',\n",
       "  'score': 0.1296645998954773},\n",
       " {'start': 0,\n",
       "  'end': 118,\n",
       "  'text': 'Identification of novel proteins differentially expressed in pluripotent embryonic stem cells and differentiated cells.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '26399336',\n",
       "  'score': 0.271136999130249},\n",
       " {'start': 0,\n",
       "  'end': 84,\n",
       "  'text': 'Mammalian pluripotent stem cells possess properties of self-renewal and pluripotency.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '26399336',\n",
       "  'score': 0.2874419093132019},\n",
       " {'start': 86,\n",
       "  'end': 257,\n",
       "  'text': 'These abilities are maintained by the strict regulation of pluripotent stem cell-specific transcription factor network and unique properties of chromatin in the stem cells.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '26399336',\n",
       "  'score': 0.3912312984466553},\n",
       " {'start': 259,\n",
       "  'end': 475,\n",
       "  'text': 'Although these major signaling pathways robustly control the characteristics of stem cells, other regulatory factors, such as metabolic pathways, are also known to modulate stem cell proliferation and differentiation.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '26399336',\n",
       "  'score': 0.11688193678855896},\n",
       " {'start': 477,\n",
       "  'end': 622,\n",
       "  'text': 'In this study, we fractionated protein samples from mouse embryonic stem (ES) cells cultured with or without the leukemia inhibitory factor (LIF).',\n",
       "  'is_title': False,\n",
       "  'doc_id': '26399336',\n",
       "  'score': 0.12306603789329529},\n",
       " {'start': 624,\n",
       "  'end': 717,\n",
       "  'text': 'Protein expression was quantified by 2-dimensional differential gel electrophoresis (2D-DIGE).',\n",
       "  'is_title': False,\n",
       "  'doc_id': '26399336',\n",
       "  'score': 0.16299530863761902},\n",
       " {'start': 719,\n",
       "  'end': 849,\n",
       "  'text': 'In total, 44 proteins were identified as being differentially expressed in the pluripotent stem cells and the differentiated cells.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '26399336',\n",
       "  'score': 0.29089367389678955},\n",
       " {'start': 851,\n",
       "  'end': 1023,\n",
       "  'text': 'Surprisingly, half of the identified proteins were the proteins localized in mitochondria, which supply cellular energy and regulate cell cycle, development, and cell death.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '26399336',\n",
       "  'score': 0.12032938003540039},\n",
       " {'start': 1025,\n",
       "  'end': 1132,\n",
       "  'text': 'Some of these identified proteins are involved in the metabolic function and the regulation of pluripotency.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '26399336',\n",
       "  'score': 0.38898223638534546},\n",
       " {'start': 1134,\n",
       "  'end': 1256,\n",
       "  'text': 'Further analysis of the identified proteins could provide new information for the manipulation of pluripotency in ES cells.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '26399336',\n",
       "  'score': 0.31028246879577637},\n",
       " {'start': 0,\n",
       "  'end': 78,\n",
       "  'text': 'Generation of a Three-Dimensional Kidney Structure from Pluripotent Stem Cells.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '28361318',\n",
       "  'score': 0.39970484375953674},\n",
       " {'start': 0,\n",
       "  'end': 143,\n",
       "  'text': 'The kidney is a vital organ that has an important role in the maintenance of homeostasis by fluid volume regulation and waste product excretion.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28361318',\n",
       "  'score': 0.13902965188026428},\n",
       " {'start': 145,\n",
       "  'end': 233,\n",
       "  'text': 'This role cannot be performed without the three-dimensional (3D) structure of the kidney.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28361318',\n",
       "  'score': 0.1641119420528412},\n",
       " {'start': 235,\n",
       "  'end': 390,\n",
       "  'text': 'Therefore, it is important to generate the 3D structure of the kidney when inducing functional kidney tissue or the whole organ from pluripotent stem cells.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28361318',\n",
       "  'score': 0.3040994703769684},\n",
       " {'start': 392,\n",
       "  'end': 549,\n",
       "  'text': 'In this chapter, we describe the detailed methods to induce kidney progenitor cells from pluripotent stem cells, which are based on embryological development.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28361318',\n",
       "  'score': 0.26723235845565796},\n",
       " {'start': 551,\n",
       "  'end': 653,\n",
       "  'text': 'We also provide a method to generate 3D kidney tissue with vascularized glomeruli upon transplantation.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28361318',\n",
       "  'score': 0.1442633867263794},\n",
       " {'start': 0,\n",
       "  'end': 91,\n",
       "  'text': 'Three-dimensional nanofibrillar surfaces promote self-renewal in mouse embryonic stem cells.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '16150921',\n",
       "  'score': 0.1721205711364746},\n",
       " {'start': 0,\n",
       "  'end': 167,\n",
       "  'text': 'The regulation of mouse embryonic stem cell (mESC) fate is controlled by the interplay of signaling networks that either promote self-renewal or induce differentiation.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '16150921',\n",
       "  'score': 0.16373324394226074},\n",
       " {'start': 169,\n",
       "  'end': 301,\n",
       "  'text': 'Leukemia inhibitory factor (LIF) is a cytokine that is required for stem cell renewal in mouse but not in human embryonic stem cells.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '16150921',\n",
       "  'score': 0.11700060963630676},\n",
       " {'start': 303,\n",
       "  'end': 638,\n",
       "  'text': 'However, feeder layers of embryonic fibroblasts are capable of inducing stem cell renewal in both cell types, suggesting that the self-renewal signaling pathways may also be promoted by other triggers, such as alternative cytokines and/or chemical or physical properties of the extracellular matrix (ECM) secreted by feeder fibroblasts.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '16150921',\n",
       "  'score': 0.11970141530036926},\n",
       " {'start': 640,\n",
       "  'end': 794,\n",
       "  'text': 'We have recently used a synthetic polyamide matrix (Ultra-Web) whose three-dimensional (3D) nanofibrillar organization resembles the ECM/basement membrane.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '16150921',\n",
       "  'score': 0.19708123803138733},\n",
       " {'start': 796,\n",
       "  'end': 1006,\n",
       "  'text': 'Growth of mESCs on this nanofibrillar surface greatly enhanced proliferation and self-renewal in comparison with growth on tissue culture surfaces without nanofibers, despite the presence of LIF in both systems.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '16150921',\n",
       "  'score': 0.29000887274742126},\n",
       " {'start': 1008,\n",
       "  'end': 1306,\n",
       "  'text': 'Enhanced proliferation and self-renewal of the stem cells on nanofibrillar surfaces were correlated with the activation of the small GTPase Rac, the activation of phosphoinositide 3-kinase (PI3K) pathway, and the enhanced expression of Nanog, a homeoprotein required for maintenance of pluripotency.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '16150921',\n",
       "  'score': 0.2947448790073395},\n",
       " {'start': 1308,\n",
       "  'end': 1411,\n",
       "  'text': 'Inhibitors of PI3K reduced the expression level of Nanog in mESCs cultured on 3D nanofibrillar surfaces.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '16150921',\n",
       "  'score': 0.3962307870388031},\n",
       " {'start': 1413,\n",
       "  'end': 1639,\n",
       "  'text': 'These results provide support for the view that the three-dimensionality of the culture surface may function as a cue for the activation of Rac and PI3K signaling pathways, resulting in stem cell proliferation and self-renewal.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '16150921',\n",
       "  'score': 0.12439730763435364},\n",
       " {'start': 0,\n",
       "  'end': 76,\n",
       "  'text': 'Gain of CTCF-Anchored Chromatin Loops Marks the Exit from Naive Pluripotency.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '30414923',\n",
       "  'score': 0.2822280824184418},\n",
       " {'start': 0,\n",
       "  'end': 160,\n",
       "  'text': 'The genome of pluripotent stem cells adopts a unique three-dimensional architecture featuring weakly condensed heterochromatin and large nucleosome-free regions.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '30414923',\n",
       "  'score': 0.3918892443180084},\n",
       " {'start': 162,\n",
       "  'end': 325,\n",
       "  'text': 'Yet, it is unknown whether structural loops and contact domains display characteristics that distinguish embryonic stem cells (ESCs) from differentiated cell types.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '30414923',\n",
       "  'score': 0.2605474591255188},\n",
       " {'start': 327,\n",
       "  'end': 493,\n",
       "  'text': 'We used genome-wide chromosome conformation capture and super-resolution imaging to determine nuclear organization in mouse ESC and neural stem cell (NSC) derivatives.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '30414923',\n",
       "  'score': 0.1448041796684265},\n",
       " {'start': 495,\n",
       "  'end': 583,\n",
       "  'text': 'We found that loss of pluripotency is accompanied by widespread gain of structural loops.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '30414923',\n",
       "  'score': 0.31085795164108276},\n",
       " {'start': 585,\n",
       "  'end': 770,\n",
       "  'text': 'This general architectural change correlates with enhanced binding of CTCF and cohesins and more pronounced insulation of contacts across chromatin boundaries in lineage-committed cells.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '30414923',\n",
       "  'score': 0.1192331314086914},\n",
       " {'start': 772,\n",
       "  'end': 858,\n",
       "  'text': 'Reprogramming NSCs to pluripotency restores the unique features of ESC domain topology.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '30414923',\n",
       "  'score': 0.28120577335357666},\n",
       " {'start': 860,\n",
       "  'end': 969,\n",
       "  'text': 'Domains defined by the anchors of loops established upon differentiation are enriched for developmental genes.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '30414923',\n",
       "  'score': 0.11662551760673523},\n",
       " {'start': 971,\n",
       "  'end': 1160,\n",
       "  'text': 'Chromatin loop formation is a pervasive structural alteration to the genome that accompanies exit from pluripotency and delineates the spatial segregation of developmentally regulated genes.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '30414923',\n",
       "  'score': 0.28757819533348083},\n",
       " {'start': 0,\n",
       "  'end': 85,\n",
       "  'text': 'Functional and Sustainable 3D Human Neural Network Models from Pluripotent Stem Cells.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '33304995',\n",
       "  'score': 0.3980183005332947},\n",
       " {'start': 0,\n",
       "  'end': 229,\n",
       "  'text': 'Three-dimensional in vitro cell culture models, particularly for the central nervous system, allow for the exploration of mechanisms of organ development, cellular interactions, and disease progression within defined environments.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '33304995',\n",
       "  'score': 0.1784084439277649},\n",
       " {'start': 231,\n",
       "  'end': 405,\n",
       "  'text': 'Here we describe the development and characterization of three-dimensional tissue models that promote the differentiation and long-term survival of functional neural networks.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '33304995',\n",
       "  'score': 0.19315552711486816},\n",
       " {'start': 407,\n",
       "  'end': 649,\n",
       "  'text': 'These tissue cultures show diverse cell populations including neurons and glial cells (astrocytes) interacting in 3D with spontaneous neural activity confirmed through electrophysiological recordings and calcium imaging over at least 8 months.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '33304995',\n",
       "  'score': 0.15080520510673523},\n",
       " {'start': 651,\n",
       "  'end': 964,\n",
       "  'text': 'This approach allows for the direct integration of pluripotent stem cells into the 3D construct bypassing early neural differentiation steps (embryoid bodies and neural rosettes), which streamlines the process while also providing a system that can be manipulated to support a variety of experimental applications.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '33304995',\n",
       "  'score': 0.3149849772453308},\n",
       " {'start': 966,\n",
       "  'end': 1250,\n",
       "  'text': \"This tissue model has been tested in stem cells derived from healthy individuals as well as Alzheimer's and Parkinson's disease patients, with similar growth and gene expression responses indicating potential use in the modeling of disease states related to neurodegenerative diseases.\",\n",
       "  'is_title': False,\n",
       "  'doc_id': '33304995',\n",
       "  'score': 0.1207895576953888},\n",
       " {'start': 0,\n",
       "  'end': 84,\n",
       "  'text': 'A genome-wide RNAi screen reveals determinants of human embryonic stem cell identity.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '20953172',\n",
       "  'score': 0.11572203040122986},\n",
       " {'start': 0,\n",
       "  'end': 117,\n",
       "  'text': 'The derivation of human ES cells (hESCs) from human blastocysts represents one of the milestones in stem cell biology.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '20953172',\n",
       "  'score': 0.26213860511779785},\n",
       " {'start': 119,\n",
       "  'end': 285,\n",
       "  'text': 'The full potential of hESCs in research and clinical applications requires a detailed understanding of the genetic network that governs the unique properties of hESCs.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '20953172',\n",
       "  'score': 0.34298673272132874},\n",
       " {'start': 287,\n",
       "  'end': 423,\n",
       "  'text': 'Here, we report a genome-wide RNA interference screen to identify genes which regulate self-renewal and pluripotency properties in hESCs.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '20953172',\n",
       "  'score': 0.27338868379592896},\n",
       " {'start': 425,\n",
       "  'end': 583,\n",
       "  'text': 'Interestingly, functionally distinct complexes involved in transcriptional regulation and chromatin remodelling are among the factors identified in the screen.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '20953172',\n",
       "  'score': 0.14615768194198608},\n",
       " {'start': 585,\n",
       "  'end': 768,\n",
       "  'text': 'To understand the roles of these potential regulators of hESCs, we studied transcription factor PRDM14 to gain new insights into its functional roles in the regulation of pluripotency.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '20953172',\n",
       "  'score': 0.3653643727302551},\n",
       " {'start': 770,\n",
       "  'end': 887,\n",
       "  'text': 'We showed that PRDM14 regulates directly the expression of key pluripotency gene POU5F1 through its proximal enhancer.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '20953172',\n",
       "  'score': 0.3836219310760498},\n",
       " {'start': 889,\n",
       "  'end': 1129,\n",
       "  'text': 'Genome-wide location profiling experiments revealed that PRDM14 colocalized extensively with other key transcription factors such as OCT4, NANOG and SOX2, indicating that PRDM14 is integrated into the core transcriptional regulatory network.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '20953172',\n",
       "  'score': 0.31281960010528564},\n",
       " {'start': 1131,\n",
       "  'end': 1311,\n",
       "  'text': 'More importantly, in a gain-of-function assay, we showed that PRDM14 is able to enhance the efficiency of reprogramming of human fibroblasts in conjunction with OCT4, SOX2 and KLF4.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '20953172',\n",
       "  'score': 0.2948090732097626},\n",
       " {'start': 1313,\n",
       "  'end': 1538,\n",
       "  'text': 'Altogether, our study uncovers a wealth of novel hESC regulators wherein PRDM14 exemplifies a key transcription factor required for the maintenance of hESC identity and the reacquisition of pluripotency in human somatic cells.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '20953172',\n",
       "  'score': 0.3012164533138275},\n",
       " {'start': 0,\n",
       "  'end': 80,\n",
       "  'text': 'Derivation of an interaction/regulation network describing pluripotency in human.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '22548825',\n",
       "  'score': 0.38108497858047485},\n",
       " {'start': 0,\n",
       "  'end': 169,\n",
       "  'text': 'Identification of the key genes/proteins of pluripotency and their interrelationships is an important step in understanding the induction and maintenance of pluripotency.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '22548825',\n",
       "  'score': 0.3298477530479431},\n",
       " {'start': 171,\n",
       "  'end': 265,\n",
       "  'text': 'Experimental approaches have accumulated large amounts of interaction/regulation data in mouse.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '22548825',\n",
       "  'score': 0.13394737243652344},\n",
       " {'start': 267,\n",
       "  'end': 418,\n",
       "  'text': 'We investigate how far such information can be transferred to human, the species of maximum interest, for which experimental data are much more limited.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '22548825',\n",
       "  'score': 0.11906912922859192},\n",
       " {'start': 420,\n",
       "  'end': 519,\n",
       "  'text': 'To address this issue, we mapped an existing mouse pluripotency network (the PluriNetWork) to human.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '22548825',\n",
       "  'score': 0.30386391282081604},\n",
       " {'start': 521,\n",
       "  'end': 701,\n",
       "  'text': 'We transferred interaction and regulation links between genes/proteins from mouse to human on the basis of orthologous relationship of the genes/proteins (called interolog mapping).',\n",
       "  'is_title': False,\n",
       "  'doc_id': '22548825',\n",
       "  'score': 0.13766825199127197},\n",
       " {'start': 703,\n",
       "  'end': 887,\n",
       "  'text': 'To reduce the number of false positives, we used four different methods: phylogenetic profiling, Gene Ontology semantic similarity, gene co-expression, and RNA interference (RNAi) data.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '22548825',\n",
       "  'score': 0.12338033318519592},\n",
       " {'start': 889,\n",
       "  'end': 1057,\n",
       "  'text': 'The methods and the resulting networks were evaluated by a novel approach using the information about the genes known to be involved in pluripotency from the literature.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '22548825',\n",
       "  'score': 0.3402690291404724},\n",
       " {'start': 1059,\n",
       "  'end': 1192,\n",
       "  'text': 'The RNAi method proved best for filtering out unlikely interactions, so it was used to construct the final human pluripotency network.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '22548825',\n",
       "  'score': 0.3018547296524048},\n",
       " {'start': 1194,\n",
       "  'end': 1330,\n",
       "  'text': 'The RNAi data are based on human embryonic stem cells (hESCs) that are generally considered to be in a (primed) epiblast stem cell state.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '22548825',\n",
       "  'score': 0.26566484570503235},\n",
       " {'start': 1332,\n",
       "  'end': 1538,\n",
       "  'text': 'Therefore, we assume that the final human network may reflect the (primed) epiblast stem cell state more closely, while the mouse network reflects the (unprimed/naïve) embryonic stem cell state more closely.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '22548825',\n",
       "  'score': 0.13441291451454163},\n",
       " {'start': 0,\n",
       "  'end': 94,\n",
       "  'text': 'Chromatin organization in pluripotent cells: emerging approaches to study and disrupt function.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '26206085',\n",
       "  'score': 0.3437819480895996},\n",
       " {'start': 0,\n",
       "  'end': 182,\n",
       "  'text': 'Translating the vast amounts of genomic and epigenomic information accumulated on the linear genome into three-dimensional models of nuclear organization is a current major challenge.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '26206085',\n",
       "  'score': 0.21401411294937134},\n",
       " {'start': 184,\n",
       "  'end': 430,\n",
       "  'text': 'In response to this challenge, recent technological innovations based on chromosome conformation capture methods in combination with increasingly powerful functional approaches have revealed exciting insights into key aspects of genome regulation.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '26206085',\n",
       "  'score': 0.13456183671951294},\n",
       " {'start': 432,\n",
       "  'end': 699,\n",
       "  'text': 'These findings have led to an emerging model where the genome is folded and compartmentalized into highly conserved topological domains that are further divided into functional subdomains containing physical loops that bring cis-regulatory elements to close proximity.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '26206085',\n",
       "  'score': 0.12079134583473206},\n",
       " {'start': 701,\n",
       "  'end': 902,\n",
       "  'text': 'Targeted functional experiments, largely based on designable DNA-binding proteins, have begun to define the major architectural proteins required to establish and maintain appropriate genome regulation.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '26206085',\n",
       "  'score': 0.1325986683368683},\n",
       " {'start': 904,\n",
       "  'end': 1105,\n",
       "  'text': 'Here, we focus on the accessible and well-characterized system of pluripotent cells to review the functional role of chromatin organization in regulating pluripotency, differentiation and reprogramming.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '26206085',\n",
       "  'score': 0.39008259773254395},\n",
       " {'start': 0,\n",
       "  'end': 137,\n",
       "  'text': 'Long-range chromatin contacts in embryonic stem cells reveal a role for pluripotency factors and polycomb proteins in genome organization.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '24035354',\n",
       "  'score': 0.3310115337371826},\n",
       " {'start': 0,\n",
       "  'end': 104,\n",
       "  'text': 'The relationship between 3D organization of the genome and gene-regulatory networks is poorly understood.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24035354',\n",
       "  'score': 0.2418501079082489},\n",
       " {'start': 106,\n",
       "  'end': 341,\n",
       "  'text': 'Here, we examined long-range chromatin interactions genome-wide in mouse embryonic stem cells (ESCs), iPSCs, and fibroblasts and uncovered a pluripotency-specific genome organization that is gradually reestablished during reprogramming.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24035354',\n",
       "  'score': 0.3077548146247864},\n",
       " {'start': 343,\n",
       "  'end': 523,\n",
       "  'text': 'Our data confirm that long-range chromatin interactions are primarily associated with the spatial segregation of open and closed chromatin, defining overall chromosome conformation.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24035354',\n",
       "  'score': 0.12268230319023132},\n",
       " {'start': 525,\n",
       "  'end': 752,\n",
       "  'text': 'Additionally, we identified two further levels of genome organization in ESCs characterized by colocalization of regions with high pluripotency factor occupancy and strong enrichment for Polycomb proteins/H3K27me3, respectively.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24035354',\n",
       "  'score': 0.35384947061538696},\n",
       " {'start': 754,\n",
       "  'end': 1001,\n",
       "  'text': 'Underlining the independence of these networks and their functional relevance for genome organization, loss of the Polycomb protein Eed diminishes interactions between Polycomb-regulated regions without altering overarching chromosome conformation.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24035354',\n",
       "  'score': 0.18397995829582214},\n",
       " {'start': 1003,\n",
       "  'end': 1252,\n",
       "  'text': 'Together, our data highlight a pluripotency-specific genome organization in which pluripotency factors such as Nanog and H3K27me3 occupy distinct nuclear spaces and reveal a role for cell-type-specific gene-regulatory networks in genome organization.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24035354',\n",
       "  'score': 0.3784448206424713},\n",
       " {'start': 0,\n",
       "  'end': 112,\n",
       "  'text': 'Coordination of m(6)A mRNA Methylation and Gene Transcription by ZFP217 Regulates Pluripotency and Reprogramming.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '26526723',\n",
       "  'score': 0.29806822538375854},\n",
       " {'start': 0,\n",
       "  'end': 160,\n",
       "  'text': 'Epigenetic and epitranscriptomic networks have important functions in maintaining the pluripotency of embryonic stem cells (ESCs) and somatic cell reprogramming.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '26526723',\n",
       "  'score': 0.38562652468681335},\n",
       " {'start': 162,\n",
       "  'end': 266,\n",
       "  'text': 'However, the mechanisms integrating the actions of these distinct networks are only partially understood.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '26526723',\n",
       "  'score': 0.14018943905830383},\n",
       " {'start': 268,\n",
       "  'end': 399,\n",
       "  'text': 'Here we show that the chromatin-associated zinc finger protein 217 (ZFP217) coordinates epigenetic and epitranscriptomic regulation.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '26526723',\n",
       "  'score': 0.14924296736717224},\n",
       " {'start': 401,\n",
       "  'end': 648,\n",
       "  'text': 'ZFP217 interacts with several epigenetic regulators, activates the transcription of key pluripotency genes, and modulates N6-methyladenosine (m(6)A) deposition on their transcripts by sequestering the enzyme m(6)A methyltransferase-like 3 (METTL3).',\n",
       "  'is_title': False,\n",
       "  'doc_id': '26526723',\n",
       "  'score': 0.268758624792099},\n",
       " {'start': 650,\n",
       "  'end': 881,\n",
       "  'text': 'Consistently, Zfp217 depletion compromises ESC self-renewal and somatic cell reprogramming, globally increases m(6)A RNA levels, and enhances m(6)A modification of the Nanog, Sox2, Klf4, and c-Myc mRNAs, promoting their degradation.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '26526723',\n",
       "  'score': 0.2757059633731842},\n",
       " {'start': 883,\n",
       "  'end': 1015,\n",
       "  'text': 'ZFP217 binds its own target gene mRNAs, which are also METTL3 associated, and is enriched at promoters of m(6)A-modified transcripts.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '26526723',\n",
       "  'score': 0.1268824338912964},\n",
       " {'start': 1017,\n",
       "  'end': 1173,\n",
       "  'text': 'Collectively, these findings shed light on how a transcription factor can tightly couple gene transcription to m(6)A RNA modification to ensure ESC identity.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '26526723',\n",
       "  'score': 0.11609819531440735},\n",
       " {'start': 0,\n",
       "  'end': 96,\n",
       "  'text': 'Regulation of protein abundance in pluripotent cells undergoing commitment to the neural lineage.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '8707862',\n",
       "  'score': 0.38138315081596375},\n",
       " {'start': 0,\n",
       "  'end': 254,\n",
       "  'text': 'The P19 cell line is a widely studied model of neural differentiation When pluripotent P19 cells are cultured as aggregates in the presence of retinoic acid for 4 days, the cells commit to the neural fate, but have not yet undergone overt differentiation.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '8707862',\n",
       "  'score': 0.27461665868759155},\n",
       " {'start': 256,\n",
       "  'end': 376,\n",
       "  'text': 'Two-dimensional polyacrylamide gel electrophoresis was used to analyze cellular protein expression during this induction.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '8707862',\n",
       "  'score': 0.1835336685180664},\n",
       " {'start': 378,\n",
       "  'end': 431,\n",
       "  'text': 'Approximately 500 abundant polypeptides were analyzed.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '8707862',\n",
       "  'score': 0.11626586318016052},\n",
       " {'start': 433,\n",
       "  'end': 577,\n",
       "  'text': 'Seventeen polypeptides were upregulated during induction; several of these were significantly regulated 48 h after the addition of retinoic acid.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '8707862',\n",
       "  'score': 0.11912712454795837},\n",
       " {'start': 579,\n",
       "  'end': 611,\n",
       "  'text': 'No downregulations were observed.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '8707862',\n",
       "  'score': 0.11626586318016052},\n",
       " {'start': 613,\n",
       "  'end': 705,\n",
       "  'text': 'Fifteen of the 17 polypeptides continued to be expressed throughout terminal differentiation.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '8707862',\n",
       "  'score': 0.11900350451469421},\n",
       " {'start': 707,\n",
       "  'end': 846,\n",
       "  'text': 'The upregulation of 14 of the 17 polypeptides requires both retinoic acid and aggregation, which alone do not induce neural differentiation.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '8707862',\n",
       "  'score': 0.12266531586647034},\n",
       " {'start': 848,\n",
       "  'end': 981,\n",
       "  'text': 'Furthermore, these regulated polypeptides are expressed in neural tissue, suggesting they are associated with neural function in vivo.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '8707862',\n",
       "  'score': 0.12146833539009094},\n",
       " {'start': 983,\n",
       "  'end': 1096,\n",
       "  'text': 'Embryonic stem cells, a totipotent line, also neurally differentiate in response to retinoic acid and aggregation.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '8707862',\n",
       "  'score': 0.12005940079689026},\n",
       " {'start': 1098,\n",
       "  'end': 1260,\n",
       "  'text': 'Comparison of embryonic stem cells to P19 cells shows that the two systems regulate a similar set of polypeptides and are thus likely to utilize a similar pathway.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '8707862',\n",
       "  'score': 0.12565675377845764},\n",
       " {'start': 1262,\n",
       "  'end': 1402,\n",
       "  'text': 'These studies are a step toward determining the full extent of regulation involved in the commitment of pluripotent cells to the neural fate.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '8707862',\n",
       "  'score': 0.3657374382019043},\n",
       " {'start': 0,\n",
       "  'end': 127,\n",
       "  'text': 'A competitive protein interaction network buffers Oct4-mediated differentiation to promote pluripotency in embryonic stem cells.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '24104477',\n",
       "  'score': 0.346407413482666},\n",
       " {'start': 0,\n",
       "  'end': 217,\n",
       "  'text': \"Pluripotency in embryonic stem cells is maintained through the activity of a small set of transcription factors centred around Oct4 and Nanog, which control the expression of 'self-renewal' and 'differentiation' genes.\",\n",
       "  'is_title': False,\n",
       "  'doc_id': '24104477',\n",
       "  'score': 0.3075852394104004},\n",
       " {'start': 219,\n",
       "  'end': 419,\n",
       "  'text': 'Here, we combine single-cell quantitative immunofluorescence microscopy and gene expression analysis, together with theoretical modelling, to investigate how the activity of those factors is regulated.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24104477',\n",
       "  'score': 0.12300476431846619},\n",
       " {'start': 421,\n",
       "  'end': 584,\n",
       "  'text': 'We uncover a key role for post-translational regulation in the maintenance of pluripotency, which complements the well-established transcriptional regulatory layer.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24104477',\n",
       "  'score': 0.3389921188354492},\n",
       " {'start': 586,\n",
       "  'end': 778,\n",
       "  'text': 'Specifically, we find that the activity of a network of protein complexes involving Nanog, Oct4, Tcf3, and β-catenin suffices to account for the behavior of ES cells under different conditions.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24104477',\n",
       "  'score': 0.3235238790512085},\n",
       " {'start': 780,\n",
       "  'end': 945,\n",
       "  'text': 'Our results suggest that the function of the network is to buffer the transcriptional activity of Oct4, which appears to be the main determinant to exit pluripotency.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24104477',\n",
       "  'score': 0.31913793087005615},\n",
       " {'start': 947,\n",
       "  'end': 1137,\n",
       "  'text': 'The protein network explains the mechanisms underlying the gain and loss of function in different mutants, and brings us closer to a full understanding of the molecular basis of pluripotency.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24104477',\n",
       "  'score': 0.3752123713493347},\n",
       " {'start': 0,\n",
       "  'end': 138,\n",
       "  'text': 'Embryonic Stem Cell Engineering with a Glycomimetic FGF2/BMP4 Co-Receptor Drives Mesodermal Differentiation in a Three-Dimensional Culture.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '30157624',\n",
       "  'score': 0.17118901014328003},\n",
       " {'start': 0,\n",
       "  'end': 300,\n",
       "  'text': 'Cell surface glycans, such as heparan sulfate (HS), are increasingly identified as co-regulators of growth factor signaling in early embryonic development; therefore, chemical tailoring of HS activity within the cellular glycocalyx of stem cells offers an opportunity to control their differentiation.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '30157624',\n",
       "  'score': 0.12159267067909241},\n",
       " {'start': 302,\n",
       "  'end': 506,\n",
       "  'text': 'The growth factors FGF2 and BMP4 are involved in mediating the exit of murine embryonic stem cells (mESCs) from their pluripotent state and their differentiation toward mesodermal cell types, respectively.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '30157624',\n",
       "  'score': 0.311210036277771},\n",
       " {'start': 508,\n",
       "  'end': 690,\n",
       "  'text': 'Here, we report a method for remodeling the glycocalyx of mutant Ext1-/- mESCs with defective biosynthesis of HS to drive their mesodermal differentiation in an embryoid body culture.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '30157624',\n",
       "  'score': 0.27280616760253906},\n",
       " {'start': 692,\n",
       "  'end': 999,\n",
       "  'text': 'Lipid-functionalized synthetic HS-mimetic glycopolymers with affinity for both FGF2 and BMP4 were introduced into the plasma membrane of Ext1-/- mESCs, where they acted as functional co-receptors of these growth factors and facilitated signal transduction through associated MAPK and Smad signaling pathways.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '30157624',\n",
       "  'score': 0.2697141170501709},\n",
       " {'start': 1001,\n",
       "  'end': 1226,\n",
       "  'text': 'We demonstrate that these materials can be employed to remodel Ext1-/- mESCs within three-dimensional embryoid body structures, providing enhanced association of BMP4 at the cell surface and driving mesodermal differentiation.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '30157624',\n",
       "  'score': 0.38241466879844666},\n",
       " {'start': 1228,\n",
       "  'end': 1491,\n",
       "  'text': 'As a more complete understanding of the function of HS in regulating development continues to emerge, this simple glycocalyx engineering method is poised to enable precise control over growth factor signaling activity and outcomes of differentiation in stem cells.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '30157624',\n",
       "  'score': 0.1398715078830719},\n",
       " {'start': 0,\n",
       "  'end': 116,\n",
       "  'text': 'A membrane-associated β-catenin/Oct4 complex correlates with ground-state pluripotency in mouse embryonic stem cells.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '23444350',\n",
       "  'score': 0.2991563379764557},\n",
       " {'start': 0,\n",
       "  'end': 255,\n",
       "  'text': 'The maintenance of pluripotency in mouse embryonic stem cells (mESCs) relies on the activity of a transcriptional network that is fuelled by the activity of three transcription factors (Nanog, Oct4 and Sox2) and balanced by the repressive activity of Tcf3.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '23444350',\n",
       "  'score': 0.37772685289382935},\n",
       " {'start': 257,\n",
       "  'end': 370,\n",
       "  'text': 'Extracellular signals modulate the activity of the network and regulate the differentiation capacity of the cells.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '23444350',\n",
       "  'score': 0.141049325466156},\n",
       " {'start': 372,\n",
       "  'end': 556,\n",
       "  'text': 'Wnt/β-catenin signaling has emerged as a significant potentiator of pluripotency: increases in the levels of β-catenin regulate the activity of Oct4 and Nanog, and enhance pluripotency.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '23444350',\n",
       "  'score': 0.3263707458972931},\n",
       " {'start': 558,\n",
       "  'end': 737,\n",
       "  'text': 'A recent report shows that β-catenin achieves some of these effects by modulating the activity of Tcf3, and that this effect does not require its transcriptional activation domain.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '23444350',\n",
       "  'score': 0.12156513333320618},\n",
       " {'start': 739,\n",
       "  'end': 952,\n",
       "  'text': 'Here, we show that during self-renewal there is negligible transcriptional activity of β-catenin and that this is due to its tight association with membranes, where we find it in a complex with Oct4 and E-cadherin.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '23444350',\n",
       "  'score': 0.2583959698677063},\n",
       " {'start': 954,\n",
       "  'end': 1079,\n",
       "  'text': 'Differentiation triggers a burst of Wnt/β-catenin transcriptional activity that coincides with the disassembly of the complex.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '23444350',\n",
       "  'score': 0.11976107954978943},\n",
       " {'start': 1081,\n",
       "  'end': 1223,\n",
       "  'text': 'Our results establish that β-catenin, but not its transcriptional activity, is central to pluripotency acting through a β-catenin/Oct4 complex.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '23444350',\n",
       "  'score': 0.28451353311538696},\n",
       " {'start': 0,\n",
       "  'end': 33,\n",
       "  'text': 'CARM1 mediates modulation of Sox2.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '22046437',\n",
       "  'score': 0.11844712495803833},\n",
       " {'start': 0,\n",
       "  'end': 127,\n",
       "  'text': 'Sox2 is a key component of the transcription factor network that maintains the pluripotent state of embryonic stem cells (ESCs).',\n",
       "  'is_title': False,\n",
       "  'doc_id': '22046437',\n",
       "  'score': 0.34139561653137207},\n",
       " {'start': 129,\n",
       "  'end': 263,\n",
       "  'text': 'Sox2 is regulated by multiple post-translational modifications, including ubiquitination, sumoylation, acetylation and phosphorylation.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '22046437',\n",
       "  'score': 0.12168490886688232},\n",
       " {'start': 265,\n",
       "  'end': 462,\n",
       "  'text': 'Here we report that Sox2 is in association with and methylated by coactivator-associated arginine methyltransferase 1 (CARM1), a protein arginine methyltransferase that plays a pivotal role in ESCs.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '22046437',\n",
       "  'score': 0.30046582221984863},\n",
       " {'start': 464,\n",
       "  'end': 570,\n",
       "  'text': 'We found that CARM1 facilitates Sox2-mediated transactivation and directly methylates Sox2 at arginine 113.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '22046437',\n",
       "  'score': 0.11919710040092468},\n",
       " {'start': 572,\n",
       "  'end': 625,\n",
       "  'text': 'This methylation event enhances Sox2 self-association.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '22046437',\n",
       "  'score': 0.11716225743293762},\n",
       " {'start': 627,\n",
       "  'end': 725,\n",
       "  'text': 'Furthermore, the physiological retention of Sox2 on chromatin restricts the Sox2 methylation level.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '22046437',\n",
       "  'score': 0.12018075585365295},\n",
       " {'start': 727,\n",
       "  'end': 902,\n",
       "  'text': 'Our study reveals the direct regulation of Sox2 by CARM1 that sheds lights on how arginine methylation signals are integrated into the pluripotent transcription factor network.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '22046437',\n",
       "  'score': 0.3782678246498108},\n",
       " {'start': 0,\n",
       "  'end': 64,\n",
       "  'text': 'Chaperones and Beyond as Key Players in Pluripotency Maintenance.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '31428613',\n",
       "  'score': 0.2854463458061218},\n",
       " {'start': 0,\n",
       "  'end': 161,\n",
       "  'text': 'Pluripotency is orchestrated by distinct players and chaperones and their partners have emerged as pivotal molecules in proteostasis control to maintain stemness.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31428613',\n",
       "  'score': 0.30126267671585083},\n",
       " {'start': 163,\n",
       "  'end': 344,\n",
       "  'text': 'The proteostasis network consists of diverse interconnected pathways that function dynamically according to the needs of the cell to quality control and maintain protein homeostasis.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31428613',\n",
       "  'score': 0.15134593844413757},\n",
       " {'start': 346,\n",
       "  'end': 529,\n",
       "  'text': 'The proteostasis machinery of pluripotent stem cells (PSCs) is finely adjusted in response to distinct stimuli during cell fate commitment to determine successful organism development.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31428613',\n",
       "  'score': 0.3076562285423279},\n",
       " {'start': 531,\n",
       "  'end': 635,\n",
       "  'text': 'Growing evidence has shown different classes of chaperones regulating crucial cellular processes in PSCs.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31428613',\n",
       "  'score': 0.13689666986465454},\n",
       " {'start': 637,\n",
       "  'end': 811,\n",
       "  'text': \"Histones chaperones promote proper nucleosome assembly and modulate the epigenetic regulation of factors involved in PSCs' rapid turnover from pluripotency to differentiation.\",\n",
       "  'is_title': False,\n",
       "  'doc_id': '31428613',\n",
       "  'score': 0.3611997961997986},\n",
       " {'start': 813,\n",
       "  'end': 1021,\n",
       "  'text': 'The life cycle of pluripotency proteins from synthesis and folding, transport and degradation is finely regulated by chaperones and co-factors either to maintain the stemness status or to cell fate commitment.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31428613',\n",
       "  'score': 0.3082446753978729},\n",
       " {'start': 1023,\n",
       "  'end': 1175,\n",
       "  'text': 'Here, we summarize current knowledge of the chaperone network that govern stemness and present the versatile role of chaperones in stem cells resilience.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31428613',\n",
       "  'score': 0.14538276195526123},\n",
       " {'start': 1177,\n",
       "  'end': 1443,\n",
       "  'text': 'Elucidation of the intricate regulation of pluripotency, dissecting in detail molecular determinants and drivers, is fundamental to understanding the properties of stem cells in order to provide a reliable foundation for biomedical research and regenerative medicine.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31428613',\n",
       "  'score': 0.37499818205833435},\n",
       " {'start': 0,\n",
       "  'end': 118,\n",
       "  'text': 'Qualitative modeling identifies IL-11 as a novel regulator in maintaining self-renewal in human pluripotent stem cells.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '24194720',\n",
       "  'score': 0.2609670162200928},\n",
       " {'start': 0,\n",
       "  'end': 161,\n",
       "  'text': 'Pluripotency in human embryonic stem cells (hESCs) and induced pluripotent stem cells (iPSCs) is regulated by three transcription factors-OCT3/4, SOX2, and NANOG.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24194720',\n",
       "  'score': 0.3290077745914459},\n",
       " {'start': 163,\n",
       "  'end': 329,\n",
       "  'text': 'To fully exploit the therapeutic potential of these cells it is essential to have a good mechanistic understanding of the maintenance of self-renewal and pluripotency.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24194720',\n",
       "  'score': 0.293235182762146},\n",
       " {'start': 331,\n",
       "  'end': 567,\n",
       "  'text': 'In this study, we demonstrate a powerful systems biology approach in which we first expand literature-based network encompassing the core regulators of pluripotency by assessing the behavior of genes targeted by perturbation experiments.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24194720',\n",
       "  'score': 0.3735601305961609},\n",
       " {'start': 569,\n",
       "  'end': 752,\n",
       "  'text': 'We focused our attention on highly regulated genes encoding cell surface and secreted proteins as these can be more easily manipulated by the use of inhibitors or recombinant proteins.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24194720',\n",
       "  'score': 0.1216198205947876},\n",
       " {'start': 754,\n",
       "  'end': 911,\n",
       "  'text': 'Qualitative modeling based on combining boolean networks and in silico perturbation experiments were employed to identify novel pluripotency-regulating genes.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24194720',\n",
       "  'score': 0.372661292552948},\n",
       " {'start': 913,\n",
       "  'end': 1112,\n",
       "  'text': 'We validated Interleukin-11 (IL-11) and demonstrate that this cytokine is a novel pluripotency-associated factor capable of supporting self-renewal in the absence of exogenously added bFGF in culture.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24194720',\n",
       "  'score': 0.33625221252441406},\n",
       " {'start': 1114,\n",
       "  'end': 1267,\n",
       "  'text': 'To date, the various protocols for hESCs maintenance require supplementation with bFGF to activate the Activin/Nodal branch of the TGFβ signaling pathway.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24194720',\n",
       "  'score': 0.2501968741416931},\n",
       " {'start': 1269,\n",
       "  'end': 1460,\n",
       "  'text': 'Additional evidence supporting our findings is that IL-11 belongs to the same protein family as LIF, which is known to be necessary for maintaining pluripotency in mouse but not in human ESCs.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24194720',\n",
       "  'score': 0.2922254800796509},\n",
       " {'start': 1462,\n",
       "  'end': 1552,\n",
       "  'text': 'These cytokines operate through the same gp130 receptor which interacts with Janus kinases.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24194720',\n",
       "  'score': 0.12059038877487183},\n",
       " {'start': 1554,\n",
       "  'end': 1694,\n",
       "  'text': 'Our finding might explain why mESCs are in a more naïve cell state compared to hESCs and how to convert primed hESCs back to the naïve state.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24194720',\n",
       "  'score': 0.2690315842628479},\n",
       " {'start': 1696,\n",
       "  'end': 1926,\n",
       "  'text': 'Taken together, our integrative modeling approach has identified novel genes as putative candidates to be incorporated into the expansion of the current gene regulatory network responsible for inducing and maintaining pluripotency.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24194720',\n",
       "  'score': 0.3067598342895508},\n",
       " {'start': 0,\n",
       "  'end': 114,\n",
       "  'text': 'mTOR supports long-term self-renewal and suppresses mesoderm and endoderm activities of human embryonic stem cells.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '19416884',\n",
       "  'score': 0.12078273296356201},\n",
       " {'start': 0,\n",
       "  'end': 237,\n",
       "  'text': 'Despite the recent identification of the transcriptional regulatory circuitry involving SOX2, NANOG, and OCT-4, the intracellular signaling networks that control pluripotency of human embryonic stem cells (hESCs) remain largely undefined.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '19416884',\n",
       "  'score': 0.37591099739074707},\n",
       " {'start': 239,\n",
       "  'end': 407,\n",
       "  'text': 'Here, we demonstrate an essential role for the serine/threonine protein kinase mammalian target of rapamycin (mTOR) in regulating hESC long-term undifferentiated growth.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '19416884',\n",
       "  'score': 0.14699479937553406},\n",
       " {'start': 409,\n",
       "  'end': 533,\n",
       "  'text': 'Inhibition of mTOR impairs pluripotency, prevents cell proliferation, and enhances mesoderm and endoderm activities in hESCs.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '19416884',\n",
       "  'score': 0.30433404445648193},\n",
       " {'start': 535,\n",
       "  'end': 779,\n",
       "  'text': 'At the molecular level, mTOR integrates signals from extrinsic pluripotency-supporting factors and represses the transcriptional activities of a subset of developmental and growth-inhibitory genes, as revealed by genome-wide microarray analyses.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '19416884',\n",
       "  'score': 0.2596684694290161},\n",
       " {'start': 781,\n",
       "  'end': 880,\n",
       "  'text': 'Repression of the developmental genes by mTOR is necessary for the maintenance of hESC pluripotency.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '19416884',\n",
       "  'score': 0.3138730823993683},\n",
       " {'start': 882,\n",
       "  'end': 978,\n",
       "  'text': 'These results uncover a novel signaling mechanism by which mTOR controls fate decisions in hESCs.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '19416884',\n",
       "  'score': 0.2693483829498291},\n",
       " {'start': 980,\n",
       "  'end': 1066,\n",
       "  'text': 'Our findings may contribute to effective strategies for tissue repair and regeneration.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '19416884',\n",
       "  'score': 0.11936423182487488},\n",
       " {'start': 0,\n",
       "  'end': 104,\n",
       "  'text': 'Microvessel Network Formation and Interactions with Pancreatic Islets in Three-Dimensional Chip Cultures.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '31724494',\n",
       "  'score': 0.20889729261398315},\n",
       " {'start': 0,\n",
       "  'end': 157,\n",
       "  'text': 'The pancreatic islet is a highly vascularized micro-organ, and rapid revascularization postislet transplantation is important for islet survival and function.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31724494',\n",
       "  'score': 0.12302398681640625},\n",
       " {'start': 159,\n",
       "  'end': 314,\n",
       "  'text': 'However, the various mechanisms involved in islet revascularization are not fully understood, and we currently lack good in vitro platforms to explore this.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31724494',\n",
       "  'score': 0.128957599401474},\n",
       " {'start': 316,\n",
       "  'end': 555,\n",
       "  'text': 'Our aim for this study was to generate perfusable microvascular networks in a microfluidic chip device, in which islets could be easily integrated, to establish an in vitro platform for investigations on islet-microvasculature interactions.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31724494',\n",
       "  'score': 0.14003503322601318},\n",
       " {'start': 557,\n",
       "  'end': 942,\n",
       "  'text': 'We compared the ability of mesenchymal stem cells (MSCs) and fibroblasts to support microvascular network formation by human umbilical vein endothelial cells (HUVECs) and human induced pluripotent stem cell-derived endothelial colony-forming cell in two-dimensional and three-dimensional models of angiogenesis, and tested the effect of different culture media on microvessel formation.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31724494',\n",
       "  'score': 0.4021335542201996},\n",
       " {'start': 944,\n",
       "  'end': 1093,\n",
       "  'text': 'HUVECs that were supported by MSCs formed patent and perfusable networks in a fibrin gel, whereas networks supported by fibroblasts rapidly regressed.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31724494',\n",
       "  'score': 0.1437842845916748},\n",
       " {'start': 1095,\n",
       "  'end': 1182,\n",
       "  'text': 'Network morphology could be controlled by adjusting relative cell numbers and densities.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31724494',\n",
       "  'score': 0.14021751284599304},\n",
       " {'start': 1184,\n",
       "  'end': 1379,\n",
       "  'text': 'Incorporation of isolated rat islets demonstrated that islets recruit local microvasculature in vitro, but that the microvessels did not invade islets, at least during the course of these studies.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31724494',\n",
       "  'score': 0.12237435579299927},\n",
       " {'start': 1381,\n",
       "  'end': 1616,\n",
       "  'text': 'This in vitro microvascularization platform can provide a useful tool to study how various parameters affect islet integration with microvascular networks and could also be utilized for studies of vascularization of other organ systems.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31724494',\n",
       "  'score': 0.14367058873176575},\n",
       " {'start': 1618,\n",
       "  'end': 1760,\n",
       "  'text': 'Impact statement To improve pancreatic islet graft survival and function posttransplantation, rapid and adequate revascularization is critical.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31724494',\n",
       "  'score': 0.12122046947479248},\n",
       " {'start': 1762,\n",
       "  'end': 1897,\n",
       "  'text': 'Efforts to improve islet revascularization are demanding due to an insufficient understanding of the mechanisms involved in the process.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31724494',\n",
       "  'score': 0.12830641865730286},\n",
       " {'start': 1899,\n",
       "  'end': 2086,\n",
       "  'text': 'We have applied a microfluidics platform to generate microvascular networks, and by incorporating pancreatic islets, we were able to study microvasculature-islet interactions in real time.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31724494',\n",
       "  'score': 0.14255136251449585},\n",
       " {'start': 2088,\n",
       "  'end': 2259,\n",
       "  'text': 'This platform can provide a useful tool to study islet integration with microvascular networks, and could be utilized for studies of vascularization of other organ systems.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31724494',\n",
       "  'score': 0.14122667908668518},\n",
       " {'start': 2261,\n",
       "  'end': 2367,\n",
       "  'text': 'Moreover, this work may be adapted toward developing a prevascularized islet construct for transplantation.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31724494',\n",
       "  'score': 0.11760681867599487},\n",
       " {'start': 0,\n",
       "  'end': 109,\n",
       "  'text': 'Microfibrous Scaffolds Enhance Endothelial Differentiation and Organization of Induced Pluripotent Stem Cells.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '28936269',\n",
       "  'score': 0.35005834698677063},\n",
       " {'start': 0,\n",
       "  'end': 180,\n",
       "  'text': 'INTRODUCTION\\nHuman induced pluripotent stem cells (iPSCs) are a promising source of endothelial cells (iPSC-ECs) for engineering three-dimensional (3D) vascularized cardiac tissues.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28936269',\n",
       "  'score': 0.3848375380039215},\n",
       " {'start': 182,\n",
       "  'end': 470,\n",
       "  'text': 'To mimic cardiac microvasculature, in which capillaries are oriented in parallel, we hypothesized that endothelial differentiation of iPSCs within topographically aligned 3D scaffolds would be a facile one-step approach to generate iPSC-ECs as well as induce aligned vascular organization.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28936269',\n",
       "  'score': 0.19390136003494263},\n",
       " {'start': 474,\n",
       "  'end': 650,\n",
       "  'text': 'METHODS\\nHuman iPSCs underwent endothelial differentiation within electrospun 3D polycaprolactone (PCL) scaffolds having either randomly oriented or parallel-aligned microfibers.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28936269',\n",
       "  'score': 0.15493014454841614},\n",
       " {'start': 652,\n",
       "  'end': 879,\n",
       "  'text': 'Using transcriptional, protein, and endothelial functional assays, endothelial differentiation was compared between conventional two-dimensional (2D) films and 3D scaffolds having either randomly oriented or aligned microfibers.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28936269',\n",
       "  'score': 0.17965486645698547},\n",
       " {'start': 881,\n",
       "  'end': 1001,\n",
       "  'text': 'Furthermore, the role of parallel-aligned microfiber patterning on the organization of vessel-like networks was assessed.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28936269',\n",
       "  'score': 0.17952904105186462},\n",
       " {'start': 1005,\n",
       "  'end': 1207,\n",
       "  'text': 'RESULTS\\nThe cells in both the randomly oriented and aligned 3D scaffolds demonstrated an 11-fold upregulation in gene expression of the endothelial phenotypic marker, CD31, compared to cells on 2D films.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28936269',\n",
       "  'score': 0.16266173124313354},\n",
       " {'start': 1209,\n",
       "  'end': 1324,\n",
       "  'text': 'This upregulation corresponded to >3-fold increase in CD31 protein expression in 3D scaffolds, compared to 2D films.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28936269',\n",
       "  'score': 0.16749972105026245},\n",
       " {'start': 1326,\n",
       "  'end': 1533,\n",
       "  'text': 'Concomitantly, other endothelial phenotypic markers including CD144 and endothelial nitric oxide synthase also showed significant transcriptional upregulation in 3D scaffolds by >7-fold, compared to 2D films.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28936269',\n",
       "  'score': 0.1590895652770996},\n",
       " {'start': 1535,\n",
       "  'end': 1689,\n",
       "  'text': 'Nitric oxide production, which is characteristic of endothelial function, was produced 4-fold more abundantly in 3D scaffolds, compared to on 2D PCL films.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28936269',\n",
       "  'score': 0.16083160042762756},\n",
       " {'start': 1691,\n",
       "  'end': 1950,\n",
       "  'text': 'Within aligned scaffolds, the iPSC-ECs displayed parallel-aligned vascular-like networks with 70% longer branch length, compared to cells in randomly oriented scaffolds, suggesting that fiber topography modulates vascular network-like formation and patterning.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28936269',\n",
       "  'score': 0.14416229724884033},\n",
       " {'start': 1954,\n",
       "  'end': 2184,\n",
       "  'text': 'CONCLUSION\\nTogether, these results demonstrate that 3D scaffold structure promotes endothelial differentiation, compared to 2D substrates, and that aligned topographical patterning induces anisotropic vascular network organization.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28936269',\n",
       "  'score': 0.2151528298854828},\n",
       " {'start': 0,\n",
       "  'end': 186,\n",
       "  'text': 'Proteomic analysis of early reprogramming events in murine somatic cells incubated with Xenopus laevis oocyte extracts demonstrates network associations with induced pluripotency markers.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '23768116',\n",
       "  'score': 0.3259560763835907},\n",
       " {'start': 0,\n",
       "  'end': 192,\n",
       "  'text': 'The reprogramming of somatic cells into a pluripotent/embryonic-like state holds great potential for regenerative medicine, bypassing ethical issues associated with embryonic stem cells (ESCs).',\n",
       "  'is_title': False,\n",
       "  'doc_id': '23768116',\n",
       "  'score': 0.3115348219871521},\n",
       " {'start': 194,\n",
       "  'end': 456,\n",
       "  'text': 'Numerous methods, including somatic cell nuclear transfer (SCNT), fusion to pluripotent cells, the use of cell extracts, and expression of transcription factors, have been used to reprogram cells into ES-like cells [termed induced pluripotent stem cells (iPSCs)].',\n",
       "  'is_title': False,\n",
       "  'doc_id': '23768116',\n",
       "  'score': 0.2667217254638672},\n",
       " {'start': 458,\n",
       "  'end': 735,\n",
       "  'text': 'This study investigated early events in the nuclei of permeabilized murine somatic cells incubated in cytoplasmic extract prepared from Xenopus laevis germinal vesicle-stage oocytes by identifying proteins that showed significant quantitative changes using proteomic techniques.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '23768116',\n",
       "  'score': 0.12321022152900696},\n",
       " {'start': 737,\n",
       "  'end': 939,\n",
       "  'text': 'A total of 69 protein spots from two-dimensional electrophoresis were identified as being significantly altered in expression after treatment, and 38 proteins were identified by tandem mass spectrometry.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '23768116',\n",
       "  'score': 0.17819857597351074},\n",
       " {'start': 941,\n",
       "  'end': 1175,\n",
       "  'text': 'Network analysis was used to highlight pathway connections and interactions between these identified proteins, which were found to be involved in many functions--primarily nuclear structure and dynamics, transcription, and translation.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '23768116',\n",
       "  'score': 0.15581241250038147},\n",
       " {'start': 1177,\n",
       "  'end': 1462,\n",
       "  'text': 'The pluripotency markers Klf4, c-Myc, Nanog, and POU5F1 were highlighted by the interaction network analysis, as well as other compounds/proteins known to be repressed in pluripotent cells [e.g., protein kinase C (PRKC)] or enhanced during differentiation of ESCs (e.g., retinoic acid).',\n",
       "  'is_title': False,\n",
       "  'doc_id': '23768116',\n",
       "  'score': 0.37398213148117065},\n",
       " {'start': 1464,\n",
       "  'end': 1583,\n",
       "  'text': 'The network analysis also indicated additional proteins and pathways potentially involved in early reprogramming events.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '23768116',\n",
       "  'score': 0.15523281693458557},\n",
       " {'start': 0,\n",
       "  'end': 46,\n",
       "  'text': 'Establishing pluripotency in early development.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '25857441',\n",
       "  'score': 0.3000926971435547},\n",
       " {'start': 0,\n",
       "  'end': 197,\n",
       "  'text': 'The earliest steps of embryonic development involve important changes in chromatin and transcription factor networks, which are orchestrated to establish pluripotent cells that will form the embryo.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '25857441',\n",
       "  'score': 0.32913291454315186},\n",
       " {'start': 199,\n",
       "  'end': 399,\n",
       "  'text': 'DNA methylation, histone modifications, the pluripotency regulatory network of transcription factors, maternal factors and newly translated proteins all contribute to these transitions in dynamic ways.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '25857441',\n",
       "  'score': 0.3381897211074829},\n",
       " {'start': 401,\n",
       "  'end': 474,\n",
       "  'text': 'Moreover, these dynamics are linked to the onset of zygotic transcription.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '25857441',\n",
       "  'score': 0.1210588812828064},\n",
       " {'start': 476,\n",
       "  'end': 679,\n",
       "  'text': 'We will review recent progress in our understanding of chromatin state and regulation of gene expression in the context of embryonic development in vertebrates, in particular mouse, Xenopus and zebrafish.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '25857441',\n",
       "  'score': 0.1436864137649536},\n",
       " {'start': 681,\n",
       "  'end': 856,\n",
       "  'text': 'We include work on mouse embryonic stem cells and highlight work that illustrates how early embryonic dynamics establish gene regulatory networks and the state of pluripotency.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '25857441',\n",
       "  'score': 0.3701986074447632},\n",
       " {'start': 0,\n",
       "  'end': 53,\n",
       "  'text': 'Role of CTCF in the regulation of microRNA expression.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '23056006',\n",
       "  'score': 0.14439508318901062},\n",
       " {'start': 0,\n",
       "  'end': 93,\n",
       "  'text': 'MicroRNAs (miRNAs) are small non-coding RNAs that regulate expression of various target genes.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '23056006',\n",
       "  'score': 0.11944600939750671},\n",
       " {'start': 95,\n",
       "  'end': 222,\n",
       "  'text': 'miRNAs are expressed in a tissue-specific manner and play important roles in cell proliferation, apoptosis, and differentiation.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '23056006',\n",
       "  'score': 0.12247636914253235},\n",
       " {'start': 224,\n",
       "  'end': 385,\n",
       "  'text': 'Epigenetic alterations such as DNA methylation and histone modification are essential for chromatin remodeling and regulation of gene expression including miRNAs.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '23056006',\n",
       "  'score': 0.13989046216011047},\n",
       " {'start': 387,\n",
       "  'end': 593,\n",
       "  'text': 'The CCCTC-binding factor, CTCF, is known to bind insulators and exhibits an enhancer-blocking and barrier function, and more recently, it also contributes to the three-dimensional organization of the genome.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '23056006',\n",
       "  'score': 0.34126001596450806},\n",
       " {'start': 595,\n",
       "  'end': 742,\n",
       "  'text': 'CTCF can also serve as a barrier against the spread of DNA methylation and histone repressive marks over promoter regions of tumor suppressor genes.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '23056006',\n",
       "  'score': 0.1955692172050476},\n",
       " {'start': 744,\n",
       "  'end': 907,\n",
       "  'text': 'Recent studies have shown that CTCF is also involved in the regulation of miRNAs such as miR-125b1, miR-375, and the miR-290 cluster in cancer cells and stem cells.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '23056006',\n",
       "  'score': 0.15107238292694092},\n",
       " {'start': 909,\n",
       "  'end': 992,\n",
       "  'text': 'miR-125b1 is a candidate of tumor suppressor and is silenced in breast cancer cells.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '23056006',\n",
       "  'score': 0.12520340085029602},\n",
       " {'start': 994,\n",
       "  'end': 1092,\n",
       "  'text': 'On the other hand, miR-375 may have oncogenic function and is overexpressed in breast cancer cells.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '23056006',\n",
       "  'score': 0.12647035717964172},\n",
       " {'start': 1094,\n",
       "  'end': 1253,\n",
       "  'text': 'CTCF is involved in the regulation of both miR-125b1 and miR-375, indicating that there are various patterns of CTCF-associated epigenetic regulation of miRNAs.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '23056006',\n",
       "  'score': 0.16101661324501038},\n",
       " {'start': 1255,\n",
       "  'end': 1355,\n",
       "  'text': 'CTCF may also play a key role in the pluripotency of cells through the regulation of miR-290 cluster.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '23056006',\n",
       "  'score': 0.3826563060283661},\n",
       " {'start': 1357,\n",
       "  'end': 1494,\n",
       "  'text': 'These observations suggest that CTCF-mediated regulation of miRNAs could be a novel approach for cancer therapy and regenerative medicine.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '23056006',\n",
       "  'score': 0.13874012231826782},\n",
       " {'start': 0,\n",
       "  'end': 95,\n",
       "  'text': 'Expanding the Circuitry of Pluripotency by Selective Isolation of Chromatin-Associated Proteins.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '27773674',\n",
       "  'score': 0.32780998945236206},\n",
       " {'start': 0,\n",
       "  'end': 255,\n",
       "  'text': 'Maintenance of pluripotency is regulated by a network of transcription factors coordinated by Oct4, Sox2, and Nanog (OSN), yet a systematic investigation of the composition and dynamics of the OSN protein network specifically on chromatin is still missing.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '27773674',\n",
       "  'score': 0.37387555837631226},\n",
       " {'start': 257,\n",
       "  'end': 461,\n",
       "  'text': 'Here we have developed a method combining ChIP with selective isolation of chromatin-associated proteins (SICAP) followed by mass spectrometry to identify chromatin-bound partners of a protein of interest.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '27773674',\n",
       "  'score': 0.1308966875076294},\n",
       " {'start': 463,\n",
       "  'end': 630,\n",
       "  'text': 'ChIP-SICAP in mouse embryonic stem cells (ESCs) identified over 400 proteins associating with OSN, including several whose interaction depends on the pluripotent state.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '27773674',\n",
       "  'score': 0.26358672976493835},\n",
       " {'start': 632,\n",
       "  'end': 817,\n",
       "  'text': 'Trim24, a previously unrecognized protein in the network, converges with OSN on multiple enhancers and suppresses the expression of developmental genes while activating cell cycle genes.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '27773674',\n",
       "  'score': 0.24970591068267822},\n",
       " {'start': 819,\n",
       "  'end': 968,\n",
       "  'text': 'Consistently, Trim24 significantly improved efficiency of cellular reprogramming, demonstrating its direct functionality in establishing pluripotency.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '27773674',\n",
       "  'score': 0.2762302756309509},\n",
       " {'start': 970,\n",
       "  'end': 1125,\n",
       "  'text': 'Collectively, ChIP-SICAP provides a powerful tool to decode chromatin protein composition, further enhanced by its integrative capacity to perform ChIP-seq.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '27773674',\n",
       "  'score': 0.12471142411231995},\n",
       " {'start': 0,\n",
       "  'end': 113,\n",
       "  'text': 'The role of Hippo signaling pathway and mechanotransduction in tuning embryoid body formation and differentiation.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '31951024',\n",
       "  'score': 0.12618011236190796},\n",
       " {'start': 0,\n",
       "  'end': 149,\n",
       "  'text': 'Embryoid bodies (EBs) are the three-dimensional aggregates of pluripotent stem cells that are used as a model system for the in vitro differentiation.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31951024',\n",
       "  'score': 0.40712907910346985},\n",
       " {'start': 151,\n",
       "  'end': 273,\n",
       "  'text': 'EBs mimic the early stages of embryogenesis and are considered as a potential biomimetic body in tuning the stem cell fate.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31951024',\n",
       "  'score': 0.1248508095741272},\n",
       " {'start': 275,\n",
       "  'end': 513,\n",
       "  'text': 'Although EBs have a spheroid shape, they are not formed accidentally by the agglomeration of cells; they are formed by the deliberate and programmed aggregation of stem cells in a complex topological and biophysical microstructure instead.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31951024',\n",
       "  'score': 0.12491637468338013},\n",
       " {'start': 515,\n",
       "  'end': 695,\n",
       "  'text': 'EBs could be programmed to promisingly differentiate into the desired germ layers with specific cell lineages, in response to intra- and extra-biochemical and biomechanical signals.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31951024',\n",
       "  'score': 0.12371566891670227},\n",
       " {'start': 697,\n",
       "  'end': 813,\n",
       "  'text': 'Hippo signaling and mechanotransduction are the key pathways in controlling the formation and differentiation of EBs.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31951024',\n",
       "  'score': 0.12538403272628784},\n",
       " {'start': 815,\n",
       "  'end': 1003,\n",
       "  'text': 'The activity of the Hippo pathway strongly relies on cell-cell junctions, cell polarity, cellular architecture, cellular metabolism, and mechanical cues in the surrounding microenvironment.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31951024',\n",
       "  'score': 0.12559983134269714},\n",
       " {'start': 1005,\n",
       "  'end': 1246,\n",
       "  'text': 'Although the Hippo pathway was initially thought to limit the size of the organ by inhibiting the proliferation and the promotion of apoptosis, the evidence suggests that this pathway even regulates stem cell self-renewal and differentiation.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31951024',\n",
       "  'score': 0.12291550636291504},\n",
       " {'start': 1248,\n",
       "  'end': 1500,\n",
       "  'text': 'Considering the abovementioned explanations, the present study investigated the interplay of the Hippo signaling pathway, mechanotransduction, differentiation, and proliferation pathways to draw the molecular network involved in the control of EBs fate.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31951024',\n",
       "  'score': 0.15443626046180725},\n",
       " {'start': 1502,\n",
       "  'end': 1688,\n",
       "  'text': 'In addition, this study highlighted several neglected critical parameters regarding EB formation, in the interplay with the Hippo core component involved in the promising differentiation.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31951024',\n",
       "  'score': 0.12807956337928772},\n",
       " {'start': 0,\n",
       "  'end': 160,\n",
       "  'text': 'Linking Hematopoietic Differentiation to Co-Expressed Sets of Pluripotency-Associated and Imprinted Genes and to Regulatory microRNA-Transcription Factor Motifs.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '28052084',\n",
       "  'score': 0.3573504388332367},\n",
       " {'start': 0,\n",
       "  'end': 177,\n",
       "  'text': 'Maintenance of cell pluripotency, differentiation, and reprogramming are regulated by complex gene regulatory networks (GRNs) including monoallelically-expressed imprinted genes.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28052084',\n",
       "  'score': 0.37089645862579346},\n",
       " {'start': 179,\n",
       "  'end': 289,\n",
       "  'text': 'Besides transcriptional control, epigenetic modifications and microRNAs contribute to cellular differentiation.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28052084',\n",
       "  'score': 0.12057551741600037},\n",
       " {'start': 291,\n",
       "  'end': 531,\n",
       "  'text': 'As a model system for studying the capacity of cells to preserve their pluripotency state and the onset of differentiation and subsequent specialization, murine hematopoiesis was used and compared to embryonic stem cells (ESCs) as a control.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28052084',\n",
       "  'score': 0.2716447710990906},\n",
       " {'start': 533,\n",
       "  'end': 696,\n",
       "  'text': 'Using published microarray data, the expression profiles of two sets of genes, pluripotent and imprinted, were compared to a third set of known hematopoietic genes.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28052084',\n",
       "  'score': 0.27532678842544556},\n",
       " {'start': 698,\n",
       "  'end': 845,\n",
       "  'text': 'We found that more than half of the pluripotent and imprinted genes are clearly upregulated in ESCs but subsequently repressed during hematopoiesis.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28052084',\n",
       "  'score': 0.2780001163482666},\n",
       " {'start': 847,\n",
       "  'end': 952,\n",
       "  'text': 'The remaining genes were either upregulated in hematopoietic progenitors or in differentiated blood cells.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28052084',\n",
       "  'score': 0.12251365184783936},\n",
       " {'start': 954,\n",
       "  'end': 1126,\n",
       "  'text': 'The three gene sets each consist of three similarly behaving gene groups with similar expression profiles in various lineages of the hematopoietic system as well as in ESCs.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28052084',\n",
       "  'score': 0.28763651847839355},\n",
       " {'start': 1128,\n",
       "  'end': 1344,\n",
       "  'text': 'To explain this co-regulation behavior, we explored the transcriptional and post-transcriptional mechanisms of pluripotent and imprinted genes and their regulator/target miRNAs in six different hematopoietic lineages.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28052084',\n",
       "  'score': 0.33364272117614746},\n",
       " {'start': 1346,\n",
       "  'end': 1519,\n",
       "  'text': 'Therewith, lineage-specific transcription factor (TF)-miRNA regulatory networks were generated and their topologies and functional impacts during hematopoiesis were analyzed.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28052084',\n",
       "  'score': 0.14304688572883606},\n",
       " {'start': 1521,\n",
       "  'end': 1756,\n",
       "  'text': 'This led to the identification of TF-miRNA co-regulatory motifs, for which we validated the contribution to the cellular development of the corresponding lineage in terms of statistical significance and relevance to biological evidence.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28052084',\n",
       "  'score': 0.12596312165260315},\n",
       " {'start': 1758,\n",
       "  'end': 1876,\n",
       "  'text': 'This analysis also identified key miRNAs and TFs/genes that might play important roles in the derived lineage networks.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28052084',\n",
       "  'score': 0.14833849668502808},\n",
       " {'start': 1878,\n",
       "  'end': 2205,\n",
       "  'text': 'These molecular associations suggest new aspects of the cellular regulation of the onset of cellular differentiation and during hematopoiesis involving, on one hand, pluripotent genes that were previously not discussed in the context of hematopoiesis and, on the other hand, involve genes that are related to genomic imprinting.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28052084',\n",
       "  'score': 0.3046821355819702},\n",
       " {'start': 2207,\n",
       "  'end': 2329,\n",
       "  'text': 'These are new links between hematopoiesis and cellular differentiation and the important field of epigenetic modifications.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28052084',\n",
       "  'score': 0.1235848069190979},\n",
       " {'start': 0,\n",
       "  'end': 107,\n",
       "  'text': 'RAD21 cooperates with pluripotency transcription factors in the maintenance of embryonic stem cell identity.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '21589869',\n",
       "  'score': 0.30433428287506104},\n",
       " {'start': 0,\n",
       "  'end': 240,\n",
       "  'text': 'For self-renewal, embryonic stem cells (ESCs) require the expression of specific transcription factors accompanied by a particular chromosome organization to maintain a balance between pluripotency and the capacity for rapid differentiation.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '21589869',\n",
       "  'score': 0.3559303879737854},\n",
       " {'start': 242,\n",
       "  'end': 349,\n",
       "  'text': 'However, how transcriptional regulation is linked to chromosome organization in ESCs is not well understood.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '21589869',\n",
       "  'score': 0.36969661712646484},\n",
       " {'start': 351,\n",
       "  'end': 517,\n",
       "  'text': 'Here we show that the cohesin component RAD21 exhibits a functional role in maintaining ESC identity through association with the pluripotency transcriptional network.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '21589869',\n",
       "  'score': 0.31612884998321533},\n",
       " {'start': 519,\n",
       "  'end': 743,\n",
       "  'text': 'ChIP-seq analyses of RAD21 reveal an ESC specific cohesin binding pattern that is characterized by CTCF independent co-localization of cohesin with pluripotency related transcription factors Oct4, Nanog, Sox2, Esrrb and Klf4.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '21589869',\n",
       "  'score': 0.3019070029258728},\n",
       " {'start': 745,\n",
       "  'end': 967,\n",
       "  'text': 'Upon ESC differentiation, most of these binding sites disappear and instead new CTCF independent RAD21 binding sites emerge, which are enriched for binding sites of transcription factors implicated in early differentiation.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '21589869',\n",
       "  'score': 0.1257777214050293},\n",
       " {'start': 969,\n",
       "  'end': 1191,\n",
       "  'text': 'Furthermore, knock-down of RAD21 causes expression changes that are similar to expression changes after Nanog depletion, demonstrating the functional relevance of the RAD21--pluripotency transcriptional network association.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '21589869',\n",
       "  'score': 0.3287307918071747},\n",
       " {'start': 1193,\n",
       "  'end': 1345,\n",
       "  'text': 'Finally, we show that Nanog physically interacts with the cohesin or cohesin interacting proteins STAG1 and WAPL further substantiating this association.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '21589869',\n",
       "  'score': 0.2604931890964508},\n",
       " {'start': 1347,\n",
       "  'end': 1534,\n",
       "  'text': 'Based on these findings we propose that a dynamic placement of cohesin by pluripotency transcription factors contributes to a chromosome organization supporting the ESC expression program.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '21589869',\n",
       "  'score': 0.332438588142395},\n",
       " {'start': 0,\n",
       "  'end': 91,\n",
       "  'text': 'Roles of Dppa2 in the regulation of the present status and future of pluripotent stem cells.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '22641845',\n",
       "  'score': 0.3610232174396515},\n",
       " {'start': 0,\n",
       "  'end': 156,\n",
       "  'text': 'Maintenance of undifferentiated states of pluripotent embryonic stem cells is regulated by a complex network of transcription factors and signaling pathways.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '22641845',\n",
       "  'score': 0.36264264583587646},\n",
       " {'start': 158,\n",
       "  'end': 372,\n",
       "  'text': 'Recent reports revealed that developmental pluripotency-associated 2 (Dppa2), which regulates chromatin structures, plays important roles in the maintenance of pluripotency and proliferation of embryonic stem cells.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '22641845',\n",
       "  'score': 0.3484847843647003},\n",
       " {'start': 374,\n",
       "  'end': 543,\n",
       "  'text': 'Interestingly, developmental pluripotency-associated 2 is involved not only in the normal development of lung but also in the pathogenesis of non-small cell lung cancers.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '22641845',\n",
       "  'score': 0.36342453956604004},\n",
       " {'start': 545,\n",
       "  'end': 683,\n",
       "  'text': 'These results suggest that an epigenetic regulator of pluripotent stem cells plays important roles in normal development and tumorigenesis.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '22641845',\n",
       "  'score': 0.31424176692962646},\n",
       " {'start': 0,\n",
       "  'end': 111,\n",
       "  'text': 'AutoSOME: a clustering method for identifying gene expression modules without prior knowledge of cluster number.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '20202218',\n",
       "  'score': 0.1210375726222992},\n",
       " {'start': 0,\n",
       "  'end': 142,\n",
       "  'text': 'BACKGROUND\\nClustering the information content of large high-dimensional gene expression datasets has widespread application in \"omics\" biology.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '20202218',\n",
       "  'score': 0.16202124953269958},\n",
       " {'start': 144,\n",
       "  'end': 342,\n",
       "  'text': 'Unfortunately, the underlying structure of these natural datasets is often fuzzy, and the computational identification of data clusters generally requires knowledge about cluster number and geometry.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '20202218',\n",
       "  'score': 0.1266900599002838},\n",
       " {'start': 346,\n",
       "  'end': 545,\n",
       "  'text': 'RESULTS\\nWe integrated strategies from machine learning, cartography, and graph theory into a new informatics method for automatically clustering self-organizing map ensembles of high-dimensional data.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '20202218',\n",
       "  'score': 0.1627674102783203},\n",
       " {'start': 547,\n",
       "  'end': 745,\n",
       "  'text': 'Our new method, called AutoSOME, readily identifies discrete and fuzzy data clusters without prior knowledge of cluster number or structure in diverse datasets including whole genome microarray data.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '20202218',\n",
       "  'score': 0.12472221255302429},\n",
       " {'start': 747,\n",
       "  'end': 901,\n",
       "  'text': 'Visualization of AutoSOME output using network diagrams and differential heat maps reveals unexpected variation among well-characterized cancer cell lines.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '20202218',\n",
       "  'score': 0.14467346668243408},\n",
       " {'start': 903,\n",
       "  'end': 1214,\n",
       "  'text': 'Co-expression analysis of data from human embryonic and induced pluripotent stem cells using AutoSOME identifies >3400 up-regulated genes associated with pluripotency, and indicates that a recently identified protein-protein interaction network characterizing pluripotency was underestimated by a factor of four.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '20202218',\n",
       "  'score': 0.39451032876968384},\n",
       " {'start': 1218,\n",
       "  'end': 1462,\n",
       "  'text': 'CONCLUSIONS\\nBy effectively extracting important information from high-dimensional microarray data without prior knowledge or the need for data filtration, AutoSOME can yield systems-level insights from whole genome microarray expression studies.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '20202218',\n",
       "  'score': 0.15597054362297058},\n",
       " {'start': 1464,\n",
       "  'end': 1636,\n",
       "  'text': 'Due to its generality, this new method should also have practical utility for a variety of data-intensive applications, including the results of deep sequencing experiments.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '20202218',\n",
       "  'score': 0.12257242202758789},\n",
       " {'start': 1638,\n",
       "  'end': 1726,\n",
       "  'text': 'AutoSOME is available for download at http://jimcooperlab.mcdb.ucsb.edu/autosome webcite.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '20202218',\n",
       "  'score': 0.12189999222755432},\n",
       " {'start': 0,\n",
       "  'end': 178,\n",
       "  'text': 'Glycogen synthase kinase-3 inhibition enhances translation of pluripotency-associated transcription factors to contribute to maintenance of mouse embryonic stem cell self-renewal.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '23577087',\n",
       "  'score': 0.3627505302429199},\n",
       " {'start': 0,\n",
       "  'end': 167,\n",
       "  'text': 'Maintenance of embryonic stem cell (ESC) self-renewal and pluripotency are controlled by extrinsic factors, molecular signaling pathways and transcriptional regulators.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '23577087',\n",
       "  'score': 0.30400508642196655},\n",
       " {'start': 169,\n",
       "  'end': 370,\n",
       "  'text': 'While many of the key players have been studied in depth, how the molecular signals interact with transcription factors of the pluripotency network to regulate their action remains less well understood.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '23577087',\n",
       "  'score': 0.3423035740852356},\n",
       " {'start': 372,\n",
       "  'end': 675,\n",
       "  'text': 'Inhibition of glycogen synthase kinase 3 (Gsk-3) has been implicated in the maintenance of mouse ESC pluripotency, although there is contradictory data on its role, with enhancement of cell survival and metabolism, stabilisation of c-Myc and activation of Wnt signalling proposed as potential mechanisms.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '23577087',\n",
       "  'score': 0.3175240755081177},\n",
       " {'start': 677,\n",
       "  'end': 856,\n",
       "  'text': 'We have discovered that suppression of Gsk-3 activity leads to enhanced protein levels of key transcriptional regulators of the pluripotency network, notably Nanog, Tbx3 and c-Myc.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '23577087',\n",
       "  'score': 0.36189576983451843},\n",
       " {'start': 858,\n",
       "  'end': 1343,\n",
       "  'text': 'Protein stability was unchanged following Gsk-3 inhibition, although interestingly, Nanog and Tbx3 proteins were found to have half-lives of 1-3 h, while that of Oct4 protein was longer, at 6 h. We demonstrate that the effects on protein levels seen following inhibition of Gsk-3 are due to both enhanced de novo synthesis of Nanog protein and increases in the proportion of Nanog and Tbx3 RNAs bound to polysomes, findings consistent with Gsk-3 regulating translation of these factors.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '23577087',\n",
       "  'score': 0.3224031627178192},\n",
       " {'start': 1345,\n",
       "  'end': 1499,\n",
       "  'text': \"These effects were not due to changes in regulators of general translation initiation machinery nor mediated via the 5' or 3' UTR sequences of Nanog alone.\",\n",
       "  'is_title': False,\n",
       "  'doc_id': '23577087',\n",
       "  'score': 0.3050016760826111},\n",
       " {'start': 1501,\n",
       "  'end': 1764,\n",
       "  'text': 'The data we present provide both new conceptual insight into the mechanisms regulated by Gsk-3 that may contribute to ESC self-renewal and, importantly, establish control of protein translation as an additional mechanism involved in modulation of ESC pluripotency.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '23577087',\n",
       "  'score': 0.342555433511734},\n",
       " {'start': 0,\n",
       "  'end': 109,\n",
       "  'text': 'An Optimized System for Effective Derivation of Three-Dimensional Retinal Tissue via Wnt Signaling Regulation.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '29999566',\n",
       "  'score': 0.1967775523662567},\n",
       " {'start': 0,\n",
       "  'end': 217,\n",
       "  'text': 'Effective derivation of three-dimensional (3D) retinal tissue from human-induced pluripotent stem cells (hiPSCs) could provide models for drug screening and facilitate patient-specific retinal cell replacement therapy.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '29999566',\n",
       "  'score': 0.3850555419921875},\n",
       " {'start': 219,\n",
       "  'end': 351,\n",
       "  'text': 'However, some hiPSC lines cannot undergo 3D self-organization and show inadequate differentiation efficiency to meet clinical demand.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '29999566',\n",
       "  'score': 0.21032124757766724},\n",
       " {'start': 353,\n",
       "  'end': 436,\n",
       "  'text': 'In this study, we developed an optimized system for derivation of 3D retinal tissue.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '29999566',\n",
       "  'score': 0.1707819700241089},\n",
       " {'start': 438,\n",
       "  'end': 600,\n",
       "  'text': 'We found that the Wnt signaling pathway antagonist Dickkopf-related protein 1 (DKK-1) rescued the inability of differentiated retinal progenitors to self-organize.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '29999566',\n",
       "  'score': 0.1292334496974945},\n",
       " {'start': 602,\n",
       "  'end': 789,\n",
       "  'text': 'By evaluating DKK-1 expression and supplying DKK-1 if necessary, retinal organoids were differentiated from six hiPSC lines, which were reprogramed from three common initiating cell types.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '29999566',\n",
       "  'score': 0.1281784474849701},\n",
       " {'start': 791,\n",
       "  'end': 908,\n",
       "  'text': 'Retinal tissues derived from the optimized system were well organized and capable of surviving for further maturation.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '29999566',\n",
       "  'score': 0.1246325671672821},\n",
       " {'start': 910,\n",
       "  'end': 1009,\n",
       "  'text': 'Thus, using this system, we generated retinal tissues from various hiPSC lines with high efficiency.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '29999566',\n",
       "  'score': 0.1201859712600708},\n",
       " {'start': 1011,\n",
       "  'end': 1107,\n",
       "  'text': 'This novel system has many potential applications in regenerative therapy and precision medicine.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '29999566',\n",
       "  'score': 0.12381336092948914},\n",
       " {'start': 1109,\n",
       "  'end': 1137,\n",
       "  'text': 'Stem Cells 2018;36:1709-1722.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '29999566',\n",
       "  'score': 0.1201859712600708},\n",
       " {'start': 0,\n",
       "  'end': 34,\n",
       "  'text': 'Nuclear architecture in stem cells.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '21222196',\n",
       "  'score': 0.12231820821762085},\n",
       " {'start': 0,\n",
       "  'end': 203,\n",
       "  'text': 'Fundamental features of genome regulation depend on the linear DNA sequence, cell type specific modification of DNA and chromatin-associated proteins, which locally control the expression of single genes.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '21222196',\n",
       "  'score': 0.14852508902549744},\n",
       " {'start': 205,\n",
       "  'end': 468,\n",
       "  'text': 'Architectural features of genome organization within the three-dimensional (3D) nuclear space establish preferential positioning of genes relative to nuclear subcompartments associated with specific biochemical activities, thereby influencing states of expression.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '21222196',\n",
       "  'score': 0.22650891542434692},\n",
       " {'start': 470,\n",
       "  'end': 715,\n",
       "  'text': 'The structural and temporal organization of the genome within the nucleus of stem cells, together with specific features of epigenetic and transcriptional regulation are emerging as key players that influence pluripotency and differentiation.1,2.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '21222196',\n",
       "  'score': 0.38951271772384644},\n",
       " {'start': 0,\n",
       "  'end': 44,\n",
       "  'text': 'Patz1 regulates embryonic stem cell identity.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '24380431',\n",
       "  'score': 0.1203262209892273},\n",
       " {'start': 0,\n",
       "  'end': 97,\n",
       "  'text': 'Embryonic stem cells (ESCs) derived from the inner cell mass (ICM) of blastocysts are pluripotent.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24380431',\n",
       "  'score': 0.2773019075393677},\n",
       " {'start': 99,\n",
       "  'end': 200,\n",
       "  'text': 'Pluripotency is maintained by a transcriptional network in which Oct4 and Nanog are master regulators.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24380431',\n",
       "  'score': 0.3662261962890625},\n",
       " {'start': 202,\n",
       "  'end': 289,\n",
       "  'text': 'Notably, several zinc finger transcription factors have important roles in this network.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24380431',\n",
       "  'score': 0.14457696676254272},\n",
       " {'start': 291,\n",
       "  'end': 417,\n",
       "  'text': 'Patz1, a BTB/POZ-domain-containing zinc finger protein, is expressed at higher levels in the ICM relative to the trophectoderm.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24380431',\n",
       "  'score': 0.13567835092544556},\n",
       " {'start': 419,\n",
       "  'end': 480,\n",
       "  'text': 'However, its function in pluripotency has been poorly studied.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24380431',\n",
       "  'score': 0.2901040315628052},\n",
       " {'start': 482,\n",
       "  'end': 556,\n",
       "  'text': 'Here, we show that Patz1 is an important regulator of pluripotency in ESCs.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24380431',\n",
       "  'score': 0.3411720395088196},\n",
       " {'start': 558,\n",
       "  'end': 683,\n",
       "  'text': 'Patz1 RNAi, chromatin immunoprecipitation (ChIP), and reporter assays indicate that Patz1 directly regulates Pou5f1 and Nanog.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24380431',\n",
       "  'score': 0.2827426791191101},\n",
       " {'start': 685,\n",
       "  'end': 845,\n",
       "  'text': 'Global transcriptome changes upon Patz1 knockdown largely involve upregulation of apoptotic genes and downregulation of cell cycle and cellular metabolism genes.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24380431',\n",
       "  'score': 0.12364223599433899},\n",
       " {'start': 847,\n",
       "  'end': 988,\n",
       "  'text': 'Patz1 ChIP sequencing further identified more than 5,000 binding sites of Patz1 in mouse genome, from which two binding motifs were extracted.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24380431',\n",
       "  'score': 0.13018068671226501},\n",
       " {'start': 990,\n",
       "  'end': 1121,\n",
       "  'text': 'Further, gene ontology analysis of genes associated with the binding sites displays enrichment for proximity to developmental genes.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24380431',\n",
       "  'score': 0.1267530918121338},\n",
       " {'start': 1123,\n",
       "  'end': 1205,\n",
       "  'text': 'In addition, embryoid body assays suggest that Patz1 represses developmental genes.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24380431',\n",
       "  'score': 0.12269043922424316},\n",
       " {'start': 1207,\n",
       "  'end': 1283,\n",
       "  'text': 'Together, these results propose that Patz1 is important for ESC pluripotency.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24380431',\n",
       "  'score': 0.2886764407157898},\n",
       " {'start': 0,\n",
       "  'end': 77,\n",
       "  'text': 'Protein Kinases and Their Inhibitors in Pluripotent Stem Cell Fate Regulation.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '31428157',\n",
       "  'score': 0.32658523321151733},\n",
       " {'start': 0,\n",
       "  'end': 193,\n",
       "  'text': 'Protein kinases modulate the reversible postmodifications of substrate proteins to their phosphorylated forms as an essential process in regulating intracellular signaling transduction cascades.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31428157',\n",
       "  'score': 0.1494547426700592},\n",
       " {'start': 195,\n",
       "  'end': 480,\n",
       "  'text': 'Moreover, phosphorylation has recently been shown to tightly control the regulatory network of kinases responsible for the induction and maintenance of pluripotency, defined as the particular ability to differentiate pluripotent stem cells (PSCs) into every cell type in the adult body.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31428157',\n",
       "  'score': 0.3523107171058655},\n",
       " {'start': 482,\n",
       "  'end': 670,\n",
       "  'text': 'In particular, emerging evidence indicates that the balance between the self-renewal and differentiation of PSCs is regulated by the small molecules that modulate kinase signaling pathways.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31428157',\n",
       "  'score': 0.13033080101013184},\n",
       " {'start': 672,\n",
       "  'end': 923,\n",
       "  'text': 'Furthermore, new reprogramming technologies have been developed using kinase modulators, which have provided novel insight of the mechanisms underlying the kinase regulatory networks involved in the generation of induced pluripotent stem cells (iPSCs).',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31428157',\n",
       "  'score': 0.3503033518791199},\n",
       " {'start': 925,\n",
       "  'end': 1184,\n",
       "  'text': 'In this review, we highlight the recent progress made in defining the roles of protein kinase signaling pathways and their small molecule modulators in regulating the pluripotent states, self-renewal, reprogramming process, and lineage differentiation of PSCs.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31428157',\n",
       "  'score': 0.3596958816051483},\n",
       " {'start': 0,\n",
       "  'end': 79,\n",
       "  'text': 'The Transcription Factor-microRNA Regulatory Network during hESC-chondrogenesis.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '32179818',\n",
       "  'score': 0.14655354619026184},\n",
       " {'start': 0,\n",
       "  'end': 96,\n",
       "  'text': 'Human embryonic stem cells (ESCs) offer a promising therapeutic approach for osteoarthritis (OA).',\n",
       "  'is_title': False,\n",
       "  'doc_id': '32179818',\n",
       "  'score': 0.27788612246513367},\n",
       " {'start': 98,\n",
       "  'end': 263,\n",
       "  'text': 'The unlimited source of cells capable of differentiating to chondrocytes has potential for repairing damaged cartilage or to generate disease models via gene editing.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '32179818',\n",
       "  'score': 0.12423697113990784},\n",
       " {'start': 265,\n",
       "  'end': 343,\n",
       "  'text': 'However their use is limited by the efficiency of chondrogenic differentiation.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '32179818',\n",
       "  'score': 0.12568259239196777},\n",
       " {'start': 345,\n",
       "  'end': 517,\n",
       "  'text': 'An improved understanding of the transcriptional and post-transcriptional regulation of chondrogenesis will enable us to improve hESC chondrogenic differentiation protocols.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '32179818',\n",
       "  'score': 0.1457291543483734},\n",
       " {'start': 519,\n",
       "  'end': 632,\n",
       "  'text': 'Small RNA-seq and whole transcriptome sequencing was performed on distinct stages of hESC-directed chondrogenesis.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '32179818',\n",
       "  'score': 0.12464895844459534},\n",
       " {'start': 634,\n",
       "  'end': 875,\n",
       "  'text': 'This revealed significant changes in the expression of several microRNAs including upregulation of known cartilage associated microRNAs and those transcribed from the Hox complexes, and the downregulation of pluripotency associated microRNAs.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '32179818',\n",
       "  'score': 0.3660372197628021},\n",
       " {'start': 877,\n",
       "  'end': 1151,\n",
       "  'text': 'Integration of miRomes and transcriptomes generated during hESC-directed chondrogenesis identified key functionally related clusters of co-expressed microRNAs and protein coding genes, associated with pluripotency, primitive streak, limb development and extracellular matrix.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '32179818',\n",
       "  'score': 0.32369381189346313},\n",
       " {'start': 1153,\n",
       "  'end': 1474,\n",
       "  'text': \"Analysis identified regulators of hESC-directed chondrogenesis such as miR-29c-3p with 10 of its established targets identified as co-regulated 'ECM organisation' genes and miR-22-3p which is highly co-expressed with ECM genes and may regulate these genes indirectly by targeting the chondrogenic regulators SP1 and HDAC4.\",\n",
       "  'is_title': False,\n",
       "  'doc_id': '32179818',\n",
       "  'score': 0.16089516878128052},\n",
       " {'start': 1476,\n",
       "  'end': 1662,\n",
       "  'text': 'We identified several upregulated transcription factors including HOXA9/A10/D13 involved in limb patterning and RELA, JUN and NFAT5, which have targets enriched with ECM associated genes.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '32179818',\n",
       "  'score': 0.1360066831111908},\n",
       " {'start': 1664,\n",
       "  'end': 1913,\n",
       "  'text': 'We have developed an unbiased approach for integrating transcriptome and miRome using protein-protein interactions, transcription factor regulation and miRNA target interactions and identified key regulatory networks prominent in hESC chondrogenesis.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '32179818',\n",
       "  'score': 0.18140000104904175},\n",
       " {'start': 0,\n",
       "  'end': 95,\n",
       "  'text': 'PRDM15 safeguards naive pluripotency by transcriptionally regulating WNT and MAPK-ERK signaling.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '28740264',\n",
       "  'score': 0.32720839977264404},\n",
       " {'start': 0,\n",
       "  'end': 173,\n",
       "  'text': 'The transcriptional network acting downstream of LIF, WNT and MAPK-ERK to stabilize mouse embryonic stem cells (ESCs) in their naive state has been extensively characterized.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28740264',\n",
       "  'score': 0.3634990453720093},\n",
       " {'start': 175,\n",
       "  'end': 271,\n",
       "  'text': 'However, the upstream factors regulating these three signaling pathways remain largely uncharted.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28740264',\n",
       "  'score': 0.14423120021820068},\n",
       " {'start': 273,\n",
       "  'end': 437,\n",
       "  'text': 'PR-domain-containing proteins (PRDMs) are zinc-finger sequence-specific chromatin factors that have essential roles in embryonic development and cell fate decisions.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28740264',\n",
       "  'score': 0.12530022859573364},\n",
       " {'start': 439,\n",
       "  'end': 574,\n",
       "  'text': 'Here we characterize the transcriptional regulator PRDM15, which acts independently of PRDM14 to regulate the naive state of mouse ESCs.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28740264',\n",
       "  'score': 0.2784096300601959},\n",
       " {'start': 576,\n",
       "  'end': 716,\n",
       "  'text': 'Mechanistically, PRDM15 modulates WNT and MAPK-ERK signaling by directly promoting the expression of Rspo1 (R-spondin1) and Spry1 (Sprouty1).',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28740264',\n",
       "  'score': 0.1260775327682495},\n",
       " {'start': 718,\n",
       "  'end': 966,\n",
       "  'text': 'Consistent with these findings, CRISPR-Cas9-mediated disruption of PRDM15-binding sites in the Rspo1 and Spry1 promoters recapitulates PRDM15 depletion, both in terms of local chromatin organization and the transcriptional modulation of these genes.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28740264',\n",
       "  'score': 0.16184517741203308},\n",
       " {'start': 968,\n",
       "  'end': 1171,\n",
       "  'text': 'Collectively, our findings uncover an essential role for PRDM15 as a chromatin factor that modulates the transcription of upstream regulators of WNT and MAPK-ERK signaling to safeguard naive pluripotency.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28740264',\n",
       "  'score': 0.2829882502555847},\n",
       " {'start': 0,\n",
       "  'end': 126,\n",
       "  'text': 'DNA Replication Inhibitor Geminin and Retinoic Acid Signaling Participate in Complex Interactions Associated With Pluripotency.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '31659113',\n",
       "  'score': 0.3197113871574402},\n",
       " {'start': 0,\n",
       "  'end': 112,\n",
       "  'text': 'BACKGROUND/AIM\\nSeveral links between DNA replication, pluripotency and development have been recently identified.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31659113',\n",
       "  'score': 0.2944391965866089},\n",
       " {'start': 114,\n",
       "  'end': 228,\n",
       "  'text': 'The involvement of miRNA in the regulation of cell cycle events and pluripotency factors has also gained attention.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31659113',\n",
       "  'score': 0.36008602380752563},\n",
       " {'start': 232,\n",
       "  'end': 461,\n",
       "  'text': 'MATERIALS AND METHODS\\nIn the present study, we used the g:Profiler platform to analyze transcription factor binding sites, miRNA networks and protein-protein interactions to identify novel links among the aforementioned processes.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31659113',\n",
       "  'score': 0.16147297620773315},\n",
       " {'start': 465,\n",
       "  'end': 653,\n",
       "  'text': 'RESULTS AND CONCLUSION\\nA complex circuitry between retinoic acid signaling, SWI/SNF components, pluripotency factors including Oct4, Sox2 and Nanog and cell cycle regulators was identified.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31659113',\n",
       "  'score': 0.2799634337425232},\n",
       " {'start': 655,\n",
       "  'end': 752,\n",
       "  'text': 'It is suggested that the DNA replication inhibitor geminin plays a central role in this circuitry.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31659113',\n",
       "  'score': 0.12780997157096863},\n",
       " {'start': 0,\n",
       "  'end': 59,\n",
       "  'text': 'Nanog, Oct4 and Tet1 interplay in establishing pluripotency.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '27146218',\n",
       "  'score': 0.3121066093444824},\n",
       " {'start': 0,\n",
       "  'end': 163,\n",
       "  'text': \"A few central transcription factors inside mouse embryonic stem (ES) cells and induced pluripotent stem (iPS) cells are believed to control the cells' pluripotency.\",\n",
       "  'is_title': False,\n",
       "  'doc_id': '27146218',\n",
       "  'score': 0.2721973657608032},\n",
       " {'start': 165,\n",
       "  'end': 271,\n",
       "  'text': 'Characterizations of pluripotent state were put forward on both transcription factor and epigenetic levels.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '27146218',\n",
       "  'score': 0.3134813606739044},\n",
       " {'start': 273,\n",
       "  'end': 460,\n",
       "  'text': 'Whereas core players have been identified, it is desirable to map out gene regulatory networks which govern the reprogramming of somatic cells as well as the early developmental decisions.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '27146218',\n",
       "  'score': 0.15231949090957642},\n",
       " {'start': 462,\n",
       "  'end': 652,\n",
       "  'text': 'Here we propose a multiple level model where the regulatory network of Oct4, Nanog and Tet1 includes positive feedback loops involving DNA-demethylation around the promoters of Oct4 and Tet1.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '27146218',\n",
       "  'score': 0.35640037059783936},\n",
       " {'start': 654,\n",
       "  'end': 1168,\n",
       "  'text': 'We put forward a mechanistic understanding of the regulatory dynamics which account for i) Oct4 overexpression is sufficient to induce pluripotency in somatic cell types expressing the other Yamanaka reprogramming factors endogenously; ii) Tet1 can replace Oct4 in reprogramming cocktail; iii) Nanog is not necessary for reprogramming however its over-expression leads to enhanced self-renewal; iv) DNA methylation is the key to the regulation of pluripotency genes; v) Lif withdrawal leads to loss of pluripotency.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '27146218',\n",
       "  'score': 0.35113525390625},\n",
       " {'start': 1170,\n",
       "  'end': 1416,\n",
       "  'text': 'Overall, our paper proposes a novel framework combining transcription regulation with DNA methylation modifications which, takes into account the multi-layer nature of regulatory mechanisms governing pluripotency acquisition through reprogramming.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '27146218',\n",
       "  'score': 0.31794777512550354},\n",
       " {'start': 0,\n",
       "  'end': 88,\n",
       "  'text': 'Protein kinases and associated pathways in pluripotent state and lineage differentiation.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '24998240',\n",
       "  'score': 0.31817471981048584},\n",
       " {'start': 0,\n",
       "  'end': 177,\n",
       "  'text': 'Protein kinases (PKs) mediate the reversible conversion of substrate proteins to phosphorylated forms, a key process in controlling intracellular signaling transduction cascades.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24998240',\n",
       "  'score': 0.13507691025733948},\n",
       " {'start': 179,\n",
       "  'end': 358,\n",
       "  'text': 'Pluripotency is, among others, characterized by specifically expressed PKs forming a highly interconnected regulatory network that culminates in a finely-balanced molecular switch.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24998240',\n",
       "  'score': 0.35417917370796204},\n",
       " {'start': 360,\n",
       "  'end': 511,\n",
       "  'text': 'Current high-throughput phosphoproteomic approaches have shed light on the specific regulatory PKs and their function in controlling pluripotent states.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24998240',\n",
       "  'score': 0.2677290439605713},\n",
       " {'start': 513,\n",
       "  'end': 681,\n",
       "  'text': 'Pluripotent cell-derived endothelial and hematopoietic developments represent an example of the importance of pluripotency in cancer therapeutics and organ regeneration.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24998240',\n",
       "  'score': 0.33453667163848877},\n",
       " {'start': 683,\n",
       "  'end': 834,\n",
       "  'text': 'This review attempts to provide the hitherto known kinome profile and the individual characterization of PK-related pathways that regulate pluripotency.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24998240',\n",
       "  'score': 0.2740197777748108},\n",
       " {'start': 836,\n",
       "  'end': 1213,\n",
       "  'text': 'Elucidating the underlying intrinsic and extrinsic signals may improve our understanding of the different pluripotent states, the maintenance or induction of pluripotency, and the ability to tailor lineage differentiation, with a particular focus on endothelial cell differentiation for anti-cancer treatment, cell-based tissue engineering, and regenerative medicine strategies.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24998240',\n",
       "  'score': 0.31702467799186707},\n",
       " {'start': 0,\n",
       "  'end': 57,\n",
       "  'text': 'Mechanosensitive mechanisms in transcriptional regulation.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '22797927',\n",
       "  'score': 0.14120548963546753},\n",
       " {'start': 0,\n",
       "  'end': 144,\n",
       "  'text': 'Transcriptional regulation contributes to the maintenance of pluripotency, self-renewal and differentiation in embryonic cells and in stem cells.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '22797927',\n",
       "  'score': 0.3630203902721405},\n",
       " {'start': 146,\n",
       "  'end': 349,\n",
       "  'text': 'Therefore, control of gene expression at the level of transcription is crucial for embryonic development, as well as for organogenesis, functional adaptation, and regeneration in adult tissues and organs.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '22797927',\n",
       "  'score': 0.13322484493255615},\n",
       " {'start': 351,\n",
       "  'end': 560,\n",
       "  'text': 'In the past, most work has focused on how transcriptional regulation results from the complex interplay between chemical cues, adhesion signals, transcription factors and their co-regulators during development.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '22797927',\n",
       "  'score': 0.14933377504348755},\n",
       " {'start': 562,\n",
       "  'end': 766,\n",
       "  'text': 'However, chemical signaling alone is not sufficient to explain how three-dimensional (3D) tissues and organs are constructed and maintained through the spatiotemporal control of transcriptional activities.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '22797927',\n",
       "  'score': 0.1849413812160492},\n",
       " {'start': 768,\n",
       "  'end': 855,\n",
       "  'text': 'Accumulated evidence indicates that mechanical cues, which include physical forces (e.g.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '22797927',\n",
       "  'score': 0.12499180436134338},\n",
       " {'start': 857,\n",
       "  'end': 1123,\n",
       "  'text': 'tension, compression or shear stress), alterations in extracellular matrix (ECM) mechanics and changes in cell shape, are transmitted to the nucleus directly or indirectly to orchestrate transcriptional activities that are crucial for embryogenesis and organogenesis.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '22797927',\n",
       "  'score': 0.12888354063034058},\n",
       " {'start': 1125,\n",
       "  'end': 1429,\n",
       "  'text': 'In this Commentary, we review how the mechanical control of gene transcription contributes to the maintenance of pluripotency, determination of cell fate, pattern formation and organogenesis, as well as how it is involved in the control of cell and tissue function throughout embryogenesis and adult life.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '22797927',\n",
       "  'score': 0.35613518953323364},\n",
       " {'start': 1431,\n",
       "  'end': 1592,\n",
       "  'text': 'A deeper understanding of these mechanosensitive transcriptional control mechanisms should lead to new approaches to tissue engineering and regenerative medicine.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '22797927',\n",
       "  'score': 0.1266523003578186},\n",
       " {'start': 0,\n",
       "  'end': 59,\n",
       "  'text': 'Transcriptional regulatory networks in embryonic stem cells.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '21141733',\n",
       "  'score': 0.1519191861152649},\n",
       " {'start': 0,\n",
       "  'end': 172,\n",
       "  'text': 'Transcriptional regulation is one of the most fundamental processes in biology, governing the morphology, function, and behavior of cells and thus the survival of organisms.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '21141733',\n",
       "  'score': 0.14974936842918396},\n",
       " {'start': 174,\n",
       "  'end': 299,\n",
       "  'text': 'The embryonic stem cell (ESC) provides a good model for the understanding of transcriptional regulation in vertebrate systems.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '21141733',\n",
       "  'score': 0.14885759353637695},\n",
       " {'start': 301,\n",
       "  'end': 452,\n",
       "  'text': 'Recent efforts have led to the identification of molecular events, which confer upon these cells the unique properties of pluripotency and self renewal.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '21141733',\n",
       "  'score': 0.32400214672088623},\n",
       " {'start': 454,\n",
       "  'end': 566,\n",
       "  'text': 'The core regulatory network maintaining the ESC identity involves three master regulators: Oct4, Sox2, and Nanog.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '21141733',\n",
       "  'score': 0.3651459217071533},\n",
       " {'start': 568,\n",
       "  'end': 730,\n",
       "  'text': 'Large-scale mapping studies interrogating the binding sites of these and other transcription factors showed co-occupancy of distinct sets of transcription factors.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '21141733',\n",
       "  'score': 0.1291850209236145},\n",
       " {'start': 732,\n",
       "  'end': 879,\n",
       "  'text': 'The assembly of multitranscription factor complexes could serve as a mechanism for providing specificity in regulating ESC-specific gene expression.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '21141733',\n",
       "  'score': 0.14708253741264343},\n",
       " {'start': 881,\n",
       "  'end': 993,\n",
       "  'text': 'These studies are also beginning to unravel the transcriptional regulatory networks that govern the ESC identity.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '21141733',\n",
       "  'score': 0.14661765098571777},\n",
       " {'start': 995,\n",
       "  'end': 1119,\n",
       "  'text': 'Loss-of-function RNAi screens also identified novel regulatory molecules involved in the stable propagation of the ESC state.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '21141733',\n",
       "  'score': 0.1355399191379547},\n",
       " {'start': 1121,\n",
       "  'end': 1403,\n",
       "  'text': 'This argues for an ESC transcriptional regulation program in which interconnected transcriptional regulatory networks involving large numbers of transcription factors and epigenetic modifiers work in concert on ESC- and differentiation-specific genes to achieve cell state stability.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '21141733',\n",
       "  'score': 0.17287743091583252},\n",
       " {'start': 1405,\n",
       "  'end': 1606,\n",
       "  'text': 'This chapter traces the major efforts made over the past decade in dissecting the transcriptional regulatory network governing ESC identity and offers perspectives on the future directions of the field.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '21141733',\n",
       "  'score': 0.1503174901008606},\n",
       " {'start': 0,\n",
       "  'end': 71,\n",
       "  'text': 'Dynamic lineage priming is driven via direct enhancer regulation by ERK.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '31695196',\n",
       "  'score': 0.22688543796539307},\n",
       " {'start': 0,\n",
       "  'end': 185,\n",
       "  'text': 'Central to understanding cellular behaviour in multi-cellular organisms is the question of how a cell exits one transcriptional state to adopt and eventually become committed to another.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31695196',\n",
       "  'score': 0.13288241624832153},\n",
       " {'start': 187,\n",
       "  'end': 442,\n",
       "  'text': 'Fibroblast growth factor-extracellular signal-regulated kinase (FGF -ERK) signalling drives differentiation of mouse embryonic stem cells (ES cells) and pre-implantation embryos towards primitive endoderm, and inhibiting ERK supports ES cell self-renewal1.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31695196',\n",
       "  'score': 0.12736180424690247},\n",
       " {'start': 444,\n",
       "  'end': 631,\n",
       "  'text': 'Paracrine FGF-ERK signalling induces heterogeneity, whereby cells reversibly progress from pluripotency towards primitive endoderm while retaining their capacity to re-enter self-renewal2.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31695196',\n",
       "  'score': 0.308447927236557},\n",
       " {'start': 633,\n",
       "  'end': 800,\n",
       "  'text': 'Here we find that ERK reversibly regulates transcription in ES cells by directly affecting enhancer activity without requiring a change in transcription factor binding.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31695196',\n",
       "  'score': 0.19502729177474976},\n",
       " {'start': 802,\n",
       "  'end': 1035,\n",
       "  'text': 'ERK triggers the reversible association and disassociation of RNA polymerase II and associated co-factors from genes and enhancers with the mediator component MED24 having an essential role in ERK-dependent transcriptional regulation.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31695196',\n",
       "  'score': 0.22788551449775696},\n",
       " {'start': 1037,\n",
       "  'end': 1284,\n",
       "  'text': 'Though the binding of mediator components responds directly to signalling, the persistent binding of pluripotency factors to both induced and repressed genes marks them for activation and/or reactivation in response to fluctuations in ERK activity.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31695196',\n",
       "  'score': 0.3259464502334595},\n",
       " {'start': 1286,\n",
       "  'end': 1519,\n",
       "  'text': 'Among the repressed genes are several core components of the pluripotency network that act to drive their own expression and maintain the ES cell state; if their binding is lost, the ability to reactivate transcription is compromised.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31695196',\n",
       "  'score': 0.3520236015319824},\n",
       " {'start': 1521,\n",
       "  'end': 1671,\n",
       "  'text': 'Thus, as long as transcription factor occupancy is maintained, so is plasticity, enabling cells to distinguish between transient and sustained signals.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31695196',\n",
       "  'score': 0.1280554234981537},\n",
       " {'start': 1673,\n",
       "  'end': 1830,\n",
       "  'text': 'If ERK signalling persists, pluripotency transcription factor levels are reduced by protein turnover and irreversible gene silencing and commitment can occur.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31695196',\n",
       "  'score': 0.3071673512458801},\n",
       " {'start': 0,\n",
       "  'end': 47,\n",
       "  'text': 'Taming human brain organoids one cell at a time.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '32718852',\n",
       "  'score': 0.12541189789772034},\n",
       " {'start': 0,\n",
       "  'end': 210,\n",
       "  'text': 'Human brain organoids are self-organizing three-dimensional structures that emerge from human pluripotent stem cells and mimic aspects of the cellular composition and functionality of the developing human brain.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '32718852',\n",
       "  'score': 0.37641382217407227},\n",
       " {'start': 212,\n",
       "  'end': 467,\n",
       "  'text': 'Despite their impressive self-organizing capacity, organoids lack the stereotypic structural anatomy of their in vivo counterpart, making conventional analysis techniques underpowered to assess cellular composition and gene network regulation in organoids.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '32718852',\n",
       "  'score': 0.17145931720733643},\n",
       " {'start': 469,\n",
       "  'end': 705,\n",
       "  'text': 'Advances in single cell transcriptomics have recently allowed characterization and improvement of organoid protocols, as they continue to evolve, by enabling identification of cell types and states along with their developmental origins.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '32718852',\n",
       "  'score': 0.1307586431503296},\n",
       " {'start': 707,\n",
       "  'end': 857,\n",
       "  'text': \"In this review, we summarize recent approaches, progresses and challenges in resolving brain organoid's complexity through single-cell transcriptomics.\",\n",
       "  'is_title': False,\n",
       "  'doc_id': '32718852',\n",
       "  'score': 0.12988168001174927},\n",
       " {'start': 859,\n",
       "  'end': 1054,\n",
       "  'text': 'We then discuss emerging technologies that may complement single-cell RNA sequencing by providing additional readouts of cellular states to generate an organ-level view of developmental processes.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '32718852',\n",
       "  'score': 0.1264784336090088},\n",
       " {'start': 1056,\n",
       "  'end': 1352,\n",
       "  'text': 'Altogether, these integrative technologies will allow monitoring of global gene regulation in thousands of individual cells and will offer an unprecedented opportunity to investigate features of human brain development and disease across multiple cellular modalities and with cell-type resolution.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '32718852',\n",
       "  'score': 0.14569297432899475},\n",
       " {'start': 0,\n",
       "  'end': 115,\n",
       "  'text': 'Assembly of FN-silk with laminin-521 to integrate hPSCs into a three-dimensional culture for neural differentiation.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '32215392',\n",
       "  'score': 0.18806323409080505},\n",
       " {'start': 0,\n",
       "  'end': 185,\n",
       "  'text': 'Three-dimensional (3D) neural tissue cultures recapitulate the basic concepts during development and disease better than what can be obtained using conventional two-dimensional cultures.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '32215392',\n",
       "  'score': 0.1900118589401245},\n",
       " {'start': 187,\n",
       "  'end': 410,\n",
       "  'text': 'Here, we use a recombinant spider silk protein functionalized with a cell binding motif from fibronectin (FN-silk) in combination with a human recombinant laminin 521 (LN-521) to create a fully defined stem cell niche in 3D.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '32215392',\n",
       "  'score': 0.17017602920532227},\n",
       " {'start': 412,\n",
       "  'end': 648,\n",
       "  'text': 'A novel method to assemble silk blended with LN-521 together with human pluripotent stem cells (hPSC) is used to create centimeter-sized foams, which upon cultivation develop into 3D cell constructs supported by a microfibrillar network.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '32215392',\n",
       "  'score': 0.36491045355796814},\n",
       " {'start': 650,\n",
       "  'end': 837,\n",
       "  'text': 'After initial cell expansion, neural differentiation was induced to form a homogenous layer of continuous neuroectodermal tissue that allows further differentiation into neuronal subtypes.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '32215392',\n",
       "  'score': 0.12646859884262085},\n",
       " {'start': 839,\n",
       "  'end': 1047,\n",
       "  'text': 'The silk-supported 3D cell constructs could then be detached from the bottom of the well and cultured as floating entities, where cells appeared in distinctive radial organization resembling early neural tube.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '32215392',\n",
       "  'score': 0.2044319212436676},\n",
       " {'start': 1049,\n",
       "  'end': 1178,\n",
       "  'text': 'This shows that the neural progenitors retain their cellular self-organization ability in the FN-silk/LN-521-supported 3D culture.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '32215392',\n",
       "  'score': 0.20968371629714966},\n",
       " {'start': 1180,\n",
       "  'end': 1288,\n",
       "  'text': 'Calcium imaging demonstrated spontaneous activity, which is important for the formation of neuronal networks.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '32215392',\n",
       "  'score': 0.1544681191444397},\n",
       " {'start': 1290,\n",
       "  'end': 1456,\n",
       "  'text': 'Together, the results show that hPSCs integrated into FN-silk/LN-521 foam develop into neural progenitors and that these stay viable during long-term differentiations.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '32215392',\n",
       "  'score': 0.1290324330329895},\n",
       " {'start': 1458,\n",
       "  'end': 1606,\n",
       "  'text': 'FN-silk/LN-521 also supports morphogenesis mimicking the human brain development and can serve as base for engineering of hPSC-derived neural tissue.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '32215392',\n",
       "  'score': 0.13067004084587097},\n",
       " {'start': 0,\n",
       "  'end': 90,\n",
       "  'text': 'Proteomic and network analysis of proteins regulated by REX1 in human embryonic stem cells.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '25736782',\n",
       "  'score': 0.15493479371070862},\n",
       " {'start': 0,\n",
       "  'end': 138,\n",
       "  'text': 'Recent studies have suggested that REX1 (reduced expression 1) plays an important role in pluripotency, proliferation, and differentiation.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '25736782',\n",
       "  'score': 0.29200220108032227},\n",
       " {'start': 140,\n",
       "  'end': 256,\n",
       "  'text': 'However, the molecular mechanisms involved in REX1-dependent regulation of diverse cellular processes remain unclear.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '25736782',\n",
       "  'score': 0.15819910168647766},\n",
       " {'start': 258,\n",
       "  'end': 428,\n",
       "  'text': 'To elucidate the regulatory functions of REX1 in human embryonic stem cells (hESCs), comparative proteomic analysis was performed on REX1 RNAi specifically silenced hESCs.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '25736782',\n",
       "  'score': 0.3034142851829529},\n",
       " {'start': 430,\n",
       "  'end': 744,\n",
       "  'text': 'Analysis of the proteome via nano-LC-MS/MS identified 140 differentially expressed proteins (DEPs) displaying a >2-fold difference in expression level between control and REX1 knockdown (KD) hESCs, which were then compared with transcriptome data and validated by quantitative real-time RT-PCR and Western blotting.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '25736782',\n",
       "  'score': 0.27733102440834045},\n",
       " {'start': 746,\n",
       "  'end': 905,\n",
       "  'text': 'These DEPs were analyzed by GO, pathway, and functional clustering analyses to determine the molecular functions of the proteins and pathways regulated by REX1.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '25736782',\n",
       "  'score': 0.13070592284202576},\n",
       " {'start': 907,\n",
       "  'end': 1056,\n",
       "  'text': 'The REX1 KD-mediated DEPs mapped to major biological processes involved in the regulation of ribosome-mediated translation and mitochondrial function.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '25736782',\n",
       "  'score': 0.16216900944709778},\n",
       " {'start': 1058,\n",
       "  'end': 1283,\n",
       "  'text': 'Functional network analysis revealed a highly interconnected network among these DEPs and indicated that these interconnected proteins are predominantly involved in translation and the regulation of mitochondrial organization.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '25736782',\n",
       "  'score': 0.2326642870903015},\n",
       " {'start': 1285,\n",
       "  'end': 1527,\n",
       "  'text': 'These findings regarding REX1-mediated regulatory network have revealed the contributions of REX1 to maintaining the status of hESCs and have improved our understanding of the molecular events that underlie the fundamental properties of hESCs.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '25736782',\n",
       "  'score': 0.34918534755706787},\n",
       " {'start': 0,\n",
       "  'end': 104,\n",
       "  'text': 'The p53 Family Coordinates Wnt and Nodal Inputs in Mesendodermal Differentiation of Embryonic Stem Cells.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '27889317',\n",
       "  'score': 0.13433071970939636},\n",
       " {'start': 0,\n",
       "  'end': 224,\n",
       "  'text': 'In this study, we outline a regulatory network that involves the p53 tumor suppressor family and the Wnt pathway acting together with the TGF-β pathway in mesendodermal differentiation of mouse and human embryonic stem cells.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '27889317',\n",
       "  'score': 0.15927597880363464},\n",
       " {'start': 226,\n",
       "  'end': 400,\n",
       "  'text': 'Knockout of all three members, p53, p63, and p73, shows that the p53 family is essential for mesendoderm specification during exit from pluripotency in embryos and in culture.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '27889317',\n",
       "  'score': 0.3394626975059509},\n",
       " {'start': 402,\n",
       "  'end': 584,\n",
       "  'text': 'Wnt3 and its receptor Fzd1 are direct p53 family target genes in this context, and induction of Wnt\\xa0signaling by p53 is critical for activation of\\xa0mesendodermal differentiation genes.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '27889317',\n",
       "  'score': 0.1339360773563385},\n",
       " {'start': 586,\n",
       "  'end': 764,\n",
       "  'text': 'Globally, Wnt3-activated Tcf3 and nodal-activated Smad2/3 transcription factors depend on each other for co-occupancy of target enhancers associated with key differentiation loci.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '27889317',\n",
       "  'score': 0.21422529220581055},\n",
       " {'start': 766,\n",
       "  'end': 1021,\n",
       "  'text': 'Our results therefore highlight an unanticipated role for p53 family proteins in a regulatory network that integrates essential Wnt-Tcf and nodal-Smad inputs in a selective and interdependent way to drive mesendodermal differentiation of pluripotent cells.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '27889317',\n",
       "  'score': 0.3530239760875702},\n",
       " {'start': 0,\n",
       "  'end': 88,\n",
       "  'text': 'On human pluripotent stem cell control: The rise of 3D bioengineering and mechanobiology.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '25818411',\n",
       "  'score': 0.35448741912841797},\n",
       " {'start': 0,\n",
       "  'end': 147,\n",
       "  'text': 'Human pluripotent stem cells (hPSCs) provide promising resources for regenerating tissues and organs and modeling development and diseases in vitro.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '25818411',\n",
       "  'score': 0.27423030138015747},\n",
       " {'start': 149,\n",
       "  'end': 363,\n",
       "  'text': 'To fulfill their promise, the fate, function, and organization of hPSCs need to be precisely regulated in a three-dimensional (3D) environment to mimic cellular structures and functions of native tissues and organs.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '25818411',\n",
       "  'score': 0.23433026671409607},\n",
       " {'start': 365,\n",
       "  'end': 527,\n",
       "  'text': 'In the past decade, innovations in 3D culture systems with functional biomaterials have enabled efficient and versatile control of hPSC fate at the cellular level.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '25818411',\n",
       "  'score': 0.18543899059295654},\n",
       " {'start': 529,\n",
       "  'end': 670,\n",
       "  'text': 'However, we are just at the beginning of bringing hPSC-based regeneration and development and disease modeling to the tissue and organ levels.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '25818411',\n",
       "  'score': 0.13100382685661316},\n",
       " {'start': 672,\n",
       "  'end': 886,\n",
       "  'text': 'In this review, we summarize existing bioengineered culture platforms for controlling hPSC fate and function by regulating inductive mechanical and biochemical cues coexisting in the synthetic cell microenvironment.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '25818411',\n",
       "  'score': 0.151963472366333},\n",
       " {'start': 888,\n",
       "  'end': 1047,\n",
       "  'text': 'We highlight recent excitements in developing 3D hPSC-based in vitro tissue and organ models with in vivo-like cellular structures, interactions, and functions.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '25818411',\n",
       "  'score': 0.17055419087409973},\n",
       " {'start': 1049,\n",
       "  'end': 1268,\n",
       "  'text': 'We further discuss an emerging multifaceted mechanotransductive signaling network--with transcriptional coactivators YAP and TAZ at the center stage--that regulate fates and behaviors of mammalian cells, including hPSCs.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '25818411',\n",
       "  'score': 0.15300211310386658},\n",
       " {'start': 1270,\n",
       "  'end': 1520,\n",
       "  'text': 'Future development of 3D biomaterial systems should incorporate dynamically modulated mechanical and chemical properties targeting specific intracellular signaling events leading to desirable hPSC fate patterning and functional tissue formation in 3D.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '25818411',\n",
       "  'score': 0.17026853561401367},\n",
       " {'start': 0,\n",
       "  'end': 61,\n",
       "  'text': 'Navigating the epigenetic landscape of pluripotent stem cells.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '22820889',\n",
       "  'score': 0.3059931993484497},\n",
       " {'start': 0,\n",
       "  'end': 236,\n",
       "  'text': 'Pluripotent stem cells, which include embryonic stem cells and induced pluripotent stem cells, use a complex network of genetic and epigenetic pathways to maintain a delicate balance between self-renewal and multilineage differentiation.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '22820889',\n",
       "  'score': 0.3193248212337494},\n",
       " {'start': 238,\n",
       "  'end': 366,\n",
       "  'text': 'Recently developed high-throughput genomic tools greatly facilitate the study of epigenetic regulation in pluripotent stem cells.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '22820889',\n",
       "  'score': 0.34369325637817383},\n",
       " {'start': 368,\n",
       "  'end': 501,\n",
       "  'text': 'Increasing evidence suggests the existence of extensive crosstalk among epigenetic pathways that modify DNA, histones and nucleosomes.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '22820889',\n",
       "  'score': 0.13160884380340576},\n",
       " {'start': 503,\n",
       "  'end': 818,\n",
       "  'text': 'Novel methods of mapping higher-order chromatin structure and chromatin-nuclear matrix interactions also provide the first insight into the three-dimensional organization of the genome and a framework in which existing genomic data of epigenetic regulation can be integrated to discover new rules of gene regulation.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '22820889',\n",
       "  'score': 0.2500002384185791},\n",
       " {'start': 0,\n",
       "  'end': 94,\n",
       "  'text': 'Cap-independent translation by DAP5 controls cell fate decisions in human embryonic stem cells.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '27664238',\n",
       "  'score': 0.12957125902175903},\n",
       " {'start': 0,\n",
       "  'end': 100,\n",
       "  'text': 'Multiple transcriptional and epigenetic changes drive differentiation of embryonic stem cells (ESCs).',\n",
       "  'is_title': False,\n",
       "  'doc_id': '27664238',\n",
       "  'score': 0.2989666163921356},\n",
       " {'start': 102,\n",
       "  'end': 249,\n",
       "  'text': 'This study unveils an additional level of gene expression regulation involving noncanonical, cap-independent translation of a select group of mRNAs.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '27664238',\n",
       "  'score': 0.14663276076316833},\n",
       " {'start': 251,\n",
       "  'end': 384,\n",
       "  'text': 'This is driven by death-associated protein 5 (DAP5/eIF4G2/NAT1), a translation initiation factor mediating IRES-dependent translation.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '27664238',\n",
       "  'score': 0.14073753356933594},\n",
       " {'start': 386,\n",
       "  'end': 618,\n",
       "  'text': 'We found that the DAP5 knockdown from human ESCs (hESCs) resulted in persistence of pluripotent gene expression, delayed induction of differentiation-associated genes in different cell lineages, and defective embryoid body formation.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '27664238',\n",
       "  'score': 0.34000664949417114},\n",
       " {'start': 620,\n",
       "  'end': 727,\n",
       "  'text': 'The latter involved improper cellular organization, lack of cavitation, and enhanced mislocalized apoptosis.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '27664238',\n",
       "  'score': 0.16661763191223145},\n",
       " {'start': 729,\n",
       "  'end': 853,\n",
       "  'text': 'RNA sequencing of polysome-associated mRNAs identified candidates with reduced translation efficiency in DAP5-depleted hESCs.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '27664238',\n",
       "  'score': 0.3179702162742615},\n",
       " {'start': 855,\n",
       "  'end': 1126,\n",
       "  'text': 'These were enriched in mitochondrial proteins involved in oxidative respiration, a pathway essential for differentiation, the significance of which was confirmed by the aberrant mitochondrial morphology and decreased oxidative respiratory activity in DAP5 knockdown cells.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '27664238',\n",
       "  'score': 0.14124780893325806},\n",
       " {'start': 1128,\n",
       "  'end': 1283,\n",
       "  'text': 'Further analysis identified the chromatin modifier HMGN3 as a cap-independent DAP5 translation target whose knockdown resulted in defective differentiation.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '27664238',\n",
       "  'score': 0.13116124272346497},\n",
       " {'start': 1285,\n",
       "  'end': 1502,\n",
       "  'text': 'Thus, DAP5-mediated translation of a specific set of proteins is critical for the transition from pluripotency to differentiation, highlighting the importance of cap-independent translation in stem cell fate decisions.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '27664238',\n",
       "  'score': 0.3211459517478943},\n",
       " {'start': 0,\n",
       "  'end': 107,\n",
       "  'text': 'The high mobility group protein HMGA2: a co-regulator of chromatin structure and pluripotency in stem cells?',\n",
       "  'is_title': True,\n",
       "  'doc_id': '19551524',\n",
       "  'score': 0.34748154878616333},\n",
       " {'start': 0,\n",
       "  'end': 162,\n",
       "  'text': 'The small, chromatin-associated HMGA proteins contain three separate DNA binding domains, so-called AT hooks, which bind preferentially to short AT-rich sequences.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '19551524',\n",
       "  'score': 0.14240971207618713},\n",
       " {'start': 164,\n",
       "  'end': 312,\n",
       "  'text': 'These proteins are abundant in pluripotent embryonic stem (ES) cells and most malignant human tumors, but are not detectable in normal somatic cells.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '19551524',\n",
       "  'score': 0.28186798095703125},\n",
       " {'start': 314,\n",
       "  'end': 513,\n",
       "  'text': 'They act both as activator and repressor of gene expression, and most likely facilitate DNA architectural changes during formation of specialized nucleoprotein structures at selected promoter regions.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '19551524',\n",
       "  'score': 0.20867693424224854},\n",
       " {'start': 515,\n",
       "  'end': 704,\n",
       "  'text': 'For example, HMGA2 is involved in transcriptional activation of certain cell proliferation genes, which likely contributes to its well-established oncogenic potential during tumor formation.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '19551524',\n",
       "  'score': 0.14384377002716064},\n",
       " {'start': 706,\n",
       "  'end': 861,\n",
       "  'text': 'However, surprisingly little is known about how HMGA proteins bind DNA packaged in chromatin and how this affects the chromatin structure at a larger scale.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '19551524',\n",
       "  'score': 0.13657796382904053},\n",
       " {'start': 863,\n",
       "  'end': 963,\n",
       "  'text': 'Experimental evidence suggests that HMGA2 competes with binding of histone H1 in the chromatin fiber.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '19551524',\n",
       "  'score': 0.1348886489868164},\n",
       " {'start': 965,\n",
       "  'end': 1102,\n",
       "  'text': 'This could substantially alter chromatin domain structures in ES cells and contribute to the activation of certain transcription networks.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '19551524',\n",
       "  'score': 0.16274115443229675},\n",
       " {'start': 1104,\n",
       "  'end': 1220,\n",
       "  'text': 'HMGA2 also seems capable of recruiting enzymes directly involved in histone modifications to trigger gene expression.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '19551524',\n",
       "  'score': 0.1381511688232422},\n",
       " {'start': 1222,\n",
       "  'end': 1351,\n",
       "  'text': 'Furthermore, it was shown that multiple HMGA2 molecules bind stably to a single nucleosome core particle whose structure is known.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '19551524',\n",
       "  'score': 0.13096222281455994},\n",
       " {'start': 1353,\n",
       "  'end': 1446,\n",
       "  'text': 'How these features of HMGA2 impinge on chromatin organization inside a living cell is unknown.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '19551524',\n",
       "  'score': 0.16340762376785278},\n",
       " {'start': 1448,\n",
       "  'end': 1685,\n",
       "  'text': 'In this commentary, we propose that HMGA2, through the action of three independent DNA binding domains, substantially contributes to the plasticity of ES cell chromatin and is involved in the maintenance of a un-differentiated cell state.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '19551524',\n",
       "  'score': 0.1536206305027008},\n",
       " {'start': 0,\n",
       "  'end': 68,\n",
       "  'text': 'Metabolic remodeling during the loss and acquisition of pluripotency.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '28196802',\n",
       "  'score': 0.3320280909538269},\n",
       " {'start': 0,\n",
       "  'end': 183,\n",
       "  'text': 'Pluripotent cells from the early stages of embryonic development have the unlimited capacity to self-renew and undergo differentiation into all of the cell types of the adult organism.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28196802',\n",
       "  'score': 0.30469852685928345},\n",
       " {'start': 185,\n",
       "  'end': 401,\n",
       "  'text': 'These properties are regulated by tightly controlled networks of gene expression, which in turn are governed by the availability of transcription factors and their interaction with the underlying epigenetic landscape.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28196802',\n",
       "  'score': 0.16373902559280396},\n",
       " {'start': 403,\n",
       "  'end': 558,\n",
       "  'text': 'Recent data suggest that, perhaps unexpectedly, some key epigenetic marks, and thereby gene expression, are regulated by the levels of specific metabolites.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28196802',\n",
       "  'score': 0.13528311252593994},\n",
       " {'start': 560,\n",
       "  'end': 696,\n",
       "  'text': 'Hence, cellular metabolism plays a vital role beyond simply the production of energy, and may be involved in the regulation of cell fate.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28196802',\n",
       "  'score': 0.1640891432762146},\n",
       " {'start': 698,\n",
       "  'end': 1014,\n",
       "  'text': 'In this Review, we discuss the metabolic changes that occur during the transitions between different pluripotent states both in vitro and in vivo, including during reprogramming to pluripotency and the onset of differentiation, and we discuss the extent to which distinct metabolites might regulate these transitions.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28196802',\n",
       "  'score': 0.32108139991760254},\n",
       " {'start': 0,\n",
       "  'end': 63,\n",
       "  'text': 'PRC1 proteins orchestrate three-dimensional genome architecture.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '26417860',\n",
       "  'score': 0.1906348466873169},\n",
       " {'start': 0,\n",
       "  'end': 144,\n",
       "  'text': 'The three-dimensional organization of the genome has an important role in orchestrating gene expression, but its regulation is poorly understood.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '26417860',\n",
       "  'score': 0.27219128608703613},\n",
       " {'start': 146,\n",
       "  'end': 316,\n",
       "  'text': 'Now, a new study uncovers a major role for Polycomb components of the PRC1 complex in organizing physical networks of genes that are co-repressed to maintain pluripotency.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '26417860',\n",
       "  'score': 0.3448425233364105},\n",
       " {'start': 0,\n",
       "  'end': 111,\n",
       "  'text': 'Wnt/Yes-Associated Protein Interactions During Neural Tissue Patterning of Human Induced Pluripotent Stem Cells.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '28726548',\n",
       "  'score': 0.33925867080688477},\n",
       " {'start': 0,\n",
       "  'end': 135,\n",
       "  'text': 'Human induced pluripotent stem cells (hiPSCs) have special ability to self-assemble into neural spheroids or mini-brain-like structures.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28726548',\n",
       "  'score': 0.2946568727493286},\n",
       " {'start': 137,\n",
       "  'end': 304,\n",
       "  'text': 'During the self-assembly process, Wnt signaling plays an important role in regional patterning and establishing positional identity of hiPSC-derived neural progenitors.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28726548',\n",
       "  'score': 0.1388946771621704},\n",
       " {'start': 306,\n",
       "  'end': 521,\n",
       "  'text': 'Recently, the role of Wnt signaling in regulating Yes-associated protein (YAP) expression (nuclear or cytoplasmic), the pivotal regulator during organ growth and tissue generation, has attracted increasing interests.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28726548',\n",
       "  'score': 0.1723305881023407},\n",
       " {'start': 523,\n",
       "  'end': 642,\n",
       "  'text': 'However, the interactions between Wnt and YAP expression for neural lineage commitment of hiPSCs remain poorly explored.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28726548',\n",
       "  'score': 0.13696685433387756},\n",
       " {'start': 644,\n",
       "  'end': 827,\n",
       "  'text': 'The objective of this study is to investigate the effects of Wnt signaling and YAP expression on the cellular population in three-dimensional (3D) neural spheroids derived from hiPSCs.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28726548',\n",
       "  'score': 0.20046281814575195},\n",
       " {'start': 829,\n",
       "  'end': 974,\n",
       "  'text': 'In this study, Wnt signaling was activated using CHIR99021 for 3D neural spheroids derived from human iPSK3 cells through embryoid body formation.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28726548',\n",
       "  'score': 0.18246540427207947},\n",
       " {'start': 976,\n",
       "  'end': 1130,\n",
       "  'text': 'Our results indicate that Wnt activation induces nuclear localization of YAP and upregulates the expression of HOXB4, the marker for hindbrain/spinal cord.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28726548',\n",
       "  'score': 0.13647699356079102},\n",
       " {'start': 1132,\n",
       "  'end': 1245,\n",
       "  'text': 'By contrast, the cells exhibit more rostral forebrain neural identity (expression of TBR1) without Wnt activation.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28726548',\n",
       "  'score': 0.13439282774925232},\n",
       " {'start': 1247,\n",
       "  'end': 1355,\n",
       "  'text': 'Cytochalasin D was then used to induce cytoplasmic YAP and the results showed the decreased HOXB4 expression.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28726548',\n",
       "  'score': 0.13506647944450378},\n",
       " {'start': 1357,\n",
       "  'end': 1488,\n",
       "  'text': 'In addition, the incorporation of microparticles in the neural spheroids was investigated for the perturbation of neural patterning.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28726548',\n",
       "  'score': 0.1369967758655548},\n",
       " {'start': 1490,\n",
       "  'end': 1722,\n",
       "  'text': 'This study may indicate the bidirectional interactions of Wnt signaling and YAP expression during neural tissue patterning, which have the significance in neurological disease modeling, drug screening, and neural tissue regeneration.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28726548',\n",
       "  'score': 0.14020735025405884},\n",
       " {'start': 0,\n",
       "  'end': 110,\n",
       "  'text': 'Microphthalamia-associated transcription factor: a critical regulator of pigment cell development and survival.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '12789278',\n",
       "  'score': 0.13920381665229797},\n",
       " {'start': 0,\n",
       "  'end': 162,\n",
       "  'text': 'The microphthalamia-associated transcription factor (MITF) is an integral transcriptional regulator in melanocyte, the lineage from which melanoma cells originate.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '12789278',\n",
       "  'score': 0.143486887216568},\n",
       " {'start': 164,\n",
       "  'end': 339,\n",
       "  'text': 'This basic-helix-loop-helix-leucine-zipper (bHLHzip) protein is critical for melanocyte cell-fate choice during commitment from pluripotent precursor cells in the neural crest.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '12789278',\n",
       "  'score': 0.35395193099975586},\n",
       " {'start': 341,\n",
       "  'end': 561,\n",
       "  'text': 'Its role in differentiation pathways has been highlighted by its potent transcriptional and lineage-specific regulation of the three major pigment enzymes: tyrosinase, Tyrp1, and Dct as well as other pigmentation factors.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '12789278',\n",
       "  'score': 0.1626070737838745},\n",
       " {'start': 563,\n",
       "  'end': 754,\n",
       "  'text': 'However, the cellular functions of MITF seem to be wider than differentiation and cell-fate pathways alone, since melanocytes and melanoma cells appear to require an expression of this factor.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '12789278',\n",
       "  'score': 0.13690334558486938},\n",
       " {'start': 756,\n",
       "  'end': 902,\n",
       "  'text': 'Here, we discuss the transcriptional networks in which MITF is thought to reside and describe signaling pathways in the cell which impinge on MITF.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '12789278',\n",
       "  'score': 0.16772770881652832},\n",
       " {'start': 904,\n",
       "  'end': 1062,\n",
       "  'text': 'Accumulating evidence supports the notion that MITF is involved in survival pathways during normal development as well as during neoplastic growth of melanoma.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '12789278',\n",
       "  'score': 0.14641043543815613},\n",
       " {'start': 0,\n",
       "  'end': 90,\n",
       "  'text': 'Polycomb repressive complex PRC1 spatially constrains the mouse embryonic stem cell genome.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '26323060',\n",
       "  'score': 0.13326194882392883},\n",
       " {'start': 0,\n",
       "  'end': 157,\n",
       "  'text': 'The Polycomb repressive complexes PRC1 and PRC2 maintain embryonic stem cell (ESC) pluripotency by silencing lineage-specifying developmental regulator genes.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '26323060',\n",
       "  'score': 0.33374887704849243},\n",
       " {'start': 159,\n",
       "  'end': 261,\n",
       "  'text': 'Emerging evidence suggests that Polycomb complexes act through controlling spatial genome organization.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '26323060',\n",
       "  'score': 0.15924254059791565},\n",
       " {'start': 263,\n",
       "  'end': 407,\n",
       "  'text': 'We show that PRC1 functions as a master regulator of mouse ESC genome architecture by organizing genes in three-dimensional interaction networks.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '26323060',\n",
       "  'score': 0.23159152269363403},\n",
       " {'start': 409,\n",
       "  'end': 579,\n",
       "  'text': 'The strongest spatial network is composed of the four Hox gene clusters and early developmental transcription factor genes, the majority of which contact poised enhancers.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '26323060',\n",
       "  'score': 0.24867889285087585},\n",
       " {'start': 581,\n",
       "  'end': 685,\n",
       "  'text': 'Removal of Polycomb repression leads to disruption of promoter-promoter contacts in the Hox gene network.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '26323060',\n",
       "  'score': 0.2731490135192871},\n",
       " {'start': 687,\n",
       "  'end': 934,\n",
       "  'text': 'In contrast, promoter-enhancer contacts are maintained in the absence of Polycomb repression, with accompanying widespread acquisition of active chromatin signatures at network enhancers and pronounced transcriptional upregulation of network genes.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '26323060',\n",
       "  'score': 0.2736440896987915},\n",
       " {'start': 936,\n",
       "  'end': 1070,\n",
       "  'text': 'Thus, PRC1 physically constrains developmental transcription factor genes and their enhancers in a silenced but poised spatial network.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '26323060',\n",
       "  'score': 0.23642989993095398},\n",
       " {'start': 1072,\n",
       "  'end': 1213,\n",
       "  'text': 'We propose that the selective release of genes from this spatial network underlies cell fate specification during early embryonic development.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '26323060',\n",
       "  'score': 0.15765175223350525},\n",
       " {'start': 0,\n",
       "  'end': 104,\n",
       "  'text': 'Poly(ADP-ribose) polymerase 1 regulates nuclear reprogramming and promotes iPSC generation without c-Myc.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '23277454',\n",
       "  'score': 0.13552239537239075},\n",
       " {'start': 0,\n",
       "  'end': 152,\n",
       "  'text': 'Poly(ADP-ribose) polymerase 1 (Parp1) catalyzes poly(ADP-ribosylation) (PARylation) and induces replication networks involved in multiple nuclear events.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '23277454',\n",
       "  'score': 0.17222949862480164},\n",
       " {'start': 154,\n",
       "  'end': 406,\n",
       "  'text': 'Using mass spectrometry and Western blotting, Parp1 and PARylation activity were intensively detected in induced pluripotent stem cells (iPSCs) and embryonic stem cells, but they were lower in mouse embryonic fibroblasts (MEFs) and differentiated cells.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '23277454',\n",
       "  'score': 0.27230048179626465},\n",
       " {'start': 408,\n",
       "  'end': 563,\n",
       "  'text': 'We show that knockdown of Parp1 and pharmacological inhibition of PARylation both reduced the efficiency of iPSC generation induced by Oct4/Sox2/Klf4/c-Myc.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '23277454',\n",
       "  'score': 0.3131505250930786},\n",
       " {'start': 565,\n",
       "  'end': 661,\n",
       "  'text': 'Furthermore, Parp1 is able to replace Klf4 or c-Myc to enhance the efficiency of iPSC generation.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '23277454',\n",
       "  'score': 0.13825595378875732},\n",
       " {'start': 663,\n",
       "  'end': 764,\n",
       "  'text': 'In addition, mouse iPSCs generated from Oct4/Sox2/Parp1-overexpressing MEFs formed chimeric offspring.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '23277454',\n",
       "  'score': 0.32201915979385376},\n",
       " {'start': 766,\n",
       "  'end': 892,\n",
       "  'text': 'Notably, the endogenous Parp1 and PARylation activity was enhanced by overexpression of c-Myc and repressed by c-Myc knockdown.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '23277454',\n",
       "  'score': 0.13864994049072266},\n",
       " {'start': 894,\n",
       "  'end': 994,\n",
       "  'text': 'A chromatin immunoprecipitation assay revealed a direct interaction of c-Myc with the Parp1 promoter.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '23277454',\n",
       "  'score': 0.21714571118354797},\n",
       " {'start': 996,\n",
       "  'end': 1211,\n",
       "  'text': 'PAR-resin pulldown, followed by proteomic analysis, demonstrated high levels of PARylated Chd1L, DNA ligase III, SSrp1, Xrcc-6/Ku70, and Parp2 in pluripotent cells, which decreased during the differentiation process.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '23277454',\n",
       "  'score': 0.2946053743362427},\n",
       " {'start': 1213,\n",
       "  'end': 1428,\n",
       "  'text': 'These data show that the activation of Parp1, partly regulated by endogenous c-Myc, effectively promotes iPSC production and helps to maintain a pluripotent state by posttranslationally modulating protein PARylation.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '23277454',\n",
       "  'score': 0.31614190340042114},\n",
       " {'start': 0,\n",
       "  'end': 69,\n",
       "  'text': 'Super-enhancers in transcriptional regulation and genome organization.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '31724731',\n",
       "  'score': 0.3204605281352997},\n",
       " {'start': 0,\n",
       "  'end': 187,\n",
       "  'text': 'Gene expression is precisely controlled in a stage and cell-type-specific manner, largely through the interaction between cis-regulatory elements and their associated trans-acting factors.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31724731',\n",
       "  'score': 0.14798542857170105},\n",
       " {'start': 189,\n",
       "  'end': 425,\n",
       "  'text': \"Where these components aggregate in promoters and enhancers, they are able to cooperate to modulate chromatin structure and support the engagement in long-range 3D superstructures that shape the dynamics of a cell's genomic architecture.\",\n",
       "  'is_title': False,\n",
       "  'doc_id': '31724731',\n",
       "  'score': 0.260100781917572},\n",
       " {'start': 427,\n",
       "  'end': 653,\n",
       "  'text': \"Recently, the term 'super-enhancer' has been introduced to describe a hyper-active regulatory domain comprising a complex array of sequence elements that work together to control the key gene networks involved in cell identity.\",\n",
       "  'is_title': False,\n",
       "  'doc_id': '31724731',\n",
       "  'score': 0.27670320868492126},\n",
       " {'start': 655,\n",
       "  'end': 848,\n",
       "  'text': 'Here, we survey the unique characteristics of super-enhancers compared to other enhancer types and summarize the recent advances in our understanding of their biological role in gene regulation.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31724731',\n",
       "  'score': 0.23841539025306702},\n",
       " {'start': 850,\n",
       "  'end': 1051,\n",
       "  'text': 'In particular, we discuss their capacity to attract the formation of phase-separated condensates, and capacity to generate three-dimensional genome structures that precisely activate their target genes.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31724731',\n",
       "  'score': 0.20552003383636475},\n",
       " {'start': 1053,\n",
       "  'end': 1262,\n",
       "  'text': 'We also propose a multi-stage transition model to explain the evolutionary pressure driving the development of super-enhancers in complex organisms, and highlight the potential for involvement in tumorigenesis.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31724731',\n",
       "  'score': 0.21455782651901245},\n",
       " {'start': 1264,\n",
       "  'end': 1401,\n",
       "  'text': 'Finally, we discuss more broadly the role of super-enhancers in human health disorders and related potential in therapeutic interventions.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31724731',\n",
       "  'score': 0.22466906905174255},\n",
       " {'start': 0,\n",
       "  'end': 91,\n",
       "  'text': 'Making a Kidney Organoid Using the Directed Differentiation of Human Pluripotent Stem Cells.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '28361319',\n",
       "  'score': 0.30767688155174255},\n",
       " {'start': 0,\n",
       "  'end': 114,\n",
       "  'text': 'An organoid can be defined as a three-dimensional organ-like structure formed from organ-specific progenitor cells.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28361319',\n",
       "  'score': 0.19676324725151062},\n",
       " {'start': 116,\n",
       "  'end': 254,\n",
       "  'text': 'Organ progenitor cells were empirically found to self-organize three-dimensional tissues when they were aggregated and cultivated in vitro.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28361319',\n",
       "  'score': 0.20012500882148743},\n",
       " {'start': 256,\n",
       "  'end': 581,\n",
       "  'text': 'While this nature power of progenitor cells has an amazing potential to recreate artificial organs in vitro, there had been difficulty to apply this technology to human organs due to the inaccessibility to human progenitor cells until human-induced pluripotent stem cell (hiPSC) was invented by Takahashi and Yamanaka in 2007.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28361319',\n",
       "  'score': 0.28519439697265625},\n",
       " {'start': 583,\n",
       "  'end': 738,\n",
       "  'text': 'As embryonic stem cells do, hiPSCs also have pluripotency to give rise to any organs/tissues cell types, including the kidney, via directed differentiation.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28361319',\n",
       "  'score': 0.30977320671081543},\n",
       " {'start': 740,\n",
       "  'end': 843,\n",
       "  'text': 'Here, we provide a detailed protocol for generating kidney organoids using human pluripotent stem cells.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28361319',\n",
       "  'score': 0.2971879839897156},\n",
       " {'start': 845,\n",
       "  'end': 937,\n",
       "  'text': 'The protocol differentiates human pluripotent stem cells into the posterior primitive streak.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28361319',\n",
       "  'score': 0.286166787147522},\n",
       " {'start': 939,\n",
       "  'end': 1121,\n",
       "  'text': 'This is followed by the simultaneous induction of posterior and anterior intermediate mesoderm that are subsequently aggregated and undergo self-organization into the kidney organoid.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28361319',\n",
       "  'score': 0.17479455471038818},\n",
       " {'start': 1123,\n",
       "  'end': 1371,\n",
       "  'text': 'Such kidney organoids are comprised of all anticipated kidney cell types including nephrons segmented into the glomerulus, proximal tubule, loop of Henle, and distal tubule as well as the collecting duct, endothelial network, and renal interstitium.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28361319',\n",
       "  'score': 0.16491156816482544},\n",
       " {'start': 0,\n",
       "  'end': 60,\n",
       "  'text': 'Nanoengineered Platforms to Guide Pluripotent Stem Cell Fate.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '26918198',\n",
       "  'score': 0.3092994689941406},\n",
       " {'start': 0,\n",
       "  'end': 129,\n",
       "  'text': 'Tissue engineering utilizes cells, signaling molecules, and scaffolds towards creating functional tissue to repair damaged organs.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '26918198',\n",
       "  'score': 0.1405961513519287},\n",
       " {'start': 131,\n",
       "  'end': 301,\n",
       "  'text': 'Pluripotent stem cells (PSCs) are a promising cell source due to their ability to self-renewal indefinitely and their potential to differentiate into almost any cell type.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '26918198',\n",
       "  'score': 0.30063775181770325},\n",
       " {'start': 303,\n",
       "  'end': 451,\n",
       "  'text': 'Great strides have been taken to parse the physiological mechanisms by which PSCs respond to their microenvironment and commit to a specific lineage.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '26918198',\n",
       "  'score': 0.1437987983226776},\n",
       " {'start': 453,\n",
       "  'end': 681,\n",
       "  'text': 'The combination of physical cues and chemical factors is thought to have the most profound influence on stem cell behavior, therefore a major focus of tissue engineering strategies is scaffold design to incorporate these signals.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '26918198',\n",
       "  'score': 0.1452803611755371},\n",
       " {'start': 683,\n",
       "  'end': 909,\n",
       "  'text': 'One overlooked component of the in vivo microenvironment researchers attempt to recapitulate with three dimensional (3D) substrates is the nanoarchitecture formed by the fibrillar network of extracellular matrix (ECM) proteins.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '26918198',\n",
       "  'score': 0.2397085428237915},\n",
       " {'start': 911,\n",
       "  'end': 1026,\n",
       "  'text': 'These nanoscale features have the ability to impact cell adhesion, migration, proliferation, and lineage commitment.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '26918198',\n",
       "  'score': 0.1422543227672577},\n",
       " {'start': 1028,\n",
       "  'end': 1284,\n",
       "  'text': 'Significant advances have been made in deciphering how these nanoscale cues interact with stem cells to determine phenotype, but much is still unknown as to how the interplay between physical and chemical signals regulate in vitro and in vivo cellular fate.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '26918198',\n",
       "  'score': 0.14635738730430603},\n",
       " {'start': 1286,\n",
       "  'end': 1468,\n",
       "  'text': 'This review dives deeper to investigate nanoscale platforms for engineering tissue, as well use the use of these nanotechnologies to drive pluripotent stem cell lineage determination.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '26918198',\n",
       "  'score': 0.2841383218765259},\n",
       " {'start': 0,\n",
       "  'end': 122,\n",
       "  'text': 'Chromatin insulator factors involved in long-range DNA interactions and their role in the folding of the Drosophila genome.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '25165871',\n",
       "  'score': 0.15337121486663818},\n",
       " {'start': 0,\n",
       "  'end': 121,\n",
       "  'text': 'Chromatin insulators are genetic elements implicated in the organization of chromatin and the regulation of transcription.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '25165871',\n",
       "  'score': 0.2160864770412445},\n",
       " {'start': 123,\n",
       "  'end': 255,\n",
       "  'text': 'In Drosophila, different insulator types were characterized by their locus-specific composition of insulator proteins and co-factors.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '25165871',\n",
       "  'score': 0.14510449767112732},\n",
       " {'start': 257,\n",
       "  'end': 499,\n",
       "  'text': 'Insulators mediate specific long-range DNA contacts required for the three dimensional organization of the interphase nucleus and for transcription regulation, but the mechanisms underlying the formation of these contacts is currently unknown.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '25165871',\n",
       "  'score': 0.2787514626979828},\n",
       " {'start': 501,\n",
       "  'end': 813,\n",
       "  'text': 'Here, we investigate the molecular associations between different components of insulator complexes (BEAF32, CP190 and Chromator) by biochemical and biophysical means, and develop a novel single-molecule assay to determine what factors are necessary and essential for the formation of long-range DNA interactions.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '25165871',\n",
       "  'score': 0.1438683569431305},\n",
       " {'start': 815,\n",
       "  'end': 939,\n",
       "  'text': 'We show that BEAF32 is able to bind DNA specifically and with high affinity, but not to bridge long-range interactions (LRI).',\n",
       "  'is_title': False,\n",
       "  'doc_id': '25165871',\n",
       "  'score': 0.14279693365097046},\n",
       " {'start': 941,\n",
       "  'end': 1077,\n",
       "  'text': 'In contrast, we show that CP190 and Chromator are able to mediate LRI between specifically-bound BEAF32 nucleoprotein complexes in vitro.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '25165871',\n",
       "  'score': 0.14328351616859436},\n",
       " {'start': 1079,\n",
       "  'end': 1253,\n",
       "  'text': 'This ability of CP190 and Chromator to establish LRI requires specific contacts between BEAF32 and their C-terminal domains, and dimerization through their N-terminal domains.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '25165871',\n",
       "  'score': 0.14200228452682495},\n",
       " {'start': 1255,\n",
       "  'end': 1399,\n",
       "  'text': 'In particular, the BTB/POZ domains of CP190 form a strict homodimer, and its C-terminal domain interacts with several insulator binding proteins.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '25165871',\n",
       "  'score': 0.14701470732688904},\n",
       " {'start': 1401,\n",
       "  'end': 1644,\n",
       "  'text': 'We propose a general model for insulator function in which BEAF32/dCTCF/Su(HW) provide DNA specificity (first layer proteins) whereas CP190/Chromator are responsible for the physical interactions required for long-range contacts (second layer).',\n",
       "  'is_title': False,\n",
       "  'doc_id': '25165871',\n",
       "  'score': 0.1443822681903839},\n",
       " {'start': 1646,\n",
       "  'end': 1949,\n",
       "  'text': 'This network of organized, multi-layer interactions could explain the different activities of insulators as chromatin barriers, enhancer blockers, and transcriptional regulators, and suggest a general mechanism for how insulators may shape the organization of higher-order chromatin during cell division.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '25165871',\n",
       "  'score': 0.3165259063243866},\n",
       " {'start': 0,\n",
       "  'end': 85,\n",
       "  'text': 'Chromosome conformation elucidates regulatory relationships in developing human brain.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '27760116',\n",
       "  'score': 0.14131638407707214},\n",
       " {'start': 0,\n",
       "  'end': 123,\n",
       "  'text': 'Three-dimensional physical interactions within chromosomes dynamically regulate gene expression in a tissue-specific manner.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '27760116',\n",
       "  'score': 0.20864352583885193},\n",
       " {'start': 125,\n",
       "  'end': 331,\n",
       "  'text': 'However, the 3D organization of chromosomes during human brain development and its role in regulating gene networks dysregulated in neurodevelopmental disorders, such as autism or schizophrenia, are unknown.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '27760116',\n",
       "  'score': 0.31868794560432434},\n",
       " {'start': 333,\n",
       "  'end': 570,\n",
       "  'text': 'Here we generate high-resolution 3D maps of chromatin contacts during human corticogenesis, permitting large-scale annotation of previously uncharacterized regulatory relationships relevant to the evolution of human cognition and disease.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '27760116',\n",
       "  'score': 0.19164606928825378},\n",
       " {'start': 572,\n",
       "  'end': 769,\n",
       "  'text': 'Our analyses identify hundreds of genes that physically interact with enhancers gained on the human lineage, many of which are under purifying selection and associated with human cognitive function.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '27760116',\n",
       "  'score': 0.22818276286125183},\n",
       " {'start': 771,\n",
       "  'end': 1172,\n",
       "  'text': 'We integrate chromatin contacts with non-coding variants identified in schizophrenia genome-wide association studies (GWAS), highlighting multiple candidate schizophrenia risk genes and pathways, including transcription factors involved in neurogenesis, and cholinergic signalling molecules, several of which are supported by independent expression quantitative trait loci and gene expression analyses.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '27760116',\n",
       "  'score': 0.1516258716583252},\n",
       " {'start': 1174,\n",
       "  'end': 1361,\n",
       "  'text': 'Genome editing in human neural progenitors suggests that one of these distal schizophrenia GWAS loci regulates FOXG1 expression, supporting its potential role as a schizophrenia risk gene.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '27760116',\n",
       "  'score': 0.14247703552246094},\n",
       " {'start': 1363,\n",
       "  'end': 1585,\n",
       "  'text': 'This work provides a framework for understanding the effect of non-coding regulatory elements on human brain development and the evolution of cognition, and highlights novel mechanisms underlying neuropsychiatric disorders.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '27760116',\n",
       "  'score': 0.14432692527770996},\n",
       " {'start': 0,\n",
       "  'end': 94,\n",
       "  'text': 'Paracrine signals regulate human liver organoid maturation from induced pluripotent stem cells.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '28275009',\n",
       "  'score': 0.3126864433288574},\n",
       " {'start': 0,\n",
       "  'end': 108,\n",
       "  'text': 'A self-organizing organoid model provides a new approach to study the mechanism of human liver organogenesis.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28275009',\n",
       "  'score': 0.14264750480651855},\n",
       " {'start': 110,\n",
       "  'end': 250,\n",
       "  'text': 'Previous animal models documented that simultaneous paracrine signaling and cell-to-cell surface contact regulate hepatocyte differentiation.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28275009',\n",
       "  'score': 0.14155784249305725},\n",
       " {'start': 252,\n",
       "  'end': 510,\n",
       "  'text': 'To dissect the relative contributions of the paracrine effects, we first established a liver organoid using human induced pluripotent stem cells (iPSCs), mesenchymal stem cells (MSCs) and human umbilical vein endothelial cells (HUVECs) as previously reported.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28275009',\n",
       "  'score': 0.3001575767993927},\n",
       " {'start': 512,\n",
       "  'end': 671,\n",
       "  'text': 'Time-lapse imaging showed that hepatic-specified endoderm iPSCs (HE-iPSCs) self-assembled into three-dimensional organoids, resulting in hepatic gene induction.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28275009',\n",
       "  'score': 0.19780898094177246},\n",
       " {'start': 673,\n",
       "  'end': 786,\n",
       "  'text': 'Progressive differentiation was demonstrated by hepatic protein production after in vivo organoid transplantation.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28275009',\n",
       "  'score': 0.14900216460227966},\n",
       " {'start': 788,\n",
       "  'end': 927,\n",
       "  'text': 'To assess the paracrine contributions, we employed a Transwell system in which HE-iPSCs were separately co-cultured with MSCs and/or HUVECs.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28275009',\n",
       "  'score': 0.1466226875782013},\n",
       " {'start': 929,\n",
       "  'end': 1107,\n",
       "  'text': 'Although the three-dimensional structure did not form, their soluble factors induced a hepatocyte-like phenotype in HE-iPSCs, resulting in the expression of bile salt export pump.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28275009',\n",
       "  'score': 0.20642510056495667},\n",
       " {'start': 1109,\n",
       "  'end': 1281,\n",
       "  'text': 'In conclusion, the mesoderm-derived paracrine signals promote hepatocyte maturation in liver organoids, but organoid self-organization requires cell-to-cell surface contact.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28275009',\n",
       "  'score': 0.17506101727485657},\n",
       " {'start': 1283,\n",
       "  'end': 1427,\n",
       "  'text': 'Our in vitro model demonstrates a novel approach to identify developmental paracrine signals regulating the differentiation of human hepatocytes.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '28275009',\n",
       "  'score': 0.1632860004901886},\n",
       " {'start': 0,\n",
       "  'end': 84,\n",
       "  'text': 'Chromatin connectivity maps reveal dynamic promoter-enhancer long-range associations.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '24213634',\n",
       "  'score': 0.23710381984710693},\n",
       " {'start': 0,\n",
       "  'end': 147,\n",
       "  'text': 'In multicellular organisms, transcription regulation is one of the central mechanisms modelling lineage differentiation and cell-fate determination.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24213634',\n",
       "  'score': 0.17123103141784668},\n",
       " {'start': 149,\n",
       "  'end': 273,\n",
       "  'text': 'Transcription requires dynamic chromatin configurations between promoters and their corresponding distal regulatory elements.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24213634',\n",
       "  'score': 0.14240393042564392},\n",
       " {'start': 275,\n",
       "  'end': 444,\n",
       "  'text': 'It is believed that their communication occurs within large discrete foci of aggregated RNA polymerases termed transcription factories in three-dimensional nuclear space.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24213634',\n",
       "  'score': 0.208286315202713},\n",
       " {'start': 446,\n",
       "  'end': 551,\n",
       "  'text': 'However, the dynamic nature of chromatin connectivity has not been characterized at the genome-wide level.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24213634',\n",
       "  'score': 0.14365434646606445},\n",
       " {'start': 553,\n",
       "  'end': 927,\n",
       "  'text': 'Here, through a chromatin interaction analysis with paired-end tagging approach using an antibody that primarily recognizes the pre-initiation complexes of RNA polymerase II, we explore the transcriptional interactomes of three mouse cells of progressive lineage commitment, including pluripotent embryonic stem cells, neural stem cells and neurosphere stem/progenitor cells.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24213634',\n",
       "  'score': 0.3128579556941986},\n",
       " {'start': 929,\n",
       "  'end': 1121,\n",
       "  'text': 'Our global chromatin connectivity maps reveal approximately 40,000 long-range interactions, suggest precise enhancer-promoter associations and delineate cell-type-specific chromatin structures.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24213634',\n",
       "  'score': 0.22451671957969666},\n",
       " {'start': 1123,\n",
       "  'end': 1259,\n",
       "  'text': 'Analysis of the complex regulatory repertoire shows that there are extensive colocalizations among promoters and distal-acting enhancers.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24213634',\n",
       "  'score': 0.22484946250915527},\n",
       " {'start': 1261,\n",
       "  'end': 1459,\n",
       "  'text': 'Most of the enhancers associate with promoters located beyond their nearest active genes, indicating that the linear juxtaposition is not the only guiding principle driving enhancer target selection.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24213634',\n",
       "  'score': 0.22459429502487183},\n",
       " {'start': 1461,\n",
       "  'end': 1644,\n",
       "  'text': 'Although promoter-enhancer interactions exhibit high cell-type specificity, promoters involved in interactions are found to be generally common and mostly active among different cells.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24213634',\n",
       "  'score': 0.2469051480293274},\n",
       " {'start': 1646,\n",
       "  'end': 1877,\n",
       "  'text': 'Chromatin connectivity networks reveal that the pivotal genes of reprogramming functions are transcribed within physical proximity to each other in embryonic stem cells, linking chromatin architecture to coordinated gene expression.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24213634',\n",
       "  'score': 0.174902081489563},\n",
       " {'start': 1879,\n",
       "  'end': 2020,\n",
       "  'text': 'Our study sets the stage for the full-scale dissection of spatial and temporal genome structures and their roles in orchestrating development.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24213634',\n",
       "  'score': 0.14775970578193665},\n",
       " {'start': 0,\n",
       "  'end': 71,\n",
       "  'text': 'High-dimensional switches and the modelling of cellular differentiation.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '15652148',\n",
       "  'score': 0.19714757800102234},\n",
       " {'start': 0,\n",
       "  'end': 191,\n",
       "  'text': 'Many genes have been identified as driving cellular differentiation, but because of their complex interactions, the understanding of their collective behaviour requires mathematical modelling.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '15652148',\n",
       "  'score': 0.14331617951393127},\n",
       " {'start': 193,\n",
       "  'end': 453,\n",
       "  'text': 'Intriguingly, it has been observed in numerous developmental contexts, and particularly haematopoiesis, that genes regulating differentiation are initially co-expressed in progenitors despite their antagonism, before one is upregulated and others downregulated.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '15652148',\n",
       "  'score': 0.1641356647014618},\n",
       " {'start': 455,\n",
       "  'end': 877,\n",
       "  'text': 'We characterise conditions under which three classes of generic \"master regulatory networks\", modelled at the molecular level after experimentally observed interactions (including bHLH protein dimerisation), and including an arbitrary number of antagonistic components, can behave as a \"multi-switch\", directing differentiation in an all-or-none fashion to a specific cell-type chosen among more than two possible outcomes.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '15652148',\n",
       "  'score': 0.18904632329940796},\n",
       " {'start': 879,\n",
       "  'end': 1029,\n",
       "  'text': 'bHLH dimerisation networks can readily display coexistence of many antagonistic factors when competition is low (a simple characterisation is derived).',\n",
       "  'is_title': False,\n",
       "  'doc_id': '15652148',\n",
       "  'score': 0.1690405309200287},\n",
       " {'start': 1031,\n",
       "  'end': 1349,\n",
       "  'text': 'Decision-making can be forced by a transient increase in competition, which could correspond to some unexplained experimental observations related to Id proteins; the speed of response varies with the initial conditions the network is subjected to, which could explain some aspects of cell behaviour upon reprogramming.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '15652148',\n",
       "  'score': 0.16980090737342834},\n",
       " {'start': 1351,\n",
       "  'end': 1588,\n",
       "  'text': 'The coexistence of antagonistic factors at low levels, early in the differentiation process or in pluripotent stem cells, could be an intrinsic property of the interaction between those factors, not requiring a specific regulatory system.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '15652148',\n",
       "  'score': 0.3194504678249359},\n",
       " {'start': 0,\n",
       "  'end': 63,\n",
       "  'text': 'The SUMO E3 Ligase SIZ1 Negatively Regulates Shoot Regeneration.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '32611787',\n",
       "  'score': 0.14275389909744263},\n",
       " {'start': 0,\n",
       "  'end': 94,\n",
       "  'text': 'Plants form calluses and regenerate new organs when incubated on phytohormone-containing media.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '32611787',\n",
       "  'score': 0.14283525943756104},\n",
       " {'start': 96,\n",
       "  'end': 358,\n",
       "  'text': 'While accumulating evidence suggests that these regenerative processes are governed by transcriptional networks orchestrating wound response and developmental transitions, it remains unknown if posttranslational regulatory mechanisms are involved in this process.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '32611787',\n",
       "  'score': 0.18269506096839905},\n",
       " {'start': 360,\n",
       "  'end': 639,\n",
       "  'text': 'In this study, we demonstrate that SAP AND MIZ1 DOMAIN- CONTAINING LIGASE1 (SIZ1), an E3 ligase-catalyzing attachment of the SMALL UBIQUITIN-LIKE MODIFIER (SUMO) to proteins, regulates wound-induced signal transduction and organ regeneration in Arabidopsis (Arabidopsis thaliana).',\n",
       "  'is_title': False,\n",
       "  'doc_id': '32611787',\n",
       "  'score': 0.1490306258201599},\n",
       " {'start': 641,\n",
       "  'end': 1040,\n",
       "  'text': 'We show that loss-of-function mutants for SIZ1 exhibit overproduction of shoot meristems under in vitro tissue culture conditions, while this defect is rescued in a complementation line expressing pSIZ1::SIZ1 RNA sequencing analysis revealed that siz1-2 mutants exhibit enhanced transcriptional responses to wound stress, resulting in the hyper-induction of over 400 genes immediately after wounding.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '32611787',\n",
       "  'score': 0.14854979515075684},\n",
       " {'start': 1042,\n",
       "  'end': 1352,\n",
       "  'text': 'Among them, we show that elevated levels of WOUND INDUCED DEDIFFERENTIATION1 (WIND1) and WIND2 contribute to the enhanced shoot regeneration observed in siz1 mutants, as expression of the dominant-negative chimeric protein WIND1-SRDX (SUPERMAN repression domain) in siz1-3 mutants partly rescues this phenotype.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '32611787',\n",
       "  'score': 0.15515625476837158},\n",
       " {'start': 1354,\n",
       "  'end': 1671,\n",
       "  'text': 'Although compromised SIZ1 function does not modify the transcription of genes implicated in auxin-induced callus formation and/or pluripotency acquisition, it does lead to enhanced induction of cytokinin-induced shoot meristem regulators such as WUSCHEL, promoting the formation of WUSCHEL-expressing foci in explants.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '32611787',\n",
       "  'score': 0.3235282301902771},\n",
       " {'start': 1673,\n",
       "  'end': 1811,\n",
       "  'text': 'This study thus suggests that SIZ1 negatively regulates shoot regeneration in part by repressing wound-induced developmental reprogramming.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '32611787',\n",
       "  'score': 0.14303883910179138},\n",
       " {'start': 0,\n",
       "  'end': 38,\n",
       "  'text': 'Gene regulation in the third dimension.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '18369139',\n",
       "  'score': 0.17092499136924744},\n",
       " {'start': 0,\n",
       "  'end': 122,\n",
       "  'text': 'Analysis of the spatial organization of chromosomes reveals complex three-dimensional networks of chromosomal interactions.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '18369139',\n",
       "  'score': 0.30421969294548035},\n",
       " {'start': 124,\n",
       "  'end': 322,\n",
       "  'text': 'These interactions affect gene expression at multiple levels, including long-range control by distant enhancers and repressors, coordinated expression of genes, and modification of epigenetic states.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '18369139',\n",
       "  'score': 0.23709824681282043},\n",
       " {'start': 324,\n",
       "  'end': 491,\n",
       "  'text': 'Major challenges now include deciphering the mechanisms by which loci come together and understanding the functional consequences of these often transient associations.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '18369139',\n",
       "  'score': 0.1520516574382782},\n",
       " {'start': 0,\n",
       "  'end': 106,\n",
       "  'text': 'Specific Contributions of Cohesin-SA1 and Cohesin-SA2 to TADs and Polycomb Domains in Embryonic Stem Cells.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '31216471',\n",
       "  'score': 0.15367034077644348},\n",
       " {'start': 0,\n",
       "  'end': 62,\n",
       "  'text': 'Cohesin exists in two variants carrying either STAG/SA1 or SA2.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31216471',\n",
       "  'score': 0.1548101305961609},\n",
       " {'start': 64,\n",
       "  'end': 338,\n",
       "  'text': 'Here we have addressed their specific contributions to the unique spatial organization of the mouse embryonic stem cell genome, which ensures super-enhancer-dependent transcription of\\xa0pluripotency factors and repression of lineage-specification genes within Polycomb domains.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31216471',\n",
       "  'score': 0.2934204339981079},\n",
       " {'start': 340,\n",
       "  'end': 606,\n",
       "  'text': 'We find that cohesin-SA2 facilitates Polycomb domain compaction through Polycomb repressing complex 1 (PRC1) recruitment and promotes the establishment of long-range interaction networks between distant Polycomb-bound promoters that are important for gene repression.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31216471',\n",
       "  'score': 0.17637240886688232},\n",
       " {'start': 608,\n",
       "  'end': 722,\n",
       "  'text': 'Cohesin-SA1, in contrast, disrupts these networks, while preserving topologically associating domain (TAD) borders.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31216471',\n",
       "  'score': 0.1741761863231659},\n",
       " {'start': 724,\n",
       "  'end': 823,\n",
       "  'text': 'The diverse effects of both complexes on genome topology may reflect two modes of action of cohesin.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31216471',\n",
       "  'score': 0.15721797943115234},\n",
       " {'start': 825,\n",
       "  'end': 997,\n",
       "  'text': 'One, likely involving loop extrusion, establishes overall genome arrangement in TADs together with CTCF and prevents excessive segregation of same-class compartment regions.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31216471',\n",
       "  'score': 0.1531020700931549},\n",
       " {'start': 999,\n",
       "  'end': 1140,\n",
       "  'text': 'The other is required for organization of local transcriptional hubs such as Polycomb domains and super-enhancers, which define cell identity.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31216471',\n",
       "  'score': 0.2941708564758301},\n",
       " {'start': 0,\n",
       "  'end': 35,\n",
       "  'text': 'A Cleared View on Retinal Organoids.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '31035373',\n",
       "  'score': 0.1496291160583496},\n",
       " {'start': 0,\n",
       "  'end': 306,\n",
       "  'text': 'Human induced pluripotent stem cell (hiPSC)-derived organoids mimicking tissues and organs in vitro have advanced medical research, as they opened up new possibilities for in-depth basic research on human organ development as well as providing a human in vitro model for personalized therapeutic approaches.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31035373',\n",
       "  'score': 0.2895902991294861},\n",
       " {'start': 308,\n",
       "  'end': 477,\n",
       "  'text': 'hiPSC-derived retinal organoids have proven to be of great value for modeling the human retina featuring a very similar cellular composition, layering, and functionality.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31035373',\n",
       "  'score': 0.155246764421463},\n",
       " {'start': 479,\n",
       "  'end': 647,\n",
       "  'text': 'The technically challenging imaging of three-dimensional structures such as retinal organoids has, however, raised the need for robust whole-organoid imaging techniques.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31035373',\n",
       "  'score': 0.21311861276626587},\n",
       " {'start': 649,\n",
       "  'end': 817,\n",
       "  'text': 'To improve imaging of retinal organoids we optimized a passive clearing technique (PACT), which enables high-resolution visualization of fragile intra-tissue structures.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31035373',\n",
       "  'score': 0.15271759033203125},\n",
       " {'start': 819,\n",
       "  'end': 1008,\n",
       "  'text': 'Using cleared retinal organoids, we could greatly enhance the antibody labeling efficiency and depth of imaging at high resolution, thereby improving the three-dimensional microscopy output.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31035373',\n",
       "  'score': 0.21031630039215088},\n",
       " {'start': 1010,\n",
       "  'end': 1153,\n",
       "  'text': 'In that course, we were able to identify the spatial morphological shape and organization of, e.g., photoreceptor cells and bipolar cell layers.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31035373',\n",
       "  'score': 0.20043876767158508},\n",
       " {'start': 1155,\n",
       "  'end': 1326,\n",
       "  'text': 'Moreover, we used the synaptic protein CtBP2/Ribeye to visualize the interconnection points of photoreceptor and bipolar cells forming the retinal-specific ribbon synapses.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '31035373',\n",
       "  'score': 0.16256222128868103},\n",
       " {'start': 0,\n",
       "  'end': 87,\n",
       "  'text': 'A structural network associated with the kallikrein-kinin and renin-angiotensin systems.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '20180650',\n",
       "  'score': 0.19083353877067566},\n",
       " {'start': 0,\n",
       "  'end': 181,\n",
       "  'text': 'The kallikrein-kinin and renin-angiotensin (KKS-RAS) systems represent two highly regulated proteolytic systems that are involved in several physiological and pathological processes.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '20180650',\n",
       "  'score': 0.1764903962612152},\n",
       " {'start': 183,\n",
       "  'end': 438,\n",
       "  'text': 'Although their protein-protein interactions can be studied using experimental approaches, it is difficult to differentiate between direct physical interactions and functional associations, which do not involve direct atomic contacts between macromolecules.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '20180650',\n",
       "  'score': 0.17073673009872437},\n",
       " {'start': 440,\n",
       "  'end': 541,\n",
       "  'text': 'This information can be obtained from an atomic-resolution characterization of the protein interfaces.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '20180650',\n",
       "  'score': 0.1675952672958374},\n",
       " {'start': 543,\n",
       "  'end': 655,\n",
       "  'text': 'As a result of this, various three-dimensional-based protein-protein interaction databases have become available.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '20180650',\n",
       "  'score': 0.2343921959400177},\n",
       " {'start': 657,\n",
       "  'end': 824,\n",
       "  'text': 'To gain insight into the multilayered interaction of the KKS-RAS systems, we present a protein network that is built up on three-dimensional domain-domain interactions.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '20180650',\n",
       "  'score': 0.27577322721481323},\n",
       " {'start': 826,\n",
       "  'end': 1092,\n",
       "  'text': 'The essential domains that link these systems are as follows: Cystatin, Peptidase_C1, Thyroglobulin_1, Insulin, CIMR (Cation-independent mannose-6-phosphate receptor repeat), fn2 (Fibronectin type II domain), fn1 (Fibronectin type I domain), EGF, Trypsin, and Serpin.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '20180650',\n",
       "  'score': 0.15941724181175232},\n",
       " {'start': 1094,\n",
       "  'end': 1191,\n",
       "  'text': 'We found that the CIMR domain is located at the core of the network, thus connecting both systems.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '20180650',\n",
       "  'score': 0.1862839162349701},\n",
       " {'start': 1193,\n",
       "  'end': 1348,\n",
       "  'text': 'From the latter, all domain interactors up to level 4 were retrieved, thus displaying a more comprehensive representation of the KKS-RAS structural network.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '20180650',\n",
       "  'score': 0.1840573251247406},\n",
       " {'start': 0,\n",
       "  'end': 163,\n",
       "  'text': 'Ep-CAM transfection in thymic epithelial cell lines triggers the formation of dynamic actin-rich protrusions involved in the organization of epithelial cell layers.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '11702195',\n",
       "  'score': 0.21225932240486145},\n",
       " {'start': 0,\n",
       "  'end': 117,\n",
       "  'text': 'Thymic epithelium is organized in a highly connected three-dimensional network through which thymocytes differentiate.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '11702195',\n",
       "  'score': 0.2675233483314514},\n",
       " {'start': 119,\n",
       "  'end': 190,\n",
       "  'text': 'The molecular mechanisms underlying this organization are still unknown.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '11702195',\n",
       "  'score': 0.19785204529762268},\n",
       " {'start': 192,\n",
       "  'end': 347,\n",
       "  'text': 'In thymic medulla, a major site of tolerance induction, the development of the epithelial cell net is tightly regulated by the needs of thymocyte selection.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '11702195',\n",
       "  'score': 0.16782167553901672},\n",
       " {'start': 349,\n",
       "  'end': 426,\n",
       "  'text': 'These reticulated epithelial cells express high levels of the Ep-CAM molecule.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '11702195',\n",
       "  'score': 0.16200625896453857},\n",
       " {'start': 428,\n",
       "  'end': 689,\n",
       "  'text': 'Using different thymic epithelial cell lines as a model system, we found that transfection of Ep-CAM enhances cell growth and leads to a rapid reorganization of the actin cytoskeleton by inducing the formation of numerous stress fibers and long cell protrusions.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '11702195',\n",
       "  'score': 0.1645474135875702},\n",
       " {'start': 691,\n",
       "  'end': 837,\n",
       "  'text': 'Finally, the crosslinking of the extracellular domain of a chimeric CD25ec/Ep-CAMic molecule is sufficient to trigger the formation of protrusions.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '11702195',\n",
       "  'score': 0.16455599665641785},\n",
       " {'start': 839,\n",
       "  'end': 1036,\n",
       "  'text': 'These results suggest that expression of Ep-CAM might balance the organizing capacity of cadherin molecules and may be participating in the formation of a dynamic stromal cell network in the thymus.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '11702195',\n",
       "  'score': 0.19614019989967346},\n",
       " {'start': 0,\n",
       "  'end': 61,\n",
       "  'text': 'Organization of the cytokeratin network in an epithelial cell.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '12850452',\n",
       "  'score': 0.2511327862739563},\n",
       " {'start': 0,\n",
       "  'end': 89,\n",
       "  'text': 'The cytoskeleton is a dynamic three-dimensional structure mainly located in the cytoplasm.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '12850452',\n",
       "  'score': 0.24609526991844177},\n",
       " {'start': 91,\n",
       "  'end': 201,\n",
       "  'text': 'It is involved in many cell functions such as mechanical signal transduction and maintenance of cell integrity.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '12850452',\n",
       "  'score': 0.18173441290855408},\n",
       " {'start': 203,\n",
       "  'end': 353,\n",
       "  'text': 'Among the three cytoskeletal components, intermediate filaments (the cytokeratin in epithelial cells) are the best candidates for this mechanical role.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '12850452',\n",
       "  'score': 0.1731824278831482},\n",
       " {'start': 355,\n",
       "  'end': 539,\n",
       "  'text': 'A model of the establishment of the cytokeratin network of an epithelial cell is proposed to study the dependence of its structural organization on extracellular mechanical environment.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '12850452',\n",
       "  'score': 0.2371448278427124},\n",
       " {'start': 541,\n",
       "  'end': 667,\n",
       "  'text': 'To implicitly describe the latter and its effects on the intracellular domain, we use mechanically regulated protein synthesis.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '12850452',\n",
       "  'score': 0.17581528425216675},\n",
       " {'start': 669,\n",
       "  'end': 889,\n",
       "  'text': 'Our model is a hybrid of a partial differential equation of parabolic type, governing the evolution of the concentration of cytokeratin, and a set of stochastic differential equations describing the dynamics of filaments.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '12850452',\n",
       "  'score': 0.17147886753082275},\n",
       " {'start': 891,\n",
       "  'end': 1086,\n",
       "  'text': 'Each filament is described by a stochastic differential equation that reflects both the local interactions with the environment and the non-local interactions via the past history of the filament.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '12850452',\n",
       "  'score': 0.17153769731521606},\n",
       " {'start': 1088,\n",
       "  'end': 1164,\n",
       "  'text': 'A three-dimensional simulation model is derived from this mathematical model.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '12850452',\n",
       "  'score': 0.23270553350448608},\n",
       " {'start': 1166,\n",
       "  'end': 1340,\n",
       "  'text': 'This simulation model is then used to obtain examples of cytokeratin network architectures under given mechanical conditions, and to study the influence of several parameters.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '12850452',\n",
       "  'score': 0.20032727718353271},\n",
       " {'start': 0,\n",
       "  'end': 103,\n",
       "  'text': 'Making connections: insulators organize eukaryotic chromosomes into independent cis-regulatory networks.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '24277632',\n",
       "  'score': 0.1947794258594513},\n",
       " {'start': 0,\n",
       "  'end': 204,\n",
       "  'text': 'Insulators play a central role in subdividing the chromosome into a series of discrete topologically independent domains and in ensuring that enhancers and silencers contact their appropriate target genes.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24277632',\n",
       "  'score': 0.2537003755569458},\n",
       " {'start': 206,\n",
       "  'end': 324,\n",
       "  'text': 'In this review we first discuss the general characteristics of insulator elements and their associated protein factors.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24277632',\n",
       "  'score': 0.18847844004631042},\n",
       " {'start': 326,\n",
       "  'end': 466,\n",
       "  'text': 'A growing collection of insulator proteins have been identified including a family of proteins whose expression is developmentally regulated.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24277632',\n",
       "  'score': 0.16808632016181946},\n",
       " {'start': 468,\n",
       "  'end': 612,\n",
       "  'text': 'We next consider several unexpected discoveries that require us to completely rethink how insulators function (and how they can best be assayed).',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24277632',\n",
       "  'score': 0.1661384105682373},\n",
       " {'start': 614,\n",
       "  'end': 800,\n",
       "  'text': 'These discoveries also require a reevaluation of how insulators might restrict or orchestrate (by preventing or promoting) interactions between regulatory elements and their target genes.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24277632',\n",
       "  'score': 0.16746357083320618},\n",
       " {'start': 802,\n",
       "  'end': 1058,\n",
       "  'text': 'We conclude by connecting these new insights into the mechanisms of insulator action to dynamic changes in the three-dimensional topology of the chromatin fiber and the generation of specific patterns of gene activity during development and differentiation.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '24277632',\n",
       "  'score': 0.23895177245140076},\n",
       " {'start': 0,\n",
       "  'end': 179,\n",
       "  'text': 'Grainyhead-like 2 regulates epithelial morphogenesis by establishing functional tight junctions through the organization of a molecular network among claudin3, claudin4, and Rab25.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '22696678',\n",
       "  'score': 0.2452041208744049},\n",
       " {'start': 0,\n",
       "  'end': 212,\n",
       "  'text': 'During development, epithelial progenitors establish intercellular junctions, including tight junctions (TJs), and form three-dimensional (3D) tissue structures, which are often associated with luminal structures.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '22696678',\n",
       "  'score': 0.23981311917304993},\n",
       " {'start': 214,\n",
       "  'end': 364,\n",
       "  'text': 'Here we identify grainyhead-like 2 (Grhl2) as a transcription factor that regulates the size of luminal space surrounded by polarized epithelial cells.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '22696678',\n",
       "  'score': 0.16978731751441956},\n",
       " {'start': 366,\n",
       "  'end': 510,\n",
       "  'text': 'We show that HPPL, a liver progenitor cell line, transfected with Grhl2 cDNA forms remarkably larger cysts than the control cells in 3D cultures.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '22696678',\n",
       "  'score': 0.21562442183494568},\n",
       " {'start': 512,\n",
       "  'end': 638,\n",
       "  'text': 'We find that Grhl2 up-regulates claudin (Cldn) 3 and Cldn4, and their functions are necessary for the formation of large cysts.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '22696678',\n",
       "  'score': 0.17221930623054504},\n",
       " {'start': 640,\n",
       "  'end': 696,\n",
       "  'text': 'Overexpression of Cldn3 alone induces the cyst expansion.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '22696678',\n",
       "  'score': 0.1708110272884369},\n",
       " {'start': 698,\n",
       "  'end': 793,\n",
       "  'text': 'In contrast, expression of Cldn4 alone does not induce expansion, as it is not localized at TJs.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '22696678',\n",
       "  'score': 0.1740792691707611},\n",
       " {'start': 795,\n",
       "  'end': 916,\n",
       "  'text': 'Of interest, Rab25, another Grhl2 target, not only increases the Cldn4 protein, but also enhances its localization at TJs.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '22696678',\n",
       "  'score': 0.1804715096950531},\n",
       " {'start': 918,\n",
       "  'end': 1141,\n",
       "  'text': 'Taken together, the results indicate that Grhl2 regulates epithelial morphogenesis through transcriptional up-regulation of Cldn3 and Cldn4, as well as of Rab25, which increases the Cldn4 protein and its localization at TJs.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '22696678',\n",
       "  'score': 0.20861735939979553},\n",
       " {'start': 1143,\n",
       "  'end': 1238,\n",
       "  'text': 'The results reveal a molecular network regulating epithelial lumen formation organized by Grhl2.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '22696678',\n",
       "  'score': 0.217901349067688},\n",
       " {'start': 0,\n",
       "  'end': 120,\n",
       "  'text': 'Structure function relations in PDZ-domain-containing proteins: Implications for protein networks in cellular signalling.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '29485124',\n",
       "  'score': 0.21302258968353271},\n",
       " {'start': 0,\n",
       "  'end': 143,\n",
       "  'text': 'Protein scaffolds as essential backbones for organization of supramolecular signalling complexes are a recurrent theme in several model systems.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '29485124',\n",
       "  'score': 0.22705060243606567},\n",
       " {'start': 145,\n",
       "  'end': 256,\n",
       "  'text': 'Scaffold proteins preferentially employ linear peptide binding motifs for recruiting their interaction partners.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '29485124',\n",
       "  'score': 0.16901996731758118},\n",
       " {'start': 258,\n",
       "  'end': 404,\n",
       "  'text': 'PDZ domains are one of the more commonly encountered peptide binding domains in several proteins including those involved in scaffolding functions.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '29485124',\n",
       "  'score': 0.18271741271018982},\n",
       " {'start': 406,\n",
       "  'end': 584,\n",
       "  'text': 'This domain is known for its promiscuity both in terms of ligand selection, mode of interaction with its ligands as well as its association with other protein interaction domains.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '29485124',\n",
       "  'score': 0.18577778339385986},\n",
       " {'start': 586,\n",
       "  'end': 681,\n",
       "  'text': 'PDZ domains are subject to several means of regulations by virtue of their functional diversity.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '29485124',\n",
       "  'score': 0.17047181725502014},\n",
       " {'start': 683,\n",
       "  'end': 834,\n",
       "  'text': 'Additionally, the PDZ domains are refractive to the effect of mutations and maintain their three-dimensional architecture under extreme mutational load.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '29485124',\n",
       "  'score': 0.24068427085876465},\n",
       " {'start': 836,\n",
       "  'end': 964,\n",
       "  'text': 'The biochemical and biophysical basis for this selectivity as well as promiscuity has been investigated and reviewed extensively.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '29485124',\n",
       "  'score': 0.17380714416503906},\n",
       " {'start': 966,\n",
       "  'end': 1154,\n",
       "  'text': 'The present review focuses on the plasticity inherent in PDZ domains and its implications for modular organization as well as evolution of cellular signalling pathways in higher eukaryotes.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '29485124',\n",
       "  'score': 0.21782386302947998},\n",
       " {'start': 0,\n",
       "  'end': 89,\n",
       "  'text': 'Interstitial flow regulates in\\xa0vitro three-dimensional self-organized brain micro-vessels.',\n",
       "  'is_title': True,\n",
       "  'doc_id': '32988592',\n",
       "  'score': 0.23765534162521362},\n",
       " {'start': 0,\n",
       "  'end': 122,\n",
       "  'text': 'Cell culture under medium flow has been shown to favor human brain microvascular endothelial cells function and maturation.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '32988592',\n",
       "  'score': 0.17177963256835938},\n",
       " {'start': 124,\n",
       "  'end': 453,\n",
       "  'text': 'Here a three-dimensional in\\xa0vitro model of the human brain microvasculature, comprising brain microvascular endothelial cells but also astrocytes, pericytes and a collagen type I microfiber - fibrin based matrix, was cultured under continuous medium flow in a pressure driven microphysiological system (10\\xa0kPa, in 60-30\\xa0s cycles).',\n",
       "  'is_title': False,\n",
       "  'doc_id': '32988592',\n",
       "  'score': 0.24118348956108093},\n",
       " {'start': 455,\n",
       "  'end': 528,\n",
       "  'text': 'The cells self-organized in micro-vessels perpendicular to the shear flow.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '32988592',\n",
       "  'score': 0.17417865991592407},\n",
       " {'start': 530,\n",
       "  'end': 827,\n",
       "  'text': 'Comparison with static culture showed that the resulting interstitial flow enhanced a more defined micro-vasculature network, with slightly more numerous lumens, and a higher expression of transporters, carriers and tight junction genes and proteins, essential to the blood-brain barrier functions.',\n",
       "  'is_title': False,\n",
       "  'doc_id': '32988592',\n",
       "  'score': 0.20121333003044128}]"
      ]
     },
     "execution_count": 54,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "snippets[\"6057c78994d57fd879000034\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [],
   "source": [
    "#run = create_document_run(queries, rerank, snippets)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Remove /tmp/tmpzgb_vo8x\n",
      "Remove /tmp/tmp30havt61\n",
      "Remove /tmp/tmpxg9g9cad\n",
      "Remove /tmp/tmpj6cn5epz\n",
      "Remove /tmp/tmpueui852v\n",
      "Remove /tmp/tmpbr1bg66v\n",
      "Remove /tmp/tmp33bsg8id\n",
      "Remove /tmp/tmpxlrmjsgg\n",
      "Remove /tmp/tmp7b1w23st\n",
      "Remove /tmp/tmpnp59en7u\n",
      "Remove /tmp/tmpw0sg6syr\n",
      "Remove /tmp/tmpxduwpg6b\n",
      "Remove /tmp/tmpqcfuq880\n",
      "Remove /tmp/tmpfc_z8xkm\n",
      "Remove /tmp/tmp0aa57tvm\n",
      "Remove /tmp/tmpm_82f2l1\n",
      "Remove /tmp/tmpv1n01i88\n",
      "Remove /tmp/tmpflmbqluy\n",
      "Remove /tmp/tmpeqk2jktm\n",
      "Remove /tmp/tmp0n8iv9vl\n",
      "Remove /tmp/tmpae8bglav\n",
      "Remove /tmp/tmp7inyg372\n",
      "Remove /tmp/tmpye5q_mw8\n",
      "Remove /tmp/tmpk3q4qy4j\n",
      "Remove /tmp/tmpei5_vmnn\n",
      "Remove /tmp/tmprr1epcqu\n",
      "Remove /tmp/tmpoa2srni9\n",
      "Remove /tmp/tmpkl80nh9m\n",
      "Remove /tmp/tmpqq8r6mdv\n",
      "Remove /tmp/tmpk7fgf2k4\n",
      "Remove /tmp/tmpem31wlj3\n",
      "Remove /tmp/tmp9m1mlr2n\n",
      "Remove /tmp/tmpo1d8mfie\n",
      "Remove /tmp/tmpc4fndxru\n",
      "Remove /tmp/tmp0sggiqh5\n",
      "Remove /tmp/tmpf3jycklk\n",
      "Remove /tmp/tmpms52i_6p\n",
      "Remove /tmp/tmpi_b0yiil\n",
      "Remove /tmp/tmp1zva_qaa\n",
      "Remove /tmp/tmp5c142fc_\n",
      "Remove /tmp/tmpxpcqyk23\n",
      "Remove /tmp/tmpnq1xoie9\n",
      "Remove /tmp/tmpyp2r1dgo\n",
      "Remove /tmp/tmp_h7tradf\n",
      "Remove /tmp/tmpg_xpemwi\n",
      "Remove /tmp/tmpwcxg29x5\n",
      "Remove /tmp/tmpuz589ra_\n",
      "Remove /tmp/tmpwz3wxnn1\n",
      "Remove /tmp/tmpxr48whi9\n",
      "Remove /tmp/tmpcv9l7_39\n",
      "Remove /tmp/tmp6e4xtobc\n",
      "Remove /tmp/tmpw_noa1ho\n",
      "Remove /tmp/tmpp8zxxkzj\n",
      "Remove /tmp/tmpougsnprr\n",
      "Remove /tmp/tmpqsihondi\n",
      "Remove /tmp/tmp7pqdifzc\n",
      "Remove /tmp/tmp7prg5ulq\n",
      "Remove /tmp/tmpxl_dbo3e\n",
      "Remove /tmp/tmpqjs1ubdl\n",
      "Remove /tmp/tmp0ogt63tx\n",
      "Remove /tmp/tmp2j96gi83\n",
      "Remove /tmp/tmpk_uyzri8\n",
      "Remove /tmp/tmpc5rrb0za\n",
      "Remove /tmp/tmp06f93p9_\n",
      "Remove /tmp/tmp5jk2sdss\n",
      "Remove /tmp/tmplg2_d1a_\n",
      "Remove /tmp/tmpzlskm_kp\n",
      "Remove /tmp/tmpvfes15v_\n",
      "Remove /tmp/tmp1n8w5w31\n",
      "Remove /tmp/tmp0s7n8304\n",
      "Remove /tmp/tmpa93vje6z\n",
      "Remove /tmp/tmpmqcwyri0\n",
      "Remove /tmp/tmpp8irsbz_\n",
      "Remove /tmp/tmpmt0soae6\n",
      "Remove /tmp/tmp9ufrjes9\n",
      "Remove /tmp/tmpmx9t4lzy\n"
     ]
    },
    {
     "ename": "KeyboardInterrupt",
     "evalue": "",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mKeyboardInterrupt\u001b[0m                         Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-18-cc046dbdc060>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m\u001b[0m\n\u001b[1;32m     10\u001b[0m         \u001b[0msnippets\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0msnippetRank_byThreshold\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mrerank\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0mt\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     11\u001b[0m         \u001b[0m_run\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mcreate_document_run\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mqueries\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mrerank\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0msnippets\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 12\u001b[0;31m         \u001b[0mm\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mevaluator\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mevaluate\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0m_run\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     13\u001b[0m         \u001b[0mfOut\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mwrite\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"snippetRank_byThreshold,\"\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0mstr\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mt\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m+\u001b[0m\u001b[0;34m\",\"\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0;34m\",\"\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mjoin\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m[\u001b[0m \u001b[0mstr\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mm\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0mn\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m)\u001b[0m \u001b[0;32mfor\u001b[0m \u001b[0mn\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mmetrics\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m)\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0;34m\"\\n\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     14\u001b[0m         \u001b[0mfOut\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mflush\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m~/BioASQ-9b/py-bioasq/lib/python3.6/site-packages/mmnrm/evaluation.py\u001b[0m in \u001b[0;36mevaluate\u001b[0;34m(self, run)\u001b[0m\n\u001b[1;32m    275\u001b[0m             \u001b[0;32mif\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mwrite_f\u001b[0m \u001b[0;32mis\u001b[0m \u001b[0;32mnot\u001b[0m \u001b[0;32mNone\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    276\u001b[0m                 \u001b[0mpath\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mos\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mpath\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mjoin\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mtemp_dir\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m\"retrived.json\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 277\u001b[0;31m                 \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mwrite_f\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mrun\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mpath\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    278\u001b[0m             \u001b[0;32melse\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    279\u001b[0m                 \u001b[0mpath\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mrun\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m~/BioASQ-9b/utils.py\u001b[0m in \u001b[0;36mwrite_as_bioasq\u001b[0;34m(run, file, max_docs, max_snippets)\u001b[0m\n\u001b[1;32m     72\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     73\u001b[0m \u001b[0;32mdef\u001b[0m \u001b[0mwrite_as_bioasq\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mrun\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mfile\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mmax_docs\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;36m10\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mmax_snippets\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;36m10\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 74\u001b[0;31m     \u001b[0mfinal_run\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mcopy\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mdeepcopy\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mrun\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     75\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     76\u001b[0m     \u001b[0;32mfor\u001b[0m \u001b[0mquery\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mfinal_run\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m/usr/lib/python3.6/copy.py\u001b[0m in \u001b[0;36mdeepcopy\u001b[0;34m(x, memo, _nil)\u001b[0m\n\u001b[1;32m    148\u001b[0m     \u001b[0mcopier\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0m_deepcopy_dispatch\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mget\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mcls\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    149\u001b[0m     \u001b[0;32mif\u001b[0m \u001b[0mcopier\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 150\u001b[0;31m         \u001b[0my\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mcopier\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mx\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mmemo\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    151\u001b[0m     \u001b[0;32melse\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    152\u001b[0m         \u001b[0;32mtry\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m/usr/lib/python3.6/copy.py\u001b[0m in \u001b[0;36m_deepcopy_list\u001b[0;34m(x, memo, deepcopy)\u001b[0m\n\u001b[1;32m    213\u001b[0m     \u001b[0mappend\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0my\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mappend\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    214\u001b[0m     \u001b[0;32mfor\u001b[0m \u001b[0ma\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mx\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 215\u001b[0;31m         \u001b[0mappend\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mdeepcopy\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0ma\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mmemo\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    216\u001b[0m     \u001b[0;32mreturn\u001b[0m \u001b[0my\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    217\u001b[0m \u001b[0md\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0mlist\u001b[0m\u001b[0;34m]\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0m_deepcopy_list\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m/usr/lib/python3.6/copy.py\u001b[0m in \u001b[0;36mdeepcopy\u001b[0;34m(x, memo, _nil)\u001b[0m\n\u001b[1;32m    148\u001b[0m     \u001b[0mcopier\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0m_deepcopy_dispatch\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mget\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mcls\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    149\u001b[0m     \u001b[0;32mif\u001b[0m \u001b[0mcopier\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 150\u001b[0;31m         \u001b[0my\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mcopier\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mx\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mmemo\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    151\u001b[0m     \u001b[0;32melse\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    152\u001b[0m         \u001b[0;32mtry\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m/usr/lib/python3.6/copy.py\u001b[0m in \u001b[0;36m_deepcopy_dict\u001b[0;34m(x, memo, deepcopy)\u001b[0m\n\u001b[1;32m    238\u001b[0m     \u001b[0mmemo\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0mid\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mx\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m]\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0my\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    239\u001b[0m     \u001b[0;32mfor\u001b[0m \u001b[0mkey\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mvalue\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mx\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mitems\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 240\u001b[0;31m         \u001b[0my\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0mdeepcopy\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mkey\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mmemo\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m]\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mdeepcopy\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mvalue\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mmemo\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    241\u001b[0m     \u001b[0;32mreturn\u001b[0m \u001b[0my\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    242\u001b[0m \u001b[0md\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0mdict\u001b[0m\u001b[0;34m]\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0m_deepcopy_dict\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m/usr/lib/python3.6/copy.py\u001b[0m in \u001b[0;36mdeepcopy\u001b[0;34m(x, memo, _nil)\u001b[0m\n\u001b[1;32m    148\u001b[0m     \u001b[0mcopier\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0m_deepcopy_dispatch\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mget\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mcls\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    149\u001b[0m     \u001b[0;32mif\u001b[0m \u001b[0mcopier\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 150\u001b[0;31m         \u001b[0my\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mcopier\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mx\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mmemo\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    151\u001b[0m     \u001b[0;32melse\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    152\u001b[0m         \u001b[0;32mtry\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m/usr/lib/python3.6/copy.py\u001b[0m in \u001b[0;36m_deepcopy_list\u001b[0;34m(x, memo, deepcopy)\u001b[0m\n\u001b[1;32m    213\u001b[0m     \u001b[0mappend\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0my\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mappend\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    214\u001b[0m     \u001b[0;32mfor\u001b[0m \u001b[0ma\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mx\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 215\u001b[0;31m         \u001b[0mappend\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mdeepcopy\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0ma\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mmemo\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    216\u001b[0m     \u001b[0;32mreturn\u001b[0m \u001b[0my\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    217\u001b[0m \u001b[0md\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0mlist\u001b[0m\u001b[0;34m]\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0m_deepcopy_list\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m/usr/lib/python3.6/copy.py\u001b[0m in \u001b[0;36mdeepcopy\u001b[0;34m(x, memo, _nil)\u001b[0m\n\u001b[1;32m    148\u001b[0m     \u001b[0mcopier\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0m_deepcopy_dispatch\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mget\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mcls\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    149\u001b[0m     \u001b[0;32mif\u001b[0m \u001b[0mcopier\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 150\u001b[0;31m         \u001b[0my\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mcopier\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mx\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mmemo\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    151\u001b[0m     \u001b[0;32melse\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    152\u001b[0m         \u001b[0;32mtry\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m/usr/lib/python3.6/copy.py\u001b[0m in \u001b[0;36m_deepcopy_dict\u001b[0;34m(x, memo, deepcopy)\u001b[0m\n\u001b[1;32m    238\u001b[0m     \u001b[0mmemo\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0mid\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mx\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m]\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0my\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    239\u001b[0m     \u001b[0;32mfor\u001b[0m \u001b[0mkey\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mvalue\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mx\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mitems\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 240\u001b[0;31m         \u001b[0my\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0mdeepcopy\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mkey\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mmemo\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m]\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mdeepcopy\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mvalue\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mmemo\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    241\u001b[0m     \u001b[0;32mreturn\u001b[0m \u001b[0my\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    242\u001b[0m \u001b[0md\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0mdict\u001b[0m\u001b[0;34m]\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0m_deepcopy_dict\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m/usr/lib/python3.6/copy.py\u001b[0m in \u001b[0;36mdeepcopy\u001b[0;34m(x, memo, _nil)\u001b[0m\n\u001b[1;32m    148\u001b[0m     \u001b[0mcopier\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0m_deepcopy_dispatch\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mget\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mcls\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    149\u001b[0m     \u001b[0;32mif\u001b[0m \u001b[0mcopier\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 150\u001b[0;31m         \u001b[0my\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mcopier\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mx\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mmemo\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    151\u001b[0m     \u001b[0;32melse\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    152\u001b[0m         \u001b[0;32mtry\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m/usr/lib/python3.6/copy.py\u001b[0m in \u001b[0;36m_deepcopy_list\u001b[0;34m(x, memo, deepcopy)\u001b[0m\n\u001b[1;32m    213\u001b[0m     \u001b[0mappend\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0my\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mappend\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    214\u001b[0m     \u001b[0;32mfor\u001b[0m \u001b[0ma\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mx\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 215\u001b[0;31m         \u001b[0mappend\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mdeepcopy\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0ma\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mmemo\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    216\u001b[0m     \u001b[0;32mreturn\u001b[0m \u001b[0my\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    217\u001b[0m \u001b[0md\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0mlist\u001b[0m\u001b[0;34m]\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0m_deepcopy_list\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m/usr/lib/python3.6/copy.py\u001b[0m in \u001b[0;36mdeepcopy\u001b[0;34m(x, memo, _nil)\u001b[0m\n\u001b[1;32m    148\u001b[0m     \u001b[0mcopier\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0m_deepcopy_dispatch\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mget\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mcls\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    149\u001b[0m     \u001b[0;32mif\u001b[0m \u001b[0mcopier\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 150\u001b[0;31m         \u001b[0my\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mcopier\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mx\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mmemo\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    151\u001b[0m     \u001b[0;32melse\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    152\u001b[0m         \u001b[0;32mtry\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m/usr/lib/python3.6/copy.py\u001b[0m in \u001b[0;36m_deepcopy_dict\u001b[0;34m(x, memo, deepcopy)\u001b[0m\n\u001b[1;32m    238\u001b[0m     \u001b[0mmemo\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0mid\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mx\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m]\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0my\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    239\u001b[0m     \u001b[0;32mfor\u001b[0m \u001b[0mkey\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mvalue\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mx\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mitems\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 240\u001b[0;31m         \u001b[0my\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0mdeepcopy\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mkey\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mmemo\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m]\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mdeepcopy\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mvalue\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mmemo\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    241\u001b[0m     \u001b[0;32mreturn\u001b[0m \u001b[0my\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    242\u001b[0m \u001b[0md\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0mdict\u001b[0m\u001b[0;34m]\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0m_deepcopy_dict\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m/usr/lib/python3.6/copy.py\u001b[0m in \u001b[0;36mdeepcopy\u001b[0;34m(x, memo, _nil)\u001b[0m\n\u001b[1;32m    144\u001b[0m         \u001b[0;32mreturn\u001b[0m \u001b[0my\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    145\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 146\u001b[0;31m     \u001b[0mcls\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mtype\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mx\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    147\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    148\u001b[0m     \u001b[0mcopier\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0m_deepcopy_dispatch\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mget\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mcls\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;31mKeyboardInterrupt\u001b[0m: "
     ]
    }
   ],
   "source": [
    "\n",
    "with open(\"snippet_bioasq_9b_batch_03_snippetRank_byThreshold\"+\".csv\", \"w\") as fOut:\n",
    "    metrics = [\"snippet_p@10\", \"snippet_r@10\", \"snippet_f1@10\", \"snippet_map@10\", \"snippet_gmap@10\"]\n",
    "    \n",
    "    header = \"f_name,threshold,\" + \",\".join(metrics)+\"\\n\"\n",
    "    fOut.write(header)\n",
    "    \n",
    "    T = list(map(lambda x:x/10, range(0,101,1)))\n",
    "    for t in T:\n",
    "    \n",
    "        snippets = snippetRank_byThreshold(rerank,t)\n",
    "        _run = create_document_run(queries, rerank, snippets)\n",
    "        m = evaluator.evaluate(_run)\n",
    "        fOut.write(\"snippetRank_byThreshold,\" + str(t)+\",\" + \",\".join([ str(m[n]) for n in metrics]) + \"\\n\")\n",
    "        fOut.flush()\n",
    "    # snippetRank_byThreshold_and_TopK\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Remove /tmp/tmp1wwvt48z\n",
      "Remove /tmp/tmpaequkr0h\n",
      "Remove /tmp/tmpvc15mw42\n",
      "Remove /tmp/tmp98uetr0j\n",
      "Remove /tmp/tmpq70ab_3_\n",
      "Remove /tmp/tmpjp6haap8\n",
      "Remove /tmp/tmpmw6qpwar\n",
      "Remove /tmp/tmpfxpq94lf\n",
      "Remove /tmp/tmp9q80cql1\n",
      "Remove /tmp/tmp0mox8ihe\n",
      "Remove /tmp/tmpyq6e8iad\n",
      "Remove /tmp/tmpp75bed5a\n",
      "Remove /tmp/tmp6egxz5qb\n",
      "Remove /tmp/tmpsanj4sf1\n",
      "Remove /tmp/tmp2ogewxgl\n",
      "Remove /tmp/tmputfaywd6\n",
      "Remove /tmp/tmpt165w6a1\n",
      "Remove /tmp/tmp1bfbhyka\n",
      "Remove /tmp/tmpjijj1xt0\n",
      "Remove /tmp/tmpctmbm9z1\n",
      "Remove /tmp/tmpkib_y8up\n",
      "Remove /tmp/tmpy3pql0ef\n",
      "Remove /tmp/tmp2lm4v1ek\n",
      "Remove /tmp/tmpv19mmu9e\n",
      "Remove /tmp/tmpovzu979o\n",
      "Remove /tmp/tmpfvagcnu_\n",
      "Remove /tmp/tmphe_pizsr\n",
      "Remove /tmp/tmpuoyt2so8\n",
      "Remove /tmp/tmpep9jfzpl\n",
      "Remove /tmp/tmp1svccdsy\n",
      "Remove /tmp/tmpyyifnwot\n",
      "Remove /tmp/tmpwgfrg8_b\n",
      "Remove /tmp/tmp6atqx1gn\n",
      "Remove /tmp/tmp2a9wm7qi\n",
      "Remove /tmp/tmpidldt3t4\n",
      "Remove /tmp/tmpjgd1ef77\n",
      "Remove /tmp/tmplan_kdis\n",
      "Remove /tmp/tmp0uaywycy\n",
      "Remove /tmp/tmppnlwxxj7\n",
      "Remove /tmp/tmpz2_04w8p\n",
      "Remove /tmp/tmpdwlvx8q6\n",
      "Remove /tmp/tmp42lt43sz\n",
      "Remove /tmp/tmpon8sbz_q\n",
      "Remove /tmp/tmps_pgeezk\n",
      "Remove /tmp/tmplq7j9aex\n",
      "Remove /tmp/tmpxenrxh73\n",
      "Remove /tmp/tmphty0cmyb\n",
      "Remove /tmp/tmplrlwa0pf\n",
      "Remove /tmp/tmp3vjk5o4y\n",
      "Remove /tmp/tmpyqzj4itq\n",
      "Remove /tmp/tmph3ub85ic\n",
      "Remove /tmp/tmp949kiw95\n",
      "Remove /tmp/tmpd0g_86fx\n",
      "Remove /tmp/tmp1qkzy2kr\n",
      "Remove /tmp/tmpmz5f_nc2\n",
      "Remove /tmp/tmpxazjzybr\n",
      "Remove /tmp/tmpofpn2tw8\n",
      "Remove /tmp/tmpeewd_vja\n",
      "Remove /tmp/tmphjceet4x\n",
      "Remove /tmp/tmpgs9qu8to\n",
      "Remove /tmp/tmpdp3vm3sz\n",
      "Remove /tmp/tmp9u8neb83\n",
      "Remove /tmp/tmp5fl65d0g\n",
      "Remove /tmp/tmp2vfu9whw\n",
      "Remove /tmp/tmpz3h94a7b\n",
      "Remove /tmp/tmppk9zjvpv\n",
      "Remove /tmp/tmpoznd_n9c\n",
      "Remove /tmp/tmpdri4tlfq\n",
      "Remove /tmp/tmpofkq3nxt\n",
      "Remove /tmp/tmp8fp8xx_z\n",
      "Remove /tmp/tmpgk7a95js\n",
      "Remove /tmp/tmpcmw4ldty\n",
      "Remove /tmp/tmp2as5wuuj\n",
      "Remove /tmp/tmpwcyj14wv\n",
      "Remove /tmp/tmpld5ou06n\n",
      "Remove /tmp/tmpmiaum2pl\n",
      "Remove /tmp/tmpcwbx1wy1\n",
      "Remove /tmp/tmp43sl1elk\n",
      "Remove /tmp/tmp7pw92r9s\n",
      "Remove /tmp/tmpgvsl95aw\n",
      "Remove /tmp/tmpzf3tu0x6\n",
      "Remove /tmp/tmpo9vow96b\n",
      "Remove /tmp/tmpam6c96k5\n",
      "Remove /tmp/tmpzi5r_skt\n",
      "Remove /tmp/tmp1kr1ewxd\n",
      "Remove /tmp/tmpf4ajfzi8\n",
      "Remove /tmp/tmpy6i_0bvy\n",
      "Remove /tmp/tmpptolpjef\n",
      "Remove /tmp/tmpegqmjwob\n",
      "Remove /tmp/tmp1j80ehgd\n",
      "Remove /tmp/tmphc1y9n0v\n",
      "Remove /tmp/tmp0px5io0u\n",
      "Remove /tmp/tmp06crt5h_\n",
      "Remove /tmp/tmpa0ox28wc\n",
      "Remove /tmp/tmp87e0vokd\n",
      "Remove /tmp/tmpsh0ek7fk\n",
      "Remove /tmp/tmpi2pgzkh6\n",
      "Remove /tmp/tmp1z05mrrx\n",
      "Remove /tmp/tmpr71yhuus\n",
      "Remove /tmp/tmpmaied4gd\n",
      "Remove /tmp/tmp8_ynvwz4\n",
      "Remove /tmp/tmpt7_lgrf5\n",
      "Remove /tmp/tmpa9w2dqm8\n",
      "Remove /tmp/tmppxqh9fqb\n",
      "Remove /tmp/tmpkpg4h7q3\n",
      "Remove /tmp/tmpds93enhx\n",
      "Remove /tmp/tmpu201w8t2\n",
      "Remove /tmp/tmph0i3xvsb\n",
      "Remove /tmp/tmpzvggpgs1\n",
      "Remove /tmp/tmplw32rf1f\n",
      "Remove /tmp/tmpg85lmuu2\n",
      "Remove /tmp/tmpm1067zfu\n",
      "Remove /tmp/tmpyj7g9v6i\n",
      "Remove /tmp/tmpsg_t8d9v\n",
      "Remove /tmp/tmpkxfu8kkn\n",
      "Remove /tmp/tmpw10bamau\n",
      "Remove /tmp/tmpiv7hh2ic\n",
      "Remove /tmp/tmp_ur8kkmn\n",
      "Remove /tmp/tmp_2ak_e52\n",
      "Remove /tmp/tmpk1p3lcmk\n",
      "Remove /tmp/tmpqbca3b5f\n",
      "Remove /tmp/tmpwv89461n\n",
      "Remove /tmp/tmpivusu9ue\n",
      "Remove /tmp/tmpz1svx5zh\n",
      "Remove /tmp/tmpzx78px1k\n",
      "Remove /tmp/tmpph_lgr79\n",
      "Remove /tmp/tmpljvsau1u\n",
      "Remove /tmp/tmpeey8ac12\n",
      "Remove /tmp/tmpnu0bj8yc\n",
      "Remove /tmp/tmpx8q4dsx4\n",
      "Remove /tmp/tmpwbgklj28\n",
      "Remove /tmp/tmp_7qf_rrk\n",
      "Remove /tmp/tmpqy0llt5w\n",
      "Remove /tmp/tmpuxx2llz9\n",
      "Remove /tmp/tmp7k3wsbiw\n",
      "Remove /tmp/tmpvh247kad\n",
      "Remove /tmp/tmpcka9hhyy\n",
      "Remove /tmp/tmpav9t9f3h\n",
      "Remove /tmp/tmp9p241ezw\n",
      "Remove /tmp/tmp4ojz5fss\n",
      "Remove /tmp/tmprlctmyca\n",
      "Remove /tmp/tmp9abs47gg\n",
      "Remove /tmp/tmpe2c27_0c\n",
      "Remove /tmp/tmp08wdtzoz\n",
      "Remove /tmp/tmplwka9gyi\n",
      "Remove /tmp/tmp63ftxrz3\n",
      "Remove /tmp/tmph5xqb7xr\n",
      "Remove /tmp/tmpe3702bg0\n",
      "Remove /tmp/tmpwlnn2ciu\n",
      "Remove /tmp/tmpqeixlcb9\n",
      "Remove /tmp/tmpg2m__iay\n",
      "Remove /tmp/tmpfx2a1e34\n",
      "Remove /tmp/tmpq0zn5khd\n",
      "Remove /tmp/tmp3bocwlvs\n",
      "Remove /tmp/tmpp_38if6y\n",
      "Remove /tmp/tmpcemmrvnm\n",
      "Remove /tmp/tmpo49fshjk\n",
      "Remove /tmp/tmprv5xzgph\n",
      "Remove /tmp/tmpwoyaixuc\n",
      "Remove /tmp/tmpzkwnjm91\n",
      "Remove /tmp/tmpqkdlohq5\n",
      "Remove /tmp/tmpdlof3i1q\n",
      "Remove /tmp/tmpohzjbjf0\n",
      "Remove /tmp/tmp0yzhc7k9\n",
      "Remove /tmp/tmprrnqrvdc\n",
      "Remove /tmp/tmpyhu1fye5\n",
      "Remove /tmp/tmp135nenil\n",
      "Remove /tmp/tmpu8l3gp1a\n",
      "Remove /tmp/tmprk8y2mmt\n",
      "Remove /tmp/tmp7jmxelsp\n",
      "Remove /tmp/tmpkmrs8rsn\n",
      "Remove /tmp/tmpnnbmk7za\n",
      "Remove /tmp/tmpdbesxpzk\n",
      "Remove /tmp/tmpxc4rw3e2\n",
      "Remove /tmp/tmpd6vqydy7\n",
      "Remove /tmp/tmpvcejl8xj\n",
      "Remove /tmp/tmpvn5b3gkx\n",
      "Remove /tmp/tmp_n4gb_b7\n",
      "Remove /tmp/tmp54a6e_b5\n",
      "Remove /tmp/tmp3i9y1587\n",
      "Remove /tmp/tmpkmahivlm\n",
      "Remove /tmp/tmpozhn_irn\n",
      "Remove /tmp/tmp914g0hli\n",
      "Remove /tmp/tmpkx092q9h\n",
      "Remove /tmp/tmp1l95308b\n",
      "Remove /tmp/tmpdrlxcyx2\n",
      "Remove /tmp/tmp6lzzxs80\n",
      "Remove /tmp/tmp70hd5ysa\n",
      "Remove /tmp/tmplu5udyhk\n",
      "Remove /tmp/tmp27rst2wm\n",
      "Remove /tmp/tmpcgyzhisj\n",
      "Remove /tmp/tmpgyvo51f4\n",
      "Remove /tmp/tmpj7b2hpje\n",
      "Remove /tmp/tmpfav3esa_\n",
      "Remove /tmp/tmpguqe5e75\n",
      "Remove /tmp/tmpfsvhfnvf\n",
      "Remove /tmp/tmp0c7f2jp7\n",
      "Remove /tmp/tmpbw38gbp0\n",
      "Remove /tmp/tmp3pjs18m3\n",
      "Remove /tmp/tmpczvbj64t\n",
      "Remove /tmp/tmpa_z7q6aw\n",
      "Remove /tmp/tmpqmgesz7f\n",
      "Remove /tmp/tmpoo9j6gn7\n",
      "Remove /tmp/tmp8dc6zf0j\n"
     ]
    }
   ],
   "source": [
    "import itertools\n",
    "\n",
    "with open(\"snippet_bioasq_9b_batch_03_snippetRank_byThreshold_and_TopK\"+\".csv\", \"w\") as fOut:\n",
    "    metrics = [\"snippet_p@10\", \"snippet_r@10\", \"snippet_f1@10\", \"snippet_map@10\", \"snippet_gmap@10\"]\n",
    "    \n",
    "    header = \"f_name,threshold,topK\" + \",\".join(metrics)+\"\\n\"\n",
    "    fOut.write(header)\n",
    "    \n",
    "    T = list(map(lambda x:x/100, range(0,51,1)))\n",
    "    TOPK = list(range(1,5,1))\n",
    "    \n",
    "    combinations = list(itertools.product(T,TOPK))\n",
    "    \n",
    "    for t,k in combinations:\n",
    "    \n",
    "        snippets = snippetRank_byThreshold_and_TopK(rerank,t,k)\n",
    "        _run = create_document_run(queries, rerank, snippets)\n",
    "        m = evaluator.evaluate(_run)\n",
    "        fOut.write(\"snippetRank_byThreshold_and_TopK,\" + str(t)+\",\" +str(k)+\",\"+ \",\".join([ str(m[n]) for n in metrics]) + \"\\n\")\n",
    "        fOut.flush()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "df = pd.read_csv(\"snippet_bioasq_9b_batch_03_snippetRank_byThreshold.csv\")\n",
    "df = df.sort_values('snippet_f1@10',ascending=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>f_name</th>\n",
       "      <th>threshold</th>\n",
       "      <th>topKsnippet_p@10</th>\n",
       "      <th>snippet_r@10</th>\n",
       "      <th>snippet_f1@10</th>\n",
       "      <th>snippet_map@10</th>\n",
       "      <th>snippet_gmap@10</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>snippetRank_byThreshold_and_TopK</th>\n",
       "      <td>0.08</td>\n",
       "      <td>1</td>\n",
       "      <td>0.167207</td>\n",
       "      <td>0.215893</td>\n",
       "      <td>0.178072</td>\n",
       "      <td>0.672674</td>\n",
       "      <td>0.000465</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>snippetRank_byThreshold_and_TopK</th>\n",
       "      <td>0.12</td>\n",
       "      <td>1</td>\n",
       "      <td>0.169362</td>\n",
       "      <td>0.212623</td>\n",
       "      <td>0.177712</td>\n",
       "      <td>0.643631</td>\n",
       "      <td>0.000459</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>snippetRank_byThreshold_and_TopK</th>\n",
       "      <td>0.11</td>\n",
       "      <td>1</td>\n",
       "      <td>0.167702</td>\n",
       "      <td>0.213821</td>\n",
       "      <td>0.177454</td>\n",
       "      <td>0.652801</td>\n",
       "      <td>0.000461</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>snippetRank_byThreshold_and_TopK</th>\n",
       "      <td>0.09</td>\n",
       "      <td>1</td>\n",
       "      <td>0.166875</td>\n",
       "      <td>0.214446</td>\n",
       "      <td>0.177414</td>\n",
       "      <td>0.659153</td>\n",
       "      <td>0.000463</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>snippetRank_byThreshold_and_TopK</th>\n",
       "      <td>0.10</td>\n",
       "      <td>1</td>\n",
       "      <td>0.167288</td>\n",
       "      <td>0.213821</td>\n",
       "      <td>0.177234</td>\n",
       "      <td>0.653516</td>\n",
       "      <td>0.000462</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>snippetRank_byThreshold_and_TopK</th>\n",
       "      <td>0.32</td>\n",
       "      <td>2</td>\n",
       "      <td>0.139549</td>\n",
       "      <td>0.227787</td>\n",
       "      <td>0.160634</td>\n",
       "      <td>0.564154</td>\n",
       "      <td>0.000898</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>snippetRank_byThreshold_and_TopK</th>\n",
       "      <td>0.33</td>\n",
       "      <td>4</td>\n",
       "      <td>0.138182</td>\n",
       "      <td>0.231159</td>\n",
       "      <td>0.160190</td>\n",
       "      <td>0.561348</td>\n",
       "      <td>0.000900</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>snippetRank_byThreshold_and_TopK</th>\n",
       "      <td>0.33</td>\n",
       "      <td>3</td>\n",
       "      <td>0.138182</td>\n",
       "      <td>0.231159</td>\n",
       "      <td>0.160190</td>\n",
       "      <td>0.561348</td>\n",
       "      <td>0.000900</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>snippetRank_byThreshold_and_TopK</th>\n",
       "      <td>0.32</td>\n",
       "      <td>3</td>\n",
       "      <td>0.137730</td>\n",
       "      <td>0.230491</td>\n",
       "      <td>0.159620</td>\n",
       "      <td>0.566314</td>\n",
       "      <td>0.000901</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>snippetRank_byThreshold_and_TopK</th>\n",
       "      <td>0.32</td>\n",
       "      <td>4</td>\n",
       "      <td>0.137730</td>\n",
       "      <td>0.230491</td>\n",
       "      <td>0.159620</td>\n",
       "      <td>0.566314</td>\n",
       "      <td>0.000901</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>204 rows × 7 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                  f_name  threshold  topKsnippet_p@10  \\\n",
       "snippetRank_byThreshold_and_TopK    0.08          1          0.167207   \n",
       "snippetRank_byThreshold_and_TopK    0.12          1          0.169362   \n",
       "snippetRank_byThreshold_and_TopK    0.11          1          0.167702   \n",
       "snippetRank_byThreshold_and_TopK    0.09          1          0.166875   \n",
       "snippetRank_byThreshold_and_TopK    0.10          1          0.167288   \n",
       "...                                  ...        ...               ...   \n",
       "snippetRank_byThreshold_and_TopK    0.32          2          0.139549   \n",
       "snippetRank_byThreshold_and_TopK    0.33          4          0.138182   \n",
       "snippetRank_byThreshold_and_TopK    0.33          3          0.138182   \n",
       "snippetRank_byThreshold_and_TopK    0.32          3          0.137730   \n",
       "snippetRank_byThreshold_and_TopK    0.32          4          0.137730   \n",
       "\n",
       "                                  snippet_r@10  snippet_f1@10  snippet_map@10  \\\n",
       "snippetRank_byThreshold_and_TopK      0.215893       0.178072        0.672674   \n",
       "snippetRank_byThreshold_and_TopK      0.212623       0.177712        0.643631   \n",
       "snippetRank_byThreshold_and_TopK      0.213821       0.177454        0.652801   \n",
       "snippetRank_byThreshold_and_TopK      0.214446       0.177414        0.659153   \n",
       "snippetRank_byThreshold_and_TopK      0.213821       0.177234        0.653516   \n",
       "...                                        ...            ...             ...   \n",
       "snippetRank_byThreshold_and_TopK      0.227787       0.160634        0.564154   \n",
       "snippetRank_byThreshold_and_TopK      0.231159       0.160190        0.561348   \n",
       "snippetRank_byThreshold_and_TopK      0.231159       0.160190        0.561348   \n",
       "snippetRank_byThreshold_and_TopK      0.230491       0.159620        0.566314   \n",
       "snippetRank_byThreshold_and_TopK      0.230491       0.159620        0.566314   \n",
       "\n",
       "                                  snippet_gmap@10  \n",
       "snippetRank_byThreshold_and_TopK         0.000465  \n",
       "snippetRank_byThreshold_and_TopK         0.000459  \n",
       "snippetRank_byThreshold_and_TopK         0.000461  \n",
       "snippetRank_byThreshold_and_TopK         0.000463  \n",
       "snippetRank_byThreshold_and_TopK         0.000462  \n",
       "...                                           ...  \n",
       "snippetRank_byThreshold_and_TopK         0.000898  \n",
       "snippetRank_byThreshold_and_TopK         0.000900  \n",
       "snippetRank_byThreshold_and_TopK         0.000900  \n",
       "snippetRank_byThreshold_and_TopK         0.000901  \n",
       "snippetRank_byThreshold_and_TopK         0.000901  \n",
       "\n",
       "[204 rows x 7 columns]"
      ]
     },
     "execution_count": 17,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import pandas as pd\n",
    "df = pd.read_csv(\"snippet_bioasq_9b_batch_03_snippetRank_byThreshold_and_TopK.csv\")\n",
    "df.sort_values('snippet_f1@10',ascending=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 45,
   "metadata": {},
   "outputs": [],
   "source": [
    "punkt_sent_tokenizer = PunktSentenceTokenizer().span_tokenize"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 47,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[(0, 6), (7, 22), (23, 32)]"
      ]
     },
     "execution_count": 47,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "list(punkt_sent_tokenizer(\"asdfa. asdf asdf asdf. asdf asdf\"))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Test Callback Validation with snippets\n",
    "from mmnrm.callbacks import Validation\n",
    "from mmnrm.evaluation import BioASQ_JavaEvaluator\n",
    "\n",
    "evaluator = BioASQ_JavaEvaluator(\"/home/tiagoalmeida/BioASQ-9b/yearly_data/8B5_golden.json\", \n",
    "                                 \"/home/tiagoalmeida/BioASQ-9b/BioASQEvaluator/BioASQEvaluation.jar\",\n",
    "                                 write_as_bioasq)\n",
    "\n",
    "validation_collection = TestCollectionV2.load(\"validation_batch_05_0.4_0.44_100\")\\\n",
    "                                    .batch_size(100)\\\n",
    "                                    .set_name(\"Validation TOP (recall) 100\")\n",
    "validation_collection.evaluator = evaluator\n",
    "\n",
    "validation_collection.save(\"validation_wSnippets_batch_05_0.4_0.44_100\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "from mmnrm.callbacks import Validation\n",
    "from mmnrm.evaluation import BioASQ_JavaEvaluator\n",
    "\n",
    "def snippetRank_byThreshold(threshold):\n",
    "    \n",
    "    def snippetRank(results):\n",
    "    \n",
    "        snippets_results = {}\n",
    "        # this will follow the document order first\n",
    "        for q in results.keys():\n",
    "            snippets_results[q] = [y for y in flat_list([x[\"snippets\"] for x in results[q]]) if y[\"score\"] >= threshold]\n",
    "\n",
    "        return snippets_results\n",
    "    \n",
    "    return snippetRank\n",
    "\n",
    "test_input_generator = get_test_generator_for_model(ranking_model, use_joint=True)\n",
    "\n",
    "evaluator = BioASQ_JavaEvaluator(\"/home/tiagoalmeida/BioASQ-9b/yearly_data/8B5_golden.json\", \n",
    "                                 \"/home/tiagoalmeida/BioASQ-9b/BioASQEvaluator/BioASQEvaluation.jar\",\n",
    "                                 write_as_bioasq)\n",
    "\n",
    "validation_collection = TestCollectionV2.load(\"validation_batch_05_0.4_0.44_100\")\\\n",
    "                                    .batch_size(100)\\\n",
    "                                    .set_transform_inputs_fn(test_input_generator)\\\n",
    "                                    .set_name(\"Validation TOP (recall) 100\")\n",
    "\n",
    "validation_collection.evaluator = evaluator\n",
    "\n",
    "snippet_rank_f = snippetRank_byThreshold(0.08) \n",
    "\n",
    "val = Validation(validation_collection = [validation_collection],# TODO\n",
    "                 output_metrics=[\"doc_r@10\", \"doc_map@10\",\"snippet_f1@10\",\"snippet_map@10\"],\n",
    "                 path_store = None,\n",
    "                 snippet_rank_f = snippet_rank_f)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Evaluation 50 | avg 50-time 0.3404881954193115\r"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "IOPub data rate exceeded.\n",
      "The notebook server will temporarily stop sending output\n",
      "to the client in order to avoid crashing it.\n",
      "To change this limit, set the config variable\n",
      "`--NotebookApp.iopub_data_rate_limit`.\n",
      "\n",
      "Current values:\n",
      "NotebookApp.iopub_data_rate_limit=1000000.0 (bytes/sec)\n",
      "NotebookApp.rate_limit_window=3.0 (secs)\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Remove /tmp/tmpqi5vduy0\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "{'doc_p@10': '0.1499999999999999',\n",
       " 'doc_r@10': '0.5829252969252969',\n",
       " 'doc_f1@10': '0.21479179024935535',\n",
       " 'doc_map@10': '0.4581660918997229',\n",
       " 'doc_gmap@10': '0.02964828870067375',\n",
       " 'snippet_p@10': '0.17166116324064698',\n",
       " 'snippet_r@10': '0.28734795914322975',\n",
       " 'snippet_f1@10': '0.19953835567849598',\n",
       " 'snippet_map@10': '0.7424392903661619',\n",
       " 'snippet_gmap@10': '0.003843435329089425'}"
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "\n",
    "val.evaluate(ranking_model.predict, validation_collection)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "\n",
    "\n",
    "validation_collection.save(\"validation_wSnippets_batch_05_0.4_0.44_100\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "validation_collection = TestCollectionV2.load(\"validation_wSnippets_batch_05_0.4_0.44_100\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "<mmnrm.evaluation.BioASQ_JavaEvaluator at 0x7fba15bebf98>"
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "validation_collection.evaluator"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Remove /tmp/tmper3ivh24\n",
      "8b-b04-bioinfo-2.json:\n",
      "\tDoc:\n",
      "\t\tMap@10: 0.40057199546485267\n",
      "\t\tRecall@10: 0.5552987012987013\n",
      "\tSnippet:\n",
      "\t\tMap@10: 0.3658896457209854\n",
      "\t\tRecall@10: 0.2584178478525874\n",
      "\t\tF1@10: 0.17229742697268072\n"
     ]
    }
   ],
   "source": [
    "runs = [\"8b-b04-bioinfo-2.json\"]\n",
    "\n",
    "evaluator = BioASQ_JavaEvaluator(\"yearly_data/8B4_golden.json\", \n",
    "                                 \"/home/tiagoalmeida/BioASQ-9b/BioASQEvaluator/BioASQEvaluation.jar\")\n",
    "\n",
    "for r in runs:\n",
    "    m = evaluator.evaluate(r)\n",
    "    out = f\"{r}:\\n\\tDoc:\\n\\t\\tMap@10: {m['doc_map@10']}\\n\\t\\tRecall@10: {m['doc_r@10']}\"\n",
    "    if float(m['snippet_r@10'])>0:\n",
    "        out += f\"\\n\\tSnippet:\\n\\t\\tMap@10: {m['snippet_map@10']}\\n\\t\\tRecall@10: {m['snippet_r@10']}\\n\\t\\tF1@10: {m['snippet_f1@10']}\"\n",
    "    print(out)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Remove /tmp/tmp5db62sdn\n",
      "8b-b04-bioinfo-2.json:\n",
      "\tDoc:\n",
      "\t\tMap@10: 0.48099051555807515\n",
      "\t\tRecall@10: 0.6006318518646105\n",
      "\tSnippet:\n",
      "\t\tMap@10: 0.40186301388153983\n",
      "\t\tRecall@10: 0.27501143521870997\n",
      "\t\tF1@10: 0.20362609203479196\n"
     ]
    }
   ],
   "source": [
    "runs = [\"8b-b04-bioinfo-2.json\"]\n",
    "\n",
    "evaluator = BioASQ_JavaEvaluator(\"yearly_data/8B4_golden.json\", \n",
    "                                 \"/home/tiagoalmeida/BioASQ-9b/BioASQEvaluator/BioASQEvaluation.jar\")\n",
    "\n",
    "for r in runs:\n",
    "    m = evaluator.evaluate(r)\n",
    "    out = f\"{r}:\\n\\tDoc:\\n\\t\\tMap@10: {m['doc_map@10']}\\n\\t\\tRecall@10: {m['doc_r@10']}\"\n",
    "    if float(m['snippet_r@10'])>0:\n",
    "        out += f\"\\n\\tSnippet:\\n\\t\\tMap@10: {m['snippet_map@10']}\\n\\t\\tRecall@10: {m['snippet_r@10']}\\n\\t\\tF1@10: {m['snippet_f1@10']}\"\n",
    "    print(out)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[0.0, 0.02, 0.04, 0.06, 0.08, 0.1, 0.12, 0.14, 0.16, 0.18]"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "[t/50 for t in range(0,10)]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "DEBUG created tokenizer bioasq_9b_RegexTokenizer\n",
      "False False\n",
      "[LOAD FROM CACHE] Load embedding matrix from /backup/BioASQ-9b/embeddings/WORD2VEC_embedding_bioasq_9b_gensim_iter_15_freq0_200_Regex_word2vec_bioasq_9b_RegexTokenizer\n",
      "Using einsum for mask bq,bps->bpqs and with embedding dim bqe,bpse->bpqs\n",
      "[EMBEDDING MATRIX SHAPE] (5322623, 200)\n"
     ]
    }
   ],
   "source": [
    "def build_data_generators(tokenizer, queries_sw=None, docs_sw=None):\n",
    "    \n",
    "    def maybe_tokenize(documents):\n",
    "        if \"tokens\" not in documents:\n",
    "            #split = nltk.sent_tokenize(documents[\"text\"])\n",
    "            documents[\"tokens\"] = tokenizer.texts_to_sequences(map(lambda x: x[\"text\"],documents[\"snippets\"]))\n",
    "            if docs_sw is not None:\n",
    "                for tokenized_sentence in documents[\"tokens\"]:\n",
    "                    tokenized_sentence = [token for token in tokenized_sentence if token not in docs_sw]\n",
    "    \n",
    "    def train_generator(data_generator):\n",
    "        while True:\n",
    "\n",
    "            # get the batch triplet\n",
    "            query, pos_docs, pos_label, neg_docs = next(data_generator)\n",
    "\n",
    "            # tokenization, this can be cached for efficientcy porpuses NOTE!!\n",
    "            tokenized_query = tokenizer.texts_to_sequences(query)\n",
    "\n",
    "            if queries_sw is not None:\n",
    "                for tokens in tokenized_query:\n",
    "                    tokenized_query = [token for token in tokens if token not in queries_sw] \n",
    "            \n",
    "            saveReturn = True\n",
    "            \n",
    "            for batch_index in range(len(pos_docs)):\n",
    "                \n",
    "                # tokenizer with cache in [batch_index][tokens]\n",
    "                maybe_tokenize(pos_docs[batch_index])\n",
    "                \n",
    "                # assertion\n",
    "                if all([ len(sentence)==0  for sentence in pos_docs[batch_index][\"tokens\"]]):\n",
    "                    saveReturn = False\n",
    "                    break # try a new resampling, NOTE THIS IS A EASY FIX PLS REDO THIS!!!!!!!\n",
    "                          # for obvious reasons\n",
    "                \n",
    "                maybe_tokenize(neg_docs[batch_index])\n",
    "                \n",
    "            if saveReturn: # this is not true, if the batch is rejected\n",
    "                yield tokenized_query, pos_docs, pos_label, neg_docs\n",
    "                \n",
    "            \n",
    "    return train_generator, None\n",
    "\n",
    "def train_generator_for_model(model):\n",
    "\n",
    "    if \"model\" in model.savable_config:\n",
    "        cfg = model.savable_config[\"model\"]\n",
    "    \n",
    "    train_gen, test_gen = build_data_generators(model.tokenizer)\n",
    "    \n",
    "    pad_tokens = lambda x, max_len, dtype='int32': tf.keras.preprocessing.sequence.pad_sequences(x, \n",
    "                                                                                           maxlen=max_len,\n",
    "                                                                                           dtype=dtype, \n",
    "                                                                                           padding='post', \n",
    "                                                                                           truncating='post', \n",
    "                                                                                           value=0)\n",
    "\n",
    "    pad_sentences = lambda x, max_lim, dtype='int32': x[:max_lim] + [[]]*(max_lim-len(x))\n",
    "    \n",
    "    pad_labels = lambda x, max_lim, dtype='int32': x[:max_lim] + [0]*(max_lim-len(x))\n",
    "    \n",
    "    def maybe_padding(document, labels = None):\n",
    "        if isinstance(document[\"tokens\"], list):\n",
    "            #overflow prevention\n",
    "            bounded_doc_passage = min(cfg[\"max_passages\"],len(document[\"tokens\"]))\n",
    "            document[\"sentences_mask\"] = [True] * bounded_doc_passage + [False] * (cfg[\"max_passages\"]-bounded_doc_passage)\n",
    "            document[\"tokens\"] = pad_tokens(pad_sentences(document[\"tokens\"], cfg[\"max_passages\"]), cfg[\"max_p_terms\"])\n",
    "            if labels is not None:\n",
    "                document[\"sentences_labels\"] = pad_labels(labels, cfg[\"max_passages\"])\n",
    "            \n",
    "    def train_generator(data_generator):\n",
    " \n",
    "        for query, pos_docs, pos_label, neg_docs in train_gen(data_generator):\n",
    "            \n",
    "            query = pad_tokens(query, cfg[\"max_q_terms\"])\n",
    "            \n",
    "            pos_docs_array = []\n",
    "            pos_snippets_labels = []\n",
    "            pos_docs_mask_array = []\n",
    "            neg_docs_array = []\n",
    "            neg_docs_mask_array = []\n",
    "            \n",
    "            # pad docs, use cache here\n",
    "            for batch_index in range(len(pos_docs)):\n",
    "                maybe_padding(pos_docs[batch_index], pos_label[batch_index])\n",
    "                pos_docs_array.append(pos_docs[batch_index][\"tokens\"])\n",
    "                pos_snippets_labels.append(pos_docs[batch_index][\"sentences_labels\"])\n",
    "                pos_docs_mask_array.append(pos_docs[batch_index][\"sentences_mask\"])\n",
    "                maybe_padding(neg_docs[batch_index])\n",
    "                neg_docs_array.append(neg_docs[batch_index][\"tokens\"])\n",
    "                neg_docs_mask_array.append(neg_docs[batch_index][\"sentences_mask\"])\n",
    "            \n",
    "            yield [query, np.array(pos_docs_array), np.array(pos_docs_mask_array), np.array(pos_snippets_labels)], [query, np.array(neg_docs_array), np.array(neg_docs_mask_array)]\n",
    "            \n",
    "    return train_generator\n",
    "\n",
    "model = load_model(\"trained_models/earthy-glade-11_val_collection0_map@10\")\n",
    "train_input_generator = train_generator_for_model(model)\n",
    "\n",
    "training_data_used = \"joint_training_batch_05_0.6_0.51_250\"\n",
    "train_collection = TrainSnippetsCollectionV2\\\n",
    "                            .load(training_data_used)\\\n",
    "                            .batch_size(32)\\\n",
    "                            .set_transform_inputs_fn(train_input_generator)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "ename": "ValueError",
     "evalue": "not enough values to unpack (expected 4, got 2)",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mValueError\u001b[0m                                Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-6-87271044d224>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m\u001b[0m\n\u001b[0;32m----> 1\u001b[0;31m \u001b[0mq\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mpos_doc\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mpos_l_snippet\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mneg_doc\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mnext\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mtrain_collection\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mgenerator\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
      "\u001b[0;31mValueError\u001b[0m: not enough values to unpack (expected 4, got 2)"
     ]
    }
   ],
   "source": [
    "q, pos_doc, pos_l_snippet, neg_doc = next(train_collection.generator())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "pos_doc"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "py-bioasq",
   "language": "python",
   "name": "py-bioasq"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.13"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
